<!DOCTYPE html>
<html class="grade-c" lang="en">
<head>
<title>Adaptive partitioning of a gene locus to the nuclear envelope in Saccharomyces cerevisiae is driven by polymer-polymer phase separation | Nature Communications</title>
<link href="https://www.nature.com/ncomms.rss" rel="alternate" type="application/rss+xml"/>
<script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
<link crossorigin="" href="https://cmp.nature.com" rel="preconnect"/>
<meta content="IE=edge" http-equiv="X-UA-Compatible"/>
<meta content="pc,mobile" name="applicable-device"/>
<meta content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes" name="viewport"/>
<meta content="5a2dc4ab3fcb9b0393241ffbbb490480" name="360-site-verification"/>
<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"chromatin-remodelling;gene-expression;gene-regulation","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Communications","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41467-023-36391-6"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Lidice González","Daniel Kolbin","Christian Trahan","Célia Jeronimo","François Robert","Marlene Oeffinger","Kerry Bloom","Stephen W. Michnick"],"publishedAt":1677542400,"publishedAtString":"2023-02-28","title":"Adaptive partitioning of a gene locus to the nuclear envelope in Saccharomyces cerevisiae is driven by polymer-polymer phase separation","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Chromatin remodelling,Gene expression,Gene regulation"},"journal":{"pcode":"ncomms","title":"nature communications","volume":"14","issue":"1","id":41467,"publishingModel":"Open Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>
<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
<style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>
<link data-inline-css-source="critical-css" data-test="critical-css-handler" href="/static/css/enhanced-article-nature-branded-d54f9178e8.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null" rel="stylesheet"/>
<noscript>
<link href="/static/css/enhanced-article-nature-branded-d54f9178e8.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" rel="stylesheet" type="text/css"/>
</noscript>
<link href="/static/css/article-print-122346e276.css" media="print" rel="stylesheet" type="text/css"/>
<link href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png" rel="apple-touch-icon" sizes="180x180"/>
<link href="/static/images/favicons/nature/favicon-48x48-b52890008c.png" rel="icon" sizes="48x48" type="image/png"/>
<link href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png" rel="icon" sizes="32x32" type="image/png"/>
<link href="/static/images/favicons/nature/favicon-16x16-951651ab72.png" rel="icon" sizes="16x16" type="image/png"/>
<link crossorigin="use-credentials" href="/static/manifest.json" rel="manifest"/>
<link color="#000000" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" rel="mask-icon"/>
<link href="/static/images/favicons/nature/favicon.ico" rel="shortcut icon"/>
<meta content="#000000" name="msapplication-TileColor"/>
<meta content="/static/browserconfig.xml" name="msapplication-config"/>
<meta content="#000000" name="theme-color"/>
<meta content="Nature" name="application-name"/>
<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>
<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->
<script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-74.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-74.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>
<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-606cb67187.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-e919764a53.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>
<meta content="noarchive" name="robots"/>
<meta content="Yes" name="access"/>
<link href="https://www.nature.com/search" rel="search"/>
<link href="https://www.nature.com/opensearch/opensearch.xml" rel="search" title="nature.com" type="application/opensearchdescription+xml"/>
<link href="https://www.nature.com/opensearch/request" rel="search" title="nature.com" type="application/sru+xml"/>
<script type="application/ld+json">{"mainEntity":{"headline":"Adaptive partitioning of a gene locus to the nuclear envelope in Saccharomyces cerevisiae is driven by polymer-polymer phase separation","description":"Partitioning of active gene loci to the nuclear envelope (NE) is a mechanism by which organisms increase the speed of adaptation and metabolic robustness to fluctuating resources in the environment. In the yeast Saccharomyces cerevisiae, adaptation to nutrient depletion or other stresses, manifests as relocalization of active gene loci from nucleoplasm to the NE, resulting in more efficient transport and translation of mRNA. The mechanism by which this partitioning occurs remains a mystery. Here, we demonstrate that the yeast inositol depletion-responsive gene locus INO1 partitions to the nuclear envelope, driven by local histone acetylation-induced polymer-polymer phase separation from the nucleoplasmic phase. This demixing is consistent with recent evidence for chromatin phase separation by acetylation-mediated dissolution of multivalent histone association and fits a physical model where increased bending stiffness of acetylated chromatin polymer causes its phase separation from de-acetylated chromatin. Increased chromatin spring stiffness could explain nucleation of transcriptional machinery at active gene loci. Partitioning of active genes to the nuclear periphery is commonly observed but not understood. Here, the authors show that this process occurs by phase separation of transcriptionally active chromatin providing for efficient mRNA transport and memory.","datePublished":"2023-02-28T00:00:00Z","dateModified":"2023-04-21T00:00:00Z","pageStart":"1","pageEnd":"15","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41467-023-36391-6","keywords":["Chromatin remodelling","Gene expression","Gene regulation","Science","Humanities and Social Sciences","multidisciplinary"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig6_HTML.png"],"isPartOf":{"name":"Nature Communications","issn":["2041-1723"],"volumeNumber":"14","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group UK","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Lidice González","affiliation":[{"name":"Université de Montréal","address":{"name":"Département de Biochimie, Université de Montréal, Montréal, Canada","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Daniel Kolbin","affiliation":[{"name":"University of North Carolina at Chapel Hill","address":{"name":"Department of Biology, University of North Carolina at Chapel Hill, Chapel Hill, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Christian Trahan","url":"http://orcid.org/0000-0002-2800-6370","affiliation":[{"name":"Institut de recherches cliniques de Montréal","address":{"name":"Institut de recherches cliniques de Montréal, Montréal, Canada","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Célia Jeronimo","affiliation":[{"name":"Institut de recherches cliniques de Montréal","address":{"name":"Institut de recherches cliniques de Montréal, Montréal, Canada","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"François Robert","url":"http://orcid.org/0000-0003-1054-5920","affiliation":[{"name":"Institut de recherches cliniques de Montréal","address":{"name":"Institut de recherches cliniques de Montréal, Montréal, Canada","@type":"PostalAddress"},"@type":"Organization"},{"name":"McGill University","address":{"name":"Faculty of Medicine, Division of Experimental Medicine, McGill University, Montréal, Canada","@type":"PostalAddress"},"@type":"Organization"},{"name":"Université de Montréal","address":{"name":"Département de Médecine, Faculté de Médecine, Université de Montréal, Montréal, Canada","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Marlene Oeffinger","affiliation":[{"name":"Université de Montréal","address":{"name":"Département de Biochimie, Université de Montréal, Montréal, Canada","@type":"PostalAddress"},"@type":"Organization"},{"name":"Institut de recherches cliniques de Montréal","address":{"name":"Institut de recherches cliniques de Montréal, Montréal, Canada","@type":"PostalAddress"},"@type":"Organization"},{"name":"McGill University","address":{"name":"Faculty of Medicine, Division of Experimental Medicine, McGill University, Montréal, Canada","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Kerry Bloom","affiliation":[{"name":"University of North Carolina at Chapel Hill","address":{"name":"Department of Biology, University of North Carolina at Chapel Hill, Chapel Hill, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Stephen W. Michnick","url":"http://orcid.org/0000-0002-9366-6762","affiliation":[{"name":"Université de Montréal","address":{"name":"Département de Biochimie, Université de Montréal, Montréal, Canada","@type":"PostalAddress"},"@type":"Organization"}],"email":"stephen.michnick@umontreal.ca","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>
<link href="https://www.nature.com/articles/s41467-023-36391-6" rel="canonical"/>
<meta content="41467" name="journal_id"/>
<meta content="Adaptive partitioning of a gene locus to the nuclear envelope in Saccharomyces cerevisiae is driven by polymer-polymer phase separation" name="dc.title"/>
<meta content="Nature Communications 2023 14:1" name="dc.source"/>
<meta content="text/html" name="dc.format"/>
<meta content="Nature Publishing Group" name="dc.publisher"/>
<meta content="2023-02-28" name="dc.date"/>
<meta content="OriginalPaper" name="dc.type"/>
<meta content="En" name="dc.language"/>
<meta content="2023 The Author(s)" name="dc.copyright"/>
<meta content="2023 The Author(s)" name="dc.rights"/>
<meta content="journalpermissions@springernature.com" name="dc.rightsAgent"/>
<meta content="Partitioning of active gene loci to the nuclear envelope (NE) is a mechanism by which organisms increase the speed of adaptation and metabolic robustness to fluctuating resources in the environment. In the yeast Saccharomyces cerevisiae, adaptation to nutrient depletion or other stresses, manifests as relocalization of active gene loci from nucleoplasm to the NE, resulting in more efficient transport and translation of mRNA. The mechanism by which this partitioning occurs remains a mystery. Here, we demonstrate that the yeast inositol depletion-responsive gene locus INO1 partitions to the nuclear envelope, driven by local histone acetylation-induced polymer-polymer phase separation from the nucleoplasmic phase. This demixing is consistent with recent evidence for chromatin phase separation by acetylation-mediated dissolution of multivalent histone association and fits a physical model where increased bending stiffness of acetylated chromatin polymer causes its phase separation from de-acetylated chromatin. Increased chromatin spring stiffness could explain nucleation of transcriptional machinery at active gene loci. Partitioning of active genes to the nuclear periphery is commonly observed but not understood. Here, the authors show that this process occurs by phase separation of transcriptionally active chromatin providing for efficient mRNA transport and memory." name="dc.description"/>
<meta content="2041-1723" name="prism.issn"/>
<meta content="Nature Communications" name="prism.publicationName"/>
<meta content="2023-02-28" name="prism.publicationDate"/>
<meta content="14" name="prism.volume"/>
<meta content="1" name="prism.number"/>
<meta content="OriginalPaper" name="prism.section"/>
<meta content="1" name="prism.startingPage"/>
<meta content="15" name="prism.endingPage"/>
<meta content="2023 The Author(s)" name="prism.copyright"/>
<meta content="journalpermissions@springernature.com" name="prism.rightsAgent"/>
<meta content="https://www.nature.com/articles/s41467-023-36391-6" name="prism.url"/>
<meta content="doi:10.1038/s41467-023-36391-6" name="prism.doi"/>
<meta content="https://www.nature.com/articles/s41467-023-36391-6.pdf" name="citation_pdf_url"/>
<meta content="https://www.nature.com/articles/s41467-023-36391-6" name="citation_fulltext_html_url"/>
<meta content="Nature Communications" name="citation_journal_title"/>
<meta content="Nat Commun" name="citation_journal_abbrev"/>
<meta content="Nature Publishing Group" name="citation_publisher"/>
<meta content="2041-1723" name="citation_issn"/>
<meta content="Adaptive partitioning of a gene locus to the nuclear envelope in Saccharomyces cerevisiae is driven by polymer-polymer phase separation" name="citation_title"/>
<meta content="14" name="citation_volume"/>
<meta content="1" name="citation_issue"/>
<meta content="2023/02/28" name="citation_online_date"/>
<meta content="1" name="citation_firstpage"/>
<meta content="15" name="citation_lastpage"/>
<meta content="Article" name="citation_article_type"/>
<meta content="" name="citation_fulltext_world_readable"/>
<meta content="en" name="citation_language"/>
<meta content="doi:10.1038/s41467-023-36391-6" name="dc.identifier"/>
<meta content="10.1038/s41467-023-36391-6" name="DOI"/>
<meta content="288319" name="size"/>
<meta content="10.1038/s41467-023-36391-6" name="citation_doi"/>
<meta content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41467-023-36391-6&amp;api_key=" name="citation_springer_api_url"/>
<meta content="Partitioning of active gene loci to the nuclear envelope (NE) is a mechanism by which organisms increase the speed of adaptation and metabolic robustness to fluctuating resources in the environment. In the yeast Saccharomyces cerevisiae, adaptation to nutrient depletion or other stresses, manifests as relocalization of active gene loci from nucleoplasm to the NE, resulting in more efficient transport and translation of mRNA. The mechanism by which this partitioning occurs remains a mystery. Here, we demonstrate that the yeast inositol depletion-responsive gene locus INO1 partitions to the nuclear envelope, driven by local histone acetylation-induced polymer-polymer phase separation from the nucleoplasmic phase. This demixing is consistent with recent evidence for chromatin phase separation by acetylation-mediated dissolution of multivalent histone association and fits a physical model where increased bending stiffness of acetylated chromatin polymer causes its phase separation from de-acetylated chromatin. Increased chromatin spring stiffness could explain nucleation of transcriptional machinery at active gene loci. Partitioning of active genes to the nuclear periphery is commonly observed but not understood. Here, the authors show that this process occurs by phase separation of transcriptionally active chromatin providing for efficient mRNA transport and memory." name="description"/>
<meta content="González, Lidice" name="dc.creator"/>
<meta content="Kolbin, Daniel" name="dc.creator"/>
<meta content="Trahan, Christian" name="dc.creator"/>
<meta content="Jeronimo, Célia" name="dc.creator"/>
<meta content="Robert, François" name="dc.creator"/>
<meta content="Oeffinger, Marlene" name="dc.creator"/>
<meta content="Bloom, Kerry" name="dc.creator"/>
<meta content="Michnick, Stephen W." name="dc.creator"/>
<meta content="Chromatin remodelling" name="dc.subject"/>
<meta content="Gene expression" name="dc.subject"/>
<meta content="Gene regulation" name="dc.subject"/>
<meta content="citation_journal_title=PLoS Biol.; citation_title=Gene recruitment of the activated INO1 locus to the nuclear membrane; citation_author=JH Brickner, P Walter; citation_volume=2; citation_publication_date=2004; citation_pages=e342; citation_doi=10.1371/journal.pbio.0020342; citation_id=CR1" name="citation_reference"/>
<meta content="citation_journal_title=Genes Dev.; citation_title=Global histone acetylation induces functional genomic reorganization at mammalian nuclear pore complexes; citation_author=CR Brown, CJ Kennedy, VA Delmar, DJ Forbes, PA Silver; citation_volume=22; citation_publication_date=2008; citation_pages=627-639; citation_doi=10.1101/gad.1632708; citation_id=CR2" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Genome-wide localization of the nuclear transport machinery couples transcriptional status and nuclear organization; citation_author=JM Casolari; citation_volume=117; citation_publication_date=2004; citation_pages=427-439; citation_doi=10.1016/S0092-8674(04)00448-9; citation_id=CR3" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Nucleoporins directly stimulate expression of developmental and cell-cycle genes inside the nucleoplasm; citation_author=B Kalverda, H Pickersgill, VV Shloma, M Fornerod; citation_volume=140; citation_publication_date=2010; citation_pages=360-371; citation_doi=10.1016/j.cell.2010.01.011; citation_id=CR4" name="citation_reference"/>
<meta content="citation_journal_title=J. Cell Biol.; citation_title=Promoter- and RNA polymerase II-dependent hsp-16 gene association with nuclear pores in Caenorhabditis elegans; citation_author=S Rohner; citation_volume=200; citation_publication_date=2013; citation_pages=589-604; citation_doi=10.1083/jcb.201207024; citation_id=CR5" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Subnuclear compartmentalization of immunoglobulin loci during lymphocyte development; citation_author=ST Kosak; citation_volume=296; citation_publication_date=2002; citation_pages=158-162; citation_doi=10.1126/science.1068768; citation_id=CR6" name="citation_reference"/>
<meta content="citation_journal_title=J. Cell Biol.; citation_title=Transcription-dependent spatial arrangements of CFTR and adjacent genes in human cell nuclei; citation_author=D Zink; citation_volume=166; citation_publication_date=2004; citation_pages=815-825; citation_doi=10.1083/jcb.200404107; citation_id=CR7" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=SAGA interacting factors confine sub-diffusion of transcribed genes to the nuclear envelope; citation_author=GG Cabal; citation_volume=441; citation_publication_date=2006; citation_pages=770-773; citation_doi=10.1038/nature04752; citation_id=CR8" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Nuclear pore association confers optimal expression levels for an inducible yeast gene; citation_author=A Taddei; citation_volume=441; citation_publication_date=2006; citation_pages=774-778; citation_doi=10.1038/nature04845; citation_id=CR9" name="citation_reference"/>
<meta content="citation_journal_title=J. Cell Sci.; citation_title=Neural induction promotes large-scale chromatin reorganisation of the Mash1 locus; citation_author=RR Williams; citation_volume=119; citation_publication_date=2006; citation_pages=132-140; citation_doi=10.1242/jcs.02727; citation_id=CR10" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Cell Biol.; citation_title=DNA zip codes control an ancient mechanism for gene targeting to the nuclear periphery; citation_author=S Ahmed; citation_volume=12; citation_publication_date=2010; citation_pages=111-118; citation_doi=10.1038/ncb2011; citation_id=CR11" name="citation_reference"/>
<meta content="citation_journal_title=Dev. Cell; citation_title=Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery; citation_author=DG Brickner; citation_volume=22; citation_publication_date=2012; citation_pages=1234-1246; citation_doi=10.1016/j.devcel.2012.03.012; citation_id=CR12" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Biol. Cell; citation_title=Subnuclear positioning and interchromosomal clustering of the GAL1-10 locus are controlled by separable, interdependent mechanisms; citation_author=DG Brickner; citation_volume=27; citation_publication_date=2016; citation_pages=2980-2993; citation_doi=10.1091/mbc.E16-03-0174; citation_id=CR13" name="citation_reference"/>
<meta content="citation_journal_title=PLoS Biol.; citation_title=H2A.Z-mediated localization of genes at the nuclear periphery confers epigenetic memory of previous transcriptional state; citation_author=DG Brickner; citation_volume=5; citation_publication_date=2007; citation_pages=e81; citation_doi=10.1371/journal.pbio.0050081; citation_id=CR14" name="citation_reference"/>
<meta content="citation_journal_title=J. Cell Biol.; citation_title=Strategies to regulate transcription factor-mediated gene positioning and interchromosomal clustering at the nuclear periphery; citation_author=C Randise-Hinchliff; citation_volume=212; citation_publication_date=2016; citation_pages=633-646; citation_doi=10.1083/jcb.201508068; citation_id=CR15" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=High-resolution statistical mapping reveals gene territories in live yeast; citation_author=AB Berger; citation_volume=5; citation_publication_date=2008; citation_pages=1031-1037; citation_doi=10.1038/nmeth.1266; citation_id=CR16" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Centromere tethering confines chromosome domains; citation_author=JS Verdaasdonk; citation_volume=52; citation_publication_date=2013; citation_pages=819-831; citation_doi=10.1016/j.molcel.2013.10.021; citation_id=CR17" name="citation_reference"/>
<meta content="citation_journal_title=Chromosoma; citation_title=Beyond the code: the mechanical properties of DNA as they relate to mitosis; citation_author=KS Bloom; citation_volume=117; citation_publication_date=2008; citation_pages=103-110; citation_doi=10.1007/s00412-007-0138-0; citation_id=CR18" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Long-range compaction and flexibility of interphase chromatin in budding yeast analyzed by high-resolution imaging techniques; citation_author=K Bystricky, P Heun, L Gehlen, J Langowski, SM Gasser; citation_volume=101; citation_publication_date=2004; citation_pages=16495-16500; citation_doi=10.1073/pnas.0402766101; citation_id=CR19" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Pulling a single chromatin fiber reveals the forces that maintain its higher-order structure; citation_author=Y Cui, C Bustamante; citation_volume=97; citation_publication_date=2000; citation_pages=127-132; citation_doi=10.1073/pnas.97.1.127; citation_id=CR20" name="citation_reference"/>
<meta content="citation_journal_title=Phys. A Stat. Mech. Appl.; citation_title=The persistence length and length per base of single-stranded DNA obtained from fluorescence correlation spectroscopy measurements using mean field theory; citation_author=Q Chi, G Wang, J Jiang; citation_volume=392; citation_publication_date=2013; citation_pages=1072-1079; citation_doi=10.1016/j.physa.2012.09.022; citation_id=CR21" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Phys.; citation_title=Persistence length and scaling properties of single-stranded DNA adsorbed on modified graphite; citation_author=K Rechendorff, G Witz, J Adamcik, G Dietler; citation_volume=131; citation_publication_date=2009; citation_pages=095103; citation_doi=10.1063/1.3216111; citation_id=CR22" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Overstretching B-DNA: the elastic response of individual double-stranded and single-stranded DNA molecules; citation_author=SB Smith, Y Cui, C Bustamante; citation_volume=271; citation_publication_date=1996; citation_pages=795-799; citation_doi=10.1126/science.271.5250.795; citation_id=CR23" name="citation_reference"/>
<meta content="citation_journal_title=Nucleus; citation_title=A role for DNA sequence in controlling the spatial organization of the genome; citation_author=S Ahmed, JH Brickner; citation_volume=1; citation_publication_date=2010; citation_pages=402-406; citation_doi=10.4161/nucl.1.5.12637; citation_id=CR24" name="citation_reference"/>
<meta content="citation_journal_title=Genes Dev.; citation_title=Targeted INO80 enhances subnuclear chromatin movement and ectopic homologous recombination; citation_author=FR Neumann; citation_volume=26; citation_publication_date=2012; citation_pages=369-383; citation_doi=10.1101/gad.176156.111; citation_id=CR25" name="citation_reference"/>
<meta content="citation_journal_title=Phys. Rev. Lett.; citation_title=Brushlike interactions between thermoresponsive microgel particles; citation_author=F Scheffold; citation_volume=104; citation_publication_date=2010; citation_pages=128304; citation_doi=10.1103/PhysRevLett.104.128304; citation_id=CR26" name="citation_reference"/>
<meta content="citation_journal_title=Phys. Rev.; citation_title=On the theory of the brownian motion; citation_author=GE Uhlenbeck, LS Ornstein; citation_volume=36; citation_publication_date=1930; citation_pages=823-841; citation_doi=10.1103/PhysRev.36.823; citation_id=CR27" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Genome-wide replication-independent histone H3 exchange occurs predominantly at promoters and implicates H3 K56 acetylation and Asf1; citation_author=A Rufiange, PE Jacques, W Bhat, F Robert, A Nourani; citation_volume=27; citation_publication_date=2007; citation_pages=393-405; citation_doi=10.1016/j.molcel.2007.07.011; citation_id=CR28" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Yeast Rtt109 promotes genome stability by acetylating histone H3 on lysine 56; citation_author=R Driscoll, A Hudson, SP Jackson; citation_volume=315; citation_publication_date=2007; citation_pages=649-652; citation_doi=10.1126/science.1135862; citation_id=CR29" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Distinct transcriptional roles for Histone H3-K56 acetylation during the cell cycle in Yeast; citation_author=S Topal, P Vasseur, M Radman-Livaja, CL Peterson; citation_volume=10; citation_publication_date=2019; citation_doi=10.1038/s41467-019-12400-5; citation_id=CR30" name="citation_reference"/>
<meta content="citation_journal_title=J. Biol. Chem.; citation_title=Regulation of histone H3 lysine 56 acetylation in Schizosaccharomyces pombe; citation_author=B Xhemalce; citation_volume=282; citation_publication_date=2007; citation_pages=15040-15047; citation_doi=10.1074/jbc.M701197200; citation_id=CR31" name="citation_reference"/>
<meta content="citation_journal_title=Yeast; citation_title=An expanded tool kit for the auxin-inducible degron system in budding yeast; citation_author=M Morawska, HD Ulrich; citation_volume=30; citation_publication_date=2013; citation_pages=341-351; citation_doi=10.1002/yea.2967; citation_id=CR32" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Snf1-a histone kinase that works in concert with the histone acetyltransferase Gcn5 to regulate transcription; citation_author=W-S Lo; citation_volume=293; citation_publication_date=2001; citation_pages=1142-1146; citation_doi=10.1126/science.1062322; citation_id=CR33" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Rtt109 acetylates histone H3 lysine 56 and functions in DNA replication; citation_author=J Han; citation_volume=315; citation_publication_date=2007; citation_pages=653-655; citation_doi=10.1126/science.1133234; citation_id=CR34" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=The TAF(II)250 subunit of TFIID has histone acetyltransferase activity; citation_author=CA Mizzen; citation_volume=87; citation_publication_date=1996; citation_pages=1261-1270; citation_doi=10.1016/S0092-8674(00)81821-8; citation_id=CR35" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell. Biol.; citation_title=Genome-wide relationships between TAF1 and histone acetyltransferases in Saccharomyces cerevisiae; citation_author=M Durant, BF Pugh; citation_volume=26; citation_publication_date=2006; citation_pages=2791-2802; citation_doi=10.1128/MCB.26.7.2791-2802.2006; citation_id=CR36" name="citation_reference"/>
<meta content="citation_journal_title=J. Biol. Chem.; citation_title=Histone H3 lysine 14 (H3K14) acetylation facilitates DNA repair in a positioned nucleosome by stabilizing the binding of the chromatin Remodeler RSC (Remodels Structure of Chromatin); citation_author=MR Duan, MJ Smerdon; citation_volume=289; citation_publication_date=2014; citation_pages=8353-8363; citation_doi=10.1074/jbc.M113.540732; citation_id=CR37" name="citation_reference"/>
<meta content="citation_journal_title=Curr. Opin. Cell Biol.; citation_title=DNA repair and the role of chromatin structure; citation_author=MJ Smerdon; citation_volume=3; citation_publication_date=1991; citation_pages=422-428; citation_doi=10.1016/0955-0674(91)90069-B; citation_id=CR38" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Gcn4 activator targets Gcn5 histone acetyltransferase to specific promoters independently of transcription; citation_author=MH Kuo, E vom Baur, K Struhl, CD Allis; citation_volume=6; citation_publication_date=2000; citation_pages=1309-1320; citation_doi=10.1016/S1097-2765(00)00129-5; citation_id=CR39" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Transcription-linked acetylation by Gcn5p of histones H3 and H4 at specific lysines; citation_author=MH Kuo; citation_volume=383; citation_publication_date=1996; citation_pages=269-272; citation_doi=10.1038/383269a0; citation_id=CR40" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Struct. Mol. Biol.; citation_title=Molecular functions of the histone acetyltransferase chaperone complex Rtt109-Vps75; citation_author=CE Berndsen; citation_volume=15; citation_publication_date=2008; citation_pages=948-956; citation_doi=10.1038/nsmb.1459; citation_id=CR41" name="citation_reference"/>
<meta content="citation_journal_title=J. Proteom.; citation_title=Chaperone-mediated acetylation of histones by Rtt109 identified by quantitative proteomics; citation_author=N Abshiru; citation_volume=81; citation_publication_date=2013; citation_pages=80-90; citation_doi=10.1016/j.jprot.2012.09.026; citation_id=CR42" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell. Proteom. MCP; citation_title=Capturing the Asc1p/receptor for activated C kinase 1 (RACK1) microenvironment at the head region of the 40 S ribosome with quantitative BioID in yeast; citation_author=N Opitz; citation_volume=16; citation_publication_date=2017; citation_pages=2199-2218; citation_doi=10.1074/mcp.M116.066654; citation_id=CR43" name="citation_reference"/>
<meta content="citation_journal_title=Nucleus; citation_title=Determination of local chromatin composition by CasID; citation_author=E Schmidtmann, T Anton, P Rombaut, F Herzog, H Leonhardt; citation_volume=7; citation_publication_date=2016; citation_pages=476-484; citation_doi=10.1080/19491034.2016.1239000; citation_id=CR44" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=A chromatin remodelling complex involved in transcription and DNA processing; citation_author=X Shen, G Mizuguchi, A Hamiche, C Wu; citation_volume=406; citation_publication_date=2000; citation_pages=541-544; citation_doi=10.1038/35020123; citation_id=CR45" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Molecular architecture of the ATP-dependent chromatin-remodeling complex SWR1; citation_author=VQ Nguyen; citation_volume=154; citation_publication_date=2013; citation_pages=1220-1231; citation_doi=10.1016/j.cell.2013.08.018; citation_id=CR46" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Syst. Biol.; citation_title=Combinatorial depletion analysis to assemble the network architecture of the SAGA and ADA chromatin remodeling complexes; citation_author=KK Lee; citation_volume=7; citation_publication_date=2011; citation_pages=503; citation_doi=10.1038/msb.2011.40; citation_id=CR47" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell. Biol.; citation_title=Proteomics of the eukaryotic transcription machinery: identification of proteins associated with components of yeast TFIID by multidimensional mass spectrometry; citation_author=SL Sanders, J Jennings, A Canutescu, AJ Link, PA Weil; citation_volume=22; citation_publication_date=2002; citation_pages=4723-4738; citation_doi=10.1128/MCB.22.13.4723-4738.2002; citation_id=CR48" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell. Proteom. MCP; citation_title=Proteomic and genomic analyses of the Rvb1 and Rvb2 interaction network upon deletion of R2TP complex components; citation_author=M Lakshminarasimhan; citation_volume=15; citation_publication_date=2016; citation_pages=960-974; citation_doi=10.1074/mcp.M115.053165; citation_id=CR49" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=FACT interacts with Set3 HDAC and fine-tunes GAL1 transcription in response to environmental stimulation; citation_author=H Leng; citation_volume=49; citation_publication_date=2021; citation_pages=5502-5519; citation_doi=10.1093/nar/gkab312; citation_id=CR50" name="citation_reference"/>
<meta content="citation_journal_title=Cell Rep.; citation_title=The histone chaperone FACT contributes to DNA replication-coupled nucleosome assembly; citation_author=J Yang; citation_volume=14; citation_publication_date=2016; citation_pages=1128-1141; citation_doi=10.1016/j.celrep.2015.12.096; citation_id=CR51" name="citation_reference"/>
<meta content="citation_journal_title=Macromol. Theory Simul.; citation_title=Intramolecular phase separation of a copolymer chain with mobile primary structure; citation_author=EE Dormidontova, AY Grosberg, AR Khokhlov; citation_volume=1; citation_publication_date=1992; citation_pages=375-385; citation_doi=10.1002/mats.1992.040010603; citation_id=CR52" name="citation_reference"/>
<meta content="citation_journal_title=Proc. R. Soc. Lond. Ser. A. Math. Phys. Sci.; citation_title=Statistical thermodynamics of semi-flexible chain molecules; citation_author=PJ Flory, G Gee; citation_volume=234; citation_publication_date=1956; citation_pages=60-73; citation_id=CR53" name="citation_reference"/>
<meta content="citation_journal_title=Ann. N. Y. Acad. Sci.; citation_title=Thermodynamic properties of solutions of long-chain compounds; citation_author=ML Huggins; citation_volume=43; citation_publication_date=1942; citation_pages=1-32; citation_doi=10.1111/j.1749-6632.1942.tb47940.x; citation_id=CR54" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=Enrichment of dynamic chromosomal crosslinks drive phase separation of the nucleolus; citation_author=C Hult; citation_volume=45; citation_publication_date=2017; citation_pages=11159-11173; citation_doi=10.1093/nar/gkx741; citation_id=CR55" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Organization of chromatin by intrinsic and regulated phase separation; citation_author=BA Gibson; citation_volume=179; citation_publication_date=2019; citation_pages=470-484.e421; citation_doi=10.1016/j.cell.2019.08.037; citation_id=CR56" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=HP1 reshapes nucleosome core to promote heterochromatin phase separation; citation_author=S Sanulli; citation_volume=575; citation_publication_date=2019; citation_pages=390-394; citation_doi=10.1038/s41586-019-1669-2; citation_id=CR57" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Chromosome dynamics in the yeast interphase nucleus; citation_author=P Heun, T Laroche, K Shimada, P Furrer, SM Gasser; citation_volume=294; citation_publication_date=2001; citation_pages=2181-2186; citation_doi=10.1126/science.1065366; citation_id=CR58" name="citation_reference"/>
<meta content="citation_journal_title=Curr. Opin. Genet. Dev.; citation_title=Chromosome dynamics during interphase: a biophysical perspective; citation_author=MM Tortora, H Salari, D Jost; citation_volume=61; citation_publication_date=2020; citation_pages=37-43; citation_doi=10.1016/j.gde.2020.03.001; citation_id=CR59" name="citation_reference"/>
<meta content="citation_journal_title=Biophysical J.; citation_title=Formation of chromatin subcompartments by phase separation; citation_author=F Erdel, K Rippe; citation_volume=114; citation_publication_date=2018; citation_pages=2262-2270; citation_doi=10.1016/j.bpj.2018.03.011; citation_id=CR60" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Liquid nuclear condensates mechanically sense and restructure the genome; citation_author=Y Shin; citation_volume=175; citation_publication_date=2018; citation_pages=1481-1491.e1413; citation_doi=10.1016/j.cell.2018.10.057; citation_id=CR61" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Phys.; citation_title=Force generation by protein–DNA co-condensation; citation_author=T Quail; citation_volume=17; citation_publication_date=2021; citation_pages=1007-1012; citation_doi=10.1038/s41567-021-01285-1; citation_id=CR62" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=The rDNA is biomolecular condensate formed by polymer-polymer phase separation and is sequestered in the nucleolus by transcription and R-loops; citation_author=J Lawrimore; citation_volume=49; citation_publication_date=2021; citation_pages=4586-4598; citation_doi=10.1093/nar/gkab229; citation_id=CR63" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=RNA-mediated feedback control of transcriptional condensates; citation_author=JE Henninger; citation_volume=184; citation_publication_date=2021; citation_pages=207-225 e224; citation_doi=10.1016/j.cell.2020.11.030; citation_id=CR64" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Coactivator condensation at super-enhancers links phase separation and gene control; citation_author=BR Sabari; citation_volume=361; citation_publication_date=2018; citation_pages=eaar3958; citation_doi=10.1126/science.aar3958; citation_id=CR65" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Extended and dynamic linker histone-DNA Interactions control chromatosome compaction; citation_author=S Rudnizky; citation_volume=81; citation_publication_date=2021; citation_pages=3410-3421 e3414; citation_doi=10.1016/j.molcel.2021.06.006; citation_id=CR66" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Phase separation drives heterochromatin domain formation; citation_author=AR Strom; citation_volume=547; citation_publication_date=2017; citation_pages=241-245; citation_doi=10.1038/nature22989; citation_id=CR67" name="citation_reference"/>
<meta content="citation_journal_title=Cold Spring Harb. Perspect. Biol.; citation_title=Essential roles for RNA in shaping nuclear organization; citation_author=SA Quinodoz, M Guttman; citation_volume=14; citation_publication_date=2021; citation_pages=a039719; citation_doi=10.1101/cshperspect.a039719; citation_id=CR68" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Cell Biol.; citation_title=Functional transcription promoters at DNA double-strand breaks mediate RNA-driven phase separation of damage-response factors; citation_author=F Pessina; citation_volume=21; citation_publication_date=2019; citation_pages=1286-1299; citation_doi=10.1038/s41556-019-0392-4; citation_id=CR69" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Entropy-driven spatial organization of highly confined polymers: lessons for the bacterial chromosome; citation_author=S Jun, B Mulder; citation_volume=103; citation_publication_date=2006; citation_pages=12388-12393; citation_doi=10.1073/pnas.0605305103; citation_id=CR70" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Engineering a model cell for rational tuning of GPCR signaling; citation_author=WM Shaw; citation_volume=177; citation_publication_date=2019; citation_pages=782-796.e727; citation_doi=10.1016/j.cell.2019.02.023; citation_id=CR71" name="citation_reference"/>
<meta content="citation_journal_title=ACS Synth. Biol.; citation_title=A highly characterized yeast toolkit for modular, multipart assembly; citation_author=ME Lee, WC DeLoache, B Cervantes, JE Dueber; citation_volume=4; citation_publication_date=2015; citation_pages=975-986; citation_doi=10.1021/sb500366v; citation_id=CR72" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=The scaffold protein Ste5 directly controls a switch-like mating decision in yeast; citation_author=MK Malleshaiah, V Shahrezaei, PS Swain, SW Michnick; citation_volume=465; citation_publication_date=2010; citation_pages=101-105; citation_doi=10.1038/nature08946; citation_id=CR73" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Chromosome arm length and nuclear constraints determine the dynamic relationship of yeast subtelomeres; citation_author=P Therizols, T Duong, B Dujon, C Zimmer, E Fabre; citation_volume=107; citation_publication_date=2010; citation_pages=2025-2030; citation_doi=10.1073/pnas.0914187107; citation_id=CR74" name="citation_reference"/>
<meta content="citation_journal_title=PLoS One; citation_title=Real-time analysis and visualization for single-molecule based super-resolution microscopy; citation_author=A Kechkar, D Nair, M Heilemann, D Choquet, JB Sibarita; citation_volume=8; citation_publication_date=2013; citation_pages=e62918; citation_doi=10.1371/journal.pone.0062918; citation_id=CR75" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=Fiji: an open-source platform for biological-image analysis; citation_author=J Schindelin; citation_volume=9; citation_publication_date=2012; citation_pages=676-682; citation_doi=10.1038/nmeth.2019; citation_id=CR76" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Biol. Cell; citation_title=ChromoShake: a chromosome dynamics simulator reveals that chromatin loops stiffen centromeric chromatin; citation_author=J Lawrimore; citation_volume=27; citation_publication_date=2016; citation_pages=153-166; citation_doi=10.1091/mbc.E15-08-0575; citation_id=CR77" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=DNA relaxation dynamics as a probe for the intracellular environment; citation_author=JK Fisher; citation_volume=106; citation_publication_date=2009; citation_pages=9250-9255; citation_doi=10.1073/pnas.0812723106; citation_id=CR78" name="citation_reference"/>
<meta content="citation_journal_title=Cell Rep.; citation_title=Histone recycling by FACT and Spt6 during transcription prevents the scrambling of histone modifications; citation_author=C Jeronimo, C Poitras, F Robert; citation_volume=28; citation_publication_date=2019; citation_pages=1206-1218.e1208; citation_doi=10.1016/j.celrep.2019.06.097; citation_id=CR79" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=Comprehensive analysis of diverse ribonucleoprotein complexes; citation_author=M Oeffinger; citation_volume=4; citation_publication_date=2007; citation_pages=951-956; citation_doi=10.1038/nmeth1101; citation_id=CR80" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=Saccharomyces Genome Database: the genomics resource of budding yeast; citation_author=JM Cherry; citation_volume=40; citation_publication_date=2012; citation_pages=D700-D705; citation_doi=10.1093/nar/gkr1029; citation_id=CR81" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=UniProt: the universal protein knowledgebase in 2021; citation_author=; citation_volume=49; citation_publication_date=2021; citation_pages=D480-D489; citation_doi=10.1093/nar/gkaa1100; citation_id=CR82" name="citation_reference"/>
<meta content="citation_journal_title=Protein Sci.; citation_title=The BioGRID database: a comprehensive biomedical resource of curated protein, genetic, and chemical interactions; citation_author=R Oughtred; citation_volume=30; citation_publication_date=2021; citation_pages=187-200; citation_doi=10.1002/pro.3978; citation_id=CR83" name="citation_reference"/>
<meta content="González, Lidice" name="citation_author"/>
<meta content="Département de Biochimie, Université de Montréal, Montréal, Canada" name="citation_author_institution"/>
<meta content="Kolbin, Daniel" name="citation_author"/>
<meta content="Department of Biology, University of North Carolina at Chapel Hill, Chapel Hill, USA" name="citation_author_institution"/>
<meta content="Trahan, Christian" name="citation_author"/>
<meta content="Institut de recherches cliniques de Montréal, Montréal, Canada" name="citation_author_institution"/>
<meta content="Jeronimo, Célia" name="citation_author"/>
<meta content="Institut de recherches cliniques de Montréal, Montréal, Canada" name="citation_author_institution"/>
<meta content="Robert, François" name="citation_author"/>
<meta content="Institut de recherches cliniques de Montréal, Montréal, Canada" name="citation_author_institution"/>
<meta content="Faculty of Medicine, Division of Experimental Medicine, McGill University, Montréal, Canada" name="citation_author_institution"/>
<meta content="Département de Médecine, Faculté de Médecine, Université de Montréal, Montréal, Canada" name="citation_author_institution"/>
<meta content="Oeffinger, Marlene" name="citation_author"/>
<meta content="Département de Biochimie, Université de Montréal, Montréal, Canada" name="citation_author_institution"/>
<meta content="Institut de recherches cliniques de Montréal, Montréal, Canada" name="citation_author_institution"/>
<meta content="Faculty of Medicine, Division of Experimental Medicine, McGill University, Montréal, Canada" name="citation_author_institution"/>
<meta content="Bloom, Kerry" name="citation_author"/>
<meta content="Department of Biology, University of North Carolina at Chapel Hill, Chapel Hill, USA" name="citation_author_institution"/>
<meta content="Michnick, Stephen W." name="citation_author"/>
<meta content="Département de Biochimie, Université de Montréal, Montréal, Canada" name="citation_author_institution"/>
<meta content="https://www.nature.com/platform/readcube-access" name="access_endpoint"/>
<meta content="@NatureComms" name="twitter:site"/>
<meta content="summary_large_image" name="twitter:card"/>
<meta content="Content cover image" name="twitter:image:alt"/>
<meta content="Adaptive partitioning of a gene locus to the nuclear envelope in Saccharomyces cerevisiae is driven by polymer-polymer phase separation" name="twitter:title"/>
<meta content="Nature Communications - Partitioning of active genes to the nuclear periphery is commonly observed but not understood. Here, the authors show that this process occurs by phase separation of..." name="twitter:description"/>
<meta content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig1_HTML.png" name="twitter:image"/>
<meta content="https://www.nature.com/articles/s41467-023-36391-6" property="og:url"/>
<meta content="article" property="og:type"/>
<meta content="Nature" property="og:site_name"/>
<meta content="Adaptive partitioning of a gene locus to the nuclear envelope in Saccharomyces cerevisiae is driven by polymer-polymer phase separation - Nature Communications" property="og:title"/>
<meta content="Partitioning of active genes to the nuclear periphery is commonly observed but not understood. Here, the authors show that this process occurs by phase separation of transcriptionally active chromatin providing for efficient mRNA transport and memory." property="og:description"/>
<meta content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig1_HTML.png" property="og:image"/>
<script>
        window.eligibleForRa21 = 'false'; 
    </script>
</head>
<body class="article-page">
<noscript><iframe height="0" src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ" style="display:none;visibility:hidden" width="0"></iframe></noscript>
<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
<a class="c-skip-link" href="#content">Skip to main content</a>
<div class="c-grade-c-banner u-hide">
<div class="c-grade-c-banner__container">
<p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>
</div>
</div>
<div class="u-hide u-show-following-ad"></div>
<aside class="c-ad c-ad--728x90">
<div class="c-ad__inner" data-container-type="banner-advert">
<p class="c-ad__label">Advertisement</p>
<div class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-gpt="" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41467-023-36391-6;doi=10.1038/s41467-023-36391-6;techmeta=14,15,19,38,45,58,63,82,88,90;subjmeta=100,102,199,200,208,337,631;kwrd=Chromatin+remodelling,Gene+expression,Gene+regulation" data-gpt-unitpath="/285/nature_communications/article" data-pa11y-ignore="" data-test="top-ad" id="div-gpt-ad-top-1">
<noscript>
<a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature_communications/article&amp;sz=728x90&amp;c=-569079819&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41467-023-36391-6%26doi%3D10.1038/s41467-023-36391-6%26techmeta%3D14,15,19,38,45,58,63,82,88,90%26subjmeta%3D100,102,199,200,208,337,631%26kwrd%3DChromatin+remodelling,Gene+expression,Gene+regulation">
<img alt="Advertisement" data-test="gpt-advert-fallback-img" height="90" src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature_communications/article&amp;sz=728x90&amp;c=-569079819&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41467-023-36391-6%26doi%3D10.1038/s41467-023-36391-6%26techmeta%3D14,15,19,38,45,58,63,82,88,90%26subjmeta%3D100,102,199,200,208,337,631%26kwrd%3DChromatin+remodelling,Gene+expression,Gene+regulation" width="728"/></a>
</noscript>
</div>
</div>
</aside>
<header class="c-header" data-header="" data-track-component="nature-150-split-header" id="header" style="border-color:#e63323">
<div class="c-header__row">
<div class="c-header__container">
<div class="c-header__split">
<div class="c-header__logo-container">
<a data-track="click" data-track-action="home" data-track-label="image" href="/ncomms">
<picture class="c-header__logo">
<source media="(min-width: 875px)" srcset="https://media.springernature.com/full/nature-cms/uploads/product/ncomms/header-03d2e325c0a02f6df509e5730e9be304.svg">
<img alt="Nature Communications" height="32" src="https://media.springernature.com/full/nature-cms/uploads/product/ncomms/header-7001f06bc3fe2437048388e9f2f44215.svg"/>
</source></picture>
</a>
</div>
<ul class="c-header__menu c-header__menu--global">
<li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
<a class="c-header__link" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link" href="https://www.nature.com/siteindex">
<span>View all journals</span>
</a>
</li>
<li class="c-header__item c-header__item--padding c-header__item--pipe">
<a class="c-header__link c-header__link--search" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" href="#search-menu">
<svg aria-hidden="true" focusable="false" height="22" role="img" viewbox="0 0 18 18" width="22" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
</a>
</li>
<li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
<a class="c-header__link eds-c-header__link" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41467-023-36391-6" id="identity-account-widget"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
</li>
</ul>
</div>
</div>
</div>
<div class="c-header__row">
<div class="c-header__container" data-test="navigation-row">
<div class="c-header__split">
<ul class="c-header__menu c-header__menu--journal">
<li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
<a class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" href="#explore">
<span><span class="c-header__show-text">Explore</span> content</span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--dropdown-menu">
<a class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" href="#about-the-journal">
<span>About <span class="c-header__show-text">the journal</span></span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
<a class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" href="#publish-with-us">
<span>Publish <span class="c-header__show-text">with us</span></span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
</ul>
<ul class="c-header__menu c-header__menu--hide-lg-max">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (desktop site header)" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D264%26journal-link%3Dhttps%253A%252F%252Fwww.nature.com%252Fncomms%252F" rel="nofollow">
<span>Sign up for alerts</span><svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--pipe">
<a class="c-header__link" data-track="click" data-track-action="rss feed" data-track-label="link" href="https://www.nature.com/ncomms.rss">
<span>RSS feed</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</header>
<nav aria-label="breadcrumbs" class="u-mb-16">
<div class="u-container">
<ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
<li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature" href="/" itemprop="item"><span itemprop="name">nature</span></a><meta content="1" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature communications" href="/ncomms" itemprop="item"><span itemprop="name">nature communications</span></a><meta content="2" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles" href="/ncomms/articles?type=article" itemprop="item"><span itemprop="name">articles</span></a><meta content="3" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
<span itemprop="name">article</span><meta content="4" itemprop="position"/></li>
</ol>
</div>
</nav>
</div>
<div class="u-container u-mt-32 u-mb-32 u-clearfix" data-component="article-container" data-container-type="article" id="content">
<main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
<div aria-hidden="true" class="c-context-bar u-hide" data-context-bar="" data-test="context-bar">
<div class="c-context-bar__container u-container" data-track-context="sticky banner">
<div class="c-context-bar__title">
                            Adaptive partitioning of a gene locus to the nuclear envelope in <i>Saccharomyces cerevisiae</i> is driven by polymer-polymer phase separation
                        </div>
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41467-023-36391-6.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
</div>
<article lang="en">
<div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
<div class="c-pdf-container" data-track-context="article body">
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41467-023-36391-6.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
</div>
<div class="c-article-header">
<header>
<ul class="c-article-identifiers" data-test="article-identifier">
<li class="c-article-identifiers__item" data-test="article-category">Article</li>
<li class="c-article-identifiers__item">
<a class="u-color-open-access" data-test="open-access" data-track="click" data-track-action="open access" data-track-label="link" href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research">Open access</a>
</li>
<li class="c-article-identifiers__item">Published: <time datetime="2023-02-28">28 February 2023</time></li>
</ul>
<h1 class="c-article-title" data-article-title="" data-test="article-title">Adaptive partitioning of a gene locus to the nuclear envelope in <i>Saccharomyces cerevisiae</i> is driven by polymer-polymer phase separation</h1>
<ul class="c-article-author-list c-article-author-list--short" data-component-authors-activator="authors-list" data-test="authors-list"><li class="c-article-author-list__item"><a data-author-popup="auth-Lidice-Gonz_lez-Aff1" data-author-search="González, Lidice" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Lidice-Gonz_lez-Aff1">Lidice González</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-author-popup="auth-Daniel-Kolbin-Aff2" data-author-search="Kolbin, Daniel" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Daniel-Kolbin-Aff2">Daniel Kolbin</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Christian-Trahan-Aff3" data-author-search="Trahan, Christian" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Christian-Trahan-Aff3">Christian Trahan</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-2800-6370"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2800-6370</a></span><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-C_lia-Jeronimo-Aff3" data-author-search="Jeronimo, Célia" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-C_lia-Jeronimo-Aff3">Célia Jeronimo</a><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Fran_ois-Robert-Aff3-Aff4-Aff5" data-author-search="Robert, François" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Fran_ois-Robert-Aff3-Aff4-Aff5">François Robert</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-1054-5920"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1054-5920</a></span><sup class="u-js-hide"><a href="#Aff3">3</a>,<a href="#Aff4">4</a>,<a href="#Aff5">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Marlene-Oeffinger-Aff1-Aff3-Aff4" data-author-search="Oeffinger, Marlene" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Marlene-Oeffinger-Aff1-Aff3-Aff4">Marlene Oeffinger</a><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff3">3</a>,<a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Kerry-Bloom-Aff2" data-author-search="Bloom, Kerry" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Kerry-Bloom-Aff2">Kerry Bloom</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup> &amp; </li><li aria-label="Show all 8 authors for this article" class="c-article-author-list__show-more" title="Show all 8 authors for this article">…</li><li class="c-article-author-list__item"><a data-author-popup="auth-Stephen_W_-Michnick-Aff1" data-author-search="Michnick, Stephen W." data-corresp-id="c1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Stephen_W_-Michnick-Aff1">Stephen W. Michnick<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-mail-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-9366-6762"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-9366-6762</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup> </li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-down-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg><span>Show authors</span></button>
<p class="c-article-info-details" data-container-section="info">
<a data-test="journal-link" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link" href="/ncomms"><i data-test="journal-title">Nature Communications</i></a>
<b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 14</b>, Article number: <span data-test="article-number">1135</span> (<span data-test="article-publication-year">2023</span>)
            <a class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link" href="#citeas">Cite this article</a>
</p>
<div class="c-article-metrics-bar__wrapper u-clear-both">
<ul class="c-article-metrics-bar u-list-reset">
<li class="c-article-metrics-bar__item" data-test="access-count">
<p class="c-article-metrics-bar__count">3036 <span class="c-article-metrics-bar__label">Accesses</span></p>
</li>
<li class="c-article-metrics-bar__item" data-test="citation-count">
<p class="c-article-metrics-bar__count">3 <span class="c-article-metrics-bar__label">Citations</span></p>
</li>
<li class="c-article-metrics-bar__item" data-test="altmetric-score">
<p class="c-article-metrics-bar__count">2 <span class="c-article-metrics-bar__label">Altmetric</span></p>
</li>
<li class="c-article-metrics-bar__item">
<p class="c-article-metrics-bar__details"><a data-track="click" data-track-action="view metrics" data-track-label="link" href="/articles/s41467-023-36391-6/metrics" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
</li>
</ul>
</div>
</header>
<div class="u-js-hide" data-component="article-subject-links">
<h3 class="c-article__sub-heading">Subjects</h3>
<ul class="c-article-subject-list">
<li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/chromatin-remodelling">Chromatin remodelling</a></li><li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/gene-expression">Gene expression</a></li><li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/gene-regulation">Gene regulation</a></li>
</ul>
</div>
<div class="u-mb-8 c-status-message c-status-message--boxed c-status-message--info">
<span class="c-status-message__icon">
<svg aria-hidden="true" class="u-icon" focusable="false" height="18" width="18">
<use xlink:href="#icon-eds-i-info-filled-medium"></use>
</svg>
</span>
<p class="u-mt-0">A <a class="relation-link" data-track="click" data-track-action="view linked article" data-track-label="link" href="https://doi.org/10.1038/s41467-023-38076-6">Publisher Correction</a> to this article was published on 21 April 2023</p>
</div>
<div class="u-mb-8 c-status-message c-status-message--boxed c-status-message--info">
<span class="c-status-message__icon">
<svg aria-hidden="true" class="u-icon" focusable="false" height="18" width="18">
<use xlink:href="#icon-eds-i-info-filled-medium"></use>
</svg>
</span>
<p class="u-mt-0">This article has been <a href="#change-history">updated</a></p>
</div>
</div>
<div class="c-article-body">
<section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Partitioning of active gene loci to the nuclear envelope (NE) is a mechanism by which organisms increase the speed of adaptation and metabolic robustness to fluctuating resources in the environment. In the yeast <i>Saccharomyces cerevisiae</i>, adaptation to nutrient depletion or other stresses, manifests as relocalization of active gene loci from nucleoplasm to the NE, resulting in more efficient transport and translation of mRNA. The mechanism by which this partitioning occurs remains a mystery. Here, we demonstrate that the yeast inositol depletion-responsive gene locus <i>INO1</i> partitions to the nuclear envelope, driven by local histone acetylation-induced polymer-polymer phase separation from the nucleoplasmic phase. This demixing is consistent with recent evidence for chromatin phase separation by acetylation-mediated dissolution of multivalent histone association and fits a physical model where increased bending stiffness of acetylated chromatin polymer causes its phase separation from de-acetylated chromatin. Increased chromatin spring stiffness could explain nucleation of transcriptional machinery at active gene loci.</p></div></div></section>
<section aria-labelledby="inline-recommendations" class="c-article-recommendations" data-title="Inline Recommendations" data-track-component="inline-recommendations">
<h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
<div class="c-article-recommendations-list">
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41594-024-01318-2/MediaObjects/41594_2024_1318_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_1" data-track-action="click recommendations inline - 1" data-track-context="inline recommendations" data-track-label="10.1038/s41594-024-01318-2" href="https://www.nature.com/articles/s41594-024-01318-2?fromPaywallRec=false" itemprop="url">The yeast genome is globally accessible in living cells
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__access-type">Open access</span>
<span class="c-article-meta-recommendations__date">25 November 2024</span>
</div>
</div>
</article>
</div>
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-023-43322-y/MediaObjects/41467_2023_43322_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_2" data-track-action="click recommendations inline - 2" data-track-context="inline recommendations" data-track-label="10.1038/s41467-023-43322-y" href="https://www.nature.com/articles/s41467-023-43322-y?fromPaywallRec=false" itemprop="url">The environmentally-regulated interplay between local three-dimensional chromatin organisation and transcription of <i>proVWX</i> in <i>E. coli</i>
</a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__access-type">Open access</span>
<span class="c-article-meta-recommendations__date">17 November 2023</span>
</div>
</div>
</article>
</div>
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41594-022-00736-4/MediaObjects/41594_2022_736_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_3" data-track-action="click recommendations inline - 3" data-track-context="inline recommendations" data-track-label="10.1038/s41594-022-00736-4" href="https://www.nature.com/articles/s41594-022-00736-4?fromPaywallRec=false" itemprop="url">Acetylation-dependent SAGA complex dimerization promotes nucleosome acetylation and gene transcription
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__date">17 March 2022</span>
</div>
</div>
</article>
</div>
</div>
</section>
<script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1740770291,
                        embedded_user: 'null'
                    }
                });
            </script>
<div class="main-content">
<section data-title="Introduction"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Introduction</h2><div class="c-article-section__content" id="Sec1-content"><p>Genes relocalize to specific regions inside the nucleus depending on whether they are active or repressed, including the nuclear periphery or the periphery of their chromosomal territories, a phenomenon observed in many different eukaryotes from budding yeast to mammalian cells<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR1" id="ref-link-section-d8519934e599" title="Brickner, J. H. &amp; Walter, P. Gene recruitment of the activated INO1 locus to the nuclear membrane. PLoS Biol. 2, e342 (2004).">1</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR2" id="ref-link-section-d8519934e599_1" title="Brown, C. R., Kennedy, C. J., Delmar, V. A., Forbes, D. J. &amp; Silver, P. A. Global histone acetylation induces functional genomic reorganization at mammalian nuclear pore complexes. Genes Dev. 22, 627–639 (2008).">2</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR3" id="ref-link-section-d8519934e599_2" title="Casolari, J. M. et al. Genome-wide localization of the nuclear transport machinery couples transcriptional status and nuclear organization. Cell 117, 427–439 (2004).">3</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR4" id="ref-link-section-d8519934e599_3" title="Kalverda, B., Pickersgill, H., Shloma, V. V. &amp; Fornerod, M. Nucleoporins directly stimulate expression of developmental and cell-cycle genes inside the nucleoplasm. Cell 140, 360–371 (2010).">4</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR5" id="ref-link-section-d8519934e599_4" title="Rohner, S. et al. Promoter- and RNA polymerase II-dependent hsp-16 gene association with nuclear pores in Caenorhabditis elegans. J. Cell Biol. 200, 589–604 (2013).">5</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR6" id="ref-link-section-d8519934e599_5" title="Kosak, S. T. et al. Subnuclear compartmentalization of immunoglobulin loci during lymphocyte development. Science 296, 158–162 (2002).">6</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR7" id="ref-link-section-d8519934e599_6" title="Zink, D. et al. Transcription-dependent spatial arrangements of CFTR and adjacent genes in human cell nuclei. J. Cell Biol. 166, 815–825 (2004).">7</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR8" id="ref-link-section-d8519934e599_7" title="Cabal, G. G. et al. SAGA interacting factors confine sub-diffusion of transcribed genes to the nuclear envelope. Nature 441, 770–773 (2006).">8</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR9" id="ref-link-section-d8519934e599_8" title="Taddei, A. et al. Nuclear pore association confers optimal expression levels for an inducible yeast gene. Nature 441, 774–778 (2006).">9</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR10" id="ref-link-section-d8519934e599_9" title="Williams, R. R. et al. Neural induction promotes large-scale chromatin reorganisation of the Mash1 locus. J. Cell Sci. 119, 132–140 (2006).">10</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR11" id="ref-link-section-d8519934e599_10" title="Ahmed, S. et al. DNA zip codes control an ancient mechanism for gene targeting to the nuclear periphery. Nat. Cell Biol. 12, 111–118 (2010).">11</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR12" id="ref-link-section-d8519934e599_11" title="Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. Dev. Cell 22, 1234–1246 (2012).">12</a>,<a aria-label="Reference 13" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR13" id="ref-link-section-d8519934e602" title="Brickner, D. G. et al. Subnuclear positioning and interchromosomal clustering of the GAL1-10 locus are controlled by separable, interdependent mechanisms. Mol. Biol. Cell 27, 2980–2993 (2016).">13</a></sup>. This has been particularly well studied for <i>Saccharomyces cerevisiae</i> gene loci, for which several stress response genes, including nutrient related genes such as the <i>GAL</i> loci and heat shock response genes, translocate to the nuclear periphery upon activation<sup><a aria-label="Reference 1" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR1" id="ref-link-section-d8519934e612" title="Brickner, J. H. &amp; Walter, P. Gene recruitment of the activated INO1 locus to the nuclear membrane. PLoS Biol. 2, e342 (2004).">1</a>,<a aria-label="Reference 3" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR3" id="ref-link-section-d8519934e615" title="Casolari, J. M. et al. Genome-wide localization of the nuclear transport machinery couples transcriptional status and nuclear organization. Cell 117, 427–439 (2004).">3</a>,<a aria-label="Reference 8" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR8" id="ref-link-section-d8519934e618" title="Cabal, G. G. et al. SAGA interacting factors confine sub-diffusion of transcribed genes to the nuclear envelope. Nature 441, 770–773 (2006).">8</a>,<a aria-label="Reference 9" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR9" id="ref-link-section-d8519934e621" title="Taddei, A. et al. Nuclear pore association confers optimal expression levels for an inducible yeast gene. Nature 441, 774–778 (2006).">9</a></sup>. These genes remain at the periphery after returning to their repressed condition, a form of transcriptional memory, which primes the genes to be more rapidly transcribed, translocated to the cytosol and translated, as nutrients vary between being depleted and enriched in a fluctuating environment<sup><a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR11" id="ref-link-section-d8519934e625" title="Ahmed, S. et al. DNA zip codes control an ancient mechanism for gene targeting to the nuclear periphery. Nat. Cell Biol. 12, 111–118 (2010).">11</a>,<a aria-label="Reference 14" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR14" id="ref-link-section-d8519934e628" title="Brickner, D. G. et al. H2A.Z-mediated localization of genes at the nuclear periphery confers epigenetic memory of previous transcriptional state. PLoS Biol. 5, e81 (2007).">14</a></sup>.</p><p>The <i>INO1</i> gene has been among the most studied to determine a molecular mechanism for gene locus translocation<sup><a aria-label="Reference 1" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR1" id="ref-link-section-d8519934e638" title="Brickner, J. H. &amp; Walter, P. Gene recruitment of the activated INO1 locus to the nuclear membrane. PLoS Biol. 2, e342 (2004).">1</a>,<a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR12" id="ref-link-section-d8519934e641" title="Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. Dev. Cell 22, 1234–1246 (2012).">12</a>,<a aria-label="Reference 15" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR15" id="ref-link-section-d8519934e644" title="Randise-Hinchliff, C. et al. Strategies to regulate transcription factor-mediated gene positioning and interchromosomal clustering at the nuclear periphery. J. Cell Biol. 212, 633–646 (2016).">15</a></sup>. Efforts to understand locus localization to the nuclear envelope (NE) have focused on attempts to identify protein-protein or protein-DNA interactions that could account for physical tethering of chromatin to the NE<sup><a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR11" id="ref-link-section-d8519934e648" title="Ahmed, S. et al. DNA zip codes control an ancient mechanism for gene targeting to the nuclear periphery. Nat. Cell Biol. 12, 111–118 (2010).">11</a>,<a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR12" id="ref-link-section-d8519934e651" title="Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. Dev. Cell 22, 1234–1246 (2012).">12</a></sup>. Thus, interactions of component proteins of the Nuclear Pore Complex (NPC) or associated components such as subunits of the SAGA complex have been suggested to induce, or their deletion has been shown to prevent <i>INO1</i> translocation<sup><a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR11" id="ref-link-section-d8519934e658" title="Ahmed, S. et al. DNA zip codes control an ancient mechanism for gene targeting to the nuclear periphery. Nat. Cell Biol. 12, 111–118 (2010).">11</a>,<a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR12" id="ref-link-section-d8519934e661" title="Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. Dev. Cell 22, 1234–1246 (2012).">12</a></sup>. However, so far, there are no direct evidence that these proteins are involved in physical tethering of the <i>INO1</i> locus to the NE. Furthermore, 5’ <i>cis</i> elements in the <i>INO1</i> locus called Gene Recruitment Sequences (GRS) or “Zip Codes”, have been shown to be essential to <i>INO1</i> and now other loci translocations<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR11" id="ref-link-section-d8519934e678" title="Ahmed, S. et al. DNA zip codes control an ancient mechanism for gene targeting to the nuclear periphery. Nat. Cell Biol. 12, 111–118 (2010).">11</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR12" id="ref-link-section-d8519934e678_1" title="Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. Dev. Cell 22, 1234–1246 (2012).">12</a>,<a aria-label="Reference 13" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR13" id="ref-link-section-d8519934e681" title="Brickner, D. G. et al. Subnuclear positioning and interchromosomal clustering of the GAL1-10 locus are controlled by separable, interdependent mechanisms. Mol. Biol. Cell 27, 2980–2993 (2016).">13</a></sup>. It has been proposed that GRSs could be anchoring sites for the tethering of gene loci to the NE via protein-DNA or protein-protein interactions<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR11" id="ref-link-section-d8519934e685" title="Ahmed, S. et al. DNA zip codes control an ancient mechanism for gene targeting to the nuclear periphery. Nat. Cell Biol. 12, 111–118 (2010).">11</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR12" id="ref-link-section-d8519934e685_1" title="Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. Dev. Cell 22, 1234–1246 (2012).">12</a>,<a aria-label="Reference 13" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR13" id="ref-link-section-d8519934e688" title="Brickner, D. G. et al. Subnuclear positioning and interchromosomal clustering of the GAL1-10 locus are controlled by separable, interdependent mechanisms. Mol. Biol. Cell 27, 2980–2993 (2016).">13</a></sup>. To date, however, evidence of interactions between GRSs and proteins at the NE have not been reported. Furthermore, studies of the statistical distributions of translocated genes among populations of cells, as well as gene tracking measurements, show that instead of being bound at the NE, as are, for instance, the NPC or Spindle Pole Body, activated gene loci are simply statistically more tightly distributed towards the NE<sup><a aria-label="Reference 8" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR8" id="ref-link-section-d8519934e693" title="Cabal, G. G. et al. SAGA interacting factors confine sub-diffusion of transcribed genes to the nuclear envelope. Nature 441, 770–773 (2006).">8</a>,<a aria-label="Reference 9" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR9" id="ref-link-section-d8519934e696" title="Taddei, A. et al. Nuclear pore association confers optimal expression levels for an inducible yeast gene. Nature 441, 774–778 (2006).">9</a>,<a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR12" id="ref-link-section-d8519934e699" title="Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. Dev. Cell 22, 1234–1246 (2012).">12</a>,<a aria-label="Reference 16" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR16" id="ref-link-section-d8519934e702" title="Berger, A. B. et al. High-resolution statistical mapping reveals gene territories in live yeast. Nat. Methods 5, 1031–1037 (2008).">16</a></sup>. These observations led us to postulate that active gene locus translocation could be the result of differential partitioning of active gene loci to the periphery by a local phase transition of the chromatin caused by histone acetylation and other modifications, resulting in increased rates of histone exchange and associated decompaction of the chromatin. Here we present evidence that the GRSs are sites of association of chromatin remodeling complexes, including HATs, which catalyze histone H3 acetylation. Consequently, there is an increase in H3 exchange between nucleosomes and nucleoplasm within and surrounding the activated <i>INO1</i> locus. The resulting decompaction of chromatin causes phase separation of the <i>INO1</i> locus from the compacted chromatin found in the nucleoplasm to the decompacted chromatin adjacent to the nuclear envelope.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><p>Translocation of the activated <i>INO1</i> locus and decompaction of its chromatin were studied as follows: first, we obtained a yeast strain in which 128 LacO binding motifs are integrated into the gene and expression of the LacO binding protein LacI is fused to GFP<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR12" id="ref-link-section-d8519934e723" title="Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. Dev. Cell 22, 1234–1246 (2012).">12</a></sup>. Furthermore, reference markers for the nuclear envelope (Nup49-GFP) and nucleolus (Nop1-mCherry) are also expressed. This strain could then be used to monitor and compare the statistical distribution of <i>INO1</i> in cells grown in the presence (inactive <i>INO1</i>) or absence (active <i>INO1</i>) of inositol (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig1">1a, b</a>). Next, we could monitor the phase transition of active chromatin based on the observation that the consequent decompaction of the chromatin, is reflected in changes in its mechanical properties and geometric organization<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e740" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a>,<a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR18" id="ref-link-section-d8519934e743" title="Bloom, K. S. Beyond the code: the mechanical properties of DNA as they relate to mitosis. Chromosoma 117, 103–110 (2008).">18</a></sup>. These changes can be understood as follows: first, the persistence length (<i>L</i><sub><i>p</i></sub>) of DNA, as for all polymers, is the length over which a thermal perturbation would cause it to bend. The <i>L</i><sub><i>p</i></sub> for naked DNA and transcriptionally active chromatin is smaller than that of chromatin on small lenght scales (5–50 nm versus 170–220 nm)<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR19" id="ref-link-section-d8519934e759" title="Bystricky, K., Heun, P., Gehlen, L., Langowski, J. &amp; Gasser, S. M. Long-range compaction and flexibility of interphase chromatin in budding yeast analyzed by high-resolution imaging techniques. Proc. Natl. Acad. Sci. USA 101, 16495–16500 (2004).">19</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR20" id="ref-link-section-d8519934e759_1" title="Cui, Y. &amp; Bustamante, C. Pulling a single chromatin fiber reveals the forces that maintain its higher-order structure. Proc. Natl. Acad. Sci. USA 97, 127–132 (2000).">20</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR21" id="ref-link-section-d8519934e759_2" title="Chi, Q., Wang, G. &amp; Jiang, J. The persistence length and length per base of single-stranded DNA obtained from fluorescence correlation spectroscopy measurements using mean field theory. Phys. A Stat. Mech. Appl. 392, 1072–1079 (2013).">21</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR22" id="ref-link-section-d8519934e759_3" title="Rechendorff, K., Witz, G., Adamcik, J. &amp; Dietler, G. Persistence length and scaling properties of single-stranded DNA adsorbed on modified graphite. J. Chem. Phys. 131, 095103 (2009).">22</a>,<a aria-label="Reference 23" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR23" id="ref-link-section-d8519934e762" title="Smith, S. B., Cui, Y. &amp; Bustamante, C. Overstretching B-DNA: the elastic response of individual double-stranded and single-stranded DNA molecules. Science 271, 795–799 (1996).">23</a></sup> given that these forms of DNA have more degrees of freedom than compact, inactive chromatin<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e767" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a>,<a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR18" id="ref-link-section-d8519934e770" title="Bloom, K. S. Beyond the code: the mechanical properties of DNA as they relate to mitosis. Chromosoma 117, 103–110 (2008).">18</a></sup>. Notably, the opposite is true over long length scales. This increase in degrees of freedom implies a higher number of configurations that can be explored per unit time by naked DNA or active chromatin, so its movements are more confined compared to chromatin. Consequently, chromatin behaves as an “entropic spring” of increasing spring stiffness as it transitions from compact to decompacted states<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e774" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a>,<a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR18" id="ref-link-section-d8519934e777" title="Bloom, K. S. Beyond the code: the mechanical properties of DNA as they relate to mitosis. Chromosoma 117, 103–110 (2008).">18</a></sup>. The spring constant of a polymer such as chromatin (<i>k</i><sub><i>s</i></sub>) is inversely proportional to <i>L</i><sub><i>p</i></sub> (Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Equ1">1</a>)) and consequently <i>k</i><sub><i>s</i></sub> is predicted to increase as chromatin decompacts. Furthermore, its trajectories through space are shorter, a characteristic that is determined from the radius of confinement (<i>R</i><sub><i>c</i></sub>) of the chromatin<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e809" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a>,<a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR18" id="ref-link-section-d8519934e812" title="Bloom, K. S. Beyond the code: the mechanical properties of DNA as they relate to mitosis. Chromosoma 117, 103–110 (2008).">18</a></sup>. Such a model is highly simplified, but for our purposes, measuring these values for the <i>INO1</i> gene locus would serve to quantify material properties of the chromatin and how these change under different conditions. Values for <i>k</i><sub><i>s</i></sub> and <i>R</i><sub><i>c</i></sub> can be determined from measurements of mean-square displacement (MSD) and localization coordinates of the GFP-labeled LacO array embedded in the <i>INO1</i> gene locus<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e835" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a></sup>.</p><div class="c-article-equation" id="Equ1"><div class="c-article-equation__content"><span class="mathjax-tex">$${k}_{s}=\frac{{k}_{B}T}{{L}_{p}^{2}}$$</span></div><div class="c-article-equation__number">
                    (1)
                </div></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Activated INO1 NE localization is disrupted by the deletion of both GRS I and II." id="figure-1"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig1">Fig. 1: Activated <i>INO1</i> NE localization is disrupted by the deletion of both GRS I and II.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-023-36391-6/figures/1" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig1_HTML.png?as=webp" type="image/webp"><img alt="figure 1" aria-describedby="Fig1" height="601" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig1_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b> Representative photomicrographs of yeast nuclei containing the LacO repressor array integrated into the <i>INO1</i> locus and expressing GFP-LacI allows for visualization of dynamics in the locus with GFP-Nup49-labeled NE and Nop1-mCherry-labeled nucleolus as spatial reference markers (maximum projection of 250 nm Z stacks). Scale bar represents 1 µm. These results are representative of three independent experiments. <b>b</b> Yeast nucleus landmarks used as coordinates for this analysis. NE (green ellipsoid), nucleolus (red ellipsoid) and nucleus center (green x). The gray lines represent the central axes of the ellipsoid; α, angle from the longer central axis. Orange, red, and green blobs connected by a trace represent the spindle pole body, telomeres and gene locus, respectively. <b>c</b> <i>INO1</i> probability maps obtained from the analysis of 1399 nuclei of <i>wild type</i> cells grown in the absence of inositol (active), and <b>d</b> 972 nuclei of <i>wild type</i> cells in the presence of inositol 100 µM (repressed). Dashed yellow line, NE; dashed red line, position of the nucleolus; the color scale represents the probability density distribution of the locus inside the nucleus. <b>e</b>–<b>g</b> <i>INO1</i> probability maps obtained from the analysis of 959 (GRSI∆), 1274 (GRSII∆) and 1294 (GRSI, II∆) nuclei in cells in which GRS I, GRS II or both GRS I and II are deleted, respectively, and that were grown in the absence of inositol (active).</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 1" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure1 Full size image" data-track-label="button" href="/articles/s41467-023-36391-6/figures/1" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec3">Deletion of GRS I and II prevents localization of active <i>INO1</i> to the nuclear envelope</h3><p>The relocalization of the active <i>INO1</i> locus to the nuclear periphery is prevented in strains in which the GRSI, and II of the <i>INO1</i> locus are both deleted (hereafter referred to as the GRSI, IIΔ double mutant)<sup><a aria-label="Reference 24" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR24" id="ref-link-section-d8519934e987" title="Ahmed, S. &amp; Brickner, J. H. A role for DNA sequence in controlling the spatial organization of the genome. Nucleus 1, 402–406 (2010).">24</a></sup>. To access the effects of GRS deletions on the statistical distribution of the <i>INO1</i> locus in large populations of cells, we used a high-resolution computational approach that generates probabilistic maps of the locus distribution in the nucleus (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig1">1a, b</a>)<sup><a aria-label="Reference 16" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR16" id="ref-link-section-d8519934e998" title="Berger, A. B. et al. High-resolution statistical mapping reveals gene territories in live yeast. Nat. Methods 5, 1031–1037 (2008).">16</a></sup>. Using the LacI-GFP-labeled LacO array as reporter for the <i>INO1</i> locus and, Nop1-mCherry and Nup49-GFP as nucleolus and nuclear envelope reference markers, respectively, we mapped the distribution of the <i>INO1</i> locus with respect to these landmarks for the <i>wild type</i> and single and double mutants of the GRS sites under repressing (+ inositol) and activating (− inositol) conditions.</p><p>Consistent with previous studies, the repressed <i>INO1</i> locus was distributed throughout the nucleoplasm, but the activated <i>INO1</i> locus was mainly confined to a specific region in the vicinity of the NE (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig1">1c, d</a>)<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR12" id="ref-link-section-d8519934e1023" title="Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. Dev. Cell 22, 1234–1246 (2012).">12</a></sup>. Single GRS deletion mutants reduced and the double mutant GRSI, IIΔ prevented partitioning of the active <i>INO1</i> locus towards the NE, as reported previously (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig1">1e–g</a>)<sup><a aria-label="Reference 24" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR24" id="ref-link-section-d8519934e1034" title="Ahmed, S. &amp; Brickner, J. H. A role for DNA sequence in controlling the spatial organization of the genome. Nucleus 1, 402–406 (2010).">24</a></sup>. The <i>INO1</i> locus was distributed towards, but not fixed at the NE<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR12" id="ref-link-section-d8519934e1041" title="Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. Dev. Cell 22, 1234–1246 (2012).">12</a></sup>. The effects of the mutants do not reflect changes in transcription of <i>INO1</i> as disrupting transcription was previously shown to have no effect on <i>INO1</i> relocalization<sup><a aria-label="Reference 14" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR14" id="ref-link-section-d8519934e1052" title="Brickner, D. G. et al. H2A.Z-mediated localization of genes at the nuclear periphery confers epigenetic memory of previous transcriptional state. PLoS Biol. 5, e81 (2007).">14</a></sup>.</p><p>The activated <i>INO1</i> locus shows a characteristic spatial distribution at the NE that perhaps reflects its position in the middle of the 3’ arm of Chromosome X. NE localization of other activated gene loci also reflects their chromosome positions, including the <i>GAL1-10</i> locus, which is located closer to the centromere region in Chromosome II and thus close to the Spindle Pole Body<sup><a aria-label="Reference 16" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR16" id="ref-link-section-d8519934e1065" title="Berger, A. B. et al. High-resolution statistical mapping reveals gene territories in live yeast. Nat. Methods 5, 1031–1037 (2008).">16</a></sup> and the <i>HXK1</i> gene locus, located in a subtelomeric region of Chromosome VI, that is localized towards the NE whether active or repressed, but with a more compact distribution when activated<sup><a aria-label="Reference 9" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR9" id="ref-link-section-d8519934e1072" title="Taddei, A. et al. Nuclear pore association confers optimal expression levels for an inducible yeast gene. Nature 441, 774–778 (2006).">9</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec4">Deletion of GRSs prevents decompaction of the active <i>INO1</i> locus</h3><p>We measured differences in the mechanical spring stiffness of the <i>INO1</i> locus in the repressed and activated states. To do this, we tracked the GFP-LacI-labeled LacO array in the <i>INO1</i> locus for 1 min at intervals of 500 milliseconds and calculated MSD values in each case. The slope and plateaus of the MSD trajectories for the <i>INO1</i> locus were markedly reduced under activating versus repressed conditions (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig2">2a</a>). In addition, loci closer to the NE appear to be excluded from the interior of the nucleoplasm as observed before for the <i>GAL1-10</i> genes<sup><a aria-label="Reference 8" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR8" id="ref-link-section-d8519934e1104" title="Cabal, G. G. et al. SAGA interacting factors confine sub-diffusion of transcribed genes to the nuclear envelope. Nature 441, 770–773 (2006).">8</a></sup>, which also contain GRS sequences required for relocalization of the <i>GAL1-10</i> loci to the NE<sup><a aria-label="Reference 13" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR13" id="ref-link-section-d8519934e1111" title="Brickner, D. G. et al. Subnuclear positioning and interchromosomal clustering of the GAL1-10 locus are controlled by separable, interdependent mechanisms. Mol. Biol. Cell 27, 2980–2993 (2016).">13</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="INO1 dynamics and material properties depend on the presence of both GRS I and II." id="figure-2"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig2">Fig. 2: <i>INO1</i> dynamics and material properties depend on the presence of both GRS I and II.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-023-36391-6/figures/2" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig2_HTML.png?as=webp" type="image/webp"><img alt="figure 2" aria-describedby="Fig2" height="419" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig2_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b> MSD curves for <i>INO1</i> locus in <i>wild type</i> (purple and blue lines, <i>n</i> = 38 and 34 cells, respectively) and the GRSI, II∆ (dark and light green lines, <i>n</i> = 51 and 45 cells, respectively) strains under activating and repressed conditions, and for a population of fixed cells (black line, <i>n</i> = 8 cells). Data are presented as mean values +/− SEM. <b>b</b>–<b>d</b> Radius of confinement (R<sub>c</sub>) significantly decreases in <i>wild type</i>, GRSIΔ, and GRSIIΔ cells from repressed (purple) to activated (blue) conditions. <b>e</b> R<sub>c</sub> for GRSI, II∆ mutant cells under repressing (purple) and activating (blue) conditions show no significant differences. <b>f</b>–<b>h</b> Spring constant (k<sub>s</sub>) for <i>wild type</i>, GRSIΔ, and GRSIIΔ cells increases significantly from repressing (purple) to activating (blue) conditions. <b>i</b> k<sub>s</sub> for GRSI, II∆ mutant shows no significant difference between repressing (purple) and activating (blue) conditions. Source data are provided as a Source Data file. <b>j</b> Schematic of <i>INO1</i> locus size measurements of aspect ratio variation over time. Green circles represent nuclei of two cells showing loci spots with different long and short axis ratios. <b>k</b> Aspect ratio change over time for two <i>INO1</i> loci in cells grown under repressed (purple) or activated (blue) conditions measured by Structured Illumination Microscopy. In the box and whisker plots (<b>b</b>–<b>i</b>), the median is indicated as middle line, 25th and 75th percentile as boxes and the whiskers represent minimum and maximum values; <i>n</i> = 38 and 34 cells, respectively (<b>b</b>, <b>f</b>), <i>n</i> = 46 and 51 cells, respectively (<b>c</b>, <b>g</b>), <i>n</i> = 31 cells (<b>d</b>, <b>h</b>), <i>n</i> = 51 and 45 cells, respectively (<b>e</b>, <b>i</b>). Data were analyzed by unpaired two-sided Student’s <i>t</i> test in <b>b</b>–<b>i</b>.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 2" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure2 Full size image" data-track-label="button" href="/articles/s41467-023-36391-6/figures/2" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>We then used established relationships between MSD statistics and <i>R</i><sub><i>c</i></sub> and <i>k</i><sub><i>s</i></sub> to calculate their values as follows:<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e1271" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a></sup> <i>R</i><sub><i>c</i></sub> could be calculated directly from the MSD plateaus (See Methods, Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Equ3">3</a>))<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e1285" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a>,<a aria-label="Reference 25" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR25" id="ref-link-section-d8519934e1288" title="Neumann, F. R. et al. Targeted INO80 enhances subnuclear chromatin movement and ectopic homologous recombination. Genes Dev. 26, 369–383 (2012).">25</a></sup>. <i>R</i><sub><i>c</i></sub> could independently be determined from the standard deviation of spot positions, σ, and the average squared deviation from the mean position, <span class="mathjax-tex">\(\left\langle \varDelta {r}_{0}^{2}\right\rangle\)</span>, by applying the equipartition theorem (See Methods, Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Equ4">4</a>))<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e1343" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a>,<a aria-label="Reference 26" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR26" id="ref-link-section-d8519934e1346" title="Scheffold, F. et al. Brushlike interactions between thermoresponsive microgel particles. Phys. Rev. Lett. 104, 128304 (2010).">26</a>,<a aria-label="Reference 27" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR27" id="ref-link-section-d8519934e1349" title="Uhlenbeck, G. E. &amp; Ornstein, L. S. On the theory of the brownian motion. Phys. Rev. 36, 823–841 (1930).">27</a></sup>.</p><p>The <i>INO1</i> locus in <i>wild type</i> strains (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig2">2b</a>) and single GRS mutant cells (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig2">2c, d</a>) showed significantly larger <i>R</i><sub><i>c</i></sub> for the repressed versus the activated state. The GRSI, IIΔ double deletion, however, resulted in no significant differences in <i>R</i><sub><i>c</i></sub> between repressed and activated states (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig2">2e</a>). These results were also consistent with statistical distributions of loci in populations of cells, demonstrating that the observed changes in <i>R</i><sub><i>c</i></sub> in single cells are pervasive in the population.</p><p>Using the equipartition theorem and standard deviation (σ) for the distribution of steps from the mean we calculated <i>k</i><sub><i>s</i></sub> (See Methods, Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Equ5">5</a>))<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e1402" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a>,<a aria-label="Reference 26" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR26" id="ref-link-section-d8519934e1405" title="Scheffold, F. et al. Brushlike interactions between thermoresponsive microgel particles. Phys. Rev. Lett. 104, 128304 (2010).">26</a></sup>. The <i>k</i><sub><i>s</i></sub> of the <i>INO1</i> loci significantly increased in cells with active versus repressed loci in both <i>wild type</i> (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig2">2f</a>) and the GRS single deletion strains (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig2">2g, h</a>), suggesting a stiffening of decompacted chromatin. There was no significant change in <i>k</i><sub><i>s</i></sub> in the GRSI, IIΔ double deletion strain under activating conditions (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig2">2i</a>).</p><h3 class="c-article__sub-heading" id="Sec5">The active <i>INO1</i> locus behaves as an elastic filament</h3><p>As discussed above, the increase in spring stiffness of decompacted chromatin reflects in a decrease in the rate and amplitude of <i>INO1</i> locus expansion-contraction<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e1453" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a></sup>. This behavior can be visualized with the LacI-GFP-labeled LacO array and quantified by fitting an ellipsoid function to the GFP signal imaged at super-resolution by Structured Illumination Microscopy and calculating the aspect ratio between long and short axes of the ellipsoid (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig2">2j</a>). Consistent with predicted behavior, the <i>INO1</i> locus changes shape with higher frequencies and amplitudes in the repressed versus the activated states (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig2">2k</a>). These observations are also consistent with previous results in which a decrease in <i>L</i><sub><i>p</i></sub> caused by H3 histone depletion resulted in a decrease in the rate and amplitude of expansion of the chromatin<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e1473" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec6">Activation of the <i>INO1</i> locus increases rates of exchange of histone H3</h3><p>To test whether the changes in <i>INO1</i> locus mechanical properties follow from an increase in histone exchange, we used a strain to probe changes in histone H3 nucleosome exchange in <i>wild type</i> and GRSI, IIΔ double mutant strains. In this strain histones H3 and H4 genes are deleted from the genome and replaced with both constitutively and inducible expressed copies (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig3">3a</a>)<sup><a aria-label="Reference 28" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR28" id="ref-link-section-d8519934e1499" title="Rufiange, A., Jacques, P. E., Bhat, W., Robert, F. &amp; Nourani, A. Genome-wide replication-independent histone H3 exchange occurs predominantly at promoters and implicates H3 K56 acetylation and Asf1. Mol. Cell 27, 393–405 (2007).">28</a></sup>. In a galactose inducible cassette, H3 is fused to a FLAG-tag, such that when expressed can be used to quantify newly synthesized H3 that is incorporated into nucleosomes at specific positions by Chromatin Immunoprecipitation followed by real time PCR (ChIP-qPCR) to measure nucleosome exchange in different chromatin regions under different conditions. To avoid the contribution of histone incorporation due to DNA replication, cells are also arrested in G1 by the addition of α-factor. We probed H3-FLAG incorporation in several regions in the <i>INO1</i> promoter and Open Reading Frame (ORF), plus sites in upstream (<i>SNA3</i> and <i>DAS1</i>) and downstream (<i>VPS35</i>) genes to determine whether changes in histone exchange are confined to the <i>INO1</i> locus (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig3">3b</a>, Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">1</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="H3 exchange increase in the activated INO1 is affected when GRS I and II are deleted." id="figure-3"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig3">Fig. 3: H3 exchange increase in the activated <i>INO1</i> is affected when GRS I and II are deleted.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-023-36391-6/figures/3" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig3_HTML.png?as=webp" type="image/webp"><img alt="figure 3" aria-describedby="Fig3" height="489" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig3_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b> Schematic representation of the strain used to measure the incorporation of newly synthesized FLAG-H3 into nucleosomes<sup><a aria-label="Reference 28" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR28" id="ref-link-section-d8519934e1544" title="Rufiange, A., Jacques, P. E., Bhat, W., Robert, F. &amp; Nourani, A. Genome-wide replication-independent histone H3 exchange occurs predominantly at promoters and implicates H3 K56 acetylation and Asf1. Mol. Cell 27, 393–405 (2007).">28</a></sup>. <b>b</b> Cell sample preparation workflow before ChIP. For the repressed condition, inositol was added in all the media used. <b>c</b> FLAG-H3 enrichment, as determined by ChIP-qPCR and expressed in percentage of input, in the vicinity of GRS I and GRS II regions within <i>DAS1, SNA3, INO1 and VPS35</i> gene loci for the <i>wild type</i> strain under repressed (purple) and activated (blue) conditions, and the GRSI, II∆ (light yellow) under activated conditions. Source data are provided as a Source Data file. Data in panel <b>c</b> represent mean ± SD; <i>n</i> = 4 biological samples. Data were analyzed by unpaired two-sided Student’s <i>t</i> test. <i>p</i>-values are shown for the regions with significant differences between the GRS double mutant and the <i>wild type</i> active gene.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 3" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure3 Full size image" data-track-label="button" href="/articles/s41467-023-36391-6/figures/3" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>ChIP-qPCR results with an anti-FLAG antibody showed higher incorporation of newly synthesized histone H3 at the <i>INO1</i> promoter in cells under activated versus repressed conditions for the <i>INO1</i> promoter and ORF regions (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig3">3c</a>). Results for repressed cells were consistent with genome-wide ChIP-chip data<sup><a aria-label="Reference 28" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR28" id="ref-link-section-d8519934e1596" title="Rufiange, A., Jacques, P. E., Bhat, W., Robert, F. &amp; Nourani, A. Genome-wide replication-independent histone H3 exchange occurs predominantly at promoters and implicates H3 K56 acetylation and Asf1. Mol. Cell 27, 393–405 (2007).">28</a></sup>, validating our method. For the GRSI, IIΔ double mutant, significant reductions in H3 exchange under activated conditions were observed in the <i>INO1</i> locus promoter at the GRS I site and in the <i>SNA3</i> promoter and coding sequence where GRS II is located but not in the <i>INO1</i> ORF sequences (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig3">3c</a>). These results suggest that GRS I and II may be HAT and/or other chromatin remodeling complex association sites mediating H3 exchange in topologically adjacent nucleosomes. The fact that H3 exchange is not affected in the <i>INO1</i> ORF sequence is consistent with the fact that there are other known remodeling enzymes that bind near this region. Thus, consistent with ours and previous results, an increase in nucleosome histone H3 exchange at the <i>INO1</i> locus chromatin is associated with an increase in its spring stiffness reflecting decompaction of the chromatin<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e1619" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec7">Depletion of yeast HATs prevents active <i>INO1</i> locus partitioning to the nuclear envelope</h3><p>The increase in H3 exchange at the <i>INO1</i> locus is consistent with histone post-translational modifications, and specifically lysine acetylation by histone acetyltransferases (HATs)<sup><a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR18" id="ref-link-section-d8519934e1638" title="Bloom, K. S. Beyond the code: the mechanical properties of DNA as they relate to mitosis. Chromosoma 117, 103–110 (2008).">18</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR28" id="ref-link-section-d8519934e1641" title="Rufiange, A., Jacques, P. E., Bhat, W., Robert, F. &amp; Nourani, A. Genome-wide replication-independent histone H3 exchange occurs predominantly at promoters and implicates H3 K56 acetylation and Asf1. Mol. Cell 27, 393–405 (2007).">28</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR29" id="ref-link-section-d8519934e1641_1" title="Driscoll, R., Hudson, A. &amp; Jackson, S. P. Yeast Rtt109 promotes genome stability by acetylating histone H3 on lysine 56. Science 315, 649–652 (2007).">29</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR30" id="ref-link-section-d8519934e1641_2" title="Topal, S., Vasseur, P., Radman-Livaja, M. &amp; Peterson, C. L. Distinct transcriptional roles for Histone H3-K56 acetylation during the cell cycle in Yeast. Nat. Commun. 10, 4372 (2019).">30</a>,<a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR31" id="ref-link-section-d8519934e1644" title="Xhemalce, B. et al. Regulation of histone H3 lysine 56 acetylation in Schizosaccharomyces pombe. J. Biol. Chem. 282, 15040–15047 (2007).">31</a></sup>. We therefore hypothesized that the GRSs could be association sites for HATs and other chromatin remodeling complexes that alter the rates of histone-nucleosome exchange. Evidence for specific functions of HATs and histone deacetylases in controlling active <i>INO1</i> partitioning towards the NE exists. For instance, the Rpd3(L) histone deacetylase blocks partitioning of the <i>INO1</i> locus towards the NE and deletion of the gene encoding this deacetylase results in increased acetylation of nucleosome-associated histone complexes in the promoter of <i>INO1</i><sup><a aria-label="Reference 15" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR15" id="ref-link-section-d8519934e1656" title="Randise-Hinchliff, C. et al. Strategies to regulate transcription factor-mediated gene positioning and interchromosomal clustering at the nuclear periphery. J. Cell Biol. 212, 633–646 (2016).">15</a></sup>. Additionally, deleting subunits of the SAGA HAT complex decreases active <i>INO1</i> locus partitioning to the NE<sup><a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR11" id="ref-link-section-d8519934e1664" title="Ahmed, S. et al. DNA zip codes control an ancient mechanism for gene targeting to the nuclear periphery. Nat. Cell Biol. 12, 111–118 (2010).">11</a></sup>, also observed for the <i>GAL</i> loci<sup><a aria-label="Reference 8" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR8" id="ref-link-section-d8519934e1671" title="Cabal, G. G. et al. SAGA interacting factors confine sub-diffusion of transcribed genes to the nuclear envelope. Nature 441, 770–773 (2006).">8</a></sup>.</p><p>Since both GRS I &amp; II, plus other unidentified GRS sites could be associated with HATs, we attempted to identify HATs associated with the <i>INO1</i> locus by screening for loss of activated <i>INO1</i> locus NE localization in cells in which an auxin-inducible degron (AID) coding sequence was integrated 3’ to the ORFs of the 12 known yeast HAT catalytic subunits. The addition of auxin to growth medium then induces targeting of the HAT-AID degron fusion proteins for rapid proteasomal degradation (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig4">4a</a>)<sup><a aria-label="Reference 32" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR32" id="ref-link-section-d8519934e1687" title="Morawska, M. &amp; Ulrich, H. D. An expanded tool kit for the auxin-inducible degron system in budding yeast. Yeast 30, 341–351 (2013).">32</a></sup>. To induce fast degradation of each HAT–AID, cells were treated with 500 μM indole-3-acetic acid (IAA) for 1.5 h under inositol starvation (<i>INO1</i> induction) conditions. This condition was sufficient to reduce the expression of each HAT–AID protein as observed by Western blot (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">2</a>). After incubation, cells were immobilized on ConA-coated microtiter plate wells and imaged to generate high-resolution probabilistic maps of the localization of the <i>INO1</i> locus.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Specific histone acetyltransferases (HATs) are required for active INO1 localization." id="figure-4"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig4">Fig. 4: Specific histone acetyltransferases (HATs) are required for active <i>INO1</i> localization.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-023-36391-6/figures/4" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig4_HTML.png?as=webp" type="image/webp"><img alt="figure 4" aria-describedby="Fig4" height="792" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig4_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b> Schematic representation of the auxin-inducible degron (AID) system used to deplete targeted HAT catalytic subunits for degradation in the presence of auxin. <b>b</b>–<b>g</b> <i>INO1</i> probability maps obtained from the analysis of thousands of nuclei in strains with different HATs tagged with the AID degron and grown in the absence of inositol (active). <b>b</b>–<b>d</b> HAT-AID tagged strains where there is no effect on the statistical distribution of the <i>INO1</i> locus after treating the cells with auxin. <b>e</b>–<b>g</b> HAT-AID tagged strains where the addition of auxin results in disruption of the <i>INO1</i> locus relocalization pattern to the nuclear periphery. <b>h</b> H3K14ac enrichment around and within the active <i>INO1</i> gene, as determined by ChIP-qPCR and expressed as H3K14ac/H3 ratio, in the vicinity of <i>DAS1, SNA3, INO1 and VPS35</i> gene loci for <i>wild type</i> and three different HAT-AID strains treated with auxin and under inositol activating conditions. <b>i</b> mRNA levels as quantified by RT-qPCR, relative to <i>ACT1</i>, for <i>DAS1, SNA3, INO1</i> and <i>VPS35</i> loci in <i>wild type</i> and Gcn5-AID, Taf1-AID, and Rtt109-AID strains in the presence of auxin and <i>INO1</i> activating conditions. Source data are provided as a Source Data file. Data represent mean ± SD (<b>h</b> and <b>i</b>); <i>n</i> = 3 biological samples. Data were analyzed by unpaired two-sided Student’s <i>t</i> test and <i>p</i>-values are shown for significant differences between <i>wild type</i> (blue) and Gcn5-AID mutant (light purple). See also Supplementary Figs. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">2</a>–<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">4</a>.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 4" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure4 Full size image" data-track-label="button" href="/articles/s41467-023-36391-6/figures/4" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>Among the 12 HAT catalytic subunits, depletion of three of these reduced partitioning of the active <i>INO1</i> locus towards the NE: Gcn5, Rtt109, and Taf1 (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig4">4b–g</a>, Supplementary Figs. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">3</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">4</a>). Deletion of <i>GCN5</i>, coding for a subunit of the SAGA complex, has previously been shown to prevent active <i>INO1</i> locus partitioning towards the NE<sup><a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR11" id="ref-link-section-d8519934e1834" title="Ahmed, S. et al. DNA zip codes control an ancient mechanism for gene targeting to the nuclear periphery. Nat. Cell Biol. 12, 111–118 (2010).">11</a></sup>. The SAGA complex has also been implicated directly in the activation of <i>INO1</i> through transcription factors that bind to the upstream activating sequence (UAS) sites close to the TATA box of <i>INO1</i><sup><a aria-label="Reference 33" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR33" id="ref-link-section-d8519934e1843" title="Lo, W.-S. et al. Snf1-a histone kinase that works in concert with the histone acetyltransferase Gcn5 to regulate transcription. Science 293, 1142–1146 (2001).">33</a></sup>. Rtt109 is not part of any known chromatin remodeling complex and has not been implicated in <i>INO1</i> gene activation previously. This HAT acetylates lysine 56 on the histone H3 core domain (H3K56), a modification that has been globally observed on newly synthesized histones in <i>S. cerevisiae</i> and <i>S. pombe</i><sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR28" id="ref-link-section-d8519934e1856" title="Rufiange, A., Jacques, P. E., Bhat, W., Robert, F. &amp; Nourani, A. Genome-wide replication-independent histone H3 exchange occurs predominantly at promoters and implicates H3 K56 acetylation and Asf1. Mol. Cell 27, 393–405 (2007).">28</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR29" id="ref-link-section-d8519934e1856_1" title="Driscoll, R., Hudson, A. &amp; Jackson, S. P. Yeast Rtt109 promotes genome stability by acetylating histone H3 on lysine 56. Science 315, 649–652 (2007).">29</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR30" id="ref-link-section-d8519934e1856_2" title="Topal, S., Vasseur, P., Radman-Livaja, M. &amp; Peterson, C. L. Distinct transcriptional roles for Histone H3-K56 acetylation during the cell cycle in Yeast. Nat. Commun. 10, 4372 (2019).">30</a>,<a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR31" id="ref-link-section-d8519934e1859" title="Xhemalce, B. et al. Regulation of histone H3 lysine 56 acetylation in Schizosaccharomyces pombe. J. Biol. Chem. 282, 15040–15047 (2007).">31</a>,<a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR34" id="ref-link-section-d8519934e1862" title="Han, J. et al. Rtt109 acetylates histone H3 lysine 56 and functions in DNA replication. Science 315, 653–655 (2007).">34</a></sup>. Interestingly, H3K56 acetylation also correlates with higher histone exchange rates in global analyses of the <i>S. cerevisiae</i> genome<sup><a aria-label="Reference 28" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR28" id="ref-link-section-d8519934e1869" title="Rufiange, A., Jacques, P. E., Bhat, W., Robert, F. &amp; Nourani, A. Genome-wide replication-independent histone H3 exchange occurs predominantly at promoters and implicates H3 K56 acetylation and Asf1. Mol. Cell 27, 393–405 (2007).">28</a></sup>. Finally, Taf1 is a subunit of the large TFIID complex, a general transcription initiation factor for RNA polymerase II (Pol II). Taf1 has been shown to have histone acetyltransferase activity in in vitro assays, but its role in vivo is less well understood<sup><a aria-label="Reference 35" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR35" id="ref-link-section-d8519934e1873" title="Mizzen, C. A. et al. The TAF(II)250 subunit of TFIID has histone acetyltransferase activity. Cell 87, 1261–1270 (1996).">35</a>,<a aria-label="Reference 36" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR36" id="ref-link-section-d8519934e1876" title="Durant, M. &amp; Pugh, B. F. Genome-wide relationships between TAF1 and histone acetyltransferases in Saccharomyces cerevisiae. Mol. Cell. Biol. 26, 2791–2802 (2006).">36</a></sup>.</p><p>We analyzed the effect of the depletion of each of these three HATs on a specific acetylation modification in the regions surrounding the <i>INO1</i> locus including the GRS sites. The H3K14 acetylation is known to be catalyzed by Gcn5<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR37" id="ref-link-section-d8519934e1886" title="Duan, M. R. &amp; Smerdon, M. J. Histone H3 lysine 14 (H3K14) acetylation facilitates DNA repair in a positioned nucleosome by stabilizing the binding of the chromatin Remodeler RSC (Remodels Structure of Chromatin). J. Biol. Chem. 289, 8353–8363 (2014).">37</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR38" id="ref-link-section-d8519934e1886_1" title="Smerdon, M. J. DNA repair and the role of chromatin structure. Curr. Opin. Cell Biol. 3, 422–428 (1991).">38</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR39" id="ref-link-section-d8519934e1886_2" title="Kuo, M. H., vom Baur, E., Struhl, K. &amp; Allis, C. D. Gcn4 activator targets Gcn5 histone acetyltransferase to specific promoters independently of transcription. Mol. Cell 6, 1309–1320 (2000).">39</a>,<a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR40" id="ref-link-section-d8519934e1889" title="Kuo, M. H. et al. Transcription-linked acetylation by Gcn5p of histones H3 and H4 at specific lysines. Nature 383, 269–272 (1996).">40</a></sup>, and it has been observed in vitro that Taf1 also acetylates H3 K14<sup><a aria-label="Reference 35" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR35" id="ref-link-section-d8519934e1893" title="Mizzen, C. A. et al. The TAF(II)250 subunit of TFIID has histone acetyltransferase activity. Cell 87, 1261–1270 (1996).">35</a></sup>, although this is more controversial<sup><a aria-label="Reference 36" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR36" id="ref-link-section-d8519934e1897" title="Durant, M. &amp; Pugh, B. F. Genome-wide relationships between TAF1 and histone acetyltransferases in Saccharomyces cerevisiae. Mol. Cell. Biol. 26, 2791–2802 (2006).">36</a></sup>. Rtt109 has been also observed to acetylate H3K14 in in vitro assays<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR41" id="ref-link-section-d8519934e1901" title="Berndsen, C. E. et al. Molecular functions of the histone acetyltransferase chaperone complex Rtt109-Vps75. Nat. Struct. Mol. Biol. 15, 948–956 (2008).">41</a>,<a aria-label="Reference 42" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR42" id="ref-link-section-d8519934e1904" title="Abshiru, N. et al. Chaperone-mediated acetylation of histones by Rtt109 identified by quantitative proteomics. J. Proteom. 81, 80–90 (2013).">42</a></sup>. This modification was quantified by ChIP-qPCR on the <i>INO1</i> locus and on proximal genes in the AID strains for each of the three HATs. Results are presented as the ratio between each modification and H3 levels in each region analyzed. The regions screened correspond to the same ones used when measuring the rate of incorporation of newly synthesized histones (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig3">3</a>).</p><p>The results obtained (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig4">4h</a>), show that there is a significant decrease in the H3K14ac/H3 ratio for the strain where Gcn5 was depleted compared to <i>wild type</i> in all regions analyzed. A significant decrease is also observed for the other two HATs, mainly for the regions close to the GRSII site, however not as significant as that for the Gcn5-AID strain. This result indicates that Gcn5 could be the dominant HAT for acetylation in these regions.</p><p>We quantified mRNA levels for these loci after depleting each of the HATs in the AID strains. We observed a decrease in <i>INO1</i> mRNA levels for Gcn5-AID and Rtt109-AID depletion strains, and a decrease for <i>SNA3</i> mRNA levels, in this case for all three HAT-AID strains, although more significantly for Gcn5-AID and Rtt109-AID strains (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig4">4i</a>). For the <i>DAS1</i> and <i>VPS35</i> loci, there was no apparent difference in mRNA levels between all four strains (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig4">4i</a>). These results indicate that the transcription is affected for <i>INO1</i> and the GRS containing upstream locus <i>SNA3</i>, when at least two of the three HATs are depleted. Previous results have shown that deleting the GRS sites cause a decrease in <i>INO1</i> mRNA levels<sup><a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR11" id="ref-link-section-d8519934e1956" title="Ahmed, S. et al. DNA zip codes control an ancient mechanism for gene targeting to the nuclear periphery. Nat. Cell Biol. 12, 111–118 (2010).">11</a></sup>.</p><p>To probe for HATs and other chromatin remodeling complexes proximal to specific GRSs, we developed a proximity-dependent biotinylation (BioID) approach, based on guide RNA (sgRNA)-directed catalytically dead (d)Cas9 (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig5">5a</a>)<sup><a aria-label="Reference 43" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR43" id="ref-link-section-d8519934e1966" title="Opitz, N. et al. Capturing the Asc1p/receptor for activated C kinase 1 (RACK1) microenvironment at the head region of the 40 S ribosome with quantitative BioID in yeast. Mol. Cell. Proteom. MCP 16, 2199–2218 (2017).">43</a></sup>. We created BirA*-dCas9 ligase fusions in different N- or C-terminal orientations and using short (12) or long (38) amino acid linkers. These constructions were used to provide for more or less steric constraint and flexibility and promiscuity of biotin labeling. The short linker was used in a previous BioID study in yeast (GSSGSSGSSGSS)<sup><a aria-label="Reference 43" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR43" id="ref-link-section-d8519934e1970" title="Opitz, N. et al. Capturing the Asc1p/receptor for activated C kinase 1 (RACK1) microenvironment at the head region of the 40 S ribosome with quantitative BioID in yeast. Mol. Cell. Proteom. MCP 16, 2199–2218 (2017).">43</a></sup> and the long sequence in a CasID study in mammalian cells (LERPPLCWISAEFHHTGLVDPSSVPSLSLNRGSGSGSG)<sup><a aria-label="Reference 44" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR44" id="ref-link-section-d8519934e1974" title="Schmidtmann, E., Anton, T., Rombaut, P., Herzog, F. &amp; Leonhardt, H. Determination of local chromatin composition by CasID. Nucleus 7, 476–484 (2016).">44</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Chromatin remodeling subunits bind GRS site near INO1 locus." id="figure-5"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig5">Fig. 5: Chromatin remodeling subunits bind GRS site near <i>INO1</i> locus.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-023-36391-6/figures/5" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig5_HTML.png?as=webp" type="image/webp"><img alt="figure 5" aria-describedby="Fig5" height="940" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig5_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b> BirA*-dCas9 approach designed to target protospacer adjacent motif (PAM) sites in the vicinity of GRS I site by using different sgRNAs for site-specific binding of BirA*-dCas9 and biotinylation of proximal proteins. Proteins were then affinity purified with streptavidin, digested, and analyzed by mass spectrometry. <b>b</b>, <b>d</b> Micro array (MA) plots representing average protein intensities from biological triplicates from sgRNA-BirA*-dCas9 samples S5 (<b>b</b>) and S7 (<b>c</b>) plotted against protein quantity fold changes from these sgRNA-BirA*-dCas9 samples over those from their respective negative control triplicates (BirA*-dCas9). Protein fold change <i>p</i>-values calculated from a two-sided unpaired Student’s <i>t</i> test with equal variance are color coded, and the gray dashed lines represent 1.5-fold quantity increase. <i>p</i>-values are indicated next to labels. <b>c</b>, <b>e</b> Box plots of measured protein intensities of candidates from sgRNA-BirA*-dCas9 samples S5 (<b>d</b>) and S7 (<b>e</b>) triplicates along with their respective negative control triplicates. Source data are provided as a Source Data file. Median is indicated as middle line, average as a white dot, 25th and 75th percentile as boxes and whiskers represent 5th and 95th percentile. False discovery rates thresholds for mass spectrometry identification of peptide spectrum match, peptides, and proteins were set to 0.1. See also Supplementary Data <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM4">1</a>–<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM4">6</a> and Supplementary Figs. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">5</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">6</a>.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 5" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure5 Full size image" data-track-label="button" href="/articles/s41467-023-36391-6/figures/5" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>We transformed a <i>wild type</i> strain with the different plasmids encoding the BirA*-dCas9 constructs and four different sgRNAs targeting genomic sequences proximal to the GRS I site. We also generated control strains that were transformed with the BirA*-dCas9 constructs but not with sgRNAs (Supplementary Tables <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">1</a>–<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">3</a>). Western blot confirmed the expression of the different fusion proteins generated (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">5</a>). Biotinylated proteins were pulled down on streptavidin-sepharose beads, bound proteins were trypsin digested, and resulting peptides were analyzed by tandem mass spectrometry (LC-MS/MS). We observed enrichment of peptides derived from proteins associated with several nuclear complexes in extracts from strains expressing versus those not expressing sgRNAs (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig5">5b–e</a>, Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">6</a>) (Table <a data-track="click" data-track-action="table anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Tab1">1</a>; Supplementary Data <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM4">1</a>–<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM4">6</a>). Five of the complexes are associated with chromatin remodeling activity including Ino80, Swr1, ASTRA, Set3, and FACT complexes (Table <a data-track="click" data-track-action="table anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Tab1">1</a>). Among the enriched peptides, those belonging to the ATP-dependent DNA helicases Rvb1 and/or Rvb2 were enriched in all separate sgRNA-BirA*-dCas9 samples. Rvb1 and Rvb2 are components of Ino80<sup><a aria-label="Reference 45" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR45" id="ref-link-section-d8519934e2085" title="Shen, X., Mizuguchi, G., Hamiche, A. &amp; Wu, C. A chromatin remodelling complex involved in transcription and DNA processing. Nature 406, 541–544 (2000).">45</a></sup>, Swr1<sup><a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR46" id="ref-link-section-d8519934e2090" title="Nguyen, V. Q. et al. Molecular architecture of the ATP-dependent chromatin-remodeling complex SWR1. Cell 154, 1220–1231 (2013).">46</a></sup>, and ASTRA<sup><a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR46" id="ref-link-section-d8519934e2094" title="Nguyen, V. Q. et al. Molecular architecture of the ATP-dependent chromatin-remodeling complex SWR1. Cell 154, 1220–1231 (2013).">46</a></sup> complexes and they are also linked to two (Gcn5 and Taf1)<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR47" id="ref-link-section-d8519934e2098" title="Lee, K. K. et al. Combinatorial depletion analysis to assemble the network architecture of the SAGA and ADA chromatin remodeling complexes. Mol. Syst. Biol. 7, 503 (2011).">47</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR48" id="ref-link-section-d8519934e2098_1" title="Sanders, S. L., Jennings, J., Canutescu, A., Link, A. J. &amp; Weil, P. A. Proteomics of the eukaryotic transcription machinery: identification of proteins associated with components of yeast TFIID by multidimensional mass spectrometry. Mol. Cell. Biol. 22, 4723–4738 (2002).">48</a>,<a aria-label="Reference 49" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR49" id="ref-link-section-d8519934e2101" title="Lakshminarasimhan, M. et al. Proteomic and genomic analyses of the Rvb1 and Rvb2 interaction network upon deletion of R2TP complex components. Mol. Cell. Proteom. MCP 15, 960–974 (2016).">49</a></sup> of the three HAT catalytic subunits, whose depletion resulted in reduced partitioning of the active <i>INO1</i> towards the NE. Specifically, Gcn5 and Taf1 make protein-protein interactions with Rvb1 (Gcn5) and Rbv2 (Gcn5 and Taf1) and Taf1 also interacts with Ino80, another subunit of the Ino80 complex. Both subunits of the FACT complex Spt16 and Pob3 showed peptides enriched in one of the sgRNA-BirA*-dCas9 samples, and there are reported interactions of these proteins with several HATs including Gcn5<sup><a aria-label="Reference 47" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR47" id="ref-link-section-d8519934e2108" title="Lee, K. K. et al. Combinatorial depletion analysis to assemble the network architecture of the SAGA and ADA chromatin remodeling complexes. Mol. Syst. Biol. 7, 503 (2011).">47</a>,<a aria-label="Reference 50" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR50" id="ref-link-section-d8519934e2111" title="Leng, H. et al. FACT interacts with Set3 HDAC and fine-tunes GAL1 transcription in response to environmental stimulation. Nucleic Acids Res. 49, 5502–5519 (2021).">50</a></sup> and Rtt109<sup><a aria-label="Reference 51" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR51" id="ref-link-section-d8519934e2115" title="Yang, J. et al. The histone chaperone FACT contributes to DNA replication-coupled nucleosome assembly. Cell Rep. 14, 1128–1141 (2016).">51</a></sup>. Analysis of protein enrichment in the samples expressing sgRNAs revealed that some proteins were only enriched in one of the samples, or the enrichment of several proteins varied between samples. This may indicate steric label constraint or hindrance due to sgRNA localization on chromatin, BirA*-dCas9 fusion orientation and linker length (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">6</a>, Supplementary Data <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM4">4</a>–<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM4">6</a>).</p><div class="c-article-table" data-container-section="table" data-test="inline-table" id="table-1"><figure><figcaption class="c-article-table__figcaption"><b data-test="table-caption" id="Tab1">Table 1 Enriched biotinylated proteins associated with chromatin remodeling and Histone Acetyltransferase (HAT) complexes</b></figcaption><div class="u-text-right u-hide-print"><a aria-label="Full size table 1" class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" href="/articles/s41467-023-36391-6/tables/1" rel="nofollow"><span>Full size table</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>Taken together, our observations imply a probable role for Gcn5, Taf1, and Rtt109 in initiating the cascade of histone acetylation and other remodeling activities that results in increased histone exchange and decompaction of the <i>INO1</i> locus chromatin. That there are likely alternative sites of association of these HATs or other chromatin remodeling proteins is consistent with our observation that the GRSI, IIΔ double deletion only results in a partial decrease in histone H3 exchange at the active <i>INO1</i> locus.</p><h3 class="c-article__sub-heading" id="Sec8">Decompacted chromosome segments can undergo polymer-polymer phase separation</h3><p>Our results provide biochemical evidence for the mechanism by which changes in material properties of the activated <i>INO1</i> gene locus is initiated<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e2302" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a>,<a aria-label="Reference 19" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR19" id="ref-link-section-d8519934e2305" title="Bystricky, K., Heun, P., Gehlen, L., Langowski, J. &amp; Gasser, S. M. Long-range compaction and flexibility of interphase chromatin in budding yeast analyzed by high-resolution imaging techniques. Proc. Natl. Acad. Sci. USA 101, 16495–16500 (2004).">19</a></sup>. How then are these changes in material properties consistent with a phase transition that causes partitioning of the <i>INO1</i> locus to the NE? Would a change in properties of a single gene locus towards the middle of a chromosome arm cause such narrow localization of the locus to the NE? Phase separation of segments of polymers or polymer-polymer phase separation (PPPS) have been predicted based on Flory–Huggins theory, in which self-interacting ligand molecules occupy multiple sites throughout the polymer<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR52" id="ref-link-section-d8519934e2312" title="Dormidontova, E. E., Grosberg, A. Y. &amp; Khokhlov, A. R. Intramolecular phase separation of a copolymer chain with mobile primary structure. Macromol. Theory Simul. 1, 375–385 (1992).">52</a></sup>. Phase separation between two segments of the polymer will occur if the density of ligands decreases in one region. The high ligand occupancy region forms a dense phase through the high valency ligand-ligand interactions and the low-density segment forms an expanded, “dilute” phase.</p><p>Bead-spring models of chromatin provide the means to explore the physical properties of the chromatin fiber and to test hypotheses about mechanisms responsible for polymer, including DNA phase separation. For instance, as predicted by Flory–Huggins theory<sup><a aria-label="Reference 53" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR53" id="ref-link-section-d8519934e2319" title="Flory, P. J. &amp; Gee, G. Statistical thermodynamics of semi-flexible chain molecules. Proc. R. Soc. Lond. Ser. A. Math. Phys. Sci. 234, 60–73 (1956).">53</a>,<a aria-label="Reference 54" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR54" id="ref-link-section-d8519934e2322" title="Huggins, M. L. Thermodynamic properties of solutions of long-chain compounds. Ann. N. Y. Acad. Sci. 43, 1–32 (1942).">54</a></sup>, invoking dynamic chromosome cross-linkers (bound ligand) into a model of the yeast genome recapitulates the segregation and morphology of the nucleolus<sup><a aria-label="Reference 55" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR55" id="ref-link-section-d8519934e2326" title="Hult, C. et al. Enrichment of dynamic chromosomal crosslinks drive phase separation of the nucleolus. Nucleic Acids Res. 45, 11159–11173 (2017).">55</a></sup>. The physical basis for partitioning the <i>INO1</i> gene locus extends beyond recruitment to the NE. The radius of confinement and mobility of the locus are decreased, indicating a change in the polymeric state that accompanies its migration to the NE. While we can readily identify transcriptionally active domains relative to DNA sequence, we are just starting to appreciate how gene activity manifests in structural changes at the chromosome level. Active genes can be sequestered into chromosome loops (topologically associated domains) or into membrane-less compartments<sup><a aria-label="Reference 56" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR56" id="ref-link-section-d8519934e2333" title="Gibson, B. A. et al. Organization of chromatin by intrinsic and regulated phase separation. Cell 179, 470–484.e421 (2019).">56</a>,<a aria-label="Reference 57" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR57" id="ref-link-section-d8519934e2336" title="Sanulli, S. et al. HP1 reshapes nucleosome core to promote heterochromatin phase separation. Nature 575, 390–394 (2019).">57</a></sup>.</p><p>Although Flory–Huggins theory alone can explain segmental phase separation of a polymer as described above<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR52" id="ref-link-section-d8519934e2343" title="Dormidontova, E. E., Grosberg, A. Y. &amp; Khokhlov, A. R. Intramolecular phase separation of a copolymer chain with mobile primary structure. Macromol. Theory Simul. 1, 375–385 (1992).">52</a></sup>, we also wanted to test if we could reproduce the dynamic behaviors of the <i>INO1</i> locus observed experimentally and see whether segmental phase separation could occur by including parameters in our bead-spring model that are known to accompany changes in gene expression. One of these parameters is the persistence length of the chromatin fiber. As discussed above, persistence length (<i>L</i><sub><i>p</i></sub>) is the length scale over which the polymer tends to remain linear and resist bending (expected value of cosine of angle θ from the horizontal at length L over <i>L</i><sub><i>p</i></sub> falls off exponentially according to Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Equ2">2</a>)). For the large fraction of non-transcribed chromatin, persistence length measurements range in the hundreds of nanometers. In the case of transcribing gene loci or regions of reduced histone occupancy, these regions have a much shorter persistence length and therefore can explore a greater number of configurations<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR21" id="ref-link-section-d8519934e2366" title="Chi, Q., Wang, G. &amp; Jiang, J. The persistence length and length per base of single-stranded DNA obtained from fluorescence correlation spectroscopy measurements using mean field theory. Phys. A Stat. Mech. Appl. 392, 1072–1079 (2013).">21</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR22" id="ref-link-section-d8519934e2366_1" title="Rechendorff, K., Witz, G., Adamcik, J. &amp; Dietler, G. Persistence length and scaling properties of single-stranded DNA adsorbed on modified graphite. J. Chem. Phys. 131, 095103 (2009).">22</a>,<a aria-label="Reference 23" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR23" id="ref-link-section-d8519934e2369" title="Smith, S. B., Cui, Y. &amp; Bustamante, C. Overstretching B-DNA: the elastic response of individual double-stranded and single-stranded DNA molecules. Science 271, 795–799 (1996).">23</a></sup>.</p><div class="c-article-equation" id="Equ2"><div class="c-article-equation__content"><span class="mathjax-tex">$$\left\langle \cos \theta \right\rangle={e}^{-\frac{L}{{L}_{p}}}$$</span></div><div class="c-article-equation__number">
                    (2)
                </div></div><p>To simulate the experimental condition of active genes flanked by inactive genes, we tuned the model such that the midsection of beads representing active genes has the lowest bending stiffness, <i>L</i><sub><i>p</i></sub> = 5 nm, flanked by sections of inactive chromatin with <i>L</i><sub><i>p</i></sub> = 250 nm. The MSD of the 5 nm <i>L</i><sub><i>p</i></sub> segment was considerably reduced relative to the 250 nm <i>L</i><sub><i>p</i></sub> segment (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig6">6a</a>), recapitulating our experimental observations (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig2">2</a>). Likewise, the increase in <i>k</i><sub><i>s</i></sub> and reduced <i>R</i><sub><i>c</i></sub> of the 5 nm <i>L</i><sub><i>p</i></sub> segment further reflect our experimental results (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig6">6b, c</a>). The 5 nm <i>L</i><sub><i>p</i></sub> chain collapsed into a much smaller area compared to the 250 nm <i>L</i><sub><i>p</i></sub> segments, due to the increase in number of entropic states that they can explore. Consequently, the 5 nm <i>L</i><sub><i>p</i></sub> region appears spherical compared to the ellipsoid shape of the 250 nm <i>L</i><sub><i>p</i></sub> due to their increased randomness, and hence increased shape symmetry. The result is that the 250 nm <i>L</i><sub><i>p</i></sub> segments showed increased frequencies and amplitudes of expansion-contraction compared to the 5 nm <i>L</i><sub><i>p</i></sub> (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig6">6d</a>). These behaviors reflect our experimental observations of compression/expansion of the <i>INO1</i> locus in repressed versus activated states (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig2">2k</a>). Finally, simulations of chain dynamics reveal phase separation of the 5 and 250 nm <i>L</i><sub><i>p</i></sub> segments modeling the phase separation observed in the analytical model and in our experimental results for the active <i>INO1</i> locus (Supplementary Movie <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM5">1</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Elastic wormlike chain (bead-spring) modeling recapitulates active (Lp = 5) and repressed (Lp = 250) chromatin material properties." id="figure-6"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig6">Fig. 6: Elastic wormlike chain (bead-spring) modeling recapitulates active (Lp = 5) and repressed (Lp = 250) chromatin material properties.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-023-36391-6/figures/6" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig6_HTML.png?as=webp" type="image/webp"><img alt="figure 6" aria-describedby="Fig6" height="358" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_Fig6_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p><b>a</b> Aggregated MSDs for 26-bead subsegment within the high persistence length (L<sub>p</sub> = 250 nm) segment represents repressed chromatin (purple, <i>n</i> = 5 randomly seeded simulations), low persistence length (L<sub>p</sub> 5 nm) segment represents active chromatin (blue, <i>n</i> = 6 randomly seeded simulations). Data are presented as mean values+/− SEM. Decreasing L<sub>p</sub> to 5 nm reduces the radius of confinement (<b>b</b>) and increases apparent effective spring constant (k<sub>s</sub>) for the 26-bead-subsegment by 2-fold (<b>c</b>), consistent with experimental observations in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig2">2</a>b, f. <b>d</b> Aspect ratio of modeled fluorescent loci (occupied by the 26-beads) at 38 time points from the simulation averages. Source data are provided as a Source Data file. <b>e</b> Cartoon illustration of bead-spring model. At 10 nm rest length, springs experience no force. Beads are soft spheres 9 nm diameter that will resist compression via excluded volume force. <b>f</b> Hinge forces applied to beads proportional to varied persistence length will affect bead positions. In the box and whiskers plots (<b>b</b>, <b>c</b>) the median is indicated as middle line, 25th and 75th percentile as boxes and the whiskers represent minimum and maximum values.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 6" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure6 Full size image" data-track-label="button" href="/articles/s41467-023-36391-6/figures/6" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>We can reproduce segmental changes in dynamics and phase separation of a model polymer, we cannot explain the dynamics itself, which are known to be driven by active, ATP-dependent processes that remain obscure, but which we model as Brownian dynamics with tuning of temperature<sup><a aria-label="Reference 58" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR58" id="ref-link-section-d8519934e2621" title="Heun, P., Laroche, T., Shimada, K., Furrer, P. &amp; Gasser, S. M. Chromosome dynamics in the yeast interphase nucleus. Science 294, 2181–2186 (2001).">58</a>,<a aria-label="Reference 59" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR59" id="ref-link-section-d8519934e2624" title="Tortora, M. M., Salari, H. &amp; Jost, D. Chromosome dynamics during interphase: a biophysical perspective. Curr. Opin. Genet. Dev. 61, 37–43 (2020).">59</a></sup>.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec9-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec9">Discussion</h2><div class="c-article-section__content" id="Sec9-content"><p>Our results for <i>INO1</i> locus chromatin can now be put into molecular terms to explain the partitioning of the active gene to the NE. Recent evidence suggests that liquid-liquid phase separation (LLPS) of histones combines with PPPS of chromatin to form condensates, a potentially important mechanism by which the spatiotemporal organization of the genome can be regulated<sup><a aria-label="Reference 56" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR56" id="ref-link-section-d8519934e2640" title="Gibson, B. A. et al. Organization of chromatin by intrinsic and regulated phase separation. Cell 179, 470–484.e421 (2019).">56</a>,<a aria-label="Reference 57" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR57" id="ref-link-section-d8519934e2643" title="Sanulli, S. et al. HP1 reshapes nucleosome core to promote heterochromatin phase separation. Nature 575, 390–394 (2019).">57</a></sup>. Specifically, acetylation-deacetylation of the histone tails in nucleosomal arrays has been shown to underlie PPPS in which deacetylated histones induce formation of a protein-DNA condensate and acetylation causes its dissolution<sup><a aria-label="Reference 56" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR56" id="ref-link-section-d8519934e2647" title="Gibson, B. A. et al. Organization of chromatin by intrinsic and regulated phase separation. Cell 179, 470–484.e421 (2019).">56</a></sup>. We propose that the disruption of histone interactions that occurs due to acetylation in the active <i>INO1</i> locus causes it to “dissolve” out of the dense nucleoplasmic chromatin phase into the less dense chromatin phase adjacent to the NE, as predicted previously for a single copolymer chain<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR52" id="ref-link-section-d8519934e2654" title="Dormidontova, E. E., Grosberg, A. Y. &amp; Khokhlov, A. R. Intramolecular phase separation of a copolymer chain with mobile primary structure. Macromol. Theory Simul. 1, 375–385 (1992).">52</a></sup>.</p><p>The increased spring stiffness of the active <i>INO1</i> locus we observed in experiments and simulations also have implications for LLPS of proteins associated with sub-compartmentalization of different processes in the nucleus. The contribution of PPPS to formation of chromatin-containing condensates has been considered both theoretically<sup><a aria-label="Reference 60" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR60" id="ref-link-section-d8519934e2664" title="Erdel, F. &amp; Rippe, K. Formation of chromatin subcompartments by phase separation. Biophysical J. 114, 2262–2270 (2018).">60</a></sup> and most recently, demonstrated both in vivo and in vitro<sup><a aria-label="Reference 61" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR61" id="ref-link-section-d8519934e2668" title="Shin, Y. et al. Liquid nuclear condensates mechanically sense and restructure the genome. Cell 175, 1481–1491.e1413 (2018).">61</a>,<a aria-label="Reference 62" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR62" id="ref-link-section-d8519934e2671" title="Quail, T. et al. Force generation by protein–DNA co-condensation. Nat. Phys. 17, 1007–1012 (2021).">62</a></sup>. Quail et al. recently showed that LLPS of a transcription factor required it to be bound to DNA in a tension-dependent manner. They presented evidence to support the postulate that less dense DNA forms an ideal template for nucleating LLPS of the bound protein and that protein-DNA co-condensation is associated with the difference in chemical potential of the DNA, as reflected in its difference in spring stiffness. Shin et al. observed that light-activated condensates in the nucleus selectively formed in regions of low-density chromatin as observed in vitro<sup><a aria-label="Reference 61" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR61" id="ref-link-section-d8519934e2675" title="Shin, Y. et al. Liquid nuclear condensates mechanically sense and restructure the genome. Cell 175, 1481–1491.e1413 (2018).">61</a>,<a aria-label="Reference 62" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR62" id="ref-link-section-d8519934e2678" title="Quail, T. et al. Force generation by protein–DNA co-condensation. Nat. Phys. 17, 1007–1012 (2021).">62</a></sup>. Ribosomal DNA (rDNA) is another example of this phenomenon. rDNA is essential for the phase separation of the nucleolus<sup><a aria-label="Reference 63" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR63" id="ref-link-section-d8519934e2682" title="Lawrimore, J. et al. The rDNA is biomolecular condensate formed by polymer-polymer phase separation and is sequestered in the nucleolus by transcription and R-loops. Nucleic Acids Res. 49, 4586–4598 (2021).">63</a></sup>. When rDNA was made discontinuous (rDNA genes were expressed from individual plasmids versus genome repeats), nucleolus formation was disrupted upon transcriptional inactivation<sup><a aria-label="Reference 63" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR63" id="ref-link-section-d8519934e2687" title="Lawrimore, J. et al. The rDNA is biomolecular condensate formed by polymer-polymer phase separation and is sequestered in the nucleolus by transcription and R-loops. Nucleic Acids Res. 49, 4586–4598 (2021).">63</a></sup>. That is, rDNA is a PPPS that relies on transcription to remain phase separated from the bulk genome. Thus, combined protein-DNA LLPS-PPPS of chromatin could facilitate the nucleation of co-condensates that have been associated with transcription and its regulation<sup><a aria-label="Reference 64" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR64" id="ref-link-section-d8519934e2691" title="Henninger, J. E. et al. RNA-mediated feedback control of transcriptional condensates. Cell 184, 207–225 e224 (2021).">64</a>,<a aria-label="Reference 65" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR65" id="ref-link-section-d8519934e2694" title="Sabari, B. R. et al. Coactivator condensation at super-enhancers links phase separation and gene control. Science 361, eaar3958 (2018).">65</a></sup>, heterochromatin formation<sup><a aria-label="Reference 66" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR66" id="ref-link-section-d8519934e2698" title="Rudnizky, S. et al. Extended and dynamic linker histone-DNA Interactions control chromatosome compaction. Mol. Cell 81, 3410–3421 e3414 (2021).">66</a>,<a aria-label="Reference 67" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR67" id="ref-link-section-d8519934e2701" title="Strom, A. R. et al. Phase separation drives heterochromatin domain formation. Nature 547, 241–245 (2017).">67</a></sup>, nucleolus assembly<sup><a aria-label="Reference 68" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR68" id="ref-link-section-d8519934e2705" title="Quinodoz, S. A. &amp; Guttman, M. Essential roles for RNA in shaping nuclear organization. Cold Spring Harb. Perspect. Biol. 14, a039719 (2021).">68</a></sup> and DNA repair<sup><a aria-label="Reference 69" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR69" id="ref-link-section-d8519934e2709" title="Pessina, F. et al. Functional transcription promoters at DNA double-strand breaks mediate RNA-driven phase separation of damage-response factors. Nat. Cell Biol. 21, 1286–1299 (2019).">69</a></sup>.</p><p>The work done to sub-compartmentalize the <i>INO1</i> and other active gene loci could also be driven by PPPS. Partitioning of the repressed locus into the nucleoplasm phase is driven by transduction of the chemical potential of PPPS into mechanical work on the non-condensed DNA<sup><a aria-label="Reference 62" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR62" id="ref-link-section-d8519934e2719" title="Quail, T. et al. Force generation by protein–DNA co-condensation. Nat. Phys. 17, 1007–1012 (2021).">62</a></sup>. The work done in sub-compartmentalizing the active <i>INO1</i> locus to the nuclear periphery is simply equal and opposite to that driving condensation. It is also possible that the <i>INO</i><i>1</i> chromatin could nucleate LLPS of associated proteins combined with PPPS, resulting in a DNA-protein condensate.</p><p>The polymer modeling provides critical insight into mechanisms of translocation of an active locus. The means by which the sub-compartment comes to reside at the nuclear periphery is that it is seeking the most thermodynamically favored state. Chains with self-similar properties tend to segregate as mixing of two chains is thermodynamically disfavored in a confined space, such as the nucleus<sup><a aria-label="Reference 55" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR55" id="ref-link-section-d8519934e2734" title="Hult, C. et al. Enrichment of dynamic chromosomal crosslinks drive phase separation of the nucleolus. Nucleic Acids Res. 45, 11159–11173 (2017).">55</a>,<a aria-label="Reference 70" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR70" id="ref-link-section-d8519934e2737" title="Jun, S. &amp; Mulder, B. Entropy-driven spatial organization of highly confined polymers: lessons for the bacterial chromosome. Proc. Natl. Acad. Sci. USA 103, 12388–12393 (2006).">70</a></sup>. In the case of the nucleus, segregation of dissimilar chains leads to their migration to the furthest distance from each other, which is the outer edge of the nucleus at the nuclear envelope.</p><p>Finally, the ability to partition the genome into discrete domains that transition between physical states, either through multivalent binding of DNA-associated proteins (dense and deformable) or disruption of histone association (dilute but less deformable active subcompartments) could lead to architectural changes that can be understood from the material properties of the components, as we have demonstrated in this study.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec10-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec10">Methods</h2><div class="c-article-section__content" id="Sec10-content"><h3 class="c-article__sub-heading" id="Sec11">Strains and plasmids</h3><p>The LMY52 strain<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR12" id="ref-link-section-d8519934e2757" title="Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. Dev. Cell 22, 1234–1246 (2012).">12</a></sup> was used as a control for the statistical mapping and for the gene tracking experiments (provided by Jason Brickner, Northwestern University, USA) (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">1</a>). This strain contains a cassette of 128 repeats of the Lac operator binding site (LacO) into the <i>INO1</i> locus and expresses a GFP-Lac repressor (LacI), which binds to the LacO repeats. This strain also expresses the Nuclear Pore Complex (NPC) protein Nup49 fused to GFP and the nucleolar protein Nop1 fused to mCherry as reference markers for the NE and the nucleolus, respectively.</p><p>GRS I and II deletion mutants used for the statistical mapping and for the gene tracking experiments were generated in the LMY52 strain by employing a CRISPR-Cas9 based strategy developed by and with reagents provided by Tom Ellis at Imperial College London, UK (Shaw et al.<sup><a aria-label="Reference 71" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR71" id="ref-link-section-d8519934e2770" title="Shaw, W. M. et al. Engineering a model cell for rational tuning of GPCR signaling. Cell 177, 782–796.e727 (2019).">71</a></sup>, Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">2</a>). This system consists of a sgRNA entry vector and a range of Cas9-sgRNA expression vectors with different selection markers that are transformed into yeast cells together with donor DNA fragments containing the desired sequence changes. In this case, each sgRNA entry vector (pWS082) was completed using small fragment Golden Gate assembly<sup><a aria-label="Reference 72" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR72" id="ref-link-section-d8519934e2777" title="Lee, M. E., DeLoache, W. C., Cervantes, B. &amp; Dueber, J. E. A highly characterized yeast toolkit for modular, multipart assembly. ACS Synth. Biol. 4, 975–986 (2015).">72</a></sup> with two 26-mer oligos each containing 20 nucleotide guide sequences selected to target each GRS to be deleted. Once assembled, the plasmid was digested with EcoRV to generate a linear sgRNA expression cassette. This linearized cassette contained homology arms to the Cas9-sgRNA expression vector that served for gap repair of the linearized plasmid in yeast to reconstitute a stable Cas9-sgRNA expression vector. The Cas9-sgRNA expression vector selected for each GRS was also linearized with a BsmBI restriction digest and gel purified. The different markers on each Cas9-sgRNA vector allowed selection for successfully transformed cells expressing both Cas9 under the strong <i>PGK1</i> promoter and a sgRNA. Each linear sgRNA expression cassette and the linear Cas9-sgRNA expression vector were transformed into LMY52 together with the corresponding donor sequence consisting of a double stranded DNA sequence of around 100 bp obtained by synthesizing and annealing mutagenic primers containing the corresponding GRS mutated sites flanked by homology arms to the integration region at the <i>INO1</i> locus.</p><p>The YAN1001 strain<sup><a aria-label="Reference 28" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR28" id="ref-link-section-d8519934e2790" title="Rufiange, A., Jacques, P. E., Bhat, W., Robert, F. &amp; Nourani, A. Genome-wide replication-independent histone H3 exchange occurs predominantly at promoters and implicates H3 K56 acetylation and Asf1. Mol. Cell 27, 393–405 (2007).">28</a></sup> (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">1</a>) was used as control to measure the incorporation of newly synthesized histone H3 into nucleosomes (provided by Amine Nourani, Université Laval, Canada). In this strain, endogenous genes for histones H3 and H4 are deleted and expressed in two different ways: 1, constitutively from a plasmid, under the control of their own promoter, and 2, from another region in the genome under the control of the <i>GAL1-10</i> promoter, with H3 fused to a FLAG tag. This allows to measure incorporation of newly synthesized H3-FLAG into nucleosomes following galactose induction. GRS I and II (GRSI, IIΔ) deletion mutants (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">1</a>) were generated in the YAN1001 by the CRISPR-Cas9 strategy described above.</p><p>To measure the effect of rapid depletion of different HATs on active <i>INO1</i> localization to the NE, 12 known yeast HAT catalytic subunits were tagged with a degron peptide IAA17 (71/116) that targets the protein for proteasomal degradation in the presence of the plant hormone auxin<sup><a aria-label="Reference 32" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR32" id="ref-link-section-d8519934e2809" title="Morawska, M. &amp; Ulrich, H. D. An expanded tool kit for the auxin-inducible degron system in budding yeast. Yeast 30, 341–351 (2013).">32</a></sup>. First, the <i>TIR1</i> gene that mediates the auxin-IAA degradation was integrated into the genome of the LMY52 strain by transformation with the Yeast Integrating plasmid YIp204-PADH1-atTIR1-9myc (Addgene #99532, Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">2</a>). This strain was used as background to tag each HAT 3’ to the coding sequence to produce C-terminal fusion products with a cassette containing the IAA17 degron sequence as well as six FLAG epitopes amplified from a plasmid generated in a previous study (pHyg–AID<sup>*</sup>–6FLAG, Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">2</a>)<sup><a aria-label="Reference 32" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR32" id="ref-link-section-d8519934e2825" title="Morawska, M. &amp; Ulrich, H. D. An expanded tool kit for the auxin-inducible degron system in budding yeast. Yeast 30, 341–351 (2013).">32</a></sup>. These FLAG tags allowed us to verify the efficiency of degradation by Western Blot.</p><p>BirA*-dCas9 fusions to identify candidate GRS-binding proteins and specifically, histone acetyl transferases (HATs), were generated based on a previously described Proximity-dependent Biotin Identification (BioID) strategy that was successful in <i>S. cerevisiae</i><sup><a aria-label="Reference 43" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR43" id="ref-link-section-d8519934e2835" title="Opitz, N. et al. Capturing the Asc1p/receptor for activated C kinase 1 (RACK1) microenvironment at the head region of the 40 S ribosome with quantitative BioID in yeast. Mol. Cell. Proteom. MCP 16, 2199–2218 (2017).">43</a></sup>. The BirA* ligase sequence was provided in a plasmid vector by Oliver Valerius at Georg-August-University, Germany (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">2</a>). dCas9 mutant was first obtained in one of the Cas9-sgRNA expression vectors (pWS174) to introduce the mutations D10A and H840A resulting in a catalytically dead (d)Cas9. Then, by linearizing this vector at the selected fusion site and using the GeneArt Seamless Cloning and Assembly kit (ThermoFisher Scientific), the sequence encoding for BirA* with specific linker sequences (linker 1: GSSGSSGSSGSS or linker 2: LERPPLCWISAEFHHTGLVDPSSVPSLSLNRGSGSGSG) were fused 5’ or 3’ to the dCas9 gene to produce N-terminal or C-terminal fusion products, respectively. Once we obtained the desired BirA*-dCas9 expression vectors, the same procedure was performed as described above for CRISPR/Cas9 genome editing by transforming both BirA*-dCas9 and sgRNA linearized cassettes into the LMY52 strain to obtain the strains used in this approach (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">1</a>), this time without the need of using any donor DNA sequence. We also transformed the LMY52 strain with the different BirA*-dCas9 expression vectors alone with empty sgRNA cassettes as negative controls for non-targeted biotinylation of proteins.</p><h3 class="c-article__sub-heading" id="Sec12">Statistical mapping of the <i>INO1</i> locus</h3><p>Statistical mapping of the <i>INO1</i> locus was performed as previously described<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR12" id="ref-link-section-d8519934e2860" title="Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. Dev. Cell 22, 1234–1246 (2012).">12</a></sup>. Briefly, overnight cultures in synthetic, defined medium (SDC) without inositol or SDC containing 100 µM of inositol were diluted to 10<sup>6</sup> cells/mL and after two generations cells were immobilized on Concanavalin A-coated well slides<sup><a aria-label="Reference 73" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR73" id="ref-link-section-d8519934e2866" title="Malleshaiah, M. K., Shahrezaei, V., Swain, P. S. &amp; Michnick, S. W. The scaffold protein Ste5 directly controls a switch-like mating decision in yeast. Nature 465, 101–105 (2010).">73</a></sup>. Images were acquired with a Zeiss Axio Observer Z1 Yokogawa spinning disk confocal microscope using a 100× 1.40 NA oil objective. Z-stacks with a slice spacing of 250 nm were taken at an acquisition time of 50 ms for GFP (50%, 3 mW 488 nm excitation), and 100 ms for mCherry (50%, 3.3 mW 561 nm excitation). Images were processed by nucloc software modified to display probability density maps as percentiles using a kernel density estimate<sup><a aria-label="Reference 74" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR74" id="ref-link-section-d8519934e2870" title="Therizols, P., Duong, T., Dujon, B., Zimmer, C. &amp; Fabre, E. Chromosome arm length and nuclear constraints determine the dynamic relationship of yeast subtelomeres. Proc. Natl. Acad. Sci. USA 107, 2025–2030 (2010).">74</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec13">Particle tracking of the <i>INO1</i> locus</h3><p>To track the <i>INO1</i> locus in live yeast cells, LMY52 and GRS I or II mutant strains containing the gene tagged with the LacO array and expressing GFP-LacI as well as the NE tagged with Nup49-GFP were used. Images of the cells were also recorded on a Zeiss spinning disk Axio Observer Z1 confocal microscope at an acquisition time of 50 ms (50%, 3 mW 488 nm excitation) for 1 min at an interval of 500 ms. To check for possible stage-drift the GFP signal was also imaged in fixed cells.</p><p>Images of GFP foci corresponding to the <i>INO1</i> locus were automatically tracked using the WaveTracer tool in MetaMorph software<sup><a aria-label="Reference 75" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR75" id="ref-link-section-d8519934e2895" title="Kechkar, A., Nair, D., Heilemann, M., Choquet, D. &amp; Sibarita, J. B. Real-time analysis and visualization for single-molecule based super-resolution microscopy. PLoS One 8, e62918 (2013).">75</a></sup>. The algorithm segments circular objects with a particular size and intensity cut-off and then calculates the centroid position for each object in each frame through Gaussian fitting. For each locus monitored, the software quantifies the movement and calculates mean squared displacement (MSD). Only cells whose MSD curves exhibited a linear slope were included in subsequent analysis.</p><p>To calculate the radius of confinement, the MSD plateau values were used in each case since the plateau reached is proportional to the square of <i>R</i><sub><i>c</i></sub>, as previously described (Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Equ3">3</a>))<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e2911" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a>,<a aria-label="Reference 25" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR25" id="ref-link-section-d8519934e2914" title="Neumann, F. R. et al. Targeted INO80 enhances subnuclear chromatin movement and ectopic homologous recombination. Genes Dev. 26, 369–383 (2012).">25</a></sup>.</p><div class="c-article-equation" id="Equ3"><div class="c-article-equation__content"><span class="mathjax-tex">$${R}_{c}=\frac{5}{4}\sqrt{{{MSD}}_{{plateau}}}$$</span></div><div class="c-article-equation__number">
                    (3)
                </div></div><p>or independently, from the standard deviation of spot positions, σ, and the average squared deviation from the mean position, <span class="mathjax-tex">\(\left\langle \varDelta {r}_{0}^{2}\right\rangle\)</span>, by applying the equipartition theorem (Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Equ4">4</a>))<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e3045" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a>,<a aria-label="Reference 26" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR26" id="ref-link-section-d8519934e3048" title="Scheffold, F. et al. Brushlike interactions between thermoresponsive microgel particles. Phys. Rev. Lett. 104, 128304 (2010).">26</a>,<a aria-label="Reference 27" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR27" id="ref-link-section-d8519934e3051" title="Uhlenbeck, G. E. &amp; Ornstein, L. S. On the theory of the brownian motion. Phys. Rev. 36, 823–841 (1930).">27</a></sup>.</p><div class="c-article-equation" id="Equ4"><div class="c-article-equation__content"><span class="mathjax-tex">$${R}_{c}=\frac{5}{4}\sqrt{2{\sigma }^{2}+\left\langle \varDelta {r}_{0}^{2}\right\rangle }$$</span></div><div class="c-article-equation__number">
                    (4)
                </div></div><p>Effective spring constants (<i>k</i><sub><i>s</i></sub>) for the <i>INO1</i> locus in repressed versus activated conditions were determined by using the equipartition theorem to measure the standard deviation (σ) of each step from the mean position to calculate <i>k</i><sub><i>s</i></sub> (Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Equ5">5</a>))<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e3171" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a>,<a aria-label="Reference 26" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR26" id="ref-link-section-d8519934e3174" title="Scheffold, F. et al. Brushlike interactions between thermoresponsive microgel particles. Phys. Rev. Lett. 104, 128304 (2010).">26</a></sup>.</p><div class="c-article-equation" id="Equ5"><div class="c-article-equation__content"><span class="mathjax-tex">$${k}_{s}=\frac{{k}_{B}T}{{\sigma }^{2}}$$</span></div><div class="c-article-equation__number">
                    (5)
                </div></div><p>Statistical differences for <i>R</i><sub><i>c</i></sub> and effective <i>k</i><sub><i>s</i></sub> in strains grown under different conditions were determined by Student’s <i>t</i> tests.</p><p>To analyze the rate and amplitude of <i>INO1</i> locus expansion-contraction, the GFP array was also acquired at the same frequency as mentioned before but using a Zeiss Elyra PS.1 system. SIM images were acquired in this case with a 63× 1.40 NA oil objective in the GFP channel (15 %, 20 mW 488 nm HR diode laser), exposure time 24 ms and 3 rotations. Each locus was then fitted to an ellipsoid function to quantify the ratio between long and short axes by using the Fiji software<sup><a aria-label="Reference 76" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR76" id="ref-link-section-d8519934e3272" title="Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. Nat. Methods 9, 676–682 (2012).">76</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec14">Chromatin polymer simulations</h3><p>An elastic worm-like chain consisting of 201 beads (masses) connected by linear springs was sectored into three equal segments: two flanking segments composed of beads 1–67 and 135–201, and one middle segment with beads 68–134. Initially the chain was homogenized with <i>L</i><sub><i>p</i></sub> for all segments set to 250 nm (in <i>n</i> = 5 randomly seeded simulations). In the experimental condition we lowered the <i>L</i><sub><i>p</i></sub> for the middle segment to 5 nm (in <i>n</i> = 6, randomly seeded simulations) by adjusting the value of hinge force on the beads. Whenever three beads (or two springs) are not colinear, hinge forces act to restore the positions of the beads to lie on a line (see Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig6">6e, f</a>). Hinge forces are based on the Euler–Bernoulli formula (Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Equ6">6</a>)), where <i>E</i> is Young’s modulus, proportional to a polymer’s bending stiffness, and is linearly related to the persistence length (for more details see methods Lawrimore et al.<sup><a aria-label="Reference 77" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR77" id="ref-link-section-d8519934e3312" title="Lawrimore, J. et al. ChromoShake: a chromosome dynamics simulator reveals that chromatin loops stiffen centromeric chromatin. Mol. Biol. Cell 27, 153–166 (2016).">77</a></sup>). In our approach the Young’s modulus (E) in Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Equ6">6</a>) is varied only in calculation of bending stiffness, and does not affect the tensile stiffness of individual DNA springs. Hinge forces were manually set to constant <i>L</i><sub><i>p</i></sub> = 250 nm for the flanking segments, and set to either <i>L</i><sub><i>p</i></sub> = 250 nm and <i>L</i><sub><i>p</i></sub> = 5 nm for the variable middle segment.</p><div class="c-article-equation" id="Equ6"><div class="c-article-equation__content"><span class="mathjax-tex">$${{{{{{\rm{F}}}}}}}_{{{{{{\rm{h}}}}}}}=\frac{4{EI}\,\theta }{{d}^{2}}$$</span></div><div class="c-article-equation__number">
                    (6)
                </div></div><p>In the simulations we obtained 35,000 timepoints of bead position data (x<sub>t</sub>, y<sub>t</sub>, z<sub>t</sub>) for a duration of 70 ms at 2 μsec resolution. With a working estimate of cellular nuclear viscosity of 141 Pa<sup><a aria-label="Reference 78" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR78" id="ref-link-section-d8519934e3413" title="Fisher, J. K. et al. DNA relaxation dynamics as a probe for the intracellular environment. Proc. Natl. Acad. Sci. USA 106, 9250–9255 (2009).">78</a></sup> the model’s parameters reflected a fluid environment set to 0.01 Pa allowing a 14,100X speedup of physical time for the simulation. An equilibration time corresponding to 25% of the total trajectories was set to allow for meaningful chain configurations from its starting linear conformation. Second, to eliminate the effects of drift on the entire chain, we considered individual bead positions relative to bead number 55 within the invariable portion of the chain (<i>L</i><sub><i>p</i></sub> 250 nm segments) throughout our analyses. Positions of a center of mass of a 26-bead long subsegment (beads 90–115) within the middle segment representing a fluorescent segment (virtual LacO array segment), were tracked for each of 15,000 time intervals (t values) amounting to 35 ms of simulation time (equivalent to 423 s of real time), and MSD was computed for each independent experiment in MATLAB with <i>computespotmsd.m</i> (Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Equ7">7</a>)). Average MSD with standard error bars for a spot (26-bead subsegment) within <i>L</i><sub><i>p</i></sub> 250 (<i>n</i> = 5) compared to spot within <i>L</i><sub><i>p</i></sub> 5 nm (<i>n</i> = 6) chain segments are displayed in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig6">6a</a>.</p><div class="c-article-equation" id="Equ7"><div class="c-article-equation__content"><span class="mathjax-tex">$${MSD}=\left\langle {\left|{x}_{t}-{x}_{o}\right|}^{2}\right\rangle$$</span></div><div class="c-article-equation__number">
                    (7)
                </div></div><p>We computed <span class="mathjax-tex">\({R}_{c}\)</span>, for the simulated with Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Equ4">4</a>)<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR17" id="ref-link-section-d8519934e3562" title="Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. Mol. Cell 52, 819–831 (2013). S1097-2765(13)00787-9 [pii].">17</a></sup>.</p><div class="c-article-equation" id="Equ8"><div class="c-article-equation__content"><span class="mathjax-tex">$${R}_{c}=\frac{5}{4}\sqrt{2{\sigma }^{2}+\left\langle \varDelta {r}_{0}^{2}\right\rangle }$$</span></div><div class="c-article-equation__number">
                    (8)
                </div></div><p>Considering 35 ms of simulation time and using the standard deviation of spot position, <i>σ</i>, and the averaged square deviations from the mean position, <span class="mathjax-tex">\(\left\langle \varDelta {r}_{0}^{2}\right\rangle\)</span> we compared spatial confinements of <i>L</i><sub><i>p</i></sub> 250 nm (<i>n</i> = 5) to <i>L</i><sub><i>p</i></sub> 5 nm segments (<i>n</i> = 6). Effective spring constants were estimated by using the standard deviations obtained from simulation’s spot positions, σ, as in Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Equ5">5</a>).</p><div class="c-article-equation" id="Equ9"><div class="c-article-equation__content"><span class="mathjax-tex">$${k}_{s}=\frac{{k}_{B}T}{{\sigma }^{2}}$$</span></div><div class="c-article-equation__number">
                    (9)
                </div></div><p>We used principal component analysis to estimate 2D shapes of simulated fluorescent spots (26-beads) to obtain their aspect ratios. Given averaged bead position data at 38 regularly spaced time intervals we used MATLAB <i>aspect_ratio.m</i> to calculate the aspect ratio for the point cloud representing the fluorescent spot (26-bead subsegment) by taking the square root of the ratio of the first principal component’s scaling factor over the second.</p><h3 class="c-article__sub-heading" id="Sec15">ChIP experiments</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec16">Cell cultures and crosslinking</h4><p>ChIP experiments were performed in four biological replicates. YAN1001 or the corresponding GRS mutant strains were inoculated on 100 mL SDC + 2% glucose with or without 100 µM inositol medium and incubated until they reached OD<sub>600</sub> 0.2–0.3. Cells were recovered by centrifugation and switched to 100 mL medium with 2% raffinose and incubated overnight. In the morning, cell cycle was arrested with 5 mM α-factor for 3.5 h and inositol was added again to 100 µM inositol final concentration to the repressed cells. Efficient G1 arrest (at least 90% of cells) was confirmed by microscope as percentage of cells with shmoo morphology. Cells were collected by centrifugation at 2000 × <i>g</i>, 5 min switched to 2% galactose medium and incubated for 1 h. At every step, OD<sub>600</sub> cells were grown only long enough to reach values of &lt;0.5, to avoid losing the repressed phenotype.</p><p>After 1 hour incubation in galactose, cells were crosslinked with 1 % formaldehyde for 30 min with gentle rotation at room temperature. To quench formaldehyde, 2.5 M glycine was added to a final concentration of 125 mM. Crosslinked cells were recovered by centrifugation and washed twice with 25 mL ice cold 1X Tris-buffered saline (TBS: 20 mM Tris, 150 mM NaCl, pH 7.6). Pellets were resuspended in the remaining liquid and transferred to 1.5 mL tubes. Cells were harvested by centrifugation at 2000 × <i>g</i>, 4 °C, 5 min, immediately frozen in liquid nitrogen and stored at −80 °C until processed.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec17">Chromatin immunoprecipitation (ChIP) and quantitative PCR (qPCR)</h4><p>Immunoprecipitation assays were performed as previously described<sup><a aria-label="Reference 79" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR79" id="ref-link-section-d8519934e3839" title="Jeronimo, C., Poitras, C. &amp; Robert, F. Histone recycling by FACT and Spt6 during transcription prevents the scrambling of histone modifications. Cell Rep. 28, 1206–1218.e1208 (2019).">79</a></sup> with slight modifications. Cells were resuspended in 700 µL of lysis buffer (50 mM HEPES-KOH pH 7.5, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% Na-deoxycholate, 1 mM PMSF, 1 mM Benzamidine, 10 μg/mL Aprotinin, 1 μg/mL Leupeptin, 1 μg/mL Pepstatin) and mechanically lysed with glass bead beating. Cells were spun at 5000 × <i>g</i> for 10 s and supernatants were transferred to new tubes. Cells were sonicated in a Sonic dismembrator Model 100 equipped with a micro probe (Fisher Scientific), 4 × 20 s at output of 7 Watts, with a 1-minute break between sonication cycles. Supernatants containing solubilized chromatin fragments were recovered by centrifugation at 16,000 × <i>g</i>, 5 min and transferred to new tubes. An aliquot of 5 µL (1 %) per sample of this whole-cell extract was kept at −20 °C to be used as input sample, which represents the amount of chromatin used in the ChIP, usually 1% of starting chromatin. 500 µL of the remaining material was used for immunoprecipitation (IP). For each IP sample, 50 µL of Dynabeads™ Pan Mouse IgG (ThermoFisher Scientific) pre-coupled to 5 µg of mouse Monoclonal Anti-FLAG M2 antibody (Sigma) was added to the 500 µL of chromatin sample and incubated overnight at 4 °C.</p><p>Beads were washed twice with Lysis buffer, twice with Lysis buffer 500 (Lysis buffer + 360 mM NaCl), twice with Wash buffer (10 mM Tris-HCl pH 8.0, 250 mM LiCl, 0.5% NP40, 0.5% Na-deoxycholate, 1 mM EDTA) and once with TE (10 mM Tris-HCl pH 8.0, 1 mM EDTA). Immunoprecipitated chromatin was eluted and reverse-crosslinked with 50 μL TE/SDS (TE + 1% SDS) by incubating overnight at 65 °C. Input samples were also reverse-crosslinked with 50 μL TE/SDS and incubated overnight at 65 °C. Eluted chromatin as well as Input samples were treated with RNase A (345 μL TE, 3 μL 10 mg/mL RNase A, 2 μL 20 mg/mL Glycogen) at 37 °C, for 2 h and subsequently digested with Proteinase K (15 μL 10% SDS, 7.5 μL 20 mg/mL Proteinase K) at 37 °C for 2 h. Samples were extracted twice with phenol/chloroform/isoamyl alcohol (25:24:1) and precipitated with 200 mM NaCl and 70% EtOH. Precipitated DNA was resuspended in 50 μL of TE before being used for ChIP-qPCR experiments.</p><p>The immunoprecipitated DNA and DNA from 1% of the Input samples were analyzed by quantitative PCR on a ViiA 7 Real-Time PCR System (Thermo Fisher Scientific) using the PowerUp™ SYBR™ Green Master (Thermo Fisher Scientific). The amount of newly synthesized FLAG-H3 incorporated at the nucleosomes was calculated as the percent of Input using the following formula: 100*2ˆ(Ct<sub>Input</sub>−6.644-Ct<sub>IP</sub>), Ct being the value where the PCR curve crosses the threshold line above background levels of detection. Subtracting 6.644 from Ct<sub>Input</sub> allows to adjust for the fact that 1 % of Input was used. In this case since the starting input fraction is 1%, then a dilution factor of 100 or 6.644 cycles (log<sub>2</sub> of 100) is subtracted from the Ct value of diluted input. Primer sequences for the regions analyzed are described in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">4</a>.</p><h3 class="c-article__sub-heading" id="Sec18">HATs depletion using an auxin degron system</h3><p>All 12 HAT-AID fusion strains (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">1</a>) were generated to rapidly deplete specific HAT activity and the <i>wild type</i> cells were cultured in SDC medium without inositol until OD<sub>600</sub> 0.3. At this point, cultures were split in two and auxin (I2886, Sigma-Aldrich) was added to one of the tubes for each sample to a final concentration of 500 μM. Cells were incubated with auxin for 1 h 30 min followed by immobilization on ConA-coated Nunc 96 well Optical-Bottom plates (164588, ThermoFisher). Imaging and nucloc analysis were performed as previously described<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR12" id="ref-link-section-d8519934e3883" title="Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. Dev. Cell 22, 1234–1246 (2012).">12</a></sup> for the Statistical Mapping of the <i>INO1</i> locus <i>wild type</i> and the GRS mutants. Western Blots were performed on HAT-AID fusion strains to verify the degradation of the corresponding HAT once the auxin was added to growth medium.</p><p>ChIP-qPCR experiments for H3K14ac and H3 in the AID strains were performed in three biological replicates similarly as described before for the H3-FLAG with modifications for the sample preparation and the IP procedures. In this case, cells were grown under the same conditions as described for the AID experiments before crosslinking. For the IP protocols, 50 μL of Dynabeads™ Protein G beads (ThermoFisher Scientific) pre-coupled to 2 µL of rabbit anti-H3K14ac (Millipore) or 2 µg of rabbit anti-H3 (Abcam) were used per IP sample.</p><h3 class="c-article__sub-heading" id="Sec19">Reverse transcriptase real-time quantitative PCR (RT-qPCR)</h3><p>Cells were prepared under the same conditions described before for AID and inositol starvation. Total RNA was isolated from yeast cells using TRIzol reagent (Invitrogen) following manufacturer’s instructions. Approximately 2 µg of total RNA was reverse transcribed using All-In-One 5X RT MasterMix (abm). qPCR reactions were performed using FastStart Essential DNA Green Master (Roche Diagnostics) and gene expression levels for <i>INO1</i> and proximal loci (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">4</a>) were determined with LightCycler 96 (software version 1.1). All assays were performed in biological triplicates.</p><h3 class="c-article__sub-heading" id="Sec20">dCas9-BioID assays</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec21">Cell cultures and streptavidin affinity purification</h4><p>The protocol for biotinylated proteins purification was modified from Opitz et al. <sup><a aria-label="Reference 43" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR43" id="ref-link-section-d8519934e3923" title="Opitz, N. et al. Capturing the Asc1p/receptor for activated C kinase 1 (RACK1) microenvironment at the head region of the 40 S ribosome with quantitative BioID in yeast. Mol. Cell. Proteom. MCP 16, 2199–2218 (2017).">43</a></sup>. LMY52 <i>wild type</i> and LMY52-BirA*-dCas9 with or without sgRNA mutants generated (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">3</a>) were inoculated in 4 L of SDC medium without inositol plus D-biotin (B4639, Sigma-Aldrich) added to final concentration of 10 μM. Cells were grown to OD<sub>600</sub> of 0.8 and harvested by filtration using a Kontes™ Ultra-Ware™ Microfiltration Assembly with Fritted Glass Support (90 mm) and MF-Millipore™ Membrane Filters, 0.45 µm pore size (HAWP0900 Millipore). Pellets were immediately frozen in liquid nitrogen and kept at −80 °C. Cell lysis was performed under cryogenic conditions using solid phase milling in a planetary ball mill (Retsch PM 100) producing a fine cell grindate<sup><a aria-label="Reference 80" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR80" id="ref-link-section-d8519934e3935" title="Oeffinger, M. et al. Comprehensive analysis of diverse ribonucleoprotein complexes. Nat. Methods 4, 951–956 (2007).">80</a></sup>. Grindate samples were stored at −80 °C until processed. Biotinylated proteins were purified using streptavidin-sepharose beads (GE Healthcare, 17-5113-01). For each sample, 0.7 g of crude extract was resuspended in 6.3 mL of buffer A: commercial buffer W (100 mM Tris-Cl, 150 M NaCl, pH 8, 2-1003-100, IBA) containing 0.1 % SDS, 0.5 % Sodium Deoxycholate, and protease inhibitors mixture (cOmplete™, Mini, EDTA-free Protease Inhibitor Cocktail, 11836170001, Roche). Samples were homogenized with a vortex agitator, polytron (PT1200E, for 30 s, one time), and incubated at 65 °C for 5 min in a water bath. Subsequent clarification was performed by two rounds of centrifugation at 2465 × <i>g</i>, 4 °C, 5 min, and 20,292 × <i>g</i>, 4 °C, 20 min, respectively. Triplicates for each sample were prepared; clear extracts were incubated with 30 µL of beads previously equilibrated with buffer A. Samples with beads were incubated for 3 hours at 4 °C with gentle mixing (VWR® Nutating Mixer). Beads were washed with buffer A and transferred to 1.5 mL Protein LoBind tubes (Eppendorf, 0030108422) and three consecutive washes with buffer A were performed, followed by four quick washes with 50 mM Ammonium Bicarbonate pH 8 (ABC). 30 µL of 50 mM ABC pH 8 containing 6 mM fresh DTT were added to each tube followed by 30 min incubation at 37 °C with gentle agitation in a ThermoMixer (Eppendorf) at 350 rpm. Samples were spun quickly and 5μL IAA 126 mM dissolved in ABC 50 mM pH 8 were added to each tube and incubated for another 30 min in the dark at room temperature. Tubes were quickly spun again and 5 µL Trypsin/LysC Mix (Promega, V5073) dissolved in ABC 50 mM pH 8 at 200 ng/µL were added to each one. Samples were digested on-beads overnight at 37 °C, 350 rpm. The next day, after a quick spin, 60 µL of MS grade H<sub>2</sub>O were added to the beads, mixed, and centrifuged at low speed (1 min, 400 × <i>g</i>). Eighty (80) µL of the peptides digests were transferred to new labeled Protein LoBind tubes before adding formic acid (FA) to a final concentration of 4% v/v. The peptides were dried in a SpeedVac at room temperature and then stored at −80 °C.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec22">Mass spectrometry and data analysis</h4><p>Samples were resuspended in 11 µL of 2 % acetonitrile (ACN) and 1 % FA containing 0.5X of iRT standard peptides (Biognosys, Ki-3002-1), of which, 2 µL of each sample was further diluted 1/5 in the same buffer. Diluted samples were loaded (2.5 µL each) at 450 nL/min on a 17 cm × 75 µm i.d. PicoFrit fused silica capillary column (New Objective), packed in-house with Jupiter 5 µm C18 300 Å (Phenomenex). The column was mounted in an Easy-nLC II system (Proxeon Biosystems) and coupled to an Orbitrap Fusion mass spectrometer (ThermoFisher Scientific) equipped with a Nanospray Flex Ion source (Proxeon Biosystems). Peptides were eluted at a flow rate of 250 nL/min on 2-slope gradient made with 0.2% FA in water (buffer A) and 0.2% FA in 100% ACN (buffer B). Concentration of buffer B first increased from 2% to 36% over 105 minutes, and from 36 % to 80 % over 12 minutes. The mass spectrometer was operated in data-independent acquisition mode. Full MS scans in the range of 400-1000 m/z range were acquired in the Orbitrap at a resolution of 120 K. Each full scan was followed by 25 MS2 acquisition windows with 24 m/z increments and 0.5 m/z overlaps, covering each full scan. Matching MS2 m/z window precursor ions were fragmented by HCD at a 30% collision energy and acquired in the Orbitrap at a 30 K resolution. Both MS and MS2 AGC target were set to 4e5 with maximum fill times of 60 ms. The lock mass option (lock mass: m/z 371.101233) was used for internal calibration. MS RAW files were analyzed with Spectronaut v15.1.210713 (Biognosys) by directDIA. The Fasta database was downloaded from Uniprot and consisted of all <i>Saccharomyces cerevisiae</i> S288c verified nuclear proteins entries (1749). Considered peptide length for Pulsar search was 6-52 amino acids with two missed cleavages tolerance, using the semi-specific Trypsin/P rules. Carbamidomethylation of cysteines was set as fixed modifications while oxidation of methionines, protein N-termini acetylation, biotinylation of lysines and protein N-termini as well as phosphorylation (STY) were set as variable modifications, with a maximum of five. Peptide spectrum match, peptides, and protein group FDR were set at 0.01. Machine learning was applied per run and both precursor and protein were identified at a Qvalue cutoff of 0.01. Proteins were quantified at the MS2 level on stripped peptide median quantities. Unique peptides were used for quantification. Only proteins for which values from all biological triplicates were measured in the sgRNA-BirA*-dCas9 were kept for quantification, and no values were imputed during the analysis. The resulting protein datasets were crosschecked with a list of chromatin and transcription related proteins obtained from Uniprot (Supplementary Data <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM4">5</a>). The protein list and associated ontologies was built from searches using the keyword chromatin (461 reviewed entries), and transcription (1211 reviewed entries) in <i>S. cerevisiae</i>. After merging both lists and removing duplicates, 1320 entries remained (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM1">6</a>). A two-sided unpaired Student’s t-test with equal variance was used to help in the evaluation of significant protein enrichment from sgRNA-BirA*-dCas9 triplicates. Analyses of evidence for association of chromatin remodeling subunits with nuclear protein complexes was performed for all significant candidates with definitions of subunit compositions of protein complexes taken from the Saccharomyces Genome Database (SGD)<sup><a aria-label="Reference 81" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR81" id="ref-link-section-d8519934e3971" title="Cherry, J. M. et al. Saccharomyces Genome Database: the genomics resource of budding yeast. Nucleic Acids Res. 40, D700–D705 (2012).">81</a></sup> and UniProt<sup><a aria-label="Reference 82" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR82" id="ref-link-section-d8519934e3976" title="The UniProt, C. UniProt: the universal protein knowledgebase in 2021. Nucleic Acids Res. 49, D480–D489 (2021).">82</a></sup>, and protein-protein interactions were identified in The Biological General Repository for Interaction Datasets (BioGrid)<sup><a aria-label="Reference 83" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-023-36391-6#ref-CR83" id="ref-link-section-d8519934e3980" title="Oughtred, R. et al. The BioGRID database: a comprehensive biomedical resource of curated protein, genetic, and chemical interactions. Protein Sci. 30, 187–200 (2021).">83</a></sup>. For representation clarity, proteins having a log2 fold change above 2.5 were limited at the 2.5 value on MA plots (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Fig5">5</a>). We removed two sgRNA-BirA*-dCas9 samples out of a total of 4 because they did not reveal significant fold changes for many or most of the candidates that were identified in the samples kept for further analysis. Both MA and box plots were made in RStudio (v2022.07.2) using the ggplot2 package (v3.3.6). Data are available via ProteomeXchange with identifier PXD029913.</p><h3 class="c-article__sub-heading" id="Sec23">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM6">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
</div>
<div class="u-mt-32">
<section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
<p>Mass spectrometric data are publicly available <i>via</i> ProteomeXchange. Project accession: <a href="https://www.ebi.ac.uk/pride/archive/projects/PXD029913">PXD029913</a>. The data supporting the results presented in this work are available in the main article and the supplementary information. Any additional information is available upon request from the corresponding author. <a data-track="click" data-track-action="section anchor" data-track-label="link" href="/articles/s41467-023-36391-6#Sec25">Source data</a> are provided with this paper.</p>
</div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
<p>All original code for matlab methods specifically referenced in the text are accessible in a GitHub repository publicly available: <a href="https://github.com/kolbincode/INO1-paper">https://github.com/kolbincode/INO1-paper</a>. <a href="https://doi.org/10.5281/zenodo.7492891">https://doi.org/10.5281/zenodo.7492891</a>.</p>
</div></div></section><section data-title="Change history"><div class="c-article-section" id="change-history-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="change-history">Change history</h2><div class="c-article-section__content" id="change-history-content"><ul class="c-article-change-list"><li class="c-article-change-list__item u-mb-24" id="chg1"><ins datetime="2023-04-21"><h3 class="c-article-change-list__heading u-h3 u-pr-8 u-display-inline">21 April 2023</h3><div class="c-article-change-list__details"><p>A Correction to this paper has been published: <a href="https://doi.org/10.1038/s41467-023-38076-6">https://doi.org/10.1038/s41467-023-38076-6</a></p></div></ins></li></ul></div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Brickner, J. H. &amp; Walter, P. Gene recruitment of the activated INO1 locus to the nuclear membrane. <i>PLoS Biol.</i> <b>2</b>, e342 (2004).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 1" data-doi="10.1371/journal.pbio.0020342" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1371/journal.pbio.0020342" data-track-label="10.1371/journal.pbio.0020342" data-track-value="article reference" href="https://doi.org/10.1371%2Fjournal.pbio.0020342" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 1" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15455074" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 1" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC519002" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 1" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20recruitment%20of%20the%20activated%20INO1%20locus%20to%20the%20nuclear%20membrane&amp;journal=PLoS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.0020342&amp;volume=2&amp;publication_year=2004&amp;author=Brickner%2CJH&amp;author=Walter%2CP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Brown, C. R., Kennedy, C. J., Delmar, V. A., Forbes, D. J. &amp; Silver, P. A. Global histone acetylation induces functional genomic reorganization at mammalian nuclear pore complexes. <i>Genes Dev.</i> <b>22</b>, 627–639 (2008).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 2" data-doi="10.1101/gad.1632708" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1101/gad.1632708" data-track-label="10.1101/gad.1632708" data-track-value="article reference" href="https://doi.org/10.1101%2Fgad.1632708" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 2" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXjsVKntL8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 2" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18316479" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 2" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2259032" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 2" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Global%20histone%20acetylation%20induces%20functional%20genomic%20reorganization%20at%20mammalian%20nuclear%20pore%20complexes&amp;journal=Genes%20Dev.&amp;doi=10.1101%2Fgad.1632708&amp;volume=22&amp;pages=627-639&amp;publication_year=2008&amp;author=Brown%2CCR&amp;author=Kennedy%2CCJ&amp;author=Delmar%2CVA&amp;author=Forbes%2CDJ&amp;author=Silver%2CPA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Casolari, J. M. et al. Genome-wide localization of the nuclear transport machinery couples transcriptional status and nuclear organization. <i>Cell</i> <b>117</b>, 427–439 (2004).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 3" data-doi="10.1016/S0092-8674(04)00448-9" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/S0092-8674(04)00448-9" data-track-label="10.1016/S0092-8674(04)00448-9" data-track-value="article reference" href="https://doi.org/10.1016%2FS0092-8674%2804%2900448-9" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 3" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXktlCqtr0%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 3" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15137937" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 3" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20localization%20of%20the%20nuclear%20transport%20machinery%20couples%20transcriptional%20status%20and%20nuclear%20organization&amp;journal=Cell&amp;doi=10.1016%2FS0092-8674%2804%2900448-9&amp;volume=117&amp;pages=427-439&amp;publication_year=2004&amp;author=Casolari%2CJM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Kalverda, B., Pickersgill, H., Shloma, V. V. &amp; Fornerod, M. Nucleoporins directly stimulate expression of developmental and cell-cycle genes inside the nucleoplasm. <i>Cell</i> <b>140</b>, 360–371 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 4" data-doi="10.1016/j.cell.2010.01.011" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2010.01.011" data-track-label="10.1016/j.cell.2010.01.011" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2010.01.011" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 4" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXlt1Klsb0%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 4" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20144760" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 4" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Nucleoporins%20directly%20stimulate%20expression%20of%20developmental%20and%20cell-cycle%20genes%20inside%20the%20nucleoplasm&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2010.01.011&amp;volume=140&amp;pages=360-371&amp;publication_year=2010&amp;author=Kalverda%2CB&amp;author=Pickersgill%2CH&amp;author=Shloma%2CVV&amp;author=Fornerod%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Rohner, S. et al. Promoter- and RNA polymerase II-dependent hsp-16 gene association with nuclear pores in Caenorhabditis elegans. <i>J. Cell Biol.</i> <b>200</b>, 589–604 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 5" data-doi="10.1083/jcb.201207024" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1083/jcb.201207024" data-track-label="10.1083/jcb.201207024" data-track-value="article reference" href="https://doi.org/10.1083%2Fjcb.201207024" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 5" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXjvVygs70%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 5" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23460676" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 5" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3587839" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 5" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Promoter-%20and%20RNA%20polymerase%20II-dependent%20hsp-16%20gene%20association%20with%20nuclear%20pores%20in%20Caenorhabditis%20elegans&amp;journal=J.%20Cell%20Biol.&amp;doi=10.1083%2Fjcb.201207024&amp;volume=200&amp;pages=589-604&amp;publication_year=2013&amp;author=Rohner%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Kosak, S. T. et al. Subnuclear compartmentalization of immunoglobulin loci during lymphocyte development. <i>Science</i> <b>296</b>, 158–162 (2002).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 6" data-doi="10.1126/science.1068768" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.1068768" data-track-label="10.1126/science.1068768" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.1068768" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 6" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2002Sci...296..158K" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 6" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38XivVWjsLg%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 6" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11935030" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 6" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Subnuclear%20compartmentalization%20of%20immunoglobulin%20loci%20during%20lymphocyte%20development&amp;journal=Science&amp;doi=10.1126%2Fscience.1068768&amp;volume=296&amp;pages=158-162&amp;publication_year=2002&amp;author=Kosak%2CST" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Zink, D. et al. Transcription-dependent spatial arrangements of CFTR and adjacent genes in human cell nuclei. <i>J. Cell Biol.</i> <b>166</b>, 815–825 (2004).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 7" data-doi="10.1083/jcb.200404107" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1083/jcb.200404107" data-track-label="10.1083/jcb.200404107" data-track-value="article reference" href="https://doi.org/10.1083%2Fjcb.200404107" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 7" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXns1yhtLs%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 7" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15364959" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 7" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2172106" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 7" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcription-dependent%20spatial%20arrangements%20of%20CFTR%20and%20adjacent%20genes%20in%20human%20cell%20nuclei&amp;journal=J.%20Cell%20Biol.&amp;doi=10.1083%2Fjcb.200404107&amp;volume=166&amp;pages=815-825&amp;publication_year=2004&amp;author=Zink%2CD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Cabal, G. G. et al. SAGA interacting factors confine sub-diffusion of transcribed genes to the nuclear envelope. <i>Nature</i> <b>441</b>, 770–773 (2006).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 8" data-doi="10.1038/nature04752" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature04752" data-track-label="10.1038/nature04752" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature04752" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 8" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2006Natur.441..770C" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 8" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XltlKqurw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 8" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16760982" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 8" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=SAGA%20interacting%20factors%20confine%20sub-diffusion%20of%20transcribed%20genes%20to%20the%20nuclear%20envelope&amp;journal=Nature&amp;doi=10.1038%2Fnature04752&amp;volume=441&amp;pages=770-773&amp;publication_year=2006&amp;author=Cabal%2CGG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Taddei, A. et al. Nuclear pore association confers optimal expression levels for an inducible yeast gene. <i>Nature</i> <b>441</b>, 774–778 (2006).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 9" data-doi="10.1038/nature04845" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature04845" data-track-label="10.1038/nature04845" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature04845" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 9" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2006Natur.441..774T" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 9" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XltlKqu7o%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 9" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16760983" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 9" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Nuclear%20pore%20association%20confers%20optimal%20expression%20levels%20for%20an%20inducible%20yeast%20gene&amp;journal=Nature&amp;doi=10.1038%2Fnature04845&amp;volume=441&amp;pages=774-778&amp;publication_year=2006&amp;author=Taddei%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Williams, R. R. et al. Neural induction promotes large-scale chromatin reorganisation of the Mash1 locus. <i>J. Cell Sci.</i> <b>119</b>, 132–140 (2006).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 10" data-doi="10.1242/jcs.02727" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1242/jcs.02727" data-track-label="10.1242/jcs.02727" data-track-value="article reference" href="https://doi.org/10.1242%2Fjcs.02727" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 10" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XhtFGju7g%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 10" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16371653" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 10" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Neural%20induction%20promotes%20large-scale%20chromatin%20reorganisation%20of%20the%20Mash1%20locus&amp;journal=J.%20Cell%20Sci.&amp;doi=10.1242%2Fjcs.02727&amp;volume=119&amp;pages=132-140&amp;publication_year=2006&amp;author=Williams%2CRR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Ahmed, S. et al. DNA zip codes control an ancient mechanism for gene targeting to the nuclear periphery. <i>Nat. Cell Biol.</i> <b>12</b>, 111–118 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 11" data-doi="10.1038/ncb2011" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/ncb2011" data-track-label="10.1038/ncb2011" data-track-value="article reference" href="https://doi.org/10.1038%2Fncb2011" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 11" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlCqtbo%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 11" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20098417" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 11" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2835469" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 11" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20zip%20codes%20control%20an%20ancient%20mechanism%20for%20gene%20targeting%20to%20the%20nuclear%20periphery&amp;journal=Nat.%20Cell%20Biol.&amp;doi=10.1038%2Fncb2011&amp;volume=12&amp;pages=111-118&amp;publication_year=2010&amp;author=Ahmed%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Brickner, D. G. et al. Transcription factor binding to a DNA zip code controls interchromosomal clustering at the nuclear periphery. <i>Dev. Cell</i> <b>22</b>, 1234–1246 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 12" data-doi="10.1016/j.devcel.2012.03.012" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.devcel.2012.03.012" data-track-label="10.1016/j.devcel.2012.03.012" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.devcel.2012.03.012" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 12" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XmvFKnu7Y%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 12" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22579222" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 12" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3376219" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 12" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcription%20factor%20binding%20to%20a%20DNA%20zip%20code%20controls%20interchromosomal%20clustering%20at%20the%20nuclear%20periphery&amp;journal=Dev.%20Cell&amp;doi=10.1016%2Fj.devcel.2012.03.012&amp;volume=22&amp;pages=1234-1246&amp;publication_year=2012&amp;author=Brickner%2CDG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Brickner, D. G. et al. Subnuclear positioning and interchromosomal clustering of the GAL1-10 locus are controlled by separable, interdependent mechanisms. <i>Mol. Biol. Cell</i> <b>27</b>, 2980–2993 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 13" data-doi="10.1091/mbc.E16-03-0174" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1091/mbc.E16-03-0174" data-track-label="10.1091/mbc.E16-03-0174" data-track-value="article reference" href="https://doi.org/10.1091%2Fmbc.E16-03-0174" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 13" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhvFaksrzK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 13" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27489341" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 13" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042583" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 13" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Subnuclear%20positioning%20and%20interchromosomal%20clustering%20of%20the%20GAL1-10%20locus%20are%20controlled%20by%20separable%2C%20interdependent%20mechanisms&amp;journal=Mol.%20Biol.%20Cell&amp;doi=10.1091%2Fmbc.E16-03-0174&amp;volume=27&amp;pages=2980-2993&amp;publication_year=2016&amp;author=Brickner%2CDG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Brickner, D. G. et al. H2A.Z-mediated localization of genes at the nuclear periphery confers epigenetic memory of previous transcriptional state. <i>PLoS Biol.</i> <b>5</b>, e81 (2007).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 14" data-doi="10.1371/journal.pbio.0050081" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1371/journal.pbio.0050081" data-track-label="10.1371/journal.pbio.0050081" data-track-value="article reference" href="https://doi.org/10.1371%2Fjournal.pbio.0050081" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 14" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17373856" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 14" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1828143" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 14" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=H2A.Z-mediated%20localization%20of%20genes%20at%20the%20nuclear%20periphery%20confers%20epigenetic%20memory%20of%20previous%20transcriptional%20state&amp;journal=PLoS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.0050081&amp;volume=5&amp;publication_year=2007&amp;author=Brickner%2CDG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Randise-Hinchliff, C. et al. Strategies to regulate transcription factor-mediated gene positioning and interchromosomal clustering at the nuclear periphery. <i>J. Cell Biol.</i> <b>212</b>, 633–646 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 15" data-doi="10.1083/jcb.201508068" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1083/jcb.201508068" data-track-label="10.1083/jcb.201508068" data-track-value="article reference" href="https://doi.org/10.1083%2Fjcb.201508068" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 15" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1Ort7fK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 15" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26953353" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 15" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4792077" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 15" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Strategies%20to%20regulate%20transcription%20factor-mediated%20gene%20positioning%20and%20interchromosomal%20clustering%20at%20the%20nuclear%20periphery&amp;journal=J.%20Cell%20Biol.&amp;doi=10.1083%2Fjcb.201508068&amp;volume=212&amp;pages=633-646&amp;publication_year=2016&amp;author=Randise-Hinchliff%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Berger, A. B. et al. High-resolution statistical mapping reveals gene territories in live yeast. <i>Nat. Methods</i> <b>5</b>, 1031–1037 (2008).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 16" data-doi="10.1038/nmeth.1266" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nmeth.1266" data-track-label="10.1038/nmeth.1266" data-track-value="article reference" href="https://doi.org/10.1038%2Fnmeth.1266" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 16" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsVWgsrzK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 16" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18978785" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 16" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=High-resolution%20statistical%20mapping%20reveals%20gene%20territories%20in%20live%20yeast&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.1266&amp;volume=5&amp;pages=1031-1037&amp;publication_year=2008&amp;author=Berger%2CAB" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Verdaasdonk, J. S. et al. Centromere tethering confines chromosome domains. <i>Mol. Cell</i> <b>52</b>, 819–831 (2013). S1097-2765(13)00787-9 [pii].</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 17" data-doi="10.1016/j.molcel.2013.10.021" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2013.10.021" data-track-label="10.1016/j.molcel.2013.10.021" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2013.10.021" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 17" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvVKgtrrF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 17" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24268574" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 17" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Centromere%20tethering%20confines%20chromosome%20domains&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2013.10.021&amp;volume=52&amp;pages=819-831&amp;publication_year=2013&amp;author=Verdaasdonk%2CJS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Bloom, K. S. Beyond the code: the mechanical properties of DNA as they relate to mitosis. <i>Chromosoma</i> <b>117</b>, 103–110 (2008).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 18" data-doi="10.1007/s00412-007-0138-0" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1007/s00412-007-0138-0" data-track-label="10.1007/s00412-007-0138-0" data-track-value="article reference" href="https://link.springer.com/doi/10.1007/s00412-007-0138-0" rel="noopener">Article</a> 
    <a aria-label="PubMed reference 18" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18060422" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 18" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Beyond%20the%20code%3A%20the%20mechanical%20properties%20of%20DNA%20as%20they%20relate%20to%20mitosis&amp;journal=Chromosoma&amp;doi=10.1007%2Fs00412-007-0138-0&amp;volume=117&amp;pages=103-110&amp;publication_year=2008&amp;author=Bloom%2CKS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Bystricky, K., Heun, P., Gehlen, L., Langowski, J. &amp; Gasser, S. M. Long-range compaction and flexibility of interphase chromatin in budding yeast analyzed by high-resolution imaging techniques. <i>Proc. Natl. Acad. Sci. USA</i> <b>101</b>, 16495–16500 (2004).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 19" data-doi="10.1073/pnas.0402766101" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.0402766101" data-track-label="10.1073/pnas.0402766101" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.0402766101" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 19" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2004PNAS..10116495B" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 19" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXhtVOisbvI" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 19" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15545610" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 19" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC534505" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 19" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Long-range%20compaction%20and%20flexibility%20of%20interphase%20chromatin%20in%20budding%20yeast%20analyzed%20by%20high-resolution%20imaging%20techniques&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0402766101&amp;volume=101&amp;pages=16495-16500&amp;publication_year=2004&amp;author=Bystricky%2CK&amp;author=Heun%2CP&amp;author=Gehlen%2CL&amp;author=Langowski%2CJ&amp;author=Gasser%2CSM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Cui, Y. &amp; Bustamante, C. Pulling a single chromatin fiber reveals the forces that maintain its higher-order structure. <i>Proc. Natl. Acad. Sci. USA</i> <b>97</b>, 127–132 (2000).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 20" data-doi="10.1073/pnas.97.1.127" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.97.1.127" data-track-label="10.1073/pnas.97.1.127" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.97.1.127" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 20" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2000PNAS...97..127C" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 20" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3cXjvVCrsw%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 20" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10618382" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 20" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC26627" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 20" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Pulling%20a%20single%20chromatin%20fiber%20reveals%20the%20forces%20that%20maintain%20its%20higher-order%20structure&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.97.1.127&amp;volume=97&amp;pages=127-132&amp;publication_year=2000&amp;author=Cui%2CY&amp;author=Bustamante%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Chi, Q., Wang, G. &amp; Jiang, J. The persistence length and length per base of single-stranded DNA obtained from fluorescence correlation spectroscopy measurements using mean field theory. <i>Phys. A Stat. Mech. Appl.</i> <b>392</b>, 1072–1079 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 21" data-doi="10.1016/j.physa.2012.09.022" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.physa.2012.09.022" data-track-label="10.1016/j.physa.2012.09.022" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.physa.2012.09.022" rel="nofollow noopener">Article</a> 
    <a aria-label="MathSciNet reference 21" data-track="click_references" data-track-action="mathscinet reference" data-track-item_id="link" data-track-label="link" data-track-value="mathscinet reference" href="http://www.ams.org/mathscinet-getitem?mr=3010988" rel="nofollow noopener">MathSciNet</a> 
    <a aria-label="CAS reference 21" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVWit7rP" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 21" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20persistence%20length%20and%20length%20per%20base%20of%20single-stranded%20DNA%20obtained%20from%20fluorescence%20correlation%20spectroscopy%20measurements%20using%20mean%20field%20theory&amp;journal=Phys.%20A%20Stat.%20Mech.%20Appl.&amp;doi=10.1016%2Fj.physa.2012.09.022&amp;volume=392&amp;pages=1072-1079&amp;publication_year=2013&amp;author=Chi%2CQ&amp;author=Wang%2CG&amp;author=Jiang%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Rechendorff, K., Witz, G., Adamcik, J. &amp; Dietler, G. Persistence length and scaling properties of single-stranded DNA adsorbed on modified graphite. <i>J. Chem. Phys.</i> <b>131</b>, 095103 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 22" data-doi="10.1063/1.3216111" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1063/1.3216111" data-track-label="10.1063/1.3216111" data-track-value="article reference" href="https://doi.org/10.1063%2F1.3216111" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 22" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2009JChPh.131i5103R" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 22" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19739875" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 22" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Persistence%20length%20and%20scaling%20properties%20of%20single-stranded%20DNA%20adsorbed%20on%20modified%20graphite&amp;journal=J.%20Chem.%20Phys.&amp;doi=10.1063%2F1.3216111&amp;volume=131&amp;publication_year=2009&amp;author=Rechendorff%2CK&amp;author=Witz%2CG&amp;author=Adamcik%2CJ&amp;author=Dietler%2CG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Smith, S. B., Cui, Y. &amp; Bustamante, C. Overstretching B-DNA: the elastic response of individual double-stranded and single-stranded DNA molecules. <i>Science</i> <b>271</b>, 795–799 (1996).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 23" data-doi="10.1126/science.271.5250.795" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.271.5250.795" data-track-label="10.1126/science.271.5250.795" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.271.5250.795" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 23" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1996Sci...271..795S" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 23" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK28XptlyjtQ%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 23" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8628994" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 23" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Overstretching%20B-DNA%3A%20the%20elastic%20response%20of%20individual%20double-stranded%20and%20single-stranded%20DNA%20molecules&amp;journal=Science&amp;doi=10.1126%2Fscience.271.5250.795&amp;volume=271&amp;pages=795-799&amp;publication_year=1996&amp;author=Smith%2CSB&amp;author=Cui%2CY&amp;author=Bustamante%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Ahmed, S. &amp; Brickner, J. H. A role for DNA sequence in controlling the spatial organization of the genome. <i>Nucleus</i> <b>1</b>, 402–406 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 24" data-doi="10.4161/nucl.1.5.12637" data-track="click_references" data-track-action="article reference" data-track-item_id="10.4161/nucl.1.5.12637" data-track-label="10.4161/nucl.1.5.12637" data-track-value="article reference" href="https://doi.org/10.4161%2Fnucl.1.5.12637" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 24" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21326823" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 24" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3037535" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 24" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20role%20for%20DNA%20sequence%20in%20controlling%20the%20spatial%20organization%20of%20the%20genome&amp;journal=Nucleus&amp;doi=10.4161%2Fnucl.1.5.12637&amp;volume=1&amp;pages=402-406&amp;publication_year=2010&amp;author=Ahmed%2CS&amp;author=Brickner%2CJH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Neumann, F. R. et al. Targeted INO80 enhances subnuclear chromatin movement and ectopic homologous recombination. <i>Genes Dev.</i> <b>26</b>, 369–383 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 25" data-doi="10.1101/gad.176156.111" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1101/gad.176156.111" data-track-label="10.1101/gad.176156.111" data-track-value="article reference" href="https://doi.org/10.1101%2Fgad.176156.111" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 25" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xjt1Clsro%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 25" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22345518" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 25" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3289885" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 25" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20INO80%20enhances%20subnuclear%20chromatin%20movement%20and%20ectopic%20homologous%20recombination&amp;journal=Genes%20Dev.&amp;doi=10.1101%2Fgad.176156.111&amp;volume=26&amp;pages=369-383&amp;publication_year=2012&amp;author=Neumann%2CFR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Scheffold, F. et al. Brushlike interactions between thermoresponsive microgel particles. <i>Phys. Rev. Lett.</i> <b>104</b>, 128304 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 26" data-doi="10.1103/PhysRevLett.104.128304" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1103/PhysRevLett.104.128304" data-track-label="10.1103/PhysRevLett.104.128304" data-track-value="article reference" href="https://doi.org/10.1103%2FPhysRevLett.104.128304" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 26" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2010PhRvL.104l8304S" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 26" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20366570" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 26" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Brushlike%20interactions%20between%20thermoresponsive%20microgel%20particles&amp;journal=Phys.%20Rev.%20Lett.&amp;doi=10.1103%2FPhysRevLett.104.128304&amp;volume=104&amp;publication_year=2010&amp;author=Scheffold%2CF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Uhlenbeck, G. E. &amp; Ornstein, L. S. On the theory of the brownian motion. <i>Phys. Rev.</i> <b>36</b>, 823–841 (1930).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 27" data-doi="10.1103/PhysRev.36.823" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1103/PhysRev.36.823" data-track-label="10.1103/PhysRev.36.823" data-track-value="article reference" href="https://doi.org/10.1103%2FPhysRev.36.823" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 27" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1930PhRv...36..823U" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 27" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaA3MXitFKr" rel="nofollow noopener">CAS</a> 
    <a aria-label="MATH reference 27" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?56.1277.03" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 27" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=On%20the%20theory%20of%20the%20brownian%20motion&amp;journal=Phys.%20Rev.&amp;doi=10.1103%2FPhysRev.36.823&amp;volume=36&amp;pages=823-841&amp;publication_year=1930&amp;author=Uhlenbeck%2CGE&amp;author=Ornstein%2CLS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Rufiange, A., Jacques, P. E., Bhat, W., Robert, F. &amp; Nourani, A. Genome-wide replication-independent histone H3 exchange occurs predominantly at promoters and implicates H3 K56 acetylation and Asf1. <i>Mol. Cell</i> <b>27</b>, 393–405 (2007).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 28" data-doi="10.1016/j.molcel.2007.07.011" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2007.07.011" data-track-label="10.1016/j.molcel.2007.07.011" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2007.07.011" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 28" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXptlymsrw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 28" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17679090" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 28" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20replication-independent%20histone%20H3%20exchange%20occurs%20predominantly%20at%20promoters%20and%20implicates%20H3%20K56%20acetylation%20and%20Asf1&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2007.07.011&amp;volume=27&amp;pages=393-405&amp;publication_year=2007&amp;author=Rufiange%2CA&amp;author=Jacques%2CPE&amp;author=Bhat%2CW&amp;author=Robert%2CF&amp;author=Nourani%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Driscoll, R., Hudson, A. &amp; Jackson, S. P. Yeast Rtt109 promotes genome stability by acetylating histone H3 on lysine 56. <i>Science</i> <b>315</b>, 649–652 (2007).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 29" data-doi="10.1126/science.1135862" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.1135862" data-track-label="10.1126/science.1135862" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.1135862" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 29" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2007Sci...315..649D" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 29" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVyisL8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 29" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17272722" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 29" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3334813" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 29" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Yeast%20Rtt109%20promotes%20genome%20stability%20by%20acetylating%20histone%20H3%20on%20lysine%2056&amp;journal=Science&amp;doi=10.1126%2Fscience.1135862&amp;volume=315&amp;pages=649-652&amp;publication_year=2007&amp;author=Driscoll%2CR&amp;author=Hudson%2CA&amp;author=Jackson%2CSP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Topal, S., Vasseur, P., Radman-Livaja, M. &amp; Peterson, C. L. Distinct transcriptional roles for Histone H3-K56 acetylation during the cell cycle in Yeast. <i>Nat. Commun.</i> <b>10</b>, 4372 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 30" data-doi="10.1038/s41467-019-12400-5" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-019-12400-5" data-track-label="10.1038/s41467-019-12400-5" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-019-12400-5" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 30" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019NatCo..10.4372T" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 30" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31558720" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 30" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6763489" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 30" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Distinct%20transcriptional%20roles%20for%20Histone%20H3-K56%20acetylation%20during%20the%20cell%20cycle%20in%20Yeast&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-12400-5&amp;volume=10&amp;publication_year=2019&amp;author=Topal%2CS&amp;author=Vasseur%2CP&amp;author=Radman-Livaja%2CM&amp;author=Peterson%2CCL" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Xhemalce, B. et al. Regulation of histone H3 lysine 56 acetylation in Schizosaccharomyces pombe. <i>J. Biol. Chem.</i> <b>282</b>, 15040–15047 (2007).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 31" data-doi="10.1074/jbc.M701197200" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1074/jbc.M701197200" data-track-label="10.1074/jbc.M701197200" data-track-value="article reference" href="https://doi.org/10.1074%2Fjbc.M701197200" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 31" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXltFelsrw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 31" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17369611" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 31" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20histone%20H3%20lysine%2056%20acetylation%20in%20Schizosaccharomyces%20pombe&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M701197200&amp;volume=282&amp;pages=15040-15047&amp;publication_year=2007&amp;author=Xhemalce%2CB" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Morawska, M. &amp; Ulrich, H. D. An expanded tool kit for the auxin-inducible degron system in budding yeast. <i>Yeast</i> <b>30</b>, 341–351 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 32" data-doi="10.1002/yea.2967" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/yea.2967" data-track-label="10.1002/yea.2967" data-track-value="article reference" href="https://doi.org/10.1002%2Fyea.2967" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 32" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFCisLrL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 32" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23836714" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 32" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20expanded%20tool%20kit%20for%20the%20auxin-inducible%20degron%20system%20in%20budding%20yeast&amp;journal=Yeast&amp;doi=10.1002%2Fyea.2967&amp;volume=30&amp;pages=341-351&amp;publication_year=2013&amp;author=Morawska%2CM&amp;author=Ulrich%2CHD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Lo, W.-S. et al. Snf1-a histone kinase that works in concert with the histone acetyltransferase Gcn5 to regulate transcription. <i>Science</i> <b>293</b>, 1142–1146 (2001).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 33" data-doi="10.1126/science.1062322" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.1062322" data-track-label="10.1126/science.1062322" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.1062322" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 33" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3MXmtVWqt7s%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 33" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11498592" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 33" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Snf1-a%20histone%20kinase%20that%20works%20in%20concert%20with%20the%20histone%20acetyltransferase%20Gcn5%20to%20regulate%20transcription&amp;journal=Science&amp;doi=10.1126%2Fscience.1062322&amp;volume=293&amp;pages=1142-1146&amp;publication_year=2001&amp;author=Lo%2CW-S" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Han, J. et al. Rtt109 acetylates histone H3 lysine 56 and functions in DNA replication. <i>Science</i> <b>315</b>, 653–655 (2007).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 34" data-doi="10.1126/science.1133234" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.1133234" data-track-label="10.1126/science.1133234" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.1133234" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 34" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2007Sci...315..653H" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 34" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVyisLw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 34" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17272723" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 34" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Rtt109%20acetylates%20histone%20H3%20lysine%2056%20and%20functions%20in%20DNA%20replication&amp;journal=Science&amp;doi=10.1126%2Fscience.1133234&amp;volume=315&amp;pages=653-655&amp;publication_year=2007&amp;author=Han%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Mizzen, C. A. et al. The TAF(II)250 subunit of TFIID has histone acetyltransferase activity. <i>Cell</i> <b>87</b>, 1261–1270 (1996).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 35" data-doi="10.1016/S0092-8674(00)81821-8" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/S0092-8674(00)81821-8" data-track-label="10.1016/S0092-8674(00)81821-8" data-track-value="article reference" href="https://doi.org/10.1016%2FS0092-8674%2800%2981821-8" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 35" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK2sXit1OqsA%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 35" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8980232" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 35" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20TAF%28II%29250%20subunit%20of%20TFIID%20has%20histone%20acetyltransferase%20activity&amp;journal=Cell&amp;doi=10.1016%2FS0092-8674%2800%2981821-8&amp;volume=87&amp;pages=1261-1270&amp;publication_year=1996&amp;author=Mizzen%2CCA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Durant, M. &amp; Pugh, B. F. Genome-wide relationships between TAF1 and histone acetyltransferases in Saccharomyces cerevisiae. <i>Mol. Cell. Biol.</i> <b>26</b>, 2791–2802 (2006).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 36" data-doi="10.1128/MCB.26.7.2791-2802.2006" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1128/MCB.26.7.2791-2802.2006" data-track-label="10.1128/MCB.26.7.2791-2802.2006" data-track-value="article reference" href="https://doi.org/10.1128%2FMCB.26.7.2791-2802.2006" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 36" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XivFSrt70%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 36" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16537921" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 36" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1430310" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 36" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20relationships%20between%20TAF1%20and%20histone%20acetyltransferases%20in%20Saccharomyces%20cerevisiae&amp;journal=Mol.%20Cell.%20Biol.&amp;doi=10.1128%2FMCB.26.7.2791-2802.2006&amp;volume=26&amp;pages=2791-2802&amp;publication_year=2006&amp;author=Durant%2CM&amp;author=Pugh%2CBF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Duan, M. R. &amp; Smerdon, M. J. Histone H3 lysine 14 (H3K14) acetylation facilitates DNA repair in a positioned nucleosome by stabilizing the binding of the chromatin Remodeler RSC (Remodels Structure of Chromatin). <i>J. Biol. Chem.</i> <b>289</b>, 8353–8363 (2014).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 37" data-doi="10.1074/jbc.M113.540732" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1074/jbc.M113.540732" data-track-label="10.1074/jbc.M113.540732" data-track-value="article reference" href="https://doi.org/10.1074%2Fjbc.M113.540732" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 37" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXks1Oltrw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 37" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24515106" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 37" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3961661" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 37" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Histone%20H3%20lysine%2014%20%28H3K14%29%20acetylation%20facilitates%20DNA%20repair%20in%20a%20positioned%20nucleosome%20by%20stabilizing%20the%20binding%20of%20the%20chromatin%20Remodeler%20RSC%20%28Remodels%20Structure%20of%20Chromatin%29&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M113.540732&amp;volume=289&amp;pages=8353-8363&amp;publication_year=2014&amp;author=Duan%2CMR&amp;author=Smerdon%2CMJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Smerdon, M. J. DNA repair and the role of chromatin structure. <i>Curr. Opin. Cell Biol.</i> <b>3</b>, 422–428 (1991).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 38" data-doi="10.1016/0955-0674(91)90069-B" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/0955-0674(91)90069-B" data-track-label="10.1016/0955-0674(91)90069-B" data-track-value="article reference" href="https://doi.org/10.1016%2F0955-0674%2891%2990069-B" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 38" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK3MXmsVyqs7g%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 38" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1892653" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 38" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20repair%20and%20the%20role%20of%20chromatin%20structure&amp;journal=Curr.%20Opin.%20Cell%20Biol.&amp;doi=10.1016%2F0955-0674%2891%2990069-B&amp;volume=3&amp;pages=422-428&amp;publication_year=1991&amp;author=Smerdon%2CMJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Kuo, M. H., vom Baur, E., Struhl, K. &amp; Allis, C. D. Gcn4 activator targets Gcn5 histone acetyltransferase to specific promoters independently of transcription. <i>Mol. Cell</i> <b>6</b>, 1309–1320 (2000).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 39" data-doi="10.1016/S1097-2765(00)00129-5" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/S1097-2765(00)00129-5" data-track-label="10.1016/S1097-2765(00)00129-5" data-track-value="article reference" href="https://doi.org/10.1016%2FS1097-2765%2800%2900129-5" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 39" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3MXlvFahsQ%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 39" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11163205" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 39" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Gcn4%20activator%20targets%20Gcn5%20histone%20acetyltransferase%20to%20specific%20promoters%20independently%20of%20transcription&amp;journal=Mol.%20Cell&amp;doi=10.1016%2FS1097-2765%2800%2900129-5&amp;volume=6&amp;pages=1309-1320&amp;publication_year=2000&amp;author=Kuo%2CMH&amp;author=vom%20Baur%2CE&amp;author=Struhl%2CK&amp;author=Allis%2CCD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Kuo, M. H. et al. Transcription-linked acetylation by Gcn5p of histones H3 and H4 at specific lysines. <i>Nature</i> <b>383</b>, 269–272 (1996).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 40" data-doi="10.1038/383269a0" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/383269a0" data-track-label="10.1038/383269a0" data-track-value="article reference" href="https://doi.org/10.1038%2F383269a0" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 40" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1996Natur.383..269K" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 40" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK28Xlsl2ksLY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 40" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8805705" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 40" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcription-linked%20acetylation%20by%20Gcn5p%20of%20histones%20H3%20and%20H4%20at%20specific%20lysines&amp;journal=Nature&amp;doi=10.1038%2F383269a0&amp;volume=383&amp;pages=269-272&amp;publication_year=1996&amp;author=Kuo%2CMH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Berndsen, C. E. et al. Molecular functions of the histone acetyltransferase chaperone complex Rtt109-Vps75. <i>Nat. Struct. Mol. Biol.</i> <b>15</b>, 948–956 (2008).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 41" data-doi="10.1038/nsmb.1459" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nsmb.1459" data-track-label="10.1038/nsmb.1459" data-track-value="article reference" href="https://doi.org/10.1038%2Fnsmb.1459" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 41" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtVKrsLnM" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 41" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19172748" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 41" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2678805" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 41" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20functions%20of%20the%20histone%20acetyltransferase%20chaperone%20complex%20Rtt109-Vps75&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;doi=10.1038%2Fnsmb.1459&amp;volume=15&amp;pages=948-956&amp;publication_year=2008&amp;author=Berndsen%2CCE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Abshiru, N. et al. Chaperone-mediated acetylation of histones by Rtt109 identified by quantitative proteomics. <i>J. Proteom.</i> <b>81</b>, 80–90 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 42" data-doi="10.1016/j.jprot.2012.09.026" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.jprot.2012.09.026" data-track-label="10.1016/j.jprot.2012.09.026" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.jprot.2012.09.026" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 42" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhsFCltL7O" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 42" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chaperone-mediated%20acetylation%20of%20histones%20by%20Rtt109%20identified%20by%20quantitative%20proteomics&amp;journal=J.%20Proteom.&amp;doi=10.1016%2Fj.jprot.2012.09.026&amp;volume=81&amp;pages=80-90&amp;publication_year=2013&amp;author=Abshiru%2CN" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Opitz, N. et al. Capturing the Asc1p/receptor for activated C kinase 1 (RACK1) microenvironment at the head region of the 40 S ribosome with quantitative BioID in yeast. <i>Mol. Cell. Proteom. MCP</i> <b>16</b>, 2199–2218 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 43" data-doi="10.1074/mcp.M116.066654" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1074/mcp.M116.066654" data-track-label="10.1074/mcp.M116.066654" data-track-value="article reference" href="https://doi.org/10.1074%2Fmcp.M116.066654" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 43" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvVOhug%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 43" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28982715" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 43" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Capturing%20the%20Asc1p%2Freceptor%20for%20activated%20C%20kinase%201%20%28RACK1%29%20microenvironment%20at%20the%20head%20region%20of%20the%2040%E2%80%89S%20ribosome%20with%20quantitative%20BioID%20in%20yeast&amp;journal=Mol.%20Cell.%20Proteom.%20MCP&amp;doi=10.1074%2Fmcp.M116.066654&amp;volume=16&amp;pages=2199-2218&amp;publication_year=2017&amp;author=Opitz%2CN" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Schmidtmann, E., Anton, T., Rombaut, P., Herzog, F. &amp; Leonhardt, H. Determination of local chromatin composition by CasID. <i>Nucleus</i> <b>7</b>, 476–484 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 44" data-doi="10.1080/19491034.2016.1239000" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1080/19491034.2016.1239000" data-track-label="10.1080/19491034.2016.1239000" data-track-value="article reference" href="https://doi.org/10.1080%2F19491034.2016.1239000" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 44" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhslCgt7vN" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 44" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27676121" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 44" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5120596" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 44" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Determination%20of%20local%20chromatin%20composition%20by%20CasID&amp;journal=Nucleus&amp;doi=10.1080%2F19491034.2016.1239000&amp;volume=7&amp;pages=476-484&amp;publication_year=2016&amp;author=Schmidtmann%2CE&amp;author=Anton%2CT&amp;author=Rombaut%2CP&amp;author=Herzog%2CF&amp;author=Leonhardt%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Shen, X., Mizuguchi, G., Hamiche, A. &amp; Wu, C. A chromatin remodelling complex involved in transcription and DNA processing. <i>Nature</i> <b>406</b>, 541–544 (2000).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 45" data-doi="10.1038/35020123" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/35020123" data-track-label="10.1038/35020123" data-track-value="article reference" href="https://doi.org/10.1038%2F35020123" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 45" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2000Natur.406..541S" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 45" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3cXmtFensbc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 45" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10952318" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 45" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20chromatin%20remodelling%20complex%20involved%20in%20transcription%20and%20DNA%20processing&amp;journal=Nature&amp;doi=10.1038%2F35020123&amp;volume=406&amp;pages=541-544&amp;publication_year=2000&amp;author=Shen%2CX&amp;author=Mizuguchi%2CG&amp;author=Hamiche%2CA&amp;author=Wu%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Nguyen, V. Q. et al. Molecular architecture of the ATP-dependent chromatin-remodeling complex SWR1. <i>Cell</i> <b>154</b>, 1220–1231 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 46" data-doi="10.1016/j.cell.2013.08.018" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2013.08.018" data-track-label="10.1016/j.cell.2013.08.018" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2013.08.018" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 46" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsVClsbrL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 46" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24034246" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 46" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3776929" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 46" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20architecture%20of%20the%20ATP-dependent%20chromatin-remodeling%20complex%20SWR1&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2013.08.018&amp;volume=154&amp;pages=1220-1231&amp;publication_year=2013&amp;author=Nguyen%2CVQ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Lee, K. K. et al. Combinatorial depletion analysis to assemble the network architecture of the SAGA and ADA chromatin remodeling complexes. <i>Mol. Syst. Biol.</i> <b>7</b>, 503 (2011).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 47" data-doi="10.1038/msb.2011.40" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/msb.2011.40" data-track-label="10.1038/msb.2011.40" data-track-value="article reference" href="https://doi.org/10.1038%2Fmsb.2011.40" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 47" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21734642" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 47" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3159981" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 47" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Combinatorial%20depletion%20analysis%20to%20assemble%20the%20network%20architecture%20of%20the%20SAGA%20and%20ADA%20chromatin%20remodeling%20complexes&amp;journal=Mol.%20Syst.%20Biol.&amp;doi=10.1038%2Fmsb.2011.40&amp;volume=7&amp;publication_year=2011&amp;author=Lee%2CKK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Sanders, S. L., Jennings, J., Canutescu, A., Link, A. J. &amp; Weil, P. A. Proteomics of the eukaryotic transcription machinery: identification of proteins associated with components of yeast TFIID by multidimensional mass spectrometry. <i>Mol. Cell. Biol.</i> <b>22</b>, 4723–4738 (2002).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 48" data-doi="10.1128/MCB.22.13.4723-4738.2002" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1128/MCB.22.13.4723-4738.2002" data-track-label="10.1128/MCB.22.13.4723-4738.2002" data-track-value="article reference" href="https://doi.org/10.1128%2FMCB.22.13.4723-4738.2002" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 48" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38Xks1yrtr8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 48" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12052880" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 48" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC133885" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 48" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Proteomics%20of%20the%20eukaryotic%20transcription%20machinery%3A%20identification%20of%20proteins%20associated%20with%20components%20of%20yeast%20TFIID%20by%20multidimensional%20mass%20spectrometry&amp;journal=Mol.%20Cell.%20Biol.&amp;doi=10.1128%2FMCB.22.13.4723-4738.2002&amp;volume=22&amp;pages=4723-4738&amp;publication_year=2002&amp;author=Sanders%2CSL&amp;author=Jennings%2CJ&amp;author=Canutescu%2CA&amp;author=Link%2CAJ&amp;author=Weil%2CPA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Lakshminarasimhan, M. et al. Proteomic and genomic analyses of the Rvb1 and Rvb2 interaction network upon deletion of R2TP complex components. <i>Mol. Cell. Proteom. MCP</i> <b>15</b>, 960–974 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 49" data-doi="10.1074/mcp.M115.053165" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1074/mcp.M115.053165" data-track-label="10.1074/mcp.M115.053165" data-track-value="article reference" href="https://doi.org/10.1074%2Fmcp.M115.053165" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 49" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XjsVGrtb4%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 49" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26831523" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 49" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Proteomic%20and%20genomic%20analyses%20of%20the%20Rvb1%20and%20Rvb2%20interaction%20network%20upon%20deletion%20of%20R2TP%20complex%20components&amp;journal=Mol.%20Cell.%20Proteom.%20MCP&amp;doi=10.1074%2Fmcp.M115.053165&amp;volume=15&amp;pages=960-974&amp;publication_year=2016&amp;author=Lakshminarasimhan%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Leng, H. et al. FACT interacts with Set3 HDAC and fine-tunes GAL1 transcription in response to environmental stimulation. <i>Nucleic Acids Res.</i> <b>49</b>, 5502–5519 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 50" data-doi="10.1093/nar/gkab312" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkab312" data-track-label="10.1093/nar/gkab312" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkab312" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 50" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhs1WjtbnK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 50" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33963860" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 50" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8191775" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 50" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=FACT%20interacts%20with%20Set3%20HDAC%20and%20fine-tunes%20GAL1%20transcription%20in%20response%20to%20environmental%20stimulation&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkab312&amp;volume=49&amp;pages=5502-5519&amp;publication_year=2021&amp;author=Leng%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Yang, J. et al. The histone chaperone FACT contributes to DNA replication-coupled nucleosome assembly. <i>Cell Rep.</i> <b>14</b>, 1128–1141 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 51" data-doi="10.1016/j.celrep.2015.12.096" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.celrep.2015.12.096" data-track-label="10.1016/j.celrep.2015.12.096" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.celrep.2015.12.096" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 51" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVKhsr0%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 51" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26804921" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 51" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20histone%20chaperone%20FACT%20contributes%20to%20DNA%20replication-coupled%20nucleosome%20assembly&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2015.12.096&amp;volume=14&amp;pages=1128-1141&amp;publication_year=2016&amp;author=Yang%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Dormidontova, E. E., Grosberg, A. Y. &amp; Khokhlov, A. R. Intramolecular phase separation of a copolymer chain with mobile primary structure. <i>Macromol. Theory Simul.</i> <b>1</b>, 375–385 (1992).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 52" data-doi="10.1002/mats.1992.040010603" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/mats.1992.040010603" data-track-label="10.1002/mats.1992.040010603" data-track-value="article reference" href="https://doi.org/10.1002%2Fmats.1992.040010603" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 52" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK38XmtlCrsbk%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 52" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Intramolecular%20phase%20separation%20of%20a%20copolymer%20chain%20with%20mobile%20primary%20structure&amp;journal=Macromol.%20Theory%20Simul.&amp;doi=10.1002%2Fmats.1992.040010603&amp;volume=1&amp;pages=375-385&amp;publication_year=1992&amp;author=Dormidontova%2CEE&amp;author=Grosberg%2CAY&amp;author=Khokhlov%2CAR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Flory, P. J. &amp; Gee, G. Statistical thermodynamics of semi-flexible chain molecules. <i>Proc. R. Soc. Lond. Ser. A. Math. Phys. Sci.</i> <b>234</b>, 60–73 (1956).</p><p class="c-article-references__links u-hide-print"><a aria-label="ADS reference 53" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1956RSPSA.234...60F" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 53" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaG28XjsVKlsw%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 53" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Statistical%20thermodynamics%20of%20semi-flexible%20chain%20molecules&amp;journal=Proc.%20R.%20Soc.%20Lond.%20Ser.%20A.%20Math.%20Phys.%20Sci.&amp;volume=234&amp;pages=60-73&amp;publication_year=1956&amp;author=Flory%2CPJ&amp;author=Gee%2CG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Huggins, M. L. Thermodynamic properties of solutions of long-chain compounds. <i>Ann. N. Y. Acad. Sci.</i> <b>43</b>, 1–32 (1942).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 54" data-doi="10.1111/j.1749-6632.1942.tb47940.x" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1111/j.1749-6632.1942.tb47940.x" data-track-label="10.1111/j.1749-6632.1942.tb47940.x" data-track-value="article reference" href="https://doi.org/10.1111%2Fj.1749-6632.1942.tb47940.x" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 54" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1942NYASA..43....1H" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 54" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaH38XisVeksg%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 54" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Thermodynamic%20properties%20of%20solutions%20of%20long-chain%20compounds&amp;journal=Ann.%20N.%20Y.%20Acad.%20Sci.&amp;doi=10.1111%2Fj.1749-6632.1942.tb47940.x&amp;volume=43&amp;pages=1-32&amp;publication_year=1942&amp;author=Huggins%2CML" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Hult, C. et al. Enrichment of dynamic chromosomal crosslinks drive phase separation of the nucleolus. <i>Nucleic Acids Res.</i> <b>45</b>, 11159–11173 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 55" data-doi="10.1093/nar/gkx741" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkx741" data-track-label="10.1093/nar/gkx741" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkx741" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 55" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017qnsm.book.....H" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 55" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVyitrrK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 55" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28977453" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 55" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5737219" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 55" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Enrichment%20of%20dynamic%20chromosomal%20crosslinks%20drive%20phase%20separation%20of%20the%20nucleolus&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkx741&amp;volume=45&amp;pages=11159-11173&amp;publication_year=2017&amp;author=Hult%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Gibson, B. A. et al. Organization of chromatin by intrinsic and regulated phase separation. <i>Cell</i> <b>179</b>, 470–484.e421 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 56" data-doi="10.1016/j.cell.2019.08.037" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2019.08.037" data-track-label="10.1016/j.cell.2019.08.037" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.08.037" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 56" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvVentbzE" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 56" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31543265" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 56" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6778041" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 56" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Organization%20of%20chromatin%20by%20intrinsic%20and%20regulated%20phase%20separation&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.08.037&amp;volume=179&amp;pages=470-484.e421&amp;publication_year=2019&amp;author=Gibson%2CBA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Sanulli, S. et al. HP1 reshapes nucleosome core to promote heterochromatin phase separation. <i>Nature</i> <b>575</b>, 390–394 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 57" data-doi="10.1038/s41586-019-1669-2" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41586-019-1669-2" data-track-label="10.1038/s41586-019-1669-2" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41586-019-1669-2" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 57" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019Natur.575..390S" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 57" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFWnur%2FP" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 57" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31618757" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 57" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7039410" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 57" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=HP1%20reshapes%20nucleosome%20core%20to%20promote%20heterochromatin%20phase%20separation&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1669-2&amp;volume=575&amp;pages=390-394&amp;publication_year=2019&amp;author=Sanulli%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Heun, P., Laroche, T., Shimada, K., Furrer, P. &amp; Gasser, S. M. Chromosome dynamics in the yeast interphase nucleus. <i>Science</i> <b>294</b>, 2181–2186 (2001).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 58" data-doi="10.1126/science.1065366" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.1065366" data-track-label="10.1126/science.1065366" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.1065366" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 58" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2001Sci...294.2181H" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 58" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3MXptVOmtrk%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 58" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11739961" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 58" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromosome%20dynamics%20in%20the%20yeast%20interphase%20nucleus&amp;journal=Science&amp;doi=10.1126%2Fscience.1065366&amp;volume=294&amp;pages=2181-2186&amp;publication_year=2001&amp;author=Heun%2CP&amp;author=Laroche%2CT&amp;author=Shimada%2CK&amp;author=Furrer%2CP&amp;author=Gasser%2CSM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Tortora, M. M., Salari, H. &amp; Jost, D. Chromosome dynamics during interphase: a biophysical perspective. <i>Curr. Opin. Genet. Dev.</i> <b>61</b>, 37–43 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 59" data-doi="10.1016/j.gde.2020.03.001" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.gde.2020.03.001" data-track-label="10.1016/j.gde.2020.03.001" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.gde.2020.03.001" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 59" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXntVKrtbg%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 59" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32304901" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 59" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromosome%20dynamics%20during%20interphase%3A%20a%20biophysical%20perspective&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;doi=10.1016%2Fj.gde.2020.03.001&amp;volume=61&amp;pages=37-43&amp;publication_year=2020&amp;author=Tortora%2CMM&amp;author=Salari%2CH&amp;author=Jost%2CD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Erdel, F. &amp; Rippe, K. Formation of chromatin subcompartments by phase separation. <i>Biophysical J.</i> <b>114</b>, 2262–2270 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 60" data-doi="10.1016/j.bpj.2018.03.011" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.bpj.2018.03.011" data-track-label="10.1016/j.bpj.2018.03.011" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.bpj.2018.03.011" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 60" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018BpJ...114.2262E" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 60" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXmsl2hsLs%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 60" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Formation%20of%20chromatin%20subcompartments%20by%20phase%20separation&amp;journal=Biophysical%20J.&amp;doi=10.1016%2Fj.bpj.2018.03.011&amp;volume=114&amp;pages=2262-2270&amp;publication_year=2018&amp;author=Erdel%2CF&amp;author=Rippe%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Shin, Y. et al. Liquid nuclear condensates mechanically sense and restructure the genome. <i>Cell</i> <b>175</b>, 1481–1491.e1413 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 61" data-doi="10.1016/j.cell.2018.10.057" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2018.10.057" data-track-label="10.1016/j.cell.2018.10.057" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2018.10.057" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 61" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXitl2iur%2FL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 61" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30500535" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 61" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6724728" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 61" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Liquid%20nuclear%20condensates%20mechanically%20sense%20and%20restructure%20the%20genome&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.10.057&amp;volume=175&amp;pages=1481-1491.e1413&amp;publication_year=2018&amp;author=Shin%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Quail, T. et al. Force generation by protein–DNA co-condensation. <i>Nat. Phys.</i> <b>17</b>, 1007–1012 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 62" data-doi="10.1038/s41567-021-01285-1" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41567-021-01285-1" data-track-label="10.1038/s41567-021-01285-1" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41567-021-01285-1" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 62" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFWis77L" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 62" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Force%20generation%20by%20protein%E2%80%93DNA%20co-condensation&amp;journal=Nat.%20Phys.&amp;doi=10.1038%2Fs41567-021-01285-1&amp;volume=17&amp;pages=1007-1012&amp;publication_year=2021&amp;author=Quail%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Lawrimore, J. et al. The rDNA is biomolecular condensate formed by polymer-polymer phase separation and is sequestered in the nucleolus by transcription and R-loops. <i>Nucleic Acids Res.</i> <b>49</b>, 4586–4598 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 63" data-doi="10.1093/nar/gkab229" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkab229" data-track-label="10.1093/nar/gkab229" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkab229" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 63" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFWgtbbE" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 63" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33836082" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 63" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8096216" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 63" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20rDNA%20is%20biomolecular%20condensate%20formed%20by%20polymer-polymer%20phase%20separation%20and%20is%20sequestered%20in%20the%20nucleolus%20by%20transcription%20and%20R-loops&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkab229&amp;volume=49&amp;pages=4586-4598&amp;publication_year=2021&amp;author=Lawrimore%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Henninger, J. E. et al. RNA-mediated feedback control of transcriptional condensates. <i>Cell</i> <b>184</b>, 207–225 e224 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 64" data-doi="10.1016/j.cell.2020.11.030" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2020.11.030" data-track-label="10.1016/j.cell.2020.11.030" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2020.11.030" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 64" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1GntrzO" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 64" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33333019" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 64" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=RNA-mediated%20feedback%20control%20of%20transcriptional%20condensates&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2020.11.030&amp;volume=184&amp;pages=207-225%20e224&amp;publication_year=2021&amp;author=Henninger%2CJE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Sabari, B. R. et al. Coactivator condensation at super-enhancers links phase separation and gene control. <i>Science</i> <b>361</b>, eaar3958 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 65" data-doi="10.1126/science.aar3958" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.aar3958" data-track-label="10.1126/science.aar3958" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.aar3958" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 65" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29930091" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 65" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6092193" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 65" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Coactivator%20condensation%20at%20super-enhancers%20links%20phase%20separation%20and%20gene%20control&amp;journal=Science&amp;doi=10.1126%2Fscience.aar3958&amp;volume=361&amp;publication_year=2018&amp;author=Sabari%2CBR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Rudnizky, S. et al. Extended and dynamic linker histone-DNA Interactions control chromatosome compaction. <i>Mol. Cell</i> <b>81</b>, 3410–3421 e3414 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 66" data-doi="10.1016/j.molcel.2021.06.006" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2021.06.006" data-track-label="10.1016/j.molcel.2021.06.006" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2021.06.006" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 66" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVKlsb3K" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 66" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34192510" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 66" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Extended%20and%20dynamic%20linker%20histone-DNA%20Interactions%20control%20chromatosome%20compaction&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2021.06.006&amp;volume=81&amp;pages=3410-3421%20e3414&amp;publication_year=2021&amp;author=Rudnizky%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Strom, A. R. et al. Phase separation drives heterochromatin domain formation. <i>Nature</i> <b>547</b>, 241–245 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 67" data-doi="10.1038/nature22989" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature22989" data-track-label="10.1038/nature22989" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature22989" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 67" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017Natur.547..241S" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 67" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVequrfN" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 67" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28636597" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 67" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6022742" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 67" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Phase%20separation%20drives%20heterochromatin%20domain%20formation&amp;journal=Nature&amp;doi=10.1038%2Fnature22989&amp;volume=547&amp;pages=241-245&amp;publication_year=2017&amp;author=Strom%2CAR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Quinodoz, S. A. &amp; Guttman, M. Essential roles for RNA in shaping nuclear organization. <i>Cold Spring Harb. Perspect. Biol.</i> <b>14</b>, a039719 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 68" data-doi="10.1101/cshperspect.a039719" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1101/cshperspect.a039719" data-track-label="10.1101/cshperspect.a039719" data-track-value="article reference" href="https://doi.org/10.1101%2Fcshperspect.a039719" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 68" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Essential%20roles%20for%20RNA%20in%20shaping%20nuclear%20organization&amp;journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&amp;doi=10.1101%2Fcshperspect.a039719&amp;volume=14&amp;publication_year=2021&amp;author=Quinodoz%2CSA&amp;author=Guttman%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Pessina, F. et al. Functional transcription promoters at DNA double-strand breaks mediate RNA-driven phase separation of damage-response factors. <i>Nat. Cell Biol.</i> <b>21</b>, 1286–1299 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 69" data-doi="10.1038/s41556-019-0392-4" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41556-019-0392-4" data-track-label="10.1038/s41556-019-0392-4" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41556-019-0392-4" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 69" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvVOqu77E" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 69" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31570834" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 69" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6859070" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 69" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20transcription%20promoters%20at%20DNA%20double-strand%20breaks%20mediate%20RNA-driven%20phase%20separation%20of%20damage-response%20factors&amp;journal=Nat.%20Cell%20Biol.&amp;doi=10.1038%2Fs41556-019-0392-4&amp;volume=21&amp;pages=1286-1299&amp;publication_year=2019&amp;author=Pessina%2CF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Jun, S. &amp; Mulder, B. Entropy-driven spatial organization of highly confined polymers: lessons for the bacterial chromosome. <i>Proc. Natl. Acad. Sci. USA</i> <b>103</b>, 12388–12393 (2006).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 70" data-doi="10.1073/pnas.0605305103" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.0605305103" data-track-label="10.1073/pnas.0605305103" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.0605305103" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 70" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2006PNAS..10312388J" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 70" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28Xos1Kmu7s%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 70" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16885211" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 70" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1525299" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 70" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Entropy-driven%20spatial%20organization%20of%20highly%20confined%20polymers%3A%20lessons%20for%20the%20bacterial%20chromosome&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0605305103&amp;volume=103&amp;pages=12388-12393&amp;publication_year=2006&amp;author=Jun%2CS&amp;author=Mulder%2CB" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Shaw, W. M. et al. Engineering a model cell for rational tuning of GPCR signaling. <i>Cell</i> <b>177</b>, 782–796.e727 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 71" data-doi="10.1016/j.cell.2019.02.023" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2019.02.023" data-track-label="10.1016/j.cell.2019.02.023" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.02.023" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 71" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXmvFShtbc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 71" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30955892" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 71" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6476273" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 71" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Engineering%20a%20model%20cell%20for%20rational%20tuning%20of%20GPCR%20signaling&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.02.023&amp;volume=177&amp;pages=782-796.e727&amp;publication_year=2019&amp;author=Shaw%2CWM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Lee, M. E., DeLoache, W. C., Cervantes, B. &amp; Dueber, J. E. A highly characterized yeast toolkit for modular, multipart assembly. <i>ACS Synth. Biol.</i> <b>4</b>, 975–986 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 72" data-doi="10.1021/sb500366v" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/sb500366v" data-track-label="10.1021/sb500366v" data-track-value="article reference" href="https://doi.org/10.1021%2Fsb500366v" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 72" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXmsVelsr8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 72" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25871405" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 72" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20highly%20characterized%20yeast%20toolkit%20for%20modular%2C%20multipart%20assembly&amp;journal=ACS%20Synth.%20Biol.&amp;doi=10.1021%2Fsb500366v&amp;volume=4&amp;pages=975-986&amp;publication_year=2015&amp;author=Lee%2CME&amp;author=DeLoache%2CWC&amp;author=Cervantes%2CB&amp;author=Dueber%2CJE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Malleshaiah, M. K., Shahrezaei, V., Swain, P. S. &amp; Michnick, S. W. The scaffold protein Ste5 directly controls a switch-like mating decision in yeast. <i>Nature</i> <b>465</b>, 101–105 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 73" data-doi="10.1038/nature08946" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature08946" data-track-label="10.1038/nature08946" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature08946" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 73" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2010Natur.465..101M" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 73" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXkvVCjtL0%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 73" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20400943" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 73" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20scaffold%20protein%20Ste5%20directly%20controls%20a%20switch-like%20mating%20decision%20in%20yeast&amp;journal=Nature&amp;doi=10.1038%2Fnature08946&amp;volume=465&amp;pages=101-105&amp;publication_year=2010&amp;author=Malleshaiah%2CMK&amp;author=Shahrezaei%2CV&amp;author=Swain%2CPS&amp;author=Michnick%2CSW" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Therizols, P., Duong, T., Dujon, B., Zimmer, C. &amp; Fabre, E. Chromosome arm length and nuclear constraints determine the dynamic relationship of yeast subtelomeres. <i>Proc. Natl. Acad. Sci. USA</i> <b>107</b>, 2025–2030 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 74" data-doi="10.1073/pnas.0914187107" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.0914187107" data-track-label="10.1073/pnas.0914187107" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.0914187107" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 74" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2010PNAS..107.2025T" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 74" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhvFCit7o%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 74" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20080699" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 74" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836701" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 74" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromosome%20arm%20length%20and%20nuclear%20constraints%20determine%20the%20dynamic%20relationship%20of%20yeast%20subtelomeres&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0914187107&amp;volume=107&amp;pages=2025-2030&amp;publication_year=2010&amp;author=Therizols%2CP&amp;author=Duong%2CT&amp;author=Dujon%2CB&amp;author=Zimmer%2CC&amp;author=Fabre%2CE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Kechkar, A., Nair, D., Heilemann, M., Choquet, D. &amp; Sibarita, J. B. Real-time analysis and visualization for single-molecule based super-resolution microscopy. <i>PLoS One</i> <b>8</b>, e62918 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 75" data-doi="10.1371/journal.pone.0062918" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1371/journal.pone.0062918" data-track-label="10.1371/journal.pone.0062918" data-track-value="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0062918" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 75" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013PLoSO...862918K" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 75" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXnsVKhurc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 75" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23646160" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 75" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3639901" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 75" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Real-time%20analysis%20and%20visualization%20for%20single-molecule%20based%20super-resolution%20microscopy&amp;journal=PLoS%20One&amp;doi=10.1371%2Fjournal.pone.0062918&amp;volume=8&amp;publication_year=2013&amp;author=Kechkar%2CA&amp;author=Nair%2CD&amp;author=Heilemann%2CM&amp;author=Choquet%2CD&amp;author=Sibarita%2CJB" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. <i>Nat. Methods</i> <b>9</b>, 676–682 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 76" data-doi="10.1038/nmeth.2019" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nmeth.2019" data-track-label="10.1038/nmeth.2019" data-track-value="article reference" href="https://doi.org/10.1038%2Fnmeth.2019" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 76" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVKnurbJ" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 76" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22743772" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 76" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Fiji%3A%20an%20open-source%20platform%20for%20biological-image%20analysis&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.2019&amp;volume=9&amp;pages=676-682&amp;publication_year=2012&amp;author=Schindelin%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Lawrimore, J. et al. ChromoShake: a chromosome dynamics simulator reveals that chromatin loops stiffen centromeric chromatin. <i>Mol. Biol. Cell</i> <b>27</b>, 153–166 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 77" data-doi="10.1091/mbc.E15-08-0575" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1091/mbc.E15-08-0575" data-track-label="10.1091/mbc.E15-08-0575" data-track-value="article reference" href="https://doi.org/10.1091%2Fmbc.E15-08-0575" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 77" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xpt12nsbo%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 77" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26538024" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 77" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4694754" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 77" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=ChromoShake%3A%20a%20chromosome%20dynamics%20simulator%20reveals%20that%20chromatin%20loops%20stiffen%20centromeric%20chromatin&amp;journal=Mol.%20Biol.%20Cell&amp;doi=10.1091%2Fmbc.E15-08-0575&amp;volume=27&amp;pages=153-166&amp;publication_year=2016&amp;author=Lawrimore%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Fisher, J. K. et al. DNA relaxation dynamics as a probe for the intracellular environment. <i>Proc. Natl. Acad. Sci. USA</i> <b>106</b>, 9250–9255 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 78" data-doi="10.1073/pnas.0812723106" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.0812723106" data-track-label="10.1073/pnas.0812723106" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.0812723106" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 78" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2009PNAS..106.9250F" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 78" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1CgtL0%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 78" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19478070" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 78" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2695107" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 78" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20relaxation%20dynamics%20as%20a%20probe%20for%20the%20intracellular%20environment&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0812723106&amp;volume=106&amp;pages=9250-9255&amp;publication_year=2009&amp;author=Fisher%2CJK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Jeronimo, C., Poitras, C. &amp; Robert, F. Histone recycling by FACT and Spt6 during transcription prevents the scrambling of histone modifications. <i>Cell Rep.</i> <b>28</b>, 1206–1218.e1208 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 79" data-doi="10.1016/j.celrep.2019.06.097" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.celrep.2019.06.097" data-track-label="10.1016/j.celrep.2019.06.097" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.celrep.2019.06.097" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 79" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFSrsbjL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 79" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31365865" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 79" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Histone%20recycling%20by%20FACT%20and%20Spt6%20during%20transcription%20prevents%20the%20scrambling%20of%20histone%20modifications&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2019.06.097&amp;volume=28&amp;pages=1206-1218.e1208&amp;publication_year=2019&amp;author=Jeronimo%2CC&amp;author=Poitras%2CC&amp;author=Robert%2CF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80."><p class="c-article-references__text" id="ref-CR80">Oeffinger, M. et al. Comprehensive analysis of diverse ribonucleoprotein complexes. <i>Nat. Methods</i> <b>4</b>, 951–956 (2007).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 80" data-doi="10.1038/nmeth1101" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nmeth1101" data-track-label="10.1038/nmeth1101" data-track-value="article reference" href="https://doi.org/10.1038%2Fnmeth1101" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 80" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXht1aqur3O" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 80" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17922018" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 80" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Comprehensive%20analysis%20of%20diverse%20ribonucleoprotein%20complexes&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth1101&amp;volume=4&amp;pages=951-956&amp;publication_year=2007&amp;author=Oeffinger%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81."><p class="c-article-references__text" id="ref-CR81">Cherry, J. M. et al. Saccharomyces Genome Database: the genomics resource of budding yeast. <i>Nucleic Acids Res.</i> <b>40</b>, D700–D705 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 81" data-doi="10.1093/nar/gkr1029" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkr1029" data-track-label="10.1093/nar/gkr1029" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkr1029" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 81" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs12htLfF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 81" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22110037" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 81" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Saccharomyces%20Genome%20Database%3A%20the%20genomics%20resource%20of%20budding%20yeast&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkr1029&amp;volume=40&amp;pages=D700-D705&amp;publication_year=2012&amp;author=Cherry%2CJM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="82."><p class="c-article-references__text" id="ref-CR82">The UniProt, C. UniProt: the universal protein knowledgebase in 2021. <i>Nucleic Acids Res.</i> <b>49</b>, D480–D489 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 82" data-doi="10.1093/nar/gkaa1100" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkaa1100" data-track-label="10.1093/nar/gkaa1100" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkaa1100" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 82" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=UniProt%3A%20the%20universal%20protein%20knowledgebase%20in%202021&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkaa1100&amp;volume=49&amp;pages=D480-D489&amp;publication_year=2021" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="83."><p class="c-article-references__text" id="ref-CR83">Oughtred, R. et al. The BioGRID database: a comprehensive biomedical resource of curated protein, genetic, and chemical interactions. <i>Protein Sci.</i> <b>30</b>, 187–200 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 83" data-doi="10.1002/pro.3978" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/pro.3978" data-track-label="10.1002/pro.3978" data-track-value="article reference" href="https://doi.org/10.1002%2Fpro.3978" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 83" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitl2nu7vO" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 83" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33070389" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 83" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20BioGRID%20database%3A%20a%20comprehensive%20biomedical%20resource%20of%20curated%20protein%2C%20genetic%2C%20and%20chemical%20interactions&amp;journal=Protein%20Sci.&amp;doi=10.1002%2Fpro.3978&amp;volume=30&amp;pages=187-200&amp;publication_year=2021&amp;author=Oughtred%2CR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1038/s41467-023-36391-6?format=refman&amp;flavour=references" rel="nofollow">Download references<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-download-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>The authors thank Jason Brickner (Northwestern University), Tom Ellis (Imperial College London), Amine Nourani (Université Laval), Oliver Valerius (Georg-August-University), and Daniel Zenklusen (Université de Montréal) for providing reagents, plasmids and strains. The authors acknowledge support from Canadian Institutes of Health Research (CIHR) grant MOP-GMX-152556 and Human Frontier Science Program grant RGP0034/2017 (S.W.M.); CIHR grant PJT-153313 (M.O.); CIHR grant PJT-162334 (F.R.); and National Institutes of Health grant R37 GM32238 (K.B.).</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Département de Biochimie, Université de Montréal, C.P. 6128, Succursale centre-ville, Montréal, QC, H3C 3J7, Canada</p><p class="c-article-author-affiliation__authors-list">Lidice González, Marlene Oeffinger &amp; Stephen W. Michnick</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Department of Biology, University of North Carolina at Chapel Hill, Chapel Hill, NC, 27599, USA</p><p class="c-article-author-affiliation__authors-list">Daniel Kolbin &amp; Kerry Bloom</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Institut de recherches cliniques de Montréal, 110 Avenue des Pins Ouest, Montréal, QC, H2W 1R7, Canada</p><p class="c-article-author-affiliation__authors-list">Christian Trahan, Célia Jeronimo, François Robert &amp; Marlene Oeffinger</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Faculty of Medicine, Division of Experimental Medicine, McGill University, Montréal, QC, H3A 1A3, Canada</p><p class="c-article-author-affiliation__authors-list">François Robert &amp; Marlene Oeffinger</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Département de Médecine, Faculté de Médecine, Université de Montréal, 2900 Boul. Édouard-Montpetit, Montréal, QC, H3T 1J4, Canada</p><p class="c-article-author-affiliation__authors-list">François Robert</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Lidice-Gonz_lez-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Lidice González</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Lidice%20Gonz%C3%A1lez" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Lidice%20Gonz%C3%A1lez" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Lidice%20Gonz%C3%A1lez%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Daniel-Kolbin-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Daniel Kolbin</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Daniel%20Kolbin" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Daniel%20Kolbin" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Daniel%20Kolbin%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Christian-Trahan-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Christian Trahan</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Christian%20Trahan" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Christian%20Trahan" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Christian%20Trahan%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-C_lia-Jeronimo-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Célia Jeronimo</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=C%C3%A9lia%20Jeronimo" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=C%C3%A9lia%20Jeronimo" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22C%C3%A9lia%20Jeronimo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Fran_ois-Robert-Aff3-Aff4-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">François Robert</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Fran%C3%A7ois%20Robert" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Fran%C3%A7ois%20Robert" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Fran%C3%A7ois%20Robert%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Marlene-Oeffinger-Aff1-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Marlene Oeffinger</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Marlene%20Oeffinger" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Marlene%20Oeffinger" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Marlene%20Oeffinger%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Kerry-Bloom-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Kerry Bloom</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Kerry%20Bloom" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kerry%20Bloom" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kerry%20Bloom%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Stephen_W_-Michnick-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Stephen W. Michnick</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Stephen%20W.%20Michnick" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Stephen%20W.%20Michnick" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Stephen%20W.%20Michnick%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>Conceptualization: L.G. and S.W.M.; Methodology: L.G., D.K., C.T., C.J.; Investigation: L.G., D.K., C.T., C.J.; Data analysis: L.G., D.K., C.T., C.J., F.R., M.O., K.B., S.W.M.; Writing-review: L.G., D.K., C.T., C.J., F.R., M.O., K.B., S.W.M.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a href="mailto:stephen.michnick@umontreal.ca" id="corresp-c1">Stephen W. Michnick</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
<h3 class="c-article__sub-heading" id="FPar2">Competing interests</h3>
<p>The authors declare no competing interests.</p>
</div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
<h3 class="c-article__sub-heading" id="FPar1">Peer review information</h3>
<p><i>Nature Communications</i> thanks the anonymous reviewer(s) for their contribution to the peer review of this work. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-023-36391-6#MOESM2">Peer reviewer reports</a> are available.</p>
</div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec24-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec24">Supplementary information</h2><div class="c-article-section__content" id="Sec24-content"><div data-test="supplementary-info"><div class="c-article-figshare-container" data-test="figshare-container" id="figshareContainer"></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_MOESM1_ESM.pdf">Supplementary Information</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="peer review file" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_MOESM2_ESM.pdf">Peer Review File</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="description of additional supplementary files" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_MOESM3_ESM.pdf">Description of Additional Supplementary Files</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary data 1-6" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_MOESM4_ESM.zip">Supplementary Data 1-6</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM5"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary movie 1" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_MOESM5_ESM.mp4">Supplementary Movie 1</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_MOESM6_ESM.pdf">Reporting Summary</a></h3></div></div></div></div></section><section data-title="Source data"><div class="c-article-section" id="Sec25-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec25">Source data</h2><div class="c-article-section__content" id="Sec25-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="source data" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-023-36391-6/MediaObjects/41467_2023_36391_MOESM7_ESM.zip">Source Data</a></h3></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
<p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Adaptive%20partitioning%20of%20a%20gene%20locus%20to%20the%20nuclear%20envelope%20in%20Saccharomyces%20cerevisiae%20is%20driven%20by%20polymer-polymer%20phase%20separation&amp;author=Lidice%20Gonz%C3%A1lez%20et%20al&amp;contentID=10.1038%2Fs41467-023-36391-6&amp;copyright=The%20Author%28s%29&amp;publication=2041-1723&amp;publicationDate=2023-02-28&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41467-023-36391-6" data-test="crossmark" data-track="click" data-track-action="Click Crossmark" data-track-label="link" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41467-023-36391-6" rel="noopener" target="_blank"><img alt="Check for updates. Verify currency and authenticity via CrossMark" height="81" loading="lazy" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>" width="57"/></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">González, L., Kolbin, D., Trahan, C. <i>et al.</i> Adaptive partitioning of a gene locus to the nuclear envelope in <i>Saccharomyces cerevisiae</i> is driven by polymer-polymer phase separation.
                    <i>Nat Commun</i> <b>14</b>, 1135 (2023). https://doi.org/10.1038/s41467-023-36391-6</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-external="" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1038/s41467-023-36391-6?format=refman&amp;flavour=citation" rel="nofollow">Download citation<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-download-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-05-28">28 May 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2023-01-30">30 January 2023</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2023-02-28">28 February 2023</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41467-023-36391-6</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" data-track="click" data-track-action="get shareable link" data-track-external="" data-track-label="button" id="get-share-url" type="button">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" data-track="click" data-track-action="select share url" data-track-label="button" id="share-url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" data-track="click" data-track-action="copy share url" data-track-external="" data-track-label="button" id="copy-share-url" type="button">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
</div>
</div>
</article>
</main>
<aside aria-label="Article navigation" class="c-article-extras u-hide-print" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
<div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41467-023-36391-6.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
<div class="c-reading-companion">
<div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
<div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
<div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu="">
<div class="c-ad c-ad--300x250">
<div class="c-ad__inner">
<p class="c-ad__label">Advertisement</p>
<div class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-gpt="" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41467-023-36391-6;doi=10.1038/s41467-023-36391-6;techmeta=14,15,19,38,45,58,63,82,88,90;subjmeta=100,102,199,200,208,337,631;kwrd=Chromatin+remodelling,Gene+expression,Gene+regulation" data-gpt-unitpath="/285/nature_communications/article" data-pa11y-ignore="" data-test="right-ad" id="div-gpt-ad-right-2">
<noscript>
<a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature_communications/article&amp;sz=300x250&amp;c=1127059744&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41467-023-36391-6%26doi%3D10.1038/s41467-023-36391-6%26techmeta%3D14,15,19,38,45,58,63,82,88,90%26subjmeta%3D100,102,199,200,208,337,631%26kwrd%3DChromatin+remodelling,Gene+expression,Gene+regulation">
<img alt="Advertisement" data-test="gpt-advert-fallback-img" height="250" src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature_communications/article&amp;sz=300x250&amp;c=1127059744&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41467-023-36391-6%26doi%3D10.1038/s41467-023-36391-6%26techmeta%3D14,15,19,38,45,58,63,82,88,90%26subjmeta%3D100,102,199,200,208,337,631%26kwrd%3DChromatin+remodelling,Gene+expression,Gene+regulation" width="300"/></a>
</noscript>
</div>
</div>
</div>
</div>
</div>
<div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
<div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
</div>
</div>
</aside>
</div>
<nav aria-labelledby="Explore-content" class="c-header__dropdown" data-test="Explore-content" data-track-component="nature-150-split-header" id="explore">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="Explore-content">Explore content</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="research articles" data-track-label="link" href="/ncomms/research-articles">
                                    Research articles
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" href="/ncomms/reviews-and-analysis">
                                    Reviews &amp; Analysis
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="news &amp; comment" data-track-label="link" href="/ncomms/news-and-comment">
                                    News &amp; Comment
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="videos" data-track-label="link" href="/ncomms/video">
                                    Videos
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="collections" data-track-label="link" href="/ncomms/collections">
                                    Collections
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="subjects" data-track-label="link" href="/ncomms/browse-subjects">
                                    Subjects
                                </a>
</li>
</ul>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="facebook" data-track-label="link" href="https://www.facebook.com/NatureCommunications">Follow us on Facebook
                            </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="twitter" data-track-label="link" href="https://twitter.com/NatureComms">Follow us on Twitter
                            </a>
</li>
<li class="c-header__item c-header__item--hide-lg">
<a class="c-header__link" data-track="click" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=264" rel="nofollow">Sign up for alerts<svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--hide-lg">
<a class="c-header__link" data-track="click" data-track-action="rss feed" data-track-label="link" href="https://www.nature.com/ncomms.rss">
<span>RSS feed</span>
</a>
</li>
</ul>
</div>
</nav>
<nav aria-labelledby="About-the-journal" class="c-header__dropdown" data-test="about-the-journal" data-track-component="nature-150-split-header" id="about-the-journal">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="About-the-journal">About the journal</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="aims &amp; scope" data-track-label="link" href="/ncomms/aims">
                                    Aims &amp; Scope
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editors" data-track-label="link" href="/ncomms/editors">
                                    Editors
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="journal information" data-track-label="link" href="/ncomms/journal-information">
                                    Journal Information
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="open access fees and funding" data-track-label="link" href="/ncomms/open-access">
                                    Open Access Fees and Funding
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="calls for papers" data-track-label="link" href="/ncomms/calls-for-papers">
                                    Calls for Papers
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editorial values statement" data-track-label="link" href="/ncomms/editorial-values-statement">
                                    Editorial Values Statement
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="journal metrics" data-track-label="link" href="/ncomms/journal-impact">
                                    Journal Metrics
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editors' highlights" data-track-label="link" href="/ncomms/editorshighlights">
                                    Editors' Highlights
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="contact" data-track-label="link" href="/ncomms/contact">
                                    Contact
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editorial policies" data-track-label="link" href="/ncomms/editorial-policies">
                                    Editorial policies
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="top articles" data-track-label="link" href="/ncomms/top-articles">
                                    Top Articles
                                </a>
</li>
</ul>
</div>
</nav>
<nav aria-labelledby="Publish-with-us-label" class="c-header__dropdown" data-test="publish-with-us" data-track-component="nature-150-split-header" id="publish-with-us">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="Publish-with-us-label">Publish with us</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="for authors" data-track-label="link" href="/ncomms/submit">
                                    For authors
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="for reviewers" data-track-label="link" href="/ncomms/for-reviewers">
                                    For Reviewers
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-action="manuscript author services" data-track-context="header publish with us dropdown menu" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
</li>
<li class="c-header__item c-header__item--keyline">
<a class="c-header__link" data-gtm-criteo="submit-manuscript" data-track="click_submit_manuscript" data-track-action="submit manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-external="" data-track-label="link (publish with us dropdown menu)" href="https://mts-ncomms.nature.com/">Submit manuscript<svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
</a>
</li>
</ul>
</div>
</nav>
<div class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header" id="search-menu">
<div class="c-header__container">
<h2 class="c-header__visually-hidden">Search</h2>
<form action="/search" autocomplete="off" class="c-header__search-form" data-test="inline-search" method="get" role="search">
<label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
<div class="c-header__search-layout c-header__search-layout--max-width">
<div>
<input class="c-header__input" id="keywords" name="q" required="" type="text" value=""/>
</div>
<div class="c-header__search-layout">
<div>
<label class="c-header__visually-hidden" for="results-from">Show results from</label>
<select class="c-header__select" id="results-from" name="journal">
<option selected="" value="">All journals</option>
<option value="ncomms">This journal</option>
</select>
</div>
<div>
<button class="c-header__search-button" type="submit">Search</button>
</div>
</div>
</div>
</form>
<div class="c-header__flush">
<a class="c-header__link" data-track="click" data-track-action="advanced search" data-track-label="link" href="/search/advanced">
                Advanced search
            </a>
</div>
<h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
<ul class="c-header__list">
<li><a class="c-header__link" data-track="click" data-track-action="explore articles by subject" data-track-label="link" href="/subjects">Explore articles by subject</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="find a job" data-track-label="link" href="/naturecareers">Find a job</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="guide to authors" data-track-label="link" href="/authors/index.html">Guide to authors</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="editorial policies" data-track-label="link" href="/authors/editorial_policies/">Editorial policies</a></li>
</ul>
</div>
</div>
<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
<meta content="Springer Nature" itemprop="publisher"/>
<div class="u-mt-16 u-mb-16">
<div class="u-container">
<div class="u-display-flex u-flex-wrap u-justify-content-space-between">
<p class="c-meta u-ma-0 u-flex-shrink">
<span class="c-meta__item">
                    Nature Communications (<i>Nat Commun</i>)
                </span>
<span class="c-meta__item">
<abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">2041-1723</span> (online)
    </span>
</p>
</div>
</div>
</div>
<div class="c-footer">
<div class="u-hide-print" data-track-component="footer">
<h2 class="u-visually-hidden">nature.com sitemap</h2>
<div class="c-footer__container">
<div class="c-footer__grid c-footer__group--separator">
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="about us" data-track-label="link" href="https://www.nature.com/npg_/company_info/index.html">About us</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="press releases" data-track-label="link" href="https://www.nature.com/npg_/press_room/press_releases.html">Press releases</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="press office" data-track-label="link" href="https://press.nature.com/">Press office</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="contact us" data-track-label="link" href="https://support.nature.com/support/home">Contact us</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Discover content</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="journals a-z" data-track-label="link" href="https://www.nature.com/siteindex">Journals A-Z</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="article by subject" data-track-label="link" href="https://www.nature.com/subjects">Articles by subject</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="protocols.io" data-track-label="link" href="https://www.protocols.io/">protocols.io</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature index" data-track-label="link" href="https://www.natureindex.com/">Nature Index</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link" href="https://www.nature.com/authors/editorial_policies">Nature portfolio policies</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="open access" data-track-label="link" href="https://www.nature.com/nature-research/open-access">Open access</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="reprints and permissions" data-track-label="link" href="https://www.nature.com/reprints">Reprints &amp; permissions</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="data research service" data-track-label="link" href="https://www.springernature.com/gp/authors/research-data">Research data</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="language editing" data-track-label="link" href="https://authorservices.springernature.com/language-editing/">Language editing</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="scientific editing" data-track-label="link" href="https://authorservices.springernature.com/scientific-editing/">Scientific editing</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature masterclasses" data-track-label="link" href="https://masterclasses.nature.com/">Nature Masterclasses</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="research solutions" data-track-label="link" href="https://solutions.springernature.com/">Research Solutions</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="librarian service and tools" data-track-label="link" href="https://www.springernature.com/gp/librarians/tools-services">Librarian service &amp; tools</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="librarian portal" data-track-label="link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal">Librarian portal</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="open research" data-track-label="link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions">Open research</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="Recommend to library" data-track-label="link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library">Recommend to library</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="advertising" data-track-label="link" href="https://partnerships.nature.com/product/digital-advertising/">Advertising</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="partnerships and services" data-track-label="link" href="https://partnerships.nature.com/">Partnerships &amp; Services</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="media kits" data-track-label="link" href="https://partnerships.nature.com/media-kits/">Media kits</a>
</li>
<li class="c-footer__item"><a class="c-footer__link" data-track-action="branded content" data-track-label="link" href="https://partnerships.nature.com/product/branded-content-native-advertising/">Branded
                        content</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Professional development</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature careers" data-track-label="link" href="https://www.nature.com/naturecareers/">Nature Careers</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature conferences" data-track-label="link" href="https://conferences.nature.com">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Regional websites</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature africa" data-track-label="link" href="https://www.nature.com/natafrica">Nature Africa</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature china" data-track-label="link" href="http://www.naturechina.com">Nature China</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature india" data-track-label="link" href="https://www.nature.com/nindia">Nature India</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature Italy" data-track-label="link" href="https://www.nature.com/natitaly">Nature Italy</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature japan" data-track-label="link" href="https://www.natureasia.com/ja-jp">Nature Japan</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature middle east" data-track-label="link" href="https://www.nature.com/nmiddleeast">Nature Middle East</a></li>
</ul>
</div>
</div>
</div>
<div class="c-footer__container">
<ul class="c-footer__links">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="privacy policy" data-track-label="link" href="https://www.nature.com/info/privacy">Privacy
                Policy</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="use of cookies" data-track-label="link" href="https://www.nature.com/info/cookies">Use
                of cookies</a></li>
<li class="c-footer__item">
<button class="optanon-toggle-display c-footer__link" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link" onclick="javascript:;">Your privacy choices/Manage cookies
                </button>
</li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="legal notice" data-track-label="link" href="https://www.nature.com/info/legal-notice">Legal
                notice</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="accessibility statement" data-track-label="link" href="https://www.nature.com/info/accessibility-statement">Accessibility
                statement</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="terms and conditions" data-track-label="link" href="https://www.nature.com/info/terms-and-conditions">Terms &amp; Conditions</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="california privacy statement" data-track-label="link" href="https://www.springernature.com/ccpa">Your US state privacy rights</a></li>
</ul>
</div>
</div>
<div class="c-footer__container">
<a class="c-footer__link" href="https://www.springernature.com/">
<img alt="Springer Nature" height="20" loading="lazy" src="/static/images/logos/sn-logo-white-ea63208b81.svg" width="200"/>
</a>
<p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>
</div>
<div aria-hidden="true" class="u-visually-hidden">
<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path d="M0 .74h56.72v55.24H0z" id="a"></path></defs><symbol id="icon-access" viewbox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewbox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewbox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewbox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewbox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewbox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewbox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewbox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewbox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewbox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewbox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewbox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewbox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewbox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewbox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewbox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewbox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewbox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewbox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewbox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewbox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewbox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewbox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewbox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewbox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewbox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewbox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewbox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewbox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewbox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewbox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewbox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewbox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewbox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewbox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewbox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewbox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewbox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewbox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewbox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewbox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewbox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewbox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewbox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewbox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewbox="0 0 56.69 56.69"><path d="M0 0h56.69v56.69H0z" style="fill:none"></path><clippath id="b"><use style="overflow:visible" xlink:href="#a"></use></clippath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewbox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewbox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewbox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewbox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewbox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewbox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-download-medium" viewbox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-mail-medium" viewbox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewbox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewbox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewbox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewbox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewbox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewbox="0 0 100 100"><path clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z" fill-rule="evenodd"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewbox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>
<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif" data-component-expirydays="30" data-component-id="nature-briefing-banner" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Flagship">
<div class="c-site-messages__banner-large">
<div class="c-site-messages__close-container">
<button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
<svg aria-hidden="true" focusable="false" height="25px" version="1.1" viewbox="0 0 25 25" width="25px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<title>Close banner</title>
<defs></defs>
<g fill="none" fill-rule="evenodd" stroke="none" stroke-width="1">
<rect height="25" opacity="0" width="25" x="0" y="0"></rect>
<path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
</g>
</svg>
<span class="visually-hidden">Close</span>
</button>
</div>
<div class="c-site-messages__form-container">
<div class="grid grid-12 last">
<div class="grid grid-4">
<img alt="Nature Briefing" height="40" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250"/>
<p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
</div>
<div class="grid grid-8 last">
<form action="https://www.nature.com/briefing/briefing" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship" method="post">
<input id="briefing-banner-signup-form-input-track-originReferralPoint" name="track_originReferralPoint" type="hidden" value="MainBriefingBanner"/>
<input id="briefing-banner-signup-form-input-track-formType" name="track_formType" type="hidden" value="DirectEmailBanner"/>
<input id="gdpr_tick_banner" name="gdpr_tick" type="hidden" value="false"/>
<input id="marketing_input_banner" name="marketing" type="hidden" value="false"/>
<input id="marketing_tick_banner" name="marketing_tick" type="hidden" value="false"/>
<input id="brieferEntryPoint_banner" name="brieferEntryPoint" type="hidden" value="MainBriefingBanner"/>
<label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>
<div class="nature-briefing-banner__email-wrapper">
<input class="nature-briefing-banner__email-input box-sizing text14" data-test-element="briefing-emailbanner-email-input" id="emailAddress" name="emailAddress" placeholder="e.g. jo.smith@university.ac.uk" required="" type="email" value=""/>
<input id="defaultNewsletter_banner" name="N:nature_briefing_daily" type="hidden" value="true"/>
<button class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button" type="submit">Sign up</button>
</div>
<div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
<input class="nature-briefing-banner__checkbox-checkbox" data-test-element="briefing-emailbanner-gdpr-checkbox" id="gdpr-briefing-banner-checkbox" name="gdpr" required="" type="checkbox" value="true"/>
<label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
</div>
</form>
</div>
</div>
</div>
</div>
<div class="c-site-messages__banner-small">
<div class="c-site-messages__close-container">
<button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
<svg aria-hidden="true" focusable="false" height="25px" version="1.1" viewbox="0 0 25 25" width="25px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<title>Close banner</title>
<defs></defs>
<g fill="none" fill-rule="evenodd" stroke="none" stroke-width="1">
<rect height="25" opacity="0" width="25" x="0" y="0"></rect>
<path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
</g>
</svg>
<span class="visually-hidden">Close</span>
</button>
</div>
<div class="c-site-messages__content text14">
<span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
<a class="nature-briefing__link text14 sans-serif" data-test-element="briefing-banner-link" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner" rel="noreferrer noopener" target="_blank">Sign up for Nature Briefing
            </a>
</div>
</div>
</div>
<noscript>
<img alt="" height="0" hidden="" src="https://verify.nature.com/verify/nature.png" style="display: none" width="0"/>
</noscript>
<script async="" src="//content.readcube.com/ping?doi=10.1038/s41467-023-36391-6&amp;format=js&amp;last_modified=2023-04-21"></script>
</body>
</html>