<!DOCTYPE html>
<html class="grade-c" lang="en">
<head>
<title>CHARMM-GUI Multicomponent Assembler for modeling and simulation of complex multicomponent systems | Nature Communications</title>
<link href="https://www.nature.com/ncomms.rss" rel="alternate" type="application/rss+xml"/>
<script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
<link crossorigin="" href="https://cmp.nature.com" rel="preconnect"/>
<meta content="IE=edge" http-equiv="X-UA-Compatible"/>
<meta content="pc,mobile" name="applicable-device"/>
<meta content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes" name="viewport"/>
<meta content="5a2dc4ab3fcb9b0393241ffbbb490480" name="360-site-verification"/>
<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"biochemistry;computational-biophysics;computational-platforms-and-environments","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Communications","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41467-024-49700-4"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Nathan R. Kern","Jumin Lee","Yeol Kyo Choi","Wonpil Im"],"publishedAt":1719446400,"publishedAtString":"2024-06-27","title":"CHARMM-GUI Multicomponent Assembler for modeling and simulation of complex multicomponent systems","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Biochemistry,Computational biophysics,Computational platforms and environments"},"journal":{"pcode":"ncomms","title":"nature communications","volume":"15","issue":"1","id":41467,"publishingModel":"Open Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>
<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
<style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>
<link data-inline-css-source="critical-css" data-test="critical-css-handler" href="/static/css/enhanced-article-nature-branded-d54f9178e8.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null" rel="stylesheet"/>
<noscript>
<link href="/static/css/enhanced-article-nature-branded-d54f9178e8.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" rel="stylesheet" type="text/css"/>
</noscript>
<link href="/static/css/article-print-122346e276.css" media="print" rel="stylesheet" type="text/css"/>
<link href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png" rel="apple-touch-icon" sizes="180x180"/>
<link href="/static/images/favicons/nature/favicon-48x48-b52890008c.png" rel="icon" sizes="48x48" type="image/png"/>
<link href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png" rel="icon" sizes="32x32" type="image/png"/>
<link href="/static/images/favicons/nature/favicon-16x16-951651ab72.png" rel="icon" sizes="16x16" type="image/png"/>
<link crossorigin="use-credentials" href="/static/manifest.json" rel="manifest"/>
<link color="#000000" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" rel="mask-icon"/>
<link href="/static/images/favicons/nature/favicon.ico" rel="shortcut icon"/>
<meta content="#000000" name="msapplication-TileColor"/>
<meta content="/static/browserconfig.xml" name="msapplication-config"/>
<meta content="#000000" name="theme-color"/>
<meta content="Nature" name="application-name"/>
<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>
<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->
<script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-74.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-74.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>
<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-606cb67187.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-e919764a53.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>
<meta content="noarchive" name="robots"/>
<meta content="Yes" name="access"/>
<link href="https://www.nature.com/search" rel="search"/>
<link href="https://www.nature.com/opensearch/opensearch.xml" rel="search" title="nature.com" type="application/opensearchdescription+xml"/>
<link href="https://www.nature.com/opensearch/request" rel="search" title="nature.com" type="application/sru+xml"/>
<script type="application/ld+json">{"mainEntity":{"headline":"CHARMM-GUI Multicomponent Assembler for modeling and simulation of complex multicomponent systems","description":"Atomic-scale molecular modeling and simulation are powerful tools for computational biology. However, constructing models with large, densely packed molecules, non-water solvents, or with combinations of multiple biomembranes, polymers, and nanomaterials remains challenging and requires significant time and expertise. Furthermore, existing tools do not support such assemblies under the periodic boundary conditions (PBC) necessary for molecular simulation. Here, we describe Multicomponent Assembler in CHARMM-GUI that automates complex molecular assembly and simulation input preparation under the PBC. In this work, we demonstrate its versatility by preparing 6 challenging systems with varying density of large components: (1) solvated proteins, (2) solvated proteins with a pre-equilibrated membrane, (3) solvated proteins with a sheet-like nanomaterial, (4) solvated proteins with a sheet-like polymer, (5) a mixed membrane-nanomaterial system, and (6) a sheet-like polymer with gaseous solvent. Multicomponent Assembler is expected to be a unique cyberinfrastructure to study complex interactions between small molecules, biomacromolecules, polymers, and nanomaterials. Building atomistic models with molecules of varying size and density requires significant time and expertize. Here, authors describe CHARMM-GUI Multicomponent Assembler that guides complex molecular assembly and simulation input preparation under periodic boundary conditions.","datePublished":"2024-06-27T00:00:00Z","dateModified":"2024-06-27T00:00:00Z","pageStart":"1","pageEnd":"14","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41467-024-49700-4","keywords":["Biochemistry","Computational biophysics","Computational platforms and environments","Science","Humanities and Social Sciences","multidisciplinary"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig6_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig7_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig8_HTML.png"],"isPartOf":{"name":"Nature Communications","issn":["2041-1723"],"volumeNumber":"15","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group UK","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Nathan R. Kern","url":"http://orcid.org/0000-0003-0936-7274","affiliation":[{"name":"Lehigh University","address":{"name":"Department of Computer Science & Engineering, Lehigh University, Bethlehem, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jumin Lee","url":"http://orcid.org/0000-0002-1008-0118","affiliation":[{"name":"Lehigh University","address":{"name":"Department of Biological Sciences, Lehigh University, Bethlehem, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Yeol Kyo Choi","url":"http://orcid.org/0000-0002-4218-7139","affiliation":[{"name":"Lehigh University","address":{"name":"Department of Biological Sciences, Lehigh University, Bethlehem, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Wonpil Im","url":"http://orcid.org/0000-0001-5642-6041","affiliation":[{"name":"Lehigh University","address":{"name":"Department of Computer Science & Engineering, Lehigh University, Bethlehem, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Lehigh University","address":{"name":"Department of Biological Sciences, Lehigh University, Bethlehem, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Lehigh University","address":{"name":"Department of Bioengineering, Lehigh University, Bethlehem, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"wonpil@lehigh.edu","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>
<link href="https://www.nature.com/articles/s41467-024-49700-4" rel="canonical"/>
<meta content="41467" name="journal_id"/>
<meta content="CHARMM-GUI Multicomponent Assembler for modeling and simulation of complex multicomponent systems" name="dc.title"/>
<meta content="Nature Communications 2024 15:1" name="dc.source"/>
<meta content="text/html" name="dc.format"/>
<meta content="Nature Publishing Group" name="dc.publisher"/>
<meta content="2024-06-27" name="dc.date"/>
<meta content="OriginalPaper" name="dc.type"/>
<meta content="En" name="dc.language"/>
<meta content="2024 The Author(s)" name="dc.copyright"/>
<meta content="2024 The Author(s)" name="dc.rights"/>
<meta content="journalpermissions@springernature.com" name="dc.rightsAgent"/>
<meta content="Atomic-scale molecular modeling and simulation are powerful tools for computational biology. However, constructing models with large, densely packed molecules, non-water solvents, or with combinations of multiple biomembranes, polymers, and nanomaterials remains challenging and requires significant time and expertise. Furthermore, existing tools do not support such assemblies under the periodic boundary conditions (PBC) necessary for molecular simulation. Here, we describe Multicomponent Assembler in CHARMM-GUI that automates complex molecular assembly and simulation input preparation under the PBC. In this work, we demonstrate its versatility by preparing 6 challenging systems with varying density of large components: (1) solvated proteins, (2) solvated proteins with a pre-equilibrated membrane, (3) solvated proteins with a sheet-like nanomaterial, (4) solvated proteins with a sheet-like polymer, (5) a mixed membrane-nanomaterial system, and (6) a sheet-like polymer with gaseous solvent. Multicomponent Assembler is expected to be a unique cyberinfrastructure to study complex interactions between small molecules, biomacromolecules, polymers, and nanomaterials. Building atomistic models with molecules of varying size and density requires significant time and expertize. Here, authors describe CHARMM-GUI Multicomponent Assembler that guides complex molecular assembly and simulation input preparation under periodic boundary conditions." name="dc.description"/>
<meta content="2041-1723" name="prism.issn"/>
<meta content="Nature Communications" name="prism.publicationName"/>
<meta content="2024-06-27" name="prism.publicationDate"/>
<meta content="15" name="prism.volume"/>
<meta content="1" name="prism.number"/>
<meta content="OriginalPaper" name="prism.section"/>
<meta content="1" name="prism.startingPage"/>
<meta content="14" name="prism.endingPage"/>
<meta content="2024 The Author(s)" name="prism.copyright"/>
<meta content="journalpermissions@springernature.com" name="prism.rightsAgent"/>
<meta content="https://www.nature.com/articles/s41467-024-49700-4" name="prism.url"/>
<meta content="doi:10.1038/s41467-024-49700-4" name="prism.doi"/>
<meta content="https://www.nature.com/articles/s41467-024-49700-4.pdf" name="citation_pdf_url"/>
<meta content="https://www.nature.com/articles/s41467-024-49700-4" name="citation_fulltext_html_url"/>
<meta content="Nature Communications" name="citation_journal_title"/>
<meta content="Nat Commun" name="citation_journal_abbrev"/>
<meta content="Nature Publishing Group" name="citation_publisher"/>
<meta content="2041-1723" name="citation_issn"/>
<meta content="CHARMM-GUI Multicomponent Assembler for modeling and simulation of complex multicomponent systems" name="citation_title"/>
<meta content="15" name="citation_volume"/>
<meta content="1" name="citation_issue"/>
<meta content="2024/06/27" name="citation_online_date"/>
<meta content="1" name="citation_firstpage"/>
<meta content="14" name="citation_lastpage"/>
<meta content="Article" name="citation_article_type"/>
<meta content="" name="citation_fulltext_world_readable"/>
<meta content="en" name="citation_language"/>
<meta content="doi:10.1038/s41467-024-49700-4" name="dc.identifier"/>
<meta content="10.1038/s41467-024-49700-4" name="DOI"/>
<meta content="221474" name="size"/>
<meta content="10.1038/s41467-024-49700-4" name="citation_doi"/>
<meta content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41467-024-49700-4&amp;api_key=" name="citation_springer_api_url"/>
<meta content="Atomic-scale molecular modeling and simulation are powerful tools for computational biology. However, constructing models with large, densely packed molecules, non-water solvents, or with combinations of multiple biomembranes, polymers, and nanomaterials remains challenging and requires significant time and expertise. Furthermore, existing tools do not support such assemblies under the periodic boundary conditions (PBC) necessary for molecular simulation. Here, we describe Multicomponent Assembler in CHARMM-GUI that automates complex molecular assembly and simulation input preparation under the PBC. In this work, we demonstrate its versatility by preparing 6 challenging systems with varying density of large components: (1) solvated proteins, (2) solvated proteins with a pre-equilibrated membrane, (3) solvated proteins with a sheet-like nanomaterial, (4) solvated proteins with a sheet-like polymer, (5) a mixed membrane-nanomaterial system, and (6) a sheet-like polymer with gaseous solvent. Multicomponent Assembler is expected to be a unique cyberinfrastructure to study complex interactions between small molecules, biomacromolecules, polymers, and nanomaterials. Building atomistic models with molecules of varying size and density requires significant time and expertize. Here, authors describe CHARMM-GUI Multicomponent Assembler that guides complex molecular assembly and simulation input preparation under periodic boundary conditions." name="description"/>
<meta content="Kern, Nathan R." name="dc.creator"/>
<meta content="Lee, Jumin" name="dc.creator"/>
<meta content="Choi, Yeol Kyo" name="dc.creator"/>
<meta content="Im, Wonpil" name="dc.creator"/>
<meta content="Biochemistry" name="dc.subject"/>
<meta content="Computational biophysics" name="dc.subject"/>
<meta content="Computational platforms and environments" name="dc.subject"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Heterogeneity in homogeneous nucleation from billion-atom molecular dynamics simulation of solidification of pure metal; citation_author=Y Shibuta; citation_volume=8; citation_publication_date=2017; citation_doi=10.1038/s41467-017-00017-5; citation_id=CR1" name="citation_reference"/>
<meta content="citation_journal_title=AIP Conf. Proc.; citation_title=Large-scale molecular dynamics studies of sliding friction in nanocrystalline aluminum; citation_author=JE Hammerberg, RJ Ravelo, TC Germann; citation_volume=1979; citation_publication_date=2018; citation_pages=050009; citation_doi=10.1063/1.5044792; citation_id=CR2" name="citation_reference"/>
<meta content="citation_journal_title=Acta Mater.; citation_title=Developing realistic grain boundary networks for use in molecular dynamics simulations; citation_author=AG Frøseth, H Swygenhoven, PM Derlet; citation_volume=53; citation_publication_date=2005; citation_pages=4847-4856; citation_doi=10.1016/j.actamat.2005.06.032; citation_id=CR3" name="citation_reference"/>
<meta content="citation_journal_title=Model. Simul. Mater. Sci. Eng.; citation_title=Micrometer-scale molecular dynamics simulation of microstructure formation linked with multi-phase-field simulation in same space scale; citation_author=Y Shibuta, S Sakane, E Miyoshi, T Takaki, M Ohno; citation_volume=27; citation_publication_date=2019; citation_pages=054002; citation_doi=10.1088/1361-651X/ab1d28; citation_id=CR4" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Atoms to phenotypes: molecular design principles of cellular energy metabolism; citation_author=A Singharoy; citation_volume=179; citation_publication_date=2019; citation_pages=1098-1111; citation_doi=10.1016/j.cell.2019.10.021; citation_id=CR5" name="citation_reference"/>
<meta content="citation_journal_title=J. Mol. Graph. Model.; citation_title=Complete atomistic model of a bacterial cytoplasm for integrating physics, biochemistry, and systems biology; citation_author=M Feig; citation_volume=58; citation_publication_date=2015; citation_pages=1-9; citation_doi=10.1016/j.jmgm.2015.02.004; citation_id=CR6" name="citation_reference"/>
<meta content="citation_journal_title=eLife; citation_title=Biomolecular interactions modulate macromolecular structure and dynamics in atomistic model of a bacterial cytoplasm; citation_author=I Yu; citation_volume=5; citation_publication_date=2016; citation_pages=e19274; citation_doi=10.7554/eLife.19274; citation_id=CR7" name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Chem.; citation_title=Scaling molecular dynamics beyond 100,000 processor cores for large-scale biophysical simulations; citation_author=J Jung; citation_volume=40; citation_publication_date=2019; citation_pages=1919-1930; citation_doi=10.1002/jcc.25840; citation_id=CR8" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Composition of isolated synaptic boutons reveals the amounts of vesicle trafficking proteins; citation_author=BG Wilhelm; citation_volume=344; citation_publication_date=2014; citation_pages=1023-1028; citation_doi=10.1126/science.1252884; citation_id=CR9" name="citation_reference"/>
<meta content="citation_journal_title=Faraday Discuss.; citation_title=3D molecular models of whole HIV-1 virions generated with cellPACK; citation_author=GT Johnson; citation_volume=169; citation_publication_date=2014; citation_pages=23-44; citation_doi=10.1039/C4FD00017J; citation_id=CR10" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=cellPACK: a virtual mesoscope to model and visualize structural systems biology; citation_author=GT Johnson; citation_volume=12; citation_publication_date=2015; citation_pages=85-91; citation_doi=10.1038/nmeth.3204; citation_id=CR11" name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Chem.; citation_title=FFParam: standalone package for CHARMM additive and drude polarizable force field parametrization of small molecules; citation_author=A Kumar, O Yoluk, AD MacKerell; citation_volume=41; citation_publication_date=2020; citation_pages=958-970; citation_doi=10.1002/jcc.26138; citation_id=CR12" name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Chem.; citation_title=Rapid parameterization of small molecules using the force field toolkit; citation_author=CG Mayne, J Saam, K Schulten, E Tajkhorshid, JC Gumbart; citation_volume=34; citation_publication_date=2013; citation_pages=2757-2770; citation_doi=10.1002/jcc.23422; citation_id=CR13" name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Chem.; citation_title=SwissParam: a fast force field generation tool for small organic molecules; citation_author=V Zoete, MA Cuendet, A Grosdidier, O Michielin; citation_volume=32; citation_publication_date=2011; citation_pages=2359-2368; citation_doi=10.1002/jcc.21816; citation_id=CR14" name="citation_reference"/>
<meta content="Case, D. A. et al. AMBERTools. J. Chem. Inf. Model. 63, 6183–6191 (2023)." name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Inf. Model.; citation_title=Automation of the CHARMM general force field (CGenFF) I: bond perception and atom typing; citation_author=K Vanommeslaeghe, AD MacKerell; citation_volume=52; citation_publication_date=2012; citation_pages=3144-3154; citation_doi=10.1021/ci300363c; citation_id=CR16" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Inf. Model.; citation_title=Automation of the CHARMM general force field (CGenFF) II: assignment of bonded parameters and partial atomic charges; citation_author=K Vanommeslaeghe, EP Raman, AD MacKerell; citation_volume=52; citation_publication_date=2012; citation_pages=3155-3168; citation_doi=10.1021/ci3003649; citation_id=CR17" name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Chem.; citation_title=CHARMM general force field: a force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields; citation_author=K Vanommeslaeghe; citation_volume=31; citation_publication_date=2010; citation_pages=671-690; citation_doi=10.1002/jcc.21367; citation_id=CR18" name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Chem.; citation_title=MATCH: an atom-typing toolset for molecular mechanics force fields; citation_author=JD Yesselman, DJ Price, JL Knight, CL Brooks III; citation_volume=33; citation_publication_date=2012; citation_pages=189-202; citation_doi=10.1002/jcc.21963; citation_id=CR19" name="citation_reference"/>
<meta content="Wagner, J. et al. openforcefield/openff-toolkit: 0.14.0 API deprecating and bugfix release. Zenodo 
                  https://doi.org/10.5281/zenodo.8102071
                  
                 (2023)." name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Chem.; citation_title=CHARMM-GUI ligand reader and modeler for CHARMM force field generation of small molecules; citation_author=S Kim; citation_volume=38; citation_publication_date=2017; citation_pages=1879-1886; citation_doi=10.1002/jcc.24829; citation_id=CR21" name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Chem.; citation_title=PACKMOL: a package for building initial configurations for molecular dynamics simulations; citation_author=L Martínez, R Andrade, EG Birgin, JM Martínez; citation_volume=30; citation_publication_date=2009; citation_pages=2157-2164; citation_doi=10.1002/jcc.21224; citation_id=CR22" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Inf. Model.; citation_title=PACKMOL-Memgen: a simple-to-use, generalized workflow for membrane-protein–lipid-bilayer system building; citation_author=S Schott-Verdugo, H Gohlke; citation_volume=59; citation_publication_date=2019; citation_pages=2522-2528; citation_doi=10.1021/acs.jcim.9b00269; citation_id=CR23" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Inf. Model.; citation_title=Wrapping up viruses at multiscale resolution: optimizing PACKMOL and SIRAH execution for simulating the zika virus; citation_author=M Soñora, L Martínez, S Pantano, MR Machado; citation_volume=61; citation_publication_date=2021; citation_pages=408-422; citation_doi=10.1021/acs.jcim.0c01205; citation_id=CR24" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Backmapping triangulated surfaces to coarse-grained membrane models; citation_author=W Pezeshkian, M König, TA Wassenaar, SJ Marrink; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-16094-y; citation_id=CR25" name="citation_reference"/>
<meta content="citation_journal_title=PLOS Comput. Biol.; citation_title=LipidWrapper: an algorithm for generating large-scale membrane models of arbitrary geometry; citation_author=JD Durrant, RE Amaro; citation_volume=10; citation_publication_date=2014; citation_pages=e1003720; citation_doi=10.1371/journal.pcbi.1003720; citation_id=CR26" name="citation_reference"/>
<meta content="Oliveira Bortot, L., Bashardanesh, Z. &amp; van der Spoel, D. Making soup: preparing and validating models of the bacterial cytoplasm for molecular simulation. J. Chem. Inf. Model. 
                  https://doi.org/10.1021/acs.jcim.9b00971
                  
                 (2019)." name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Polyply; a python suite for facilitating simulations of macromolecules and nanomaterials; citation_author=F Grünewald; citation_volume=13; citation_publication_date=2022; citation_doi=10.1038/s41467-021-27627-4; citation_id=CR28" name="citation_reference"/>
<meta content="citation_journal_title=SoftwareX; citation_title=pysimm: a python package for simulation of molecular systems; citation_author=ME Fortunato, CM Colina; citation_volume=6; citation_publication_date=2017; citation_pages=7-12; citation_doi=10.1016/j.softx.2016.12.002; citation_id=CR29" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Theory Comput.; citation_title=CHARMM-GUI Polymer builder for modeling and simulation of synthetic polymers; citation_author=YK Choi; citation_volume=17; citation_publication_date=2021; citation_pages=2431-2443; citation_doi=10.1021/acs.jctc.1c00169; citation_id=CR30" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Theory Comput.; citation_title=CHARMM-GUI Input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field; citation_author=J Lee; citation_volume=12; citation_publication_date=2016; citation_pages=405-413; citation_doi=10.1021/acs.jctc.5b00935; citation_id=CR31" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Phys.; citation_title=CHARMM-GUI supports the Amber force fields; citation_author=J Lee; citation_volume=153; citation_publication_date=2020; citation_pages=035103; citation_doi=10.1063/5.0012280; citation_id=CR32" name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Aided Mol. Des.; citation_title=Lessons learned from comparing molecular dynamics engines on the SAMPL5 dataset; citation_author=MR Shirts; citation_volume=31; citation_publication_date=2017; citation_pages=147-161; citation_doi=10.1007/s10822-016-9977-1; citation_id=CR33" name="citation_reference"/>
<meta content="Páll, S., Abraham, M. J., Kutzner, C., Hess, B. &amp; Lindahl, E. Tackling exascale software challenges in molecular dynamics simulations with GROMACS. in Solving Software Challenges for Exascale (eds. Markidis, S. &amp; Laure, E.) 3–27 
                  https://doi.org/10.1007/978-3-319-15976-8_1
                  
                 (Springer International Publishing, Cham, 2015)." name="citation_reference"/>
<meta content="citation_journal_title=SoftwareX; citation_title=GROMACS: high performance molecular simulations through multi-level parallelism from laptops to supercomputers; citation_author=MJ Abraham; citation_volume=1–2; citation_publication_date=2015; citation_pages=19-25; citation_doi=10.1016/j.softx.2015.06.001; citation_id=CR35" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Phys.; citation_title=Scalable molecular dynamics on CPU and GPU architectures with NAMD; citation_author=JC Phillips; citation_volume=153; citation_publication_date=2020; citation_pages=044130; citation_doi=10.1063/5.0014475; citation_id=CR36" name="citation_reference"/>
<meta content="citation_journal_title=J. Mol. Graph.; citation_title=VMD: visual molecular dynamics; citation_author=W Humphrey, A Dalke, K Schulten; citation_volume=14; citation_publication_date=1996; citation_pages=33-38; citation_doi=10.1016/0263-7855(96)00018-5; citation_id=CR37" name="citation_reference"/>
<meta content="citation_journal_title=PLOS Comput. Biol.; citation_title=OpenMM 7: rapid development of high performance algorithms for molecular dynamics; citation_author=P Eastman; citation_volume=13; citation_publication_date=2017; citation_pages=e1005659; citation_doi=10.1371/journal.pcbi.1005659; citation_id=CR38" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Inf. Model.; citation_title=CELLmicrocosmos 2.2 membraneEditor: a modular interactive shape-based software approach to solve heterogeneous membrane packing problems; citation_author=B Sommer; citation_volume=51; citation_publication_date=2011; citation_pages=1165-1182; citation_doi=10.1021/ci1003619; citation_id=CR39" name="citation_reference"/>
<meta content="citation_journal_title=Biophys. J.; citation_title=CHARMM-GUI membrane builder for mixed bilayers and its application to yeast membranes; citation_author=S Jo, JB Lim, JB Klauda, W Im; citation_volume=97; citation_publication_date=2009; citation_pages=50-58; citation_doi=10.1016/j.bpj.2009.04.013; citation_id=CR40" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Theory Comput.; citation_title=CHARMM-GUI membrane builder for complex biological membrane simulations with Glycolipids and Lipoglycans; citation_author=J Lee; citation_volume=15; citation_publication_date=2019; citation_pages=775-786; citation_doi=10.1021/acs.jctc.8b01066; citation_id=CR41" name="citation_reference"/>
<meta content="citation_journal_title=Bioinformatics; citation_title=MemGen: a general web server for the setup of lipid membrane simulation systems; citation_author=CJ Knight, JS Hub; citation_volume=31; citation_publication_date=2015; citation_pages=2897-2899; citation_doi=10.1093/bioinformatics/btv292; citation_id=CR42" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Theory Comput.; citation_title=Computational lipidomics with insane: a versatile tool for generating custom membranes for molecular simulations; citation_author=TA Wassenaar, HI Ingólfsson, RA Böckmann, DP Tieleman, SJ Marrink; citation_volume=11; citation_publication_date=2015; citation_pages=2144-2155; citation_doi=10.1021/acs.jctc.5b00209; citation_id=CR43" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Theory Comput.; citation_title=High-throughput automated preparation and simulation of membrane proteins with HTMD; citation_author=S Doerr, T Giorgino, G Martínez-Rosell, JM Damas, G Fabritiis; citation_volume=13; citation_publication_date=2017; citation_pages=4003-4011; citation_doi=10.1021/acs.jctc.7b00480; citation_id=CR44" name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Chem.; citation_title=CHARMM-GUI: a web-based graphical user interface for CHARMM; citation_author=S Jo, T Kim, VG Iyer, W Im; citation_volume=29; citation_publication_date=2008; citation_pages=1859-1865; citation_doi=10.1002/jcc.20945; citation_id=CR45" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Theory Comput.; citation_title=CHARMM-GUI Nanomaterial modeler for modeling and simulation of nanomaterial systems; citation_author=YK Choi; citation_volume=18; citation_publication_date=2022; citation_pages=479-493; citation_doi=10.1021/acs.jctc.1c00996; citation_id=CR46" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=CHARMM36m: an improved force field for folded and intrinsically disordered proteins; citation_author=J Huang; citation_volume=14; citation_publication_date=2017; citation_pages=71-73; citation_doi=10.1038/nmeth.4067; citation_id=CR47" name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Chem.; citation_title=Extension of the CHARMM general force field to sulfonyl-containing compounds and its utility in biomolecular simulations; citation_author=W Yu, X He, K Vanommeslaeghe, AD MacKerell; citation_volume=33; citation_publication_date=2012; citation_pages=2451-2468; citation_doi=10.1002/jcc.23067; citation_id=CR48" name="citation_reference"/>
<meta content="citation_journal_title=Bioorg. Med. Chem.; citation_title=Parametrization of halogen bonds in the CHARMM general force field: Improved treatment of ligand–protein interactions; citation_author=I Soteras Gutiérrez; citation_volume=24; citation_publication_date=2016; citation_pages=4812-4825; citation_doi=10.1016/j.bmc.2016.06.034; citation_id=CR49" name="citation_reference"/>
<meta content="citation_journal_title=J. Phys. Chem. C.; citation_title=Accurate force field parameters and pH resolved surface models for hydroxyapatite to understand structure, mechanics, hydration, and biological interfaces; citation_author=T-J Lin, H Heinz; citation_volume=120; citation_publication_date=2016; citation_pages=4975-4992; citation_doi=10.1021/acs.jpcc.5b12504; citation_id=CR50" name="citation_reference"/>
<meta content="citation_journal_title=J. Phys. Chem. C.; citation_title=Accurate simulation of surfaces and interfaces of face-centered cubic metals using 12−6 and 9−6 Lennard-Jones potentials; citation_author=H Heinz, RA Vaia, BL Farmer, RR Naik; citation_volume=112; citation_publication_date=2008; citation_pages=17281-17290; citation_doi=10.1021/jp801931d; citation_id=CR51" name="citation_reference"/>
<meta content="citation_journal_title=Chem. Mater.; citation_title=Force field and a surface model database for Silica to simulate interfacial properties in atomic resolution; citation_author=FS Emami; citation_volume=26; citation_publication_date=2014; citation_pages=2647-2658; citation_doi=10.1021/cm500365c; citation_id=CR52" name="citation_reference"/>
<meta content="citation_journal_title=Cem. Concr. Res.; citation_title=Force field for calcium sulfate minerals to predict structural, hydration, and interfacial properties; citation_author=RK Mishra, K Kanhaiya, JJ Winetrout, RJ Flatt, H Heinz; citation_volume=139; citation_publication_date=2021; citation_pages=106262; citation_doi=10.1016/j.cemconres.2020.106262; citation_id=CR53" name="citation_reference"/>
<meta content="citation_journal_title=Chem. Sci.; citation_title=Interpretable molecular models for molybdenum disulfide and insight into selective peptide recognition; citation_author=J Liu; citation_volume=11; citation_publication_date=2020; citation_pages=8708-8722; citation_doi=10.1039/D0SC01443E; citation_id=CR54" name="citation_reference"/>
<meta content="citation_journal_title=Langmuir; citation_title=Thermodynamically consistent force fields for the assembly of inorganic, organic, and biological nanostructures: the INTERFACE force field; citation_author=H Heinz, T-J Lin, R Kishore Mishra, FS Emami; citation_volume=29; citation_publication_date=2013; citation_pages=1754-1765; citation_doi=10.1021/la3038846; citation_id=CR55" name="citation_reference"/>
<meta content="Park, S.-J., Kern, N., Brown, T., Lee, J. &amp; Im, W. CHARMM-GUI PDB Manipulator: various PDB structural modifications for biomolecular Modeling and Simulation. J. Mol. Biol. 435, 167995 (2023)." name="citation_reference"/>
<meta content="citation_journal_title=Bioinformatics; citation_title=Glycan reader is improved to recognize most sugar types and chemical modifications in the protein data bank; citation_author=S-J Park; citation_volume=33; citation_publication_date=2017; citation_pages=3051-3057; citation_doi=10.1093/bioinformatics/btx358; citation_id=CR57" name="citation_reference"/>
<meta content="citation_journal_title=PLOS ONE; citation_title=Automated builder and database of protein/membrane complexes for molecular dynamics simulations; citation_author=S Jo, T Kim, W Im; citation_volume=2; citation_publication_date=2007; citation_pages=e880; citation_doi=10.1371/journal.pone.0000880; citation_id=CR58" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl Acad. Sci.; citation_title=Clustering and dynamics of crowded proteins near membranes and their influence on membrane bending; citation_author=G Nawrocki, W Im, Y Sugita, M Feig; citation_volume=116; citation_publication_date=2019; citation_pages=24562-24567; citation_doi=10.1073/pnas.1910771116; citation_id=CR59" name="citation_reference"/>
<meta content="citation_journal_title=Biochim. Biophys. Acta BBA - Biomembr.; citation_title=Lipid diffusion in planar membranes investigated by fluorescence correlation spectroscopy; citation_author=R Macháň, M Hof; citation_volume=1798; citation_publication_date=2010; citation_pages=1377-1391; citation_doi=10.1016/j.bbamem.2010.02.014; citation_id=CR60" name="citation_reference"/>
<meta content="citation_journal_title=J. Open Source Softw.; citation_title=Computing diffusion coefficients in macromolecular simulations: the diffusion coefficient tool for VMD; citation_author=T Giorgino; citation_volume=4; citation_publication_date=2019; citation_pages=1698; citation_doi=10.21105/joss.01698; citation_id=CR61" name="citation_reference"/>
<meta content="citation_journal_title=Langmuir; citation_title=Measuring diffusion of lipid-like probes in artificial and natural membranes by raster image correlation spectroscopy (RICS): use of a commercial laser-scanning microscope with analog detection; citation_author=E Gielen; citation_volume=25; citation_publication_date=2009; citation_pages=5209-5218; citation_doi=10.1021/la8040538; citation_id=CR62" name="citation_reference"/>
<meta content="citation_journal_title=Biophys. J.; citation_title=Structure of an adsorbed dimyristoylphosphatidylcholine bilayer measured with specular reflection of neutrons; citation_author=SJ Johnson; citation_volume=59; citation_publication_date=1991; citation_pages=289-294; citation_doi=10.1016/S0006-3495(91)82222-6; citation_id=CR63" name="citation_reference"/>
<meta content="citation_journal_title=Int. J. Life Cycle Assess.; citation_title=Life cycle energy and GHG emissions of PET recycling: change-oriented effects; citation_author=L Shen, E Nieuwlaar, E Worrell, MK Patel; citation_volume=16; citation_publication_date=2011; citation_pages=522-536; citation_doi=10.1007/s11367-011-0296-4; citation_id=CR64" name="citation_reference"/>
<meta content="citation_journal_title=Energy Environ. Sci.; citation_title=Replacing fossil based PET with biobased PEF; process analysis, energy and GHG balance; citation_author=EAJJ Eerhart, APC Faaij, MK Patel; citation_volume=5; citation_publication_date=2012; citation_pages=6407-6422; citation_doi=10.1039/c2ee02480b; citation_id=CR65" name="citation_reference"/>
<meta content="citation_journal_title=Macromolecules; citation_title=Carbon dioxide sorption and transport in amorphous poly(ethylene furanoate); citation_author=SK Burgess, RM Kriegel, WJ Koros; citation_volume=48; citation_publication_date=2015; citation_pages=2184-2193; citation_doi=10.1021/acs.macromol.5b00333; citation_id=CR66" name="citation_reference"/>
<meta content="citation_journal_title=Front. Chem.; citation_title=Molecular dynamics simulation of an entire cell; citation_author=JA Stevens; citation_volume=11; citation_publication_date=2023; citation_pages=1106495; citation_doi=10.3389/fchem.2023.1106495; citation_id=CR67" name="citation_reference"/>
<meta content="citation_journal_title=J. Mol. Biol.; citation_title=Moltemplate: a tool for coarse-grained modeling of complex biological matter and soft condensed matter physics; citation_author=AI Jewett; citation_volume=433; citation_publication_date=2021; citation_pages=166841; citation_doi=10.1016/j.jmb.2021.166841; citation_id=CR68" name="citation_reference"/>
<meta content="Buckner, J. et al. pyCHARMM: embedding CHARMM functionality in a python framework. J. Chem. Theory. Comput. 19, 3752–3762 (2023)." name="citation_reference"/>
<meta content="Jewett, A. I. et al. Moltemplate: A tool for coarse-grained modeling of complex biological matter and soft condensed matter physics. J. Mol. Biol. 433, 166841 (2021)." name="citation_reference"/>
<meta content="Ribeiro, J. et al. VMD psfgen Plugin, Version 2.0. 
                  https://www.ks.uiuc.edu/Research/vmd/plugins/psfgen/
                  
                 (2020)." name="citation_reference"/>
<meta content="citation_journal_title=J. Phys. Chem. B; citation_title=Slow-down in diffusion in crowded protein solutions correlates with transient cluster Formation; citation_author=G Nawrocki, P Wang, I Yu, Y Sugita, M Feig; citation_volume=121; citation_publication_date=2017; citation_pages=11072-11084; citation_doi=10.1021/acs.jpcb.7b08785; citation_id=CR72" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Inf. Model.; citation_title=CHARMM-GUI Supports hydrogen mass repartitioning and different protonation states of phosphates in lipopolysaccharides; citation_author=Y Gao; citation_volume=61; citation_publication_date=2021; citation_pages=831-839; citation_doi=10.1021/acs.jcim.0c01360; citation_id=CR73" name="citation_reference"/>
<meta content="Kern, N. R., Lee, J., Choi, Y. K. &amp; Im, W. CHARMM-GUI multicomponent assembler for modeling and simulation of complex multicomponent systems. MCA Scripts 
                  https://doi.org/10.5281/zenodo.11205908
                  
                 (2024)." name="citation_reference"/>
<meta content="Kern, Nathan R." name="citation_author"/>
<meta content="Department of Computer Science &amp; Engineering, Lehigh University, Bethlehem, USA" name="citation_author_institution"/>
<meta content="Lee, Jumin" name="citation_author"/>
<meta content="Department of Biological Sciences, Lehigh University, Bethlehem, USA" name="citation_author_institution"/>
<meta content="Choi, Yeol Kyo" name="citation_author"/>
<meta content="Department of Biological Sciences, Lehigh University, Bethlehem, USA" name="citation_author_institution"/>
<meta content="Im, Wonpil" name="citation_author"/>
<meta content="Department of Computer Science &amp; Engineering, Lehigh University, Bethlehem, USA" name="citation_author_institution"/>
<meta content="Department of Biological Sciences, Lehigh University, Bethlehem, USA" name="citation_author_institution"/>
<meta content="Department of Bioengineering, Lehigh University, Bethlehem, USA" name="citation_author_institution"/>
<meta content="https://www.nature.com/platform/readcube-access" name="access_endpoint"/>
<meta content="@NatureComms" name="twitter:site"/>
<meta content="summary_large_image" name="twitter:card"/>
<meta content="Content cover image" name="twitter:image:alt"/>
<meta content="CHARMM-GUI Multicomponent Assembler for modeling and simulation of complex multicomponent systems" name="twitter:title"/>
<meta content="Nature Communications - Building atomistic models with molecules of varying size and density requires significant time and expertize. Here, authors describe CHARMM-GUI Multicomponent Assembler that..." name="twitter:description"/>
<meta content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig1_HTML.png" name="twitter:image"/>
<meta content="https://www.nature.com/articles/s41467-024-49700-4" property="og:url"/>
<meta content="article" property="og:type"/>
<meta content="Nature" property="og:site_name"/>
<meta content="CHARMM-GUI Multicomponent Assembler for modeling and simulation of complex multicomponent systems - Nature Communications" property="og:title"/>
<meta content="Building atomistic models with molecules of varying size and density requires significant time and expertize. Here, authors describe CHARMM-GUI Multicomponent Assembler that guides complex molecular assembly and simulation input preparation under periodic boundary conditions." property="og:description"/>
<meta content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig1_HTML.png" property="og:image"/>
<script>
        window.eligibleForRa21 = 'false'; 
    </script>
</head>
<body class="article-page">
<noscript><iframe height="0" src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ" style="display:none;visibility:hidden" width="0"></iframe></noscript>
<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
<a class="c-skip-link" href="#content">Skip to main content</a>
<div class="c-grade-c-banner u-hide">
<div class="c-grade-c-banner__container">
<p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>
</div>
</div>
<div class="u-hide u-show-following-ad"></div>
<aside class="c-ad c-ad--728x90">
<div class="c-ad__inner" data-container-type="banner-advert">
<p class="c-ad__label">Advertisement</p>
<div class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-gpt="" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41467-024-49700-4;doi=10.1038/s41467-024-49700-4;techmeta=118,119,139,9;subjmeta=114,2266,2398,45,57,631;kwrd=Biochemistry,Computational+biophysics,Computational+platforms+and+environments" data-gpt-unitpath="/285/nature_communications/article" data-pa11y-ignore="" data-test="top-ad" id="div-gpt-ad-top-1">
<noscript>
<a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature_communications/article&amp;sz=728x90&amp;c=1752994186&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41467-024-49700-4%26doi%3D10.1038/s41467-024-49700-4%26techmeta%3D118,119,139,9%26subjmeta%3D114,2266,2398,45,57,631%26kwrd%3DBiochemistry,Computational+biophysics,Computational+platforms+and+environments">
<img alt="Advertisement" data-test="gpt-advert-fallback-img" height="90" src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature_communications/article&amp;sz=728x90&amp;c=1752994186&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41467-024-49700-4%26doi%3D10.1038/s41467-024-49700-4%26techmeta%3D118,119,139,9%26subjmeta%3D114,2266,2398,45,57,631%26kwrd%3DBiochemistry,Computational+biophysics,Computational+platforms+and+environments" width="728"/></a>
</noscript>
</div>
</div>
</aside>
<header class="c-header" data-header="" data-track-component="nature-150-split-header" id="header" style="border-color:#e63323">
<div class="c-header__row">
<div class="c-header__container">
<div class="c-header__split">
<div class="c-header__logo-container">
<a data-track="click" data-track-action="home" data-track-label="image" href="/ncomms">
<picture class="c-header__logo">
<source media="(min-width: 875px)" srcset="https://media.springernature.com/full/nature-cms/uploads/product/ncomms/header-03d2e325c0a02f6df509e5730e9be304.svg">
<img alt="Nature Communications" height="32" src="https://media.springernature.com/full/nature-cms/uploads/product/ncomms/header-7001f06bc3fe2437048388e9f2f44215.svg"/>
</source></picture>
</a>
</div>
<ul class="c-header__menu c-header__menu--global">
<li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
<a class="c-header__link" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link" href="https://www.nature.com/siteindex">
<span>View all journals</span>
</a>
</li>
<li class="c-header__item c-header__item--padding c-header__item--pipe">
<a class="c-header__link c-header__link--search" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" href="#search-menu">
<svg aria-hidden="true" focusable="false" height="22" role="img" viewbox="0 0 18 18" width="22" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
</a>
</li>
<li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
<a class="c-header__link eds-c-header__link" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41467-024-49700-4" id="identity-account-widget"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
</li>
</ul>
</div>
</div>
</div>
<div class="c-header__row">
<div class="c-header__container" data-test="navigation-row">
<div class="c-header__split">
<ul class="c-header__menu c-header__menu--journal">
<li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
<a class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" href="#explore">
<span><span class="c-header__show-text">Explore</span> content</span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--dropdown-menu">
<a class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" href="#about-the-journal">
<span>About <span class="c-header__show-text">the journal</span></span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
<a class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" href="#publish-with-us">
<span>Publish <span class="c-header__show-text">with us</span></span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
</ul>
<ul class="c-header__menu c-header__menu--hide-lg-max">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (desktop site header)" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D264%26journal-link%3Dhttps%253A%252F%252Fwww.nature.com%252Fncomms%252F" rel="nofollow">
<span>Sign up for alerts</span><svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--pipe">
<a class="c-header__link" data-track="click" data-track-action="rss feed" data-track-label="link" href="https://www.nature.com/ncomms.rss">
<span>RSS feed</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</header>
<nav aria-label="breadcrumbs" class="u-mb-16">
<div class="u-container">
<ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
<li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature" href="/" itemprop="item"><span itemprop="name">nature</span></a><meta content="1" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature communications" href="/ncomms" itemprop="item"><span itemprop="name">nature communications</span></a><meta content="2" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles" href="/ncomms/articles?type=article" itemprop="item"><span itemprop="name">articles</span></a><meta content="3" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
<span itemprop="name">article</span><meta content="4" itemprop="position"/></li>
</ol>
</div>
</nav>
</div>
<div class="u-container u-mt-32 u-mb-32 u-clearfix" data-component="article-container" data-container-type="article" id="content">
<main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
<div aria-hidden="true" class="c-context-bar u-hide" data-context-bar="" data-test="context-bar">
<div class="c-context-bar__container u-container" data-track-context="sticky banner">
<div class="c-context-bar__title">
                            CHARMM-GUI Multicomponent Assembler for modeling and simulation of complex multicomponent systems
                        </div>
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41467-024-49700-4.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
</div>
<article lang="en">
<div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
<div class="c-pdf-container" data-track-context="article body">
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41467-024-49700-4.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
</div>
<div class="c-article-header">
<header>
<ul class="c-article-identifiers" data-test="article-identifier">
<li class="c-article-identifiers__item" data-test="article-category">Article</li>
<li class="c-article-identifiers__item">
<a class="u-color-open-access" data-test="open-access" data-track="click" data-track-action="open access" data-track-label="link" href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research">Open access</a>
</li>
<li class="c-article-identifiers__item">Published: <time datetime="2024-06-27">27 June 2024</time></li>
</ul>
<h1 class="c-article-title" data-article-title="" data-test="article-title">CHARMM-GUI Multicomponent Assembler for modeling and simulation of complex multicomponent systems</h1>
<ul class="c-article-author-list c-article-author-list--short" data-component-authors-activator="authors-list" data-test="authors-list"><li class="c-article-author-list__item"><a data-author-popup="auth-Nathan_R_-Kern-Aff1" data-author-search="Kern, Nathan R." data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Nathan_R_-Kern-Aff1">Nathan R. Kern</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-0936-7274"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-0936-7274</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-author-popup="auth-Jumin-Lee-Aff2" data-author-search="Lee, Jumin" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jumin-Lee-Aff2">Jumin Lee</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-1008-0118"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-1008-0118</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Yeol_Kyo-Choi-Aff2" data-author-search="Choi, Yeol Kyo" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Yeol_Kyo-Choi-Aff2">Yeol Kyo Choi</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-4218-7139"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-4218-7139</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup> &amp; </li><li aria-label="Show all 4 authors for this article" class="c-article-author-list__show-more" title="Show all 4 authors for this article">…</li><li class="c-article-author-list__item"><a data-author-popup="auth-Wonpil-Im-Aff1-Aff2-Aff3" data-author-search="Im, Wonpil" data-corresp-id="c1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Wonpil-Im-Aff1-Aff2-Aff3">Wonpil Im<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-mail-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0001-5642-6041"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-5642-6041</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a>,<a href="#Aff3">3</a></sup> </li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-down-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg><span>Show authors</span></button>
<p class="c-article-info-details" data-container-section="info">
<a data-test="journal-link" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link" href="/ncomms"><i data-test="journal-title">Nature Communications</i></a>
<b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 15</b>, Article number: <span data-test="article-number">5459</span> (<span data-test="article-publication-year">2024</span>)
            <a class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link" href="#citeas">Cite this article</a>
</p>
<div class="c-article-metrics-bar__wrapper u-clear-both">
<ul class="c-article-metrics-bar u-list-reset">
<li class="c-article-metrics-bar__item" data-test="access-count">
<p class="c-article-metrics-bar__count">12k <span class="c-article-metrics-bar__label">Accesses</span></p>
</li>
<li class="c-article-metrics-bar__item" data-test="citation-count">
<p class="c-article-metrics-bar__count">9 <span class="c-article-metrics-bar__label">Citations</span></p>
</li>
<li class="c-article-metrics-bar__item" data-test="altmetric-score">
<p class="c-article-metrics-bar__count">34 <span class="c-article-metrics-bar__label">Altmetric</span></p>
</li>
<li class="c-article-metrics-bar__item">
<p class="c-article-metrics-bar__details"><a data-track="click" data-track-action="view metrics" data-track-label="link" href="/articles/s41467-024-49700-4/metrics" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
</li>
</ul>
</div>
</header>
<div class="u-js-hide" data-component="article-subject-links">
<h3 class="c-article__sub-heading">Subjects</h3>
<ul class="c-article-subject-list">
<li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/biochemistry">Biochemistry</a></li><li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/computational-biophysics">Computational biophysics</a></li><li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/computational-platforms-and-environments">Computational platforms and environments</a></li>
</ul>
</div>
</div>
<div class="c-article-body">
<section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Atomic-scale molecular modeling and simulation are powerful tools for computational biology. However, constructing models with large, densely packed molecules, non-water solvents, or with combinations of multiple biomembranes, polymers, and nanomaterials remains challenging and requires significant time and expertise. Furthermore, existing tools do not support such assemblies under the periodic boundary conditions (PBC) necessary for molecular simulation. Here, we describe Multicomponent Assembler in CHARMM-GUI that automates complex molecular assembly and simulation input preparation under the PBC. In this work, we demonstrate its versatility by preparing 6 challenging systems with varying density of large components: (1) solvated proteins, (2) solvated proteins with a pre-equilibrated membrane, (3) solvated proteins with a sheet-like nanomaterial, (4) solvated proteins with a sheet-like polymer, (5) a mixed membrane-nanomaterial system, and (6) a sheet-like polymer with gaseous solvent. Multicomponent Assembler is expected to be a unique cyberinfrastructure to study complex interactions between small molecules, biomacromolecules, polymers, and nanomaterials.</p></div></div></section>
<section aria-labelledby="inline-recommendations" class="c-article-recommendations" data-title="Inline Recommendations" data-track-component="inline-recommendations">
<h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
<div class="c-article-recommendations-list">
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41596-021-00640-z/MediaObjects/41596_2021_640_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_1" data-track-action="click recommendations inline - 1" data-track-context="inline recommendations" data-track-label="10.1038/s41596-021-00640-z" href="https://www.nature.com/articles/s41596-021-00640-z?fromPaywallRec=false" itemprop="url">Integrative structural modeling of macromolecular complexes using Assembline
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__date">29 November 2021</span>
</div>
</div>
</article>
</div>
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-024-47654-1/MediaObjects/41467_2024_47654_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_2" data-track-action="click recommendations inline - 2" data-track-context="inline recommendations" data-track-label="10.1038/s41467-024-47654-1" href="https://www.nature.com/articles/s41467-024-47654-1?fromPaywallRec=false" itemprop="url">GENESIS CGDYN: large-scale coarse-grained MD simulation with dynamic load balancing for heterogeneous biomolecular systems
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__access-type">Open access</span>
<span class="c-article-meta-recommendations__date">20 April 2024</span>
</div>
</div>
</article>
</div>
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41594-025-01490-z/MediaObjects/41594_2025_1490_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_3" data-track-action="click recommendations inline - 3" data-track-context="inline recommendations" data-track-label="10.1038/s41594-025-01490-z" href="https://www.nature.com/articles/s41594-025-01490-z?fromPaywallRec=false" itemprop="url">Local structural flexibility drives oligomorphism in computationally designed protein assemblies
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__access-type">Open access</span>
<span class="c-article-meta-recommendations__date">26 February 2025</span>
</div>
</div>
</article>
</div>
</div>
</section>
<script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1740765561,
                        embedded_user: 'null'
                    }
                });
            </script>
<div class="main-content">
<section data-title="Introduction"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Introduction</h2><div class="c-article-section__content" id="Sec1-content"><p>Molecular dynamics (MD) simulation becomes essential to study diverse molecular phenomena at atomic resolutions. Advances in computational power and algorithms have enabled simulation models with atom counts in the range of 100 million to several billion, including studies of metal nucleation and sliding<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR1" id="ref-link-section-d22328748e454" title="Shibuta, Y. et al. Heterogeneity in homogeneous nucleation from billion-atom molecular dynamics simulation of solidification of pure metal. Nat. Commun. 8, 10 (2017).">1</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR2" id="ref-link-section-d22328748e454_1" title="Hammerberg, J. E., Ravelo, R. J. &amp; Germann, T. C. Large-scale molecular dynamics studies of sliding friction in nanocrystalline aluminum. AIP Conf. Proc. 1979, 050009 (2018).">2</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR3" id="ref-link-section-d22328748e454_2" title="Frøseth, A. G., Van Swygenhoven, H. &amp; Derlet, P. M. Developing realistic grain boundary networks for use in molecular dynamics simulations. Acta Mater. 53, 4847–4856 (2005).">3</a>,<a aria-label="Reference 4" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR4" id="ref-link-section-d22328748e457" title="Shibuta, Y., Sakane, S., Miyoshi, E., Takaki, T. &amp; Ohno, M. Micrometer-scale molecular dynamics simulation of microstructure formation linked with multi-phase-field simulation in same space scale. Model. Simul. Mater. Sci. Eng. 27, 054002 (2019).">4</a></sup>, bacterial cytoplasm<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR5" id="ref-link-section-d22328748e461" title="Singharoy, A. et al. Atoms to phenotypes: molecular design principles of cellular energy metabolism. Cell 179, 1098–1111 (2019).">5</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR6" id="ref-link-section-d22328748e461_1" title="Feig, M. et al. Complete atomistic model of a bacterial cytoplasm for integrating physics, biochemistry, and systems biology. J. Mol. Graph. Model. 58, 1–9 (2015).">6</a>,<a aria-label="Reference 7" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR7" id="ref-link-section-d22328748e464" title="Yu, I. et al. Biomolecular interactions modulate macromolecular structure and dynamics in atomistic model of a bacterial cytoplasm. eLife 5, e19274 (2016).">7</a></sup>, eukaryotic gene loci<sup><a aria-label="Reference 8" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR8" id="ref-link-section-d22328748e468" title="Jung, J. et al. Scaling molecular dynamics beyond 100,000 processor cores for large-scale biophysical simulations. J. Comput. Chem. 40, 1919–1930 (2019).">8</a></sup>, synaptic bouton<sup><a aria-label="Reference 9" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR9" id="ref-link-section-d22328748e472" title="Wilhelm, B. G. et al. Composition of isolated synaptic boutons reveals the amounts of vesicle trafficking proteins. Science 344, 1023–1028 (2014).">9</a></sup>, and viral capsids<sup><a aria-label="Reference 10" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR10" id="ref-link-section-d22328748e476" title="Johnson, G. T. et al. 3D molecular models of whole HIV-1 virions generated with cellPACK. Faraday Discuss. 169, 23–44 (2014).">10</a>,<a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR11" id="ref-link-section-d22328748e479" title="Johnson, G. T. et al. cellPACK: a virtual mesoscope to model and visualize structural systems biology. Nat. Methods 12, 85–91 (2015).">11</a></sup>. Preparing MD simulation systems typically requires determining the model size and composition, solving a challenging packing problem, and having topological information of each molecule and associated force field (FF) together with simulation input parameters. However, each of these steps presents a challenging barrier for researchers attempting to enter the field, including many experimental researchers who seek to supplement their studies with molecular modeling and simulation.</p><p>Many software applications have been developed to facilitate various steps of atomistic model preparation, including FFParam<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR12" id="ref-link-section-d22328748e486" title="Kumar, A., Yoluk, O. &amp; MacKerell, A. D. FFParam: standalone package for CHARMM additive and drude polarizable force field parametrization of small molecules. J. Comput. Chem. 41, 958–970 (2020).">12</a></sup>, FFTK<sup><a aria-label="Reference 13" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR13" id="ref-link-section-d22328748e490" title="Mayne, C. G., Saam, J., Schulten, K., Tajkhorshid, E. &amp; Gumbart, J. C. Rapid parameterization of small molecules using the force field toolkit. J. Comput. Chem. 34, 2757–2770 (2013).">13</a></sup>, SwissParam<sup><a aria-label="Reference 14" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR14" id="ref-link-section-d22328748e494" title="Zoete, V., Cuendet, M. A., Grosdidier, A. &amp; Michielin, O. SwissParam: a fast force field generation tool for small organic molecules. J. Comput. Chem. 32, 2359–2368 (2011).">14</a></sup>, Antechamber<sup><a aria-label="Reference 15" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR15" id="ref-link-section-d22328748e498" title="Case, D. A. et al. AMBERTools. J. Chem. Inf. Model. 63, 6183–6191 (2023).">15</a></sup>, CGenFF<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR16" id="ref-link-section-d22328748e502" title="Vanommeslaeghe, K. &amp; MacKerell, A. D. Jr. Automation of the CHARMM general force field (CGenFF) I: bond perception and atom typing. J. Chem. Inf. Model. 52, 3144–3154 (2012).">16</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR17" id="ref-link-section-d22328748e502_1" title="Vanommeslaeghe, K., Raman, E. P. &amp; MacKerell, A. D. Jr. Automation of the CHARMM general force field (CGenFF) II: assignment of bonded parameters and partial atomic charges. J. Chem. Inf. Model. 52, 3155–3168 (2012).">17</a>,<a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR18" id="ref-link-section-d22328748e505" title="Vanommeslaeghe, K. et al. CHARMM general force field: a force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields. J. Comput. Chem. 31, 671–690 (2010).">18</a></sup>, MATCH<sup><a aria-label="Reference 19" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR19" id="ref-link-section-d22328748e510" title="Yesselman, J. D., Price, D. J., Knight, J. L. &amp; Brooks III, C. L. MATCH: an atom-typing toolset for molecular mechanics force fields. J. Comput. Chem. 33, 189–202 (2012).">19</a></sup>, OpenFF<sup><a aria-label="Reference 20" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR20" id="ref-link-section-d22328748e514" title="Wagner, J. et al. openforcefield/openff-toolkit: 0.14.0 API deprecating and bugfix release. Zenodo 
                  https://doi.org/10.5281/zenodo.8102071
                  
                 (2023).">20</a></sup>, and CHARMM-GUI Ligand Reader &amp; Modeler<sup><a aria-label="Reference 21" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR21" id="ref-link-section-d22328748e518" title="Kim, S. et al. CHARMM-GUI ligand reader and modeler for CHARMM force field generation of small molecules. J. Comput. Chem. 38, 1879–1886 (2017).">21</a></sup> for FF preparation; PACKMOL<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR22" id="ref-link-section-d22328748e522" title="Martínez, L., Andrade, R., Birgin, E. G. &amp; Martínez, J. M. PACKMOL: a package for building initial configurations for molecular dynamics simulations. J. Comput. Chem. 30, 2157–2164 (2009).">22</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR23" id="ref-link-section-d22328748e522_1" title="Schott-Verdugo, S. &amp; Gohlke, H. PACKMOL-Memgen: a simple-to-use, generalized workflow for membrane-protein–lipid-bilayer system building. J. Chem. Inf. Model. 59, 2522–2528 (2019).">23</a>,<a aria-label="Reference 24" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR24" id="ref-link-section-d22328748e525" title="Soñora, M., Martínez, L., Pantano, S. &amp; Machado, M. R. Wrapping up viruses at multiscale resolution: optimizing PACKMOL and SIRAH execution for simulating the zika virus. J. Chem. Inf. Model. 61, 408–422 (2021).">24</a></sup>, cellPACK<sup><a aria-label="Reference 10" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR10" id="ref-link-section-d22328748e529" title="Johnson, G. T. et al. 3D molecular models of whole HIV-1 virions generated with cellPACK. Faraday Discuss. 169, 23–44 (2014).">10</a>,<a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR11" id="ref-link-section-d22328748e532" title="Johnson, G. T. et al. cellPACK: a virtual mesoscope to model and visualize structural systems biology. Nat. Methods 12, 85–91 (2015).">11</a></sup>, TS2CG<sup><a aria-label="Reference 25" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR25" id="ref-link-section-d22328748e536" title="Pezeshkian, W., König, M., Wassenaar, T. A. &amp; Marrink, S. J. Backmapping triangulated surfaces to coarse-grained membrane models. Nat. Commun. 11, 2296 (2020).">25</a></sup>, LipidWrapper<sup><a aria-label="Reference 26" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR26" id="ref-link-section-d22328748e541" title="Durrant, J. D. &amp; Amaro, R. E. LipidWrapper: an algorithm for generating large-scale membrane models of arbitrary geometry. PLOS Comput. Biol. 10, e1003720 (2014).">26</a></sup>, and Soup<sup><a aria-label="Reference 27" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR27" id="ref-link-section-d22328748e545" title="Oliveira Bortot, L., Bashardanesh, Z. &amp; van der Spoel, D. Making soup: preparing and validating models of the bacterial cytoplasm for molecular simulation. J. Chem. Inf. Model. 
                  https://doi.org/10.1021/acs.jcim.9b00971
                  
                 (2019).">27</a></sup> for molecular packing; polyply<sup><a aria-label="Reference 28" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR28" id="ref-link-section-d22328748e549" title="Grünewald, F. et al. Polyply; a python suite for facilitating simulations of macromolecules and nanomaterials. Nat. Commun. 13, 68 (2022).">28</a></sup>, pysimm<sup><a aria-label="Reference 29" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR29" id="ref-link-section-d22328748e553" title="Fortunato, M. E. &amp; Colina, C. M. pysimm: a python package for simulation of molecular systems. SoftwareX 6, 7–12 (2017).">29</a></sup>, and Polymer Builder<sup><a aria-label="Reference 30" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR30" id="ref-link-section-d22328748e557" title="Choi, Y. K. et al. CHARMM-GUI Polymer builder for modeling and simulation of synthetic polymers. J. Chem. Theory Comput. 17, 2431–2443 (2021).">30</a></sup> for building and assembling long polymer chains; and FF-Converter<sup><a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR31" id="ref-link-section-d22328748e561" title="Lee, J. et al. CHARMM-GUI Input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field. J. Chem. Theory Comput. 12, 405–413 (2016).">31</a>,<a aria-label="Reference 32" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR32" id="ref-link-section-d22328748e564" title="Lee, J. et al. CHARMM-GUI supports the Amber force fields. J. Chem. Phys. 153, 035103 (2020).">32</a></sup> and ParmEd<sup><a aria-label="Reference 33" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR33" id="ref-link-section-d22328748e569" title="Shirts, M. R. et al. Lessons learned from comparing molecular dynamics engines on the SAMPL5 dataset. J. Comput. Aided Mol. Des. 31, 147–161 (2017).">33</a></sup> for preparing and converting inputs for several simulation programs. Although all software packages require familiarity with the fundamentals of molecular modeling, there are still opportunities to lower the entry barriers for modeling systems with multiple proteins and/or metabolites of interest at the all-atom resolution. Certain common molecular configurations are already handled by simulation and analysis programs directly, such as AmberTools<sup><a aria-label="Reference 15" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR15" id="ref-link-section-d22328748e573" title="Case, D. A. et al. AMBERTools. J. Chem. Inf. Model. 63, 6183–6191 (2023).">15</a></sup>, GROMACS<sup><a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR34" id="ref-link-section-d22328748e577" title="Páll, S., Abraham, M. J., Kutzner, C., Hess, B. &amp; Lindahl, E. Tackling exascale software challenges in molecular dynamics simulations with GROMACS. in Solving Software Challenges for Exascale (eds. Markidis, S. &amp; Laure, E.) 3–27 
                  https://doi.org/10.1007/978-3-319-15976-8_1
                  
                 (Springer International Publishing, Cham, 2015).">34</a>,<a aria-label="Reference 35" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR35" id="ref-link-section-d22328748e580" title="Abraham, M. J. et al. GROMACS: high performance molecular simulations through multi-level parallelism from laptops to supercomputers. SoftwareX 1–2, 19–25 (2015).">35</a></sup>, NAMD/VMD<sup><a aria-label="Reference 36" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR36" id="ref-link-section-d22328748e584" title="Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. J. Chem. Phys. 153, 044130 (2020).">36</a>,<a aria-label="Reference 37" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR37" id="ref-link-section-d22328748e587" title="Humphrey, W., Dalke, A. &amp; Schulten, K. VMD: visual molecular dynamics. J. Mol. Graph. 14, 33–38 (1996).">37</a></sup>, and OpenMM<sup><a aria-label="Reference 38" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR38" id="ref-link-section-d22328748e591" title="Eastman, P. et al. OpenMM 7: rapid development of high performance algorithms for molecular dynamics. PLOS Comput. Biol. 13, e1005659 (2017).">38</a></sup>, which can add a box of water and ions to a single molecule of interest or to a pre-arranged molecular complex or mixture. Similarly, several tools exist to facilitate building combined protein-membrane models by using the template copying method for water placement and by using either a membrane template scheme for positioning lipids or solving the packing problem for the membrane-protein only<sup><a aria-label="Reference 25" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR25" id="ref-link-section-d22328748e595" title="Pezeshkian, W., König, M., Wassenaar, T. A. &amp; Marrink, S. J. Backmapping triangulated surfaces to coarse-grained membrane models. Nat. Commun. 11, 2296 (2020).">25</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR39" id="ref-link-section-d22328748e598" title="Sommer, B. et al. CELLmicrocosmos 2.2 membraneEditor: a modular interactive shape-based software approach to solve heterogeneous membrane packing problems. J. Chem. Inf. Model. 51, 1165–1182 (2011).">39</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR40" id="ref-link-section-d22328748e598_1" title="Jo, S., Lim, J. B., Klauda, J. B. &amp; Im, W. CHARMM-GUI membrane builder for mixed bilayers and its application to yeast membranes. Biophys. J. 97, 50–58 (2009).">40</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR41" id="ref-link-section-d22328748e598_2" title="Lee, J. et al. CHARMM-GUI membrane builder for complex biological membrane simulations with Glycolipids and Lipoglycans. J. Chem. Theory Comput. 15, 775–786 (2019).">41</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR42" id="ref-link-section-d22328748e598_3" title="Knight, C. J. &amp; Hub, J. S. MemGen: a general web server for the setup of lipid membrane simulation systems. Bioinformatics 31, 2897–2899 (2015).">42</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR43" id="ref-link-section-d22328748e598_4" title="Wassenaar, T. A., Ingólfsson, H. I., Böckmann, R. A., Tieleman, D. P. &amp; Marrink, S. J. Computational lipidomics with insane: a versatile tool for generating custom membranes for molecular simulations. J. Chem. Theory Comput. 11, 2144–2155 (2015).">43</a>,<a aria-label="Reference 44" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR44" id="ref-link-section-d22328748e601" title="Doerr, S., Giorgino, T., Martínez-Rosell, G., Damas, J. M. &amp; De Fabritiis, G. High-throughput automated preparation and simulation of membrane proteins with HTMD. J. Chem. Theory Comput. 13, 4003–4011 (2017).">44</a></sup>. In each of these cases, only a single protein or pre-oriented protein complex can be handled per molecular model. Furthermore, while PACKMOL, cellPACK, and LipidWrapper can pack a wide variety of molecules into nearly any defined shape, neither program currently handles the periodic boundary conditions (PBC) necessary for MD simulations. In addition, significant post-processing is typically required to use the output of these tools to prepare inputs for different simulation programs. <a href="https://www.charmm-gui.org">CHARMM-GUI</a><sup><a aria-label="Reference 45" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR45" id="ref-link-section-d22328748e612" title="Jo, S., Kim, T., Iyer, V. G. &amp; Im, W. CHARMM-GUI: a web-based graphical user interface for CHARMM. J. Comput. Chem. 29, 1859–1865 (2008).">45</a></sup> is a cyberinfrastructure that guides researchers through building simulation-ready atomistic and coarse-grained models containing a single component or complex of interest in solution or membrane environments. Recent developments have also extended its functionality to cover various nanomaterials and polymer models<sup><a aria-label="Reference 30" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR30" id="ref-link-section-d22328748e616" title="Choi, Y. K. et al. CHARMM-GUI Polymer builder for modeling and simulation of synthetic polymers. J. Chem. Theory Comput. 17, 2431–2443 (2021).">30</a>,<a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR46" id="ref-link-section-d22328748e619" title="Choi, Y. K. et al. CHARMM-GUI Nanomaterial modeler for modeling and simulation of nanomaterial systems. J. Chem. Theory Comput. 18, 479–493 (2022).">46</a></sup>.</p><p>In this work, we present CHARMM-GUI Multicomponent Assembler (MCA) that solves a complex packing problem for many components of interest under the PBC, enables using pre-equilibrated membrane or membrane-like (nanomaterials and polymer) materials, and generalizes the template-based solvent building approach to cover non-water and mixed solvents. To illustrate MCA’s versatility for heterogeneous system building, we prepare and simulate 6 challenging systems at various densities, for a total of 20 systems, using only components generated by other CHARMM-GUI modules: (1) solvated proteins, (2) solvated proteins with a pre-equilibrated membrane, (3) solvated proteins with a sheet-like nanomaterial, (4) solvated proteins with a sheet-like polymer, (5) a mixed membrane-nanomaterial system, and (6) a sheet-like polymer with gaseous solvent.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Workflow of Multicomponent Assembler</h3><p>MCA handles molecular components by grouping them into 5 categories (Table <a data-track="click" data-track-action="table anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Tab1">1</a>) that differ by their general positioning requirements and assumptions. MCA combines components into 6 overall molecular configurations (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Fig1">1</a>) using the workflow described below and in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Fig2">2</a>. Note that MCA currently uses the CHARMM36(m) FF<sup><a aria-label="Reference 47" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR47" id="ref-link-section-d22328748e647" title="Huang, J. et al. CHARMM36m: an improved force field for folded and intrinsically disordered proteins. Nat. Methods 14, 71–73 (2017).">47</a></sup> for protein and lipid components, CGenFF<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR16" id="ref-link-section-d22328748e651" title="Vanommeslaeghe, K. &amp; MacKerell, A. D. Jr. Automation of the CHARMM general force field (CGenFF) I: bond perception and atom typing. J. Chem. Inf. Model. 52, 3144–3154 (2012).">16</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR17" id="ref-link-section-d22328748e651_1" title="Vanommeslaeghe, K., Raman, E. P. &amp; MacKerell, A. D. Jr. Automation of the CHARMM general force field (CGenFF) II: assignment of bonded parameters and partial atomic charges. J. Chem. Inf. Model. 52, 3155–3168 (2012).">17</a>,<a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR18" id="ref-link-section-d22328748e654" title="Vanommeslaeghe, K. et al. CHARMM general force field: a force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields. J. Comput. Chem. 31, 671–690 (2010).">18</a>,<a aria-label="Reference 48" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR48" id="ref-link-section-d22328748e657" title="Yu, W., He, X., Vanommeslaeghe, K. &amp; MacKerell, A. D. Jr. Extension of the CHARMM general force field to sulfonyl-containing compounds and its utility in biomolecular simulations. J. Comput. Chem. 33, 2451–2468 (2012).">48</a>,<a aria-label="Reference 49" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR49" id="ref-link-section-d22328748e660" title="Soteras Gutiérrez, I. et al. Parametrization of halogen bonds in the CHARMM general force field: Improved treatment of ligand–protein interactions. Bioorg. Med. Chem. 24, 4812–4825 (2016).">49</a></sup> for polymers, and INTERFACE FF<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR50" id="ref-link-section-d22328748e665" title="Lin, T.-J. &amp; Heinz, H. Accurate force field parameters and pH resolved surface models for hydroxyapatite to understand structure, mechanics, hydration, and biological interfaces. J. Phys. Chem. C. 120, 4975–4992 (2016).">50</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR51" id="ref-link-section-d22328748e665_1" title="Heinz, H., Vaia, R. A., Farmer, B. L. &amp; Naik, R. R. Accurate simulation of surfaces and interfaces of face-centered cubic metals using 12−6 and 9−6 Lennard-Jones potentials. J. Phys. Chem. C. 112, 17281–17290 (2008).">51</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR52" id="ref-link-section-d22328748e665_2" title="Emami, F. S. et al. Force field and a surface model database for Silica to simulate interfacial properties in atomic resolution. Chem. Mater. 26, 2647–2658 (2014).">52</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR53" id="ref-link-section-d22328748e665_3" title="Mishra, R. K., Kanhaiya, K., Winetrout, J. J., Flatt, R. J. &amp; Heinz, H. Force field for calcium sulfate minerals to predict structural, hydration, and interfacial properties. Cem. Concr. Res. 139, 106262 (2021).">53</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR54" id="ref-link-section-d22328748e665_4" title="Liu, J. et al. Interpretable molecular models for molybdenum disulfide and insight into selective peptide recognition. Chem. Sci. 11, 8708–8722 (2020).">54</a>,<a aria-label="Reference 55" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR55" id="ref-link-section-d22328748e668" title="Heinz, H., Lin, T.-J., Kishore Mishra, R. &amp; Emami, F. S. Thermodynamically consistent force fields for the assembly of inorganic, organic, and biological nanostructures: the INTERFACE force field. Langmuir 29, 1754–1765 (2013).">55</a></sup> for nanomaterials.</p><div class="c-article-table" data-container-section="table" data-test="inline-table" id="table-1"><figure><figcaption class="c-article-table__figcaption"><b data-test="table-caption" id="Tab1">Table 1 Supported component types</b></figcaption><div class="u-text-right u-hide-print"><a aria-label="Full size table 1" class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" href="/articles/s41467-024-49700-4/tables/1" rel="nofollow"><span>Full size table</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Example system configurations generated by Multicomponent Assembler." id="figure-1"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig1">Fig. 1: Example system configurations generated by Multicomponent Assembler.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-024-49700-4/figures/1" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig1_HTML.png?as=webp" type="image/webp"><img alt="figure 1" aria-describedby="Fig1" height="467" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig1_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p>In all figures containing protein, ubiquitin (PDB: 1UBQ) is orange, villin (PDB: 1VII) is green, and protein G (PDB: 3GB1) is cyan. K<sup>+</sup> and Cl<sup>-</sup> ions are shown as orange and purple spheres, respectively, and water molecules as blue translucent lines. When included, KCl concentration is 0.15 M. <b>A</b> Proteins with water and ions only. <b>B</b> Pre-equilibrated axolemma membrane combined with proteins. <b>C</b> Proteins with a hydroxyapatite slab centered on the unit cell’s Z boundary. <b>D</b> Artificial membrane of polyethylene oxide-poly(ethylethylene) (EO<sub>40</sub>EE<sub>37</sub>) with proteins. <b>E</b> Supported lipid bilayer consisting of 1-palmitoyl-2-oleoyl-phosphatidylcholine (POPC) membrane and mica separated by a 20 Å water layer. <b>F</b> CO<sub>2</sub> adsorption on a polyethylene terephthalate (PET) membrane.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 1" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure1 Full size image" data-track-label="button" href="/articles/s41467-024-49700-4/figures/1" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Multicomponent Assembler workflow." id="figure-2"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig2">Fig. 2: Multicomponent Assembler workflow.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-024-49700-4/figures/2" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig2_HTML.png?as=webp" type="image/webp"><img alt="figure 2" aria-describedby="Fig2" height="431" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig2_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p>Step colors indicate which model types they use. The Copy &amp; Pack procedure is used to place solutes and to create a small solvent template. CG: coarse-grained. MD: molecular dynamics.<sup>*</sup>Positions are refined using the algorithm in Supplementary Algorithm <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">1</a>.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 2" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure2 Full size image" data-track-label="button" href="/articles/s41467-024-49700-4/figures/2" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec4">STEP 1 – Read structures and FFs</h4><p>Because CHARMM-GUI uses CHARMM for model building and manipulation, it is required to provide molecular structures in both CHARMM protein structure file (PSF) and coordinate (CRD) formats. Additionally, for periodic (nanomaterial) structures with image bonds (between the primary system and the PBC systems), listing the bonds in a CHARMM-formatted image PSF file is necessary. Molecules containing residues already present in the CHARMM and INTERFACE FFs are recognized automatically, but any additional residue topologies and FFs must be provided in CHARMM’s residue topology file (RTF) and parameter (PRM) formats with unique identifiers that do not conflict with the existing FFs. Note that these files can be obtained from other CHARMM-GUI modules such as PDB Reader &amp; Modeler<sup><a aria-label="Reference 56" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR56" id="ref-link-section-d22328748e903" title="Park, S.-J., Kern, N., Brown, T., Lee, J. &amp; Im, W. CHARMM-GUI PDB Manipulator: various PDB structural modifications for biomolecular Modeling and Simulation. J. Mol. Biol. 435, 167995 (2023).">56</a></sup>, Glycan Reader &amp; Modeler<sup><a aria-label="Reference 57" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR57" id="ref-link-section-d22328748e907" title="Park, S.-J. et al. Glycan reader is improved to recognize most sugar types and chemical modifications in the protein data bank. Bioinformatics 33, 3051–3057 (2017).">57</a></sup>, Ligand Reader &amp; Modeler<sup><a aria-label="Reference 21" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR21" id="ref-link-section-d22328748e911" title="Kim, S. et al. CHARMM-GUI ligand reader and modeler for CHARMM force field generation of small molecules. J. Comput. Chem. 38, 1879–1886 (2017).">21</a></sup>, Nanomaterial Modeler<sup><a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR46" id="ref-link-section-d22328748e915" title="Choi, Y. K. et al. CHARMM-GUI Nanomaterial modeler for modeling and simulation of nanomaterial systems. J. Chem. Theory Comput. 18, 479–493 (2022).">46</a></sup>, and Polymer Builder<sup><a aria-label="Reference 30" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR30" id="ref-link-section-d22328748e919" title="Choi, Y. K. et al. CHARMM-GUI Polymer builder for modeling and simulation of synthetic polymers. J. Chem. Theory Comput. 17, 2431–2443 (2021).">30</a></sup>.</p><p>For each uploaded component, MCA uses CHARMM internal functions to determine the molecular dimensions, volume, solvent-accessible volume, mass, charge, number of residues, and radius of gyration. To facilitate membrane building, the uploaded coordinates are used to determine the volume and center of mass of the molecular regions that would be located within, above, and below a membrane centered at <i>Z</i> = 0 with a hydrophobic thickness of 24 Å. The component’s dimensions are calculated as its bounding box when the component’s primary, secondary, and tertiary axes are aligned with the <i>X, Y</i>, and <i>Z</i> axes, respectively. The component’s length is determined as the maximum distance between any two atoms within the structure. This value is needed to estimate the minimum box size that prevents any component from interacting with its own PBC images. The molecular volume is calculated by polling coordinates using a grid spacing of 0.5 Å within any atom’s van der Waals radius, with a maximum probe range of 6 Å, and counting the empty holes within a molecule toward its volume. For solvent-accessible volume, the procedure is repeated with atomic radii increased by 1.4 Å, corresponding to the water radius.</p><p>To ensure that segment identifiers are both unique and meaningful, all segments from each input structure are written to separate PDB files to determine the segment type (protein, DNA, RNA, heterogen, carbohydrate, or water). Each input segment is then re-written to a PSF such that the first letter corresponds to the segment type (protein: P, DNA: D, RNA: R, heterogen: H, carbohydrate: C, water: W), and the next two letters designate a unique ID. Thus, the first input protein segment is renamed to PAA, the 27<sup>th</sup> to PBA, etc. A human-readable map of input-to-output segment IDs is written to a file (<span class="u-monospace">rename_map.txt</span>) for the user’s convenience.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec5">STEP 2 – Determine system size and positional constraints, and pack solutes</h4><p>To properly specify the components in a given system, users must identify the type of each component as listed in Table <a data-track="click" data-track-action="table anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Tab1">1</a>. If there are no membrane-embedded components, users must specify whether a new membrane should be generated.</p><p>In the simplest case, the user already knows the exact number of copies of each component and system dimensions. However, commonly only the relative ratio of components with respect to each other is known. Similarly, instead of knowing the exact system dimensions, the user may know only the fraction of available volume that should be occupied by uploaded components (not including solvent and ions that are handled later). To guide in determining an appropriate system size and number of components, one can specify the relative component ratios, and specify either exact system dimensions or volume fraction and an approximation of the other quantity, as shown in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Fig3">3</a>. MCA recommends the closest match to the approximated quantity with the following constraints: (1) no dimension is small enough that a component can interact with its own PBC images, and (2) the volume fraction is low enough that packing can plausibly succeed. To quickly estimate an upper bound on the maximum packing density, we rely on a heuristic that empty space should be no less than the total solvent accessible volume of all components. However, in our observation, achieving volume fractions higher than 30% usually requires significant trial and error. As shown in our benchmark study (see Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">1</a> and Comparison with other programs), the difficulty of finding a collision-free packing depends on the shape of the components and the desired system density. In particular, for high volume fractions (e.g., &gt; 30%), as shown in our benchmark testing, a trial-and-error process is unavoidable in the current approach.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="User interface and corresponding packing results with varied volume fraction and component ratios." id="figure-3"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig3">Fig. 3: User interface and corresponding packing results with varied volume fraction and component ratios.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-024-49700-4/figures/3" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig3_HTML.png?as=webp" type="image/webp"><img alt="figure 3" aria-describedby="Fig3" height="707" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig3_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p>10 % v/v is used in (<b>A</b>) and 30 % v/v in (<b>B</b>). Both parameters result in a cube with a side length of 74.19 Å. The resulting component count is updated after clicking “Calculate System Size”. Packing results of (<b>A</b>) and (<b>B</b>) are shown in (<b>C</b>) and (<b>D</b>), respectively. Molecules may cross the system boundaries only if collisions with periodic image atoms are avoided. In (<b>C</b>) and (<b>D</b>), all atoms are wrapped to the primary cell to illustrate the use of available space. Ubiquitin (PDB: 1UBQ), villin (PDB: 1VII), and protein G (PDB: 3GB1) are colored orange, green, and cyan, respectively.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 3" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure3 Full size image" data-track-label="button" href="/articles/s41467-024-49700-4/figures/3" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>Systems without a membrane or periodic component are modeled with a cubic crystal lattice (X = Y = Z, 90° angles); those with a membrane are tetragonal (X = Y ≠ Z, 90° angles) and require the system width (X or Y) and height (Z) to be determined separately (see Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">2a</a>); those with a periodic component are orthorhombic (X ≠ Y ≠ Z, 90° angles), with system <i>X</i> and <i>Y</i> dimensions defined by the periodic component’s dimensions. For the purpose of estimating the available space for packing, the approximate membrane or periodic component’s thickness along <i>Z</i> must be provided, if applicable (see Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">2b</a>). Furthermore, the provided membrane or periodic component’s <i>XYZ</i> dimensions are reserved for that component. Note that their volumes are subtracted from the available space for system density calculations and solvent/solute components are prevented from being initialized within or entering the reserved regions.</p><p>After the system dimensions and component counts are determined, the user can specify positional constraints (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Fig4">4</a>) whose options differ slightly between component types. For solvated components, the options are: (1) “none”, which accepts any collision-free position and orientation, (2) “planar (Z) restraint”, which fixes the center of mass (COM) of the component to a given Z position, but allows translation along X and Y, and allows any rotation about the component’s COM, and (3) “fixed XYZ”, in which the component’s coordinates are translated so that its COM lies at a given coordinate, and no further movement is allowed. For membrane-embedded components, the options are (1) “none”, in which the component’s Z positions remain unchanged from their uploaded values, but the component may translate in the X and Y directions and rotate about the <i>Z</i> axis, (2) “planar (Z) restraint”, which is like “none”, but the Z position may be changed a bit from its uploaded value, and (3) “fixed XY”, in which the Z position remains at its initial value, but the user can specify a fixed X, and Y coordinate for the component’s COM. Finally, for XY periodic components, the user can specify a COM-Z position since the X and Y lengths of periodic components already define the system’s length in X and Y. If no options are selected, the largest component is fixed at the system center. In all cases, the default constraint type is “none”.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="User interface of positioning options for solvated components." id="figure-4"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig4">Fig. 4: User interface of positioning options for solvated components.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-024-49700-4/figures/4" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig4_HTML.png?as=webp" type="image/webp"><img alt="figure 4" aria-describedby="Fig4" height="670" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig4_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>Any component can have its positioning constraints modified by clicking “Set Position”, as shown in (<b>A</b>). Exact positions can be specified for (<b>B</b>) individual components’ center of mass (COM), or (<b>C</b>) identical planar restraints for all copies. Results of applying the settings in (<b>C</b>) and (<b>B</b>) are shown in (<b>D</b>) and (<b>E</b>), respectively. A translucent plane shows the location of each planar restraint, which is applied to both component copies in (<b>D</b>), and only the pink copy in (<b>E</b>). Corresponding coarse-grained (CG) particles are shown as translucent spheres.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 4" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure4 Full size image" data-track-label="button" href="/articles/s41467-024-49700-4/figures/4" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>The non-solvent components are first packed using an uncharged coarse-grained (CG) sphere model that uses 1 to 3 large atoms per copy of each component (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Fig5">5</a>). Positions are initialized according to the aforementioned positioning constraints chosen by the user. The CG model is then simulated in 5 iterations with Langevin dynamics for 100 ps using a 2 fs timestep at 500 K with a switching function applied to nonbonded forces at 10 Å plus the largest sphere’s radius of gyration. Each iteration of dynamics uses a system size decreasing from 150% of the user’s target to 100%. Solvated components are initially positioned by applying a random rotation and aligning the component’s COM with the center of its corresponding CG particle. For membrane components, a root-mean-squared best-fit alignment is performed between the resulting CG coordinates and the reference coordinates calculated in STEP 1; the <i>Z</i> axis rotation and X/Y translation results of this best fit are applied to the all-atom model. Then, collisions are minimized with up to 7 iterations of the greedy conformation search (see Supplementary Algorithm <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">1</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Coarse-grained representations of molecular components." id="figure-5"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig5">Fig. 5: Coarse-grained representations of molecular components.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-024-49700-4/figures/5" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig5_HTML.png?as=webp" type="image/webp"><img alt="figure 5" aria-describedby="Fig5" height="583" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig5_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p>For solvated components, a single particle represents the whole component; for membrane-embedded components, 1–3 particles are used, corresponding to the above-membrane, intra-membrane, and below-membrane regions of the component in a 24 Å thick implicit membrane region centered at <i>Z</i> = 0. The van der Waals radius of each particle is set to the radius of gyration (R<sub>g</sub>) of the atoms contained in the corresponding above/intra/below membrane region or omitted if there are no atoms in that region. The solvent accessible volume around a particle corresponds to its molecular volume plus a 1.4 Å thick shell around that particle’s volume. LPS: lipopolysaccharide. LptD/E: Lipopolysaccharide transporter protein subunits D and E.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 5" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure5 Full size image" data-track-label="button" href="/articles/s41467-024-49700-4/figures/5" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>If packing fails to find a collision-free configuration, MCA stops and emits an error. Otherwise, segments are renamed using the map from STEP 1. Individual copies of a component are distinguished by appending a copy ID to the segment ID, so that the first protein segment’s copy becomes PAA1, etc.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec6">STEP 3 – Build membrane (membrane systems only)</h4><p>If the system contains membrane-embedded components or the user chooses to generate a membrane, the membrane lipid composition is determined using Membrane Builder<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR40" id="ref-link-section-d22328748e1136" title="Jo, S., Lim, J. B., Klauda, J. B. &amp; Im, W. CHARMM-GUI membrane builder for mixed bilayers and its application to yeast membranes. Biophys. J. 97, 50–58 (2009).">40</a>,<a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR41" id="ref-link-section-d22328748e1139" title="Lee, J. et al. CHARMM-GUI membrane builder for complex biological membrane simulations with Glycolipids and Lipoglycans. J. Chem. Theory Comput. 15, 775–786 (2019).">41</a>,<a aria-label="Reference 58" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR58" id="ref-link-section-d22328748e1142" title="Jo, S., Kim, T. &amp; Im, W. Automated builder and database of protein/membrane complexes for molecular dynamics simulations. PLOS ONE 2, e880 (2007).">58</a></sup> with the system dimensions determined in STEP 2. The lipid packing and replacement procedures are the same as in Membrane Builder, except that the system dimensions during lipid packing are decreased in 4 iterations from 150% to 120% of target membrane width, and with 21 iterations from 120% to 100%. All membrane lipids built by Membrane Builder are given the segment ID MEMB.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec7">STEP 4 – Build solution</h4><p>The water and ion building procedures in this study follow the same protocols as in Solution Builder<sup><a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR31" id="ref-link-section-d22328748e1155" title="Lee, J. et al. CHARMM-GUI Input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field. J. Chem. Theory Comput. 12, 405–413 (2016).">31</a>,<a aria-label="Reference 32" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR32" id="ref-link-section-d22328748e1158" title="Lee, J. et al. CHARMM-GUI supports the Amber force fields. J. Chem. Phys. 153, 035103 (2020).">32</a>,<a aria-label="Reference 45" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR45" id="ref-link-section-d22328748e1161" title="Jo, S., Kim, T., Iyer, V. G. &amp; Im, W. CHARMM-GUI: a web-based graphical user interface for CHARMM. J. Comput. Chem. 29, 1859–1865 (2008).">45</a></sup>, with the additional ability to utilize user-uploaded ions. Users can build any combination of ions supported by the CHARMM36 FF, containing up to 7 atoms, in addition to uploading custom ions. Solvent options allow users to specify either the concentration of any solvent, including water, or a solvent density and ratio of solvent volumes. The solvent options page estimates the number of each generated solvent and ion with the user’s settings, although the exact number may differ as described below in STEP 5.</p><p>To maintain the approximate solvent ratios, user-uploaded solvent molecules are initially packed into a small box whose size is chosen based on heuristics. The packing procedure follows the same protocols described in STEP 2. After packing, the box is replicated to reach the target box size, and any molecules outside the system boundaries are deleted.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec8">STEP 5 – Assemble system</h4><p>The solvent and non-solvent components are combined by superimposing the solvents with the non-solvents and deleting any solvent molecules that collide with non-solvent molecules within 2.8 Å. If the system contains a membrane or XY periodic component, solvents located in the component’s interior (as defined by the user in STEP 2) are also deleted. If a solvent is present in a higher concentration than the user’s target, extras are deleted randomly as needed. However, extra solvent molecules are not created to fill space when system assembly results in fewer than the target number of molecules. Finally, any uploaded components with the segment ID MEMB and membrane generated by Membrane Builder (if any) are joined into a single segment, so that appropriate restraints can be generated in STEP 6.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec9">STEP 6 – Generate simulation inputs</h4><p>MCA uses FF-Converter to generate simulation inputs for various MD engines<sup><a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR31" id="ref-link-section-d22328748e1184" title="Lee, J. et al. CHARMM-GUI Input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field. J. Chem. Theory Comput. 12, 405–413 (2016).">31</a>,<a aria-label="Reference 32" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR32" id="ref-link-section-d22328748e1187" title="Lee, J. et al. CHARMM-GUI supports the Amber force fields. J. Chem. Phys. 153, 035103 (2020).">32</a></sup>. For systems containing periodic nanomaterials or polymers, the generated NVT (constant number of particles, volume, and temperature) equilibration inputs use a multi-step scheme where protein restraints are progressively released. Inputs generated for NPT (constant number of particles, pressure, and temperature) production use anisotropic pressure coupling, and NPAT (constant number of particles, pressure, membrane area, and temperature) production inputs use semi-isotropic pressure coupling with pressure applied only along the <i>Z</i> dimension.</p><h3 class="c-article__sub-heading" id="Sec10">Protein-protein and protein-membrane contacts</h3><p>The effects of protein crowding near surfaces are important in biology and industry. The prevalence of membrane surfaces in cells suggests that even cytosolic proteins must interact with membranes, especially in organelles with high surface area such as the endoplasmic reticulum and mitochondria. In laboratory settings, solvated proteins can bind to their container surfaces and leave residues that require strong acids to remove. Nawrocki et al.<sup><a aria-label="Reference 59" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR59" id="ref-link-section-d22328748e1203" title="Nawrocki, G., Im, W., Sugita, Y. &amp; Feig, M. Clustering and dynamics of crowded proteins near membranes and their influence on membrane bending. Proc. Natl Acad. Sci. 116, 24562–24567 (2019).">59</a></sup> recently reported the contacts between crowded proteins and a membrane of cholesterol (CHL1), 1-palmitoyl-2-oleoyl-phosphocholine (POPC), and palmitoyl-sphingomyelin (PSM) via atomistic MD simulation. Protein crowding is difficult to model in atomic details due to the high possibilities for collisions between atoms that cannot be resolved by energy minimization. To study how different surfaces affect protein contact behaviors in crowded environments and to demonstrate the versatility of MCA’s modeling capabilities, we reproduced each of the models in Nawrocki et al. and modeled 3 additional membrane types with varying protein concentrations (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Fig1">1a–d</a>).</p><p>For the models containing proteins and a membrane-like component (see “Methods”), we measured the contact probabilities (<i>p</i>) between all pairs of components and aggregated them by component type (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Fig6">6</a>). Our results show that protein contact preferences depend greatly on the membrane environment and protein volume fraction. At 5% v/v, proteins are very likely (<i>p</i> &gt; 0.5) to contact hydroxyapatite (HAP) and polyethylene oxide-poly(ethylethylene) (EO<sub>40</sub>EE<sub>37</sub>) but unlikely (<i>p</i> &lt; 0.1) to contact CHL1/POPC/PSM membranes. Only ubiquitin and villin show high contact probabilities near the axolemma membrane at 5% v/v. For all membrane types, increasing protein v/v has the effect of increasing the relative proportion of contacts between proteins and other proteins, though contacts between protein and membrane remain frequent (<i>p</i> &gt; 0.14) for all protein types and membranes except CHL1/POPC/PSM. Indeed, all protein-CHL1/POPC/PSM contacts surprisingly decrease between 5–10% v/v and increase between 10–30% v/v, suggesting a strong preference for protein-protein contacts that is only overcome by increased protein crowding. The higher protein-membrane contacts at 5% v/v can be explained by the lack of opportunities for protein cluster formation<sup><a aria-label="Reference 59" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR59" id="ref-link-section-d22328748e1233" title="Nawrocki, G., Im, W., Sugita, Y. &amp; Feig, M. Clustering and dynamics of crowded proteins near membranes and their influence on membrane bending. Proc. Natl Acad. Sci. 116, 24562–24567 (2019).">59</a></sup>. It should be noted that this trend has a small magnitude: even the highest protein-CHL1/POPC/PSM contact fraction (7.6% for ubiquitin-membrane at 30% v/v) is lower than the lowest protein-membrane contact fraction with other membranes (14.5% for protein G-axolemma at 5% v/v). We believe more simulation replicas are required to establish the statistical significance of this trend reversal. The overall low protein-CHL1/POPC/PSM contacts and high contacts with the other membranes we simulated are consistent with Nawrocki et al.’s<sup><a aria-label="Reference 59" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR59" id="ref-link-section-d22328748e1237" title="Nawrocki, G., Im, W., Sugita, Y. &amp; Feig, M. Clustering and dynamics of crowded proteins near membranes and their influence on membrane bending. Proc. Natl Acad. Sci. 116, 24562–24567 (2019).">59</a></sup> finding that proteins are thermodynamically excluded from membranes with no charged lipids.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Protein contact probability for all systems containing membrane-like components." id="figure-6"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig6">Fig. 6: Protein contact probability for all systems containing membrane-like components.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-024-49700-4/figures/6" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig6_HTML.png?as=webp" type="image/webp"><img alt="figure 6" aria-describedby="Fig6" height="606" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig6_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p>A contact is defined when Cα-Cα distance ≤ 7 Å. Note that for calculation of % v/v in CHL1/POPC/PSM systems only, protein volume includes the solvent accessible volume, as described in STEP 1. CHL1: cholesterol, POPC: 1-palmitoyl-2-oleoyl-phosphatidylcholine, PSM: palmitoylsphingomyelin (PSM). EO<sub>40</sub>EE<sub>37</sub>: polyethylene oxide-poly(ethylethylene).</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 6" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure6 Full size image" data-track-label="button" href="/articles/s41467-024-49700-4/figures/6" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>Proteins uniformly prefer contacts with the same type of protein than with other protein types, with two notable exceptions. First, at 5% protein v/v, villin prefers contacts with axolemma membrane over contacts with other copies of villin. Second, no obvious contact patterns between proteins are observed near HAP for protein v/v &lt; 30% because contacts with HAP are strongly preferred.</p><p>Visualization of the HAP and EO<sub>40</sub>EE<sub>37</sub> simulation trajectories suggests different reasons for their frequent protein contacts. The duration of contact events between proteins and HAP is higher than that in other membranes, with several contact events lasting longer than 100 ns (Supplemental Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">3</a>). In contrast, the hydrophilic EO domain of EO<sub>40</sub>EE<sub>37</sub> dissolved so much into water throughout simulation that there was almost no region of the solution where EO was absent (Supplemental Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">4</a>) as shown by the width of EO<sub>40</sub>EE<sub>37</sub>’s distribution in Supplemental Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">5</a>. This behavior suggests that EO<sub>40</sub>EE<sub>37</sub> should be equilibrated separately with a thick water layer before being combined with proteins via MCA.</p><p>Although we modeled the same CHL1/POPC/PSM membrane and proteins as Nawrocki et al.<sup><a aria-label="Reference 59" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR59" id="ref-link-section-d22328748e1301" title="Nawrocki, G., Im, W., Sugita, Y. &amp; Feig, M. Clustering and dynamics of crowded proteins near membranes and their influence on membrane bending. Proc. Natl Acad. Sci. 116, 24562–24567 (2019).">59</a></sup>, we observed overall higher protein aggregation. For example, their contact fraction between villin and other villin molecules (villin-villin contacts) ranges between 25-30% whereas ours is between 27–73%. Although we both report overall higher protein contact probabilities with increasing protein concentrations, there are exceptions to this trend: Nawrocki et al. show a statistically insignificant decrease in villin-villin contact between 5% and 10% v/v. In contrast, only our protein G contact fraction decreases between 5% and 10%. Notably, they used the CHARMM36 FF with modified ion parameters, and they also decreased protein aggregation by multiplying protein-water Lennard-Jones (LJ) potentials by 1.09, whereas we used the CHARMM36m FF that uses an updated CMAP potential to reduce left-handed α-helix formation but does not directly address protein-water interaction strength by default<sup><a aria-label="Reference 47" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR47" id="ref-link-section-d22328748e1305" title="Huang, J. et al. CHARMM36m: an improved force field for folded and intrinsically disordered proteins. Nat. Methods 14, 71–73 (2017).">47</a></sup>. Due to these methodological differences, a direct comparison of our results with those of Nawrocki et al. is challenging. Further study could see if reintroducing water interaction scaling reconciles this discrepancy.</p><h3 class="c-article__sub-heading" id="Sec11">POPC diffusion near mica-supported lipid bilayers</h3><p>Planar-supported lipid bilayers (SLBs), artificial membranes consisting of a lipid bilayer deposited on a solid support, have become an essential tool for investigating the properties and functions of biological membranes. Compared to other membrane models, such as giant unilamellar vesicles or cell membranes, SLBs offer numerous advantages, including their simplicity, reproducibility, and versatility in analytical techniques, such as fluorescence microscopy, surface plasmon resonance, and atomic force microscopy<sup><a aria-label="Reference 60" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR60" id="ref-link-section-d22328748e1317" title="Macháň, R. &amp; Hof, M. Lipid diffusion in planar membranes investigated by fluorescence correlation spectroscopy. Biochim. Biophys. Acta BBA - Biomembr. 1798, 1377–1391 (2010).">60</a></sup>. Therefore, SLBs have been widely utilized in the fields of biophysics, biochemistry, and cell biology to study a variety of membrane-related processes, such as membrane protein function, protein-lipid interactions, and membrane fusion. Therefore, understanding the lateral motion of lipids in SLBs is crucial in elucidating the properties and functions of biological systems. The lateral mobility of lipids is a key determinant of membrane structure and function and is influenced by various factors such as temperature, lipid composition, and the presence of membrane proteins. Thus, accurate measurements of lipid diffusion coefficients are necessary to fully understand membrane behavior and corresponding biological events.</p><p>In this study, we investigated the effect of the solid support on lipid diffusion in SLBs using model systems (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Fig1">1e</a>). We prepared three systems with a pure POPC bilayer separated by 1, 2, and 3 nm from a mica support, and analyzed the diffusion coefficient of POPC lipids in the lower (D<sub>B-1nm</sub>, D<sub>B-2nm</sub>, and D<sub>B-3nm</sub>) and upper (D<sub>T-1nm</sub>, D<sub>T-2nm</sub>, and D<sub>T-3nm</sub>) leaflets using the Diffusion Coefficient Tool<sup><a aria-label="Reference 61" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR61" id="ref-link-section-d22328748e1340" title="Giorgino, T. Computing diffusion coefficients in macromolecular simulations: the diffusion coefficient tool for VMD. J. Open Source Softw. 4, 1698 (2019).">61</a></sup> plugin in VMD<sup><a aria-label="Reference 37" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR37" id="ref-link-section-d22328748e1344" title="Humphrey, W., Dalke, A. &amp; Schulten, K. VMD: visual molecular dynamics. J. Mol. Graph. 14, 33–38 (1996).">37</a></sup>. Our MD simulation results (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Fig7">7</a>) show that D<sub>B-1nm</sub>, D<sub>B-2nm</sub>, and D<sub>B-3nm</sub> are 4.8 μm<sup>2</sup>/s, 6.6 μm<sup>2</sup>/s, and 6.4 μm<sup>2</sup>/s, respectively (all errors are less than 0.01 μm<sup>2</sup>/s). Experimental values of the POPC diffusion coefficient measured by raster image correlation spectroscopy (RICS) of a lipid-like probe in a GUV bilayer are ~7 ± 3 μm<sup>2</sup>/s<sup><a aria-label="Reference 62" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR62" id="ref-link-section-d22328748e1369" title="Gielen, E. et al. Measuring diffusion of lipid-like probes in artificial and natural membranes by raster image correlation spectroscopy (RICS): use of a commercial laser-scanning microscope with analog detection. Langmuir 25, 5209–5218 (2009).">62</a></sup>, implying that a bilayer on support behaves like free-standing lipids when the water thickness is more than 2 nm. In contrast, when the thickness of the water layer is less than 2 nm, there are strong interactions between the support and the lipids, which reduces the diffusion coefficient of the lipid in the lower leaflet of the SLB. This finding is consistent with the water thickness of dimyristoylphosphatidylcholine (DMPC) SLBs measured with the specular reflection of neutrons, which found that water thickness is in the range of 2 ~ 4 nm<sup><a aria-label="Reference 63" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR63" id="ref-link-section-d22328748e1373" title="Johnson, S. J. et al. Structure of an adsorbed dimyristoylphosphatidylcholine bilayer measured with specular reflection of neutrons. Biophys. J. 59, 289–294 (1991).">63</a></sup>. In addition to the lower leaflet, D<sub>T-1nm</sub>, D<sub>T-2nm</sub>, and D<sub>T-3nm</sub> show 5.9 μm<sup>2</sup>/s, 6.7 μm<sup>2</sup>/s, and 6.5 μm<sup>2</sup>/s, respectively (all errors are less than 0.01 μm<sup>2</sup>/s). These results suggest that the interaction between the mica support and the lower leaflet affects the diffusion of lipids in the upper leaflet as well. The decrease in D<sub>T-1nm</sub> is likely due to the coupling of the two leaflets via lipid-lipid interactions across the lipid bilayer. This finding highlights the importance of considering the behavior of both the upper and lower leaflets when studying the properties and functions of biological membranes in contact with solid supports.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="POPC diffusion near mica-supported lipid bilayers." id="figure-7"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig7">Fig. 7: POPC diffusion near mica-supported lipid bilayers.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-024-49700-4/figures/7" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig7_HTML.png?as=webp" type="image/webp"><img alt="figure 7" aria-describedby="Fig7" height="330" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig7_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-7-desc"><p>Upper and lower leaflets are distal and proximal to mica, respectively. Mean square displacement (MSD) of 1-palmitoyl-2-oleoyl-phosphatidylcholine (POPC) phosphorus is plotted against lag time, with diffusion coefficient (D) shown as a fitted line. All lag times between 1 to 250 ns were used to calculate D.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 7" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure7 Full size image" data-track-label="button" href="/articles/s41467-024-49700-4/figures/7" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>In all simulations containing Mica, the bottom POPC leaflet is the one closest to Mica. The difference between the systems is the thickness of the water layer separating Mica and POPC’s bottom leaflet (D<sub>1</sub> = 1 nm, D<sub>2</sub> = 2 nm, D<sub>3</sub> = 3 nm). Thus, D<sub>1</sub> is the system that provides the greatest interaction between Mica and POPC. To quantify this interaction, we used the CHARMM <span class="u-monospace">INTER</span> function to calculate interaction energies via the formula ∆E<sub>inter</sub> = E<sub>both</sub> – (E<sub>POPC</sub> + E<sub>Mica</sub>) for the final frame of each simulation. CHARMM energy calculations automatically separate energy terms; the VDW (∆E<sub>VDW</sub>) and electrostatic (∆E<sub>elec</sub>) terms are reported in Table <a data-track="click" data-track-action="table anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Tab2">2</a> below. We believe this interaction hinders the lateral diffusion of POPC by increasing friction between POPC and water.</p><div class="c-article-table" data-container-section="table" data-test="inline-table" id="table-2"><figure><figcaption class="c-article-table__figcaption"><b data-test="table-caption" id="Tab2">Table 2 Interaction energies between POPC and Mica in kcal mol<sup>-1</sup></b></figcaption><div class="u-text-right u-hide-print"><a aria-label="Full size table 2" class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" href="/articles/s41467-024-49700-4/tables/2" rel="nofollow"><span>Full size table</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec12">Diffusion of CO<sub>2</sub> through polymer membranes</h3><p>Polyethylene terephthalate (PET) is a cheap, recyclable plastic used in containers, clothing, and many other products with the resin ID code 1. Modern industrial production and recycling of PET causes significant non-renewable energy usage (NREU) and release of greenhouse gases (GHGs)<sup><a aria-label="Reference 64" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR64" id="ref-link-section-d22328748e1594" title="Shen, L., Nieuwlaar, E., Worrell, E. &amp; Patel, M. K. Life cycle energy and GHG emissions of PET recycling: change-oriented effects. Int. J. Life Cycle Assess. 16, 522–536 (2011).">64</a></sup>, which has led to an investigation into potential replacement plastics that could be constructed from renewable sources. One such candidate is poly(ethylene 2,5-furandicarboxylate) (PEF), which is amenable to production from biomass and which could substantially reduce NREU and GHG production<sup><a aria-label="Reference 65" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR65" id="ref-link-section-d22328748e1598" title="Eerhart, E. A. J. J., Faaij, A. P. C. &amp; Patel, M. K. Replacing fossil based PET with biobased PEF; process analysis, energy and GHG balance. Energy Environ. Sci. 5, 6407–6422 (2012).">65</a></sup>. In experimental investigations of the permeability of plastics to CO<sub>2</sub> and other gases, PEF has shown uniform improvement over PET in its CO<sub>2</sub> retention ability<sup><a aria-label="Reference 66" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR66" id="ref-link-section-d22328748e1606" title="Burgess, S. K., Kriegel, R. M. &amp; Koros, W. J. Carbon dioxide sorption and transport in amorphous poly(ethylene furanoate). Macromolecules 48, 2184–2193 (2015).">66</a></sup>. To investigate possible mechanisms for CO<sub>2</sub> diffusion through PET and PEF, we simulated 100 Å thick polymer sheets of each plastic with 1 atm initial pressure in pure CO<sub>2</sub> (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Fig1">1f</a>).</p><p>Our results show that CO<sub>2</sub> penetration in PEF<sub>95</sub> is comparable to that in PET<sub>95</sub> at each measured simulation time point. In our simulations, CO<sub>2</sub> molecules made rapid jumps between defects located at varying depths in the polymer structures, followed by long periods where the molecules remained in the same approximate region. The exact location of these defects varies greatly between replicas, as indicated by the error bars in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Fig8">8</a>, but the general trend for both polymers is an increase in overall CO<sub>2</sub> diffusion over time. Notably, between 666–1333 ns, the CO<sub>2</sub> density near the polymer center (0–12 Å) was higher for PET<sub>95</sub> than for PEF<sub>95</sub>, and the CO<sub>2</sub> density near the polymer periphery (15–37 Å) was lower for PET<sub>95</sub> than for PEF<sub>95</sub> in the same time range, indicating that PEF<sub>95</sub> is overall more resistant to CO<sub>2</sub> diffusion. Indeed, CO<sub>2</sub> diffusion we measured in the polymer center was nearly twice as high for PET<sub>95</sub> (7.8 ± 2.2 cm<sup>2</sup>/s × 10<sup>-8</sup>) versus PEF<sub>95</sub> (4.0 ± 0.3 cm<sup>2</sup>/s × 10<sup>-8</sup>), as described in Supplementary Methods: CO<sub>2</sub> Diffusion (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">1</a> and Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">6</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Symmetrized Z density profiles of CO2 in PET95 and PEF95." id="figure-8"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig8">Fig. 8: Symmetrized Z density profiles of CO<sub>2</sub> in PET<sub>95</sub> and PEF<sub>95</sub>.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-024-49700-4/figures/8" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig8_HTML.png?as=webp" type="image/webp"><img alt="figure 8" aria-describedby="Fig8" height="550" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_Fig8_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-8-desc"><p>Left and middle: mean ± SEM of 3 independent simulation replicas of the same plastic are plotted with a bin size of 2.46 Å. PEF (PEF<sub>95</sub> or polyethylene 2,5-furandicarboxylate) and CO<sub>2</sub> in PEF<sub>95</sub> values are shifted left by 2.46 Å to correct for changes in plastic thickness that occurred during equilibration in a vacuum. Bin heights are normalized to sum to 1. Right: simulation snapshots of PET (PET<sub>95</sub> or polyethylene terephthalate) and CO<sub>2</sub> taken after 1 ns and 2000 ns of NVT simulation.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 8" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure8 Full size image" data-track-label="button" href="/articles/s41467-024-49700-4/figures/8" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec13">Comparison with other programs</h3><p>Several existing programs facilitate densely packed macromolecular system modeling. For example, PACKMOL and cellPACK can pack molecules into complex shapes such as budding vesicles and viral capsids<sup><a aria-label="Reference 10" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR10" id="ref-link-section-d22328748e1722" title="Johnson, G. T. et al. 3D molecular models of whole HIV-1 virions generated with cellPACK. Faraday Discuss. 169, 23–44 (2014).">10</a>,<a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR11" id="ref-link-section-d22328748e1725" title="Johnson, G. T. et al. cellPACK: a virtual mesoscope to model and visualize structural systems biology. Nat. Methods 12, 85–91 (2015).">11</a>,<a aria-label="Reference 24" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR24" id="ref-link-section-d22328748e1728" title="Soñora, M., Martínez, L., Pantano, S. &amp; Machado, M. R. Wrapping up viruses at multiscale resolution: optimizing PACKMOL and SIRAH execution for simulating the zika virus. J. Chem. Inf. Model. 61, 408–422 (2021).">24</a></sup>; polyply and pysimm can model long polymer chains (e.g., DNA/RNA) densely packed with proteins in a cytoplasm<sup><a aria-label="Reference 29" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR29" id="ref-link-section-d22328748e1732" title="Fortunato, M. E. &amp; Colina, C. M. pysimm: a python package for simulation of molecular systems. SoftwareX 6, 7–12 (2017).">29</a>,<a aria-label="Reference 67" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR67" id="ref-link-section-d22328748e1735" title="Stevens, J. A. et al. Molecular dynamics simulation of an entire cell. Front. Chem. 11, 1106495 (2023).">67</a></sup>. Moltemplate can also model long polymers, but collision detection must be handled externally<sup><a aria-label="Reference 68" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR68" id="ref-link-section-d22328748e1739" title="Jewett, A. I. et al. Moltemplate: a tool for coarse-grained modeling of complex biological matter and soft condensed matter physics. J. Mol. Biol. 433, 166841 (2021).">68</a></sup>. Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">2</a> summarizes our analysis on the capabilities of each program in multicomponent assembly, which is elaborated below.</p><p>Pysimm is a Python API that facilitates modeling and simulation with LAMMPS. It can read molecular structures in several formats including PDB, XYZ, and MOL/MOL2. It can construct LAMMPS-compatible topologies and supports creating and positioning long polymers. Pysimm includes many functions that delegate common simulation tasks to LAMMPS, such as molecular dynamics and minimization. Pysimm is similar to pyCHARMM<sup><a aria-label="Reference 69" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR69" id="ref-link-section-d22328748e1749" title="Buckner, J. et al. pyCHARMM: embedding CHARMM functionality in a python framework. J. Chem. Theory. Comput. 19, 3752–3762 (2023).">69</a></sup> embedding CHARMM functionality in a Python framework.</p><p>Although moltemplate was designed for custom CG modeling, it has also been used in the preparation of all-atom models<sup><a aria-label="Reference 70" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR70" id="ref-link-section-d22328748e1756" title="Jewett, A. I. et al. Moltemplate: A tool for coarse-grained modeling of complex biological matter and soft condensed matter physics. J. Mol. Biol. 433, 166841 (2021).">70</a></sup>. However, external tools are required to select appropriate FF atom types and resolve collisions in prepared models. According to their website, moltemplate “is not suitable for all-atom protein simulations”. When utilizing moltemplate’s linear stacking method to generate initial polymer conformations, long equilibration simulations of tens to hundreds of nanoseconds are required before the melt is well relaxed. In contrast, the CHARMM-GUI Polymer Builder<sup><a aria-label="Reference 30" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR30" id="ref-link-section-d22328748e1760" title="Choi, Y. K. et al. CHARMM-GUI Polymer builder for modeling and simulation of synthetic polymers. J. Chem. Theory Comput. 17, 2431–2443 (2021).">30</a></sup> creates initial structures similar to fully relaxed configurations, allowing for direct production runs without long equilibration simulations. As described in our recent publication, the initial polymer configurations generated by CHARMM-GUI Polymer Builder exhibit structures similar to fully relaxed configurations, allowing for direct production runs without additional equilibration simulations. However, when utilizing moltemplate’s linear stacking method, additional equilibration simulations of tens to hundreds of nanoseconds are required even after the initial structure formation.</p><p>Polyply has been developed to generate input files and starting coordinates for polymeric molecules at CG and all-atom resolutions. However, it does not perform rigid body packing of large molecules and requires third-party software for certain biomolecules, similar to Moltemplate.</p><p>PACKMOL has been developed for finding collision-free rigid body packing solutions of arbitrary molecules in many non-periodic geometries. It implements an intuitive scripting format for describing the geometrical requirements and can read input coordinates from PDB, tinker, XYZ, and moldy formats. PACKMOL has quick runtimes with stable performance when using default settings. This makes it an ideal candidate for benchmark comparison with MCA’s packing algorithm. As both MCA and PACKMOL treat molecules as rigid bodies, neither software package is well-suited to mixing long polymers with proteins.</p><p>To test whether our packing approach can lead to more dense packing and lower runtimes even though the CHARMM executable is not optimized for packing, we selected two sets of macromolecules shown in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">3</a>: one where all molecules have asphericity between 0.07 – 0.15 (easy), and the other where asphericity varies between 0.02–0.48 (hard). We then ran 12 replicas of both macromolecule sets through PACKMOL and MCA’s packing procedure with volume fraction (v/v) starting from 10% and increasing by 1% until all 12 replicas failed at the same v/v. As shown in Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">1</a>, MCA’s runtimes were longer for low-density systems, and shorter for high-density systems. The maximum v/v achieved by PACKMOL was 30% (easy) and 18% (hard), and it was 41% (easy) and 23% (hard) for MCA. These results show that our approach can improve packing performance for periodic geometries, especially when the system density is high.</p><h3 class="c-article__sub-heading" id="Sec14">Limitations</h3><p>Although there is a large library of molecules that can be handled by MCA via other CHARMM-GUI modules, incorporating molecules that are not available in CHARMM-GUI—such as new membrane lipids, synthetic polymer building blocks, or ligands unsupported by CGenFF—could be challenging for users. Although users can download the CHARMM scripts used to generate a system, the scripts are often hundreds to thousands of lines long. While MCA allows users to upload their own CHARMM topologies (RTF) and parameters (PRM) for molecules not contained within the CHARMM36(m) or INTERFACE FFs, parameterizing molecules for the CHARMM FF is challenging. However, other programs (e.g., psfgen<sup><a aria-label="Reference 71" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR71" id="ref-link-section-d22328748e1788" title="Ribeiro, J. et al. VMD psfgen Plugin, Version 2.0. 
                  https://www.ks.uiuc.edu/Research/vmd/plugins/psfgen/
                  
                 (2020).">71</a></sup> for PSF; FFParam<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR12" id="ref-link-section-d22328748e1792" title="Kumar, A., Yoluk, O. &amp; MacKerell, A. D. FFParam: standalone package for CHARMM additive and drude polarizable force field parametrization of small molecules. J. Comput. Chem. 41, 958–970 (2020).">12</a></sup>, CGenFF<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR16" id="ref-link-section-d22328748e1796" title="Vanommeslaeghe, K. &amp; MacKerell, A. D. Jr. Automation of the CHARMM general force field (CGenFF) I: bond perception and atom typing. J. Chem. Inf. Model. 52, 3144–3154 (2012).">16</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR17" id="ref-link-section-d22328748e1796_1" title="Vanommeslaeghe, K., Raman, E. P. &amp; MacKerell, A. D. Jr. Automation of the CHARMM general force field (CGenFF) II: assignment of bonded parameters and partial atomic charges. J. Chem. Inf. Model. 52, 3155–3168 (2012).">17</a>,<a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR18" id="ref-link-section-d22328748e1799" title="Vanommeslaeghe, K. et al. CHARMM general force field: a force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields. J. Comput. Chem. 31, 671–690 (2010).">18</a></sup>) are capable of producing these topology and parameter files.</p><p>The rigid body packing used by MCA or PACKMOL tends to work well for packing problems involving macromolecules that are already fairly rigid, such as proteins, crystals, and short nucleic acid sequences. Packing long flexible molecules such as synthetic and nucleic acid polymers is unlikely to succeed because the molecule’s ability to pack tightly with other components comes from its flexibility. In such cases, the polymers should be assembled by another method, such as the random walk algorithms used by pysimm<sup><a aria-label="Reference 29" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR29" id="ref-link-section-d22328748e1806" title="Fortunato, M. E. &amp; Colina, C. M. pysimm: a python package for simulation of molecular systems. SoftwareX 6, 7–12 (2017).">29</a></sup> and polyply<sup><a aria-label="Reference 28" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR28" id="ref-link-section-d22328748e1810" title="Grünewald, F. et al. Polyply; a python suite for facilitating simulations of macromolecules and nanomaterials. Nat. Commun. 13, 68 (2022).">28</a></sup> or the CG Kuhn fragment equilibration method used by Polymer Builder<sup><a aria-label="Reference 30" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR30" id="ref-link-section-d22328748e1814" title="Choi, Y. K. et al. CHARMM-GUI Polymer builder for modeling and simulation of synthetic polymers. J. Chem. Theory Comput. 17, 2431–2443 (2021).">30</a></sup>.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec15-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec15">Discussion</h2><div class="c-article-section__content" id="Sec15-content"><p>This work presents a guided procedure for building simulation-ready, atomistic models containing heterogeneous molecular components via Multicomponent Assembler (MCA) in CHARMM-GUI. Initial positioning is facilitated by assigning component types that determine the packing strategy and using a greedy packing algorithm whose initial configuration is generated from CG simulation. The provided positioning options allow constraining components to a given position or orientation while allowing them to move within those constraints during packing.</p><p>Although MCA requires that input structures be provided in CHARMM format, other CHARMM-GUI tools—especially PDB Reader &amp; Manipulator<sup><a aria-label="Reference 56" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR56" id="ref-link-section-d22328748e1830" title="Park, S.-J., Kern, N., Brown, T., Lee, J. &amp; Im, W. CHARMM-GUI PDB Manipulator: various PDB structural modifications for biomolecular Modeling and Simulation. J. Mol. Biol. 435, 167995 (2023).">56</a></sup>, Ligand Reader &amp; Modeler<sup><a aria-label="Reference 21" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR21" id="ref-link-section-d22328748e1834" title="Kim, S. et al. CHARMM-GUI ligand reader and modeler for CHARMM force field generation of small molecules. J. Comput. Chem. 38, 1879–1886 (2017).">21</a></sup>, Glycan Reader &amp; Modeler<sup><a aria-label="Reference 57" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR57" id="ref-link-section-d22328748e1838" title="Park, S.-J. et al. Glycan reader is improved to recognize most sugar types and chemical modifications in the protein data bank. Bioinformatics 33, 3051–3057 (2017).">57</a></sup>, Nanomaterial Modeler<sup><a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR46" id="ref-link-section-d22328748e1842" title="Choi, Y. K. et al. CHARMM-GUI Nanomaterial modeler for modeling and simulation of nanomaterial systems. J. Chem. Theory Comput. 18, 479–493 (2022).">46</a></sup>, and Polymer Builder<sup><a aria-label="Reference 30" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR30" id="ref-link-section-d22328748e1846" title="Choi, Y. K. et al. CHARMM-GUI Polymer builder for modeling and simulation of synthetic polymers. J. Chem. Theory Comput. 17, 2431–2443 (2021).">30</a></sup>—can be used to prepare these initial structures. Furthermore, its integration with FF-Converter<sup><a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR31" id="ref-link-section-d22328748e1851" title="Lee, J. et al. CHARMM-GUI Input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field. J. Chem. Theory Comput. 12, 405–413 (2016).">31</a>,<a aria-label="Reference 32" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR32" id="ref-link-section-d22328748e1854" title="Lee, J. et al. CHARMM-GUI supports the Amber force fields. J. Chem. Phys. 153, 035103 (2020).">32</a></sup> allows MCA to provide simulation inputs using both CHARMM and AMBER FFs and for many other simulation programs, including NAMD<sup><a aria-label="Reference 36" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR36" id="ref-link-section-d22328748e1858" title="Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. J. Chem. Phys. 153, 044130 (2020).">36</a></sup>, GROMACS<sup><a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR34" id="ref-link-section-d22328748e1862" title="Páll, S., Abraham, M. J., Kutzner, C., Hess, B. &amp; Lindahl, E. Tackling exascale software challenges in molecular dynamics simulations with GROMACS. in Solving Software Challenges for Exascale (eds. Markidis, S. &amp; Laure, E.) 3–27 
                  https://doi.org/10.1007/978-3-319-15976-8_1
                  
                 (Springer International Publishing, Cham, 2015).">34</a>,<a aria-label="Reference 35" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR35" id="ref-link-section-d22328748e1865" title="Abraham, M. J. et al. GROMACS: high performance molecular simulations through multi-level parallelism from laptops to supercomputers. SoftwareX 1–2, 19–25 (2015).">35</a></sup>, Amber<sup><a aria-label="Reference 15" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR15" id="ref-link-section-d22328748e1869" title="Case, D. A. et al. AMBERTools. J. Chem. Inf. Model. 63, 6183–6191 (2023).">15</a></sup>, and OpenMM<sup><a aria-label="Reference 38" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR38" id="ref-link-section-d22328748e1873" title="Eastman, P. et al. OpenMM 7: rapid development of high performance algorithms for molecular dynamics. PLOS Comput. Biol. 13, e1005659 (2017).">38</a></sup>. MCA’s diverse modeling capabilities are demonstrated by building crowded solute-membrane interfaces, nano-bio interfaces, multi-membrane systems, and non-water solvents. Our modeling of polymer-bio and nano-bio interfaces is made possible by the CHARMM36(m)<sup><a aria-label="Reference 47" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR47" id="ref-link-section-d22328748e1877" title="Huang, J. et al. CHARMM36m: an improved force field for folded and intrinsically disordered proteins. Nat. Methods 14, 71–73 (2017).">47</a></sup> and INTERFACE FFs<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR50" id="ref-link-section-d22328748e1882" title="Lin, T.-J. &amp; Heinz, H. Accurate force field parameters and pH resolved surface models for hydroxyapatite to understand structure, mechanics, hydration, and biological interfaces. J. Phys. Chem. C. 120, 4975–4992 (2016).">50</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR51" id="ref-link-section-d22328748e1882_1" title="Heinz, H., Vaia, R. A., Farmer, B. L. &amp; Naik, R. R. Accurate simulation of surfaces and interfaces of face-centered cubic metals using 12−6 and 9−6 Lennard-Jones potentials. J. Phys. Chem. C. 112, 17281–17290 (2008).">51</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR52" id="ref-link-section-d22328748e1882_2" title="Emami, F. S. et al. Force field and a surface model database for Silica to simulate interfacial properties in atomic resolution. Chem. Mater. 26, 2647–2658 (2014).">52</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR53" id="ref-link-section-d22328748e1882_3" title="Mishra, R. K., Kanhaiya, K., Winetrout, J. J., Flatt, R. J. &amp; Heinz, H. Force field for calcium sulfate minerals to predict structural, hydration, and interfacial properties. Cem. Concr. Res. 139, 106262 (2021).">53</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR54" id="ref-link-section-d22328748e1882_4" title="Liu, J. et al. Interpretable molecular models for molybdenum disulfide and insight into selective peptide recognition. Chem. Sci. 11, 8708–8722 (2020).">54</a>,<a aria-label="Reference 55" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR55" id="ref-link-section-d22328748e1885" title="Heinz, H., Lin, T.-J., Kishore Mishra, R. &amp; Emami, F. S. Thermodynamically consistent force fields for the assembly of inorganic, organic, and biological nanostructures: the INTERFACE force field. Langmuir 29, 1754–1765 (2013).">55</a></sup> and greatly facilitated by Polymer Builder<sup><a aria-label="Reference 30" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR30" id="ref-link-section-d22328748e1889" title="Choi, Y. K. et al. CHARMM-GUI Polymer builder for modeling and simulation of synthetic polymers. J. Chem. Theory Comput. 17, 2431–2443 (2021).">30</a></sup> and Nanomaterial Modeler<sup><a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR46" id="ref-link-section-d22328748e1893" title="Choi, Y. K. et al. CHARMM-GUI Nanomaterial modeler for modeling and simulation of nanomaterial systems. J. Chem. Theory Comput. 18, 479–493 (2022).">46</a></sup>.</p><p>This work solves a PBC-aware packing problem for large, densely packed molecules. In combination with other CHARMM-GUI modules, MCA can generate complex molecular systems by combining many components. We hope that MCA can facilitate innovative studies of complex interactions between small (organic and inorganic) molecules, biomacromolecules, polymers, and nanomaterials.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec16-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec16">Methods</h2><div class="c-article-section__content" id="Sec16-content"><h3 class="c-article__sub-heading" id="Sec17">Test system preparation</h3><p>To test MCA’s ability to generate complex multicomponent systems reliably, we have built and simulated the systems shown in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-024-49700-4#Fig1">1</a>. Each system’s configuration is summarized in Supplementary Table 4<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">,</a> and the preparation is described below. Video demos of MCA usage for most of the below molecular configurations are available at <a href="https://charmm-gui.org/demo">https://charmm-gui.org/demo</a>. The FFs necessary for nanomaterial and polymer modeling<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR50" id="ref-link-section-d22328748e1925" title="Lin, T.-J. &amp; Heinz, H. Accurate force field parameters and pH resolved surface models for hydroxyapatite to understand structure, mechanics, hydration, and biological interfaces. J. Phys. Chem. C. 120, 4975–4992 (2016).">50</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR51" id="ref-link-section-d22328748e1925_1" title="Heinz, H., Vaia, R. A., Farmer, B. L. &amp; Naik, R. R. Accurate simulation of surfaces and interfaces of face-centered cubic metals using 12−6 and 9−6 Lennard-Jones potentials. J. Phys. Chem. C. 112, 17281–17290 (2008).">51</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR52" id="ref-link-section-d22328748e1925_2" title="Emami, F. S. et al. Force field and a surface model database for Silica to simulate interfacial properties in atomic resolution. Chem. Mater. 26, 2647–2658 (2014).">52</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR53" id="ref-link-section-d22328748e1925_3" title="Mishra, R. K., Kanhaiya, K., Winetrout, J. J., Flatt, R. J. &amp; Heinz, H. Force field for calcium sulfate minerals to predict structural, hydration, and interfacial properties. Cem. Concr. Res. 139, 106262 (2021).">53</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR54" id="ref-link-section-d22328748e1925_4" title="Liu, J. et al. Interpretable molecular models for molybdenum disulfide and insight into selective peptide recognition. Chem. Sci. 11, 8708–8722 (2020).">54</a>,<a aria-label="Reference 55" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR55" id="ref-link-section-d22328748e1928" title="Heinz, H., Lin, T.-J., Kishore Mishra, R. &amp; Emami, F. S. Thermodynamically consistent force fields for the assembly of inorganic, organic, and biological nanostructures: the INTERFACE force field. Langmuir 29, 1754–1765 (2013).">55</a></sup> are automatically included in the systems prepared by CHARMM-GUI. Note that protein-water interaction scaling previously used to reduce protein-protein interactions<sup><a aria-label="Reference 72" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR72" id="ref-link-section-d22328748e1932" title="Nawrocki, G., Wang, P., Yu, I., Sugita, Y. &amp; Feig, M. Slow-down in diffusion in crowded protein solutions correlates with transient cluster Formation. J. Phys. Chem. B 121, 11072–11084 (2017).">72</a></sup> was not used in this study.</p><h3 class="c-article__sub-heading" id="Sec18">Solution systems with three proteins (5, 10, 30% v/v)</h3><p>We built three systems described by Nawrocki et al.<sup><a aria-label="Reference 59" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR59" id="ref-link-section-d22328748e1944" title="Nawrocki, G., Im, W., Sugita, Y. &amp; Feig, M. Clustering and dynamics of crowded proteins near membranes and their influence on membrane bending. Proc. Natl Acad. Sci. 116, 24562–24567 (2019).">59</a></sup>, each containing ubiquitin (PDB ID: 1UBQ), villin (PDB ID: 1VII), and streptococcal G protein (PDB ID: 3GB1) in volume fractions (v/v) of 5%, 10%, and 30%, solvated by TIP3P water and 150 mM KCl (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">1</a>). Each system was simulated with OpenMM using the NVT (constant particle number, volume, and temperature) equilibration and NPT (constant particle number, pressure, and temperature) production protocols provided by CHARMM-GUI Solution Builder<sup><a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR31" id="ref-link-section-d22328748e1951" title="Lee, J. et al. CHARMM-GUI Input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field. J. Chem. Theory Comput. 12, 405–413 (2016).">31</a>,<a aria-label="Reference 32" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR32" id="ref-link-section-d22328748e1954" title="Lee, J. et al. CHARMM-GUI supports the Amber force fields. J. Chem. Phys. 153, 035103 (2020).">32</a>,<a aria-label="Reference 45" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR45" id="ref-link-section-d22328748e1957" title="Jo, S., Kim, T., Iyer, V. G. &amp; Im, W. CHARMM-GUI: a web-based graphical user interface for CHARMM. J. Comput. Chem. 29, 1859–1865 (2008).">45</a></sup> at 310.15 K with hydrogen mass repartitioning (HMR)<sup><a aria-label="Reference 73" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR73" id="ref-link-section-d22328748e1961" title="Gao, Y. et al. CHARMM-GUI Supports hydrogen mass repartitioning and different protonation states of phosphates in lipopolysaccharides. J. Chem. Inf. Model. 61, 831–839 (2021).">73</a></sup> for 1 µs. Preparation of a similar system is demonstrated in the video demo “Solvating Multiple Proteins” (<a href="https://charmm-gui.org/demo/multicomp/2">https://charmm-gui.org/demo/multicomp/2</a>). To rebuild these systems, use the corresponding volume fraction and component counts shown in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">4</a> on the STEP 1 page.</p><h3 class="c-article__sub-heading" id="Sec19">Membrane systems with three proteins (5, 10, 30% v/v)</h3><p>We built three systems containing the same proteins used in the solution systems, combined with a membrane containing an equal ratio of cholesterol (CHL1), 1-palmitoyl-2-oleoyl-phosphatidylcholine (POPC), and palmitoylsphingomyelin (PSM); see Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">4</a> for system information. Membranes were built de novo in MCA using the Membrane Builder protocol in STEP 3. Systems were simulated with OpenMM using the multi-step NVT/NPT equilibration and NPT production protocols provided by Membrane Builder<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR40" id="ref-link-section-d22328748e1987" title="Jo, S., Lim, J. B., Klauda, J. B. &amp; Im, W. CHARMM-GUI membrane builder for mixed bilayers and its application to yeast membranes. Biophys. J. 97, 50–58 (2009).">40</a>,<a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR41" id="ref-link-section-d22328748e1990" title="Lee, J. et al. CHARMM-GUI membrane builder for complex biological membrane simulations with Glycolipids and Lipoglycans. J. Chem. Theory Comput. 15, 775–786 (2019).">41</a></sup> at 310.15 K with HMR for 1 µs. As the CHL1/POPC/PSM membrane system was not pre-equilibrated, its size changed significantly throughout the first 100 ns of simulation. In each system, the <i>X</i> and <i>Y</i> axes shrank while the <i>Z</i> axis grew (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">7</a>). Preparation of a similar system is demonstrated in the video demo “Solvating Proteins with Membranes and Membrane Proteins” (<a href="https://charmm-gui.org/demo/multicomp/3">https://charmm-gui.org/demo/multicomp/3</a>). To rebuild these systems, use the corresponding volume fraction and component counts shown in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">4</a> on the STEP 1 page, and instead of uploading 5O8F (as shown in the demo), check the box labeled “Generate a membrane for this system” to enable the membrane size options.</p><h3 class="c-article__sub-heading" id="Sec20">Pre-equilibrated axolemma membrane systems with three proteins (5, 10, 30 % v/v)</h3><p>An axolemma membrane model that was simulated by Lee et al.<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR41" id="ref-link-section-d22328748e2025" title="Lee, J. et al. CHARMM-GUI membrane builder for complex biological membrane simulations with Glycolipids and Lipoglycans. J. Chem. Theory Comput. 15, 775–786 (2019).">41</a></sup> was prepared by manually removing water and ions from the last simulation frame. Its PSF/CRD files were then uploaded with those of 1UBQ, 1VII, and 3GB1. We used the solvated component type for the three proteins and the periodic type for the axolemma membrane. Dimensions for the axolemma were taken from the previous simulation (for X and Y) and the membrane’s Z dimension was estimated as ~50 Å. Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">4</a> summarizes the final system information. To speed up packing for the 5% and 10% v/v cases, protein components were excluded above and below 10 Å from the membrane region. Systems were simulated with OpenMM using the multi-step NVT/NPT equilibration and NPT production protocols provided by Membrane Builder<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR40" id="ref-link-section-d22328748e2032" title="Jo, S., Lim, J. B., Klauda, J. B. &amp; Im, W. CHARMM-GUI membrane builder for mixed bilayers and its application to yeast membranes. Biophys. J. 97, 50–58 (2009).">40</a>,<a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR41" id="ref-link-section-d22328748e2035" title="Lee, J. et al. CHARMM-GUI membrane builder for complex biological membrane simulations with Glycolipids and Lipoglycans. J. Chem. Theory Comput. 15, 775–786 (2019).">41</a></sup> at 310.15 K with HMR for 1 µs. Preparation of a similar system is demonstrated in the second example from video demo “Solvating Proteins with Membrane-like Polymers or Pre-Equilibrated Membranes” (<a href="https://charmm-gui.org/demo/multicomp/4">https://charmm-gui.org/demo/multicomp/4</a>) and from video demo “Building Nano-Bio Interface with Image Bonds” (<a href="https://charmm-gui.org/demo/multicomp/5">https://charmm-gui.org/demo/multicomp/5</a>). To rebuild these systems, use the corresponding volume fraction and component counts shown in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">4</a> on the STEP 1 page.</p><h3 class="c-article__sub-heading" id="Sec21">HAP with three proteins (5, 10, 30% v/v)</h3><p>We first constructed a hydroxyapatite (HAP) slab with a dimension of 103.6 Å × 114.2 Å× 42 Å at pH 10 using Nanomaterial Modeler<sup><a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR46" id="ref-link-section-d22328748e2066" title="Choi, Y. K. et al. CHARMM-GUI Nanomaterial modeler for modeling and simulation of nanomaterial systems. J. Chem. Theory Comput. 18, 479–493 (2022).">46</a></sup>. The HAP slab was then uploaded with the same proteins used in the solution systems. The solvated component type was used for the three proteins, and the periodic type was used for HAP. To achieve protein volume fractions of 5/10/30, the number of protein copies (2, 4, and 17 each) and system <i>Z</i> dimension were varied (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">1</a>). To speed up packing for all cases, protein components above and below 5 Å from the HAP region were excluded. After system assembly, 5000 steps of steepest descent (SD) minimization was followed by 5000 steps of adopted basis Newton-Raphson (ABNR) minimization using CHARMM. The systems were then simulated without HMR at 303.15 K for 1 µs using OpenMM. Preparation of a similar system is demonstrated in the second example from video demo “Solvating Proteins with Membrane-like Polymers or Pre-Equilibrated Membranes” (<a href="https://charmm-gui.org/demo/multicomp/4">https://charmm-gui.org/demo/multicomp/4</a>). To rebuild these systems, use the corresponding volume fraction and component counts shown in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">4</a> on the STEP 1 page.</p><h3 class="c-article__sub-heading" id="Sec22">Polymer EO<sub>40</sub>EE<sub>37</sub> with three proteins (5, 10, 30% v/v)</h3><p>We first constructed a polyethylene oxide-poly(ethylethylene) polymer slab (EO<sub>40</sub>EE<sub>37</sub>) in solution using Polymer Builder, as described by Choi et al.<sup><a aria-label="Reference 30" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR30" id="ref-link-section-d22328748e2103" title="Choi, Y. K. et al. CHARMM-GUI Polymer builder for modeling and simulation of synthetic polymers. J. Chem. Theory Comput. 17, 2431–2443 (2021).">30</a></sup>, with a thickness of 120 Å and a width of 107.9 Å. The slab was then uploaded with the same proteins used in the solution systems. The solvated component type and periodic type were used for the three proteins and the EO<sub>40</sub>EE<sub>37</sub>, respectively. To achieve protein volume fractions of 5/10/30, the number of protein copies (2, 4, and 13 each) and system <i>Z</i> dimension were varied (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">4</a>). To speed up packing for the 5% and 10% v/v cases, protein components were excluded above and below 10 Å from the polymer region. After system assembly, we performed 5000 steps of SD minimization followed by 5000 steps of ABNR minimization using CHARMM. Systems were then simulated without HMR at 303.15 K for 1 µs. Preparation of a similar system is demonstrated in the second example from video demo “Solvating Proteins with Membrane-like Polymers or Pre-Equilibrated Membranes” (<a href="https://charmm-gui.org/demo/multicomp/4">https://charmm-gui.org/demo/multicomp/4</a>). To rebuild these systems, use the corresponding volume fraction and component counts shown in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">4</a> on the STEP 1 page.</p><h3 class="c-article__sub-heading" id="Sec23">Diffusion of CO<sub>2</sub> through polymer membranes</h3><p>We used Polymer Builder to construct slabs containing polyethylene terephthalate (PET<sub>95</sub>) and polyethylene 2,5-furandicarboxylate (PEF<sub>95</sub>) with a monomer length of 95 each in a vacuum with a thickness of 100 Å and width near 100 Å. This resulted in 40 molecules of PET<sub>95</sub> with a width of 108.4 Å and 46 molecules of PEF<sub>95</sub> with a width of 108.4 Å. We ran equilibration and 500 ns of production for each slab separately in a vacuum using the default inputs provided by Polymer Builder at 298.15 K without HMR. We obtained a CO<sub>2</sub> structure from the CHARMM36 FF by Ligand Reader &amp; Modeler. To combine each polymer with CO<sub>2</sub>, we used the solvent component type for CO<sub>2</sub> and the periodic type for PET<sub>95</sub> or PEF<sub>95</sub>. Three replicas of each polymer  + CO<sub>2</sub> system were constructed (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">4</a>). Instead of a water solvent, we used the CO<sub>2</sub> to construct a gaseous solvent with pure CO<sub>2</sub> present at 1.98 g/L density. This resulted in 64 copies of CO<sub>2</sub> with PET<sub>95</sub> and 64 copies of CO<sub>2</sub> with PEF<sub>95</sub>. After system assembly, we performed 5000 steps of SD minimization followed by 5000 steps of ABNR minimization using CHARMM. Systems were then simulated without HMR at 298.15 K for 2 µs using OpenMM. Preparation of the PET<sub>95</sub> example is demonstrated in the third example of the video demo “Custom Solvent Composition, Gaseous Solvents” (<a href="https://charmm-gui.org/demo/multicomp/6">https://charmm-gui.org/demo/multicomp/6</a>). Preparation of the PEF<sub>95</sub> example is the same except that the PEF monomer is chosen during polymer generation.</p><h3 class="c-article__sub-heading" id="Sec24">Multi-layer system (mica + POPC membrane)</h3><p>To demonstrate the capability of MCA to build multi-layer models, we obtained a mica model of 103.8 Å× 108.2 Å× 29.9 Å from Nanomaterial Modeler and uploaded it to MCA using the periodic component type and selecting the option to generate a new membrane. To analyze the effect of mica on the membrane, we generated three such models by varying the Z position of the uploaded mica’s COM, Z<sub>1</sub> = 48.33 Å, Z<sub>2</sub> = 58.33 Å, and Z<sub>3</sub> = 68.33 Å, corresponding to 10 Å, 20 Å, and 30 Å initial separation between membrane head group atoms (initialized near Z = 19 Å) and mica. A pure POPC bilayer containing 165 lipids in each leaflet was constructed with the membrane centered at Z = 0. The systems were solvated with 0.15 M KCl followed by 100 steps of SD minimization in CHARMM and 5000 steps of minimization in OpenMM. Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">4</a> summarizes the final system information. The mica + POPC systems were equilibrated in a multistage procedure starting with 250 ps NVT simulation at 298.15 K with positional and dihedral restraints starting at 1000 kJ/mol/nm<sup>2</sup> and 1000 kJ/mol/rad<sup>2</sup> decreasing halfway to 400 kJ/mol/nm<sup>2</sup> and 400 kJ/mol/rad<sup>2</sup>. We then used an NPT ensemble at 298.15 K for 3.625 ns with restraints decreasing from 400 kJ/mol/nm2 and 200 kJ/mol/rad2 to 0 and pressure coupling applied in all dimensions at 1 atm, as shown in Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">8</a>. Finally, all systems were run for 1 µs under NVT ensemble at 298.15 K without HMR. The exact steps required to generate these examples in MCA are described in Supplementary Methods: Mica + POPC Generation.</p><h3 class="c-article__sub-heading" id="Sec25">MCA and PACKMOL packing benchmark environment</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec26">Computing environment</h4><p>We used one Dell XPS 8900 workstation with an Intel Core i7-6700 CPU (4 cores) and 8 GB of installed RAM. On the Ubuntu 22.04.2 LTS operating system, we installed PACKMOL 20.14.2 and CHARMM 48a1 (commit ID cb36cf5c6) from CHARMM’s development branch.</p><p>To set an appropriate number of simultaneous tests to run on the workstation, we observed CPU and memory usage of solo tests and determined that while PACKMOL and CHARMM consistently consumed 1 CPU at a rate near 100%, PACKMOL’s memory usage was uniformly below 9 MB, whereas CHARMM tests used 3.5 GB. To make the most of the workstation’s resources, we thus ran 4 simultaneous jobs when testing PACKMOL and 2 when testing CHARMM.</p><p>In all tests, target system densities were achieved by varying total system volume while keeping the number of molecules constant. All code used to run our benchmark can be found at <a href="https://github.com/charmm_gui/mca_scripts">https://github.com/charmm_gui/mca_scripts</a><sup><a aria-label="Reference 74" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR74" id="ref-link-section-d22328748e2241" title="Kern, N. R., Lee, J., Choi, Y. K. &amp; Im, W. CHARMM-GUI multicomponent assembler for modeling and simulation of complex multicomponent systems. MCA Scripts 
                  https://doi.org/10.5281/zenodo.11205908
                  
                 (2024).">74</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec27">MCA</h4><p>One MCA job was created on CHARMM-GUI for each set of macromolecules shown in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM1">3</a>. After selecting the number of molecules and a 10% volume fraction, we performed packing in the web GUI and downloaded the projects from the solvent options page without generating a solvent. Each macromolecule set’s project directory was copied to create 12 replicas (24 total). The directories were copied again to create one directory for each tested volume fraction at 1% intervals starting from 10% and increasing until all replicas failed at the same v/v. Random number generator (RNG) seeds were set automatically from CPU time.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec28">PACKMOL</h4><p>To avoid giving MCA an undue advantage, hydrogen atoms were stripped from PACKMOL’s input PDB files, as MCA ignores hydrogen atoms when checking for collisions. Similarly, the collision tolerance was set to 2.5 Å to match MCA’s tolerance. RNG seeds were set automatically from CPU time. Default values were used for all other settings. As with MCA, 12 replicas of each macromolecule set were tested at 1% v/v intervals starting from 10% and increasing until all replicas failed at the same v/v.</p><h3 class="c-article__sub-heading" id="Sec29">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-024-49700-4#MOESM3">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
</div>
<div class="u-mt-32">
<section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
<p>Initial simulation structures, simulation parameters, and programs used to produce figures are included in the GitHub repository described in Code Availability. Any additional data are available upon request. PDB structures referenced in this work: <a href="https://doi.org/10.2210/pdb1MJC/pdb">1MJC</a>, <a href="https://doi.org/10.2210/pdb1UBQ/pdb">1UBQ</a>, <a href="https://doi.org/10.2210/pdb1VII/pdb">1VII</a>, <a href="https://doi.org/10.2210/pdb2HAC/pdb">2HAC</a>, <a href="https://doi.org/10.2210/pdb3GB1/pdb">3GB1</a>, <a href="https://doi.org/10.2210/pdb6Y3G/pdb">6Y3G</a>. A source data archive (source_data.tgz) was added to the github repository at [<a href="https://github.com/charmmgui/mca_scripts/raw/main/source_data.tgz">https://github.com/charmmgui/mca_scripts/raw/main/source_data.tgz</a>]. Version of code submitted for publication can be found in the Zenodo repository [10.5281/zenodo.11205908].</p>
</div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
<p>CHARMM-GUI’s full licensing terms are shown here [<a href="https://charmm-gui.org/?doc=license_cgui">https://charmm-gui.org/?doc=license_cgui</a>]. CHARMM-GUI is free for academic and governmental use, but not for commercial use or any collaboration with commercial entities. These terms apply to the usage of our web service CHARMM-GUI and to any output files we provide (e.g., CHARMM scripts). Scripts used to manipulate PSF/CRD files before upload to MCA are available at [<a href="https://github.com/charmm-gui/mca_scripts">https://github.com/charmm-gui/mca_scripts</a>]<sup><a aria-label="Reference 74" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-024-49700-4#ref-CR74" id="ref-link-section-d22328748e2415" title="Kern, N. R., Lee, J., Choi, Y. K. &amp; Im, W. CHARMM-GUI multicomponent assembler for modeling and simulation of complex multicomponent systems. MCA Scripts 
                  https://doi.org/10.5281/zenodo.11205908
                  
                 (2024).">74</a></sup>.</p>
</div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Shibuta, Y. et al. Heterogeneity in homogeneous nucleation from billion-atom molecular dynamics simulation of solidification of pure metal. <i>Nat. Commun.</i> <b>8</b>, 10 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 1" data-doi="10.1038/s41467-017-00017-5" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-017-00017-5" data-track-label="10.1038/s41467-017-00017-5" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-017-00017-5" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 1" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017NatCo...8...10S" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 1" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28381864" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 1" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5431899" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 1" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Heterogeneity%20in%20homogeneous%20nucleation%20from%20billion-atom%20molecular%20dynamics%20simulation%20of%20solidification%20of%20pure%20metal&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-017-00017-5&amp;volume=8&amp;publication_year=2017&amp;author=Shibuta%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Hammerberg, J. E., Ravelo, R. J. &amp; Germann, T. C. Large-scale molecular dynamics studies of sliding friction in nanocrystalline aluminum. <i>AIP Conf. Proc.</i> <b>1979</b>, 050009 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 2" data-doi="10.1063/1.5044792" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1063/1.5044792" data-track-label="10.1063/1.5044792" data-track-value="article reference" href="https://doi.org/10.1063%2F1.5044792" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 2" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20molecular%20dynamics%20studies%20of%20sliding%20friction%20in%20nanocrystalline%20aluminum&amp;journal=AIP%20Conf.%20Proc.&amp;doi=10.1063%2F1.5044792&amp;volume=1979&amp;publication_year=2018&amp;author=Hammerberg%2CJE&amp;author=Ravelo%2CRJ&amp;author=Germann%2CTC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Frøseth, A. G., Van Swygenhoven, H. &amp; Derlet, P. M. Developing realistic grain boundary networks for use in molecular dynamics simulations. <i>Acta Mater.</i> <b>53</b>, 4847–4856 (2005).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 3" data-doi="10.1016/j.actamat.2005.06.032" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.actamat.2005.06.032" data-track-label="10.1016/j.actamat.2005.06.032" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.actamat.2005.06.032" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 3" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2005AcMat..53.4847F" rel="nofollow noopener">ADS</a> 
    <a aria-label="Google Scholar reference 3" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Developing%20realistic%20grain%20boundary%20networks%20for%20use%20in%20molecular%20dynamics%20simulations&amp;journal=Acta%20Mater.&amp;doi=10.1016%2Fj.actamat.2005.06.032&amp;volume=53&amp;pages=4847-4856&amp;publication_year=2005&amp;author=Fr%C3%B8seth%2CAG&amp;author=Swygenhoven%2CH&amp;author=Derlet%2CPM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Shibuta, Y., Sakane, S., Miyoshi, E., Takaki, T. &amp; Ohno, M. Micrometer-scale molecular dynamics simulation of microstructure formation linked with multi-phase-field simulation in same space scale. <i>Model. Simul. Mater. Sci. Eng.</i> <b>27</b>, 054002 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 4" data-doi="10.1088/1361-651X/ab1d28" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1088/1361-651X/ab1d28" data-track-label="10.1088/1361-651X/ab1d28" data-track-value="article reference" href="https://doi.org/10.1088%2F1361-651X%2Fab1d28" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 4" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019MSMSE..27e4002S" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 4" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitF2jsL%2FI" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 4" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Micrometer-scale%20molecular%20dynamics%20simulation%20of%20microstructure%20formation%20linked%20with%20multi-phase-field%20simulation%20in%20same%20space%20scale&amp;journal=Model.%20Simul.%20Mater.%20Sci.%20Eng.&amp;doi=10.1088%2F1361-651X%2Fab1d28&amp;volume=27&amp;publication_year=2019&amp;author=Shibuta%2CY&amp;author=Sakane%2CS&amp;author=Miyoshi%2CE&amp;author=Takaki%2CT&amp;author=Ohno%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Singharoy, A. et al. Atoms to phenotypes: molecular design principles of cellular energy metabolism. <i>Cell</i> <b>179</b>, 1098–1111 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 5" data-doi="10.1016/j.cell.2019.10.021" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2019.10.021" data-track-label="10.1016/j.cell.2019.10.021" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.10.021" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 5" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFymtb%2FN" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 5" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31730852" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 5" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7075482" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 5" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Atoms%20to%20phenotypes%3A%20molecular%20design%20principles%20of%20cellular%20energy%20metabolism&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.10.021&amp;volume=179&amp;pages=1098-1111&amp;publication_year=2019&amp;author=Singharoy%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Feig, M. et al. Complete atomistic model of a bacterial cytoplasm for integrating physics, biochemistry, and systems biology. <i>J. Mol. Graph. Model.</i> <b>58</b>, 1–9 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 6" data-doi="10.1016/j.jmgm.2015.02.004" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.jmgm.2015.02.004" data-track-label="10.1016/j.jmgm.2015.02.004" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.jmgm.2015.02.004" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 6" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015JMoSt1081....1F" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 6" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXjvVGgsbs%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 6" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25765281" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 6" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4388805" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 6" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Complete%20atomistic%20model%20of%20a%20bacterial%20cytoplasm%20for%20integrating%20physics%2C%20biochemistry%2C%20and%20systems%20biology&amp;journal=J.%20Mol.%20Graph.%20Model.&amp;doi=10.1016%2Fj.jmgm.2015.02.004&amp;volume=58&amp;pages=1-9&amp;publication_year=2015&amp;author=Feig%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Yu, I. et al. Biomolecular interactions modulate macromolecular structure and dynamics in atomistic model of a bacterial cytoplasm. <i>eLife</i> <b>5</b>, e19274 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 7" data-doi="10.7554/eLife.19274" data-track="click_references" data-track-action="article reference" data-track-item_id="10.7554/eLife.19274" data-track-label="10.7554/eLife.19274" data-track-value="article reference" href="https://doi.org/10.7554%2FeLife.19274" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 7" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27801646" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 7" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5089862" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 7" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Biomolecular%20interactions%20modulate%20macromolecular%20structure%20and%20dynamics%20in%20atomistic%20model%20of%20a%20bacterial%20cytoplasm&amp;journal=eLife&amp;doi=10.7554%2FeLife.19274&amp;volume=5&amp;publication_year=2016&amp;author=Yu%2CI" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Jung, J. et al. Scaling molecular dynamics beyond 100,000 processor cores for large-scale biophysical simulations. <i>J. Comput. Chem.</i> <b>40</b>, 1919–1930 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 8" data-doi="10.1002/jcc.25840" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/jcc.25840" data-track-label="10.1002/jcc.25840" data-track-value="article reference" href="https://doi.org/10.1002%2Fjcc.25840" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 8" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXnslektbw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 8" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30994934" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 8" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7153361" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 8" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Scaling%20molecular%20dynamics%20beyond%20100%2C000%20processor%20cores%20for%20large-scale%20biophysical%20simulations&amp;journal=J.%20Comput.%20Chem.&amp;doi=10.1002%2Fjcc.25840&amp;volume=40&amp;pages=1919-1930&amp;publication_year=2019&amp;author=Jung%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Wilhelm, B. G. et al. Composition of isolated synaptic boutons reveals the amounts of vesicle trafficking proteins. <i>Science</i> <b>344</b>, 1023–1028 (2014).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 9" data-doi="10.1126/science.1252884" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.1252884" data-track-label="10.1126/science.1252884" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.1252884" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 9" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2014Sci...344.1023W" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 9" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXosFSmurc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 9" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24876496" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 9" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Composition%20of%20isolated%20synaptic%20boutons%20reveals%20the%20amounts%20of%20vesicle%20trafficking%20proteins&amp;journal=Science&amp;doi=10.1126%2Fscience.1252884&amp;volume=344&amp;pages=1023-1028&amp;publication_year=2014&amp;author=Wilhelm%2CBG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Johnson, G. T. et al. 3D molecular models of whole HIV-1 virions generated with cellPACK. <i>Faraday Discuss.</i> <b>169</b>, 23–44 (2014).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 10" data-doi="10.1039/C4FD00017J" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1039/C4FD00017J" data-track-label="10.1039/C4FD00017J" data-track-value="article reference" href="https://doi.org/10.1039%2FC4FD00017J" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 10" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2014FaDi..169...23J" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 10" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhs1WjsLvN" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 10" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25253262" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 10" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4569901" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 10" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=3D%20molecular%20models%20of%20whole%20HIV-1%20virions%20generated%20with%20cellPACK&amp;journal=Faraday%20Discuss.&amp;doi=10.1039%2FC4FD00017J&amp;volume=169&amp;pages=23-44&amp;publication_year=2014&amp;author=Johnson%2CGT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Johnson, G. T. et al. cellPACK: a virtual mesoscope to model and visualize structural systems biology. <i>Nat. Methods</i> <b>12</b>, 85–91 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 11" data-doi="10.1038/nmeth.3204" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nmeth.3204" data-track-label="10.1038/nmeth.3204" data-track-value="article reference" href="https://doi.org/10.1038%2Fnmeth.3204" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 11" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXitVShtr7I" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 11" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25437435" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 11" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=cellPACK%3A%20a%20virtual%20mesoscope%20to%20model%20and%20visualize%20structural%20systems%20biology&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.3204&amp;volume=12&amp;pages=85-91&amp;publication_year=2015&amp;author=Johnson%2CGT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Kumar, A., Yoluk, O. &amp; MacKerell, A. D. FFParam: standalone package for CHARMM additive and drude polarizable force field parametrization of small molecules. <i>J. Comput. Chem.</i> <b>41</b>, 958–970 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 12" data-doi="10.1002/jcc.26138" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/jcc.26138" data-track-label="10.1002/jcc.26138" data-track-value="article reference" href="https://doi.org/10.1002%2Fjcc.26138" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 12" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFyrtw%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 12" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31886576" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 12" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=FFParam%3A%20standalone%20package%20for%20CHARMM%20additive%20and%20drude%20polarizable%20force%20field%20parametrization%20of%20small%20molecules&amp;journal=J.%20Comput.%20Chem.&amp;doi=10.1002%2Fjcc.26138&amp;volume=41&amp;pages=958-970&amp;publication_year=2020&amp;author=Kumar%2CA&amp;author=Yoluk%2CO&amp;author=MacKerell%2CAD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Mayne, C. G., Saam, J., Schulten, K., Tajkhorshid, E. &amp; Gumbart, J. C. Rapid parameterization of small molecules using the force field toolkit. <i>J. Comput. Chem.</i> <b>34</b>, 2757–2770 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 13" data-doi="10.1002/jcc.23422" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/jcc.23422" data-track-label="10.1002/jcc.23422" data-track-value="article reference" href="https://doi.org/10.1002%2Fjcc.23422" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 13" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtlyrt73J" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 13" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24000174" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 13" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Rapid%20parameterization%20of%20small%20molecules%20using%20the%20force%20field%20toolkit&amp;journal=J.%20Comput.%20Chem.&amp;doi=10.1002%2Fjcc.23422&amp;volume=34&amp;pages=2757-2770&amp;publication_year=2013&amp;author=Mayne%2CCG&amp;author=Saam%2CJ&amp;author=Schulten%2CK&amp;author=Tajkhorshid%2CE&amp;author=Gumbart%2CJC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Zoete, V., Cuendet, M. A., Grosdidier, A. &amp; Michielin, O. SwissParam: a fast force field generation tool for small organic molecules. <i>J. Comput. Chem.</i> <b>32</b>, 2359–2368 (2011).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 14" data-doi="10.1002/jcc.21816" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/jcc.21816" data-track-label="10.1002/jcc.21816" data-track-value="article reference" href="https://doi.org/10.1002%2Fjcc.21816" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 14" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXntlOqtrY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 14" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21541964" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 14" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=SwissParam%3A%20a%20fast%20force%20field%20generation%20tool%20for%20small%20organic%20molecules&amp;journal=J.%20Comput.%20Chem.&amp;doi=10.1002%2Fjcc.21816&amp;volume=32&amp;pages=2359-2368&amp;publication_year=2011&amp;author=Zoete%2CV&amp;author=Cuendet%2CMA&amp;author=Grosdidier%2CA&amp;author=Michielin%2CO" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Case, D. A. et al. AMBERTools. <i>J. Chem. Inf. Model</i>. <b>63</b>, 6183–6191 (2023).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Vanommeslaeghe, K. &amp; MacKerell, A. D. Jr. Automation of the CHARMM general force field (CGenFF) I: bond perception and atom typing. <i>J. Chem. Inf. Model.</i> <b>52</b>, 3144–3154 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 16" data-doi="10.1021/ci300363c" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/ci300363c" data-track-label="10.1021/ci300363c" data-track-value="article reference" href="https://doi.org/10.1021%2Fci300363c" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 16" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xhs1Gns7fL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 16" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23146088" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 16" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3528824" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 16" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Automation%20of%20the%20CHARMM%20general%20force%20field%20%28CGenFF%29%20I%3A%20bond%20perception%20and%20atom%20typing&amp;journal=J.%20Chem.%20Inf.%20Model.&amp;doi=10.1021%2Fci300363c&amp;volume=52&amp;pages=3144-3154&amp;publication_year=2012&amp;author=Vanommeslaeghe%2CK&amp;author=MacKerell%2CAD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Vanommeslaeghe, K., Raman, E. P. &amp; MacKerell, A. D. Jr. Automation of the CHARMM general force field (CGenFF) II: assignment of bonded parameters and partial atomic charges. <i>J. Chem. Inf. Model.</i> <b>52</b>, 3155–3168 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 17" data-doi="10.1021/ci3003649" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/ci3003649" data-track-label="10.1021/ci3003649" data-track-value="article reference" href="https://doi.org/10.1021%2Fci3003649" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 17" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xhs1Gns7fF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 17" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23145473" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 17" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3528813" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 17" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Automation%20of%20the%20CHARMM%20general%20force%20field%20%28CGenFF%29%20II%3A%20assignment%20of%20bonded%20parameters%20and%20partial%20atomic%20charges&amp;journal=J.%20Chem.%20Inf.%20Model.&amp;doi=10.1021%2Fci3003649&amp;volume=52&amp;pages=3155-3168&amp;publication_year=2012&amp;author=Vanommeslaeghe%2CK&amp;author=Raman%2CEP&amp;author=MacKerell%2CAD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Vanommeslaeghe, K. et al. CHARMM general force field: a force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields. <i>J. Comput. Chem.</i> <b>31</b>, 671–690 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 18" data-doi="10.1002/jcc.21367" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/jcc.21367" data-track-label="10.1002/jcc.21367" data-track-value="article reference" href="https://doi.org/10.1002%2Fjcc.21367" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 18" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlentbc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 18" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19575467" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 18" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2888302" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 18" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM%20general%20force%20field%3A%20a%20force%20field%20for%20drug-like%20molecules%20compatible%20with%20the%20CHARMM%20all-atom%20additive%20biological%20force%20fields&amp;journal=J.%20Comput.%20Chem.&amp;doi=10.1002%2Fjcc.21367&amp;volume=31&amp;pages=671-690&amp;publication_year=2010&amp;author=Vanommeslaeghe%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Yesselman, J. D., Price, D. J., Knight, J. L. &amp; Brooks III, C. L. MATCH: an atom-typing toolset for molecular mechanics force fields. <i>J. Comput. Chem.</i> <b>33</b>, 189–202 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 19" data-doi="10.1002/jcc.21963" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/jcc.21963" data-track-label="10.1002/jcc.21963" data-track-value="article reference" href="https://doi.org/10.1002%2Fjcc.21963" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 19" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtl2gsb7L" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 19" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22042689" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 19" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=MATCH%3A%20an%20atom-typing%20toolset%20for%20molecular%20mechanics%20force%20fields&amp;journal=J.%20Comput.%20Chem.&amp;doi=10.1002%2Fjcc.21963&amp;volume=33&amp;pages=189-202&amp;publication_year=2012&amp;author=Yesselman%2CJD&amp;author=Price%2CDJ&amp;author=Knight%2CJL&amp;author=Brooks%20III%2CCL" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Wagner, J. et al. openforcefield/openff-toolkit: 0.14.0 API deprecating and bugfix release. Zenodo <a data-track="click_references" data-track-action="external reference" data-track-label="10.5281/zenodo.8102071" data-track-value="external reference" href="https://doi.org/10.5281/zenodo.8102071">https://doi.org/10.5281/zenodo.8102071</a> (2023).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Kim, S. et al. CHARMM-GUI ligand reader and modeler for CHARMM force field generation of small molecules. <i>J. Comput. Chem.</i> <b>38</b>, 1879–1886 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 21" data-doi="10.1002/jcc.24829" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/jcc.24829" data-track-label="10.1002/jcc.24829" data-track-value="article reference" href="https://doi.org/10.1002%2Fjcc.24829" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 21" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXnslKhtLk%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 21" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28497616" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 21" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5488718" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 21" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM-GUI%20ligand%20reader%20and%20modeler%20for%20CHARMM%20force%20field%20generation%20of%20small%20molecules&amp;journal=J.%20Comput.%20Chem.&amp;doi=10.1002%2Fjcc.24829&amp;volume=38&amp;pages=1879-1886&amp;publication_year=2017&amp;author=Kim%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Martínez, L., Andrade, R., Birgin, E. G. &amp; Martínez, J. M. PACKMOL: a package for building initial configurations for molecular dynamics simulations. <i>J. Comput. Chem.</i> <b>30</b>, 2157–2164 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 22" data-doi="10.1002/jcc.21224" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/jcc.21224" data-track-label="10.1002/jcc.21224" data-track-value="article reference" href="https://doi.org/10.1002%2Fjcc.21224" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 22" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19229944" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 22" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=PACKMOL%3A%20a%20package%20for%20building%20initial%20configurations%20for%20molecular%20dynamics%20simulations&amp;journal=J.%20Comput.%20Chem.&amp;doi=10.1002%2Fjcc.21224&amp;volume=30&amp;pages=2157-2164&amp;publication_year=2009&amp;author=Mart%C3%ADnez%2CL&amp;author=Andrade%2CR&amp;author=Birgin%2CEG&amp;author=Mart%C3%ADnez%2CJM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Schott-Verdugo, S. &amp; Gohlke, H. PACKMOL-Memgen: a simple-to-use, generalized workflow for membrane-protein–lipid-bilayer system building. <i>J. Chem. Inf. Model.</i> <b>59</b>, 2522–2528 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 23" data-doi="10.1021/acs.jcim.9b00269" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jcim.9b00269" data-track-label="10.1021/acs.jcim.9b00269" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jcim.9b00269" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 23" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVWqtrvI" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 23" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31120747" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 23" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=PACKMOL-Memgen%3A%20a%20simple-to-use%2C%20generalized%20workflow%20for%20membrane-protein%E2%80%93lipid-bilayer%20system%20building&amp;journal=J.%20Chem.%20Inf.%20Model.&amp;doi=10.1021%2Facs.jcim.9b00269&amp;volume=59&amp;pages=2522-2528&amp;publication_year=2019&amp;author=Schott-Verdugo%2CS&amp;author=Gohlke%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Soñora, M., Martínez, L., Pantano, S. &amp; Machado, M. R. Wrapping up viruses at multiscale resolution: optimizing PACKMOL and SIRAH execution for simulating the zika virus. <i>J. Chem. Inf. Model.</i> <b>61</b>, 408–422 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 24" data-doi="10.1021/acs.jcim.0c01205" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jcim.0c01205" data-track-label="10.1021/acs.jcim.0c01205" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jcim.0c01205" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 24" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33415985" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 24" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Wrapping%20up%20viruses%20at%20multiscale%20resolution%3A%20optimizing%20PACKMOL%20and%20SIRAH%20execution%20for%20simulating%20the%20zika%20virus&amp;journal=J.%20Chem.%20Inf.%20Model.&amp;doi=10.1021%2Facs.jcim.0c01205&amp;volume=61&amp;pages=408-422&amp;publication_year=2021&amp;author=So%C3%B1ora%2CM&amp;author=Mart%C3%ADnez%2CL&amp;author=Pantano%2CS&amp;author=Machado%2CMR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Pezeshkian, W., König, M., Wassenaar, T. A. &amp; Marrink, S. J. Backmapping triangulated surfaces to coarse-grained membrane models. <i>Nat. Commun.</i> <b>11</b>, 2296 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 25" data-doi="10.1038/s41467-020-16094-y" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-020-16094-y" data-track-label="10.1038/s41467-020-16094-y" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-020-16094-y" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 25" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatCo..11.2296P" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 25" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXptVert74%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 25" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32385270" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 25" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7210967" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 25" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Backmapping%20triangulated%20surfaces%20to%20coarse-grained%20membrane%20models&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-16094-y&amp;volume=11&amp;publication_year=2020&amp;author=Pezeshkian%2CW&amp;author=K%C3%B6nig%2CM&amp;author=Wassenaar%2CTA&amp;author=Marrink%2CSJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Durrant, J. D. &amp; Amaro, R. E. LipidWrapper: an algorithm for generating large-scale membrane models of arbitrary geometry. <i>PLOS Comput. Biol.</i> <b>10</b>, e1003720 (2014).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 26" data-doi="10.1371/journal.pcbi.1003720" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1371/journal.pcbi.1003720" data-track-label="10.1371/journal.pcbi.1003720" data-track-value="article reference" href="https://doi.org/10.1371%2Fjournal.pcbi.1003720" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 26" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2014PLSCB..10E3720D" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 26" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25032790" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 26" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4102414" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 26" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=LipidWrapper%3A%20an%20algorithm%20for%20generating%20large-scale%20membrane%20models%20of%20arbitrary%20geometry&amp;journal=PLOS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1003720&amp;volume=10&amp;publication_year=2014&amp;author=Durrant%2CJD&amp;author=Amaro%2CRE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Oliveira Bortot, L., Bashardanesh, Z. &amp; van der Spoel, D. Making soup: preparing and validating models of the bacterial cytoplasm for molecular simulation. <i>J. Chem. Inf. Model</i>. <a data-track="click_references" data-track-action="external reference" data-track-label="10.1021/acs.jcim.9b00971" data-track-value="external reference" href="https://doi.org/10.1021/acs.jcim.9b00971">https://doi.org/10.1021/acs.jcim.9b00971</a> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Grünewald, F. et al. Polyply; a python suite for facilitating simulations of macromolecules and nanomaterials. <i>Nat. Commun.</i> <b>13</b>, 68 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 28" data-doi="10.1038/s41467-021-27627-4" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-021-27627-4" data-track-label="10.1038/s41467-021-27627-4" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-021-27627-4" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 28" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022NatCo..13...68G" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 28" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35013176" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 28" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8748707" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 28" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Polyply%3B%20a%20python%20suite%20for%20facilitating%20simulations%20of%20macromolecules%20and%20nanomaterials&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-27627-4&amp;volume=13&amp;publication_year=2022&amp;author=Gr%C3%BCnewald%2CF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Fortunato, M. E. &amp; Colina, C. M. pysimm: a python package for simulation of molecular systems. <i>SoftwareX</i> <b>6</b>, 7–12 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 29" data-doi="10.1016/j.softx.2016.12.002" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.softx.2016.12.002" data-track-label="10.1016/j.softx.2016.12.002" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.softx.2016.12.002" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 29" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017SoftX...6....7F" rel="nofollow noopener">ADS</a> 
    <a aria-label="Google Scholar reference 29" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=pysimm%3A%20a%20python%20package%20for%20simulation%20of%20molecular%20systems&amp;journal=SoftwareX&amp;doi=10.1016%2Fj.softx.2016.12.002&amp;volume=6&amp;pages=7-12&amp;publication_year=2017&amp;author=Fortunato%2CME&amp;author=Colina%2CCM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Choi, Y. K. et al. CHARMM-GUI Polymer builder for modeling and simulation of synthetic polymers. <i>J. Chem. Theory Comput.</i> <b>17</b>, 2431–2443 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 30" data-doi="10.1021/acs.jctc.1c00169" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jctc.1c00169" data-track-label="10.1021/acs.jctc.1c00169" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jctc.1c00169" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 30" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXnvVSjtr4%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 30" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33797913" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 30" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8078172" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 30" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM-GUI%20Polymer%20builder%20for%20modeling%20and%20simulation%20of%20synthetic%20polymers&amp;journal=J.%20Chem.%20Theory%20Comput.&amp;doi=10.1021%2Facs.jctc.1c00169&amp;volume=17&amp;pages=2431-2443&amp;publication_year=2021&amp;author=Choi%2CYK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Lee, J. et al. CHARMM-GUI Input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field. <i>J. Chem. Theory Comput.</i> <b>12</b>, 405–413 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 31" data-doi="10.1021/acs.jctc.5b00935" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jctc.5b00935" data-track-label="10.1021/acs.jctc.5b00935" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jctc.5b00935" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 31" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVWru7nI" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 31" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26631602" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 31" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM-GUI%20Input%20generator%20for%20NAMD%2C%20GROMACS%2C%20AMBER%2C%20OpenMM%2C%20and%20CHARMM%2FOpenMM%20simulations%20using%20the%20CHARMM36%20additive%20force%20field&amp;journal=J.%20Chem.%20Theory%20Comput.&amp;doi=10.1021%2Facs.jctc.5b00935&amp;volume=12&amp;pages=405-413&amp;publication_year=2016&amp;author=Lee%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Lee, J. et al. CHARMM-GUI supports the Amber force fields. <i>J. Chem. Phys.</i> <b>153</b>, 035103 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 32" data-doi="10.1063/5.0012280" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1063/5.0012280" data-track-label="10.1063/5.0012280" data-track-value="article reference" href="https://doi.org/10.1063%2F5.0012280" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 32" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVWrsbnL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 32" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32716185" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 32" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM-GUI%20supports%20the%20Amber%20force%20fields&amp;journal=J.%20Chem.%20Phys.&amp;doi=10.1063%2F5.0012280&amp;volume=153&amp;publication_year=2020&amp;author=Lee%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Shirts, M. R. et al. Lessons learned from comparing molecular dynamics engines on the SAMPL5 dataset. <i>J. Comput. Aided Mol. Des.</i> <b>31</b>, 147–161 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 33" data-doi="10.1007/s10822-016-9977-1" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1007/s10822-016-9977-1" data-track-label="10.1007/s10822-016-9977-1" data-track-value="article reference" href="https://link.springer.com/doi/10.1007/s10822-016-9977-1" rel="noopener">Article</a> 
    <a aria-label="ADS reference 33" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017JCAMD..31..147S" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 33" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhslGrurfK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 33" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27787702" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 33" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Lessons%20learned%20from%20comparing%20molecular%20dynamics%20engines%20on%20the%20SAMPL5%20dataset&amp;journal=J.%20Comput.%20Aided%20Mol.%20Des.&amp;doi=10.1007%2Fs10822-016-9977-1&amp;volume=31&amp;pages=147-161&amp;publication_year=2017&amp;author=Shirts%2CMR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Páll, S., Abraham, M. J., Kutzner, C., Hess, B. &amp; Lindahl, E. Tackling exascale software challenges in molecular dynamics simulations with GROMACS. in <i>Solving Software Challenges for Exascale</i> (eds. Markidis, S. &amp; Laure, E.) 3–27 <a data-track="click_references" data-track-action="external reference" data-track-label="10.1007/978-3-319-15976-8_1" data-track-value="external reference" href="https://doi.org/10.1007/978-3-319-15976-8_1">https://doi.org/10.1007/978-3-319-15976-8_1</a> (Springer International Publishing, Cham, 2015).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Abraham, M. J. et al. GROMACS: high performance molecular simulations through multi-level parallelism from laptops to supercomputers. <i>SoftwareX</i> <b>1–2</b>, 19–25 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 35" data-doi="10.1016/j.softx.2015.06.001" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.softx.2015.06.001" data-track-label="10.1016/j.softx.2015.06.001" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.softx.2015.06.001" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 35" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015SoftX...1...19A" rel="nofollow noopener">ADS</a> 
    <a aria-label="Google Scholar reference 35" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=GROMACS%3A%20high%20performance%20molecular%20simulations%20through%20multi-level%20parallelism%20from%20laptops%20to%20supercomputers&amp;journal=SoftwareX&amp;doi=10.1016%2Fj.softx.2015.06.001&amp;volume=1%E2%80%932&amp;pages=19-25&amp;publication_year=2015&amp;author=Abraham%2CMJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Phillips, J. C. et al. Scalable molecular dynamics on CPU and GPU architectures with NAMD. <i>J. Chem. Phys.</i> <b>153</b>, 044130 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 36" data-doi="10.1063/5.0014475" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1063/5.0014475" data-track-label="10.1063/5.0014475" data-track-value="article reference" href="https://doi.org/10.1063%2F5.0014475" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 36" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFajsL3J" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 36" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32752662" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 36" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7395834" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 36" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Scalable%20molecular%20dynamics%20on%20CPU%20and%20GPU%20architectures%20with%20NAMD&amp;journal=J.%20Chem.%20Phys.&amp;doi=10.1063%2F5.0014475&amp;volume=153&amp;publication_year=2020&amp;author=Phillips%2CJC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Humphrey, W., Dalke, A. &amp; Schulten, K. VMD: visual molecular dynamics. <i>J. Mol. Graph.</i> <b>14</b>, 33–38 (1996).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 37" data-doi="10.1016/0263-7855(96)00018-5" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/0263-7855(96)00018-5" data-track-label="10.1016/0263-7855(96)00018-5" data-track-value="article reference" href="https://doi.org/10.1016%2F0263-7855%2896%2900018-5" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 37" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK28Xis12nsrg%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 37" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8744570" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 37" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=VMD%3A%20visual%20molecular%20dynamics&amp;journal=J.%20Mol.%20Graph.&amp;doi=10.1016%2F0263-7855%2896%2900018-5&amp;volume=14&amp;pages=33-38&amp;publication_year=1996&amp;author=Humphrey%2CW&amp;author=Dalke%2CA&amp;author=Schulten%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Eastman, P. et al. OpenMM 7: rapid development of high performance algorithms for molecular dynamics. <i>PLOS Comput. Biol.</i> <b>13</b>, e1005659 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 38" data-doi="10.1371/journal.pcbi.1005659" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1371/journal.pcbi.1005659" data-track-label="10.1371/journal.pcbi.1005659" data-track-value="article reference" href="https://doi.org/10.1371%2Fjournal.pcbi.1005659" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 38" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28746339" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 38" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5549999" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 38" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=OpenMM%207%3A%20rapid%20development%20of%20high%20performance%20algorithms%20for%20molecular%20dynamics&amp;journal=PLOS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1005659&amp;volume=13&amp;publication_year=2017&amp;author=Eastman%2CP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Sommer, B. et al. CELLmicrocosmos 2.2 membraneEditor: a modular interactive shape-based software approach to solve heterogeneous membrane packing problems. <i>J. Chem. Inf. Model.</i> <b>51</b>, 1165–1182 (2011).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 39" data-doi="10.1021/ci1003619" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/ci1003619" data-track-label="10.1021/ci1003619" data-track-value="article reference" href="https://doi.org/10.1021%2Fci1003619" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 39" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXkvVCmurk%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 39" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21504163" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 39" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CELLmicrocosmos%202.2%20membraneEditor%3A%20a%20modular%20interactive%20shape-based%20software%20approach%20to%20solve%20heterogeneous%20membrane%20packing%20problems&amp;journal=J.%20Chem.%20Inf.%20Model.&amp;doi=10.1021%2Fci1003619&amp;volume=51&amp;pages=1165-1182&amp;publication_year=2011&amp;author=Sommer%2CB" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Jo, S., Lim, J. B., Klauda, J. B. &amp; Im, W. CHARMM-GUI membrane builder for mixed bilayers and its application to yeast membranes. <i>Biophys. J.</i> <b>97</b>, 50–58 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 40" data-doi="10.1016/j.bpj.2009.04.013" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.bpj.2009.04.013" data-track-label="10.1016/j.bpj.2009.04.013" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.bpj.2009.04.013" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 40" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2009BpJ....97...50J" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 40" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXpsFOmt7g%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 40" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19580743" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 40" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2711372" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 40" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM-GUI%20membrane%20builder%20for%20mixed%20bilayers%20and%20its%20application%20to%20yeast%20membranes&amp;journal=Biophys.%20J.&amp;doi=10.1016%2Fj.bpj.2009.04.013&amp;volume=97&amp;pages=50-58&amp;publication_year=2009&amp;author=Jo%2CS&amp;author=Lim%2CJB&amp;author=Klauda%2CJB&amp;author=Im%2CW" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Lee, J. et al. CHARMM-GUI membrane builder for complex biological membrane simulations with Glycolipids and Lipoglycans. <i>J. Chem. Theory Comput.</i> <b>15</b>, 775–786 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 41" data-doi="10.1021/acs.jctc.8b01066" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jctc.8b01066" data-track-label="10.1021/acs.jctc.8b01066" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jctc.8b01066" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 41" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVOitrbO" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 41" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30525595" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 41" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM-GUI%20membrane%20builder%20for%20complex%20biological%20membrane%20simulations%20with%20Glycolipids%20and%20Lipoglycans&amp;journal=J.%20Chem.%20Theory%20Comput.&amp;doi=10.1021%2Facs.jctc.8b01066&amp;volume=15&amp;pages=775-786&amp;publication_year=2019&amp;author=Lee%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Knight, C. J. &amp; Hub, J. S. MemGen: a general web server for the setup of lipid membrane simulation systems. <i>Bioinformatics</i> <b>31</b>, 2897–2899 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 42" data-doi="10.1093/bioinformatics/btv292" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/bioinformatics/btv292" data-track-label="10.1093/bioinformatics/btv292" data-track-value="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtv292" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 42" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1GitrnO" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 42" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25957354" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 42" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=MemGen%3A%20a%20general%20web%20server%20for%20the%20setup%20of%20lipid%20membrane%20simulation%20systems&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtv292&amp;volume=31&amp;pages=2897-2899&amp;publication_year=2015&amp;author=Knight%2CCJ&amp;author=Hub%2CJS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Wassenaar, T. A., Ingólfsson, H. I., Böckmann, R. A., Tieleman, D. P. &amp; Marrink, S. J. Computational lipidomics with insane: a versatile tool for generating custom membranes for molecular simulations. <i>J. Chem. Theory Comput.</i> <b>11</b>, 2144–2155 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 43" data-doi="10.1021/acs.jctc.5b00209" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jctc.5b00209" data-track-label="10.1021/acs.jctc.5b00209" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jctc.5b00209" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 43" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXmtlamsbo%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 43" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26574417" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 43" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Computational%20lipidomics%20with%20insane%3A%20a%20versatile%20tool%20for%20generating%20custom%20membranes%20for%20molecular%20simulations&amp;journal=J.%20Chem.%20Theory%20Comput.&amp;doi=10.1021%2Facs.jctc.5b00209&amp;volume=11&amp;pages=2144-2155&amp;publication_year=2015&amp;author=Wassenaar%2CTA&amp;author=Ing%C3%B3lfsson%2CHI&amp;author=B%C3%B6ckmann%2CRA&amp;author=Tieleman%2CDP&amp;author=Marrink%2CSJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Doerr, S., Giorgino, T., Martínez-Rosell, G., Damas, J. M. &amp; De Fabritiis, G. High-throughput automated preparation and simulation of membrane proteins with HTMD. <i>J. Chem. Theory Comput.</i> <b>13</b>, 4003–4011 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 44" data-doi="10.1021/acs.jctc.7b00480" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jctc.7b00480" data-track-label="10.1021/acs.jctc.7b00480" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jctc.7b00480" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 44" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1Sjt7zI" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 44" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28723224" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 44" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=High-throughput%20automated%20preparation%20and%20simulation%20of%20membrane%20proteins%20with%20HTMD&amp;journal=J.%20Chem.%20Theory%20Comput.&amp;doi=10.1021%2Facs.jctc.7b00480&amp;volume=13&amp;pages=4003-4011&amp;publication_year=2017&amp;author=Doerr%2CS&amp;author=Giorgino%2CT&amp;author=Mart%C3%ADnez-Rosell%2CG&amp;author=Damas%2CJM&amp;author=Fabritiis%2CG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Jo, S., Kim, T., Iyer, V. G. &amp; Im, W. CHARMM-GUI: a web-based graphical user interface for CHARMM. <i>J. Comput. Chem.</i> <b>29</b>, 1859–1865 (2008).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 45" data-doi="10.1002/jcc.20945" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/jcc.20945" data-track-label="10.1002/jcc.20945" data-track-value="article reference" href="https://doi.org/10.1002%2Fjcc.20945" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 45" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXosVKksbc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 45" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18351591" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 45" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM-GUI%3A%20a%20web-based%20graphical%20user%20interface%20for%20CHARMM&amp;journal=J.%20Comput.%20Chem.&amp;doi=10.1002%2Fjcc.20945&amp;volume=29&amp;pages=1859-1865&amp;publication_year=2008&amp;author=Jo%2CS&amp;author=Kim%2CT&amp;author=Iyer%2CVG&amp;author=Im%2CW" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Choi, Y. K. et al. CHARMM-GUI Nanomaterial modeler for modeling and simulation of nanomaterial systems. <i>J. Chem. Theory Comput.</i> <b>18</b>, 479–493 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 46" data-doi="10.1021/acs.jctc.1c00996" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jctc.1c00996" data-track-label="10.1021/acs.jctc.1c00996" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jctc.1c00996" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 46" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1Kgur3J" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 46" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34871001" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 46" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM-GUI%20Nanomaterial%20modeler%20for%20modeling%20and%20simulation%20of%20nanomaterial%20systems&amp;journal=J.%20Chem.%20Theory%20Comput.&amp;doi=10.1021%2Facs.jctc.1c00996&amp;volume=18&amp;pages=479-493&amp;publication_year=2022&amp;author=Choi%2CYK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Huang, J. et al. CHARMM36m: an improved force field for folded and intrinsically disordered proteins. <i>Nat. Methods</i> <b>14</b>, 71–73 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 47" data-doi="10.1038/nmeth.4067" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nmeth.4067" data-track-label="10.1038/nmeth.4067" data-track-value="article reference" href="https://doi.org/10.1038%2Fnmeth.4067" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 47" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhvVSiu77I" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 47" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27819658" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 47" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM36m%3A%20an%20improved%20force%20field%20for%20folded%20and%20intrinsically%20disordered%20proteins&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.4067&amp;volume=14&amp;pages=71-73&amp;publication_year=2017&amp;author=Huang%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Yu, W., He, X., Vanommeslaeghe, K. &amp; MacKerell, A. D. Jr. Extension of the CHARMM general force field to sulfonyl-containing compounds and its utility in biomolecular simulations. <i>J. Comput. Chem.</i> <b>33</b>, 2451–2468 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 48" data-doi="10.1002/jcc.23067" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/jcc.23067" data-track-label="10.1002/jcc.23067" data-track-value="article reference" href="https://doi.org/10.1002%2Fjcc.23067" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 48" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVKntL7J" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 48" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22821581" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 48" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3477297" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 48" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Extension%20of%20the%20CHARMM%20general%20force%20field%20to%20sulfonyl-containing%20compounds%20and%20its%20utility%20in%20biomolecular%20simulations&amp;journal=J.%20Comput.%20Chem.&amp;doi=10.1002%2Fjcc.23067&amp;volume=33&amp;pages=2451-2468&amp;publication_year=2012&amp;author=Yu%2CW&amp;author=He%2CX&amp;author=Vanommeslaeghe%2CK&amp;author=MacKerell%2CAD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Soteras Gutiérrez, I. et al. Parametrization of halogen bonds in the CHARMM general force field: Improved treatment of ligand–protein interactions. <i>Bioorg. Med. Chem.</i> <b>24</b>, 4812–4825 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 49" data-doi="10.1016/j.bmc.2016.06.034" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.bmc.2016.06.034" data-track-label="10.1016/j.bmc.2016.06.034" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.bmc.2016.06.034" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 49" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27353885" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 49" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5053860" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 49" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Parametrization%20of%20halogen%20bonds%20in%20the%20CHARMM%20general%20force%20field%3A%20Improved%20treatment%20of%20ligand%E2%80%93protein%20interactions&amp;journal=Bioorg.%20Med.%20Chem.&amp;doi=10.1016%2Fj.bmc.2016.06.034&amp;volume=24&amp;pages=4812-4825&amp;publication_year=2016&amp;author=Soteras%20Guti%C3%A9rrez%2CI" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Lin, T.-J. &amp; Heinz, H. Accurate force field parameters and pH resolved surface models for hydroxyapatite to understand structure, mechanics, hydration, and biological interfaces. <i>J. Phys. Chem. C.</i> <b>120</b>, 4975–4992 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 50" data-doi="10.1021/acs.jpcc.5b12504" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jpcc.5b12504" data-track-label="10.1021/acs.jpcc.5b12504" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jpcc.5b12504" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 50" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XislSgur0%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 50" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Accurate%20force%20field%20parameters%20and%20pH%20resolved%20surface%20models%20for%20hydroxyapatite%20to%20understand%20structure%2C%20mechanics%2C%20hydration%2C%20and%20biological%20interfaces&amp;journal=J.%20Phys.%20Chem.%20C.&amp;doi=10.1021%2Facs.jpcc.5b12504&amp;volume=120&amp;pages=4975-4992&amp;publication_year=2016&amp;author=Lin%2CT-J&amp;author=Heinz%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Heinz, H., Vaia, R. A., Farmer, B. L. &amp; Naik, R. R. Accurate simulation of surfaces and interfaces of face-centered cubic metals using 12−6 and 9−6 Lennard-Jones potentials. <i>J. Phys. Chem. C.</i> <b>112</b>, 17281–17290 (2008).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 51" data-doi="10.1021/jp801931d" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/jp801931d" data-track-label="10.1021/jp801931d" data-track-value="article reference" href="https://doi.org/10.1021%2Fjp801931d" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 51" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXht1ahs7zN" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 51" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Accurate%20simulation%20of%20surfaces%20and%20interfaces%20of%20face-centered%20cubic%20metals%20using%2012%E2%88%926%20and%209%E2%88%926%20Lennard-Jones%20potentials&amp;journal=J.%20Phys.%20Chem.%20C.&amp;doi=10.1021%2Fjp801931d&amp;volume=112&amp;pages=17281-17290&amp;publication_year=2008&amp;author=Heinz%2CH&amp;author=Vaia%2CRA&amp;author=Farmer%2CBL&amp;author=Naik%2CRR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Emami, F. S. et al. Force field and a surface model database for Silica to simulate interfacial properties in atomic resolution. <i>Chem. Mater.</i> <b>26</b>, 2647–2658 (2014).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 52" data-doi="10.1021/cm500365c" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/cm500365c" data-track-label="10.1021/cm500365c" data-track-value="article reference" href="https://doi.org/10.1021%2Fcm500365c" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 52" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXksVWis7s%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 52" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Force%20field%20and%20a%20surface%20model%20database%20for%20Silica%20to%20simulate%20interfacial%20properties%20in%20atomic%20resolution&amp;journal=Chem.%20Mater.&amp;doi=10.1021%2Fcm500365c&amp;volume=26&amp;pages=2647-2658&amp;publication_year=2014&amp;author=Emami%2CFS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Mishra, R. K., Kanhaiya, K., Winetrout, J. J., Flatt, R. J. &amp; Heinz, H. Force field for calcium sulfate minerals to predict structural, hydration, and interfacial properties. <i>Cem. Concr. Res.</i> <b>139</b>, 106262 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 53" data-doi="10.1016/j.cemconres.2020.106262" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cemconres.2020.106262" data-track-label="10.1016/j.cemconres.2020.106262" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cemconres.2020.106262" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 53" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1OmsL3F" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 53" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Force%20field%20for%20calcium%20sulfate%20minerals%20to%20predict%20structural%2C%20hydration%2C%20and%20interfacial%20properties&amp;journal=Cem.%20Concr.%20Res.&amp;doi=10.1016%2Fj.cemconres.2020.106262&amp;volume=139&amp;publication_year=2021&amp;author=Mishra%2CRK&amp;author=Kanhaiya%2CK&amp;author=Winetrout%2CJJ&amp;author=Flatt%2CRJ&amp;author=Heinz%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Liu, J. et al. Interpretable molecular models for molybdenum disulfide and insight into selective peptide recognition. <i>Chem. Sci.</i> <b>11</b>, 8708–8722 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 54" data-doi="10.1039/D0SC01443E" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1039/D0SC01443E" data-track-label="10.1039/D0SC01443E" data-track-value="article reference" href="https://doi.org/10.1039%2FD0SC01443E" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 54" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVWmtLbO" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 54" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34094188" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 54" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8162032" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 54" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Interpretable%20molecular%20models%20for%20molybdenum%20disulfide%20and%20insight%20into%20selective%20peptide%20recognition&amp;journal=Chem.%20Sci.&amp;doi=10.1039%2FD0SC01443E&amp;volume=11&amp;pages=8708-8722&amp;publication_year=2020&amp;author=Liu%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Heinz, H., Lin, T.-J., Kishore Mishra, R. &amp; Emami, F. S. Thermodynamically consistent force fields for the assembly of inorganic, organic, and biological nanostructures: the INTERFACE force field. <i>Langmuir</i> <b>29</b>, 1754–1765 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 55" data-doi="10.1021/la3038846" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/la3038846" data-track-label="10.1021/la3038846" data-track-value="article reference" href="https://doi.org/10.1021%2Fla3038846" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 55" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXjslKh" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 55" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23276161" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 55" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Thermodynamically%20consistent%20force%20fields%20for%20the%20assembly%20of%20inorganic%2C%20organic%2C%20and%20biological%20nanostructures%3A%20the%20INTERFACE%20force%20field&amp;journal=Langmuir&amp;doi=10.1021%2Fla3038846&amp;volume=29&amp;pages=1754-1765&amp;publication_year=2013&amp;author=Heinz%2CH&amp;author=Lin%2CT-J&amp;author=Kishore%20Mishra%2CR&amp;author=Emami%2CFS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Park, S.-J., Kern, N., Brown, T., Lee, J. &amp; Im, W. CHARMM-GUI PDB Manipulator: various PDB structural modifications for biomolecular Modeling and Simulation. <i>J. Mol. Biol</i>. <b>435</b>, 167995 (2023).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Park, S.-J. et al. Glycan reader is improved to recognize most sugar types and chemical modifications in the protein data bank. <i>Bioinformatics</i> <b>33</b>, 3051–3057 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 57" data-doi="10.1093/bioinformatics/btx358" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/bioinformatics/btx358" data-track-label="10.1093/bioinformatics/btx358" data-track-value="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtx358" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 57" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFGju7bO" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 57" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28582506" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 57" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5870669" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 57" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Glycan%20reader%20is%20improved%20to%20recognize%20most%20sugar%20types%20and%20chemical%20modifications%20in%20the%20protein%20data%20bank&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtx358&amp;volume=33&amp;pages=3051-3057&amp;publication_year=2017&amp;author=Park%2CS-J" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Jo, S., Kim, T. &amp; Im, W. Automated builder and database of protein/membrane complexes for molecular dynamics simulations. <i>PLOS ONE</i> <b>2</b>, e880 (2007).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 58" data-doi="10.1371/journal.pone.0000880" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1371/journal.pone.0000880" data-track-label="10.1371/journal.pone.0000880" data-track-value="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0000880" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 58" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2007PLoSO...2..880J" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 58" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17849009" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 58" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1963319" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 58" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Automated%20builder%20and%20database%20of%20protein%2Fmembrane%20complexes%20for%20molecular%20dynamics%20simulations&amp;journal=PLOS%20ONE&amp;doi=10.1371%2Fjournal.pone.0000880&amp;volume=2&amp;publication_year=2007&amp;author=Jo%2CS&amp;author=Kim%2CT&amp;author=Im%2CW" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Nawrocki, G., Im, W., Sugita, Y. &amp; Feig, M. Clustering and dynamics of crowded proteins near membranes and their influence on membrane bending. <i>Proc. Natl Acad. Sci.</i> <b>116</b>, 24562–24567 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 59" data-doi="10.1073/pnas.1910771116" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.1910771116" data-track-label="10.1073/pnas.1910771116" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.1910771116" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 59" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019PNAS..11624562N" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 59" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlejsLnK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 59" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31740611" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 59" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6900600" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 59" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Clustering%20and%20dynamics%20of%20crowded%20proteins%20near%20membranes%20and%20their%20influence%20on%20membrane%20bending&amp;journal=Proc.%20Natl%20Acad.%20Sci.&amp;doi=10.1073%2Fpnas.1910771116&amp;volume=116&amp;pages=24562-24567&amp;publication_year=2019&amp;author=Nawrocki%2CG&amp;author=Im%2CW&amp;author=Sugita%2CY&amp;author=Feig%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Macháň, R. &amp; Hof, M. Lipid diffusion in planar membranes investigated by fluorescence correlation spectroscopy. <i>Biochim. Biophys. Acta BBA - Biomembr.</i> <b>1798</b>, 1377–1391 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 60" data-doi="10.1016/j.bbamem.2010.02.014" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.bbamem.2010.02.014" data-track-label="10.1016/j.bbamem.2010.02.014" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.bbamem.2010.02.014" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 60" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Lipid%20diffusion%20in%20planar%20membranes%20investigated%20by%20fluorescence%20correlation%20spectroscopy&amp;journal=Biochim.%20Biophys.%20Acta%20BBA%20-%20Biomembr.&amp;doi=10.1016%2Fj.bbamem.2010.02.014&amp;volume=1798&amp;pages=1377-1391&amp;publication_year=2010&amp;author=Mach%C3%A1%C5%88%2CR&amp;author=Hof%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Giorgino, T. Computing diffusion coefficients in macromolecular simulations: the diffusion coefficient tool for VMD. <i>J. Open Source Softw.</i> <b>4</b>, 1698 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 61" data-doi="10.21105/joss.01698" data-track="click_references" data-track-action="article reference" data-track-item_id="10.21105/joss.01698" data-track-label="10.21105/joss.01698" data-track-value="article reference" href="https://doi.org/10.21105%2Fjoss.01698" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 61" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019JOSS....4.1698G" rel="nofollow noopener">ADS</a> 
    <a aria-label="Google Scholar reference 61" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Computing%20diffusion%20coefficients%20in%20macromolecular%20simulations%3A%20the%20diffusion%20coefficient%20tool%20for%20VMD&amp;journal=J.%20Open%20Source%20Softw.&amp;doi=10.21105%2Fjoss.01698&amp;volume=4&amp;publication_year=2019&amp;author=Giorgino%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Gielen, E. et al. Measuring diffusion of lipid-like probes in artificial and natural membranes by raster image correlation spectroscopy (RICS): use of a commercial laser-scanning microscope with analog detection. <i>Langmuir</i> <b>25</b>, 5209–5218 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 62" data-doi="10.1021/la8040538" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/la8040538" data-track-label="10.1021/la8040538" data-track-value="article reference" href="https://doi.org/10.1021%2Fla8040538" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 62" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXis12is7g%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 62" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19260653" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 62" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2728053" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 62" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Measuring%20diffusion%20of%20lipid-like%20probes%20in%20artificial%20and%20natural%20membranes%20by%20raster%20image%20correlation%20spectroscopy%20%28RICS%29%3A%20use%20of%20a%20commercial%20laser-scanning%20microscope%20with%20analog%20detection&amp;journal=Langmuir&amp;doi=10.1021%2Fla8040538&amp;volume=25&amp;pages=5209-5218&amp;publication_year=2009&amp;author=Gielen%2CE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Johnson, S. J. et al. Structure of an adsorbed dimyristoylphosphatidylcholine bilayer measured with specular reflection of neutrons. <i>Biophys. J.</i> <b>59</b>, 289–294 (1991).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 63" data-doi="10.1016/S0006-3495(91)82222-6" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/S0006-3495(91)82222-6" data-track-label="10.1016/S0006-3495(91)82222-6" data-track-value="article reference" href="https://doi.org/10.1016%2FS0006-3495%2891%2982222-6" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 63" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1991BpJ....59..289J" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 63" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK3MXhtl2hsb8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 63" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2009353" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 63" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1281145" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 63" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structure%20of%20an%20adsorbed%20dimyristoylphosphatidylcholine%20bilayer%20measured%20with%20specular%20reflection%20of%20neutrons&amp;journal=Biophys.%20J.&amp;doi=10.1016%2FS0006-3495%2891%2982222-6&amp;volume=59&amp;pages=289-294&amp;publication_year=1991&amp;author=Johnson%2CSJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Shen, L., Nieuwlaar, E., Worrell, E. &amp; Patel, M. K. Life cycle energy and GHG emissions of PET recycling: change-oriented effects. <i>Int. J. Life Cycle Assess.</i> <b>16</b>, 522–536 (2011).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 64" data-doi="10.1007/s11367-011-0296-4" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1007/s11367-011-0296-4" data-track-label="10.1007/s11367-011-0296-4" data-track-value="article reference" href="https://link.springer.com/doi/10.1007/s11367-011-0296-4" rel="noopener">Article</a> 
    <a aria-label="CAS reference 64" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXotFSksrk%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 64" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Life%20cycle%20energy%20and%20GHG%20emissions%20of%20PET%20recycling%3A%20change-oriented%20effects&amp;journal=Int.%20J.%20Life%20Cycle%20Assess.&amp;doi=10.1007%2Fs11367-011-0296-4&amp;volume=16&amp;pages=522-536&amp;publication_year=2011&amp;author=Shen%2CL&amp;author=Nieuwlaar%2CE&amp;author=Worrell%2CE&amp;author=Patel%2CMK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Eerhart, E. A. J. J., Faaij, A. P. C. &amp; Patel, M. K. Replacing fossil based PET with biobased PEF; process analysis, energy and GHG balance. <i>Energy Environ. Sci.</i> <b>5</b>, 6407–6422 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 65" data-doi="10.1039/c2ee02480b" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1039/c2ee02480b" data-track-label="10.1039/c2ee02480b" data-track-value="article reference" href="https://doi.org/10.1039%2Fc2ee02480b" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 65" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XksVKntLs%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 65" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Replacing%20fossil%20based%20PET%20with%20biobased%20PEF%3B%20process%20analysis%2C%20energy%20and%20GHG%20balance&amp;journal=Energy%20Environ.%20Sci.&amp;doi=10.1039%2Fc2ee02480b&amp;volume=5&amp;pages=6407-6422&amp;publication_year=2012&amp;author=Eerhart%2CEAJJ&amp;author=Faaij%2CAPC&amp;author=Patel%2CMK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Burgess, S. K., Kriegel, R. M. &amp; Koros, W. J. Carbon dioxide sorption and transport in amorphous poly(ethylene furanoate). <i>Macromolecules</i> <b>48</b>, 2184–2193 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 66" data-doi="10.1021/acs.macromol.5b00333" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.macromol.5b00333" data-track-label="10.1021/acs.macromol.5b00333" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.macromol.5b00333" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 66" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015MaMol..48.2184B" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 66" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXlslKitLc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 66" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Carbon%20dioxide%20sorption%20and%20transport%20in%20amorphous%20poly%28ethylene%20furanoate%29&amp;journal=Macromolecules&amp;doi=10.1021%2Facs.macromol.5b00333&amp;volume=48&amp;pages=2184-2193&amp;publication_year=2015&amp;author=Burgess%2CSK&amp;author=Kriegel%2CRM&amp;author=Koros%2CWJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Stevens, J. A. et al. Molecular dynamics simulation of an entire cell. <i>Front. Chem.</i> <b>11</b>, 1106495 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 67" data-doi="10.3389/fchem.2023.1106495" data-track="click_references" data-track-action="article reference" data-track-item_id="10.3389/fchem.2023.1106495" data-track-label="10.3389/fchem.2023.1106495" data-track-value="article reference" href="https://doi.org/10.3389%2Ffchem.2023.1106495" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 67" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2023FrCh...1106495S" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 67" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXivFKku74%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 67" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36742032" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 67" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9889929" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 67" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20dynamics%20simulation%20of%20an%20entire%20cell&amp;journal=Front.%20Chem.&amp;doi=10.3389%2Ffchem.2023.1106495&amp;volume=11&amp;publication_year=2023&amp;author=Stevens%2CJA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Jewett, A. I. et al. Moltemplate: a tool for coarse-grained modeling of complex biological matter and soft condensed matter physics. <i>J. Mol. Biol.</i> <b>433</b>, 166841 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 68" data-doi="10.1016/j.jmb.2021.166841" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.jmb.2021.166841" data-track-label="10.1016/j.jmb.2021.166841" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.jmb.2021.166841" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 68" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXjs1yrsLk%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 68" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33539886" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 68" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8119336" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 68" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Moltemplate%3A%20a%20tool%20for%20coarse-grained%20modeling%20of%20complex%20biological%20matter%20and%20soft%20condensed%20matter%20physics&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2Fj.jmb.2021.166841&amp;volume=433&amp;publication_year=2021&amp;author=Jewett%2CAI" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Buckner, J. et al. pyCHARMM: embedding CHARMM functionality in a python framework. <i>J. Chem. Theory. Comput</i>. <b>19</b>, 3752–3762 (2023).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Jewett, A. I. et al. Moltemplate: A tool for coarse-grained modeling of complex biological matter and soft condensed matter physics. <i>J. Mol. Biol</i>. <b>433</b>, 166841 (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Ribeiro, J. et al. VMD psfgen Plugin, Version 2.0. <a data-track="click_references" data-track-action="external reference" data-track-label="https://www.ks.uiuc.edu/Research/vmd/plugins/psfgen/" data-track-value="external reference" href="https://www.ks.uiuc.edu/Research/vmd/plugins/psfgen/">https://www.ks.uiuc.edu/Research/vmd/plugins/psfgen/</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Nawrocki, G., Wang, P., Yu, I., Sugita, Y. &amp; Feig, M. Slow-down in diffusion in crowded protein solutions correlates with transient cluster Formation. <i>J. Phys. Chem. B</i> <b>121</b>, 11072–11084 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 72" data-doi="10.1021/acs.jpcb.7b08785" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jpcb.7b08785" data-track-label="10.1021/acs.jpcb.7b08785" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jpcb.7b08785" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 72" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvVarsr7P" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 72" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29151345" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 72" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5951686" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 72" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Slow-down%20in%20diffusion%20in%20crowded%20protein%20solutions%20correlates%20with%20transient%20cluster%20Formation&amp;journal=J.%20Phys.%20Chem.%20B&amp;doi=10.1021%2Facs.jpcb.7b08785&amp;volume=121&amp;pages=11072-11084&amp;publication_year=2017&amp;author=Nawrocki%2CG&amp;author=Wang%2CP&amp;author=Yu%2CI&amp;author=Sugita%2CY&amp;author=Feig%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Gao, Y. et al. CHARMM-GUI Supports hydrogen mass repartitioning and different protonation states of phosphates in lipopolysaccharides. <i>J. Chem. Inf. Model.</i> <b>61</b>, 831–839 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 73" data-doi="10.1021/acs.jcim.0c01360" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jcim.0c01360" data-track-label="10.1021/acs.jcim.0c01360" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jcim.0c01360" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 73" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVOgu7k%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 73" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33442985" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 73" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7902386" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 73" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM-GUI%20Supports%20hydrogen%20mass%20repartitioning%20and%20different%20protonation%20states%20of%20phosphates%20in%20lipopolysaccharides&amp;journal=J.%20Chem.%20Inf.%20Model.&amp;doi=10.1021%2Facs.jcim.0c01360&amp;volume=61&amp;pages=831-839&amp;publication_year=2021&amp;author=Gao%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Kern, N. R., Lee, J., Choi, Y. K. &amp; Im, W. CHARMM-GUI multicomponent assembler for modeling and simulation of complex multicomponent systems. MCA Scripts <a data-track="click_references" data-track-action="external reference" data-track-label="10.5281/zenodo.11205908" data-track-value="external reference" href="https://doi.org/10.5281/zenodo.11205908">https://doi.org/10.5281/zenodo.11205908</a> (2024).</p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1038/s41467-024-49700-4?format=refman&amp;flavour=references" rel="nofollow">Download references<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-download-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>This work has been supported by NIH GM138472 and NSF OAC-1931343.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Department of Computer Science &amp; Engineering, Lehigh University, Bethlehem, PA, USA</p><p class="c-article-author-affiliation__authors-list">Nathan R. Kern &amp; Wonpil Im</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Department of Biological Sciences, Lehigh University, Bethlehem, PA, USA</p><p class="c-article-author-affiliation__authors-list">Jumin Lee, Yeol Kyo Choi &amp; Wonpil Im</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Department of Bioengineering, Lehigh University, Bethlehem, PA, USA</p><p class="c-article-author-affiliation__authors-list">Wonpil Im</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Nathan_R_-Kern-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Nathan R. Kern</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Nathan%20R.%20Kern" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Nathan%20R.%20Kern" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Nathan%20R.%20Kern%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jumin-Lee-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Jumin Lee</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Jumin%20Lee" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jumin%20Lee" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jumin%20Lee%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Yeol_Kyo-Choi-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Yeol Kyo Choi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Yeol%20Kyo%20Choi" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Yeol%20Kyo%20Choi" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Yeol%20Kyo%20Choi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Wonpil-Im-Aff1-Aff2-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Wonpil Im</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Wonpil%20Im" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Wonpil%20Im" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Wonpil%20Im%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>N.K., Y.K.C., J.L., and W.I. designed the research. N.K. performed the research, wrote the software, and drafted the paper with input from the co-authors. Y.K.C. wrote the text for POPC diffusion near mica-supported lipid bilayers. J.L. helped with FF Converter integration. Y.K.C. and J.L. recommended appropriate default settings for simulation inputs. Y.K.C., J.L., and W.I. helped with the interpretation of results and substantially revised the manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a href="mailto:wonpil@lehigh.edu" id="corresp-c1">Wonpil Im</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
<h3 class="c-article__sub-heading" id="FPar2">Competing interests</h3>
<p>W.I. is the co-founder and CEO of MolCube INC. J.L. is the co-founder and CTO of MolCube INC. The remaining authors declare no competing interests.</p>
</div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
<h3 class="c-article__sub-heading" id="FPar1">Peer review information</h3>
<p><i>Nature Communications</i> thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p>
</div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec30-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec30">Supplementary information</h2><div class="c-article-section__content" id="Sec30-content"><div data-test="supplementary-info"><div class="c-article-figshare-container" data-test="figshare-container" id="figshareContainer"></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_MOESM1_ESM.pdf">Supplementary Information</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="peer review file" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_MOESM2_ESM.pdf">Peer Review File</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-49700-4/MediaObjects/41467_2024_49700_MOESM3_ESM.pdf">Reporting Summary</a></h3></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
<p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=CHARMM-GUI%20Multicomponent%20Assembler%20for%20modeling%20and%20simulation%20of%20complex%20multicomponent%20systems&amp;author=Nathan%20R.%20Kern%20et%20al&amp;contentID=10.1038%2Fs41467-024-49700-4&amp;copyright=The%20Author%28s%29&amp;publication=2041-1723&amp;publicationDate=2024-06-27&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41467-024-49700-4" data-test="crossmark" data-track="click" data-track-action="Click Crossmark" data-track-label="link" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41467-024-49700-4" rel="noopener" target="_blank"><img alt="Check for updates. Verify currency and authenticity via CrossMark" height="81" loading="lazy" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>" width="57"/></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Kern, N.R., Lee, J., Choi, Y.K. <i>et al.</i> CHARMM-GUI Multicomponent Assembler for modeling and simulation of complex multicomponent systems.
                    <i>Nat Commun</i> <b>15</b>, 5459 (2024). https://doi.org/10.1038/s41467-024-49700-4</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-external="" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1038/s41467-024-49700-4?format=refman&amp;flavour=citation" rel="nofollow">Download citation<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-download-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2023-09-13">13 September 2023</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2024-06-17">17 June 2024</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2024-06-27">27 June 2024</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41467-024-49700-4</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" data-track="click" data-track-action="get shareable link" data-track-external="" data-track-label="button" id="get-share-url" type="button">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" data-track="click" data-track-action="select share url" data-track-label="button" id="share-url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" data-track="click" data-track-action="copy share url" data-track-external="" data-track-label="button" id="copy-share-url" type="button">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
</div>
<section>
<div class="c-article-section js-article-section" data-test="further-reading-section" id="further-reading-section">
<h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
<div class="c-article-section__content js-collapsible-section" id="further-reading-content">
<ul class="c-article-further-reading__list" id="further-reading-list">
<li class="c-article-further-reading__item js-ref-item">
<h3 class="c-article-further-reading__title" data-test="article-title">
<a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:c-di-GMP inhibits rRNA methylation and impairs ribosome assembly in the presence of kanamycin" href="https://doi.org/10.1038/s44319-025-00377-w">
                                        c-di-GMP inhibits rRNA methylation and impairs ribosome assembly in the presence of kanamycin
                                    </a>
</h3>
<ul class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto" data-test="author-list">
<li>Siqi Yu</li><li>Zheyao Hu</li><li>Zhaowei Xu</li>
</ul>
<p class="c-article-further-reading__journal-title"><i>EMBO Reports</i> (2025)</p>
</li>
<li class="c-article-further-reading__item js-ref-item">
<h3 class="c-article-further-reading__title" data-test="article-title">
<a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Sticky Business: Correlating Oligomeric Features of Class B Scavenger Receptors to Lipid Transport" href="https://doi.org/10.1007/s11883-024-01260-0">
                                        Sticky Business: Correlating Oligomeric Features of Class B Scavenger Receptors to Lipid Transport
                                    </a>
</h3>
<ul class="c-author-list c-author-list--compact u-sans-serif u-mb-4 u-mt-auto" data-test="author-list">
<li>Emma A. Tillison</li><li>Daisy Sahoo</li>
</ul>
<p class="c-article-further-reading__journal-title"><i>Current Atherosclerosis Reports</i> (2024)</p>
</li>
</ul>
</div>
</div>
</section>
</div>
</article>
</main>
<aside aria-label="Article navigation" class="c-article-extras u-hide-print" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
<div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41467-024-49700-4.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
<div class="c-reading-companion">
<div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
<div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
<div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu="">
<div class="c-ad c-ad--300x250">
<div class="c-ad__inner">
<p class="c-ad__label">Advertisement</p>
<div class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-gpt="" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41467-024-49700-4;doi=10.1038/s41467-024-49700-4;techmeta=118,119,139,9;subjmeta=114,2266,2398,45,57,631;kwrd=Biochemistry,Computational+biophysics,Computational+platforms+and+environments" data-gpt-unitpath="/285/nature_communications/article" data-pa11y-ignore="" data-test="right-ad" id="div-gpt-ad-right-2">
<noscript>
<a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature_communications/article&amp;sz=300x250&amp;c=-1474229589&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41467-024-49700-4%26doi%3D10.1038/s41467-024-49700-4%26techmeta%3D118,119,139,9%26subjmeta%3D114,2266,2398,45,57,631%26kwrd%3DBiochemistry,Computational+biophysics,Computational+platforms+and+environments">
<img alt="Advertisement" data-test="gpt-advert-fallback-img" height="250" src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature_communications/article&amp;sz=300x250&amp;c=-1474229589&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41467-024-49700-4%26doi%3D10.1038/s41467-024-49700-4%26techmeta%3D118,119,139,9%26subjmeta%3D114,2266,2398,45,57,631%26kwrd%3DBiochemistry,Computational+biophysics,Computational+platforms+and+environments" width="300"/></a>
</noscript>
</div>
</div>
</div>
</div>
</div>
<div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
<div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
</div>
</div>
</aside>
</div>
<nav aria-labelledby="Explore-content" class="c-header__dropdown" data-test="Explore-content" data-track-component="nature-150-split-header" id="explore">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="Explore-content">Explore content</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="research articles" data-track-label="link" href="/ncomms/research-articles">
                                    Research articles
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" href="/ncomms/reviews-and-analysis">
                                    Reviews &amp; Analysis
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="news &amp; comment" data-track-label="link" href="/ncomms/news-and-comment">
                                    News &amp; Comment
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="videos" data-track-label="link" href="/ncomms/video">
                                    Videos
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="collections" data-track-label="link" href="/ncomms/collections">
                                    Collections
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="subjects" data-track-label="link" href="/ncomms/browse-subjects">
                                    Subjects
                                </a>
</li>
</ul>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="facebook" data-track-label="link" href="https://www.facebook.com/NatureCommunications">Follow us on Facebook
                            </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="twitter" data-track-label="link" href="https://twitter.com/NatureComms">Follow us on Twitter
                            </a>
</li>
<li class="c-header__item c-header__item--hide-lg">
<a class="c-header__link" data-track="click" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=264" rel="nofollow">Sign up for alerts<svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--hide-lg">
<a class="c-header__link" data-track="click" data-track-action="rss feed" data-track-label="link" href="https://www.nature.com/ncomms.rss">
<span>RSS feed</span>
</a>
</li>
</ul>
</div>
</nav>
<nav aria-labelledby="About-the-journal" class="c-header__dropdown" data-test="about-the-journal" data-track-component="nature-150-split-header" id="about-the-journal">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="About-the-journal">About the journal</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="aims &amp; scope" data-track-label="link" href="/ncomms/aims">
                                    Aims &amp; Scope
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editors" data-track-label="link" href="/ncomms/editors">
                                    Editors
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="journal information" data-track-label="link" href="/ncomms/journal-information">
                                    Journal Information
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="open access fees and funding" data-track-label="link" href="/ncomms/open-access">
                                    Open Access Fees and Funding
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="calls for papers" data-track-label="link" href="/ncomms/calls-for-papers">
                                    Calls for Papers
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editorial values statement" data-track-label="link" href="/ncomms/editorial-values-statement">
                                    Editorial Values Statement
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="journal metrics" data-track-label="link" href="/ncomms/journal-impact">
                                    Journal Metrics
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editors' highlights" data-track-label="link" href="/ncomms/editorshighlights">
                                    Editors' Highlights
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="contact" data-track-label="link" href="/ncomms/contact">
                                    Contact
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editorial policies" data-track-label="link" href="/ncomms/editorial-policies">
                                    Editorial policies
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="top articles" data-track-label="link" href="/ncomms/top-articles">
                                    Top Articles
                                </a>
</li>
</ul>
</div>
</nav>
<nav aria-labelledby="Publish-with-us-label" class="c-header__dropdown" data-test="publish-with-us" data-track-component="nature-150-split-header" id="publish-with-us">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="Publish-with-us-label">Publish with us</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="for authors" data-track-label="link" href="/ncomms/submit">
                                    For authors
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="for reviewers" data-track-label="link" href="/ncomms/for-reviewers">
                                    For Reviewers
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-action="manuscript author services" data-track-context="header publish with us dropdown menu" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
</li>
<li class="c-header__item c-header__item--keyline">
<a class="c-header__link" data-gtm-criteo="submit-manuscript" data-track="click_submit_manuscript" data-track-action="submit manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-external="" data-track-label="link (publish with us dropdown menu)" href="https://mts-ncomms.nature.com/">Submit manuscript<svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
</a>
</li>
</ul>
</div>
</nav>
<div class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header" id="search-menu">
<div class="c-header__container">
<h2 class="c-header__visually-hidden">Search</h2>
<form action="/search" autocomplete="off" class="c-header__search-form" data-test="inline-search" method="get" role="search">
<label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
<div class="c-header__search-layout c-header__search-layout--max-width">
<div>
<input class="c-header__input" id="keywords" name="q" required="" type="text" value=""/>
</div>
<div class="c-header__search-layout">
<div>
<label class="c-header__visually-hidden" for="results-from">Show results from</label>
<select class="c-header__select" id="results-from" name="journal">
<option selected="" value="">All journals</option>
<option value="ncomms">This journal</option>
</select>
</div>
<div>
<button class="c-header__search-button" type="submit">Search</button>
</div>
</div>
</div>
</form>
<div class="c-header__flush">
<a class="c-header__link" data-track="click" data-track-action="advanced search" data-track-label="link" href="/search/advanced">
                Advanced search
            </a>
</div>
<h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
<ul class="c-header__list">
<li><a class="c-header__link" data-track="click" data-track-action="explore articles by subject" data-track-label="link" href="/subjects">Explore articles by subject</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="find a job" data-track-label="link" href="/naturecareers">Find a job</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="guide to authors" data-track-label="link" href="/authors/index.html">Guide to authors</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="editorial policies" data-track-label="link" href="/authors/editorial_policies/">Editorial policies</a></li>
</ul>
</div>
</div>
<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
<meta content="Springer Nature" itemprop="publisher"/>
<div class="u-mt-16 u-mb-16">
<div class="u-container">
<div class="u-display-flex u-flex-wrap u-justify-content-space-between">
<p class="c-meta u-ma-0 u-flex-shrink">
<span class="c-meta__item">
                    Nature Communications (<i>Nat Commun</i>)
                </span>
<span class="c-meta__item">
<abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">2041-1723</span> (online)
    </span>
</p>
</div>
</div>
</div>
<div class="c-footer">
<div class="u-hide-print" data-track-component="footer">
<h2 class="u-visually-hidden">nature.com sitemap</h2>
<div class="c-footer__container">
<div class="c-footer__grid c-footer__group--separator">
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="about us" data-track-label="link" href="https://www.nature.com/npg_/company_info/index.html">About us</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="press releases" data-track-label="link" href="https://www.nature.com/npg_/press_room/press_releases.html">Press releases</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="press office" data-track-label="link" href="https://press.nature.com/">Press office</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="contact us" data-track-label="link" href="https://support.nature.com/support/home">Contact us</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Discover content</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="journals a-z" data-track-label="link" href="https://www.nature.com/siteindex">Journals A-Z</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="article by subject" data-track-label="link" href="https://www.nature.com/subjects">Articles by subject</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="protocols.io" data-track-label="link" href="https://www.protocols.io/">protocols.io</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature index" data-track-label="link" href="https://www.natureindex.com/">Nature Index</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link" href="https://www.nature.com/authors/editorial_policies">Nature portfolio policies</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="open access" data-track-label="link" href="https://www.nature.com/nature-research/open-access">Open access</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="reprints and permissions" data-track-label="link" href="https://www.nature.com/reprints">Reprints &amp; permissions</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="data research service" data-track-label="link" href="https://www.springernature.com/gp/authors/research-data">Research data</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="language editing" data-track-label="link" href="https://authorservices.springernature.com/language-editing/">Language editing</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="scientific editing" data-track-label="link" href="https://authorservices.springernature.com/scientific-editing/">Scientific editing</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature masterclasses" data-track-label="link" href="https://masterclasses.nature.com/">Nature Masterclasses</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="research solutions" data-track-label="link" href="https://solutions.springernature.com/">Research Solutions</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="librarian service and tools" data-track-label="link" href="https://www.springernature.com/gp/librarians/tools-services">Librarian service &amp; tools</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="librarian portal" data-track-label="link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal">Librarian portal</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="open research" data-track-label="link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions">Open research</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="Recommend to library" data-track-label="link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library">Recommend to library</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="advertising" data-track-label="link" href="https://partnerships.nature.com/product/digital-advertising/">Advertising</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="partnerships and services" data-track-label="link" href="https://partnerships.nature.com/">Partnerships &amp; Services</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="media kits" data-track-label="link" href="https://partnerships.nature.com/media-kits/">Media kits</a>
</li>
<li class="c-footer__item"><a class="c-footer__link" data-track-action="branded content" data-track-label="link" href="https://partnerships.nature.com/product/branded-content-native-advertising/">Branded
                        content</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Professional development</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature careers" data-track-label="link" href="https://www.nature.com/naturecareers/">Nature Careers</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature conferences" data-track-label="link" href="https://conferences.nature.com">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Regional websites</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature africa" data-track-label="link" href="https://www.nature.com/natafrica">Nature Africa</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature china" data-track-label="link" href="http://www.naturechina.com">Nature China</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature india" data-track-label="link" href="https://www.nature.com/nindia">Nature India</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature Italy" data-track-label="link" href="https://www.nature.com/natitaly">Nature Italy</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature japan" data-track-label="link" href="https://www.natureasia.com/ja-jp">Nature Japan</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature middle east" data-track-label="link" href="https://www.nature.com/nmiddleeast">Nature Middle East</a></li>
</ul>
</div>
</div>
</div>
<div class="c-footer__container">
<ul class="c-footer__links">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="privacy policy" data-track-label="link" href="https://www.nature.com/info/privacy">Privacy
                Policy</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="use of cookies" data-track-label="link" href="https://www.nature.com/info/cookies">Use
                of cookies</a></li>
<li class="c-footer__item">
<button class="optanon-toggle-display c-footer__link" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link" onclick="javascript:;">Your privacy choices/Manage cookies
                </button>
</li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="legal notice" data-track-label="link" href="https://www.nature.com/info/legal-notice">Legal
                notice</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="accessibility statement" data-track-label="link" href="https://www.nature.com/info/accessibility-statement">Accessibility
                statement</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="terms and conditions" data-track-label="link" href="https://www.nature.com/info/terms-and-conditions">Terms &amp; Conditions</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="california privacy statement" data-track-label="link" href="https://www.springernature.com/ccpa">Your US state privacy rights</a></li>
</ul>
</div>
</div>
<div class="c-footer__container">
<a class="c-footer__link" href="https://www.springernature.com/">
<img alt="Springer Nature" height="20" loading="lazy" src="/static/images/logos/sn-logo-white-ea63208b81.svg" width="200"/>
</a>
<p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>
</div>
<div aria-hidden="true" class="u-visually-hidden">
<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path d="M0 .74h56.72v55.24H0z" id="a"></path></defs><symbol id="icon-access" viewbox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewbox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewbox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewbox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewbox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewbox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewbox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewbox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewbox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewbox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewbox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewbox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewbox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewbox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewbox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewbox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewbox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewbox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewbox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewbox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewbox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewbox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewbox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewbox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewbox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewbox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewbox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewbox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewbox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewbox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewbox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewbox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewbox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewbox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewbox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewbox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewbox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewbox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewbox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewbox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewbox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewbox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewbox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewbox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewbox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewbox="0 0 56.69 56.69"><path d="M0 0h56.69v56.69H0z" style="fill:none"></path><clippath id="b"><use style="overflow:visible" xlink:href="#a"></use></clippath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewbox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewbox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewbox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewbox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewbox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewbox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-download-medium" viewbox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-mail-medium" viewbox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewbox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewbox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewbox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewbox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewbox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewbox="0 0 100 100"><path clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z" fill-rule="evenodd"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewbox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>
<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif" data-component-expirydays="30" data-component-id="nature-briefing-banner" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Flagship">
<div class="c-site-messages__banner-large">
<div class="c-site-messages__close-container">
<button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
<svg aria-hidden="true" focusable="false" height="25px" version="1.1" viewbox="0 0 25 25" width="25px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<title>Close banner</title>
<defs></defs>
<g fill="none" fill-rule="evenodd" stroke="none" stroke-width="1">
<rect height="25" opacity="0" width="25" x="0" y="0"></rect>
<path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
</g>
</svg>
<span class="visually-hidden">Close</span>
</button>
</div>
<div class="c-site-messages__form-container">
<div class="grid grid-12 last">
<div class="grid grid-4">
<img alt="Nature Briefing" height="40" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250"/>
<p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
</div>
<div class="grid grid-8 last">
<form action="https://www.nature.com/briefing/briefing" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship" method="post">
<input id="briefing-banner-signup-form-input-track-originReferralPoint" name="track_originReferralPoint" type="hidden" value="MainBriefingBanner"/>
<input id="briefing-banner-signup-form-input-track-formType" name="track_formType" type="hidden" value="DirectEmailBanner"/>
<input id="gdpr_tick_banner" name="gdpr_tick" type="hidden" value="false"/>
<input id="marketing_input_banner" name="marketing" type="hidden" value="false"/>
<input id="marketing_tick_banner" name="marketing_tick" type="hidden" value="false"/>
<input id="brieferEntryPoint_banner" name="brieferEntryPoint" type="hidden" value="MainBriefingBanner"/>
<label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>
<div class="nature-briefing-banner__email-wrapper">
<input class="nature-briefing-banner__email-input box-sizing text14" data-test-element="briefing-emailbanner-email-input" id="emailAddress" name="emailAddress" placeholder="e.g. jo.smith@university.ac.uk" required="" type="email" value=""/>
<input id="defaultNewsletter_banner" name="N:nature_briefing_daily" type="hidden" value="true"/>
<button class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button" type="submit">Sign up</button>
</div>
<div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
<input class="nature-briefing-banner__checkbox-checkbox" data-test-element="briefing-emailbanner-gdpr-checkbox" id="gdpr-briefing-banner-checkbox" name="gdpr" required="" type="checkbox" value="true"/>
<label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
</div>
</form>
</div>
</div>
</div>
</div>
<div class="c-site-messages__banner-small">
<div class="c-site-messages__close-container">
<button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
<svg aria-hidden="true" focusable="false" height="25px" version="1.1" viewbox="0 0 25 25" width="25px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<title>Close banner</title>
<defs></defs>
<g fill="none" fill-rule="evenodd" stroke="none" stroke-width="1">
<rect height="25" opacity="0" width="25" x="0" y="0"></rect>
<path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
</g>
</svg>
<span class="visually-hidden">Close</span>
</button>
</div>
<div class="c-site-messages__content text14">
<span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
<a class="nature-briefing__link text14 sans-serif" data-test-element="briefing-banner-link" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner" rel="noreferrer noopener" target="_blank">Sign up for Nature Briefing
            </a>
</div>
</div>
</div>
<noscript>
<img alt="" height="0" hidden="" src="https://verify.nature.com/verify/nature.png" style="display: none" width="0"/>
</noscript>
<script async="" src="//content.readcube.com/ping?doi=10.1038/s41467-024-49700-4&amp;format=js&amp;last_modified=2024-06-27"></script>
</body>
</html>