<!DOCTYPE html>
<html class="grade-c" lang="en">
<head>
<title>Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure | Nature Communications</title>
<link href="https://www.nature.com/ncomms.rss" rel="alternate" type="application/rss+xml"/>
<script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
<link crossorigin="" href="https://cmp.nature.com" rel="preconnect"/>
<meta content="IE=edge" http-equiv="X-UA-Compatible"/>
<meta content="pc,mobile" name="applicable-device"/>
<meta content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes" name="viewport"/>
<meta content="5a2dc4ab3fcb9b0393241ffbbb490480" name="360-site-verification"/>
<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"archaea-genomics;chromatin-structure","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Communications","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41467-025-56197-y"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by-nc-nd/4.0/"}}},"contentInfo":{"authors":["Kodai Yamaura","Naomichi Takemata","Masashi Kariya","Ayami Osaka","Sonoko Ishino","Masataka Yamauchi","Tomonori Tamura","Itaru Hamachi","Shoji Takada","Yoshizumi Ishino","Haruyuki Atomi"],"publishedAt":1739923200,"publishedAtString":"2025-02-19","title":"Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Archaeal genomics,Chromatin structure"},"journal":{"pcode":"ncomms","title":"nature communications","volume":"16","issue":"1","id":41467,"publishingModel":"Open Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>
<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
<style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>
<link data-inline-css-source="critical-css" data-test="critical-css-handler" href="/static/css/enhanced-article-nature-branded-d54f9178e8.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null" rel="stylesheet"/>
<noscript>
<link href="/static/css/enhanced-article-nature-branded-d54f9178e8.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" rel="stylesheet" type="text/css"/>
</noscript>
<link href="/static/css/article-print-122346e276.css" media="print" rel="stylesheet" type="text/css"/>
<link href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png" rel="apple-touch-icon" sizes="180x180"/>
<link href="/static/images/favicons/nature/favicon-48x48-b52890008c.png" rel="icon" sizes="48x48" type="image/png"/>
<link href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png" rel="icon" sizes="32x32" type="image/png"/>
<link href="/static/images/favicons/nature/favicon-16x16-951651ab72.png" rel="icon" sizes="16x16" type="image/png"/>
<link crossorigin="use-credentials" href="/static/manifest.json" rel="manifest"/>
<link color="#000000" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" rel="mask-icon"/>
<link href="/static/images/favicons/nature/favicon.ico" rel="shortcut icon"/>
<meta content="#000000" name="msapplication-TileColor"/>
<meta content="/static/browserconfig.xml" name="msapplication-config"/>
<meta content="#000000" name="theme-color"/>
<meta content="Nature" name="application-name"/>
<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>
<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->
<script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-74.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-74.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>
<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-606cb67187.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-e919764a53.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>
<meta content="noarchive" name="robots"/>
<meta content="Yes" name="access"/>
<link href="https://www.nature.com/search" rel="search"/>
<link href="https://www.nature.com/opensearch/opensearch.xml" rel="search" title="nature.com" type="application/opensearchdescription+xml"/>
<link href="https://www.nature.com/opensearch/request" rel="search" title="nature.com" type="application/sru+xml"/>
<script type="application/ld+json">{"mainEntity":{"headline":"Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure","description":"In eukaryotes, structural maintenance of chromosomes (SMC) complexes form topologically associating domains (TADs) by extruding DNA loops and being stalled by roadblock proteins. It remains unclear whether a similar mechanism of domain formation exists in prokaryotes. Using high-resolution chromosome conformation capture sequencing, we show that an archaeal homolog of the bacterial Smc-ScpAB complex organizes the genome of Thermococcus kodakarensis into TAD-like domains. We find that TrmBL2, a nucleoid-associated protein that forms a stiff nucleoprotein filament, stalls the T. kodakarensis SMC complex and establishes a boundary at the site-specific recombination site dif. TrmBL2 stalls the SMC complex at tens of additional non-boundary loci with lower efficiency. Intriguingly, the stalling efficiency is correlated with structural properties of underlying DNA sequences. Our study illuminates a eukaryotic-like mechanism of domain formation in archaea and a role of intrinsic DNA structure in large-scale genome organization. Eukaryotic chromosomes are organized into arrays of compact domain structures called TADs. Here the authors show that a member of the Archaea, the prokaryotic domain closest to the Eukarya, uses a eukaryotic-like mechanism of chromosomal domain formation.","datePublished":"2025-02-19T00:00:00Z","dateModified":"2025-02-19T00:00:00Z","pageStart":"1","pageEnd":"21","license":"http://creativecommons.org/licenses/by-nc-nd/4.0/","sameAs":"https://doi.org/10.1038/s41467-025-56197-y","keywords":["Archaeal genomics","Chromatin structure","Science","Humanities and Social Sciences","multidisciplinary"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig6_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig7_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig8_HTML.png"],"isPartOf":{"name":"Nature Communications","issn":["2041-1723"],"volumeNumber":"16","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group UK","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Kodai Yamaura","affiliation":[{"name":"Kyoto University","address":{"name":"Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Naomichi Takemata","url":"http://orcid.org/0000-0003-1902-3766","affiliation":[{"name":"Kyoto University","address":{"name":"Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan","@type":"PostalAddress"},"@type":"Organization"}],"email":"takemata.naomichi.3a@kyoto-u.ac.jp","@type":"Person"},{"name":"Masashi Kariya","affiliation":[{"name":"Kyoto University","address":{"name":"Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ayami Osaka","affiliation":[{"name":"Kyoto University","address":{"name":"Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sonoko Ishino","affiliation":[{"name":"Kyushu University","address":{"name":"Department of Bioscience and Biotechnology, Graduate School of Bioresource and Bioenvironmental Sciences, Kyushu University, Fukuoka, Japan","@type":"PostalAddress"},"@type":"Organization"},{"name":"Nagahama Institute of Bio-Science and Technology","address":{"name":"Genome Editing Research Institute, Nagahama Institute of Bio-Science and Technology, Nagahama, Japan","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Masataka Yamauchi","url":"http://orcid.org/0000-0002-5123-3993","affiliation":[{"name":"Kyoto University","address":{"name":"Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto, Japan","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Tomonori Tamura","url":"http://orcid.org/0000-0003-1648-9296","affiliation":[{"name":"Kyoto University","address":{"name":"Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Itaru Hamachi","url":"http://orcid.org/0000-0002-3327-3916","affiliation":[{"name":"Kyoto University","address":{"name":"Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Shoji Takada","affiliation":[{"name":"Kyoto University","address":{"name":"Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto, Japan","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Yoshizumi Ishino","url":"http://orcid.org/0000-0001-9419-0826","affiliation":[{"name":"Kyushu University","address":{"name":"Department of Bioscience and Biotechnology, Graduate School of Bioresource and Bioenvironmental Sciences, Kyushu University, Fukuoka, Japan","@type":"PostalAddress"},"@type":"Organization"},{"name":"Nagahama Institute of Bio-Science and Technology","address":{"name":"Genome Editing Research Institute, Nagahama Institute of Bio-Science and Technology, Nagahama, Japan","@type":"PostalAddress"},"@type":"Organization"},{"name":"Institute of Science Tokyo","address":{"name":"Cell Biology Center, Institute of Innovative Research, Institute of Science Tokyo, Yokohama, Japan","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Haruyuki Atomi","url":"http://orcid.org/0000-0001-9687-6426","affiliation":[{"name":"Kyoto University","address":{"name":"Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan","@type":"PostalAddress"},"@type":"Organization"}],"email":"atomi.haruyuki.8r@kyoto-u.ac.jp","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>
<link href="https://www.nature.com/articles/s41467-025-56197-y" rel="canonical"/>
<meta content="41467" name="journal_id"/>
<meta content="Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure" name="dc.title"/>
<meta content="Nature Communications 2025 16:1" name="dc.source"/>
<meta content="text/html" name="dc.format"/>
<meta content="Nature Publishing Group" name="dc.publisher"/>
<meta content="2025-02-19" name="dc.date"/>
<meta content="OriginalPaper" name="dc.type"/>
<meta content="En" name="dc.language"/>
<meta content="2025 The Author(s)" name="dc.copyright"/>
<meta content="2025 The Author(s)" name="dc.rights"/>
<meta content="journalpermissions@springernature.com" name="dc.rightsAgent"/>
<meta content="In eukaryotes, structural maintenance of chromosomes (SMC) complexes form topologically associating domains (TADs) by extruding DNA loops and being stalled by roadblock proteins. It remains unclear whether a similar mechanism of domain formation exists in prokaryotes. Using high-resolution chromosome conformation capture sequencing, we show that an archaeal homolog of the bacterial Smc-ScpAB complex organizes the genome of Thermococcus kodakarensis into TAD-like domains. We find that TrmBL2, a nucleoid-associated protein that forms a stiff nucleoprotein filament, stalls the T. kodakarensis SMC complex and establishes a boundary at the site-specific recombination site dif. TrmBL2 stalls the SMC complex at tens of additional non-boundary loci with lower efficiency. Intriguingly, the stalling efficiency is correlated with structural properties of underlying DNA sequences. Our study illuminates a eukaryotic-like mechanism of domain formation in archaea and a role of intrinsic DNA structure in large-scale genome organization. Eukaryotic chromosomes are organized into arrays of compact domain structures called TADs. Here the authors show that a member of the Archaea, the prokaryotic domain closest to the Eukarya, uses a eukaryotic-like mechanism of chromosomal domain formation." name="dc.description"/>
<meta content="2041-1723" name="prism.issn"/>
<meta content="Nature Communications" name="prism.publicationName"/>
<meta content="2025-02-19" name="prism.publicationDate"/>
<meta content="16" name="prism.volume"/>
<meta content="1" name="prism.number"/>
<meta content="OriginalPaper" name="prism.section"/>
<meta content="1" name="prism.startingPage"/>
<meta content="21" name="prism.endingPage"/>
<meta content="2025 The Author(s)" name="prism.copyright"/>
<meta content="journalpermissions@springernature.com" name="prism.rightsAgent"/>
<meta content="https://www.nature.com/articles/s41467-025-56197-y" name="prism.url"/>
<meta content="doi:10.1038/s41467-025-56197-y" name="prism.doi"/>
<meta content="https://www.nature.com/articles/s41467-025-56197-y.pdf" name="citation_pdf_url"/>
<meta content="https://www.nature.com/articles/s41467-025-56197-y" name="citation_fulltext_html_url"/>
<meta content="Nature Communications" name="citation_journal_title"/>
<meta content="Nat Commun" name="citation_journal_abbrev"/>
<meta content="Nature Publishing Group" name="citation_publisher"/>
<meta content="2041-1723" name="citation_issn"/>
<meta content="Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure" name="citation_title"/>
<meta content="16" name="citation_volume"/>
<meta content="1" name="citation_issue"/>
<meta content="2025/02/19" name="citation_online_date"/>
<meta content="1" name="citation_firstpage"/>
<meta content="21" name="citation_lastpage"/>
<meta content="Article" name="citation_article_type"/>
<meta content="" name="citation_fulltext_world_readable"/>
<meta content="en" name="citation_language"/>
<meta content="doi:10.1038/s41467-025-56197-y" name="dc.identifier"/>
<meta content="10.1038/s41467-025-56197-y" name="DOI"/>
<meta content="410993" name="size"/>
<meta content="10.1038/s41467-025-56197-y" name="citation_doi"/>
<meta content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41467-025-56197-y&amp;api_key=" name="citation_springer_api_url"/>
<meta content="In eukaryotes, structural maintenance of chromosomes (SMC) complexes form topologically associating domains (TADs) by extruding DNA loops and being stalled by roadblock proteins. It remains unclear whether a similar mechanism of domain formation exists in prokaryotes. Using high-resolution chromosome conformation capture sequencing, we show that an archaeal homolog of the bacterial Smc-ScpAB complex organizes the genome of Thermococcus kodakarensis into TAD-like domains. We find that TrmBL2, a nucleoid-associated protein that forms a stiff nucleoprotein filament, stalls the T. kodakarensis SMC complex and establishes a boundary at the site-specific recombination site dif. TrmBL2 stalls the SMC complex at tens of additional non-boundary loci with lower efficiency. Intriguingly, the stalling efficiency is correlated with structural properties of underlying DNA sequences. Our study illuminates a eukaryotic-like mechanism of domain formation in archaea and a role of intrinsic DNA structure in large-scale genome organization. Eukaryotic chromosomes are organized into arrays of compact domain structures called TADs. Here the authors show that a member of the Archaea, the prokaryotic domain closest to the Eukarya, uses a eukaryotic-like mechanism of chromosomal domain formation." name="description"/>
<meta content="Yamaura, Kodai" name="dc.creator"/>
<meta content="Takemata, Naomichi" name="dc.creator"/>
<meta content="Kariya, Masashi" name="dc.creator"/>
<meta content="Osaka, Ayami" name="dc.creator"/>
<meta content="Ishino, Sonoko" name="dc.creator"/>
<meta content="Yamauchi, Masataka" name="dc.creator"/>
<meta content="Tamura, Tomonori" name="dc.creator"/>
<meta content="Hamachi, Itaru" name="dc.creator"/>
<meta content="Takada, Shoji" name="dc.creator"/>
<meta content="Ishino, Yoshizumi" name="dc.creator"/>
<meta content="Atomi, Haruyuki" name="dc.creator"/>
<meta content="Archaeal genomics" name="dc.subject"/>
<meta content="Chromatin structure" name="dc.subject"/>
<meta content="citation_journal_title=Annu. Rev. Genet.; citation_title=Organization of chromosomal DNA by SMC complexes; citation_author=S Yatskevich, J Rhodes, K Nasmyth; citation_volume=53; citation_publication_date=2019; citation_pages=445-482; citation_doi=10.1146/annurev-genet-112618-043633; citation_id=CR1" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_title=SMC complexes: from DNA to chromosomes; citation_author=F Uhlmann; citation_volume=17; citation_publication_date=2016; citation_pages=399-412; citation_doi=10.1038/nrm.2016.30; citation_id=CR2" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Condensin-based chromosome organization from bacteria to vertebrates; citation_author=T Hirano; citation_volume=164; citation_publication_date=2016; citation_pages=847-857; citation_doi=10.1016/j.cell.2016.01.033; citation_id=CR3" name="citation_reference"/>
<meta content="citation_journal_title=Curr. Biol.; citation_title=Evolution of condensin and cohesin complexes driven by replacement of Kite by Hawk proteins; citation_author=J. N Wells, T. G Gligoris, K. A Nasmyth, J. A Marsh; citation_volume=27; citation_publication_date=2017; citation_pages=R17-R18; citation_doi=10.1016/j.cub.2016.11.050; citation_id=CR4" name="citation_reference"/>
<meta content="citation_journal_title=Structure; citation_title=Kite proteins: a superfamily of SMC/Kleisin partners conserved across bacteria, archaea, and eukaryotes; citation_author=J. J Palecek, S Gruber; citation_volume=23; citation_publication_date=2015; citation_pages=2183-2190; citation_doi=10.1016/j.str.2015.10.004; citation_id=CR5" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_title=Genome folding through loop extrusion by SMC complexes; citation_author=I. F Davidson, J. M Peters; citation_volume=22; citation_publication_date=2021; citation_pages=445-464; citation_doi=10.1038/s41580-021-00349-7; citation_id=CR6" name="citation_reference"/>
<meta content="citation_journal_title=Annu. Rev. Biochem.; citation_title=Looping the genome with SMC complexes; citation_author=E Kim, R Barth, C Dekker; citation_volume=92; citation_publication_date=2023; citation_pages=15-41; citation_doi=10.1146/annurev-biochem-032620-110506; citation_id=CR7" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Spatial partitioning of the regulatory landscape of the X-inactivation centre; citation_author=E. P Nora; citation_volume=485; citation_publication_date=2012; citation_pages=381-385; citation_doi=10.1038/nature11049; citation_id=CR8" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping; citation_author=S. S Rao; citation_volume=159; citation_publication_date=2014; citation_pages=1665-1680; citation_doi=10.1016/j.cell.2014.11.021; citation_id=CR9" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Targeted degradation of CTCF decouples local insulation of chromosome domains from genomic compartmentalization; citation_author=E. P Nora; citation_volume=169; citation_publication_date=2017; citation_pages=930-944 e922; citation_doi=10.1016/j.cell.2017.05.004; citation_id=CR10" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Transcription shapes 3D chromatin organization by interacting with loop extrusion; citation_author=E. J Banigan; citation_volume=120; citation_publication_date=2023; citation_pages=e2210480120; citation_doi=10.1073/pnas.2210480120; citation_id=CR11" name="citation_reference"/>
<meta content="citation_journal_title=Sci. Adv.; citation_title=Cohesin-dependent chromosome loop extrusion is limited by transcription and stalled replication forks; citation_author=K Jeppsson; citation_volume=8; citation_publication_date=2022; citation_pages=eabn7063; citation_doi=10.1126/sciadv.abn7063; citation_id=CR12" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Cohesin loss eliminates all loop domains; citation_author=S. S. P Rao; citation_volume=171; citation_publication_date=2017; citation_pages=305-320 e324; citation_doi=10.1016/j.cell.2017.09.026; citation_id=CR13" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Struct. Mol. Biol.; citation_title=A cohesin traffic pattern genetically linked to gene regulation; citation_author=A. L Valton; citation_volume=29; citation_publication_date=2022; citation_pages=1239-1251; citation_doi=10.1038/s41594-022-00890-9; citation_id=CR14" name="citation_reference"/>
<meta content="citation_journal_title=Elife; citation_title=Cohesin residency determines chromatin loop patterns; citation_author=L Costantino, T. S Hsieh, R Lamothe, X Darzacq, D Koshland; citation_volume=9; citation_publication_date=2020; citation_pages=e59889; citation_doi=10.7554/eLife.59889; citation_id=CR15" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes; citation_author=A. L Sanborn; citation_volume=112; citation_publication_date=2015; citation_pages=E6456-E6465; citation_doi=10.1073/pnas.1518552112; citation_id=CR16" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=The energetics and physiological impact of cohesin extrusion; citation_author=L Vian; citation_volume=173; citation_publication_date=2018; citation_pages=1165-1178.e20; citation_doi=10.1016/j.cell.2018.03.072; citation_id=CR17" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Genet.; citation_title=Condensin-mediated remodeling of the mitotic chromatin landscape in fission yeast; citation_author=Y Kakui, A Rabinowitz, D. J Barry, F Uhlmann; citation_volume=49; citation_publication_date=2017; citation_pages=1553-1557; citation_doi=10.1038/ng.3938; citation_id=CR18" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Condensin-driven remodelling of X chromosome topology during dosage compensation; citation_author=E Crane; citation_volume=523; citation_publication_date=2015; citation_pages=240-244; citation_doi=10.1038/nature14450; citation_id=CR19" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=High-resolution mapping of the spatial organization of a bacterial chromosome; citation_author=T. B Le, M. V Imakaev, L. A Mirny, M. T Laub; citation_volume=342; citation_publication_date=2013; citation_pages=731-734; citation_doi=10.1126/science.1242059; citation_id=CR20" name="citation_reference"/>
<meta content="citation_journal_title=Genes Dev.; citation_title=Condensin promotes the juxtaposition of DNA flanking its loading site in Bacillus subtilis; citation_author=X Wang; citation_volume=29; citation_publication_date=2015; citation_pages=1661-1675; citation_doi=10.1101/gad.265876.115; citation_id=CR21" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Multiscale structuring of the E. coli chromosome by nucleoid-associated and condensin proteins; citation_author=V. S Lioy; citation_volume=172; citation_publication_date=2018; citation_pages=771-783 e718; citation_doi=10.1016/j.cell.2017.12.027; citation_id=CR22" name="citation_reference"/>
<meta content="citation_journal_title=Genes Dev.; citation_title=Characterization of a prokaryotic SMC protein involved in chromosome partitioning; citation_author=R. A Britton, D. C Lin, A. D Grossman; citation_volume=12; citation_publication_date=1998; citation_pages=1254-1259; citation_doi=10.1101/gad.12.9.1254; citation_id=CR23" name="citation_reference"/>
<meta content="citation_journal_title=EMBO J.; citation_title=The new gene mukB codes for a 177 kd protein with coiled-coil domains involved in chromosome partitioning of E. coli; citation_author=H Niki, A Jaffe, R Imamura, T Ogura, S Hiraga; citation_volume=10; citation_publication_date=1991; citation_pages=183-193; citation_doi=10.1002/j.1460-2075.1991.tb07935.x; citation_id=CR24" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Bacillus subtilis SMC complexes juxtapose chromosome arms as they travel from origin to terminus; citation_author=X Wang, H. B Brandao, T. B Le, M. T Laub, D. Z Rudner; citation_volume=355; citation_publication_date=2017; citation_pages=524-527; citation_doi=10.1126/science.aai8982; citation_id=CR25" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Organization of the Escherichia coli chromosome by a MukBEF axial core; citation_author=J Makela, D. J Sherratt; citation_volume=78; citation_publication_date=2020; citation_pages=250-260.e5; citation_doi=10.1016/j.molcel.2020.02.003; citation_id=CR26" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Recruitment of condensin to replication origin regions by ParB/SpoOJ promotes chromosome segregation in B. subtilis; citation_author=S Gruber, J Errington; citation_volume=137; citation_publication_date=2009; citation_pages=685-696; citation_doi=10.1016/j.cell.2009.02.035; citation_id=CR27" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Recruitment of SMC by ParB-parS organizes the origin region and promotes efficient chromosome segregation; citation_author=N. L Sullivan, K. A Marquis, D. Z Rudner; citation_volume=137; citation_publication_date=2009; citation_pages=697-707; citation_doi=10.1016/j.cell.2009.04.044; citation_id=CR28" name="citation_reference"/>
<meta content="citation_journal_title=Cell Rep.; citation_title=SMC Progressively aligns chromosomal arms in Caulobacter crescentus but is antagonized by convergent transcription; citation_author=N. T Tran, M. T Laub, T. B. K Le; citation_volume=20; citation_publication_date=2017; citation_pages=2057-2071; citation_doi=10.1016/j.celrep.2017.08.026; citation_id=CR29" name="citation_reference"/>
<meta content="citation_journal_title=Curr. Biol.; citation_title=Interlinked sister chromosomes arise in the absence of condensin during fast replication in B. subtilis; citation_author=S Gruber; citation_volume=24; citation_publication_date=2014; citation_pages=293-298; citation_doi=10.1016/j.cub.2013.12.049; citation_id=CR30" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Genomic exploration of the diversity, ecology, and evolution of the archaeal domain of life; citation_author=A Spang, E. F Caceres, T. J. G Ettema; citation_volume=357; citation_publication_date=2017; citation_pages=eaaf388; citation_doi=10.1126/science.aaf3883; citation_id=CR31" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Isolation of an archaeon at the prokaryote-eukaryote interface; citation_author=H Imachi; citation_volume=577; citation_publication_date=2020; citation_pages=519-525; citation_doi=10.1038/s41586-019-1916-6; citation_id=CR32" name="citation_reference"/>
<meta content="citation_journal_title=IUCrJ; citation_title=Evidence for binary Smc complexes lacking kite subunits in archaea; citation_author=J. H Jeon; citation_volume=7; citation_publication_date=2020; citation_pages=193-206; citation_doi=10.1107/S2052252519016634; citation_id=CR33" name="citation_reference"/>
<meta content="citation_journal_title=Front. Microbiol.; citation_title=A novel structural maintenance of chromosomes (SMC)-related protein family specific to Archaea; citation_author=M Yoshinaga, T Nakayama, Y Inagaki; citation_volume=13; citation_publication_date=2022; citation_pages=913088; citation_doi=10.3389/fmicb.2022.913088; citation_id=CR34" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Struct. Mol. Biol.; citation_title=An asymmetric SMC-kleisin bridge in prokaryotic condensin; citation_author=F Burmann; citation_volume=20; citation_publication_date=2013; citation_pages=371-379; citation_doi=10.1038/nsmb.2488; citation_id=CR35" name="citation_reference"/>
<meta content="citation_journal_title=Structure; citation_title=Molecular basis of SMC ATPase activation: role of internal structural changes of the regulatory subcomplex ScpAB; citation_author=K Kamada, M Miyata, T Hirano; citation_volume=21; citation_publication_date=2013; citation_pages=581-594; citation_doi=10.1016/j.str.2013.02.016; citation_id=CR36" name="citation_reference"/>
<meta content="citation_journal_title=J. Cell Sci.; citation_title=Emerging views of genome organization in Archaea; citation_author=N Takemata, S. D Bell; citation_volume=133; citation_publication_date=2020; citation_pages=jcs243782; citation_doi=10.1242/jcs.243782; citation_id=CR37" name="citation_reference"/>
<meta content="citation_journal_title=Biochem. Soc. Trans.; citation_title=Form and function of archaeal genomes; citation_author=S. D Bell; citation_volume=50; citation_publication_date=2022; citation_pages=1931-1939; citation_doi=10.1042/BST20221396; citation_id=CR38" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Multi-scale architecture of archaeal chromosomes; citation_author=N Takemata, S. D Bell; citation_volume=81; citation_publication_date=2021; citation_pages=473-487 e476; citation_doi=10.1016/j.molcel.2020.12.001; citation_id=CR39" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Physical and functional compartmentalization of archaeal chromosomes; citation_author=N Takemata, R. Y Samson, S. D Bell; citation_volume=179; citation_publication_date=2019; citation_pages=165-179.e118; citation_doi=10.1016/j.cell.2019.08.036; citation_id=CR40" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization; citation_author=C Cockram, A Thierry, A Gorlas, R Lestini, R Koszul; citation_volume=81; citation_publication_date=2021; citation_pages=459-472.e410; citation_doi=10.1016/j.molcel.2020.12.013; citation_id=CR41" name="citation_reference"/>
<meta content="Badel C. &amp; Bell S. D. Chromosome architecture in an archaeal species naturally lacking structural maintenance of chromosomes proteins. Nat. Microbiol. 9, 263–273 (2023)." name="citation_reference"/>
<meta content="citation_journal_title=Nat. Microbiol.; citation_title=Chromosome organization affects genome evolution in Sulfolobus archaea; citation_author=C Badel, R. Y Samson, S. D Bell; citation_volume=7; citation_publication_date=2022; citation_pages=820-830; citation_doi=10.1038/s41564-022-01127-7; citation_id=CR43" name="citation_reference"/>
<meta content="Sobolev AS, M. et al. 3C-seq-captured chromosome conformation of the hyperthermophilic archaeon Thermofilum adornatum. bioRxiv, (2021)." name="citation_reference"/>
<meta content="Pilatowski-Herzing E. et al. Capturing chromosome conformation in Crenarchaea. Mol. Microbiol. 
                  https://doi.org/10.1111/mmi.15245
                  
                 (2024)." name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=Systematic evaluation of chromosome conformation capture assays; citation_author=B Akgol Oksuz; citation_volume=18; citation_publication_date=2021; citation_pages=1046-1055; citation_doi=10.1038/s41592-021-01248-7; citation_id=CR46" name="citation_reference"/>
<meta content="citation_journal_title=Appl. Environ. Microbiol.; citation_title=Improved and versatile transformation system allowing multiple genetic manipulations of the hyperthermophilic archaeon Thermococcus kodakaraensis; citation_author=T Sato, T Fukui, H Atomi, T Imanaka; citation_volume=71; citation_publication_date=2005; citation_pages=3889-3899; citation_doi=10.1128/AEM.71.7.3889-3899.2005; citation_id=CR47" name="citation_reference"/>
<meta content="citation_journal_title=Genome Biol.; citation_title=Hi-C as a tool for precise detection and characterisation of chromosomal rearrangements and copy number variation in human tumours; citation_author=L Harewood; citation_volume=18; citation_publication_date=2017; citation_doi=10.1186/s13059-017-1253-8; citation_id=CR48" name="citation_reference"/>
<meta content="citation_journal_title=Genome Res.; citation_title=Complete genome sequence of the hyperthermophilic archaeon Thermococcus kodakaraensis KOD1 and comparison with Pyrococcus genomes; citation_author=T Fukui; citation_volume=15; citation_publication_date=2005; citation_pages=352-363; citation_doi=10.1101/gr.3003105; citation_id=CR49" name="citation_reference"/>
<meta content="citation_journal_title=Extremophiles; citation_title=The chromosome copy number of the hyperthermophilic archaeon Thermococcus kodakarensis KOD1; citation_author=S. K Spaans, J Oost, S. W Kengen; citation_volume=19; citation_publication_date=2015; citation_pages=741-750; citation_doi=10.1007/s00792-015-0750-5; citation_id=CR50" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=HiCDB: a sensitive and robust method for detecting contact domain boundaries; citation_author=F Chen, G Li, M. Q Zhang, Y Chen; citation_volume=46; citation_publication_date=2018; citation_pages=11239-11250; citation_doi=10.1093/nar/gky789; citation_id=CR51" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Struct. Mol. Biol.; citation_title=Transcription-induced domains form the elementary constraining building blocks of bacterial chromosomes; citation_author=A Bignaud; citation_volume=31; citation_publication_date=2024; citation_pages=489-497; citation_doi=10.1038/s41594-023-01178-2; citation_id=CR52" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Computer vision for pattern detection in chromosome contact maps; citation_author=C Matthey-Doret; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-19562-7; citation_id=CR53" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Microbiol.; citation_title=Anucleate and titan cell phenotypes caused by insertional inactivation of the structural maintenance of chromosomes (SMC) gene in the archaeon Methanococcus voltae; citation_author=S. W Long, D. M Faguy; citation_volume=52; citation_publication_date=2004; citation_pages=1567-1577; citation_doi=10.1111/j.1365-2958.2004.04084.x; citation_id=CR54" name="citation_reference"/>
<meta content="citation_journal_title=PLoS Genet.; citation_title=Evidence for a Xer/dif system for chromosome resolution in archaea; citation_author=D Cortez; citation_volume=6; citation_publication_date=2010; citation_pages=e1001166; citation_doi=10.1371/journal.pgen.1001166; citation_id=CR55" name="citation_reference"/>
<meta content="citation_journal_title=Biochimie; citation_title=Comparative genomics reveals conserved positioning of essential genomic clusters in highly rearranged Thermococcales chromosomes; citation_author=M Cossu, V Cunha, C Toffano-Nioche, P Forterre, J Oberto; citation_volume=118; citation_publication_date=2015; citation_pages=313-321; citation_doi=10.1016/j.biochi.2015.07.008; citation_id=CR56" name="citation_reference"/>
<meta content="citation_journal_title=EMBO J.; citation_title=Replication termination and chromosome dimer resolution in the archaeon Sulfolobus solfataricus; citation_author=I. G Duggin, N Dubarry, S. D Bell; citation_volume=30; citation_publication_date=2011; citation_pages=145-153; citation_doi=10.1038/emboj.2010.301; citation_id=CR57" name="citation_reference"/>
<meta content="Midonet C., Barre F. X. Xer site-specific recombination: promoting vertical and horizontal transmission of genetic information. Microbiol Spectr 2, 
                  https://doi.org/10.1128/microbiolspec.MDNA3-0056-2014
                  
                 (2014)." name="citation_reference"/>
<meta content="citation_journal_title=Genes Cells; citation_title=In vitro site-specific recombination mediated by the tyrosine recombinase XerA of Thermoplasma acidophilum; citation_author=M Jo, Y Murayama, Y Tsutsui, H Iwasaki; citation_volume=22; citation_publication_date=2017; citation_pages=646-661; citation_doi=10.1111/gtc.12503; citation_id=CR59" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=XerD unloads bacterial SMC complexes at the replication terminus; citation_author=X Karaboja; citation_volume=81; citation_publication_date=2021; citation_pages=756-766 e758; citation_doi=10.1016/j.molcel.2020.12.027; citation_id=CR60" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Biol. Cell; citation_title=Histone and TK0471/TrmBL2 form a novel heterogeneous genome architecture in the hyperthermophilic archaeon Thermococcus kodakarensis; citation_author=H Maruyama; citation_volume=22; citation_publication_date=2011; citation_pages=386-398; citation_doi=10.1091/mbc.e10-08-0668; citation_id=CR61" name="citation_reference"/>
<meta content="citation_journal_title=J. Biol. Chem.; citation_title=Transcriptional repressor TrmBL2 from Thermococcus kodakarensis forms filamentous nucleoprotein structures and competes with histones for DNA binding in a salt- and DNA supercoiling-dependent manner; citation_author=A. K Efremov; citation_volume=290; citation_publication_date=2015; citation_pages=15770-15784; citation_doi=10.1074/jbc.M114.626705; citation_id=CR62" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=The structural basis for cohesin-CTCF-anchored loops; citation_author=Y Li; citation_volume=578; citation_publication_date=2020; citation_pages=472-476; citation_doi=10.1038/s41586-019-1910-z; citation_id=CR63" name="citation_reference"/>
<meta content="citation_journal_title=J. Mol. Biol.; citation_title=Structural insights into nonspecific binding of DNA by TrmBL2, an archaeal chromatin protein; citation_author=M. U. D Ahmad; citation_volume=427; citation_publication_date=2015; citation_pages=3216-3229; citation_doi=10.1016/j.jmb.2015.08.012; citation_id=CR64" name="citation_reference"/>
<meta content="citation_journal_title=Q. Rev. Biophys.; citation_title=The unique structure of A-tracts and intrinsic DNA bending; citation_author=T. E Haran, U Mohanty; citation_volume=42; citation_publication_date=2009; citation_pages=41-81; citation_doi=10.1017/S0033583509004752; citation_id=CR65" name="citation_reference"/>
<meta content="citation_journal_title=Annu. Rev. Biochem.; citation_title=Sequence-directed curvature of DNA; citation_author=P. J Hagerman; citation_volume=59; citation_publication_date=1990; citation_pages=755-781; citation_doi=10.1146/annurev.bi.59.070190.003543; citation_id=CR66" name="citation_reference"/>
<meta content="citation_journal_title=Q Rev. Biophys.; citation_title=A molecular view of DNA flexibility; citation_author=A Marin-Gonzalez, J. G Vilhena, R Perez, F Moreno-Herrero; citation_volume=54; citation_publication_date=2021; citation_pages=e8; citation_doi=10.1017/S0033583521000068; citation_id=CR67" name="citation_reference"/>
<meta content="citation_journal_title=Curr. Opin. Struct. Biol.; citation_title=Poly(dA:dT) tracts: major determinants of nucleosome organization; citation_author=E Segal, J Widom; citation_volume=19; citation_publication_date=2009; citation_pages=65-71; citation_doi=10.1016/j.sbi.2009.01.004; citation_id=CR68" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=Understanding the paradoxical mechanical response of in-phase A-tracts at different force regimes; citation_author=A Marin-Gonzalez; citation_volume=48; citation_publication_date=2020; citation_pages=5024-5036; citation_doi=10.1093/nar/gkaa225; citation_id=CR69" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=The role of DNA shape in protein-DNA recognition; citation_author=R Rohs; citation_volume=461; citation_publication_date=2009; citation_pages=1248-1253; citation_doi=10.1038/nature08473; citation_id=CR70" name="citation_reference"/>
<meta content="Olson W. K. DNA Bending &amp; Curvature (Adenine Press, 1988)." name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Theory Comput.; citation_title=Sequence-dependent persistence lengths of DNA; citation_author=J. S Mitchell, J Glowacki, A. E Grandchamp, R. S Manning, J. H Maddocks; citation_volume=13; citation_publication_date=2017; citation_pages=1539-1555; citation_doi=10.1021/acs.jctc.6b00904; citation_id=CR72" name="citation_reference"/>
<meta content="citation_journal_title=Phys. Chem. Chem. Phys.; citation_title=SerraNA: a program to determine nucleic acids elasticity from simulation data; citation_author=V Velasco-Berrelleza; citation_volume=22; citation_publication_date=2020; citation_pages=19254-19266; citation_doi=10.1039/D0CP02713H; citation_id=CR73" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Predicting DNA structure using a deep learning method; citation_author=J Li, T. P Chiu, R Rohs; citation_volume=15; citation_publication_date=2024; citation_doi=10.1038/s41467-024-45191-5; citation_id=CR74" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=How do molecular motors fold the genome?; citation_author=C Dekker, C. H Haering, J. M Peters, B. D Rowland; citation_volume=382; citation_publication_date=2023; citation_pages=646-648; citation_doi=10.1126/science.adi8308; citation_id=CR75" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=DNA-segment-capture model for loop extrusion by structural maintenance of chromosome (SMC) protein complexes; citation_author=J. F Marko, P Los Rios, A Barducci, S Gruber; citation_volume=47; citation_publication_date=2019; citation_pages=6956-6972; citation_doi=10.1093/nar/gkz497; citation_id=CR76" name="citation_reference"/>
<meta content="citation_journal_title=Curr. Biol.; citation_title=Real-time detection of single-molecule DNA compaction by condensin I; citation_author=T. R Strick, T Kawaguchi, T Hirano; citation_volume=14; citation_publication_date=2004; citation_pages=874-880; citation_doi=10.1016/j.cub.2004.04.038; citation_id=CR77" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=Condensin extrudes DNA loops in steps up to hundreds of base pairs that are generated by ATP binding events; citation_author=J. K Ryu; citation_volume=50; citation_publication_date=2022; citation_pages=820-832; citation_doi=10.1093/nar/gkab1268; citation_id=CR78" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Struct. Mol. Biol.; citation_title=MukB acts as a macromolecular clamp in DNA condensation; citation_author=Y Cui, Z. M Petrushenko, V. V Rybenkov; citation_volume=15; citation_publication_date=2008; citation_pages=411-418; citation_doi=10.1038/nsmb.1410; citation_id=CR79" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=The SMC1-SMC3 cohesin heterodimer structures DNA through supercoiling-dependent loop formation; citation_author=M Sun, T Nishino, J. F Marko; citation_volume=41; citation_publication_date=2013; citation_pages=6149-6160; citation_doi=10.1093/nar/gkt303; citation_id=CR80" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Phys.; citation_title=Coarse-grained modeling of DNA curvature; citation_author=G. S Freeman, D. M Hinckley, J. P Lequieu, J. K Whitmer, J. J Pablo; citation_volume=141; citation_publication_date=2014; citation_pages=165103; citation_doi=10.1063/1.4897649; citation_id=CR81" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Highly accurate protein structure prediction with AlphaFold; citation_author=J Jumper; citation_volume=596; citation_publication_date=2021; citation_pages=583-589; citation_doi=10.1038/s41586-021-03819-2; citation_id=CR82" name="citation_reference"/>
<meta content="citation_journal_title=Elife; citation_title=Chromosome organization by one-sided and two-sided loop extrusion; citation_author=E. J Banigan, A. A Berg, H. B Brandao, J. F Marko, L. A Mirny; citation_volume=9; citation_publication_date=2020; citation_pages=e53558; citation_doi=10.7554/eLife.53558; citation_id=CR83" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=The Smc5/6 complex is a DNA loop-extruding motor; citation_author=B Pradhan; citation_volume=616; citation_publication_date=2023; citation_pages=843-848; citation_doi=10.1038/s41586-023-05963-3; citation_id=CR84" name="citation_reference"/>
<meta content="Pradhan, B. et al. Loop extrusion-mediated plasmid DNA cleavage by the bacterial SMC Wadjet complex. Mol. Cell 85, 107–116 (2024)." name="citation_reference"/>
<meta content="Jalal A. S., Tran N. T. &amp; Le T. B. ParB spreading on DNA requires cytidine triphosphate in vitro. Elife 9, e53515 (2020)." name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Self-organization of parS centromeres by the ParB CTP hydrolase; citation_author=Y. M Soh; citation_volume=366; citation_publication_date=2019; citation_pages=1129-1133; citation_doi=10.1126/science.aay3965; citation_id=CR87" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=ParB-type DNA segregation proteins Are CTP-dependent molecular switches; citation_author=M Osorio-Valeriano; citation_volume=179; citation_publication_date=2019; citation_pages=1512-1524.e1515; citation_doi=10.1016/j.cell.2019.11.015; citation_id=CR88" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=An archaeal ADP-dependent serine kinase involved in cysteine biosynthesis and serine metabolism; citation_author=Y Makino; citation_volume=7; citation_publication_date=2016; citation_doi=10.1038/ncomms13446; citation_id=CR89" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=The MatP/matS site-specific system organizes the terminus region of the E. coli chromosome into a macrodomain; citation_author=R Mercier; citation_volume=135; citation_publication_date=2008; citation_pages=475-485; citation_doi=10.1016/j.cell.2008.08.031; citation_id=CR90" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=MatP regulates the coordinated action of topoisomerase IV and MukBEF in chromosome segregation; citation_author=S Nolivos; citation_volume=7; citation_publication_date=2016; citation_doi=10.1038/ncomms10466; citation_id=CR91" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Microbiol.; citation_title=Cell cycle-dependent expression of an essential SMC-like protein and dynamic chromosome localization in the archaeon Halobacterium salinarum; citation_author=U Herrmann, J Soppa; citation_volume=46; citation_publication_date=2002; citation_pages=395-409; citation_doi=10.1046/j.1365-2958.2002.03181.x; citation_id=CR92" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=Extended sister-chromosome catenation leads to massive reorganization of the E. coli genome; citation_author=B Conin; citation_volume=50; citation_publication_date=2022; citation_pages=2635-2650; citation_doi=10.1093/nar/gkac105; citation_id=CR93" name="citation_reference"/>
<meta content="citation_journal_title=PLoS Genet.; citation_title=Mapping topoisomerase IV binding and activity sites on the E. coli genome; citation_author=H El Sayyed; citation_volume=12; citation_publication_date=2016; citation_pages=e1006025; citation_doi=10.1371/journal.pgen.1006025; citation_id=CR94" name="citation_reference"/>
<meta content="citation_journal_title=Front. Microbiol.; citation_title=Single-nucleotide resolution detection of Topo IV cleavage activity in the Escherichia coli genome with Topo-Seq; citation_author=D Sutormin, A Galivondzhyan, A Gafurov, K Severinov; citation_volume=14; citation_publication_date=2023; citation_pages=1160736; citation_doi=10.3389/fmicb.2023.1160736; citation_id=CR95" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Genome organization drives chromosome fragility; citation_author=A Canela; citation_volume=170; citation_publication_date=2017; citation_pages=507-521.e518; citation_doi=10.1016/j.cell.2017.06.034; citation_id=CR96" name="citation_reference"/>
<meta content="citation_journal_title=PLoS ONE; citation_title=DNA as a phosphate storage polymer and the alternative advantages of polyploidy for growth or survival; citation_author=K Zerulla; citation_volume=9; citation_publication_date=2014; citation_pages=e94819; citation_doi=10.1371/journal.pone.0094819; citation_id=CR97" name="citation_reference"/>
<meta content="citation_journal_title=Acta Crystallogr. D Biol. Crystallogr.; citation_title=Effect of Rap1 binding on DNA distortion and potassium permanganate hypersensitivity; citation_author=Y. V Bihan; citation_volume=69; citation_publication_date=2013; citation_pages=409-419; citation_doi=10.1107/S0907444912049311; citation_id=CR98" name="citation_reference"/>
<meta content="Analikwu B. T. et al. Telomere protein arrays stall DNA loop extrusion by condensin. bioRxiv, 2023.2010.2029.564563 (2023)." name="citation_reference"/>
<meta content="citation_journal_title=iScience; citation_title=Profiling a single-stranded DNA region within an rDNA segment that affects the loading of bacterial condensin; citation_author=K Yano, H Noguchi, H Niki; citation_volume=25; citation_publication_date=2022; citation_pages=105504; citation_doi=10.1016/j.isci.2022.105504; citation_id=CR100" name="citation_reference"/>
<meta content="citation_journal_title=Trends Genet.; citation_title=A hot story from comparative genomics: reverse gyrase is the only hyperthermophile-specific protein; citation_author=P Forterre; citation_volume=18; citation_publication_date=2002; citation_pages=236-237; citation_doi=10.1016/S0168-9525(02)02650-1; citation_id=CR101" name="citation_reference"/>
<meta content="citation_journal_title=J. Bacteriol.; citation_title=Reverse gyrase is not a prerequisite for hyperthermophilic life; citation_author=H Atomi, R Matsumi, T Imanaka; citation_volume=186; citation_publication_date=2004; citation_pages=4829-4833; citation_doi=10.1128/JB.186.14.4829-4833.2004; citation_id=CR102" name="citation_reference"/>
<meta content="citation_journal_title=Extremophiles; citation_title=Reverse gyrase is essential for microbial growth at 95 degrees C; citation_author=G. L Lipscomb, E. M Hahn, A. T Crowley, M. W. W Adams; citation_volume=21; citation_publication_date=2017; citation_pages=603-608; citation_doi=10.1007/s00792-017-0929-z; citation_id=CR103" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Microbiol.; citation_title=Growth temperature and chromatinization in archaea; citation_author=A Hocher; citation_volume=7; citation_publication_date=2022; citation_pages=1932-1942; citation_doi=10.1038/s41564-022-01245-2; citation_id=CR104" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=Genome-wide prediction of minor-groove electrostatic potential enables biophysical modeling of protein-DNA binding; citation_author=T. P Chiu, S Rao, R. S Mann, B Honig, R Rohs; citation_volume=45; citation_publication_date=2017; citation_pages=12565-12576; citation_doi=10.1093/nar/gkx915; citation_id=CR105" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Removal of phosphoglycolate in hyperthermophilic archaea; citation_author=Y Michimori; citation_volume=121; citation_publication_date=2024; citation_pages=e2311390121; citation_doi=10.1073/pnas.2311390121; citation_id=CR106" name="citation_reference"/>
<meta content="citation_journal_title=J. Bacteriol.; citation_title=Targeted gene disruption by homologous recombination in the hyperthermophilic archaeon Thermococcus kodakaraensis KOD1; citation_author=T Sato, T Fukui, H Atomi, T Imanaka; citation_volume=185; citation_publication_date=2003; citation_pages=210-220; citation_doi=10.1128/JB.185.1.210-220.2003; citation_id=CR107" name="citation_reference"/>
<meta content="citation_journal_title=Cell Rep.; citation_title=Specificity and function of archaeal DNA replication initiator proteins; citation_author=R. Y Samson; citation_volume=3; citation_publication_date=2013; citation_pages=485-496; citation_doi=10.1016/j.celrep.2013.01.002; citation_id=CR108" name="citation_reference"/>
<meta content="citation_journal_title=Front. Microbiol.; citation_title=Biochemical and genetic examination of two aminotransferases from the hyperthermophilic archaeon Thermococcus kodakarensis; citation_author=Y Su, Y Michimori, H Atomi; citation_volume=14; citation_publication_date=2023; citation_pages=1126218; citation_doi=10.3389/fmicb.2023.1126218; citation_id=CR109" name="citation_reference"/>
<meta content="citation_journal_title=BMC Genom.; citation_title=Primary transcriptome map of the hyperthermophilic archaeon Thermococcus kodakarensis; citation_author=D Jager, K. U Forstner, C. M Sharma, T. J Santangelo, J. N Reeve; citation_volume=15; citation_publication_date=2014; citation_doi=10.1186/1471-2164-15-684; citation_id=CR110" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=The hyperthermophilic archaeon Thermococcus kodakarensis is resistant to pervasive negative supercoiling activity of DNA gyrase; citation_author=P Villain; citation_volume=49; citation_publication_date=2021; citation_pages=12332-12347; citation_doi=10.1093/nar/gkab869; citation_id=CR111" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Microbiol.; citation_title=MicrobeJ, a tool for high throughput bacterial cell detection and quantitative analysis; citation_author=A Ducret, E. M Quardokus, Y. V Brun; citation_volume=1; citation_publication_date=2016; citation_pages=16077; citation_doi=10.1038/nmicrobiol.2016.77; citation_id=CR112" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Dynamic RNA acetylation revealed by quantitative cross-evolutionary mapping; citation_author=A Sas-Chen; citation_volume=583; citation_publication_date=2020; citation_pages=638-643; citation_doi=10.1038/s41586-020-2418-2; citation_id=CR113" name="citation_reference"/>
<meta content="citation_journal_title=Genome Biol.; citation_title=HiC-Pro: an optimized and flexible pipeline for Hi-C data processing; citation_author=N Servant; citation_volume=16; citation_publication_date=2015; citation_doi=10.1186/s13059-015-0831-x; citation_id=CR114" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=Salmon provides fast and bias-aware quantification of transcript expression; citation_author=R Patro, G Duggal, M. I Love, R. A Irizarry, C Kingsford; citation_volume=14; citation_publication_date=2017; citation_pages=417-419; citation_doi=10.1038/nmeth.4197; citation_id=CR115" name="citation_reference"/>
<meta content="citation_journal_title=Bioinformatics; citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data; citation_author=M. D Robinson, D. J McCarthy, G. K Smyth; citation_volume=26; citation_publication_date=2010; citation_pages=139-140; citation_doi=10.1093/bioinformatics/btp616; citation_id=CR116" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=Fast gapped-read alignment with Bowtie 2; citation_author=B Langmead, S. L Salzberg; citation_volume=9; citation_publication_date=2012; citation_pages=357-359; citation_doi=10.1038/nmeth.1923; citation_id=CR117" name="citation_reference"/>
<meta content="citation_journal_title=Bioinformatics; citation_title=The sequence alignment/map format and SAMtools; citation_author=H Li; citation_volume=25; citation_publication_date=2009; citation_pages=2078-2079; citation_doi=10.1093/bioinformatics/btp352; citation_id=CR118" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=deepTools2: a next generation web server for deep-sequencing data analysis; citation_author=F Ramirez; citation_volume=44; citation_publication_date=2016; citation_pages=W160-W165; citation_doi=10.1093/nar/gkw257; citation_id=CR119" name="citation_reference"/>
<meta content="citation_journal_title=Genome Biol.; citation_title=Model-based analysis of ChIP-Seq (MACS); citation_author=Y Zhang; citation_volume=9; citation_publication_date=2008; citation_doi=10.1186/gb-2008-9-9-r137; citation_id=CR120" name="citation_reference"/>
<meta content="citation_journal_title=Bioinformatics; citation_title=regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests; citation_author=B Gel; citation_volume=32; citation_publication_date=2016; citation_pages=289-291; citation_doi=10.1093/bioinformatics/btv562; citation_id=CR121" name="citation_reference"/>
<meta content="citation_journal_title=Bioinformatics; citation_title=MEME-ChIP: motif analysis of large DNA datasets; citation_author=P Machanick, T. L Bailey; citation_volume=27; citation_publication_date=2011; citation_pages=1696-1697; citation_doi=10.1093/bioinformatics/btr189; citation_id=CR122" name="citation_reference"/>
<meta content="citation_journal_title=Bioinformatics; citation_title=BEDTools: a flexible suite of utilities for comparing genomic features; citation_author=A. R Quinlan, I. M Hall; citation_volume=26; citation_publication_date=2010; citation_pages=841-842; citation_doi=10.1093/bioinformatics/btq033; citation_id=CR123" name="citation_reference"/>
<meta content="citation_journal_title=PLoS ONE; citation_title=SeqKit: a cross-platform and ultrafast toolkit for FASTA/Q file manipulation; citation_author=W Shen, S Le, Y Li, F Hu; citation_volume=11; citation_publication_date=2016; citation_pages=e0163962; citation_doi=10.1371/journal.pone.0163962; citation_id=CR124" name="citation_reference"/>
<meta content="citation_journal_title=FEMS Microbiol. Lett.; citation_title=Thermoadaptation of methanogenic bacteria by intracellular ion concentration; citation_author=R Hensel, H König; citation_volume=49; citation_publication_date=1988; citation_pages=75-79; citation_doi=10.1111/j.1574-6968.1988.tb02685.x; citation_id=CR125" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Theory Comput.; citation_title=CafeMol: A coarse-grained biomolecular simulator for simulating proteins at work; citation_author=H Kenzaki; citation_volume=7; citation_publication_date=2011; citation_pages=1979-1989; citation_doi=10.1021/ct2001045; citation_id=CR126" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Protoc.; citation_title=3DNA: a versatile, integrated software system for the analysis, rebuilding and visualization of three-dimensional nucleic-acid structures; citation_author=X. J Lu, W. K Olson; citation_volume=3; citation_publication_date=2008; citation_pages=1213-1227; citation_doi=10.1038/nprot.2008.104; citation_id=CR127" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=jPOSTrepo: an international standard data repository for proteomes; citation_author=S Okuda; citation_volume=45; citation_publication_date=2017; citation_pages=D1107-D1111; citation_doi=10.1093/nar/gkw1080; citation_id=CR128" name="citation_reference"/>
<meta content="Yamaura, Kodai" name="citation_author"/>
<meta content="Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan" name="citation_author_institution"/>
<meta content="Takemata, Naomichi" name="citation_author"/>
<meta content="Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan" name="citation_author_institution"/>
<meta content="Kariya, Masashi" name="citation_author"/>
<meta content="Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan" name="citation_author_institution"/>
<meta content="Osaka, Ayami" name="citation_author"/>
<meta content="Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan" name="citation_author_institution"/>
<meta content="Ishino, Sonoko" name="citation_author"/>
<meta content="Department of Bioscience and Biotechnology, Graduate School of Bioresource and Bioenvironmental Sciences, Kyushu University, Fukuoka, Japan" name="citation_author_institution"/>
<meta content="Genome Editing Research Institute, Nagahama Institute of Bio-Science and Technology, Nagahama, Japan" name="citation_author_institution"/>
<meta content="Yamauchi, Masataka" name="citation_author"/>
<meta content="Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto, Japan" name="citation_author_institution"/>
<meta content="Tamura, Tomonori" name="citation_author"/>
<meta content="Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan" name="citation_author_institution"/>
<meta content="Hamachi, Itaru" name="citation_author"/>
<meta content="Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan" name="citation_author_institution"/>
<meta content="Takada, Shoji" name="citation_author"/>
<meta content="Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto, Japan" name="citation_author_institution"/>
<meta content="Ishino, Yoshizumi" name="citation_author"/>
<meta content="Department of Bioscience and Biotechnology, Graduate School of Bioresource and Bioenvironmental Sciences, Kyushu University, Fukuoka, Japan" name="citation_author_institution"/>
<meta content="Genome Editing Research Institute, Nagahama Institute of Bio-Science and Technology, Nagahama, Japan" name="citation_author_institution"/>
<meta content="Cell Biology Center, Institute of Innovative Research, Institute of Science Tokyo, Yokohama, Japan" name="citation_author_institution"/>
<meta content="Atomi, Haruyuki" name="citation_author"/>
<meta content="Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan" name="citation_author_institution"/>
<meta content="https://www.nature.com/platform/readcube-access" name="access_endpoint"/>
<meta content="@NatureComms" name="twitter:site"/>
<meta content="summary_large_image" name="twitter:card"/>
<meta content="Content cover image" name="twitter:image:alt"/>
<meta content="Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure" name="twitter:title"/>
<meta content="Nature Communications - Eukaryotic chromosomes are organized into arrays of compact domain structures called TADs. Here the authors show that a member of the Archaea, the prokaryotic domain closest..." name="twitter:description"/>
<meta content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig1_HTML.png" name="twitter:image"/>
<meta content="https://www.nature.com/articles/s41467-025-56197-y" property="og:url"/>
<meta content="article" property="og:type"/>
<meta content="Nature" property="og:site_name"/>
<meta content="Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure - Nature Communications" property="og:title"/>
<meta content="Eukaryotic chromosomes are organized into arrays of compact domain structures called TADs. Here the authors show that a member of the Archaea, the prokaryotic domain closest to the Eukarya, uses a eukaryotic-like mechanism of chromosomal domain formation." property="og:description"/>
<meta content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig1_HTML.png" property="og:image"/>
<script>
        window.eligibleForRa21 = 'false'; 
    </script>
</head>
<body class="article-page">
<noscript><iframe height="0" src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ" style="display:none;visibility:hidden" width="0"></iframe></noscript>
<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
<a class="c-skip-link" href="#content">Skip to main content</a>
<div class="c-grade-c-banner u-hide">
<div class="c-grade-c-banner__container">
<p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>
</div>
</div>
<div class="u-hide u-show-following-ad"></div>
<aside class="c-ad c-ad--728x90">
<div class="c-ad__inner" data-container-type="banner-advert">
<p class="c-ad__label">Advertisement</p>
<div class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-gpt="" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41467-025-56197-y;doi=10.1038/s41467-025-56197-y;techmeta=1,119,14,15,23,38,58,63,82,91;subjmeta=100,101,2526,26,326,337,631;kwrd=Archaeal+genomics,Chromatin+structure" data-gpt-unitpath="/285/nature_communications/article" data-pa11y-ignore="" data-test="top-ad" id="div-gpt-ad-top-1">
<noscript>
<a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature_communications/article&amp;sz=728x90&amp;c=990395103&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41467-025-56197-y%26doi%3D10.1038/s41467-025-56197-y%26techmeta%3D1,119,14,15,23,38,58,63,82,91%26subjmeta%3D100,101,2526,26,326,337,631%26kwrd%3DArchaeal+genomics,Chromatin+structure">
<img alt="Advertisement" data-test="gpt-advert-fallback-img" height="90" src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature_communications/article&amp;sz=728x90&amp;c=990395103&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41467-025-56197-y%26doi%3D10.1038/s41467-025-56197-y%26techmeta%3D1,119,14,15,23,38,58,63,82,91%26subjmeta%3D100,101,2526,26,326,337,631%26kwrd%3DArchaeal+genomics,Chromatin+structure" width="728"/></a>
</noscript>
</div>
</div>
</aside>
<header class="c-header" data-header="" data-track-component="nature-150-split-header" id="header" style="border-color:#e63323">
<div class="c-header__row">
<div class="c-header__container">
<div class="c-header__split">
<div class="c-header__logo-container">
<a data-track="click" data-track-action="home" data-track-label="image" href="/ncomms">
<picture class="c-header__logo">
<source media="(min-width: 875px)" srcset="https://media.springernature.com/full/nature-cms/uploads/product/ncomms/header-03d2e325c0a02f6df509e5730e9be304.svg">
<img alt="Nature Communications" height="32" src="https://media.springernature.com/full/nature-cms/uploads/product/ncomms/header-7001f06bc3fe2437048388e9f2f44215.svg"/>
</source></picture>
</a>
</div>
<ul class="c-header__menu c-header__menu--global">
<li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
<a class="c-header__link" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link" href="https://www.nature.com/siteindex">
<span>View all journals</span>
</a>
</li>
<li class="c-header__item c-header__item--padding c-header__item--pipe">
<a class="c-header__link c-header__link--search" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" href="#search-menu">
<svg aria-hidden="true" focusable="false" height="22" role="img" viewbox="0 0 18 18" width="22" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
</a>
</li>
<li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
<a class="c-header__link eds-c-header__link" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41467-025-56197-y" id="identity-account-widget"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
</li>
</ul>
</div>
</div>
</div>
<div class="c-header__row">
<div class="c-header__container" data-test="navigation-row">
<div class="c-header__split">
<ul class="c-header__menu c-header__menu--journal">
<li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
<a class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" href="#explore">
<span><span class="c-header__show-text">Explore</span> content</span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--dropdown-menu">
<a class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" href="#about-the-journal">
<span>About <span class="c-header__show-text">the journal</span></span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
<a class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" href="#publish-with-us">
<span>Publish <span class="c-header__show-text">with us</span></span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
</ul>
<ul class="c-header__menu c-header__menu--hide-lg-max">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (desktop site header)" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D264%26journal-link%3Dhttps%253A%252F%252Fwww.nature.com%252Fncomms%252F" rel="nofollow">
<span>Sign up for alerts</span><svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--pipe">
<a class="c-header__link" data-track="click" data-track-action="rss feed" data-track-label="link" href="https://www.nature.com/ncomms.rss">
<span>RSS feed</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</header>
<nav aria-label="breadcrumbs" class="u-mb-16">
<div class="u-container">
<ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
<li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature" href="/" itemprop="item"><span itemprop="name">nature</span></a><meta content="1" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature communications" href="/ncomms" itemprop="item"><span itemprop="name">nature communications</span></a><meta content="2" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles" href="/ncomms/articles?type=article" itemprop="item"><span itemprop="name">articles</span></a><meta content="3" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
<span itemprop="name">article</span><meta content="4" itemprop="position"/></li>
</ol>
</div>
</nav>
</div>
<div class="u-container u-mt-32 u-mb-32 u-clearfix" data-component="article-container" data-container-type="article" id="content">
<main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
<div aria-hidden="true" class="c-context-bar u-hide" data-context-bar="" data-test="context-bar">
<div class="c-context-bar__container u-container" data-track-context="sticky banner">
<div class="c-context-bar__title">
                            Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure
                        </div>
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41467-025-56197-y.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
</div>
<article lang="en">
<div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
<div class="c-pdf-container" data-track-context="article body">
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41467-025-56197-y.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
</div>
<div class="c-article-header">
<header>
<ul class="c-article-identifiers" data-test="article-identifier">
<li class="c-article-identifiers__item" data-test="article-category">Article</li>
<li class="c-article-identifiers__item">
<a class="u-color-open-access" data-test="open-access" data-track="click" data-track-action="open access" data-track-label="link" href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research">Open access</a>
</li>
<li class="c-article-identifiers__item">Published: <time datetime="2025-02-19">19 February 2025</time></li>
</ul>
<h1 class="c-article-title" data-article-title="" data-test="article-title">Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure</h1>
<ul class="c-article-author-list c-article-author-list--short" data-component-authors-activator="authors-list" data-test="authors-list"><li class="c-article-author-list__item"><a data-author-popup="auth-Kodai-Yamaura-Aff1" data-author-search="Yamaura, Kodai" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Kodai-Yamaura-Aff1">Kodai Yamaura</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup><sup class="u-js-hide"> <a href="#na1">na1</a></sup>, </li><li class="c-article-author-list__item"><a data-author-popup="auth-Naomichi-Takemata-Aff1" data-author-search="Takemata, Naomichi" data-corresp-id="c1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Naomichi-Takemata-Aff1">Naomichi Takemata<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-mail-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-1902-3766"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1902-3766</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup><sup class="u-js-hide"> <a href="#na1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Masashi-Kariya-Aff1" data-author-search="Kariya, Masashi" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Masashi-Kariya-Aff1">Masashi Kariya</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Ayami-Osaka-Aff1" data-author-search="Osaka, Ayami" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ayami-Osaka-Aff1">Ayami Osaka</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Sonoko-Ishino-Aff2-Aff3" data-author-search="Ishino, Sonoko" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sonoko-Ishino-Aff2-Aff3">Sonoko Ishino</a><sup class="u-js-hide"><a href="#Aff2">2</a>,<a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Masataka-Yamauchi-Aff4" data-author-search="Yamauchi, Masataka" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Masataka-Yamauchi-Aff4">Masataka Yamauchi</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-5123-3993"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-5123-3993</a></span><sup class="u-js-hide"><a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Tomonori-Tamura-Aff1" data-author-search="Tamura, Tomonori" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Tomonori-Tamura-Aff1">Tomonori Tamura</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-1648-9296"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1648-9296</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Itaru-Hamachi-Aff1" data-author-search="Hamachi, Itaru" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Itaru-Hamachi-Aff1">Itaru Hamachi</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-3327-3916"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3327-3916</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Shoji-Takada-Aff4" data-author-search="Takada, Shoji" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Shoji-Takada-Aff4">Shoji Takada</a><sup class="u-js-hide"><a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Yoshizumi-Ishino-Aff2-Aff3-Aff5" data-author-search="Ishino, Yoshizumi" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Yoshizumi-Ishino-Aff2-Aff3-Aff5">Yoshizumi Ishino</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0001-9419-0826"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9419-0826</a></span><sup class="u-js-hide"><a href="#Aff2">2</a>,<a href="#Aff3">3</a>,<a href="#Aff5">5</a></sup> &amp; </li><li aria-label="Show all 11 authors for this article" class="c-article-author-list__show-more" title="Show all 11 authors for this article">…</li><li class="c-article-author-list__item"><a data-author-popup="auth-Haruyuki-Atomi-Aff1" data-author-search="Atomi, Haruyuki" data-corresp-id="c2" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Haruyuki-Atomi-Aff1">Haruyuki Atomi<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-mail-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0001-9687-6426"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9687-6426</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup> </li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-down-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg><span>Show authors</span></button>
<p class="c-article-info-details" data-container-section="info">
<a data-test="journal-link" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link" href="/ncomms"><i data-test="journal-title">Nature Communications</i></a>
<b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 16</b>, Article number: <span data-test="article-number">1312</span> (<span data-test="article-publication-year">2025</span>)
            <a class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link" href="#citeas">Cite this article</a>
</p>
<div class="c-article-metrics-bar__wrapper u-clear-both">
<ul class="c-article-metrics-bar u-list-reset">
<li class="c-article-metrics-bar__item" data-test="access-count">
<p class="c-article-metrics-bar__count">2388 <span class="c-article-metrics-bar__label">Accesses</span></p>
</li>
<li class="c-article-metrics-bar__item" data-test="altmetric-score">
<p class="c-article-metrics-bar__count">33 <span class="c-article-metrics-bar__label">Altmetric</span></p>
</li>
<li class="c-article-metrics-bar__item">
<p class="c-article-metrics-bar__details"><a data-track="click" data-track-action="view metrics" data-track-label="link" href="/articles/s41467-025-56197-y/metrics" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
</li>
</ul>
</div>
</header>
<div class="u-js-hide" data-component="article-subject-links">
<h3 class="c-article__sub-heading">Subjects</h3>
<ul class="c-article-subject-list">
<li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/archaea-genomics">Archaeal genomics</a></li><li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/chromatin-structure">Chromatin structure</a></li>
</ul>
</div>
</div>
<div class="c-article-body">
<section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>In eukaryotes, structural maintenance of chromosomes (SMC) complexes form topologically associating domains (TADs) by extruding DNA loops and being stalled by roadblock proteins. It remains unclear whether a similar mechanism of domain formation exists in prokaryotes. Using high-resolution chromosome conformation capture sequencing, we show that an archaeal homolog of the bacterial Smc-ScpAB complex organizes the genome of <i>Thermococcus kodakarensis</i> into TAD-like domains. We find that TrmBL2, a nucleoid-associated protein that forms a stiff nucleoprotein filament, stalls the <i>T. kodakarensis</i> SMC complex and establishes a boundary at the site-specific recombination site <i>dif</i>. TrmBL2 stalls the SMC complex at tens of additional non-boundary loci with lower efficiency. Intriguingly, the stalling efficiency is correlated with structural properties of underlying DNA sequences. Our study illuminates a eukaryotic-like mechanism of domain formation in archaea and a role of intrinsic DNA structure in large-scale genome organization.</p></div></div></section>
<section aria-labelledby="inline-recommendations" class="c-article-recommendations" data-title="Inline Recommendations" data-track-component="inline-recommendations">
<h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
<div class="c-article-recommendations-list">
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41564-023-01540-6/MediaObjects/41564_2023_1540_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_1" data-track-action="click recommendations inline - 1" data-track-context="inline recommendations" data-track-label="10.1038/s41564-023-01540-6" href="https://www.nature.com/articles/s41564-023-01540-6?fromPaywallRec=false" itemprop="url">Chromosome architecture in an archaeal species naturally lacking structural maintenance of chromosomes proteins
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__access-type">Open access</span>
<span class="c-article-meta-recommendations__date">18 December 2023</span>
</div>
</div>
</article>
</div>
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-025-86946-4/MediaObjects/41598_2025_86946_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_2" data-track-action="click recommendations inline - 2" data-track-context="inline recommendations" data-track-label="10.1038/s41598-025-86946-4" href="https://www.nature.com/articles/s41598-025-86946-4?fromPaywallRec=false" itemprop="url">SMC translocation is unaffected by an excess of nucleoid associated proteins in vivo
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__access-type">Open access</span>
<span class="c-article-meta-recommendations__date">19 January 2025</span>
</div>
</div>
</article>
</div>
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41594-023-00956-2/MediaObjects/41594_2023_956_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_3" data-track-action="click recommendations inline - 3" data-track-context="inline recommendations" data-track-label="10.1038/s41594-023-00956-2" href="https://www.nature.com/articles/s41594-023-00956-2?fromPaywallRec=false" itemprop="url">DNA segment capture by Smc5/6 holocomplexes
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__access-type">Open access</span>
<span class="c-article-meta-recommendations__date">03 April 2023</span>
</div>
</div>
</article>
</div>
</div>
</section>
<script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1740765107,
                        embedded_user: 'null'
                    }
                });
            </script>
<div class="main-content">
<section data-title="Introduction"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Introduction</h2><div class="c-article-section__content" id="Sec1-content"><p>In both eukaryotes and prokaryotes, structural maintenance of chromosomes (SMC) complexes play critical roles in regulating the 3D structure and function of genomes<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR1" id="ref-link-section-d18519924e621" title="Yatskevich, S., Rhodes, J. &amp; Nasmyth, K. Organization of chromosomal DNA by SMC complexes. Annu. Rev. Genet. 53, 445–482 (2019).">1</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR2" id="ref-link-section-d18519924e621_1" title="Uhlmann, F. SMC complexes: from DNA to chromosomes. Nat. Rev. Mol. Cell Biol. 17, 399–412 (2016).">2</a>,<a aria-label="Reference 3" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR3" id="ref-link-section-d18519924e624" title="Hirano, T. Condensin-based chromosome organization from bacteria to vertebrates. Cell 164, 847–857 (2016).">3</a></sup>. At the core of the SMC complex are two SMC proteins featured by a ~50-nm-long antiparallel coiled coil with the hinge dimerization domain at one end and the ATPase head domain at the other. The head domains of the dimer sandwich ATPs and hydrolyze them to drive conformational changes in the complex. The two SMC protomers are bridged by a Kleisin subunit to form a common tripartite ring structure. An intervening central domain of Kleisin further interacts with either Kleisin-interacting tandem winged-helix elements (Kites) or HEAT proteins associated with Kleisins (Hawks)<sup><a aria-label="Reference 4" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR4" id="ref-link-section-d18519924e628" title="Wells, J. N., Gligoris, T. G., Nasmyth, K. A. &amp; Marsh, J. A. Evolution of condensin and cohesin complexes driven by replacement of Kite by Hawk proteins. Curr. Biol. 27, R17–R18 (2017).">4</a>,<a aria-label="Reference 5" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR5" id="ref-link-section-d18519924e631" title="Palecek, J. J. &amp; Gruber, S. Kite proteins: a superfamily of SMC/Kleisin partners conserved across bacteria, archaea, and eukaryotes. Structure 23, 2183–2190 (2015).">5</a></sup>.</p><p>Single-molecule and chromosome conformation capture (3C) studies have provided a growing body of evidence that SMC complexes function as motors that progressively extrude DNA loops<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR6" id="ref-link-section-d18519924e638" title="Davidson, I. F. &amp; Peters, J. M. Genome folding through loop extrusion by SMC complexes. Nat. Rev. Mol. Cell Biol. 22, 445–464 (2021).">6</a>,<a aria-label="Reference 7" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR7" id="ref-link-section-d18519924e641" title="Kim, E., Barth, R. &amp; Dekker, C. Looping the genome with SMC complexes. Annu. Rev. Biochem. 92, 15–41 (2023).">7</a></sup>. In eukaryotes, SMC-mediated loop extrusion is postulated to fold genomes into arrays of self-interacting domains. These domain structures, often called topologically associating domains (TADs) or loop domains, are believed to play versatile regulatory roles<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR6" id="ref-link-section-d18519924e645" title="Davidson, I. F. &amp; Peters, J. M. Genome folding through loop extrusion by SMC complexes. Nat. Rev. Mol. Cell Biol. 22, 445–464 (2021).">6</a>,<a aria-label="Reference 8" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR8" id="ref-link-section-d18519924e648" title="Nora, E. P. et al. Spatial partitioning of the regulatory landscape of the X-inactivation centre. Nature 485, 381–385 (2012).">8</a>,<a aria-label="Reference 9" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR9" id="ref-link-section-d18519924e651" title="Rao, S. S. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell 159, 1665–1680 (2014).">9</a></sup>. In most cases, eukaryotic chromosomal domains are formed by the SMC complex cohesin and a number of DNA-binding proteins (CTCF, RNA polymerases, etc.) that presumably stall cohesin-mediated loop extrusion at domain boundaries<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR6" id="ref-link-section-d18519924e655" title="Davidson, I. F. &amp; Peters, J. M. Genome folding through loop extrusion by SMC complexes. Nat. Rev. Mol. Cell Biol. 22, 445–464 (2021).">6</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR10" id="ref-link-section-d18519924e658" title="Nora, E. P. et al. Targeted degradation of CTCF decouples local insulation of chromosome domains from genomic compartmentalization. Cell 169, 930–944 e922 (2017).">10</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR11" id="ref-link-section-d18519924e658_1" title="Banigan, E. J. et al. Transcription shapes 3D chromatin organization by interacting with loop extrusion. Proc. Natl. Acad. Sci. USA 120, e2210480120 (2023).">11</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR12" id="ref-link-section-d18519924e658_2" title="Jeppsson, K. et al. Cohesin-dependent chromosome loop extrusion is limited by transcription and stalled replication forks. Sci. Adv. 8, eabn7063 (2022).">12</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR13" id="ref-link-section-d18519924e658_3" title="Rao, S. S. P. et al. Cohesin loss eliminates all loop domains. Cell 171, 305–320 e324 (2017).">13</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR14" id="ref-link-section-d18519924e658_4" title="Valton, A. L. et al. A cohesin traffic pattern genetically linked to gene regulation. Nat. Struct. Mol. Biol. 29, 1239–1251 (2022).">14</a>,<a aria-label="Reference 15" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR15" id="ref-link-section-d18519924e661" title="Costantino, L., Hsieh, T. S., Lamothe, R., Darzacq, X. &amp; Koshland, D. Cohesin residency determines chromatin loop patterns. Elife 9, e59889 (2020).">15</a></sup>. The CTCF roadblock is the most studied example and can nicely explain the formation of multiple boundaries and boundary-associated Hi-C features such as stripes and boundary-to-boundary loops<sup><a aria-label="Reference 10" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR10" id="ref-link-section-d18519924e665" title="Nora, E. P. et al. Targeted degradation of CTCF decouples local insulation of chromosome domains from genomic compartmentalization. Cell 169, 930–944 e922 (2017).">10</a>,<a aria-label="Reference 16" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR16" id="ref-link-section-d18519924e668" title="Sanborn, A. L. et al. Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes. Proc. Natl. Acad. Sci. USA 112, E6456–E6465 (2015).">16</a>,<a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR17" id="ref-link-section-d18519924e671" title="Vian, L, et al. The energetics and physiological impact of cohesin extrusion. Cell 173, 1165–1178.e20 (2018).">17</a></sup>. Another eukaryotic SMC complex condensin also mediates domain formation in certain species, although it is less clear whether and how loop extrusion and boundary proteins are involved in this process<sup><a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR18" id="ref-link-section-d18519924e675" title="Kakui, Y., Rabinowitz, A., Barry, D. J. &amp; Uhlmann, F. Condensin-mediated remodeling of the mitotic chromatin landscape in fission yeast. Nat. Genet. 49, 1553–1557 (2017).">18</a>,<a aria-label="Reference 19" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR19" id="ref-link-section-d18519924e678" title="Crane, E. et al. Condensin-driven remodelling of X chromosome topology during dosage compensation. Nature 523, 240–244 (2015).">19</a></sup>. As with eukaryotes, bacteria fold their genomes into arrays of self-interacting domains called chromosomal interaction domains (CIDs)<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR20" id="ref-link-section-d18519924e683" title="Le, T. B., Imakaev, M. V., Mirny, L. A. &amp; Laub, M. T. High-resolution mapping of the spatial organization of a bacterial chromosome. Science 342, 731–734 (2013).">20</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR21" id="ref-link-section-d18519924e683_1" title="Wang, X. et al. Condensin promotes the juxtaposition of DNA flanking its loading site in Bacillus subtilis. Genes Dev. 29, 1661–1675 (2015).">21</a>,<a aria-label="Reference 22" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR22" id="ref-link-section-d18519924e686" title="Lioy, V. S. et al. Multiscale structuring of the E. coli chromosome by nucleoid-associated and condensin proteins. Cell 172, 771–783 e718 (2018).">22</a></sup>. However, two major classes of bacterial SMC complexes, Smc-ScpAB and MukBEF, do not play a role in CID formation, and they instead presumably use the loop extrusion activity to resolve replicated DNA molecules for chromosome segregation<sup><a aria-label="Reference 20" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR20" id="ref-link-section-d18519924e690" title="Le, T. B., Imakaev, M. V., Mirny, L. A. &amp; Laub, M. T. High-resolution mapping of the spatial organization of a bacterial chromosome. Science 342, 731–734 (2013).">20</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR22" id="ref-link-section-d18519924e693" title="Lioy, V. S. et al. Multiscale structuring of the E. coli chromosome by nucleoid-associated and condensin proteins. Cell 172, 771–783 e718 (2018).">22</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR23" id="ref-link-section-d18519924e693_1" title="Britton, R. A., Lin, D. C. &amp; Grossman, A. D. Characterization of a prokaryotic SMC protein involved in chromosome partitioning. Genes Dev. 12, 1254–1259 (1998).">23</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR24" id="ref-link-section-d18519924e693_2" title="Niki, H., Jaffe, A., Imamura, R., Ogura, T. &amp; Hiraga, S. The new gene mukB codes for a 177 kd protein with coiled-coil domains involved in chromosome partitioning of E. coli. EMBO J. 10, 183–193 (1991).">24</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR25" id="ref-link-section-d18519924e693_3" title="Wang, X., Brandao, H. B., Le, T. B., Laub, M. T. &amp; Rudner, D. Z. Bacillus subtilis SMC complexes juxtapose chromosome arms as they travel from origin to terminus. Science 355, 524–527 (2017).">25</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR26" id="ref-link-section-d18519924e693_4" title="Makela, J. &amp; Sherratt, D. J. Organization of the Escherichia coli chromosome by a MukBEF axial core. Mol. Cell 78, 250–260.e5 (2020).">26</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR27" id="ref-link-section-d18519924e693_5" title="Gruber, S. &amp; Errington, J. Recruitment of condensin to replication origin regions by ParB/SpoOJ promotes chromosome segregation in B. subtilis. Cell 137, 685–696 (2009).">27</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR28" id="ref-link-section-d18519924e693_6" title="Sullivan, N. L., Marquis, K. A. &amp; Rudner, D. Z. Recruitment of SMC by ParB-parS organizes the origin region and promotes efficient chromosome segregation. Cell 137, 697–707 (2009).">28</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR29" id="ref-link-section-d18519924e693_7" title="Tran, N. T., Laub, M. T. &amp; Le, T. B. K. SMC Progressively aligns chromosomal arms in Caulobacter crescentus but is antagonized by convergent transcription. Cell Rep. 20, 2057–2071 (2017).">29</a>,<a aria-label="Reference 30" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR30" id="ref-link-section-d18519924e696" title="Gruber, S. et al. Interlinked sister chromosomes arise in the absence of condensin during fast replication in B. subtilis. Curr. Biol. 24, 293–298 (2014).">30</a></sup>. CIDs are formed by high levels of transcription occurring at their boundaries<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR20" id="ref-link-section-d18519924e700" title="Le, T. B., Imakaev, M. V., Mirny, L. A. &amp; Laub, M. T. High-resolution mapping of the spatial organization of a bacterial chromosome. Science 342, 731–734 (2013).">20</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR21" id="ref-link-section-d18519924e700_1" title="Wang, X. et al. Condensin promotes the juxtaposition of DNA flanking its loading site in Bacillus subtilis. Genes Dev. 29, 1661–1675 (2015).">21</a>,<a aria-label="Reference 22" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR22" id="ref-link-section-d18519924e703" title="Lioy, V. S. et al. Multiscale structuring of the E. coli chromosome by nucleoid-associated and condensin proteins. Cell 172, 771–783 e718 (2018).">22</a></sup>. These findings have led to the prevailing view that domain formation driven by loop extrusion of SMC complexes is specific to eukaryotes.</p><p>Current evidence suggests the origin of eukaryotes within the prokaryotic domain Archaea<sup><a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR31" id="ref-link-section-d18519924e710" title="Spang, A., Caceres, E. F. &amp; Ettema, T. J. G. Genomic exploration of the diversity, ecology, and evolution of the archaeal domain of life. Science 357, eaaf388 (2017).">31</a>,<a aria-label="Reference 32" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR32" id="ref-link-section-d18519924e713" title="Imachi, H. et al. Isolation of an archaeon at the prokaryote-eukaryote interface. Nature 577, 519–525 (2020).">32</a></sup>. Most archaea, with the notable exception of Crenarchaeota, possess homologs of the bacterial Smc-ScpAB subunits, namely the Smc ATPase, the Kleisin protein ScpA, and the Kite protein ScpB<sup><a aria-label="Reference 33" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR33" id="ref-link-section-d18519924e717" title="Jeon, J. H. et al. Evidence for binary Smc complexes lacking kite subunits in archaea. IUCrJ 7, 193–206 (2020).">33</a>,<a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR34" id="ref-link-section-d18519924e720" title="Yoshinaga, M., Nakayama, T. &amp; Inagaki, Y. A novel structural maintenance of chromosomes (SMC)-related protein family specific to Archaea. Front. Microbiol. 13, 913088 (2022).">34</a></sup>. Recent in vitro experiments failed to detect a physical interaction between archaeal ScpA and ScpB, whereas their bacterial homologs form a stable subcomplex<sup><a aria-label="Reference 33" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR33" id="ref-link-section-d18519924e724" title="Jeon, J. H. et al. Evidence for binary Smc complexes lacking kite subunits in archaea. IUCrJ 7, 193–206 (2020).">33</a>,<a aria-label="Reference 35" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR35" id="ref-link-section-d18519924e727" title="Burmann, F. et al. An asymmetric SMC-kleisin bridge in prokaryotic condensin. Nat. Struct. Mol. Biol. 20, 371–379 (2013).">35</a>,<a aria-label="Reference 36" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR36" id="ref-link-section-d18519924e730" title="Kamada, K., Miyata, M. &amp; Hirano, T. Molecular basis of SMC ATPase activation: role of internal structural changes of the regulatory subcomplex ScpAB. Structure 21, 581–594 (2013).">36</a></sup>. The roles of archaeal Smc, ScpA, and ScpB in vivo are poorly understood, although a recent study has provided an important clue about their functions (see below).</p><p>Our knowledge of archaeal 3D genome organization was considerably limited for a long time, partly due to a relatively low number of cultivated archaeal species, their small size, and the extreme growth conditions of most model archaea<sup><a aria-label="Reference 37" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR37" id="ref-link-section-d18519924e737" title="Takemata, N. &amp; Bell, S. D. Emerging views of genome organization in Archaea. J. Cell Sci. 133, jcs243782 (2020).">37</a>,<a aria-label="Reference 38" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR38" id="ref-link-section-d18519924e740" title="Bell, S. D. Form and function of archaeal genomes. Biochem. Soc. Trans. 50, 1931–1939 (2022).">38</a></sup>. We and others have recently succeeded in applying genome-wide 3C techniques (Hi-C and 3C-seq) to diverse archaeal species, identifying a number of structural entities including self-interacting domains<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR39" id="ref-link-section-d18519924e744" title="Takemata, N. &amp; Bell, S. D. Multi-scale architecture of archaeal chromosomes. Mol. Cell 81, 473–487 e476 (2021).">39</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR40" id="ref-link-section-d18519924e744_1" title="Takemata, N., Samson, R. Y. &amp; Bell, S. D. Physical and functional compartmentalization of archaeal chromosomes. Cell 179, 165–179.e118 (2019).">40</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR41" id="ref-link-section-d18519924e744_2" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR42" id="ref-link-section-d18519924e744_3" title="Badel C. &amp; Bell S. D. Chromosome architecture in an archaeal species naturally lacking structural maintenance of chromosomes proteins. Nat. Microbiol. 9, 263–273 (2023).">42</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR43" id="ref-link-section-d18519924e744_4" title="Badel, C., Samson, R. Y. &amp; Bell, S. D. Chromosome organization affects genome evolution in Sulfolobus archaea. Nat. Microbiol. 7, 820–830 (2022).">43</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR44" id="ref-link-section-d18519924e744_5" title="Sobolev AS, M. et al. 3C-seq-captured chromosome conformation of the hyperthermophilic archaeon Thermofilum adornatum. bioRxiv, (2021).">44</a>,<a aria-label="Reference 45" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR45" id="ref-link-section-d18519924e747" title="Pilatowski-Herzing E. et al. Capturing chromosome conformation in Crenarchaea. Mol. Microbiol. 
                  https://doi.org/10.1111/mmi.15245
                  
                 (2024).">45</a></sup>. As is the case in bacteria, most of these archaeal chromosomal domains are demarcated by active transcription<sup><a aria-label="Reference 39" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR39" id="ref-link-section-d18519924e751" title="Takemata, N. &amp; Bell, S. D. Multi-scale architecture of archaeal chromosomes. Mol. Cell 81, 473–487 e476 (2021).">39</a>,<a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e754" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup>. Intriguingly, formation of certain domain boundaries in the halophilic archaeon <i>Halofarax volcanii</i> depends on Smc rather than transcription<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e761" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup>. This raises the possibility that SMC-mediated loop extrusion is a key driver of domain formation not only in eukaryotes but also in archaea. However, the underlying molecular mechanism is largely unknown, especially regarding how Smc-dependent boundaries are formed at specific loci.</p><p>To address this question, we set out to characterize the Smc-ScpAB homolog in the hyperthermophilic anaerobic archaeon <i>Thermococcus kodakarensis</i>. Like many other archaea, <i>T. kodakarensis</i> has a genome encoded in a single circular chromosome. Whereas <i>H. volcanii</i> lacks ScpB, <i>T. kodakarensis</i> possesses homologs of Smc (TK1017), ScpA (TK1018), and ScpB (TK1962). In this study, we provide evidence that these proteins act in concert with a roadblock nucleoid-associated protein (NAP) and sequence-dependent structural features of DNA to sculpt chromosomal domains.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Domain formation in the genome of <i>T. kodakarensis</i>
</h3><p>To investigate the role of Smc, ScpA, and ScpB for archaeal genome organization, we developed a high-resolution 3C-seq protocol for <i>T. kodakarensis</i>, starting from the Hi-C procedure for prokaryotes published recently<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e800" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup>. In this 3C-seq protocol, we used two crosslinking reagents (formaldehyde and disuccinimidyl glutarate) to capture fine-scale structures<sup><a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR46" id="ref-link-section-d18519924e804" title="Akgol Oksuz, B. et al. Systematic evaluation of chromosome conformation capture assays. Nat. Methods 18, 1046–1055 (2021).">46</a></sup>. Crosslinked DNA was digested with two four-base blunt-end cutters (AluI and HaeIII) for high resolution and uniform digestion. We applied the method to our laboratory strain KU216, a uracil-auxotrophic strain (∆<i>pyrF</i>) constructed from the wild-type <i>T. kodakarensis</i> strain KOD1<sup><a aria-label="Reference 47" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR47" id="ref-link-section-d18519924e815" title="Sato, T., Fukui, T., Atomi, H. &amp; Imanaka, T. Improved and versatile transformation system allowing multiple genetic manipulations of the hyperthermophilic archaeon Thermococcus kodakaraensis. Appl. Environ. Microbiol. 71, 3889–3899 (2005).">47</a></sup>. Cells were grown in nutrient-rich medium (ASW-YT-m1-S<sup>0</sup>) until mid-to-late log phase and immediately fixed for the experiment. An obtained contact map, binned at 5-kb resolution, was largely similar to that of KOD1 published previously (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig1">1a</a> and Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">1a</a>)<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e827" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup>. Our 3C-seq data also generated a high-coverage contact map even at 500-bp resolution, whereas the published Hi-C data generated a highly sparse contact map at this resolution (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig1">1b</a> and Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">1b</a>). Since previous Hi-C/3C-seq analyses of archaea were conducted at resolutions ranging from 1 to 30 kb<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR39" id="ref-link-section-d18519924e838" title="Takemata, N. &amp; Bell, S. D. Multi-scale architecture of archaeal chromosomes. Mol. Cell 81, 473–487 e476 (2021).">39</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR40" id="ref-link-section-d18519924e838_1" title="Takemata, N., Samson, R. Y. &amp; Bell, S. D. Physical and functional compartmentalization of archaeal chromosomes. Cell 179, 165–179.e118 (2019).">40</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR41" id="ref-link-section-d18519924e838_2" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR42" id="ref-link-section-d18519924e838_3" title="Badel C. &amp; Bell S. D. Chromosome architecture in an archaeal species naturally lacking structural maintenance of chromosomes proteins. Nat. Microbiol. 9, 263–273 (2023).">42</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR43" id="ref-link-section-d18519924e838_4" title="Badel, C., Samson, R. Y. &amp; Bell, S. D. Chromosome organization affects genome evolution in Sulfolobus archaea. Nat. Microbiol. 7, 820–830 (2022).">43</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR44" id="ref-link-section-d18519924e838_5" title="Sobolev AS, M. et al. 3C-seq-captured chromosome conformation of the hyperthermophilic archaeon Thermofilum adornatum. bioRxiv, (2021).">44</a>,<a aria-label="Reference 45" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR45" id="ref-link-section-d18519924e841" title="Pilatowski-Herzing E. et al. Capturing chromosome conformation in Crenarchaea. Mol. Microbiol. 
                  https://doi.org/10.1111/mmi.15245
                  
                 (2024).">45</a></sup>, this study has provided the highest-resolution view of archaeal genome conformation.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Identification of a genomic inversion and chromosomal domains in T. kodakarensis." id="figure-1"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig1">Fig. 1: Identification of a genomic inversion and chromosomal domains in <i>T. kodakarensis.</i></b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-025-56197-y/figures/1" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig1_HTML.png?as=webp" type="image/webp"><img alt="figure 1" aria-describedby="Fig1" height="683" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig1_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b> 3C-seq analysis on the genome of the <i>T. kodakarensis</i> strain KU216. The contact map was generated at 5-kb resolution. Two visible boundaries (boundary 1 and boundary 2) are indicated by arrows. Bins removed due to low coverage (see “Methods” for more detail) are shaded in gray throughout the paper. <b>b</b> Left panel: a magnified view of <b>a</b> showing a butterfly pattern indicative of a genomic inversion (500-bp resolution). Two proviral regions (<i>TKV2</i> and <i>TKV3</i>) are indicated by cyan and green rectangles, respectively. Homologous sequences within <i>TKV2</i> and <i>TKV3</i> (<i>TKV2</i><sub><i>HR</i></sub> and <i>TKV3</i><sub><i>HR</i></sub>, respectively) are indicated by blue triangles. Right panel: a 3C-seq contact map was generated as in the left panel, except that it was generated for a virtual genome sequence in which a DNA segment flanked by the expected inversion breakpoints was flipped. In the two panels, the orientation of the inverted segment is indicated by white arrows. <b>c</b> Models for the suggested co-existence of chromosome copies with and without the inversion. <b>d</b>, <b>e</b> Magnified views of <b>a</b> showing stripes at 2-kb resolution. Boundaries 1, 2, and part of the inverted segment are indicated by a black triangle, a white triangle, and a white box with one side open, respectively. <b>f</b> Insulation score profile of the KU216 genome. Note that the genomic inversion has caused an artificial drop in the contact insulation at the breakpoints, manifested as deep valleys in the profile. Source data are provided as a <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM8">Source Data</a> file.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 1" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure1 Full size image" data-track-label="button" href="/articles/s41467-025-56197-y/figures/1" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>Of note, the 3C-seq contact map of KU216 displayed a butterfly-like pattern indicative of genomic inversion (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig1">1b</a>, left panel)<sup><a aria-label="Reference 48" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR48" id="ref-link-section-d18519924e929" title="Harewood, L. et al. Hi-C as a tool for precise detection and characterisation of chromosomal rearrangements and copy number variation in human tumours. Genome Biol. 18, 125 (2017).">48</a></sup>. Expected inversion breakpoints were located in highly homologous ~9-kb sequences (sequence identity: 96%) in the two proviral regions <i>TKV2</i> and <i>TKV3</i><sup><a aria-label="Reference 49" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR49" id="ref-link-section-d18519924e938" title="Fukui, T. et al. Complete genome sequence of the hyperthermophilic archaeon Thermococcus kodakaraensis KOD1 and comparison with Pyrococcus genomes. Genome Res. 15, 352–363 (2005).">49</a></sup>. These sequences, here denoted as <i>TKV2</i><sub><i>HR</i></sub> and <i>TKV3</i><sub><i>HR</i></sub> respectively, are oriented in opposite directions on the chromosome, which probably caused the inversion via intra-chromosomal crossover. As <i>T. kodakarensis</i> contains 7–19 copies of the chromosome per cell<sup><a aria-label="Reference 50" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR50" id="ref-link-section-d18519924e958" title="Spaans, S. K., van der Oost, J. &amp; Kengen, S. W. The chromosome copy number of the hyperthermophilic archaeon Thermococcus kodakarensis KOD1. Extremophiles 19, 741–750 (2015).">50</a></sup>, we wondered whether the inversion exists in all copies of the chromosome in the KU216 cell. To test this, we re-generated a 3C-seq contact map of KU216 using a virtual reference genome sequence in which the intervening segment between <i>TKV2</i><sub><i>HR</i></sub> and <i>TKV3</i><sub><i>HR</i></sub> was inverted. This manipulation did not eliminate the butterfly pattern (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig1">1b</a>, right panel), suggesting that chromosome copies with and without the inversion co-exist at the single-cell level or the population level (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig1">1c</a>). Published Hi-C data of the wild-type strain KOD1<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e981" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup> also generated a weaker but visible butterfly signal on the original reference genome (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">1b</a>). Thus, the genomic heterogeneity is not specific to our laboratory strain, and chromosome copies harboring the inversion can exist at a variable ratio.</p><p>The 3C-seq contact map of KU216 displayed two visible boundaries across which genomic contacts were relatively depleted (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig1">1a</a>). These boundaries, denoted as boundary 1 and boundary 2 respectively, can also be seen in the published Hi-C contact map of KOD1 (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">1a</a>)<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e997" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup>. Boundaries 1 and 2 also formed weak stripe patterns reminiscent of those formed by eukaryotic CTCF sites<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR17" id="ref-link-section-d18519924e1001" title="Vian, L, et al. The energetics and physiological impact of cohesin extrusion. Cell 173, 1165–1178.e20 (2018).">17</a></sup>, although the intensity of the stripe at boundary 1 must be carefully interpreted since it goes across the inversion breakpoints (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig1">1d, e</a>). To determine the precise locations of the boundaries, we used the metric called insulation score<sup><a aria-label="Reference 19" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR19" id="ref-link-section-d18519924e1009" title="Crane, E. et al. Condensin-driven remodelling of X chromosome topology during dosage compensation. Nature 523, 240–244 (2015).">19</a></sup>, which represents the relative frequency of local contacts across a locus and thereby reflects contact insulation strength of the region. By determining local minima of insulation scores at 1-kb resolution, we determined the positions of boundaries 1 and 2 as 483–484 kb and 1561–1562 kb, respectively (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig1">1f</a>). Boundaries were also called using the published tool HiCDB<sup><a aria-label="Reference 51" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR51" id="ref-link-section-d18519924e1016" title="Chen, F., Li, G., Zhang, M. Q. &amp; Chen, Y. HiCDB: a sensitive and robust method for detecting contact domain boundaries. Nucleic Acids Res. 46, 11239–11250 (2018).">51</a></sup>, which identified boundaries 1 and 2 essentially at the same positions (Supplementary Data <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM3">1</a>). The insulation of local contacts was much weaker in the rest of the genome (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig1">1f</a>), leading us to conclude that boundaries 1 and 2 are the major boundaries on the <i>T. kodakarensis</i> genome. However, given the ensemble nature of the 3C-seq contact matrix, it remains to be determined how frequently these boundaries are formed and how stably they are maintained across individual chromosome copies and within individual cells.</p><p>A recent high-resolution Hi-C study on <i>Escherichia coli</i> identified transcription-induced domains (TIDs), which are featured by high levels of transcription and bundle-like patterns of short-range interactions<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR52" id="ref-link-section-d18519924e1035" title="Bignaud, A. et al. Transcription-induced domains form the elementary constraining building blocks of bacterial chromosomes. Nat. Struct. Mol. Biol. 31, 489–497 (2024).">52</a></sup>. We found that both boundaries 1 and 2 in <i>T. kodakarensis</i> were transcriptionally inactive (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">2</a>), suggesting that they are structural entities that differ from TIDs. Boundary 1 also did not exhibit TID-like enrichment of short-range interactions, whereas boundary 2 was adjacent to a ~5-kb segment that was slightly enriched for local contacts (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">2</a>).</p><p>The euryarchaeon <i>H. volcanii</i> and members of the Crenarchaeota form tens of DNA loops<sup><a aria-label="Reference 39" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR39" id="ref-link-section-d18519924e1055" title="Takemata, N. &amp; Bell, S. D. Multi-scale architecture of archaeal chromosomes. Mol. Cell 81, 473–487 e476 (2021).">39</a>,<a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e1058" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a>,<a aria-label="Reference 42" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR42" id="ref-link-section-d18519924e1061" title="Badel C. &amp; Bell S. D. Chromosome architecture in an archaeal species naturally lacking structural maintenance of chromosomes proteins. Nat. Microbiol. 9, 263–273 (2023).">42</a></sup>. To search for DNA loops in <i>T. kodakarensis</i>, we analyzed the 3C-seq data of KU216 using Chromosight<sup><a aria-label="Reference 53" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR53" id="ref-link-section-d18519924e1068" title="Matthey-Doret, C. et al. Computer vision for pattern detection in chromosome contact maps. Nat. Commun. 11, 5795 (2020).">53</a></sup>. The interactions involving the inverted segment were excluded from the analysis. Chromosight detected ~10 loops from each of three biological replicates, but none of them were reproducible in all the replicates (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">1</a>). Loop anchors might be obscured by the combined effect of the polyploidy and thermal motion of the chromosome under the high growth temperature of <i>T. kodakarensis</i>.</p><h3 class="c-article__sub-heading" id="Sec4">Smc, ScpA, and ScpB are all required to form chromosomal domains</h3><p>To investigate whether Smc, ScpA, and ScpB contribute to boundary formation in <i>T. kodakarensis</i>, we constructed five deletion mutants lacking one or two of these proteins (Δ<i>smc</i>, Δ<i>scpA</i>, Δ<i>scpB</i>, Δ<i>smc</i> Δ<i>scpA</i>, and Δ<i>smc</i> Δ<i>scpB</i>) using KU216 as a parental strain. 3C-seq uncovered that all deletions tested resulted in loss of contact insulation at boundaries 1 and 2, especially at the former, as well as a slight increase in short-range interactions up to ~150 kb (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig2">2a–d</a> and Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">3</a>). The loss of these boundaries was also confirmed by HiCDB (Supplementary Data <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM3">1</a>) and manifested as an increase in the insulation score for both loci (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig2">2e</a>). The insulation score was increased very similarly among the five mutants. From these results, we conclude that Smc, ScpA, and ScpB act in the same pathway to organize the <i>T. kodakarensis</i> genome into the domain structures. Our unpublished results also suggest that these three proteins form a ternary complex (see “Discussion”). According to these findings, we will refer to the <i>T. kodakarensis</i> counterpart of bacterial Smc-ScpAB simply as Smc-ScpAB.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Deletions of Smc, ScpA, and ScpB decrease contact insulation at boundaries 1 and 2 in a non-additive manner." id="figure-2"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig2">Fig. 2: Deletions of Smc, ScpA, and ScpB decrease contact insulation at boundaries 1 and 2 in a non-additive manner.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-025-56197-y/figures/2" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig2_HTML.png?as=webp" type="image/webp"><img alt="figure 2" aria-describedby="Fig2" height="577" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig2_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b> Comparison of 3C-seq contact maps (5-kb resolution) from KU216 and Δ<i>smc</i> cells (upper right and lower left triangles, respectively). <b>b</b> Differential contact map showing log<sub>2</sub> ratios of contact frequencies between KU216 and Δ<i>smc</i> cells at 5-kb resolution. The genomic contacts between the inverted region and the other loci (shaded in gray) were omitted from the analysis. The locations of boundaries 1 and 2 in KU216 are indicated by arrows. Note that the deletion of the long <i>smc</i> gene (3570 bp) has caused artifacts in the map (indicated by an asterisk). <b>c</b>, <b>d</b> 3C-seq contact maps of genomic regions surrounding boundary 1 (<b>c</b>) and boundary 2 (<b>d</b>). KU216 and derivative strains lacking one or two of the Smc-ScpAB subunit homologs were used for the analysis. <b>e</b> Insulation score profiles of boundary 1 (upper panel) and boundary 2 (lower panel) are shown for KU216 (black solid line) and the deletion strains (dashed lines, Δ<i>smc</i>: red, Δ<i>scpA</i>: green, Δ<i>scpB</i>: blue, Δ<i>smc</i> Δ<i>scpA</i>: orange, Δ<i>smc</i> Δ<i>scpB</i>: cyan). Note that the graphs in the upper panel cover an inversion breakpoint that has caused artificial drops in the score. Source data are provided as a <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM8">Source Data</a> file.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 2" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure2 Full size image" data-track-label="button" href="/articles/s41467-025-56197-y/figures/2" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>SMC-mediated genome organization is critical for cell viability in the bacterial model organisms <i>E. coli</i> and <i>Bacillus subtilis</i><sup><a aria-label="Reference 23" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR23" id="ref-link-section-d18519924e1217" title="Britton, R. A., Lin, D. C. &amp; Grossman, A. D. Characterization of a prokaryotic SMC protein involved in chromosome partitioning. Genes Dev. 12, 1254–1259 (1998).">23</a>,<a aria-label="Reference 24" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR24" id="ref-link-section-d18519924e1220" title="Niki, H., Jaffe, A., Imamura, R., Ogura, T. &amp; Hiraga, S. The new gene mukB codes for a 177 kd protein with coiled-coil domains involved in chromosome partitioning of E. coli. EMBO J. 10, 183–193 (1991).">24</a></sup>. An early study reported that loss of <i>smc</i> causes a discernible growth defect in the archaeon <i>Methanococcus voltae</i>, whereas Cockram et al. more recently revealed that deletion of <i>smc</i> has no apparent impact on the cellular fitness of <i>H. volcanii</i><sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e1236" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a>,<a aria-label="Reference 54" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR54" id="ref-link-section-d18519924e1239" title="Long, S. W. &amp; Faguy, D. M. Anucleate and titan cell phenotypes caused by insertional inactivation of the structural maintenance of chromosomes (SMC) gene in the archaeon Methanococcus voltae. Mol. Microbiol. 52, 1567–1577 (2004).">54</a></sup>. To explore whether the SMC-mediated domain formation has phenotypic consequences in <i>T. kodakarensis</i>, we first measured the growth of the five deletion mutants in nutrient-rich medium. All of them proliferated at similar rates as the parental strain KU216 during exponential phase, although the mutants exhibited a slight decrease in growth rate immediately before the transition to stationary phase (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">4a, b</a>). We next compared morphological and cellular features of the Δ<i>smc</i>, Δ<i>scpA</i>, and Δ<i>scpB</i> strains with those of KU216. The three mutants were slightly larger than KU216 but displayed a similar nucleoid distribution in which the nucleoid occupied most part of the cell as in KU216 (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">4c, d</a>). No anucleate cells were found among the 1000–1700 cells analyzed for each strain. Finally, RNA-seq analysis using the Δ<i>smc</i>, Δ<i>scpA</i>, and Δ<i>scpB</i> strains grown to mid-to-late exponential phase revealed only minor differences in their transcriptomes compared to that in KU216 (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">4e</a>). These results suggest that the domain structures shaped by Smc-ScpAB have a very small impact on the cellular fitness of <i>T. kodakarensis</i>, at least under the growth conditions tested here.</p><h3 class="c-article__sub-heading" id="Sec5">Boundary formation at <i>dif</i>-surrounding sequences in diverse euryarchaea</h3><p>Boundary 1 was located in a large intergenic region (~1.9 kb) between the convergently oriented genes <i>TK0561</i> and <i>TK0562</i>. We found that this intergenic region also contains a previously reported putative <i>dif</i> sequence composed of imperfect inverted repeats of 11 bp separated by a 6-bp spacer (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig3">3a</a> and Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">5a</a>)<sup><a aria-label="Reference 55" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR55" id="ref-link-section-d18519924e1306" title="Cortez, D. et al. Evidence for a Xer/dif system for chromosome resolution in archaea. PLoS Genet. 6, e1001166 (2010).">55</a>,<a aria-label="Reference 56" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR56" id="ref-link-section-d18519924e1309" title="Cossu, M., Da Cunha, V., Toffano-Nioche, C., Forterre, P. &amp; Oberto, J. Comparative genomics reveals conserved positioning of essential genomic clusters in highly rearranged Thermococcales chromosomes. Biochimie 118, 313–321 (2015).">56</a></sup>. <i>dif</i> is known as a target site of Xer site-specific recombinases (XerC and XerD in bacteria and XerA or Xer in archaea), which resolve chromosome dimers arising from an odd number of crossover events between circular sister chromosomes<sup><a aria-label="Reference 57" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR57" id="ref-link-section-d18519924e1316" title="Duggin, I. G., Dubarry, N. &amp; Bell, S. D. Replication termination and chromosome dimer resolution in the archaeon Sulfolobus solfataricus. EMBO J. 30, 145–153 (2011).">57</a>,<a aria-label="Reference 58" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR58" id="ref-link-section-d18519924e1319" title="Midonet C., Barre F. X. Xer site-specific recombination: promoting vertical and horizontal transmission of genetic information. Microbiol Spectr 2, 
                  https://doi.org/10.1128/microbiolspec.MDNA3-0056-2014
                  
                 (2014).">58</a></sup>. Chromatin immunoprecipitation sequencing (ChIP-seq) on the sole homolog of XerA (TK0777) in <i>T. kodakarensis</i> showed that the <i>dif</i> sequence was the bona fide binding site for XerA (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig3">3b</a>). We did not find any discernible DNA motif around boundary 2.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Domain boundaries are formed near dif sequences in diverse euryarchaea." id="figure-3"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig3">Fig. 3: Domain boundaries are formed near <i>dif</i> sequences in diverse euryarchaea.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-025-56197-y/figures/3" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig3_HTML.png?as=webp" type="image/webp"><img alt="figure 3" aria-describedby="Fig3" height="480" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig3_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b> Genomic position of the <i>dif</i> sequence in <i>T. kodakarensis</i>. The coordinate of the first base of the <i>dif</i> sequence is shown in parentheses. The position of boundary 1 (determined at 1-kb resolution) is indicated by a black capped line. Neighboring genes and their orientations are indicated by gray pentagons. <b>b</b> ChIP-seq tracks of XerA are shown for KU216 and Δ<i>xerA</i> cells of <i>T. kodakarensis</i>. The genomic positions of boundaries 1 and 2 in KU216 are indicated by black triangles. The genomic position of <i>dif</i> is indicated by an arrow. The coordinate of the first base of the <i>dif</i> sequence is shown in parentheses. Asterisks indicate a non-specific peak. <b>c</b> The genomic positions of <i>dif1</i> and <i>dif2</i> in <i>Thermoplasma acidophilum</i> are shown as in (<b>a</b>). <i>dif1</i> is denoted in brackets to highlight that it is not functional<sup><a aria-label="Reference 59" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR59" id="ref-link-section-d18519924e1395" title="Jo, M., Murayama, Y., Tsutsui, Y. &amp; Iwasaki, H. In vitro site-specific recombination mediated by the tyrosine recombinase XerA of Thermoplasma acidophilum. Genes Cells 22, 646–661 (2017).">59</a></sup>. <b>d</b> Left panel: 3C-seq analysis on the whole genome of <i>T. acidophilum</i> at 5-kb resolution. The genomic positions of <i>dif1</i> and <i>dif2</i> are indicated by black triangles. Right panel: a magnified view of the left panel showing the colocalization of <i>dif2</i> with a boundary structure (indicated by an arrow) at 1-kb resolution. <b>e</b> Insulation score profile of the <i>T. acidophilum</i> genome. The genomic positions of <i>dif1</i> and <i>dif2</i> are indicated by red arrows. Source data are provided as a <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM8">Source Data</a> file. <b>f</b> Left panel: the genomic position of a putative <i>dif</i> sequence in <i>Haloferax volcanii</i> are indicated as in (<b>a</b>). Cyan pentagons represent an operon containing a homolog of <i>xerA</i>. Right panel: published Hi-C data<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e1447" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup> were used to generate contact maps (5-kb resolution) around the putative <i>dif</i> sequence (indicated by an arrow) in wild-type and Δ<i>smc</i> cells of <i>H. volcanii</i> (upper right and lower left triangles, respectively). <b>g</b> Left panel: the genomic position of a putative <i>dif</i> sequence in <i>Halobacterium salinarum</i> is shown as in (<b>f</b>). Right panel: published Hi-C data<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e1473" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup> were used to generate a contact map (5-kb resolution) around the putative <i>dif</i> sequence (indicated by an arrow) in wild-type <i>H. salinarum</i>.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 3" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure3 Full size image" data-track-label="button" href="/articles/s41467-025-56197-y/figures/3" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>A previous study identified two XerA-binding sites (<i>dif1</i> and <i>dif2</i>) in <i>Thermoplasma acidophilum</i>, a euryarchaeon possessing homologs of Smc, ScpA, and ScpB (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig3">3c</a>). Of these two sites, only <i>dif2</i> can serve as a substrate for XerA-mediated recombination in vitro<sup><a aria-label="Reference 59" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR59" id="ref-link-section-d18519924e1510" title="Jo, M., Murayama, Y., Tsutsui, Y. &amp; Iwasaki, H. In vitro site-specific recombination mediated by the tyrosine recombinase XerA of Thermoplasma acidophilum. Genes Cells 22, 646–661 (2017).">59</a></sup>. By applying 3C-seq to <i>T. acidophilum</i>, we identified a boundary-like structure near <i>dif2</i> but not <i>dif1</i> (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig3">3d, e</a>). To further explore the generality of <i>dif</i>-associated boundaries, we searched for <i>dif</i>-like sequences around the previously reported Smc-dependent boundaries in the multipartite genome of <i>H. volcanii</i><sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e1535" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup>. We found that two of the nine Smc-dependent boundaries, one of which resides on the main chromosome and the other on the megaplasmid pHV3, are adjacent to putative <i>dif</i> sequences (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig3">3f</a> and Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">5b</a>). The <i>dif</i>-like element on the main chromosome is located upstream of an operon containing a <i>xerA</i> homolog (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig3">3f</a>, left panel). We also inspected a published Hi-C dataset of <i>Halobacterium salinarum</i>, another euryarchaeon with a multipartite genome<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e1561" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup>. In this organism, a <i>dif</i>-like sequence is found upstream of the same <i>xerA-</i>containing operon and located close to a boundary structure (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig3">3g</a>). It remains unknown whether this boundary is formed by an SMC complex<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e1575" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup>. Taken together, the colocalization of a boundary structure and <i>dif</i> is conserved among a wide range of euryarchaeal lineages.</p><p>A recent study has shown that bacterial XerD also serves as an unloader of Smc-ScpAB at the <i>dif</i>-like sequences named <i>XDS</i> in the replication terminus (<i>ter</i>)<sup><a aria-label="Reference 60" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR60" id="ref-link-section-d18519924e1594" title="Karaboja, X. et al. XerD unloads bacterial SMC complexes at the replication terminus. Mol. Cell 81, 756–766 e758 (2021).">60</a></sup>. Presumably due to this unloading function, an artificial array of <i>XDS</i> inserted on a bacterial chromosome inhibits the translocation of Smc-ScpAB and forces the complex to form a boundary-like structure<sup><a aria-label="Reference 60" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR60" id="ref-link-section-d18519924e1602" title="Karaboja, X. et al. XerD unloads bacterial SMC complexes at the replication terminus. Mol. Cell 81, 756–766 e758 (2021).">60</a></sup>. These findings led us to hypothesize that archaeal XerA is responsible for specifying the <i>dif</i>-associated boundary. To test this possibility, we tried to construct <i>T. kodakarensis</i> strains lacking either XerA or the whole 28-bp sequence of <i>dif</i> using KU216 as a parental strain. Consistent with the successful deletion of Xer in the crenarchaeon <i>Saccharolobus solfataricus</i><sup><a aria-label="Reference 57" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR57" id="ref-link-section-d18519924e1617" title="Duggin, I. G., Dubarry, N. &amp; Bell, S. D. Replication termination and chromosome dimer resolution in the archaeon Sulfolobus solfataricus. EMBO J. 30, 145–153 (2011).">57</a></sup>, we obtained both mutants of <i>T. kodakarensis</i>. To our surprise, 3C-seq revealed that boundary 1 was unchanged in these mutants (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig4">4</a>). In addition, conformational changes in other genomic regions were not observed in Δ<i>xerA</i> or Δ<i>dif</i>. Altogether, despite the conserved colocalization of a boundary and archaeal <i>dif</i>, the Xer/<i>dif</i> system is not essential for the boundary formation in <i>T. kodakarensis</i>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="The Xer/dif system is dispensable for the formation of boundary 1." id="figure-4"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig4">Fig. 4: The Xer/<i>dif</i> system is dispensable for the formation of boundary 1.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-025-56197-y/figures/4" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig4_HTML.png?as=webp" type="image/webp"><img alt="figure 4" aria-describedby="Fig4" height="404" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig4_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b> Comparison of 3C-seq contact maps (5-kb resolution) from KU216 and deletion strains (upper right and lower left triangles, respectively). <b>b</b> Differential contact maps displaying log<sub>2</sub> ratios of contact frequencies between KU216 and deletion strains are shown as in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig2">2b</a>. <b>c</b> 3C-seq contact maps (1-kb resolution) of a genomic region surrounding boundary 1 are shown for <i>T. kodakarensis</i> strains lacking either <i>xerA</i> (left panel) or <i>dif</i> (right panel). <b>d</b> Insulation score profiles of boundary 1 from KU216 (black solid line), Δ<i>xerA</i> (red dashed line), and Δ<i>dif</i> (orange dashed line) strains of <i>T. kodakarensis</i>. Note that the graphs cover an inversion breakpoint that has caused artificial drops in the score. Source data are provided as a <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM8">Source Data</a> file.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 4" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure4 Full size image" data-track-label="button" href="/articles/s41467-025-56197-y/figures/4" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>To investigate whether the formation of boundary 1 is dictated by an underlying DNA sequence other than <i>dif</i>, we deleted the entire part of the <i>TK0561-0562</i> intergenic region that contains boundary 1 and <i>dif</i> (referred to as the <i>boundary1</i> locus). 3C-seq revealed that deletion of <i>boundary1</i> resulted in a complete loss of contact insulation at the corresponding region (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig5">5a, b</a>). To verify this finding, we next constructed a DNA sequence that contains the whole intergenic region between <i>TK0561</i> and <i>TK0562</i> but lacks the 28-bp <i>dif</i> sequence. This construct, denoted as <i>boundary1</i><sup>Δ<i>dif</i></sup>, was introduced into an ectopic locus outside the inverted segment in KU216 and the Δ<i>boundary1</i> strain. 3C-seq showed that the insert formed a boundary structure in both strains (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig5">5a, b</a>). Since the Δ<i>boundary1</i> strain carrying the <i>boundary1</i><sup>Δ<i>dif</i></sup> insert has only one copy of the <i>dif-</i>surrounding sequence, this strain allowed us to see in detail how the insert interacted with other loci. First, the insert generated a stripe that was not contaminated by the inversion, corroborating the presence of the stripe at the endogenous <i>boundary1</i> locus (Figs. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig1">1</a>d, <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig5">5c</a>). Second, the insert formed a weak but reproducible loop on the 3C-seq contact map by interacting with boundary 2 (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig5">5c</a> and Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">6a</a>). The looping interaction was also confirmed by virtual 4C plots generated from the 3C-seq data (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig5">5d</a> and Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">6b</a>). These results support the role of the <i>dif</i>-surrounding sequence in directing the formation of boundary 1 and additionally highlight its structural properties shared with eukaryotic CTCF sites.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="The dif-surrounding intergenic sequence is required and sufficient to direct boundary formation." id="figure-5"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig5">Fig. 5: The <i>dif</i>-surrounding intergenic sequence is required and sufficient to direct boundary formation.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-025-56197-y/figures/5" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig5_HTML.png?as=webp" type="image/webp"><img alt="figure 5" aria-describedby="Fig5" height="684" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig5_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, <b>b</b> Comparison of 3C-seq contact maps (5-kb resolution) between KU216 and strains lacking the <i>boundary1</i> locus and/or possessing the <i>boundary1</i><sup>Δ<i>dif</i></sup> sequence at an ectopic location. The contact maps were generated at 5-kb (<b>a</b>) and 1-kb (<b>b</b>) resolutions. The endogenous positions of boundaries 1 and 2 are indicated by cyan ellipses and cyan rectangles, respectively. The deletion of <i>boundary1</i> is represented by X. The insertion site of <i>boundary1</i><sup>Δ<i>dif</i></sup> is indicated by magenta ellipses. The inverted segment is indicated by white rectangles (with or without one side open). In (<b>b</b>), the genomic contacts between the inverted segment and other loci are shaded in gray. <b>c</b> Left panel: a 3C-seq contact map was generated at 2-kb resolution to visualize a stripe extending from the ectopic <i>boundary1</i><sup>Δ<i>dif</i></sup> sequence that had been introduced into the Δ<i>boundary1</i> strain. The positions of boundary 2 and the <i>boundary1</i><sup>Δ<i>dif</i></sup> insert are indicated as in (<b>a</b>, <b>b</b>). Right panel: a magnified view of the left panel focusing on a loop anchored by the ectopic <i>boundary1</i><sup>Δ<i>dif</i></sup> sequence and boundary 2. <b>d</b> Virtual 4C plot confirming the looping interaction shown in (<b>c</b>). The ectopic <i>boundary1</i><sup>Δ<i>dif</i></sup> sequence was selected as a viewpoint (white triangle). The position of boundary 2 is indicated by a black triangle. The inverted segment was omitted from the analysis (shaded in gray).</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 5" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure5 Full size image" data-track-label="button" href="/articles/s41467-025-56197-y/figures/5" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec6">The NAP TrmBL2 is required for the formation of boundary 1</h3><p>To identify a protein that specifies boundary 1, we performed DNA affinity purification of proteins that bind to the intergenic region encompassing boundary 1. We prepared two biotinylated DNA probes, denoted as <i>difL</i> and <i>difR</i> respectively, that cover the intergenic region (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig6">6a</a>). The kanamycin resistance gene <i>kanR</i> was used as a control probe. These probes were conjugated to streptavidin beads and incubated with cell extracts of KU216 in the presence of 150 mM NaCl. After washing with 150 mM NaCl, bound proteins were eluted in a single step by adding SDS (“total bound proteins” in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig6">6b</a>). The <i>difL</i> and <i>difR</i> probes yielded two common specific bands (bands 1 and 2) as well as many other non-specific bands (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig6">6c</a>). We reduced these non-specific proteins by eluting the bound material with increasing concentrations of NaCl before adding SDS. Bands 1 and 2 were isolated from two eluates (eluates 1 and 2) and analyzed by mass spectrometry (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig6">6b</a>, c). Band 1 was identified as TrmBL2 (TK0471), a NAP known to form a stiff nucleoprotein filament and repress more than a hundred of genes<sup><a aria-label="Reference 61" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR61" id="ref-link-section-d18519924e1934" title="Maruyama, H. et al. Histone and TK0471/TrmBL2 form a novel heterogeneous genome architecture in the hyperthermophilic archaeon Thermococcus kodakarensis. Mol. Biol. Cell 22, 386–398 (2011).">61</a>,<a aria-label="Reference 62" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR62" id="ref-link-section-d18519924e1937" title="Efremov, A. K. et al. Transcriptional repressor TrmBL2 from Thermococcus kodakarensis forms filamentous nucleoprotein structures and competes with histones for DNA binding in a salt- and DNA supercoiling-dependent manner. J. Biol. Chem. 290, 15770–15784 (2015).">62</a></sup>. Band 2 was identified as a protein encoded by <i>TK0795</i>, which shares weak homology with the modified cytosine restriction protein McrB but is functionally uncharacterized<sup><a aria-label="Reference 49" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR49" id="ref-link-section-d18519924e1945" title="Fukui, T. et al. Complete genome sequence of the hyperthermophilic archaeon Thermococcus kodakaraensis KOD1 and comparison with Pyrococcus genomes. Genome Res. 15, 352–363 (2005).">49</a></sup>. Although both <i>difL</i> and <i>difR</i> contained <i>dif</i>, we did not see a clear specific band corresponding to XerA (33 kDa) under our experimental conditions.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Identification of boundary 1-associated proteins and their impacts on genome architecture." id="figure-6"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig6">Fig. 6: Identification of boundary 1-associated proteins and their impacts on genome architecture.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-025-56197-y/figures/6" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig6_HTML.png?as=webp" type="image/webp"><img alt="figure 6" aria-describedby="Fig6" height="708" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig6_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p><b>a</b> Genomic regions covered by the two probes (<i>difL</i> and <i>difR</i>) for DNA affinity purification are indicated by black arrows. The other features are depicted as in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig3">3a</a>. <b>b</b> Workflow of DNA affinity purification. <b>c</b> Proteins from the DNA affinity purification experiment were separated by SDS-PAGE and visualized by silver staining. 0.1% of the lysate was run as input. Positions of common specific bands for the <i>difR</i> and <i>difL</i> probes are indicated by white triangles (bands 1 and 2). Bands indicated by black rectangles were subjected to mass spectrometry (MS). Source data are provided as a Source Data file. <b>d</b> Comparison of 3C-seq contact maps (5-kb resolution) from KU216 (upper left triangles) and derivative strains lacking either <i>trmBL2</i> or <i>TK0795</i> (lower left triangles). <b>e</b> Differential contact maps displaying log<sub>2</sub> ratios of contact frequencies between KU216 and deletion strains are shown as in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig2">2b</a>. <b>f</b> Contact maps of genomic regions surrounding boundary 1 (upper panels) and boundary 2 (lower panels) were generated using the 3C-seq data from Δ<i>trmBL2</i> and Δ<i>TK0795</i> cells. <b>g</b> Insulation score profiles of boundary 1 (top panel) and boundary 2 (bottom panel) are shown for KU216 (black solid line) and deletion strains (dashed lines, Δ<i>trmBL2</i>: red, Δ<i>TK0795</i>: orange). Note that the graphs on the top panel cover an inversion breakpoint that has caused artificial drops in the score. Source data are provided as a <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM8">Source Data</a> file.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 6" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure6 Full size image" data-track-label="button" href="/articles/s41467-025-56197-y/figures/6" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>To investigate whether the identified proteins are required to build boundary 1, we constructed deletion mutants of <i>trmBL2</i> and <i>TK0795</i> using KU216 as a parental strain. 3C-seq demonstrated that the <i>trmBL2</i> deletion caused a complete loss of boundary 1, while the <i>TK0795</i> deletion rather slightly enhanced the contact insulation at boundary 1 (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig6">6d–g</a>). The loss of boundary 1 in the Δ<i>trmBL2</i> mutant was also confirmed by HiCDB (Supplementary Data <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM3">1</a>). The insulation of contacts was largely maintained at boundary 2 in both mutants. Except for the loss of boundary 1 in the Δ<i>trmBL2</i> mutant, we did not see a clear structural change in either of the mutants (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig6">6d, e</a>). Taken together, TrmBL2 is responsible for specifying boundary 1, while another, unknown factor defines boundary 2.</p><h3 class="c-article__sub-heading" id="Sec7">TrmBL2 localizes Smc-ScpAB to boundary 1 and tens of other loci</h3><p>We wondered whether TrmBL2 positions Smc-ScpAB at boundary 1 as eukaryotic CTCF does for cohesin to form TAD boundaries<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR6" id="ref-link-section-d18519924e2082" title="Davidson, I. F. &amp; Peters, J. M. Genome folding through loop extrusion by SMC complexes. Nat. Rev. Mol. Cell Biol. 22, 445–464 (2021).">6</a>,<a aria-label="Reference 10" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR10" id="ref-link-section-d18519924e2085" title="Nora, E. P. et al. Targeted degradation of CTCF decouples local insulation of chromosome domains from genomic compartmentalization. Cell 169, 930–944 e922 (2017).">10</a></sup>. To test this possibility, we carried out ChIP-seq of Smc and TrmBL2 using antisera against these proteins. The specificity of the antisera was confirmed by performing ChIP-seq on Δ<i>smc</i> and Δ<i>trmBL2</i> cells (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7a, b</a>). On the genome of KU216, Smc was most highly enriched at boundary 1 (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7a</a>). At a finer scale, boundary 1 was nestled by two sharp peaks of Smc, each of which overlapped with a distinct TrmBL2 peak (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7c</a>). Furthermore, these Smc peaks were almost completely lost in Δ<i>trmBL2</i> cells. These results support the role of TrmBL2 in localizing Smc-ScpAB to boundary 1.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="TrmBL2 is required to localize Smc-ScpAB to boundary 1 and non-boundary regions." id="figure-7"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig7">Fig. 7: TrmBL2 is required to localize Smc-ScpAB to boundary 1 and non-boundary regions.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-025-56197-y/figures/7" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig7_HTML.png?as=webp" type="image/webp"><img alt="figure 7" aria-describedby="Fig7" height="677" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig7_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-7-desc"><p><b>a</b> ChIP-seq tracks of Smc for the whole genomic regions of KU216, Δ<i>trmBL2</i>, and Δ<i>smc</i> strains. Enrichment of immunoprecipitated versus input DNA (IP/input) is shown at 1-kb resolution. The genomic positions of boundaries 1 and 2 in KU216 are indicated by gray dashed lines. <b>b</b> ChIP-seq tracks of TrmBL2 for the whole genomic regions of KU216 and Δ<i>trmBL2</i> strains are shown as in (<b>a</b>). ChIP-seq tracks of Smc (KU216: blue solid line, Δ<i>trmBL2</i>: cyan dashed line) and TrmBL2 (KU216: orange solid line) are shown for boundary 1 (<b>c</b>), boundary 2 (<b>d</b>), and non-boundary loci (<b>e</b>, <b>f</b>) in <i>T. kodakarensis</i>. Enrichment of immunoprecipitated versus input DNA (IP/input) is shown at 50-bp resolution. The locations of genes, boundaries, and <i>dif</i> are indicated as in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig3">3a</a>. <b>g</b> Venn diagram showing the overlap of Smc and TrmBL2 peaks detected by ChIP-seq analysis of KU216. Statistical significance of the overlap was determined by two-sided permutation test. <b>h</b> MEME-ChIP<sup><a aria-label="Reference 122" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR122" id="ref-link-section-d18519924e2169" title="Machanick, P. &amp; Bailey, T. L. MEME-ChIP: motif analysis of large DNA datasets. Bioinformatics 27, 1696–1697 (2011).">122</a></sup> was performed to search for DNA motifs enriched in the three peak groups in (<b>g</b>). Statistical significance of the enrichment was evaluated using <i>E-</i>values. Only the most significant motif is shown for each group. <b>i</b> Smc occupancy was calculated for non-boundary loci forming Smc peaks in KU216. Occupancy was defined as the ChIP-seq IP/input ratio of a 200-bp region centered at the peak summit. Statistical significance of the difference was determined by two-sided Wilcoxon rank sum test. Source data are provided as a Source Data file. <b>j</b> Occupancies of TrmBL2 and Smc were plotted for their common binding sites shown in (<b>g</b>). The binding sites adjacent to boundary 1 are highlighted in red. The Spearman rank correlation coefficient (<i>r</i>) and corresponding two-sided <i>p</i>-value (<i>p</i>) are also shown. Source data are provided as a <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM8">Source Data</a> file.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 7" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure7 Full size image" data-track-label="button" href="/articles/s41467-025-56197-y/figures/7" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>The peak summits of TrmBL2 at boundary 1 were reproducibly located ~100 bp inward from the cognate peak summits of Smc (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7c</a> and Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">7</a>). This distribution could be explained if TrmBL2, akin to CTCF, serves as an asymmetric barrier for the Smc-ScpAB complex that diffuses along DNA or actively extrudes a loop<sup><a aria-label="Reference 16" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR16" id="ref-link-section-d18519924e2219" title="Sanborn, A. L. et al. Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes. Proc. Natl. Acad. Sci. USA 112, E6456–E6465 (2015).">16</a>,<a aria-label="Reference 63" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR63" id="ref-link-section-d18519924e2222" title="Li, Y. et al. The structural basis for cohesin-CTCF-anchored loops. Nature 578, 472–476 (2020).">63</a></sup>. However, this scenario does not fit well with the crystal structure of DNA-bound <i>Pyrococcus furiosus</i> TrmBL2, in which TrmBL2 forms a dimer-of-dimer whose overall structure is symmetric<sup><a aria-label="Reference 64" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR64" id="ref-link-section-d18519924e2229" title="Ahmad, M. U. D. et al. Structural insights into nonspecific binding of DNA by TrmBL2, an archaeal chromatin protein. J. Mol. Biol. 427, 3216–3229 (2015).">64</a></sup>. Another possibility is that each of the two closely located TrmBL2-binding sites at boundary 1 stalls Smc-ScpAB with high efficiency, possibly ~100%. In this case, as loading of Smc-ScpAB onto the intervening region should be a relatively rare event, the left and right binding sites for TrmBL2 will encounter Smc-ScpAB mostly from the left and right, respectively. This directionally biased encounter and subsequent stalling will generate the observed ChIP-seq profile of Smc.</p><p>In KU216, a high ChIP-seq peak of Smc was also found near boundary 2 (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7a, d</a>). In contrast to boundary 1, however, this region was not enriched for TrmBL2. Moreover, deletion of <i>trmBL2</i> did not reduce the Smc binding to boundary 2 (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7d</a>). These results are consistent with a TrmBL2-independent formation of boundary 2 (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig6">6d–g</a>).</p><p>Our ChIP-seq identified a total of 90 Smc peaks in KU216, among which 66 overlapped with TrmBL2 peaks in a statistically significant manner (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7c, e–g</a>). Common and TrmBL2-specific binding sites were enriched for very similar AT-rich motifs, whereas Smc-specific binding sites were characterized by a dissimilar AT-rich motif (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7h</a>). As observed for boundary 1, many non-boundary regions that were co-occupied by Smc and TrmBL2 in KU216 showed decreased binding of Smc in Δ<i>trmBL2</i> cells (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7i</a>). This was not due to global depletion of Smc from the chromosome, because the loss of TrmBL2 did not reduce Smc occupancy at Smc-specific binding sites (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7d, i</a>). Taken together, TrmBL2 is required to position Smc-ScpAB at many non-boundary regions as well as boundary 1.</p><p>To explore the potential difference between boundary 1 and other colocalization sites of TrmBL2 and Smc-ScpAB, we plotted the occupancies of TrmBL2 and Smc at their common binding sites (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7j</a>). The two values showed a high level of positive correlation on average (<i>r</i> = 0.67). However, Smc was disproportionally enriched at a number of loci despite their modest enrichment for TrmBL2, and this disproportionality was most evident at boundary 1. The variability is also exemplified by the ChIP-seq profiles in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7e, f</a>, which display different heights of Smc peaks relative to the cognate TrmBL2 peaks. These results suggest that TrmBL2 positions Smc-ScpAB very effectively at boundary 1 (possibly with ~100% efficiency as described above) and much less, but still significantly, at non-boundary sites. Importantly, this difference cannot simply be explained by the level of TrmBL2 occupancy.</p><h3 class="c-article__sub-heading" id="Sec8">DNA sequences correlate with the efficiency of TrmBL2-dependent positioning of Smc-ScpAB</h3><p>How does TrmBL2 position Smc-ScpAB with variable efficiency? We noticed that the intergenic region encompassing boundary 1 exhibited unusually high AT content (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig8">8a</a>). The percentage was especially high around the peaks of Smc and TrmBL2, reaching a maximum of over 75% at 200-bp scale, while the overall AT content of the <i>T. kodakarensis</i> genome is 48%. AT content was also high at other loci co-occupied by TrmBL2 and Smc (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">8a, b</a>). These observations led us to hypothesize that AT-rich sequences affect how efficiently TrmBL2 positions Smc-ScpAB.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="DNA sequence features correlating with TrmBL2-dependent stalling of Smc-ScpAB." id="figure-8"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig8">Fig. 8: DNA sequence features correlating with TrmBL2-dependent stalling of Smc-ScpAB.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-025-56197-y/figures/8" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig8_HTML.png?as=webp" type="image/webp"><img alt="figure 8" aria-describedby="Fig8" height="657" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_Fig8_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-8-desc"><p><b>a</b> AT-content track of boundary 1. AT content was calculated with sliding window size of 200 bp and step size of 100 bp. The peak positions of Smc and TrmBL2 are indicated by black and white triangles, respectively. The other features are depicted as in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig3">3a</a>. <b>b</b> AT content and Smc positioning efficiency were plotted for 57 TrmBL2-binding sites that showed TrmBL2-dependent localization of Smc. The Spearman rank correlation coefficient (<i>r</i>) and corresponding two-sided <i>p</i>-value (<i>p</i>) are also shown. The two sites associated with boundary 1 are highlighted in red. Source data are provided as a Source Data file. <b>c</b> Spearman correlation coefficients between DNA tetranucleotide frequencies and Smc positioning efficiency were calculated for the 57 colocalization sites. Only the sequences with high statistical significance (two-sided <i>p-</i>value adjusted for multiple comparisons &lt;0.05) are shown with the adjusted <i>p-</i>values. Source data are provided as a Source Data file. <b>d</b> The Spearman correlation coefficient and corresponding two-sided <i>p-</i>value between the frequency of all types of A-tract tetranucleotides and Smc positioning efficiency were calculated for the 57 colocalization sites. The partial Spearman correlation coefficient and corresponding two-sided <i>p-</i>value were also calculated to examine their relationship without the contribution of AT content. Source data are provided as a Source Data file. <b>e</b> Partial Spearman correlation coefficients between Smc positioning efficiency and structural properties of the 57 colocalization sites were calculated by removing the contribution of AT content. Two-sided <i>p-</i>values adjusted for multiple comparisons are also shown. Source data are provided as a <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM8">Source Data</a> file. <b>f</b> Model for the TrmBL2-mediated regulation of Smc-ScpAB dynamics in <i>T. kodakarensis</i>. The diagram assumes that the complex functions as a dimer as suggested for bacterial Smc-ScpAB<sup><a aria-label="Reference 25" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR25" id="ref-link-section-d18519924e2360" title="Wang, X., Brandao, H. B., Le, T. B., Laub, M. T. &amp; Rudner, D. Z. Bacillus subtilis SMC complexes juxtapose chromosome arms as they travel from origin to terminus. Science 355, 524–527 (2017).">25</a></sup>. ScpB probably forms a complex with Smc (green) and ScpA (magenta) but is not included in the diagram for simplicity. See “Discussion” for more detail.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 8" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure8 Full size image" data-track-label="button" href="/articles/s41467-025-56197-y/figures/8" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>To explore this possibility, we defined a simple metric named Smc positioning efficiency. When a given TrmBL2-binding site overlapped with an Smc-binding site (each defined as a 200-bp region centered at the peak summit), their occupancy ratio log<sub>2</sub>(Smc/TrmBL2) was defined as the Smc positioning efficiency of the TrmBL2-binding site. If TrmBL2 positions Smc-ScpAB more efficiently at a given locus than another, the value of Smc positioning efficiency is expected to be higher at the former. We calculated Smc positioning efficiency for the 57 TrmBL2-binding sites that overlapped with TrmBL2-dependent Smc-binding sites (see “Methods” for more detail). Comparison of Smc positioning efficiency with the AT content of the TrmBL2-bound 200-bp region revealed that the two metrics were well correlated with each other (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig8">8b</a>). In contrast, AT content showed a weaker correlation with Smc occupancy and no significant correlation with TrmBL2 occupancy (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">8c, d</a>). Thus, AT content is strongly linked to the positioning efficiency of Smc-ScpAB but not to the DNA binding affinities of Smc-ScpAB and TrmBL2.</p><p>We further explored what feature pertinent to AT content could affect Smc positioning efficiency. The high growth temperature (85 °C) of <i>T. kodakarensis</i> raises the possibility that AT-rich sequences affect Smc positioning via DNA melting. To test this, we predicted the melting temperatures (T<sub>m</sub>) of the TrmBL2-bound 200-bp regions used for the calculation of Smc positioning efficiency (see “Methods” for more detail). The predicted T<sub>m</sub> value and Smc positioning efficiency showed a strong inverse relationship with a Spearman correlation coefficient of –0.63 (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">8e</a>). However, the predicted T<sub>m</sub> value and AT content also had a very strong negative correlation with a Spearman correlation coefficient of –0.99, making it difficult to infer which of the two factors is more directly related to Smc positioning. We next considered whether Smc positioning efficiency is associated with a specific form of AT-rich sequences. To test this, we counted occurrence of each of all possible DNA tetranucleotides in the TrmBL2-bound 200-bp regions. The Spearman correlation coefficient was calculated between the tetranucleotide frequency and Smc positioning efficiency. This analysis identified all types of so-called “A-tract” tetranucleotides (AAAA/TTTT, AAAT/ATTT, and AATT/AATT) as the most positively correlated sequences (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig8">8c</a>). A-tracts are four or more consecutive A:T base pairs without a TpA step<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR65" id="ref-link-section-d18519924e2402" title="Haran, T. E. &amp; Mohanty, U. The unique structure of A-tracts and intrinsic DNA bending. Q. Rev. Biophys. 42, 41–81 (2009).">65</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR66" id="ref-link-section-d18519924e2402_1" title="Hagerman, P. J. Sequence-directed curvature of DNA. Annu. Rev. Biochem. 59, 755–781 (1990).">66</a>,<a aria-label="Reference 67" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR67" id="ref-link-section-d18519924e2405" title="Marin-Gonzalez, A., Vilhena, J. G., Perez, R. &amp; Moreno-Herrero, F. A molecular view of DNA flexibility. Q Rev. Biophys. 54, e8 (2021).">67</a></sup>. The total number of the A-tract tetranucleotides showed a significant correlation with Smc positioning efficiency even if the contribution of AT content was removed by partial correlation analysis (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig8">8d</a>). This indicates a potential role of A-tracts in localizing Smc-ScpAB to TrmBL2-binding sites.</p><p>A-tracts display unique structural features in multiple ways. For example, long poly(A) sequences (often 10-20 bp or even longer) disfavor nucleosome wrapping in eukaryotes, while A-tracts induce DNA curvature when in phase with the helical pitch of DNA<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR67" id="ref-link-section-d18519924e2415" title="Marin-Gonzalez, A., Vilhena, J. G., Perez, R. &amp; Moreno-Herrero, F. A molecular view of DNA flexibility. Q Rev. Biophys. 54, e8 (2021).">67</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR68" id="ref-link-section-d18519924e2415_1" title="Segal, E. &amp; Widom, J. Poly(dA:dT) tracts: major determinants of nucleosome organization. Curr. Opin. Struct. Biol. 19, 65–71 (2009).">68</a>,<a aria-label="Reference 69" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR69" id="ref-link-section-d18519924e2418" title="Marin-Gonzalez, A. et al. Understanding the paradoxical mechanical response of in-phase A-tracts at different force regimes. Nucleic Acids Res. 48, 5024–5036 (2020).">69</a></sup>. A-tracts also form narrow and negatively charged minor grooves<sup><a aria-label="Reference 70" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR70" id="ref-link-section-d18519924e2422" title="Rohs, R. et al. The role of DNA shape in protein-DNA recognition. Nature 461, 1248–1253 (2009).">70</a></sup>. The observed correlation between A-tract frequency and Smc positioning efficiency raises the possibility that TrmBL2-dependent localization of Smc-ScpAB is affected by structural properties of the underlying sequence. To test this, we quantified structural features associated with A-tracts for colocalization sites solely by using their sequence information. We first focused on persistence length (PL), within which a polymer segment can be seen as a straight rod. Although the PL of DNA is often regarded as its stiffness to bending deformation, the metric can actually be decomposed into dynamic and static PLs, which reflect bona fide stiffness and intrinsic shape of the DNA chain, respectively<sup><a aria-label="Reference 71" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR71" id="ref-link-section-d18519924e2426" title="Olson W. K. DNA Bending &amp; Curvature (Adenine Press, 1988).">71</a>,<a aria-label="Reference 72" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR72" id="ref-link-section-d18519924e2429" title="Mitchell, J. S., Glowacki, J., Grandchamp, A. E., Manning, R. S. &amp; Maddocks, J. H. Sequence-dependent persistence lengths of DNA. J. Chem. Theory Comput. 13, 1539–1555 (2017).">72</a></sup>. These two types of PLs have recently been estimated for all possible DNA tetranucleotides using all-atom molecular dynamics (MD) simulation data<sup><a aria-label="Reference 73" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR73" id="ref-link-section-d18519924e2433" title="Velasco-Berrelleza, V. et al. SerraNA: a program to determine nucleic acids elasticity from simulation data. Phys. Chem. Chem. Phys. 22, 19254–19266 (2020).">73</a></sup>. We used these values to calculate the mean dynamic and static PLs of all tetranucleotide steps in each stalling site, seeing the values as the local PLs of the site. We also used the recently developed software Deep DNAshape<sup><a aria-label="Reference 74" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR74" id="ref-link-section-d18519924e2437" title="Li, J., Chiu, T. P. &amp; Rohs, R. Predicting DNA structure using a deep learning method. Nat. Commun. 15, 1243 (2024).">74</a></sup> to predict the width and electrostatic potential of the minor groove averaged over the bases of each sequence. Partial correlation analysis controlling the effect of AT content uncovered that, among these four structural features, the local static PL and the minor-groove electrostatic potential showed significant positive and negative correlations with Smc positioning efficiency, respectively (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig8">8e</a>). Thus, these two features may be causally related to the positioning of Smc-ScpAB.</p><p>Current models posit that DNA bending is prerequisite for loop-extruding SMC complexes to reel in DNA for translocation<sup><a aria-label="Reference 7" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR7" id="ref-link-section-d18519924e2449" title="Kim, E., Barth, R. &amp; Dekker, C. Looping the genome with SMC complexes. Annu. Rev. Biochem. 92, 15–41 (2023).">7</a>,<a aria-label="Reference 75" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR75" id="ref-link-section-d18519924e2452" title="Dekker, C., Haering, C. H., Peters, J. M. &amp; Rowland, B. D. How do molecular motors fold the genome? Science 382, 646–648 (2023).">75</a>,<a aria-label="Reference 76" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR76" id="ref-link-section-d18519924e2455" title="Marko, J. F., De Los Rios, P., Barducci, A. &amp; Gruber, S. DNA-segment-capture model for loop extrusion by structural maintenance of chromosome (SMC) protein complexes. Nucleic Acids Res. 47, 6956–6972 (2019).">76</a></sup>. Given this, it is somewhat surprising that the local dynamic PL of DNA did not significantly contribute to TrmBL2-dependent positioning of Smc. Experimental evidence suggests that SMC complexes capture a bent DNA segment of ~200 bp to carry out a loop extrusion cycle<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR77" id="ref-link-section-d18519924e2459" title="Strick, T. R., Kawaguchi, T. &amp; Hirano, T. Real-time detection of single-molecule DNA compaction by condensin I. Curr. Biol. 14, 874–880 (2004).">77</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR78" id="ref-link-section-d18519924e2459_1" title="Ryu, J. K. et al. Condensin extrudes DNA loops in steps up to hundreds of base pairs that are generated by ATP binding events. Nucleic Acids Res. 50, 820–832 (2022).">78</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR79" id="ref-link-section-d18519924e2459_2" title="Cui, Y., Petrushenko, Z. M. &amp; Rybenkov, V. V. MukB acts as a macromolecular clamp in DNA condensation. Nat. Struct. Mol. Biol. 15, 411–418 (2008).">79</a>,<a aria-label="Reference 80" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR80" id="ref-link-section-d18519924e2462" title="Sun, M., Nishino, T. &amp; Marko, J. F. The SMC1-SMC3 cohesin heterodimer structures DNA through supercoiling-dependent loop formation. Nucleic Acids Res. 41, 6149–6160 (2013).">80</a></sup>. The effect of DNA stiffness may need to be evaluated at this length scale. This prompted us to estimate the dynamic PLs of entire 200-bp segments (here referred to as gross dynamic PL) of colocalization sites by performing coarse-grained MD simulations<sup><a aria-label="Reference 81" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR81" id="ref-link-section-d18519924e2466" title="Freeman, G. S., Hinckley, D. M., Lequieu, J. P., Whitmer, J. K. &amp; de Pablo, J. J. Coarse-grained modeling of DNA curvature. J. Chem. Phys. 141, 165103 (2014).">81</a></sup>. To save the computational cost for the modeling, we selected and analyzed the top five and bottom five colocalization sites in terms of Smc positioning efficiency. The directional correlation decay of the modeled DNA polymer was used to calculate gross dynamic PL. The obtained value actually showed a high positive correlation with Smc positioning efficiency (<i>r</i> = 0.85) (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">8f</a>). However, the gross dynamic PL varied only from 64.8 to 68.1 nm among the ten sequences, whereas the local static PL varied in a much larger range of 88.0 to 211 nm among the 57 colocalization sites (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">8g</a>). At this stage, it remains unclear whether the small differences in the gross dynamic PLs contribute to the variability in Smc positioning efficiency. It is possible that our MD simulation analysis has underestimated the gross dynamic PL.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec9-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec9">Discussion</h2><div class="c-article-section__content" id="Sec9-content"><p>In this study, we have demonstrated that Smc, ScpA, and ScpB are all required for the domain formation in <i>T. kodakarensis</i> (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig2">2</a>). This finding raises the possibility that these proteins function, at least in some archaeal lineages, as a ternary complex akin to bacterial Smc-ScpAB. Indeed, our analyses using yeast two-hybrid assays and modeling by AlphaFold2<sup><a aria-label="Reference 82" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR82" id="ref-link-section-d18519924e2495" title="Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. Nature 596, 583–589 (2021).">82</a></sup> suggest that archaeal ScpA and ScpB can form a complex when Smc triggers a conformational change of ScpA (Takemata, Yamauchi, Takada, and Atomi, manuscript in preparation).</p><p>In <i>T. kodakarensis</i>, TrmBL2 is required to localize Smc-ScpAB to tens of loci, most strikingly at boundary 1 (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7</a>). We propose that TrmBL2, analogous to eukaryotic CTCF<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR6" id="ref-link-section-d18519924e2508" title="Davidson, I. F. &amp; Peters, J. M. Genome folding through loop extrusion by SMC complexes. Nat. Rev. Mol. Cell Biol. 22, 445–464 (2021).">6</a>,<a aria-label="Reference 10" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR10" id="ref-link-section-d18519924e2511" title="Nora, E. P. et al. Targeted degradation of CTCF decouples local insulation of chromosome domains from genomic compartmentalization. Cell 169, 930–944 e922 (2017).">10</a>,<a aria-label="Reference 63" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR63" id="ref-link-section-d18519924e2514" title="Li, Y. et al. The structural basis for cohesin-CTCF-anchored loops. Nature 578, 472–476 (2020).">63</a></sup>, stalls loop-extruding Smc-ScpAB and that the stalling occurs most frequently at the <i>dif</i>-proximal TrmBL2-binding sites, resulting in the formation of boundary 1 (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig8">8f</a>). Assuming high stalling efficiencies (possibly close to 100%) for the TrmBL2-enriched regions at boundary 1, this model can explain the distribution pattern of Smc observed at boundary 1 (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7c</a> and Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">7</a>). A potential alternative explanation for the colocalization of Smc and TrmBL2 is that TrmBL2 serves as a loader of Smc-ScpAB. A simulation study showed that, if a loop-extruding factor shapes a boundary at its loading site as postulated above, it must perform asymmetric loop extrusion combined with one-dimensional diffusion along DNA<sup><a aria-label="Reference 83" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR83" id="ref-link-section-d18519924e2531" title="Banigan, E. J., van den Berg, A. A., Brandao, H. B., Marko, J. F. &amp; Mirny, L. A. Chromosome organization by one-sided and two-sided loop extrusion. Elife 9, e53558 (2020).">83</a></sup>. In contrast to this requirement, the ability of the ectopic <i>boundary1</i><sup>Δ<i>dif</i></sup> sequence to serve as an anchor for loop- and stripe-type contacts supports the idea that this sequence can stably retain Smc-ScpAB (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig5">5c</a>). In addition, although the loop extrusion of Smc-ScpAB has not been directly observed in vitro, Hi-C studies suggested that the complex forms a dimer that extrudes a DNA loop symmetrically<sup><a aria-label="Reference 25" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR25" id="ref-link-section-d18519924e2546" title="Wang, X., Brandao, H. B., Le, T. B., Laub, M. T. &amp; Rudner, D. Z. Bacillus subtilis SMC complexes juxtapose chromosome arms as they travel from origin to terminus. Science 355, 524–527 (2017).">25</a></sup>. In vitro single-molecule experiments also demonstrated that the Kite-containing eukaryotic SMC complex SMC5/6 and the bacterial MukBEF-like complex Wadjet form dimeric supercomplexes that extrude DNA loops symmetrically<sup><a aria-label="Reference 84" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR84" id="ref-link-section-d18519924e2550" title="Pradhan, B. et al. The Smc5/6 complex is a DNA loop-extruding motor. Nature 616, 843–848 (2023).">84</a>,<a aria-label="Reference 85" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR85" id="ref-link-section-d18519924e2553" title="Pradhan, B. et al. Loop extrusion-mediated plasmid DNA cleavage by the bacterial SMC Wadjet complex. Mol. Cell 85, 107–116 (2024).">85</a></sup>. Taken together, it is more plausible that <i>T. kodakarensis</i> Smc-ScpAB is also a symmetric loop extruder and is stalled by TrmBL2 for the boundary formation.</p><p>We note a number of differences in the Smc-mediated genome organization between archaea and bacteria. First, the above model places <i>T. kodakarensis</i> Smc-ScpAB as a key player for a eukaryotic-like mechanism of domain formation, whereas bacterial Smc-ScpAB is not involved in sculpting domains<sup><a aria-label="Reference 20" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR20" id="ref-link-section-d18519924e2566" title="Le, T. B., Imakaev, M. V., Mirny, L. A. &amp; Laub, M. T. High-resolution mapping of the spatial organization of a bacterial chromosome. Science 342, 731–734 (2013).">20</a>,<a aria-label="Reference 21" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR21" id="ref-link-section-d18519924e2569" title="Wang, X. et al. Condensin promotes the juxtaposition of DNA flanking its loading site in Bacillus subtilis. Genes Dev. 29, 1661–1675 (2015).">21</a></sup>. The chromosomal loading of bacterial Smc-ScpAB is mediated by the DNA-binding CTPase ParB, which binds to <i>parS</i> centromere-like sequences scattered around the origin<sup><a aria-label="Reference 27" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR27" id="ref-link-section-d18519924e2576" title="Gruber, S. &amp; Errington, J. Recruitment of condensin to replication origin regions by ParB/SpoOJ promotes chromosome segregation in B. subtilis. Cell 137, 685–696 (2009).">27</a>,<a aria-label="Reference 28" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR28" id="ref-link-section-d18519924e2579" title="Sullivan, N. L., Marquis, K. A. &amp; Rudner, D. Z. Recruitment of SMC by ParB-parS organizes the origin region and promotes efficient chromosome segregation. Cell 137, 697–707 (2009).">28</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR86" id="ref-link-section-d18519924e2582" title="Jalal A. S., Tran N. T. &amp; Le T. B. ParB spreading on DNA requires cytidine triphosphate in vitro. Elife 9, e53515 (2020).">86</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR87" id="ref-link-section-d18519924e2582_1" title="Soh, Y. M. et al. Self-organization of parS centromeres by the ParB CTP hydrolase. Science 366, 1129–1133 (2019).">87</a>,<a aria-label="Reference 88" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR88" id="ref-link-section-d18519924e2585" title="Osorio-Valeriano, M. et al. ParB-type DNA segregation proteins Are CTP-dependent molecular switches. Cell 179, 1512–1524.e1515 (2019).">88</a></sup>. Notably, the <i>T. kodakarensis</i> homolog of ParB functions as an ADP-dependent serine kinase involved in cysteine biogenesis, and thus this enzyme does not appear to play a role in Smc-ScpAB loading<sup><a aria-label="Reference 89" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR89" id="ref-link-section-d18519924e2593" title="Makino, Y. et al. An archaeal ADP-dependent serine kinase involved in cysteine biosynthesis and serine metabolism. Nat. Commun. 7, 13446 (2016).">89</a></sup>.</p><p>In addition to these differences, our study suggests a common feature of the SMC flux in diverse prokaryotes—disfavoring SMC traversal across the chromosome dimer resolution site <i>dif</i>. <i>T. kodakarensis</i> Smc-ScpAB is likely stalled on both sides of <i>dif</i> to form boundary 1, whereas the bacterial Smc-ScpAB complex traveling from the replication origin is unloaded by XerD before the complex reaches the <i>dif</i> site in <i>ter</i><sup><a aria-label="Reference 60" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR60" id="ref-link-section-d18519924e2614" title="Karaboja, X. et al. XerD unloads bacterial SMC complexes at the replication terminus. Mol. Cell 81, 756–766 e758 (2021).">60</a></sup>. <i>E. coli</i> MukBEF is also kept away from the <i>dif-</i>containing <i>ter</i> region by the unloader protein MatP, whose binding sites are scattered in <i>ter</i><sup><a aria-label="Reference 26" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR26" id="ref-link-section-d18519924e2630" title="Makela, J. &amp; Sherratt, D. J. Organization of the Escherichia coli chromosome by a MukBEF axial core. Mol. Cell 78, 250–260.e5 (2020).">26</a>,<a aria-label="Reference 90" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR90" id="ref-link-section-d18519924e2633" title="Mercier, R. et al. The MatP/matS site-specific system organizes the terminus region of the E. coli chromosome into a macrodomain. Cell 135, 475–485 (2008).">90</a>,<a aria-label="Reference 91" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR91" id="ref-link-section-d18519924e2636" title="Nolivos, S. et al. MatP regulates the coordinated action of topoisomerase IV and MukBEF in chromosome segregation. Nat. Commun. 7, 10466 (2016).">91</a></sup>. These similarities may indicate a common role of prokaryotic SMC complexes in <i>dif</i> regions (see below).</p><p><i>dif</i> and <i>dif-</i>like sequences are found proximal to boundary structures in <i>T. acidophilum</i>, <i>H. volcanii</i>, and <i>H. salinarum</i> (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig3">3</a>). All of these organisms possess both Smc proteins and TrmBL2-like proteins<sup><a aria-label="Reference 33" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR33" id="ref-link-section-d18519924e2665" title="Jeon, J. H. et al. Evidence for binary Smc complexes lacking kite subunits in archaea. IUCrJ 7, 193–206 (2020).">33</a>,<a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e2668" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a>,<a aria-label="Reference 61" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR61" id="ref-link-section-d18519924e2671" title="Maruyama, H. et al. Histone and TK0471/TrmBL2 form a novel heterogeneous genome architecture in the hyperthermophilic archaeon Thermococcus kodakarensis. Mol. Biol. Cell 22, 386–398 (2011).">61</a>,<a aria-label="Reference 92" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR92" id="ref-link-section-d18519924e2674" title="Herrmann, U. &amp; Soppa, J. Cell cycle-dependent expression of an essential SMC-like protein and dynamic chromosome localization in the archaeon Halobacterium salinarum. Mol. Microbiol. 46, 395–409 (2002).">92</a></sup>, suggesting that the mechanism and function of the <i>dif-</i>associated boundary are conserved in multiple euryarchaeal lineages. Despite this presumed conservation, loss of the <i>dif</i> boundary does not cause any discernible phenotype in <i>T. kodakarensis</i>. What are the physiological consequences of the <i>dif</i>-associated boundary in archaea? Here, we describe two potential roles, presuming that the boundary formation involves loop extrusion by Smc-ScpAB.</p><ol class="u-list-style-none">
<li>
<span class="u-custom-list-number">(1)</span>
<p>The <i>dif-</i>boundary formation conceivably promotes chromosome dimer resolution, because two <i>dif</i> copies on the chromosome dimer will be brought into proximity by loop-extrusion-mediated formation of a loop array between the <i>dif</i> boundaries. In polyploid euryarchaeal cells, loop extrusion will also be useful to selectively drive intra-molecular pairing of <i>dif</i> sequences over inter-molecular pairing. The Smc-dependent synapsis of <i>dif</i> sites and their recombination by XerA might become crucial when chromosome dimers are frequently formed due to increased DNA damage.</p>
</li>
<li>
<span class="u-custom-list-number">(2)</span>
<p>A recent Hi-C study has proposed that the MukBEF-free <i>dif/ter</i> region in <i>E. coli</i> serves as a hub to which DNA catenanes are brought to be resolved by the type IIA topoisomerase Topo IV<sup><a aria-label="Reference 93" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR93" id="ref-link-section-d18519924e2732" title="Conin, B. et al. Extended sister-chromosome catenation leads to massive reorganization of the E. coli genome. Nucleic Acids Res. 50, 2635–2650 (2022).">93</a></sup>. In accordance with this localized decatenation activity, <i>ter</i> possesses a major Topo IV-binding site where Topo IV is targeted by XerC in <i>E. coli</i><sup><a aria-label="Reference 94" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR94" id="ref-link-section-d18519924e2741" title="El Sayyed, H. et al. Mapping topoisomerase IV binding and activity sites on the E. coli genome. PLoS Genet. 12, e1006025 (2016).">94</a>,<a aria-label="Reference 95" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR95" id="ref-link-section-d18519924e2744" title="Sutormin, D., Galivondzhyan, A., Gafurov, A. &amp; Severinov, K. Single-nucleotide resolution detection of Topo IV cleavage activity in the Escherichia coli genome with Topo-Seq. Front. Microbiol. 14, 1160736 (2023).">95</a></sup>. A similar decatenation hub may be installed in the archaeal <i>dif</i> boundary region, and loop-extruding Smc may bring catenation links to this hub. In line with this idea, loop extrusion has been suggested to direct DNA entanglements to the decatenation hub in <i>E. coli</i> and to CTCF boundaries in mammalian cells<sup><a aria-label="Reference 93" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR93" id="ref-link-section-d18519924e2755" title="Conin, B. et al. Extended sister-chromosome catenation leads to massive reorganization of the E. coli genome. Nucleic Acids Res. 50, 2635–2650 (2022).">93</a>,<a aria-label="Reference 96" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR96" id="ref-link-section-d18519924e2758" title="Canela, A. et al. Genome organization drives chromosome fragility. Cell 170, 507–521.e518 (2017).">96</a></sup>. Although it remains unclear whether a decatenation hub exists in bacteria possessing Smc-ScpAB, this complex has been implicated in minimizing sister DNA interconnections—either DNA catenanes or protein-mediated bridges—through loop extrusion to ensure efficient chromosome segregation<sup><a aria-label="Reference 25" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR25" id="ref-link-section-d18519924e2762" title="Wang, X., Brandao, H. B., Le, T. B., Laub, M. T. &amp; Rudner, D. Z. Bacillus subtilis SMC complexes juxtapose chromosome arms as they travel from origin to terminus. Science 355, 524–527 (2017).">25</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR27" id="ref-link-section-d18519924e2765" title="Gruber, S. &amp; Errington, J. Recruitment of condensin to replication origin regions by ParB/SpoOJ promotes chromosome segregation in B. subtilis. Cell 137, 685–696 (2009).">27</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR28" id="ref-link-section-d18519924e2765_1" title="Sullivan, N. L., Marquis, K. A. &amp; Rudner, D. Z. Recruitment of SMC by ParB-parS organizes the origin region and promotes efficient chromosome segregation. Cell 137, 697–707 (2009).">28</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR29" id="ref-link-section-d18519924e2765_2" title="Tran, N. T., Laub, M. T. &amp; Le, T. B. K. SMC Progressively aligns chromosomal arms in Caulobacter crescentus but is antagonized by convergent transcription. Cell Rep. 20, 2057–2071 (2017).">29</a>,<a aria-label="Reference 30" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR30" id="ref-link-section-d18519924e2768" title="Gruber, S. et al. Interlinked sister chromosomes arise in the absence of condensin during fast replication in B. subtilis. Curr. Biol. 24, 293–298 (2014).">30</a></sup>. It is tempting to speculate that archaeal Smc-ScpAB, bacterial Smc-ScpAB, and MukBEF all contribute to sister chromosome resolution through loop extrusion, despite their different outcomes in overall chromosome folding. Such an active mechanism for sister chromosome resolution might be crucial to euryarchaeal cells when, for instance, their chromosome copy numbers are reduced in response to nutrient availability<sup><a aria-label="Reference 50" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR50" id="ref-link-section-d18519924e2772" title="Spaans, S. K., van der Oost, J. &amp; Kengen, S. W. The chromosome copy number of the hyperthermophilic archaeon Thermococcus kodakarensis KOD1. Extremophiles 19, 741–750 (2015).">50</a>,<a aria-label="Reference 97" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR97" id="ref-link-section-d18519924e2775" title="Zerulla, K. et al. DNA as a phosphate storage polymer and the alternative advantages of polyploidy for growth or survival. PLoS ONE 9, e94819 (2014).">97</a></sup>.</p>
</li>
</ol><p>In any case, it will be important to further explore the physiological function of the <i>dif</i> boundary in euryarchaea, especially in the context of their polyploid nature.</p><p>According to current models, the loop extrusion cycle of SMC complexes requires a DNA bend to reel a DNA segment through the lumen of the SMC ring<sup><a aria-label="Reference 7" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR7" id="ref-link-section-d18519924e2791" title="Kim, E., Barth, R. &amp; Dekker, C. Looping the genome with SMC complexes. Annu. Rev. Biochem. 92, 15–41 (2023).">7</a>,<a aria-label="Reference 75" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR75" id="ref-link-section-d18519924e2794" title="Dekker, C., Haering, C. H., Peters, J. M. &amp; Rowland, B. D. How do molecular motors fold the genome? Science 382, 646–648 (2023).">75</a>,<a aria-label="Reference 76" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR76" id="ref-link-section-d18519924e2797" title="Marko, J. F., De Los Rios, P., Barducci, A. &amp; Gruber, S. DNA-segment-capture model for loop extrusion by structural maintenance of chromosome (SMC) protein complexes. Nucleic Acids Res. 47, 6956–6972 (2019).">76</a></sup>. We propose that TrmBL2, a NAP that forms a filamentous structure along DNA, serves as an obstacle that blocks loop extrusion by slowing or prohibiting DNA bending (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig8">8f</a>). A previous biophysical study demonstrated that TrmBL2 increases the persistence length of bound DNA in a concentration-dependent manner<sup><a aria-label="Reference 62" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR62" id="ref-link-section-d18519924e2804" title="Efremov, A. K. et al. Transcriptional repressor TrmBL2 from Thermococcus kodakarensis forms filamentous nucleoprotein structures and competes with histones for DNA binding in a salt- and DNA supercoiling-dependent manner. J. Biol. Chem. 290, 15770–15784 (2015).">62</a></sup>. The persistence length can be increased even above 90 nm, approximately twice as large as the size of the SMC ring. In line with our model, a recent study observed that the loop extrusion of eukaryotic condensin is stalled by a linear array of Rap1, a telomeric protein that can induce local DNA stiffening<sup><a aria-label="Reference 98" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR98" id="ref-link-section-d18519924e2808" title="Le Bihan, Y. V. et al. Effect of Rap1 binding on DNA distortion and potassium permanganate hypersensitivity. Acta Crystallogr. D Biol. Crystallogr. 69, 409–419 (2013).">98</a>,<a aria-label="Reference 99" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR99" id="ref-link-section-d18519924e2811" title="Analikwu B. T. et al. Telomere protein arrays stall DNA loop extrusion by condensin. bioRxiv, 2023.2010.2029.564563 (2023).">99</a></sup>.</p><p>To our surprise, we found that TrmBL2-dependent positioning of Smc-ScpAB is associated with the AT richness of the underlying sequence (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig8">8b</a>). Given the high growth temperature of <i>T. kodakarensis</i>, one could imagine that AT-rich sequences interfere with loop extrusion by being denatured into single-stranded DNA (ssDNA), even though our correlation analysis failed to address this possibility. ssDNA has been suggested to serve as a high-affinity substrate for Smc-ScpAB binding and thereby may trap the complex<sup><a aria-label="Reference 100" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR100" id="ref-link-section-d18519924e2824" title="Yano, K., Noguchi, H. &amp; Niki, H. Profiling a single-stranded DNA region within an rDNA segment that affects the loading of bacterial condensin. iScience 25, 105504 (2022).">100</a></sup>. However, due to the fragility of ssDNA that threatens genome integrity, hyperthermophiles are more likely to suppress heat-induced denaturation of genomic DNA. For this suppression, all hyperthermophiles appear to leverage reverse gyrase, a type IA topoisomerase that generates positive DNA supercoils and is critical for the thermal adaptation of hyperthermophiles<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR101" id="ref-link-section-d18519924e2828" title="Forterre, P. A hot story from comparative genomics: reverse gyrase is the only hyperthermophile-specific protein. Trends Genet. 18, 236–237 (2002).">101</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR102" id="ref-link-section-d18519924e2828_1" title="Atomi, H., Matsumi, R. &amp; Imanaka, T. Reverse gyrase is not a prerequisite for hyperthermophilic life. J. Bacteriol. 186, 4829–4833 (2004).">102</a>,<a aria-label="Reference 103" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR103" id="ref-link-section-d18519924e2831" title="Lipscomb, G. L., Hahn, E. M., Crowley, A. T. &amp; Adams, M. W. W. Reverse gyrase is essential for microbial growth at 95 degrees C. Extremophiles 21, 603–608 (2017).">103</a></sup>. Hyperthermophilic archaea also protect their genomes from thermal denaturation by coating genomic DNA with NAPs more extensively than mesophilic archaea do<sup><a aria-label="Reference 104" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR104" id="ref-link-section-d18519924e2835" title="Hocher, A. et al. Growth temperature and chromatinization in archaea. Nat. Microbiol. 7, 1932–1942 (2022).">104</a></sup>. We prefer the idea that physical properties of AT-rich sequences other than their proneness to denaturation facilitate TrmBL2-mediated stalling of Smc-ScpAB. This notion is supported by the positive correlation of Smc positioning efficiency with A-tracts, a specific type of AT-rich sequences that displays unique structural features (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig8">8c, d</a>). A-tracts are characterized by their intrinsically straight conformation and enhanced negative electrostatic potential in the minor groove<sup><a aria-label="Reference 67" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR67" id="ref-link-section-d18519924e2843" title="Marin-Gonzalez, A., Vilhena, J. G., Perez, R. &amp; Moreno-Herrero, F. A molecular view of DNA flexibility. Q Rev. Biophys. 54, e8 (2021).">67</a>,<a aria-label="Reference 70" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR70" id="ref-link-section-d18519924e2846" title="Rohs, R. et al. The role of DNA shape in protein-DNA recognition. Nature 461, 1248–1253 (2009).">70</a>,<a aria-label="Reference 73" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR73" id="ref-link-section-d18519924e2849" title="Velasco-Berrelleza, V. et al. SerraNA: a program to determine nucleic acids elasticity from simulation data. Phys. Chem. Chem. Phys. 22, 19254–19266 (2020).">73</a></sup>. It is plausible that these two features of A-tracts help TrmBL2 to impede Smc-ScpAB translocation, which is consistent with their correlations with Smc positioning efficiency (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig8">8e</a>). We also note that the DNA motifs found at TrmBL2- and Smc-binding sites contain A-tract-like sequences (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig7">7h</a>). The structural properties of A-tracts may affect their interaction with TrmBL2 or Smc-ScpAB, thereby influencing the stalling competency. In this case, the effect of A-tracts on DNA binding of TrmBL2 or Smc-ScpAB will be qualitative rather than quantitative, given that AT richness is not markedly correlated with the binding level of TrmBL2 or Smc (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">8c, d</a>). The electrostatic potential of DNA minor groove can indeed affect protein-DNA interaction via so-called shape readout mechanisms<sup><a aria-label="Reference 70" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR70" id="ref-link-section-d18519924e2862" title="Rohs, R. et al. The role of DNA shape in protein-DNA recognition. Nature 461, 1248–1253 (2009).">70</a>,<a aria-label="Reference 105" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR105" id="ref-link-section-d18519924e2865" title="Chiu, T. P., Rao, S., Mann, R. S., Honig, B. &amp; Rohs, R. Genome-wide prediction of minor-groove electrostatic potential enables biophysical modeling of protein-DNA binding. Nucleic Acids Res. 45, 12565–12576 (2017).">105</a></sup>.</p><p>In summary, our study has not only uncovered a eukaryotic-like mechanism of chromosomal domain formation in archaea, but also shed light on the potential role of intrinsic DNA structure in defining higher-order genome organization. It will be interesting to see whether a similar interplay of SMC complexes, boundary proteins, and structural features of boundary sequences shapes eukaryotic 3D genomes.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec10-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec10">Methods</h2><div class="c-article-section__content" id="Sec10-content"><h3 class="c-article__sub-heading" id="Sec11">
<i>T. kodakarensis</i> strains</h3><p>The uracil-auxotrophic strain KU216 (Δ<i>pyrF</i>)<sup><a aria-label="Reference 47" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR47" id="ref-link-section-d18519924e2891" title="Sato, T., Fukui, T., Atomi, H. &amp; Imanaka, T. Improved and versatile transformation system allowing multiple genetic manipulations of the hyperthermophilic archaeon Thermococcus kodakaraensis. Appl. Environ. Microbiol. 71, 3889–3899 (2005).">47</a></sup> was used as a parental strain for strain construction. For all experiments, cells were grown anaerobically at 85 °C. Unless otherwise stated, cells were cultivated as follows. Cells were inoculated into the ASW-YT-m1-S<sup>0</sup> rich medium<sup><a aria-label="Reference 106" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR106" id="ref-link-section-d18519924e2897" title="Michimori, Y. et al. Removal of phosphoglycolate in hyperthermophilic archaea. Proc. Natl. Acad. Sci. USA 121, e2311390121 (2024).">106</a></sup> and pre-cultured overnight. The pre-culture was inoculated into fresh ASW-YT-m1-S<sup>0</sup> rich medium to an OD<sub>660</sub> of 0.015. The culture was grown until mid-to-late log phase (OD<sub>660</sub>: 0.2-0.25) and used for experiments. ASW-YT-m1-S<sup>0</sup> was used with the redox indicator resazurin added to a concentration of 0.5 mg/L.</p><h3 class="c-article__sub-heading" id="Sec12">
<i>T. acidophilum</i> strain</h3><p>The <i>T. acidophilum</i> wild-type strain DSM 1728 was grown as described previously<sup><a aria-label="Reference 59" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR59" id="ref-link-section-d18519924e2925" title="Jo, M., Murayama, Y., Tsutsui, Y. &amp; Iwasaki, H. In vitro site-specific recombination mediated by the tyrosine recombinase XerA of Thermoplasma acidophilum. Genes Cells 22, 646–661 (2017).">59</a></sup> to an OD<sub>600</sub> of ~0.2.</p><h3 class="c-article__sub-heading" id="Sec13">
<i>E. coli</i> strain</h3><p>The BL21-CodonPlus(DE3)-RIL strain (Agilent, 230245) was used for protein expression. Unless otherwise stated, cells harboring an expression plasmid were grown in LB medium containing 100 μg/mL of ampicillin and 30 μg/mL of chloramphenicol.</p><h3 class="c-article__sub-heading" id="Sec14">Construction of <i>T. kodakarensis</i> mutants</h3><p>Except <i>xerA</i> (<i>TK0777</i>), all genes were deleted using a pop-in/pop-out approach targeting the entire (or almost entire) coding region<sup><a aria-label="Reference 47" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR47" id="ref-link-section-d18519924e2961" title="Sato, T., Fukui, T., Atomi, H. &amp; Imanaka, T. Improved and versatile transformation system allowing multiple genetic manipulations of the hyperthermophilic archaeon Thermococcus kodakaraensis. Appl. Environ. Microbiol. 71, 3889–3899 (2005).">47</a></sup>. For <i>xerA</i> deletion, its coding sequence was replaced with a marker gene via double crossover<sup><a aria-label="Reference 107" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR107" id="ref-link-section-d18519924e2968" title="Sato, T., Fukui, T., Atomi, H. &amp; Imanaka, T. Targeted gene disruption by homologous recombination in the hyperthermophilic archaeon Thermococcus kodakaraensis KOD1. J. Bacteriol. 185, 210–220 (2003).">107</a></sup>. For <i>dif</i> deletion, the entire 28-bp sequence of <i>dif</i> (TTTTGATATAATGTACCTTATATGACAA) was deleted using the pop-in/pop-out method. For ectopic insertion of <i>boundary1</i><sup>Δ<i>dif</i></sup> into the genome of the Δ<i>boundary1</i> strain, <i>boundary1</i><sup>Δ<i>dif</i></sup> was integrated into the intergenic region between <i>TK0712</i> and <i>TK0713</i> via the pop-in/pop-out method. For ectopic insertion of <i>boundary1</i><sup>Δ<i>dif</i></sup> into the genome of KU216, <i>boundary1</i><sup>Δ<i>dif</i></sup> was integrated into the <i>TK0712</i>-<i>TK0713</i> intergenic region together with a marker via double crossover.</p><h3 class="c-article__sub-heading" id="Sec15">Deletion/insertion constructs</h3><p>The primers used for the construction are listed in Supplementary Data <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM4">2</a>. For deletions of <i>smc</i> (<i>TK1017</i>), <i>scpA</i> (<i>TK1018</i>), and <i>scpB</i> (<i>TK1962</i>), each target sequence was PCR amplified together with upstream and downstream ~1-kb regions and cloned into the pUD3 plasmid, which contains the <i>pyrF</i> marker<sup><a aria-label="Reference 108" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR108" id="ref-link-section-d18519924e3058" title="Samson, R. Y. et al. Specificity and function of archaeal DNA replication initiator proteins. Cell Rep. 3, 485–496 (2013).">108</a></sup>. pUD3 was digested at the PstI and SalI sites for the cloning. The target sequence was then removed by inverse PCR to generate the deletion plasmids pUD3-Δ<i>smc</i>, pUD3-Δ<i>scpA</i>, and pUD3-Δ<i>scpB</i>. For double deletion of <i>smc</i> and <i>scpA</i>, constituting a part of an operon, pUD3-∆<i>smc</i> was amplified by inverse PCR to remove the downstream homology arm. The obtained PCR fragment was fused with a downstream ~1-kb region of <i>scpA</i> using In-Fusion HD Cloning Plus reagents (Takara Clontech, 638910) to generate pUD3-Δ<i>smc</i>Δ<i>scpA</i>. For <i>dif</i> deletion, upstream and downstream ~1-kb regions of the <i>dif</i> sequence were PCR amplified separately and cloned together into pUD3 (digested at BamHI and HindIII sites) using In-Fusion HD Cloning Plus reagents. The obtained plasmid was named pUD3-Δ<i>dif</i>. For <i>TK0795</i> deletion, upstream and downstream ~1-kb regions of the target sequence were PCR amplified separately and cloned together into the EcoRI site of pUD3 using In-Fusion HD Cloning Plus reagents. The obtained plasmid was named pUD3-Δ<i>TK0795</i>. For <i>trmBL2</i> (<i>TK0471</i>) deletion, upstream and downstream ~1-kb regions of the target sequence were PCR amplified as a single fragment using genomic DNA of the KCP1 strain<sup><a aria-label="Reference 55" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR55" id="ref-link-section-d18519924e3113" title="Cortez, D. et al. Evidence for a Xer/dif system for chromosome resolution in archaea. PLoS Genet. 6, e1001166 (2010).">55</a></sup> (Δ<i>pyrF</i> Δ<i>trpE</i> Δ<i>trmBL2</i>) as a template. The KCP1 strain was kindly provided by Hugo Maruyama. The fragment was cloned into the EcoRI site of pUD3 to generate the deletion plasmid pUD3-Δ<i>trmBL2</i>. For <i>xerA</i> deletion, a PCR fragment containing the <i>pyrF</i> marker and upstream and downstream ~1-kb regions of the target sequence were PCR amplified separately. The three fragments were cloned together into the EcoRI site of pUC118 using In-Fusion HD Cloning Plus reagents. The obtained deletion plasmid was named pUC118-Δ<i>xerA</i>. For <i>boundary1</i> deletion, upstream and downstream ~1-kb regions of the entire intergenic region between the ORFs of <i>TK0561</i> and <i>TK0562</i> were PCR amplified separately and cloned together into the EcoRI site of pUD3 using In-Fusion HD Cloning Plus reagents. The obtained plasmid was named pUD3-Δ<i>boundary1</i>.</p><p>For ectopic insertion of <i>boundary1</i><sup>Δ<i>dif</i></sup> into the genome of the Δ<i>boundary1</i> strain, upstream and downstream ~1-kb regions of the insertion site (located in the intergenic region between <i>TK0712</i> and <i>TK0713</i>) were PCR amplified and cloned into the EcoRI site of pUD3 using In-Fusion HD Cloning Plus reagents. The obtained plasmid, named pUD3-<i>TK0712</i>, was linearized at the <i>TK0712</i>-<i>TK0713</i> intergenic region by inverse PCR. We also PCR amplified a <i>boundary1</i><sup>Δ<i>dif</i></sup> fragment using the genome of the Δ<i>dif</i> strain as a template. The two PCR fragments were assembled using In-Fusion HD Cloning Plus reagents to generate the pUD3-<i>TK0712-boundary1</i><sup>Δ<i>dif</i></sup> plasmid. For ectopic insertion of <i>boundary1</i><sup>Δ<i>dif</i></sup> into the genome of KU216, the insertion site was PCR amplified and cloned into the EcoRI site of pUC118 in the same way as for the construction of pUD3-<i>TK0712</i>. The obtained plasmid, named pUC118-<i>TK0712</i>, was linearized by inverse PCR in the same way as for the construction of pUD3-<i>TK0712-boundary1</i><sup>Δ<i>dif</i></sup>. This fragment was assembled with the <i>boundary1</i><sup>Δ<i>dif</i></sup> fragment (the same sequence as used for the construction of pUD3-<i>TK0712-boundary1</i><sup>Δ<i>dif</i></sup>) and the <i>pyrF</i> marker using In-Fusion HD Cloning Plus reagents to generate the pUC118-<i>TK0712-pyrF-boundary1</i><sup>Δ<i>dif</i></sup> plasmid.</p><h3 class="c-article__sub-heading" id="Sec16">Transformation</h3><p>For gene disruption or sequence insertion via pop-in/pop-out recombination, transformation with deletion or insertion constructs described above was performed as described previously<sup><a aria-label="Reference 106" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR106" id="ref-link-section-d18519924e3252" title="Michimori, Y. et al. Removal of phosphoglycolate in hyperthermophilic archaea. Proc. Natl. Acad. Sci. USA 121, e2311390121 (2024).">106</a></sup>. The synthetic medium ASW-AA-m1-S<sup>0</sup> was made according to Su et al. <sup><a aria-label="Reference 109" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR109" id="ref-link-section-d18519924e3258" title="Su, Y., Michimori, Y. &amp; Atomi, H. Biochemical and genetic examination of two aminotransferases from the hyperthermophilic archaeon Thermococcus kodakarensis. Front. Microbiol. 14, 1126218 (2023).">109</a></sup>. For selection of ∆<i>trmBL2</i> cells, cells were spread onto solid ASW-AA-m1-S<sup>0</sup> medium containing 7.5 g/L 5-fluoroorotic acid (FUJIFILM Wako, 064-03664), 44 mM NaOH, and 5 mg/L uracil. The ∆<i>smc</i> ∆<i>scpA</i> strain was constructed by transforming KU216 with pUD3-∆<i>smc</i>∆<i>scpA</i>. The ∆<i>smc</i> ∆<i>scpB</i> strain was constructed by disrupting <i>scpB</i> in the ∆<i>smc</i> mutant.</p><p>Double-crossover deletion of <i>xerA</i> was conducted as follows. KU216 cells from 20 mL of an overnight culture were pelleted and resuspended in 200 µL of 0.8 × ASW-m1, which is composed of 0.8 × artificial seawater (ASW)<sup><a aria-label="Reference 110" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR110" id="ref-link-section-d18519924e3299" title="Jager, D., Forstner, K. U., Sharma, C. M., Santangelo, T. J. &amp; Reeve, J. N. Primary transcriptome map of the hyperthermophilic archaeon Thermococcus kodakarensis. BMC Genom. 15, 684 (2014).">110</a></sup> supplemented with 20 μM KI, 20 μM H<sub>3</sub>BO<sub>3</sub>, and 10 μM NiCl<sub>2</sub>. The cell suspension was mixed with 3 μg of pUC118-Δ<i>xerA</i> and incubated on ice for 5 min. The cells were heat shocked for 45 s at 85 °C and cooled on ice for 5 min. This heating-chilling cycle was repeated five times. The cell suspension was inoculated into 20 mL of the uracil-free ASW-AA-m1-S<sup>0</sup> medium and grown for 3 days. 400 µL of the culture were inoculated into fresh ASW-AA-m1-S<sup>0</sup> medium and further grown for 3 days. For single colony isolation, 50 µL of the culture were spread onto ASW-YT-m1 medium solidified with 10 g/L Gelrite (FUJIFILM Wako, 075-05655) and grown for 1 day. Deletion of <i>xerA</i> was confirmed by colony PCR and DNA sequencing.</p><p>Ectopic introduction of <i>boundary1</i><sup><i>Δdif</i></sup> into the genome of KU216 was conducted as follows. A fragment containing the upstream and downstream homology arms, <i>boundary1</i><sup><i>Δdif</i></sup>, and <i>pyrF</i> was PCR amplified using pUC118-<i>TK0712-pyrF-boundary1</i><sup>Δ<i>dif</i></sup> as a template. The fragment was integrated into the genome of KU216 via double crossover. Transformation was carried out as described for ∆<i>xerA</i>. Insertion was confirmed by colony PCR and DNA sequencing.</p><h3 class="c-article__sub-heading" id="Sec17">Growth measurement of <i>T. kodakarensis</i>
</h3><p>Cells from an overnight pre-culture were inoculated into three vials containing fresh ASW-YT-m1-S<sup>0</sup>. These three cultures were grown at 85 °C and used separately for OD<sub>660</sub> measurement.</p><h3 class="c-article__sub-heading" id="Sec18">Microscopy</h3><p>Microscopic observation of <i>T. kodakarensis</i> cells was performed according to a previous study<sup><a aria-label="Reference 111" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR111" id="ref-link-section-d18519924e3376" title="Villain, P. et al. The hyperthermophilic archaeon Thermococcus kodakarensis is resistant to pervasive negative supercoiling activity of DNA gyrase. Nucleic Acids Res. 49, 12332–12347 (2021).">111</a></sup>. For each strain, 500 μL of hot cell culture were dispensed into each of two 1.5-mL tubes containing 500 μL of pre-chilled 0.8 × ASW-m1. The tubes were cooled in ice-cold water for 5 min. After centrifugation (10,000 × <i>g</i>, 4 °C, 5 min), the two pellets were resuspended together in 200 μL of pre-chilled 0.8 × ASW-m1. The cells were centrifuged (10,000 × <i>g</i>, 4 °C, 5 min), resuspended in 100 μL of pre-chilled 0.8 × ASW-m1 containing 1 μg/mL Hoechst33342 (FUJIFILM Wako, 346-07951), and incubated for 10 min on ice in the dark. 3 µL of the cell suspension were mounted on a glass slide covered with a thin layer of 1% agarose containing 0.8 × ASW-m1. Bright field and fluorescence images were obtained at room temperature using Axio Imager.M2 equipped with a Plan-Apochromat 100×/1.4 NA oil immersion objective, an Axiocam 305 mono camera, and ZEN Pro software (Carl Zeiss). Hoechst33342-stained DNA was visualized using a 385-nm laser from the Colibri 7 LED illumination system (excitation filter: 385/30 nm, emission filter: 450/40 nm) (Carl Zeiss). Cell detection and quantitative analysis were performed using MicrobeJ<sup><a aria-label="Reference 112" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR112" id="ref-link-section-d18519924e3386" title="Ducret, A., Quardokus, E. M. &amp; Brun, Y. V. MicrobeJ, a tool for high throughput bacterial cell detection and quantitative analysis. Nat. Microbiol. 1, 16077 (2016).">112</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec19">3C-seq</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec20">Cell fixation</h4><p><i>T. kodakarensis</i> cells were crosslinked by dispensing 8 mL of hot cell culture into each of eight 50-mL tubes containing 32 mL of fixative solution 1 (4.3 mL of 37% formaldehyde [Nacalai Tesque, 16223-55], 6.4 mL of 4 × ASW-m1, and 21.3 mL of MilliQ water). The mixtures were incubated for 30 min at 25 °C with agitation at 75 rpm. Crosslinking was quenched by adding 10 mL of 2.5 M glycine to each tube and incubating the mixtures for 10 min at room temperature. The cells were collected by centrifugation (8000 × <i>g</i>, 15 min, 25 °C). Supernatant was removed with ~2 mL per tube left for resuspension. The cell suspensions were combined and dispensed into four 5-mL tubes. The cells were centrifuged (10,000 × <i>g</i>, 5 min, 25 °C), and two each of the pellets were resuspended in 3 mL of fixative solution 2 (30 µL of 300 mM DSG [Thermo Fisher Scientific, 20593] dissolved in DMSO [FUJIFILM Wako, 046-21981], 600 µL of 4 × ASW-m1, and 2370 µL of MilliQ water) for additional crosslinking. The two cell suspensions were incubated for 40 min at 25 °C with agitation at 75 rpm. Crosslinking was quenched by adding 750 µL of 2.5 M glycine and incubating for 5 min at room temperature. The cells were collected by centrifugation (10,000 × <i>g</i>, 5 min, 4 °C). The two pellets were resuspended together in 1 mL of 1 × PBS. The cells were centrifuged (10,000 × <i>g</i>, 5 min, 4 °C), resuspended in 1 mL of 1 × PBS, and centrifuged again (10,000 × <i>g</i>, 5 min, 4 °C). The pellet was stored at –80 °C until use.</p><p>For crosslinking of <i>T. acidophilum</i>, 50 mL of cell culture were pelleted (8000 × g, 5 min, room temperature) and resuspended in a mixture of 3.35 mL 1 × PBS and 0.65 mL 37% formaldehyde (final 6%). The reaction was incubated for 30 min at 25 °C with agitation at 100 rpm. Crosslinking was quenched by adding 680 µL of 2.5 M glycine and incubating the mixture for 10 min at room temperature. The cells were pelleted by centrifugation (10,000 × <i>g</i>, 3 min, 4 °C). The centrifugation was repeated with the opposite side of the tube outward. The cells were washed by resuspending the pellet in 1 mL of 1 × PBS and centrifuging the suspension (10,000 × <i>g</i>, 2 min, 4 °C). This step was repeated once. The pellet was stored at –80 °C until use.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec21">Restriction digestion</h4><p>A frozen pellet of <i>T. kodakarensis</i> was resuspended in wash buffer (10 mM Tris-HCl [pH 8.0], 50 mM NaCl, 10 mM MgCl<sub>2</sub>) to an OD<sub>660</sub> of 3. 1.2 mL of the cell suspension were used for the experiment. A frozen pellet of <i>T. acidophilum</i> was resuspended in 1 mL of wash buffer, and the whole cell suspension was used for the experiment. The cells were centrifuged (10,000 × <i>g</i>, 10 min, 4 °C) and resuspended in 50 μL of 1 × NEBuffer 2 (New England Biolabs, B7002S). 50 μL of the suspension were transferred to an 1.5-mL tube, mixed with 5.55 μL of 10% SDS, and incubated for 15 min at 65 °C with agitation at 600 rpm. The suspension was immediately cooled on ice for 90 s and placed at room temperature until use. 12.5 µL of the lysate were stored for purification of undigested control DNA. For DNA digestion, 37.5 µL of the lysate were mixed with 232.5 µL of premixed buffer composed of 26.3 µL of 10 × NEBuffer 2, 60 µL of 10% v/v Triton X-100, and 146.2 μL of MilliQ water. This was further mixed with 15 μL each of 10 U/μL HaeIII (New England Biolabs, R0108L) and 10 U/μL AluI (New England Biolabs, R0137L) and incubated for 3 h at 37 °C with agitation at 600 rpm. The tube was also inverted every 30 min for mixing. After the incubation was completed, the reaction was mixed with 33.3 μL of 10% SDS and incubated for 10 min at room temperature to inactivate the restriction enzymes.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec22">Proximity ligation</h4><p>A previous study demonstrated that, for prokaryotic Hi-C, using only the insoluble material for proximity ligation significantly improves the data quality<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e3460" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup>. According to this strategy, the insoluble fraction was collected by centrifugation (16,000 × <i>g</i>, 10 min, 25 °C), resuspended in 500 μL of wash buffer, and centrifuged again (16,000 × <i>g</i>, 10 min, 25 °C). The pellet was resuspended in 1185 μL of 1.01 × T4 DNA Ligase Reaction Buffer (New England Biolabs, B0202S). 395 µL of the sample were mixed with 5 µL of MilliQ water and stored for purification of un-ligated control DNA. For ligation, 790 µL of the sample were mixed with 10 μL of 400 U/μL T4 DNA Ligase (New England Biolabs, M0202L) and incubated for 3 h at 16 °C with agitation at 600 rpm. The reaction was also mixed every 30 min by inverting the tube.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec23">Reverse crosslinking</h4><p>The insoluble fraction was collected by centrifugation (16,000 × <i>g</i>, 10 min, 25 °C) and resuspended in 115 μL (for <i>T. kodakarensis</i>) or 230 µL (for <i>T. acidophilum</i>) of decrosslinking buffer (100 µL of TE buffer [pH 8], 10 µL of 10% SDS, and 5 µL of 0.5 M EDTA [pH 8]). The suspension was mixed with 3 μL (for <i>T. kodakarensis</i>) or 6 µL (for <i>T. acidophilum</i>) of 800 U/mL proteinase K (New England Biolabs, P8107). For <i>T. kodakarensis</i>, decrosslinking was performed by incubating the sample overnight at 37 °C with agitation at 600 rpm. After that, 3 μL of 800 U/mL proteinase K were additionally added, and the sample was incubated for 4 h at 37 °C with agitation at 600 rpm. For <i>T. acidophilum</i>, decrosslinking was performed by incubating the sample for 6 h at 65 °C with agitation at 600 rpm, which was followed by overnight incubation at 30 °C with agitation at 600 rpm. After that, 6 µL of 800 U/mL proteinase K were additionally added, and the sample was incubated for 4 h at 30 °C with agitation at 600 rpm.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec24">Sonication and purification of DNA</h4><p>90 μL of the sample were used for DNA sonication, while the remainder of the sample was stored for purification of unsonicated control DNA. The 90-µL aliquot was transferred to a microTUBE AFA Fiber Pre-Slit Snap-Cap 6 × 16 mm (Covaris, 520045) and sonicated for 200 s at 7 ° C using an M220 Focused-ultrasonicator (Covaris) with the peak power set to 50, the duty factor set to 20, and the cycles/burst set to 200. DNA was then extracted twice with phenol:chloroform:isoamyl alcohol (Sigma Aldrich, 77618-100ML) and ethanol-precipitated together with 2 μL of 20 mg/mL glycogen (Thermo Fisher Scientific, R0561). The DNA was dissolved in 30 μL of 10 mM Tris-HCl (pH 8.0) containing 0.05 mg/mL RNase A (Thermo Scientific, EN0531).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec25">Library construction</h4><p>The sheared DNA was used for library construction with NEBNext Ultra II DNA Library Prep with Sample Purification Beads (New England Biolabs, E7103S). Reaction was performed according to the manufacturer’s instructions with size selection for a 300–400 bp insert.</p><h3 class="c-article__sub-heading" id="Sec26">RNA-seq</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec27">RNA extraction</h4><p>10 mL of hot cell culture were dispensed into a 50-mL tube containing 20 mL of pre-chilled 0.8 × ASW-m1 and further chilled in iced water for 5 min. The cells were centrifuged (10,000 × <i>g</i>, 4 °C, 10 min), resuspended in 1 mL of pre-chilled 0.8 × ASW-m1, and centrifuged again (10,000 × <i>g</i>, 4 °C, 5 min). The pellet was resuspended in 750 µL of lysis buffer (75 µL of 10% SDS, 25 μL of 3 M sodium acetate [pH 5.2], and 650 µL of RNase-free water). The RNA was extracted twice with acid phenol (NIPPON GENE, 315-90291) and isopropanol-precipitated. The pellet was dissolved in 51 µL of RNase-free water. The solution was mixed with 6 µL of 10 × DNase I Reaction Buffer (New England Biolabs, M0303S) and 3 µL of 2000 U/mL DNase I (New England Biolabs, M0303S). After 30 min of incubation at 37 °C, the RNA was extracted with acid phenol and ethanol-precipitated. The pellet was dissolved in 25 µL of RNase-free water.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec28">Ribosomal RNA depletion</h4><p>Ribosomal RNAs (rRNAs) were removed according to a previous study using NEBNext rRNA Depletion Kit v2 (Human/Mouse/Rat) (New England Biolabs, E7400L) and a 85 μM DNA oligo mixture containing 85 DNA oligos complementary to either 23S rRNA, 16S rRNA, or 5S rRNA<sup><a aria-label="Reference 113" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR113" id="ref-link-section-d18519924e3544" title="Sas-Chen, A. et al. Dynamic RNA acetylation revealed by quantitative cross-evolutionary mapping. Nature 583, 638–643 (2020).">113</a></sup>. 1 µg of the RNA was diluted in RNase-free water for a total volume of 11 µL. The diluted RNA was mixed with 2 µL of NEBNext Probe Hybridization Buffer (New England Biolabs, E7400L) and 2 µL of the DNA oligo mixture. Annealing was performed by heating the sample for 2 min at 95 °C and cooled down to 22 °C (0.1 °C/s) in a thermocycler. After additional 5-min incubation at 22 °C, the sample was mixed with 2 µL of RNase H Reaction Buffer (New England Biolabs, E7400L), 2 µL of NEBNext Thermostable RNase H (New England Biolabs, E7400L), and 1 µL of nuclease-free water. rRNA digestion was performed by incubating the sample for 30 min at 50 °C in a thermocycler with the lid temperature set to 55 °C. The reaction was quickly chilled on ice and mixed with 5 µL of DNase I Reaction Buffer (New England Biolabs, E7400L), 2.5 µL of NEBNext RNase-free DNase I (New England Biolabs, E7400L), and 22.5 µL of nuclease-free water. This was followed by incubation for 30 min at 37 °C in a thermocycler with the lid temperature set to 40 °C. The RNA was purified using Monarch RNA Cleanup Kit (New England Biolabs, 32855).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec29">Library construction</h4><p>The cleaned-up RNA was used for library construction with NEBNext Ultra II Directional RNA Library Prep Kit for Illumina (New England Biolabs, E7765L). Reaction was performed according to the manufacturer’s instructions.</p><h3 class="c-article__sub-heading" id="Sec30">ChIP-seq</h3><p>ChIP-seq was carried out as described previously with modifications<sup><a aria-label="Reference 108" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR108" id="ref-link-section-d18519924e3566" title="Samson, R. Y. et al. Specificity and function of archaeal DNA replication initiator proteins. Cell Rep. 3, 485–496 (2013).">108</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec31">Cell fixation and lysate preparation</h4><p>For crosslinking, 80 mL of hot cell culture were mixed with 2.5 mL of 37% formaldehyde (final 1%). The mixture was immediately cooled in iced water with occasional agitation. After 5 min, the mixture was incubated for 20 min at room temperature with agitation at 75 rpm. Crosslinking was quenched by adding 4.5 mL of 2.5 M glycine (final 0.13 M). After 20 min of incubation at room temperature, the mixture was transferred to 50-mL tubes while removing as much powder of elemental sulfur as possible. The cells were collected by centrifugation (16,000 × <i>g</i>, 10 min, 4 °C) and resuspended in 1 mL of 0.8 × ASW–m1. The cell suspension was transferred to an 1.5-mL tube and centrifuged (10,000 × <i>g</i>, 5 min, 4 °C). The pellet was resuspended in 1 mL of TBS-TT (20 mM Tris-HCl [pH 8], 150 mM NaCl, 0.1% v/v Triton X-100, and 0.1% v/v Tween 20) and transferred to a milliTUBE 1 mL AFA Fiber (Covaris, 520135) for DNA fragmentation using an M220 Focused-ultrasonicator (Covaris). The fragmentation was performed for 12 min at 7 °C with the peak power set to 75, the duty factor set to 26, and the cycles/burst set to 200. The lysate was transferred to an 1.5-mL tube and centrifuged (20,400 × <i>g</i>, 30 min, 4 °C). The supernatant was transferred to a new 1.5-mL tube, flash frozen in liquid nitrogen, and stored at –80 °C until use.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec32">Immunoprecipitation</h4><p>500 μL of the lysate were mixed with 10 µL of antiserum (IP) or TBS-TT and incubated for 2.5 h at 4 °C. 50 μL of Dynabeads Protein G (Thermo Fisher Scientific, 10004D) were washed with TBS-TT and added to the IP sample. The lysate-bead mixture was incubated for 1 h at 4 °C. The beads were washed five times with TBS-TT, once with TBS-TT containing 0.5 M NaCl, and once with TBS-TT containing 0.5% v/v Triton X-100 and 0.5% v/v Tween 20. Protein-DNA complexes were eluted in 200 μL of elution buffer (20 mM Tris-HCl [pH 7.5], 10 mM EDTA [pH 8], and 0.5% SDS) for 30 min at 65 °C with agitation at 1400 rpm. 186 µL of the input sample were mixed with 4 µL of 0.5 M EDTA (pH 8) and 10 µL of 10% SDS and heated in the same way. Each of the IP and input samples was incubated with 3 μL of 800 U/mL proteinase K overnight at 37 °C. The DNA was extracted with phenol:chloroform:isoamyl alcohol and ethanol-precipitated together with 2 μL of 20 mg/mL glycogen. The DNA was dissolved in 55 μL of 10 mM Tris-HCl (pH 8.0).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec33">Library construction</h4><p>The DNA was used for library construction with NEBNext Ultra II DNA Library Prep with Sample Purification Beads. Reaction was performed according to the manufacturer’s instructions.</p><h3 class="c-article__sub-heading" id="Sec34">Protein purification and antibody production</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec35">Purification of XerA</h4><p>The coding region of <i>T. kodakarensis</i> XerA was cloned into the pET21a(+) plasmid (Merck Millipore, 69740-3CN) and expressed without any additional sequence. <i>E. coli</i> cells harboring the expression plasmid were grown to an OD<sub>600</sub> of ~0.8 in LB at 37 °C. Protein expression was induced with 0.1 mM IPTG for 15 h at 28 °C. The cells were pelleted by centrifugation, resuspended in 1 × PBS, and pelleted again. The cells were resuspended in 50 mM Tris-HCl (pH 7.5) containing 1 mM EDTA and disrupted by sonication. The lysate was cleared by centrifugation and heated for 20 min at 70 °C. The heat-denatured proteins were removed by centrifugation. The supernatant was concentrated using an Amicon Ultra Centrifugal Filter 10 kDa MWCO (Merck Millipore, UFC901096) and loaded onto a HiTrap Heparin HP affinity column (Cytiva, 17040701) and eluted with a linear gradient of NaCl (0–1 M). The main peak fractions were pooled and fractionated through a Superdex-200 10/300 GL column (Cytiva, 17517501) using 50 mM Tris-HCl (pH 7.5) containing 150 mM NaCl as a mobile phase. The main peak fractions were pooled and concentrated using an Amicon Ultra Centrifugal Filter 10 kDa MWCO. The sample was stored in the presence of 650–800 mM NaCl to avoid precipitation.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec36">Purification of Smc</h4><p>The coding sequence of <i>T. kodakarensis</i> Smc with a 5’ extension of ATGTGG was cloned into pET21a(+) and expressed without any additional sequence. <i>E. coli</i> cells harboring the expression plasmid were cultured in LB medium containing 50 μg/mL of ampicillin and 34 μg/mL of chloramphenicol. The culture was grown to an OD<sub>600</sub> of ~0.4 in LB at 37 °C. Expression of the <i>smc</i> gene was induced with 1 mM IPTG for 4 h at 37 °C. The cells were pelleted by centrifugation and resuspended in 50 mM Tris-HCl (pH 8) containing 1 mM EDTA and 500 mM NaCl. After cell disruption by sonication, the cell extract was obtained by centrifugation followed by heating for 20 min at 80 °C. The heat-denatured proteins were removed by centrifugation. The supernatant was mixed with polyethylenimine (final 0.15%) and incubated for 10 min on ice. Precipitated nucleic acids were removed by centrifugation (23,708 × <i>g</i>, 10 min, 4 °C). Ammonium sulfate was added to the supernatant to 70% saturation, and the soluble proteins were precipitated by overnight incubation at 4 °C. The proteins were recovered by centrifugation (23,708 × <i>g</i>, 20 min, 4 °C). The precipitate was dissolved in 50 mM Tris-HCl (pH 8) containing 1 mM EDTA and 1 M ammonium sulfate. Debris were removed from the solution through a 0.45-μm Minisart Filter (Sartorius, S6555—FMOSK). The sample was loaded onto a HiTrap Phenyl HP column (Cytiva, 17135101), and the chromatography was developed with a linear gradient of ammonium sulfate (1–0 M). The main peak fractions were pooled and dialyzed overnight against 50 mM Tris-HCl (pH 8) containing 1 mM EDTA and 100 mM NaCl. The dialyzed sample was loaded onto a HiTrap Heparin HP affinity column. Elution was performed with a linear gradient of NaCl (0–1 M). The main peak fractions were pooled and diluted in 50 mM Tris-HCl (pH 8) containing 1 mM EDTA so that the NaCl concentration was adjusted to 100 mM. Lastly, the Smc protein was purified through a HiTrap Q HP column (Cytiva, 17115401) with a linear gradient of NaCl (0.1–1 M).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec37">Purification of TrmBL2</h4><p>The coding sequence of <i>T. kodakarensis</i> TrmBL2 was cloned into pET21a(+) and expressed without any additional sequence. The expression plasmid was kindly provided by Hugo Maruyama<sup><a aria-label="Reference 61" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR61" id="ref-link-section-d18519924e3660" title="Maruyama, H. et al. Histone and TK0471/TrmBL2 form a novel heterogeneous genome architecture in the hyperthermophilic archaeon Thermococcus kodakarensis. Mol. Biol. Cell 22, 386–398 (2011).">61</a></sup>. Protein expression was induced as described previously<sup><a aria-label="Reference 61" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR61" id="ref-link-section-d18519924e3664" title="Maruyama, H. et al. Histone and TK0471/TrmBL2 form a novel heterogeneous genome architecture in the hyperthermophilic archaeon Thermococcus kodakarensis. Mol. Biol. Cell 22, 386–398 (2011).">61</a></sup>. A heat-stable protein extract was prepared and concentrated as described for XerA purification, except that heating was performed at 80 °C. The extract was loaded onto a HiTrap Heparin HP affinity column and eluted with a linear gradient of NaCl (0–2 M). A pool of the main peak fractions was loaded onto a ResourceQ anion exchange column (Cytiva, 17117901) and eluted with a linear gradient of NaCl (0–1 M). The main peak fractions were pooled and concentrated using an Amicon Ultra Centrifugal Filter 10 kDa MWCO. The sample was stored in 50 mM Tris-HCl (pH 8.0) containing 240 mM NaCl.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec38">Antibody production</h4><p>The purified recombinant proteins were used to raise rabbit polyclonal antibodies (Eurofins Genomics).</p><h3 class="c-article__sub-heading" id="Sec39">DNA affinity purification</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec40">DNA probes</h4><p>PCR was performed to amplify <i>difL</i> and <i>difR</i> (using genomic DNA of <i>T. kodakarensis</i> as a template) and <i>kanR</i> (using the pGBKT7 plasmid as a template) and clone them separately into the pUC19 plasmid (digested at the BamHI and SphI sites) using In-Fusion HD Cloning Plus reagents. The primer sets used for the cloning are shown in Supplementary Data <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM4">2</a>. The inserted fragments were PCR amplified to generate biotinylated DNA probes (see Supplementary Data <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM4">2</a> for oligonucleotide sequences). The probes were purified using FastGene Gel/PCR Extraction Kit (Nippon Genetics, FG-91302).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec41">Cell extract preparation and affinity purification</h4><p>80 mL of hot cell culture of KU216 were dispensed into two 50-mL tubes, chilled in iced water for 10 min, and centrifuged (10,000 × <i>g</i>, 10 min, 4 °C). The two pellets were resuspended together in 3.5 mL of buffer H (20 mM Tris-HCl [pH 7.5], 500 mM NaCl) and sonicated. The lysate was cleared by centrifugation (12,000 × <i>g</i>, 10 min, 4 °C). 3 mL of the supernatant were mixed with 7 mL of 20 mM Tris-HCl (pH 7.5) containing 0.714% v/v Tween-20. The diluted lysate corresponding to 0.9 mg protein was mixed with 45 µL of 2 mg/mL Poly(dI-dC) (Sigma Aldrich, 81349-500UG). The sample was mixed with Dynabeads MyOne Streptavidin T1 (Thermo Fisher Scientific, 65602) pre-coupled to 9 µg of a biotinylated probe. The mixture was incubated for 30 min at 25 °C with agitation at 600 rpm. The beads were recovered and resuspended in 1.5 mL of buffer MT (20 mM Tris-HCl [pH 7.5], 150 mM NaCl, 0.5% v/v Tween-20). Beads were recovered from 500 µL of the sample for total bound proteins and from 1000 µL for eluates 1 and 2 in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig6">6</a>.</p><p>The beads for total bound proteins were washed with buffer MT and resuspended in 30 µL of buffer W600 (20 mM Tris-HCl [pH 7.5], 600 mM NaCl, 0.5% v/v Tween-20). The sample was mixed with 15 µL of 3 × Blue Protein Loading Dye (New England Biolabs, B7703S) and incubated for 10 min at 37 °C with agitation at 600 rpm to elute the bound proteins. The supernatant was recovered, mixed with 1.5 µL of 1.25 M DTT (New England Biolabs, B7703S), and heated for 5 min at 95 °C. The beads for eluates 1 and 2 were first washed with buffer MT. Proteins were then sequentially eluted from the beads with (1) 56.2 µL of buffer W300 (20 mM Tris-HCl [pH 7.5], 300 mM NaCl, 0.5% v/v Tween-20), (2) 58.1 µL of buffer W450 (20 mM Tris-HCl [pH 7.5], 450 mM NaCl, 0.5% v/v Tween-20), (3) 60 µL of buffer W600, (4) 48 µL of buffer W750 (20 mM Tris-HCl [pH 7.5], 750 mM NaCl, 0.5% v/v Tween-20), (5) 36 µL of buffer W1000 (20 mM Tris-HCl [pH 7.5], 1 M NaCl, 0.5% v/v Tween-20), and (6) 90 µL of SDS sample buffer (buffer W600 containing 1 × Blue Protein Loading Dye). Each elution was performed by incubating the bead suspension for 10 min at 37 °C with agitation at 600 rpm and recovering the supernatant. The supernatant from (5), used as eluate 1, was mixed with 12 μL of 20 mM Tris-HCl (pH 7.5) and 30 µL of 3 × Blue Protein Loading Dye. The supernatant from (7) was used as eluate 2. Eluates 1 and 2 were each mixed with 3 µL of 1.25 M DTT and heated for 5 min at 95 °C.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec42">Detection of bound proteins</h4><p>20 µL of the DNA affinity purification sample per lane were subjected to SDS-PAGE using e-PAGEL HR 5–20% gradient gel (ATTO, 2331970). The gel was stained with Pierce Silver Stain for Mass Spectrometry (Thermo Fisher Scientific, 24600). Indicated gel bands were excised, digested with trypsin, and subjected to label-free quantification analysis by NanoLC-MS/MS using a Q Exactive mass spectrometer (Thermo Fisher Scientific) and an Ultimate 3000 nanoLC pump (AMR). Peptides and proteins were identified by means of automated database searching using Sequest HT (Thermo Fisher Scientific) against the <i>T. kodakarensis</i> database (Taxon ID 311400, UniProtKB/SWISS-PROT, release 2018-01) with a precursor mass tolerance of 10 p.p.m., a fragment ion mass tolerance of 0.02 Da, and trypsin specificity that allows for up to two missed cleavages. A reversed decoy database search was conducted with Percolator node, setting the false discovery rate (FDR) threshold to 5% at the peptide level.</p><h3 class="c-article__sub-heading" id="Sec43">Identification of putative <i>dif</i> sequences near domain boundaries</h3><p>Prior work reported nine Smc-dependent boundaries and their closest genes in <i>H. volcanii</i><sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e3754" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup>. We first retrieved the sequences of these genes together with their upstream and downstream 4-kb regions. To identify potential boundary-associated <i>dif</i> sequences, we searched these nine regions for imperfect inverted repeats containing TAA(N)<sub>n</sub>TTA (<i>n</i> = 6–8). We selected this motif since it is the most highly conserved feature among the previously reported <i>dif</i> sequences in archaea (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">5a</a>). We identified three sequences containing the motif, one of which is located upstream of the <i>xerA</i>-containing operon as illustrated in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig4">4E</a>. Another hit is located upstream of <i>HVO_B0249</i> as shown in Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">5b</a>. The third hit (AATCGAGAGTAAGGAGAGTATTACTAACCGGC) is located ~200 bp away from the second one. The third sequence was removed from the candidates due to a smaller number of base matches than that in the second one. The <i>dif</i>-like sequence in <i>H. salinarum</i> was identified by searching for the intergenic region upstream of the same <i>xerA</i>-containing operon.</p><h3 class="c-article__sub-heading" id="Sec44">Illumina DNA sequencing and read mapping</h3><p>The 3C-seq, RNA-seq, and ChIP-seq libraries generated in this study were paired-end sequenced on either of the Illumina HiSeq X Ten, NovaSeq 6000, NextSeq 550, and NextSeq 500 platforms. The sequencing was performed by Macrogen, Single-cell Genome Information Analysis Core at Kyoto University, and the NGS core facility of the Graduate Schools of Biostudies at Kyoto University. Sequence data were mapped to the reference genome of <i>T. kodakarensis</i> KOD1 (GCA_000009965.1), <i>T. acidophilum</i> DSM 1728 (GCA_000195915.1), <i>H. volcanii</i> DS2 (GCA_000025685.1) or <i>H. salinarum</i> NRC-1 (GCA_000006805.1). For analysis of the <i>T. kodakarensis</i> strains lacking <i>boundary1</i>, the deletion was also introduced into the reference genome sequence. For analysis of the <i>T. kodakarensis</i> strains possessing the ectopic <i>boundary1</i><sup>Δ<i>dif</i></sup> sequence, the insert was also introduced into the reference genome sequence.</p><h3 class="c-article__sub-heading" id="Sec45">3C-seq data analysis</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec46">Generation of 3C-seq contact maps</h4><p>3C-seq contact maps were generated using HiC-Pro (version 3.0.0)<sup><a aria-label="Reference 114" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR114" id="ref-link-section-d18519924e3844" title="Servant, N. et al. HiC-Pro: an optimized and flexible pipeline for Hi-C data processing. Genome Biol. 16, 259 (2015).">114</a></sup>. The data were iteratively corrected with the MAX_ITER parameter of 500. The obtained matrices were normalized so that the sum of interaction scores is equal to 1000 for each row and column. For <i>T. kodakarensis</i>, genomic bins with extremely low coverage were filtered out by setting the FILTER_LOW_COUNT_PER parameter to 0.006. 3C-seq contact maps reflecting the genomic inversion in <i>T. kodakarensis</i> were generated by flipping the sequence corresponding to the genomic coordinates 327001-520000.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec47">Insulation score analysis</h4><p>Insulation score was determined as described previously at 1-kb resolution<sup><a aria-label="Reference 39" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR39" id="ref-link-section-d18519924e3862" title="Takemata, N. &amp; Bell, S. D. Multi-scale architecture of archaeal chromosomes. Mol. Cell 81, 473–487 e476 (2021).">39</a></sup>. The size of the sliding square was set to 40 kb. The insulation score profile from KU216 formed two major valleys with the local minima at 483–484 kb and 1561–1562 kb, which was defined as the positions of boundaries 1 and 2 respectively. We also used the boundary caller RHiCDB (version 1.0), an R-packaged version of HiCDB<sup><a aria-label="Reference 51" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR51" id="ref-link-section-d18519924e3866" title="Chen, F., Li, G., Zhang, M. Q. &amp; Chen, Y. HiCDB: a sensitive and robust method for detecting contact domain boundaries. Nucleic Acids Res. 46, 11239–11250 (2018).">51</a></sup>, to verify the presence of boundaries 1 and 2 in KU216. RHiCDB was also applied to the 3C-seq data of the Δ<i>smc</i> and Δ<i>trmBL2</i> strains to test the loss of the two boundaries. RHiCDB was used with the following parameters: resolution = 1000, wd = 20, and wdsize = 4.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec48">Detection of loop structures</h4><p>DNA loops in the <i>T. kodakarensis</i> strain KU216 were searched for using Chromosight (version 1.5.0)<sup><a aria-label="Reference 53" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR53" id="ref-link-section-d18519924e3887" title="Matthey-Doret, C. et al. Computer vision for pattern detection in chromosome contact maps. Nat. Commun. 11, 5795 (2020).">53</a></sup> as described previously<sup><a aria-label="Reference 39" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR39" id="ref-link-section-d18519924e3891" title="Takemata, N. &amp; Bell, S. D. Multi-scale architecture of archaeal chromosomes. Mol. Cell 81, 473–487 e476 (2021).">39</a></sup>. 3C-seq contact maps at 1-kb resolution were used for the analysis. A genomic segment containing the inversion (the genomic coordinates 321001–525000) were omitted due to the uncertainty resulting from the genome heterogeneity caused by the inversion. The max-dist parameter was set to 1044000.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec49">Virtual 4C analysis</h4><p>3C-seq contact maps at 2-kb resolution were used for the analysis. The genomic bin 616001–618000 was used as a viewpoint, and its contact frequencies with the other bins were plotted. The genomic interval 362001–524000, containing <i>TKV2</i><sub><i>HR</i></sub>, <i>TKV3</i><sub><i>HR</i></sub>, and the inverted segment, were omitted.</p><h3 class="c-article__sub-heading" id="Sec50">RNA-seq data analysis</h3><p>Differential gene expression analysis was performed using the GFF annotation file of the <i>T. kodakarensis</i> reference genome. The 23S, 16S, and 5S rRNA genes were omitted from the analysis. The orientation of the 7S (SRP) RNA was inverted, because the original orientation in the genome annotation is probably incorrect as reported previously<sup><a aria-label="Reference 110" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR110" id="ref-link-section-d18519924e3928" title="Jager, D., Forstner, K. U., Sharma, C. M., Santangelo, T. J. &amp; Reeve, J. N. Primary transcriptome map of the hyperthermophilic archaeon Thermococcus kodakarensis. BMC Genom. 15, 684 (2014).">110</a></sup>. RNA levels were first quantified by Salmon (version 1.9.0)<sup><a aria-label="Reference 115" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR115" id="ref-link-section-d18519924e3932" title="Patro, R., Duggal, G., Love, M. I., Irizarry, R. A. &amp; Kingsford, C. Salmon provides fast and bias-aware quantification of transcript expression. Nat. Methods 14, 417–419 (2017).">115</a></sup> and then analyzed using edgeR (version 3.40.2)<sup><a aria-label="Reference 116" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR116" id="ref-link-section-d18519924e3936" title="Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139–140 (2010).">116</a></sup> as follows. The read count was normalized using the TMM normalization method. Common, trended, and tagwise dispersions were estimated using the GLM method. Statistical significance of the gene expression difference was tested using the glmFit function.</p><h3 class="c-article__sub-heading" id="Sec51">ChIP-seq data analysis</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec52">Mapping</h4><p>Reads were mapped using Bowtie 2 (version 2.3.5.1)<sup><a aria-label="Reference 117" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR117" id="ref-link-section-d18519924e3952" title="Langmead, B. &amp; Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Nat. Methods 9, 357–359 (2012).">117</a></sup>. Low-quality alignments (MAPQ &lt; 30) were removed using SAMtools (version 1.9)<sup><a aria-label="Reference 118" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR118" id="ref-link-section-d18519924e3956" title="Li, H. et al. The sequence alignment/map format and SAMtools. Bioinformatics 25, 2078–2079 (2009).">118</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec53">Generation of ChIP-seq tracks</h4><p>Generated BAM files were processed using the bamCoverage function (version 3.5.1) of deepTools<sup><a aria-label="Reference 119" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR119" id="ref-link-section-d18519924e3968" title="Ramirez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. Nucleic Acids Res. 44, W160–W165 (2016).">119</a></sup> to calculate Reads Per Kilobase region per Million mapped reads (RPKM) for genomic bins of 50 bp and 1 kb. RPKM ratios of immunoprecipitated versus input DNA were plotted as IP/input.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec54">Peak analysis</h4><p>Mapped data were processed for peak calling using MACS (version 3.0.0a6)<sup><a aria-label="Reference 120" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR120" id="ref-link-section-d18519924e3980" title="Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). Genome Biol. 9, R137 (2008).">120</a></sup> with the following setting: --fe-cutoff 3, --keep-dup all, --call-summits, -m 1 50. To identify common binding sites for Smc and TrmBL2, overlaps of ±100-bp regions from the identified peaks were examined using the numOverlaps function of regioneR (version 1.30.0)<sup><a aria-label="Reference 121" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR121" id="ref-link-section-d18519924e3984" title="Gel, B. et al. regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests. Bioinformatics 32, 289–291 (2016).">121</a></sup>. Statistical significance of the overlaps was evaluated using the overlapPermuTest function of regioneR (ntimes = 1000). DNA motifs enriched at identified peaks were determined using MEME-ChIP (version 5.5.5)<sup><a aria-label="Reference 122" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR122" id="ref-link-section-d18519924e3988" title="Machanick, P. &amp; Bailey, T. L. MEME-ChIP: motif analysis of large DNA datasets. Bioinformatics 27, 1696–1697 (2011).">122</a></sup> with the following setting: -ccut 100, -order 2, -minw 4, -maxw 15, -meme-mod zoops. To calculate protein occupancy at the identified peak, the read coverage of the ±100-bp region from the peak summit was calculated using the multicov function of BEDtools (version 2.26.0)<sup><a aria-label="Reference 123" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR123" id="ref-link-section-d18519924e3992" title="Quinlan, A. R. &amp; Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics 26, 841–842 (2010).">123</a></sup>. The coverage was normalized to the total read number, and the IP/input ratio of the normalized coverage values was used as the occupancy of the protein at the peak.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec55">Smc positioning efficiency</h4><p>Using ChIP-seq data from KU216, we first calculated Smc occupancy for the Smc peaks overlapping with those of TrmBL2. Smc occupancy in ∆<i>trmBL2</i> was also calculated for the same regions. These values were used to calculate the Smc occupancy ratio between KU216 and ∆<i>trmBL2</i>. Loci with the ∆<i>trmBL2</i>/KU216 ratio of –0.5 or larger were filtered out. We then calculated TrmBL2 occupancy for the TrmBL2 peaks overlapping with the retained Smc peaks in KU216. These peak pairs were used to calculate the ratio of Smc versus TrmBL2 occupancies, which was defined as Smc positioning efficiency.</p><h3 class="c-article__sub-heading" id="Sec56">Analysis on DNA sequence features</h3><p>AT-content tracks were generated using SeqKit (version 2.5.1)<sup><a aria-label="Reference 124" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR124" id="ref-link-section-d18519924e4022" title="Shen, W., Le, S., Li, Y. &amp; Hu, F. SeqKit: a cross-platform and ultrafast toolkit for FASTA/Q file manipulation. PLoS ONE 11, e0163962 (2016).">124</a></sup> with sliding window size of 200 bp and step size of 100 bp. For analysis on colocalization sites of TrmBL2 and Smc, 200-bp sequences centered at TrmBL2 peaks were used. The melting temperatures of colocalization sites were predicted using T<sub>m</sub> for Oligos Calculator (<a href="https://www.promega.jp/resources/tools/biomath/tm-calculator/">https://www.promega.jp/resources/tools/biomath/tm-calculator/</a>), which is based on the nearest-neighbor method. The following parameters were used for the prediction: primer concentration: 200 nM, Na<sup>+</sup>/K<sup>+</sup> concentration: 700 mM, Mg<sup>2+</sup> concentration: 1 mM. Although the high concentration of Na<sup>+</sup>/K<sup>+</sup> was adopted according to intracellular K<sup>+</sup> concentrations in thermophilic methanogenic archaea<sup><a aria-label="Reference 125" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR125" id="ref-link-section-d18519924e4048" title="Hensel, R. &amp; König, H. Thermoadaptation of methanogenic bacteria by intracellular ion concentration. FEMS Microbiol. Lett. 49, 75–79 (1988).">125</a></sup>, this value did not significantly affect the result. Occurrence of DNA tetranucleotide sequences was counted for each stalling site using the countDnaKmers function of seqTools (version 1.32.0). For non-palindrome sequences, their occurrence frequencies plus those of the reverse complements were used for analyses. The occurrence frequency dataset was then used to identify the tetranucleotide sequences that were correlated with Smc positioning efficiency. The same dataset was used to calculate the average of 4-bp-scale persistence lengths for each stalling site. The persistence length values were according to a previous study<sup><a aria-label="Reference 73" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR73" id="ref-link-section-d18519924e4052" title="Velasco-Berrelleza, V. et al. SerraNA: a program to determine nucleic acids elasticity from simulation data. Phys. Chem. Chem. Phys. 22, 19254–19266 (2020).">73</a></sup>. The other DNA properties were predicted using the Deep DNAshape web server (<a href="https://deepdnashape.usc.edu/">https://deepdnashape.usc.edu/</a>)<sup><a aria-label="Reference 74" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR74" id="ref-link-section-d18519924e4064" title="Li, J., Chiu, T. P. &amp; Rohs, R. Predicting DNA structure using a deep learning method. Nat. Commun. 15, 1243 (2024).">74</a></sup> with the Deep DNAshape layer set to 4. This server predicts a selected feature value for each base of the input sequence. The base average of the feature value was calculated for each stalling site and used for correlation analysis. Adjusted <i>p</i>-values were determined using the Benjamini-Hochberg method.</p><h3 class="c-article__sub-heading" id="Sec57">Coarse-grained MD simulations of TrmBL2/Smc colocalization sites</h3><p>The coarse-grained MD simulations were conducted with CafeMol (version 3.2.1)<sup><a aria-label="Reference 126" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR126" id="ref-link-section-d18519924e4079" title="Kenzaki, H. et al. CafeMol: A coarse-grained biomolecular simulator for simulating proteins at work. J. Chem. Theory Comput. 7, 1979–1989 (2011).">126</a></sup>. The 3SPN.2C sequence-dependent coarse-grained DNA model was used<sup><a aria-label="Reference 81" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR81" id="ref-link-section-d18519924e4083" title="Freeman, G. S., Hinckley, D. M., Lequieu, J. P., Whitmer, J. K. &amp; de Pablo, J. J. Coarse-grained modeling of DNA curvature. J. Chem. Phys. 141, 165103 (2014).">81</a></sup>. 5’- and 3’ termini of stalling sites were capped with five CG repeats to insulate the analyzed sequences from end effects, but the CG caps were omitted when gross dynamic PLs were calculated from the obtained MD data. The temperature was set to 300 K. The monovalent ion concentration of Debye-Hückel model was set to 150 mM. Langevin dynamics simulations were performed with a step size of 0.2 in the CafeMol time unit for 2 × 10<sup>8</sup> steps per stalling site, including 5 × 10<sup>7</sup> steps for equilibration. The coordinates were stored every 5 × 10<sup>3</sup> steps.</p><p>The gross dynamic PL, <span class="mathjax-tex">\({l}_{{{\rm{d}}}}\)</span>, was estimated from the directional correlation decay of DNA polymer as <span class="mathjax-tex">\({C}_{{{\rm{d}}}}\left(s\right)\equiv {C}_{{{\rm{p}}}}(s)/{C}_{{{\rm{s}}}}(s)\approx \exp \left(-s/{l}_{{{\rm{d}}}}\right)\)</span> as performed previously<sup><a aria-label="Reference 72" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR72" id="ref-link-section-d18519924e4252" title="Mitchell, J. S., Glowacki, J., Grandchamp, A. E., Manning, R. S. &amp; Maddocks, J. H. Sequence-dependent persistence lengths of DNA. J. Chem. Theory Comput. 13, 1539–1555 (2017).">72</a></sup>. Here, <span class="mathjax-tex">\({C}_{{{\rm{p}}}}\left(s\right)\)</span> is the autocorrelation function of the helical axis vectors <span class="mathjax-tex">\({{\boldsymbol{h}}}\)</span> defined as <span class="mathjax-tex">\({C}_{{{\rm{p}}}}\left(s\right)=\, &lt; {\left\{{{\boldsymbol{h}}}\left(l\right)\bullet {{\boldsymbol{h}}}\left(l+s\right)\right\}}_{l} &gt; \,\approx \exp (-s/{l}_{{{\rm{p}}}})\)</span>, <span class="mathjax-tex">\({C}_{{{\rm{s}}}}(s)\)</span> is that for minimum-energy ground state conformation defined as <span class="mathjax-tex">\({C}_{{{\rm{s}}}}\left(s\right)={\{{{{\boldsymbol{h}}}}^{\min }\left(l\right)\bullet {{{\boldsymbol{h}}}}^{\min }\left(l+s\right)\}}_{l}\approx \exp \left(-s/{l}_{{{\rm{s}}}}\right)\)</span>, <span class="mathjax-tex">\(s\)</span> is the spacing between helical axis vectors, <span class="mathjax-tex">\(l\)</span> is the position along the contour length, <span class="mathjax-tex">\({l}_{{{\rm{p}}}}\)</span> is the PL, and <span class="mathjax-tex">\({l}_{{{\rm{s}}}}\)</span> is the static PL. <span class="mathjax-tex">\(s\)</span> is a multiple of 10 in order to remove any residual effects of DNA helicity. {・} represents an average over the position along the contour length and &lt;・&gt; represents a long-time average. The sequence-dependent ground state conformation was obtained using x3DNA (version 2.4)<sup><a aria-label="Reference 127" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR127" id="ref-link-section-d18519924e4737" title="Lu, X. J. &amp; Olson, W. K. 3DNA: a versatile, integrated software system for the analysis, rebuilding and visualization of three-dimensional nucleic-acid structures. Nat. Protoc. 3, 1213–1227 (2008).">127</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec58">Data visualization and statistical test</h3><p>Unless otherwise stated, data were visualized using R software (<a href="http://www.r-project.org/">http://www.r-project.org/</a>).</p><h3 class="c-article__sub-heading" id="Sec59">Statistics and reproducibility</h3><p>Statistical test was performed using R software. Reproducibility of 3C-seq results was confirmed using two (all strains except KU216) or three (KU216) biological replicates. Reproducibility of RNA-seq results were confirmed using three biological replicates. Reproducibility of ChIP-seq results was confirmed using two biological replicates. Unless otherwise stated, reads from the biological replicates were pooled for analysis. Growth measurement of <i>T. kodakarensis</i> cells were repeated twice independently with similar results. Microscopic observation of <i>T. kodakarensis</i> cells was performed for a single culture. In the microscopic analysis, 50 images were taken for each strain, all of which show similar results. The number of cells used for the image quantification is shown in Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM1">4d</a>. The specific bands observed in the DNA affinity purification in Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Fig6">6c</a> were reproduced in three independent experiments. Samples obtained from one of the three experiments were analyzed by mass spectrometry in technical duplicates.</p><h3 class="c-article__sub-heading" id="Sec60">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56197-y#MOESM5">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
</div>
<div class="u-mt-32">
<section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
<p>The sequence data generated in this study have been deposited in the NCBI’s Gene Expression Omnibus database under the GEO series accession codes <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE267297">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE267297</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE267298">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE267298</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE267299">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE267299</a>. The mass spectrometry data generated in this study have been deposited in the ProteomeXchange Consortium via the jPOST partner repository<sup><a aria-label="Reference 128" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR128" id="ref-link-section-d18519924e4890" title="Okuda, S. et al. jPOSTrepo: an international standard data repository for proteomes. Nucleic Acids Res. 45, D1107–D1111 (2017).">128</a></sup> (<a href="http://jpostdb.org">http://jpostdb.org</a>) with the PDX identifier <a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD053073">https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD053073</a>. The raw microscopic image data generated in this study have been deposited in FigShare (<a href="https://figshare.com/projects/Microscopic_images_of_Smc-ScpAB_mutants/231431">https://figshare.com/projects/Microscopic_images_of_Smc-ScpAB_mutants/231431</a>). The published Hi-C data<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56197-y#ref-CR41" id="ref-link-section-d18519924e4916" title="Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. Mol. Cell 81, 459–472.e410 (2021).">41</a></sup> used in this study are available in the NCBI Sequence Read Archive (SRA) database under the following accession codes: <i>T. kodakarensis</i>: <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR12717836">https://www.ncbi.nlm.nih.gov/sra/?term=SRR12717836</a> and <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR12717848">https://www.ncbi.nlm.nih.gov/sra/?term=SRR12717848</a>, <i>H. volcanii</i> DS2: <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR11747722">https://www.ncbi.nlm.nih.gov/sra/?term=SRR11747722</a> and <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR11747734">https://www.ncbi.nlm.nih.gov/sra/?term=SRR11747734</a>, <i>H. volcanii</i> ∆<i>smc</i>: <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR11747720">https://www.ncbi.nlm.nih.gov/sra/?term=SRR11747720</a> and <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR11747731">https://www.ncbi.nlm.nih.gov/sra/?term=SRR11747731</a>, <i>H. salinarum</i>: <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR12717837">https://www.ncbi.nlm.nih.gov/sra/?term=SRR12717837</a> and <a href="https://www.ncbi.nlm.nih.gov/sra/?term=SRR12717838">https://www.ncbi.nlm.nih.gov/sra/?term=SRR12717838</a>. The insulation score profiles generated from 3C-seq data are available in the Source Data file. The growth curve data and uncropped gel image data generated in this study are provided in the Source Data file. <a data-track="click" data-track-action="section anchor" data-track-label="link" href="/articles/s41467-025-56197-y#Sec62">Source data</a> are provided with this paper.</p>
</div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Yatskevich, S., Rhodes, J. &amp; Nasmyth, K. Organization of chromosomal DNA by SMC complexes. <i>Annu. Rev. Genet.</i> <b>53</b>, 445–482 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 1" data-doi="10.1146/annurev-genet-112618-043633" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1146/annurev-genet-112618-043633" data-track-label="10.1146/annurev-genet-112618-043633" data-track-value="article reference" href="https://doi.org/10.1146%2Fannurev-genet-112618-043633" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 1" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvV2msbfO" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 1" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31577909" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 1" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0353.54028" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 1" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Organization%20of%20chromosomal%20DNA%20by%20SMC%20complexes&amp;journal=Annu.%20Rev.%20Genet.&amp;doi=10.1146%2Fannurev-genet-112618-043633&amp;volume=53&amp;pages=445-482&amp;publication_year=2019&amp;author=Yatskevich%2CS&amp;author=Rhodes%2CJ&amp;author=Nasmyth%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Uhlmann, F. SMC complexes: from DNA to chromosomes. <i>Nat. Rev. Mol. Cell Biol.</i> <b>17</b>, 399–412 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 2" data-doi="10.1038/nrm.2016.30" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nrm.2016.30" data-track-label="10.1038/nrm.2016.30" data-track-value="article reference" href="https://doi.org/10.1038%2Fnrm.2016.30" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 2" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XmtVSltb4%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 2" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27075410" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 2" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1157.81307" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 2" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=SMC%20complexes%3A%20from%20DNA%20to%20chromosomes&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;doi=10.1038%2Fnrm.2016.30&amp;volume=17&amp;pages=399-412&amp;publication_year=2016&amp;author=Uhlmann%2CF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Hirano, T. Condensin-based chromosome organization from bacteria to vertebrates. <i>Cell</i> <b>164</b>, 847–857 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 3" data-doi="10.1016/j.cell.2016.01.033" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2016.01.033" data-track-label="10.1016/j.cell.2016.01.033" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2016.01.033" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 3" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XjsValt74%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 3" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26919425" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 3" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?07878178" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 3" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Condensin-based%20chromosome%20organization%20from%20bacteria%20to%20vertebrates&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2016.01.033&amp;volume=164&amp;pages=847-857&amp;publication_year=2016&amp;author=Hirano%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Wells, J. N., Gligoris, T. G., Nasmyth, K. A. &amp; Marsh, J. A. Evolution of condensin and cohesin complexes driven by replacement of Kite by Hawk proteins. <i>Curr. Biol.</i> <b>27</b>, R17–R18 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 4" data-doi="10.1016/j.cub.2016.11.050" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cub.2016.11.050" data-track-label="10.1016/j.cub.2016.11.050" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cub.2016.11.050" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 4" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXotlKgsA%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 4" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28073014" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 4" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5228436" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 4" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Evolution%20of%20condensin%20and%20cohesin%20complexes%20driven%20by%20replacement%20of%20Kite%20by%20Hawk%20proteins&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2016.11.050&amp;volume=27&amp;pages=R17-R18&amp;publication_year=2017&amp;author=Wells%2CJ.%20N&amp;author=Gligoris%2CT.%20G&amp;author=Nasmyth%2CK.%20A&amp;author=Marsh%2CJ.%20A" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Palecek, J. J. &amp; Gruber, S. Kite proteins: a superfamily of SMC/Kleisin partners conserved across bacteria, archaea, and eukaryotes. <i>Structure</i> <b>23</b>, 2183–2190 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 5" data-doi="10.1016/j.str.2015.10.004" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.str.2015.10.004" data-track-label="10.1016/j.str.2015.10.004" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.str.2015.10.004" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 5" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVehtrjE" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 5" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26585514" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 5" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0048.17705" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 5" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Kite%20proteins%3A%20a%20superfamily%20of%20SMC%2FKleisin%20partners%20conserved%20across%20bacteria%2C%20archaea%2C%20and%20eukaryotes&amp;journal=Structure&amp;doi=10.1016%2Fj.str.2015.10.004&amp;volume=23&amp;pages=2183-2190&amp;publication_year=2015&amp;author=Palecek%2CJ.%20J&amp;author=Gruber%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Davidson, I. F. &amp; Peters, J. M. Genome folding through loop extrusion by SMC complexes. <i>Nat. Rev. Mol. Cell Biol.</i> <b>22</b>, 445–464 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 6" data-doi="10.1038/s41580-021-00349-7" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41580-021-00349-7" data-track-label="10.1038/s41580-021-00349-7" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41580-021-00349-7" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 6" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXnsFKjs78%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 6" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33767413" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 6" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome%20folding%20through%20loop%20extrusion%20by%20SMC%20complexes&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;doi=10.1038%2Fs41580-021-00349-7&amp;volume=22&amp;pages=445-464&amp;publication_year=2021&amp;author=Davidson%2CI.%20F&amp;author=Peters%2CJ.%20M" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Kim, E., Barth, R. &amp; Dekker, C. Looping the genome with SMC complexes. <i>Annu. Rev. Biochem.</i> <b>92</b>, 15–41 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 7" data-doi="10.1146/annurev-biochem-032620-110506" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1146/annurev-biochem-032620-110506" data-track-label="10.1146/annurev-biochem-032620-110506" data-track-value="article reference" href="https://doi.org/10.1146%2Fannurev-biochem-032620-110506" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 7" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXpsVynsLw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 7" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37137166" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 7" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1543.46031" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 7" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Looping%20the%20genome%20with%20SMC%20complexes&amp;journal=Annu.%20Rev.%20Biochem.&amp;doi=10.1146%2Fannurev-biochem-032620-110506&amp;volume=92&amp;pages=15-41&amp;publication_year=2023&amp;author=Kim%2CE&amp;author=Barth%2CR&amp;author=Dekker%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Nora, E. P. et al. Spatial partitioning of the regulatory landscape of the X-inactivation centre. <i>Nature</i> <b>485</b>, 381–385 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 8" data-doi="10.1038/nature11049" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature11049" data-track-label="10.1038/nature11049" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature11049" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 8" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012Natur.485..381N" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 8" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XlslSisrw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 8" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22495304" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 8" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3555144" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 8" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1281.62090" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 8" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Spatial%20partitioning%20of%20the%20regulatory%20landscape%20of%20the%20X-inactivation%20centre&amp;journal=Nature&amp;doi=10.1038%2Fnature11049&amp;volume=485&amp;pages=381-385&amp;publication_year=2012&amp;author=Nora%2CE.%20P" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Rao, S. S. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. <i>Cell</i> <b>159</b>, 1665–1680 (2014).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 9" data-doi="10.1016/j.cell.2014.11.021" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2014.11.021" data-track-label="10.1016/j.cell.2014.11.021" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2014.11.021" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 9" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFCrtrrK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 9" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25497547" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 9" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5635824" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 9" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0663.68043" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 9" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%203D%20map%20of%20the%20human%20genome%20at%20kilobase%20resolution%20reveals%20principles%20of%20chromatin%20looping&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2014.11.021&amp;volume=159&amp;pages=1665-1680&amp;publication_year=2014&amp;author=Rao%2CS.%20S" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Nora, E. P. et al. Targeted degradation of CTCF decouples local insulation of chromosome domains from genomic compartmentalization. <i>Cell</i> <b>169</b>, 930–944 e922 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 10" data-doi="10.1016/j.cell.2017.05.004" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2017.05.004" data-track-label="10.1016/j.cell.2017.05.004" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.05.004" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 10" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXotlOqsbs%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 10" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28525758" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 10" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5538188" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 10" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1387.00011" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 10" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20degradation%20of%20CTCF%20decouples%20local%20insulation%20of%20chromosome%20domains%20from%20genomic%20compartmentalization&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.05.004&amp;volume=169&amp;pages=930-944%20e922&amp;publication_year=2017&amp;author=Nora%2CE.%20P" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Banigan, E. J. et al. Transcription shapes 3D chromatin organization by interacting with loop extrusion. <i>Proc. Natl. Acad. Sci. USA</i> <b>120</b>, e2210480120 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 11" data-doi="10.1073/pnas.2210480120" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.2210480120" data-track-label="10.1073/pnas.2210480120" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.2210480120" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 11" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXosFOnu7s%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 11" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36897969" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 11" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10089175" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 11" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcription%20shapes%203D%20chromatin%20organization%20by%20interacting%20with%20loop%20extrusion&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2210480120&amp;volume=120&amp;publication_year=2023&amp;author=Banigan%2CE.%20J" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Jeppsson, K. et al. Cohesin-dependent chromosome loop extrusion is limited by transcription and stalled replication forks. <i>Sci. Adv.</i> <b>8</b>, eabn7063 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 12" data-doi="10.1126/sciadv.abn7063" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/sciadv.abn7063" data-track-label="10.1126/sciadv.abn7063" data-track-value="article reference" href="https://doi.org/10.1126%2Fsciadv.abn7063" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 12" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVegsLjE" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 12" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35687682" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 12" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9187231" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 12" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0825.73742" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 12" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohesin-dependent%20chromosome%20loop%20extrusion%20is%20limited%20by%20transcription%20and%20stalled%20replication%20forks&amp;journal=Sci.%20Adv.&amp;doi=10.1126%2Fsciadv.abn7063&amp;volume=8&amp;publication_year=2022&amp;author=Jeppsson%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Rao, S. S. P. et al. Cohesin loss eliminates all loop domains. <i>Cell</i> <b>171</b>, 305–320 e324 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 13" data-doi="10.1016/j.cell.2017.09.026" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2017.09.026" data-track-label="10.1016/j.cell.2017.09.026" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.09.026" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 13" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1WqsrfN" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 13" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28985562" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 13" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5846482" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 13" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohesin%20loss%20eliminates%20all%20loop%20domains&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.09.026&amp;volume=171&amp;pages=305-320%20e324&amp;publication_year=2017&amp;author=Rao%2CS.%20S.%20P" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Valton, A. L. et al. A cohesin traffic pattern genetically linked to gene regulation. <i>Nat. Struct. Mol. Biol.</i> <b>29</b>, 1239–1251 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 14" data-doi="10.1038/s41594-022-00890-9" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41594-022-00890-9" data-track-label="10.1038/s41594-022-00890-9" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41594-022-00890-9" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 14" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XjtVyltb7I" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 14" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36482254" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 14" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10228515" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 14" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0796.58019" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 14" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20cohesin%20traffic%20pattern%20genetically%20linked%20to%20gene%20regulation&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;doi=10.1038%2Fs41594-022-00890-9&amp;volume=29&amp;pages=1239-1251&amp;publication_year=2022&amp;author=Valton%2CA.%20L" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Costantino, L., Hsieh, T. S., Lamothe, R., Darzacq, X. &amp; Koshland, D. Cohesin residency determines chromatin loop patterns. <i>Elife</i> <b>9</b>, e59889 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 15" data-doi="10.7554/eLife.59889" data-track="click_references" data-track-action="article reference" data-track-item_id="10.7554/eLife.59889" data-track-label="10.7554/eLife.59889" data-track-value="article reference" href="https://doi.org/10.7554%2FeLife.59889" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 15" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXmslSrs7s%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 15" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33170773" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 15" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7655110" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 15" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohesin%20residency%20determines%20chromatin%20loop%20patterns&amp;journal=Elife&amp;doi=10.7554%2FeLife.59889&amp;volume=9&amp;publication_year=2020&amp;author=Costantino%2CL&amp;author=Hsieh%2CT.%20S&amp;author=Lamothe%2CR&amp;author=Darzacq%2CX&amp;author=Koshland%2CD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Sanborn, A. L. et al. Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes. <i>Proc. Natl. Acad. Sci. USA</i> <b>112</b>, E6456–E6465 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 16" data-doi="10.1073/pnas.1518552112" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.1518552112" data-track-label="10.1073/pnas.1518552112" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.1518552112" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 16" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhslWks7%2FN" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 16" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26499245" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 16" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4664323" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 16" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?37.0701.01" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 16" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromatin%20extrusion%20explains%20key%20features%20of%20loop%20and%20domain%20formation%20in%20wild-type%20and%20engineered%20genomes&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1518552112&amp;volume=112&amp;pages=E6456-E6465&amp;publication_year=2015&amp;author=Sanborn%2CA.%20L" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Vian, L, et al. The energetics and physiological impact of cohesin extrusion. <i>Cell</i> <b>173</b>, 1165–1178.e20 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 17" data-doi="10.1016/j.cell.2018.03.072" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2018.03.072" data-track-label="10.1016/j.cell.2018.03.072" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2018.03.072" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 17" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXosFWhsLY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 17" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29706548" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 17" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6065110" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 17" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0947.81044" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 17" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20energetics%20and%20physiological%20impact%20of%20cohesin%20extrusion&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.03.072&amp;volume=173&amp;pages=1165-1178.e20&amp;publication_year=2018&amp;author=Vian%2CL" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Kakui, Y., Rabinowitz, A., Barry, D. J. &amp; Uhlmann, F. Condensin-mediated remodeling of the mitotic chromatin landscape in fission yeast. <i>Nat. Genet.</i> <b>49</b>, 1553–1557 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 18" data-doi="10.1038/ng.3938" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/ng.3938" data-track-label="10.1038/ng.3938" data-track-value="article reference" href="https://doi.org/10.1038%2Fng.3938" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 18" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlKjtbjM" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 18" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28825727" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 18" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5621628" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 18" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Condensin-mediated%20remodeling%20of%20the%20mitotic%20chromatin%20landscape%20in%20fission%20yeast&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3938&amp;volume=49&amp;pages=1553-1557&amp;publication_year=2017&amp;author=Kakui%2CY&amp;author=Rabinowitz%2CA&amp;author=Barry%2CD.%20J&amp;author=Uhlmann%2CF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Crane, E. et al. Condensin-driven remodelling of X chromosome topology during dosage compensation. <i>Nature</i> <b>523</b>, 240–244 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 19" data-doi="10.1038/nature14450" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature14450" data-track-label="10.1038/nature14450" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature14450" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 19" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015Natur.523..240C" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 19" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFaitrvP" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 19" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26030525" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 19" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4498965" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 19" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0801.57012" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 19" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Condensin-driven%20remodelling%20of%20X%20chromosome%20topology%20during%20dosage%20compensation&amp;journal=Nature&amp;doi=10.1038%2Fnature14450&amp;volume=523&amp;pages=240-244&amp;publication_year=2015&amp;author=Crane%2CE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Le, T. B., Imakaev, M. V., Mirny, L. A. &amp; Laub, M. T. High-resolution mapping of the spatial organization of a bacterial chromosome. <i>Science</i> <b>342</b>, 731–734 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 20" data-doi="10.1126/science.1242059" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.1242059" data-track-label="10.1126/science.1242059" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.1242059" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 20" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013Sci...342..731L" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 20" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslWlsLrP" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 20" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24158908" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 20" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3927313" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 20" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1290.90073" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 20" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=High-resolution%20mapping%20of%20the%20spatial%20organization%20of%20a%20bacterial%20chromosome&amp;journal=Science&amp;doi=10.1126%2Fscience.1242059&amp;volume=342&amp;pages=731-734&amp;publication_year=2013&amp;author=Le%2CT.%20B&amp;author=Imakaev%2CM.%20V&amp;author=Mirny%2CL.%20A&amp;author=Laub%2CM.%20T" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Wang, X. et al. Condensin promotes the juxtaposition of DNA flanking its loading site in <i>Bacillus subtilis</i>. <i>Genes Dev.</i> <b>29</b>, 1661–1675 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 21" data-doi="10.1101/gad.265876.115" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1101/gad.265876.115" data-track-label="10.1101/gad.265876.115" data-track-value="article reference" href="https://doi.org/10.1101%2Fgad.265876.115" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 21" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsVemt7nK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 21" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26253537" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 21" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4536313" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 21" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1336.35236" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 21" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Condensin%20promotes%20the%20juxtaposition%20of%20DNA%20flanking%20its%20loading%20site%20in%20Bacillus%20subtilis&amp;journal=Genes%20Dev.&amp;doi=10.1101%2Fgad.265876.115&amp;volume=29&amp;pages=1661-1675&amp;publication_year=2015&amp;author=Wang%2CX" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Lioy, V. S. et al. Multiscale structuring of the <i>E. coli</i> chromosome by nucleoid-associated and condensin proteins. <i>Cell</i> <b>172</b>, 771–783 e718 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 22" data-doi="10.1016/j.cell.2017.12.027" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2017.12.027" data-track-label="10.1016/j.cell.2017.12.027" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.12.027" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 22" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlOgu78%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 22" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29358050" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 22" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1042.68554" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 22" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiscale%20structuring%20of%20the%20E.%20coli%20chromosome%20by%20nucleoid-associated%20and%20condensin%20proteins&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.12.027&amp;volume=172&amp;pages=771-783%20e718&amp;publication_year=2018&amp;author=Lioy%2CV.%20S" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Britton, R. A., Lin, D. C. &amp; Grossman, A. D. Characterization of a prokaryotic SMC protein involved in chromosome partitioning. <i>Genes Dev.</i> <b>12</b>, 1254–1259 (1998).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 23" data-doi="10.1101/gad.12.9.1254" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1101/gad.12.9.1254" data-track-label="10.1101/gad.12.9.1254" data-track-value="article reference" href="https://doi.org/10.1101%2Fgad.12.9.1254" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 23" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK1cXjtVejsro%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 23" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9573042" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 23" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC316777" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 23" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0909.62111" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 23" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Characterization%20of%20a%20prokaryotic%20SMC%20protein%20involved%20in%20chromosome%20partitioning&amp;journal=Genes%20Dev.&amp;doi=10.1101%2Fgad.12.9.1254&amp;volume=12&amp;pages=1254-1259&amp;publication_year=1998&amp;author=Britton%2CR.%20A&amp;author=Lin%2CD.%20C&amp;author=Grossman%2CA.%20D" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Niki, H., Jaffe, A., Imamura, R., Ogura, T. &amp; Hiraga, S. The new gene <i>mukB</i> codes for a 177 kd protein with coiled-coil domains involved in chromosome partitioning of <i>E. coli</i>. <i>EMBO J.</i> <b>10</b>, 183–193 (1991).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 24" data-doi="10.1002/j.1460-2075.1991.tb07935.x" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/j.1460-2075.1991.tb07935.x" data-track-label="10.1002/j.1460-2075.1991.tb07935.x" data-track-value="article reference" href="https://doi.org/10.1002%2Fj.1460-2075.1991.tb07935.x" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 24" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK3MXksVart78%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 24" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1989883" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 24" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC452628" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 24" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20new%20gene%20mukB%20codes%20for%20a%20177%20kd%20protein%20with%20coiled-coil%20domains%20involved%20in%20chromosome%20partitioning%20of%20E.%20coli&amp;journal=EMBO%20J.&amp;doi=10.1002%2Fj.1460-2075.1991.tb07935.x&amp;volume=10&amp;pages=183-193&amp;publication_year=1991&amp;author=Niki%2CH&amp;author=Jaffe%2CA&amp;author=Imamura%2CR&amp;author=Ogura%2CT&amp;author=Hiraga%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Wang, X., Brandao, H. B., Le, T. B., Laub, M. T. &amp; Rudner, D. Z. <i>Bacillus subtilis</i> SMC complexes juxtapose chromosome arms as they travel from origin to terminus. <i>Science</i> <b>355</b>, 524–527 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 25" data-doi="10.1126/science.aai8982" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.aai8982" data-track-label="10.1126/science.aai8982" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.aai8982" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 25" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017Sci...355..524W" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 25" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsl2qtrg%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 25" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28154080" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 25" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5484144" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 25" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Bacillus%20subtilis%20SMC%20complexes%20juxtapose%20chromosome%20arms%20as%20they%20travel%20from%20origin%20to%20terminus&amp;journal=Science&amp;doi=10.1126%2Fscience.aai8982&amp;volume=355&amp;pages=524-527&amp;publication_year=2017&amp;author=Wang%2CX&amp;author=Brandao%2CH.%20B&amp;author=Le%2CT.%20B&amp;author=Laub%2CM.%20T&amp;author=Rudner%2CD.%20Z" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Makela, J. &amp; Sherratt, D. J. Organization of the <i>Escherichia coli</i> chromosome by a MukBEF axial core. <i>Mol. Cell</i> <b>78</b>, 250–260.e5 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 26" data-doi="10.1016/j.molcel.2020.02.003" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2020.02.003" data-track-label="10.1016/j.molcel.2020.02.003" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2020.02.003" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 26" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXjvVKhtL8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 26" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32097603" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 26" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7163298" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 26" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Organization%20of%20the%20Escherichia%20coli%20chromosome%20by%20a%20MukBEF%20axial%20core&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2020.02.003&amp;volume=78&amp;pages=250-260.e5&amp;publication_year=2020&amp;author=Makela%2CJ&amp;author=Sherratt%2CD.%20J" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Gruber, S. &amp; Errington, J. Recruitment of condensin to replication origin regions by ParB/SpoOJ promotes chromosome segregation in <i>B. subtilis</i>. <i>Cell</i> <b>137</b>, 685–696 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 27" data-doi="10.1016/j.cell.2009.02.035" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2009.02.035" data-track-label="10.1016/j.cell.2009.02.035" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2009.02.035" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 27" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXosVClt78%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 27" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19450516" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 27" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0967.68105" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 27" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Recruitment%20of%20condensin%20to%20replication%20origin%20regions%20by%20ParB%2FSpoOJ%20promotes%20chromosome%20segregation%20in%20B.%20subtilis&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2009.02.035&amp;volume=137&amp;pages=685-696&amp;publication_year=2009&amp;author=Gruber%2CS&amp;author=Errington%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Sullivan, N. L., Marquis, K. A. &amp; Rudner, D. Z. Recruitment of SMC by ParB-parS organizes the origin region and promotes efficient chromosome segregation. <i>Cell</i> <b>137</b>, 697–707 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 28" data-doi="10.1016/j.cell.2009.04.044" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2009.04.044" data-track-label="10.1016/j.cell.2009.04.044" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2009.04.044" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 28" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXosVClt7w%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 28" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19450517" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 28" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2892783" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 28" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0224.94013" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 28" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Recruitment%20of%20SMC%20by%20ParB-parS%20organizes%20the%20origin%20region%20and%20promotes%20efficient%20chromosome%20segregation&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2009.04.044&amp;volume=137&amp;pages=697-707&amp;publication_year=2009&amp;author=Sullivan%2CN.%20L&amp;author=Marquis%2CK.%20A&amp;author=Rudner%2CD.%20Z" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Tran, N. T., Laub, M. T. &amp; Le, T. B. K. SMC Progressively aligns chromosomal arms in <i>Caulobacter crescentus</i> but is antagonized by convergent transcription. <i>Cell Rep.</i> <b>20</b>, 2057–2071 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 29" data-doi="10.1016/j.celrep.2017.08.026" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.celrep.2017.08.026" data-track-label="10.1016/j.celrep.2017.08.026" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.celrep.2017.08.026" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 29" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVSltb3O" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 29" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28854358" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 29" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5583512" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 29" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=SMC%20Progressively%20aligns%20chromosomal%20arms%20in%20Caulobacter%20crescentus%20but%20is%20antagonized%20by%20convergent%20transcription&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2017.08.026&amp;volume=20&amp;pages=2057-2071&amp;publication_year=2017&amp;author=Tran%2CN.%20T&amp;author=Laub%2CM.%20T&amp;author=Le%2CT.%20B.%20K" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Gruber, S. et al. Interlinked sister chromosomes arise in the absence of condensin during fast replication in <i>B. subtilis</i>. <i>Curr. Biol.</i> <b>24</b>, 293–298 (2014).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 30" data-doi="10.1016/j.cub.2013.12.049" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cub.2013.12.049" data-track-label="10.1016/j.cub.2013.12.049" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cub.2013.12.049" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 30" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFGksL8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 30" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24440399" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 30" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3919155" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 30" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Interlinked%20sister%20chromosomes%20arise%20in%20the%20absence%20of%20condensin%20during%20fast%20replication%20in%20B.%20subtilis&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2013.12.049&amp;volume=24&amp;pages=293-298&amp;publication_year=2014&amp;author=Gruber%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Spang, A., Caceres, E. F. &amp; Ettema, T. J. G. Genomic exploration of the diversity, ecology, and evolution of the archaeal domain of life. <i>Science</i> <b>357</b>, eaaf388 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 31" data-doi="10.1126/science.aaf3883" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.aaf3883" data-track-label="10.1126/science.aaf3883" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.aaf3883" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 31" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20exploration%20of%20the%20diversity%2C%20ecology%2C%20and%20evolution%20of%20the%20archaeal%20domain%20of%20life&amp;journal=Science&amp;doi=10.1126%2Fscience.aaf3883&amp;volume=357&amp;publication_year=2017&amp;author=Spang%2CA&amp;author=Caceres%2CE.%20F&amp;author=Ettema%2CT.%20J.%20G" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Imachi, H. et al. Isolation of an archaeon at the prokaryote-eukaryote interface. <i>Nature</i> <b>577</b>, 519–525 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 32" data-doi="10.1038/s41586-019-1916-6" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41586-019-1916-6" data-track-label="10.1038/s41586-019-1916-6" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41586-019-1916-6" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 32" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Natur.577..519I" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 32" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1Sisbg%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 32" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31942073" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 32" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7015854" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 32" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1135.82007" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 32" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Isolation%20of%20an%20archaeon%20at%20the%20prokaryote-eukaryote%20interface&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1916-6&amp;volume=577&amp;pages=519-525&amp;publication_year=2020&amp;author=Imachi%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Jeon, J. H. et al. Evidence for binary Smc complexes lacking kite subunits in archaea. <i>IUCrJ</i> <b>7</b>, 193–206 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 33" data-doi="10.1107/S2052252519016634" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1107/S2052252519016634" data-track-label="10.1107/S2052252519016634" data-track-value="article reference" href="https://doi.org/10.1107%2FS2052252519016634" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 33" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXkt1yjsL4%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 33" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32148848" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 33" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7055376" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 33" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0988.74040" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 33" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Evidence%20for%20binary%20Smc%20complexes%20lacking%20kite%20subunits%20in%20archaea&amp;journal=IUCrJ&amp;doi=10.1107%2FS2052252519016634&amp;volume=7&amp;pages=193-206&amp;publication_year=2020&amp;author=Jeon%2CJ.%20H" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Yoshinaga, M., Nakayama, T. &amp; Inagaki, Y. A novel structural maintenance of chromosomes (SMC)-related protein family specific to Archaea. <i>Front. Microbiol.</i> <b>13</b>, 913088 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 34" data-doi="10.3389/fmicb.2022.913088" data-track="click_references" data-track-action="article reference" data-track-item_id="10.3389/fmicb.2022.913088" data-track-label="10.3389/fmicb.2022.913088" data-track-value="article reference" href="https://doi.org/10.3389%2Ffmicb.2022.913088" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 34" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35992648" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 34" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9389158" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 34" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20novel%20structural%20maintenance%20of%20chromosomes%20%28SMC%29-related%20protein%20family%20specific%20to%20Archaea&amp;journal=Front.%20Microbiol.&amp;doi=10.3389%2Ffmicb.2022.913088&amp;volume=13&amp;publication_year=2022&amp;author=Yoshinaga%2CM&amp;author=Nakayama%2CT&amp;author=Inagaki%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Burmann, F. et al. An asymmetric SMC-kleisin bridge in prokaryotic condensin. <i>Nat. Struct. Mol. Biol.</i> <b>20</b>, 371–379 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 35" data-doi="10.1038/nsmb.2488" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nsmb.2488" data-track-label="10.1038/nsmb.2488" data-track-value="article reference" href="https://doi.org/10.1038%2Fnsmb.2488" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 35" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23353789" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 35" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0222.34057" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 35" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20asymmetric%20SMC-kleisin%20bridge%20in%20prokaryotic%20condensin&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;doi=10.1038%2Fnsmb.2488&amp;volume=20&amp;pages=371-379&amp;publication_year=2013&amp;author=Burmann%2CF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Kamada, K., Miyata, M. &amp; Hirano, T. Molecular basis of SMC ATPase activation: role of internal structural changes of the regulatory subcomplex ScpAB. <i>Structure</i> <b>21</b>, 581–594 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 36" data-doi="10.1016/j.str.2013.02.016" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.str.2013.02.016" data-track-label="10.1016/j.str.2013.02.016" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.str.2013.02.016" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 36" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXltVemsLw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 36" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23541893" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 36" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20basis%20of%20SMC%20ATPase%20activation%3A%20role%20of%20internal%20structural%20changes%20of%20the%20regulatory%20subcomplex%20ScpAB&amp;journal=Structure&amp;doi=10.1016%2Fj.str.2013.02.016&amp;volume=21&amp;pages=581-594&amp;publication_year=2013&amp;author=Kamada%2CK&amp;author=Miyata%2CM&amp;author=Hirano%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Takemata, N. &amp; Bell, S. D. Emerging views of genome organization in Archaea. <i>J. Cell Sci.</i> <b>133</b>, jcs243782 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 37" data-doi="10.1242/jcs.243782" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1242/jcs.243782" data-track-label="10.1242/jcs.243782" data-track-value="article reference" href="https://doi.org/10.1242%2Fjcs.243782" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 37" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1aiur7K" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 37" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32423947" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 37" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7325442" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 37" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Emerging%20views%20of%20genome%20organization%20in%20Archaea&amp;journal=J.%20Cell%20Sci.&amp;doi=10.1242%2Fjcs.243782&amp;volume=133&amp;publication_year=2020&amp;author=Takemata%2CN&amp;author=Bell%2CS.%20D" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Bell, S. D. Form and function of archaeal genomes. <i>Biochem. Soc. Trans.</i> <b>50</b>, 1931–1939 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 38" data-doi="10.1042/BST20221396" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1042/BST20221396" data-track-label="10.1042/BST20221396" data-track-value="article reference" href="https://doi.org/10.1042%2FBST20221396" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 38" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXit1elt7%2FL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 38" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36511238" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 38" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9764264" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 38" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1510.35070" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 38" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Form%20and%20function%20of%20archaeal%20genomes&amp;journal=Biochem.%20Soc.%20Trans.&amp;doi=10.1042%2FBST20221396&amp;volume=50&amp;pages=1931-1939&amp;publication_year=2022&amp;author=Bell%2CS.%20D" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Takemata, N. &amp; Bell, S. D. Multi-scale architecture of archaeal chromosomes. <i>Mol. Cell</i> <b>81</b>, 473–487 e476 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 39" data-doi="10.1016/j.molcel.2020.12.001" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2020.12.001" data-track-label="10.1016/j.molcel.2020.12.001" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2020.12.001" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 39" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXks1CqtA%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 39" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33382983" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 39" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1260.90111" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 39" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Multi-scale%20architecture%20of%20archaeal%20chromosomes&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2020.12.001&amp;volume=81&amp;pages=473-487%20e476&amp;publication_year=2021&amp;author=Takemata%2CN&amp;author=Bell%2CS.%20D" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Takemata, N., Samson, R. Y. &amp; Bell, S. D. Physical and functional compartmentalization of archaeal chromosomes. <i>Cell</i> <b>179</b>, 165–179.e118 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 40" data-doi="10.1016/j.cell.2019.08.036" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2019.08.036" data-track-label="10.1016/j.cell.2019.08.036" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.08.036" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 40" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvVenu7%2FP" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 40" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31539494" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 40" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6756186" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 40" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Physical%20and%20functional%20compartmentalization%20of%20archaeal%20chromosomes&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.08.036&amp;volume=179&amp;pages=165-179.e118&amp;publication_year=2019&amp;author=Takemata%2CN&amp;author=Samson%2CR.%20Y&amp;author=Bell%2CS.%20D" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Cockram, C., Thierry, A., Gorlas, A., Lestini, R. &amp; Koszul, R. Euryarchaeal genomes are folded into SMC-dependent loops and domains, but lack transcription-mediated compartmentalization. <i>Mol. Cell</i> <b>81</b>, 459–472.e410 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 41" data-doi="10.1016/j.molcel.2020.12.013" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2020.12.013" data-track-label="10.1016/j.molcel.2020.12.013" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2020.12.013" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 41" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXks1CqsQ%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 41" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33382984" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 41" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Euryarchaeal%20genomes%20are%20folded%20into%20SMC-dependent%20loops%20and%20domains%2C%20but%20lack%20transcription-mediated%20compartmentalization&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2020.12.013&amp;volume=81&amp;pages=459-472.e410&amp;publication_year=2021&amp;author=Cockram%2CC&amp;author=Thierry%2CA&amp;author=Gorlas%2CA&amp;author=Lestini%2CR&amp;author=Koszul%2CR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Badel C. &amp; Bell S. D. Chromosome architecture in an archaeal species naturally lacking structural maintenance of chromosomes proteins. <i>Nat. Microbiol</i>. <b>9</b>, 263–273 (2023).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Badel, C., Samson, R. Y. &amp; Bell, S. D. Chromosome organization affects genome evolution in <i>Sulfolobus</i> archaea. <i>Nat. Microbiol.</i> <b>7</b>, 820–830 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 43" data-doi="10.1038/s41564-022-01127-7" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41564-022-01127-7" data-track-label="10.1038/s41564-022-01127-7" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41564-022-01127-7" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 43" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XhsVSqu7zF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 43" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35618771" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 43" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9597579" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 43" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromosome%20organization%20affects%20genome%20evolution%20in%20Sulfolobus%20archaea&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-022-01127-7&amp;volume=7&amp;pages=820-830&amp;publication_year=2022&amp;author=Badel%2CC&amp;author=Samson%2CR.%20Y&amp;author=Bell%2CS.%20D" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Sobolev AS, M. et al. 3C-seq-captured chromosome conformation of the hyperthermophilic archaeon <i>Thermofilum adornatum</i>. <i>bioRxiv</i>, (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Pilatowski-Herzing E. et al. Capturing chromosome conformation in Crenarchaea. <i>Mol. Microbiol</i>. <a data-track="click_references" data-track-action="external reference" data-track-label="10.1111/mmi.15245" data-track-value="external reference" href="https://doi.org/10.1111/mmi.15245">https://doi.org/10.1111/mmi.15245</a> (2024).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Akgol Oksuz, B. et al. Systematic evaluation of chromosome conformation capture assays. <i>Nat. Methods</i> <b>18</b>, 1046–1055 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 46" data-doi="10.1038/s41592-021-01248-7" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41592-021-01248-7" data-track-label="10.1038/s41592-021-01248-7" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41592-021-01248-7" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 46" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFGlt73I" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 46" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34480151" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 46" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8446342" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 46" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1441.60068" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 46" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20evaluation%20of%20chromosome%20conformation%20capture%20assays&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-021-01248-7&amp;volume=18&amp;pages=1046-1055&amp;publication_year=2021&amp;author=Akgol%20Oksuz%2CB" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Sato, T., Fukui, T., Atomi, H. &amp; Imanaka, T. Improved and versatile transformation system allowing multiple genetic manipulations of the hyperthermophilic archaeon <i>Thermococcus kodakaraensis</i>. <i>Appl. Environ. Microbiol.</i> <b>71</b>, 3889–3899 (2005).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 47" data-doi="10.1128/AEM.71.7.3889-3899.2005" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1128/AEM.71.7.3889-3899.2005" data-track-label="10.1128/AEM.71.7.3889-3899.2005" data-track-value="article reference" href="https://doi.org/10.1128%2FAEM.71.7.3889-3899.2005" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 47" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2005ApEnM..71.3889S" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 47" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXmt1yltbY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 47" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16000802" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 47" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1169065" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 47" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20and%20versatile%20transformation%20system%20allowing%20multiple%20genetic%20manipulations%20of%20the%20hyperthermophilic%20archaeon%20Thermococcus%20kodakaraensis&amp;journal=Appl.%20Environ.%20Microbiol.&amp;doi=10.1128%2FAEM.71.7.3889-3899.2005&amp;volume=71&amp;pages=3889-3899&amp;publication_year=2005&amp;author=Sato%2CT&amp;author=Fukui%2CT&amp;author=Atomi%2CH&amp;author=Imanaka%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Harewood, L. et al. Hi-C as a tool for precise detection and characterisation of chromosomal rearrangements and copy number variation in human tumours. <i>Genome Biol.</i> <b>18</b>, 125 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 48" data-doi="10.1186/s13059-017-1253-8" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1186/s13059-017-1253-8" data-track-label="10.1186/s13059-017-1253-8" data-track-value="article reference" href="https://link.springer.com/doi/10.1186/s13059-017-1253-8" rel="noopener">Article</a> 
    <a aria-label="PubMed reference 48" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28655341" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 48" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5488307" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 48" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0875.62284" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 48" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Hi-C%20as%20a%20tool%20for%20precise%20detection%20and%20characterisation%20of%20chromosomal%20rearrangements%20and%20copy%20number%20variation%20in%20human%20tumours&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-017-1253-8&amp;volume=18&amp;publication_year=2017&amp;author=Harewood%2CL" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Fukui, T. et al. Complete genome sequence of the hyperthermophilic archaeon <i>Thermococcus kodakaraensis</i> KOD1 and comparison with <i>Pyrococcus</i> genomes. <i>Genome Res.</i> <b>15</b>, 352–363 (2005).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 49" data-doi="10.1101/gr.3003105" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1101/gr.3003105" data-track-label="10.1101/gr.3003105" data-track-value="article reference" href="https://doi.org/10.1101%2Fgr.3003105" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 49" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXitVWks78%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 49" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15710748" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 49" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC551561" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 49" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Complete%20genome%20sequence%20of%20the%20hyperthermophilic%20archaeon%20Thermococcus%20kodakaraensis%20KOD1%20and%20comparison%20with%20Pyrococcus%20genomes&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.3003105&amp;volume=15&amp;pages=352-363&amp;publication_year=2005&amp;author=Fukui%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Spaans, S. K., van der Oost, J. &amp; Kengen, S. W. The chromosome copy number of the hyperthermophilic archaeon <i>Thermococcus kodakarensis</i> KOD1. <i>Extremophiles</i> <b>19</b>, 741–750 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 50" data-doi="10.1007/s00792-015-0750-5" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1007/s00792-015-0750-5" data-track-label="10.1007/s00792-015-0750-5" data-track-value="article reference" href="https://link.springer.com/doi/10.1007/s00792-015-0750-5" rel="noopener">Article</a> 
    <a aria-label="PubMed reference 50" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25952670" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 50" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4502288" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 50" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20chromosome%20copy%20number%20of%20the%20hyperthermophilic%20archaeon%20Thermococcus%20kodakarensis%20KOD1&amp;journal=Extremophiles&amp;doi=10.1007%2Fs00792-015-0750-5&amp;volume=19&amp;pages=741-750&amp;publication_year=2015&amp;author=Spaans%2CS.%20K&amp;author=Oost%2CJ&amp;author=Kengen%2CS.%20W" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Chen, F., Li, G., Zhang, M. Q. &amp; Chen, Y. HiCDB: a sensitive and robust method for detecting contact domain boundaries. <i>Nucleic Acids Res.</i> <b>46</b>, 11239–11250 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 51" data-doi="10.1093/nar/gky789" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gky789" data-track-label="10.1093/nar/gky789" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgky789" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 51" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXovFWnsb4%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 51" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30184171" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 51" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6265446" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 51" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?07936768" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 51" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=HiCDB%3A%20a%20sensitive%20and%20robust%20method%20for%20detecting%20contact%20domain%20boundaries&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgky789&amp;volume=46&amp;pages=11239-11250&amp;publication_year=2018&amp;author=Chen%2CF&amp;author=Li%2CG&amp;author=Zhang%2CM.%20Q&amp;author=Chen%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Bignaud, A. et al. Transcription-induced domains form the elementary constraining building blocks of bacterial chromosomes. <i>Nat. Struct. Mol. Biol.</i> <b>31</b>, 489–497 (2024).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 52" data-doi="10.1038/s41594-023-01178-2" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41594-023-01178-2" data-track-label="10.1038/s41594-023-01178-2" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41594-023-01178-2" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 52" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB2cXmtFeqsw%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 52" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=38177686" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 52" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10948358" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 52" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcription-induced%20domains%20form%20the%20elementary%20constraining%20building%20blocks%20of%20bacterial%20chromosomes&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;doi=10.1038%2Fs41594-023-01178-2&amp;volume=31&amp;pages=489-497&amp;publication_year=2024&amp;author=Bignaud%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Matthey-Doret, C. et al. Computer vision for pattern detection in chromosome contact maps. <i>Nat. Commun.</i> <b>11</b>, 5795 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 53" data-doi="10.1038/s41467-020-19562-7" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-020-19562-7" data-track-label="10.1038/s41467-020-19562-7" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-020-19562-7" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 53" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatCo..11.5795M" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 53" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlOrtLrL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 53" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33199682" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 53" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7670471" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 53" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1543.94171" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 53" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Computer%20vision%20for%20pattern%20detection%20in%20chromosome%20contact%20maps&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-19562-7&amp;volume=11&amp;publication_year=2020&amp;author=Matthey-Doret%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Long, S. W. &amp; Faguy, D. M. Anucleate and titan cell phenotypes caused by insertional inactivation of the structural maintenance of chromosomes (<i>SMC</i>) gene in the archaeon <i>Methanococcus voltae</i>. <i>Mol. Microbiol.</i> <b>52</b>, 1567–1577 (2004).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 54" data-doi="10.1111/j.1365-2958.2004.04084.x" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1111/j.1365-2958.2004.04084.x" data-track-label="10.1111/j.1365-2958.2004.04084.x" data-track-value="article reference" href="https://doi.org/10.1111%2Fj.1365-2958.2004.04084.x" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 54" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXlsVOgu7s%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 54" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15186409" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 54" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Anucleate%20and%20titan%20cell%20phenotypes%20caused%20by%20insertional%20inactivation%20of%20the%20structural%20maintenance%20of%20chromosomes%20%28SMC%29%20gene%20in%20the%20archaeon%20Methanococcus%20voltae&amp;journal=Mol.%20Microbiol.&amp;doi=10.1111%2Fj.1365-2958.2004.04084.x&amp;volume=52&amp;pages=1567-1577&amp;publication_year=2004&amp;author=Long%2CS.%20W&amp;author=Faguy%2CD.%20M" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Cortez, D. et al. Evidence for a Xer/dif system for chromosome resolution in archaea. <i>PLoS Genet.</i> <b>6</b>, e1001166 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 55" data-doi="10.1371/journal.pgen.1001166" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1371/journal.pgen.1001166" data-track-label="10.1371/journal.pgen.1001166" data-track-value="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1001166" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 55" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20975945" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 55" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2958812" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 55" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1357.82055" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 55" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Evidence%20for%20a%20Xer%2Fdif%20system%20for%20chromosome%20resolution%20in%20archaea&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1001166&amp;volume=6&amp;publication_year=2010&amp;author=Cortez%2CD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Cossu, M., Da Cunha, V., Toffano-Nioche, C., Forterre, P. &amp; Oberto, J. Comparative genomics reveals conserved positioning of essential genomic clusters in highly rearranged Thermococcales chromosomes. <i>Biochimie</i> <b>118</b>, 313–321 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 56" data-doi="10.1016/j.biochi.2015.07.008" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.biochi.2015.07.008" data-track-label="10.1016/j.biochi.2015.07.008" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.biochi.2015.07.008" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 56" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFOlu7jJ" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 56" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26166067" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 56" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4640148" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 56" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparative%20genomics%20reveals%20conserved%20positioning%20of%20essential%20genomic%20clusters%20in%20highly%20rearranged%20Thermococcales%20chromosomes&amp;journal=Biochimie&amp;doi=10.1016%2Fj.biochi.2015.07.008&amp;volume=118&amp;pages=313-321&amp;publication_year=2015&amp;author=Cossu%2CM&amp;author=Cunha%2CV&amp;author=Toffano-Nioche%2CC&amp;author=Forterre%2CP&amp;author=Oberto%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Duggin, I. G., Dubarry, N. &amp; Bell, S. D. Replication termination and chromosome dimer resolution in the archaeon <i>Sulfolobus solfataricus</i>. <i>EMBO J.</i> <b>30</b>, 145–153 (2011).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 57" data-doi="10.1038/emboj.2010.301" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/emboj.2010.301" data-track-label="10.1038/emboj.2010.301" data-track-value="article reference" href="https://doi.org/10.1038%2Femboj.2010.301" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 57" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsVOrsLfI" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 57" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21113132" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 57" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1375.60115" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 57" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Replication%20termination%20and%20chromosome%20dimer%20resolution%20in%20the%20archaeon%20Sulfolobus%20solfataricus&amp;journal=EMBO%20J.&amp;doi=10.1038%2Femboj.2010.301&amp;volume=30&amp;pages=145-153&amp;publication_year=2011&amp;author=Duggin%2CI.%20G&amp;author=Dubarry%2CN&amp;author=Bell%2CS.%20D" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Midonet C., Barre F. X. Xer site-specific recombination: promoting vertical and horizontal transmission of genetic information. <i>Microbiol Spectr</i> <b>2</b>, <a data-track="click_references" data-track-action="external reference" data-track-label="10.1128/microbiolspec.MDNA3-0056-2014" data-track-value="external reference" href="https://doi.org/10.1128/microbiolspec.MDNA3-0056-2014">https://doi.org/10.1128/microbiolspec.MDNA3-0056-2014</a> (2014).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Jo, M., Murayama, Y., Tsutsui, Y. &amp; Iwasaki, H. In vitro site-specific recombination mediated by the tyrosine recombinase XerA of <i>Thermoplasma acidophilum</i>. <i>Genes Cells</i> <b>22</b>, 646–661 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 59" data-doi="10.1111/gtc.12503" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1111/gtc.12503" data-track-label="10.1111/gtc.12503" data-track-value="article reference" href="https://doi.org/10.1111%2Fgtc.12503" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 59" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXos1Srtro%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 59" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28557347" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 59" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1152.15305" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 59" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=In%20vitro%20site-specific%20recombination%20mediated%20by%20the%20tyrosine%20recombinase%20XerA%20of%20Thermoplasma%20acidophilum&amp;journal=Genes%20Cells&amp;doi=10.1111%2Fgtc.12503&amp;volume=22&amp;pages=646-661&amp;publication_year=2017&amp;author=Jo%2CM&amp;author=Murayama%2CY&amp;author=Tsutsui%2CY&amp;author=Iwasaki%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Karaboja, X. et al. XerD unloads bacterial SMC complexes at the replication terminus. <i>Mol. Cell</i> <b>81</b>, 756–766 e758 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 60" data-doi="10.1016/j.molcel.2020.12.027" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2020.12.027" data-track-label="10.1016/j.molcel.2020.12.027" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2020.12.027" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 60" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhs1ymt7Y%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 60" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33472056" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 60" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7897262" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 60" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0206.22101" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 60" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=XerD%20unloads%20bacterial%20SMC%20complexes%20at%20the%20replication%20terminus&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2020.12.027&amp;volume=81&amp;pages=756-766%20e758&amp;publication_year=2021&amp;author=Karaboja%2CX" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Maruyama, H. et al. Histone and TK0471/TrmBL2 form a novel heterogeneous genome architecture in the hyperthermophilic archaeon <i>Thermococcus kodakarensis</i>. <i>Mol. Biol. Cell</i> <b>22</b>, 386–398 (2011).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 61" data-doi="10.1091/mbc.e10-08-0668" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1091/mbc.e10-08-0668" data-track-label="10.1091/mbc.e10-08-0668" data-track-value="article reference" href="https://doi.org/10.1091%2Fmbc.e10-08-0668" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 61" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXitVChsbY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 61" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21148291" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 61" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3031468" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 61" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0498.90011" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 61" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Histone%20and%20TK0471%2FTrmBL2%20form%20a%20novel%20heterogeneous%20genome%20architecture%20in%20the%20hyperthermophilic%20archaeon%20Thermococcus%20kodakarensis&amp;journal=Mol.%20Biol.%20Cell&amp;doi=10.1091%2Fmbc.e10-08-0668&amp;volume=22&amp;pages=386-398&amp;publication_year=2011&amp;author=Maruyama%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Efremov, A. K. et al. Transcriptional repressor TrmBL2 from <i>Thermococcus kodakarensis</i> forms filamentous nucleoprotein structures and competes with histones for DNA binding in a salt- and DNA supercoiling-dependent manner. <i>J. Biol. Chem.</i> <b>290</b>, 15770–15784 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 62" data-doi="10.1074/jbc.M114.626705" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1074/jbc.M114.626705" data-track-label="10.1074/jbc.M114.626705" data-track-value="article reference" href="https://doi.org/10.1074%2Fjbc.M114.626705" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 62" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtVKhsLrF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 62" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25931116" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 62" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4505486" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 62" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcriptional%20repressor%20TrmBL2%20from%20Thermococcus%20kodakarensis%20forms%20filamentous%20nucleoprotein%20structures%20and%20competes%20with%20histones%20for%20DNA%20binding%20in%20a%20salt-%20and%20DNA%20supercoiling-dependent%20manner&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M114.626705&amp;volume=290&amp;pages=15770-15784&amp;publication_year=2015&amp;author=Efremov%2CA.%20K" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Li, Y. et al. The structural basis for cohesin-CTCF-anchored loops. <i>Nature</i> <b>578</b>, 472–476 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 63" data-doi="10.1038/s41586-019-1910-z" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41586-019-1910-z" data-track-label="10.1038/s41586-019-1910-z" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41586-019-1910-z" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 63" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Natur.578..472L" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 63" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXislCnsLY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 63" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31905366" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 63" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7035113" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 63" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1451.65200" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 63" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20structural%20basis%20for%20cohesin-CTCF-anchored%20loops&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1910-z&amp;volume=578&amp;pages=472-476&amp;publication_year=2020&amp;author=Li%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Ahmad, M. U. D. et al. Structural insights into nonspecific binding of DNA by TrmBL2, an archaeal chromatin protein. <i>J. Mol. Biol.</i> <b>427</b>, 3216–3229 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 64" data-doi="10.1016/j.jmb.2015.08.012" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.jmb.2015.08.012" data-track-label="10.1016/j.jmb.2015.08.012" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.jmb.2015.08.012" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 64" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26299937" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 64" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1368.62048" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 64" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20insights%20into%20nonspecific%20binding%20of%20DNA%20by%20TrmBL2%2C%20an%20archaeal%20chromatin%20protein&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2Fj.jmb.2015.08.012&amp;volume=427&amp;pages=3216-3229&amp;publication_year=2015&amp;author=Ahmad%2CM.%20U.%20D" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Haran, T. E. &amp; Mohanty, U. The unique structure of A-tracts and intrinsic DNA bending. <i>Q. Rev. Biophys.</i> <b>42</b>, 41–81 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 65" data-doi="10.1017/S0033583509004752" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1017/S0033583509004752" data-track-label="10.1017/S0033583509004752" data-track-value="article reference" href="https://doi.org/10.1017%2FS0033583509004752" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 65" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXmvF2htbk%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 65" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19508739" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 65" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1182.94066" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 65" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20unique%20structure%20of%20A-tracts%20and%20intrinsic%20DNA%20bending&amp;journal=Q.%20Rev.%20Biophys.&amp;doi=10.1017%2FS0033583509004752&amp;volume=42&amp;pages=41-81&amp;publication_year=2009&amp;author=Haran%2CT.%20E&amp;author=Mohanty%2CU" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Hagerman, P. J. Sequence-directed curvature of DNA. <i>Annu. Rev. Biochem.</i> <b>59</b>, 755–781 (1990).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 66" data-doi="10.1146/annurev.bi.59.070190.003543" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1146/annurev.bi.59.070190.003543" data-track-label="10.1146/annurev.bi.59.070190.003543" data-track-value="article reference" href="https://doi.org/10.1146%2Fannurev.bi.59.070190.003543" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 66" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK3cXls1Ontrs%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 66" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2197990" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 66" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1119.40300" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 66" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Sequence-directed%20curvature%20of%20DNA&amp;journal=Annu.%20Rev.%20Biochem.&amp;doi=10.1146%2Fannurev.bi.59.070190.003543&amp;volume=59&amp;pages=755-781&amp;publication_year=1990&amp;author=Hagerman%2CP.%20J" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Marin-Gonzalez, A., Vilhena, J. G., Perez, R. &amp; Moreno-Herrero, F. A molecular view of DNA flexibility. <i>Q Rev. Biophys.</i> <b>54</b>, e8 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 67" data-doi="10.1017/S0033583521000068" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1017/S0033583521000068" data-track-label="10.1017/S0033583521000068" data-track-value="article reference" href="https://doi.org/10.1017%2FS0033583521000068" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 67" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFygtL%2FJ" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 67" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34225835" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 67" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1458.05250" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 67" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20molecular%20view%20of%20DNA%20flexibility&amp;journal=Q%20Rev.%20Biophys.&amp;doi=10.1017%2FS0033583521000068&amp;volume=54&amp;publication_year=2021&amp;author=Marin-Gonzalez%2CA&amp;author=Vilhena%2CJ.%20G&amp;author=Perez%2CR&amp;author=Moreno-Herrero%2CF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Segal, E. &amp; Widom, J. Poly(dA:dT) tracts: major determinants of nucleosome organization. <i>Curr. Opin. Struct. Biol.</i> <b>19</b>, 65–71 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 68" data-doi="10.1016/j.sbi.2009.01.004" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.sbi.2009.01.004" data-track-label="10.1016/j.sbi.2009.01.004" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.sbi.2009.01.004" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 68" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXit1Omsbw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 68" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19208466" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 68" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2673466" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 68" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0592.35112" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 68" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Poly%28dA%3AdT%29%20tracts%3A%20major%20determinants%20of%20nucleosome%20organization&amp;journal=Curr.%20Opin.%20Struct.%20Biol.&amp;doi=10.1016%2Fj.sbi.2009.01.004&amp;volume=19&amp;pages=65-71&amp;publication_year=2009&amp;author=Segal%2CE&amp;author=Widom%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Marin-Gonzalez, A. et al. Understanding the paradoxical mechanical response of in-phase A-tracts at different force regimes. <i>Nucleic Acids Res.</i> <b>48</b>, 5024–5036 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 69" data-doi="10.1093/nar/gkaa225" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkaa225" data-track-label="10.1093/nar/gkaa225" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkaa225" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 69" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVKlurbK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 69" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32282908" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 69" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7229863" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 69" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?07962883" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 69" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Understanding%20the%20paradoxical%20mechanical%20response%20of%20in-phase%20A-tracts%20at%20different%20force%20regimes&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkaa225&amp;volume=48&amp;pages=5024-5036&amp;publication_year=2020&amp;author=Marin-Gonzalez%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Rohs, R. et al. The role of DNA shape in protein-DNA recognition. <i>Nature</i> <b>461</b>, 1248–1253 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 70" data-doi="10.1038/nature08473" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature08473" data-track-label="10.1038/nature08473" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature08473" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 70" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2009Natur.461.1248R" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 70" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtlOjtrvL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 70" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19865164" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 70" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2793086" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 70" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1284.62791" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 70" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20role%20of%20DNA%20shape%20in%20protein-DNA%20recognition&amp;journal=Nature&amp;doi=10.1038%2Fnature08473&amp;volume=461&amp;pages=1248-1253&amp;publication_year=2009&amp;author=Rohs%2CR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Olson W. K. <i>DNA Bending &amp; Curvature</i> (Adenine Press, 1988).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Mitchell, J. S., Glowacki, J., Grandchamp, A. E., Manning, R. S. &amp; Maddocks, J. H. Sequence-dependent persistence lengths of DNA. <i>J. Chem. Theory Comput.</i> <b>13</b>, 1539–1555 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 72" data-doi="10.1021/acs.jctc.6b00904" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jctc.6b00904" data-track-label="10.1021/acs.jctc.6b00904" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jctc.6b00904" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 72" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XitFOitbnL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 72" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28029797" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 72" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1370.43001" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 72" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Sequence-dependent%20persistence%20lengths%20of%20DNA&amp;journal=J.%20Chem.%20Theory%20Comput.&amp;doi=10.1021%2Facs.jctc.6b00904&amp;volume=13&amp;pages=1539-1555&amp;publication_year=2017&amp;author=Mitchell%2CJ.%20S&amp;author=Glowacki%2CJ&amp;author=Grandchamp%2CA.%20E&amp;author=Manning%2CR.%20S&amp;author=Maddocks%2CJ.%20H" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Velasco-Berrelleza, V. et al. SerraNA: a program to determine nucleic acids elasticity from simulation data. <i>Phys. Chem. Chem. Phys.</i> <b>22</b>, 19254–19266 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 73" data-doi="10.1039/D0CP02713H" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1039/D0CP02713H" data-track-label="10.1039/D0CP02713H" data-track-value="article reference" href="https://doi.org/10.1039%2FD0CP02713H" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 73" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1SgsL3P" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 73" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32902529" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 73" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1160.65353" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 73" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=SerraNA%3A%20a%20program%20to%20determine%20nucleic%20acids%20elasticity%20from%20simulation%20data&amp;journal=Phys.%20Chem.%20Chem.%20Phys.&amp;doi=10.1039%2FD0CP02713H&amp;volume=22&amp;pages=19254-19266&amp;publication_year=2020&amp;author=Velasco-Berrelleza%2CV" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Li, J., Chiu, T. P. &amp; Rohs, R. Predicting DNA structure using a deep learning method. <i>Nat. Commun.</i> <b>15</b>, 1243 (2024).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 74" data-doi="10.1038/s41467-024-45191-5" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-024-45191-5" data-track-label="10.1038/s41467-024-45191-5" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-024-45191-5" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 74" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2024NatCo..15.1243L" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 74" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB2cXksVWisro%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 74" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=38336958" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 74" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10858265" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 74" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1178.76290" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 74" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Predicting%20DNA%20structure%20using%20a%20deep%20learning%20method&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-024-45191-5&amp;volume=15&amp;publication_year=2024&amp;author=Li%2CJ&amp;author=Chiu%2CT.%20P&amp;author=Rohs%2CR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Dekker, C., Haering, C. H., Peters, J. M. &amp; Rowland, B. D. How do molecular motors fold the genome? <i>Science</i> <b>382</b>, 646–648 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 75" data-doi="10.1126/science.adi8308" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.adi8308" data-track-label="10.1126/science.adi8308" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.adi8308" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 75" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2023Sci...382..646D" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 75" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXitlKgtbvM" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 75" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37943927" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 75" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1492.94089" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 75" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=How%20do%20molecular%20motors%20fold%20the%20genome%3F&amp;journal=Science&amp;doi=10.1126%2Fscience.adi8308&amp;volume=382&amp;pages=646-648&amp;publication_year=2023&amp;author=Dekker%2CC&amp;author=Haering%2CC.%20H&amp;author=Peters%2CJ.%20M&amp;author=Rowland%2CB.%20D" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Marko, J. F., De Los Rios, P., Barducci, A. &amp; Gruber, S. DNA-segment-capture model for loop extrusion by structural maintenance of chromosome (SMC) protein complexes. <i>Nucleic Acids Res.</i> <b>47</b>, 6956–6972 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 76" data-doi="10.1093/nar/gkz497" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkz497" data-track-label="10.1093/nar/gkz497" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkz497" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 76" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXktVSmt70%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 76" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31175837" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 76" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6649773" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 76" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA-segment-capture%20model%20for%20loop%20extrusion%20by%20structural%20maintenance%20of%20chromosome%20%28SMC%29%20protein%20complexes&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkz497&amp;volume=47&amp;pages=6956-6972&amp;publication_year=2019&amp;author=Marko%2CJ.%20F&amp;author=Los%20Rios%2CP&amp;author=Barducci%2CA&amp;author=Gruber%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Strick, T. R., Kawaguchi, T. &amp; Hirano, T. Real-time detection of single-molecule DNA compaction by condensin I. <i>Curr. Biol.</i> <b>14</b>, 874–880 (2004).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 77" data-doi="10.1016/j.cub.2004.04.038" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cub.2004.04.038" data-track-label="10.1016/j.cub.2004.04.038" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cub.2004.04.038" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 77" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXkvV2gtLY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 77" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15186743" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 77" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1091.68096" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 77" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Real-time%20detection%20of%20single-molecule%20DNA%20compaction%20by%20condensin%20I&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2004.04.038&amp;volume=14&amp;pages=874-880&amp;publication_year=2004&amp;author=Strick%2CT.%20R&amp;author=Kawaguchi%2CT&amp;author=Hirano%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Ryu, J. K. et al. Condensin extrudes DNA loops in steps up to hundreds of base pairs that are generated by ATP binding events. <i>Nucleic Acids Res.</i> <b>50</b>, 820–832 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 78" data-doi="10.1093/nar/gkab1268" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkab1268" data-track-label="10.1093/nar/gkab1268" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkab1268" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 78" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XktV2ju70%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 78" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34951453" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 78" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1545.33017" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 78" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Condensin%20extrudes%20DNA%20loops%20in%20steps%20up%20to%20hundreds%20of%20base%20pairs%20that%20are%20generated%20by%20ATP%20binding%20events&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkab1268&amp;volume=50&amp;pages=820-832&amp;publication_year=2022&amp;author=Ryu%2CJ.%20K" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Cui, Y., Petrushenko, Z. M. &amp; Rybenkov, V. V. MukB acts as a macromolecular clamp in DNA condensation. <i>Nat. Struct. Mol. Biol.</i> <b>15</b>, 411–418 (2008).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 79" data-doi="10.1038/nsmb.1410" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nsmb.1410" data-track-label="10.1038/nsmb.1410" data-track-value="article reference" href="https://doi.org/10.1038%2Fnsmb.1410" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 79" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXktFyjtb0%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 79" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18376412" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 79" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=MukB%20acts%20as%20a%20macromolecular%20clamp%20in%20DNA%20condensation&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;doi=10.1038%2Fnsmb.1410&amp;volume=15&amp;pages=411-418&amp;publication_year=2008&amp;author=Cui%2CY&amp;author=Petrushenko%2CZ.%20M&amp;author=Rybenkov%2CV.%20V" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80."><p class="c-article-references__text" id="ref-CR80">Sun, M., Nishino, T. &amp; Marko, J. F. The SMC1-SMC3 cohesin heterodimer structures DNA through supercoiling-dependent loop formation. <i>Nucleic Acids Res.</i> <b>41</b>, 6149–6160 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 80" data-doi="10.1093/nar/gkt303" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkt303" data-track-label="10.1093/nar/gkt303" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkt303" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 80" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtValsbrM" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 80" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23620281" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 80" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3695518" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 80" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1394.94890" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 80" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20SMC1-SMC3%20cohesin%20heterodimer%20structures%20DNA%20through%20supercoiling-dependent%20loop%20formation&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkt303&amp;volume=41&amp;pages=6149-6160&amp;publication_year=2013&amp;author=Sun%2CM&amp;author=Nishino%2CT&amp;author=Marko%2CJ.%20F" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81."><p class="c-article-references__text" id="ref-CR81">Freeman, G. S., Hinckley, D. M., Lequieu, J. P., Whitmer, J. K. &amp; de Pablo, J. J. Coarse-grained modeling of DNA curvature. <i>J. Chem. Phys.</i> <b>141</b>, 165103 (2014).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 81" data-doi="10.1063/1.4897649" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1063/1.4897649" data-track-label="10.1063/1.4897649" data-track-value="article reference" href="https://doi.org/10.1063%2F1.4897649" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 81" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2014JChPh.141p5103F" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 81" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25362344" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 81" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Coarse-grained%20modeling%20of%20DNA%20curvature&amp;journal=J.%20Chem.%20Phys.&amp;doi=10.1063%2F1.4897649&amp;volume=141&amp;publication_year=2014&amp;author=Freeman%2CG.%20S&amp;author=Hinckley%2CD.%20M&amp;author=Lequieu%2CJ.%20P&amp;author=Whitmer%2CJ.%20K&amp;author=Pablo%2CJ.%20J" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="82."><p class="c-article-references__text" id="ref-CR82">Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <i>Nature</i> <b>596</b>, 583–589 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 82" data-doi="10.1038/s41586-021-03819-2" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41586-021-03819-2" data-track-label="10.1038/s41586-021-03819-2" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41586-021-03819-2" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 82" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021Natur.596..583J" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 82" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVaktrrL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 82" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34265844" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 82" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8371605" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 82" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?52.0677.08" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 82" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-03819-2&amp;volume=596&amp;pages=583-589&amp;publication_year=2021&amp;author=Jumper%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="83."><p class="c-article-references__text" id="ref-CR83">Banigan, E. J., van den Berg, A. A., Brandao, H. B., Marko, J. F. &amp; Mirny, L. A. Chromosome organization by one-sided and two-sided loop extrusion. <i>Elife</i> <b>9</b>, e53558 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 83" data-doi="10.7554/eLife.53558" data-track="click_references" data-track-action="article reference" data-track-item_id="10.7554/eLife.53558" data-track-label="10.7554/eLife.53558" data-track-value="article reference" href="https://doi.org/10.7554%2FeLife.53558" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 83" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32250245" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 83" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7295573" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 83" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromosome%20organization%20by%20one-sided%20and%20two-sided%20loop%20extrusion&amp;journal=Elife&amp;doi=10.7554%2FeLife.53558&amp;volume=9&amp;publication_year=2020&amp;author=Banigan%2CE.%20J&amp;author=Berg%2CA.%20A&amp;author=Brandao%2CH.%20B&amp;author=Marko%2CJ.%20F&amp;author=Mirny%2CL.%20A" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="84."><p class="c-article-references__text" id="ref-CR84">Pradhan, B. et al. The Smc5/6 complex is a DNA loop-extruding motor. <i>Nature</i> <b>616</b>, 843–848 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 84" data-doi="10.1038/s41586-023-05963-3" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41586-023-05963-3" data-track-label="10.1038/s41586-023-05963-3" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41586-023-05963-3" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 84" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2023Natur.616..843P" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 84" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXotFKktbY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 84" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37076626" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 84" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10132971" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 84" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1517.74015" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 84" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Smc5%2F6%20complex%20is%20a%20DNA%20loop-extruding%20motor&amp;journal=Nature&amp;doi=10.1038%2Fs41586-023-05963-3&amp;volume=616&amp;pages=843-848&amp;publication_year=2023&amp;author=Pradhan%2CB" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="85."><p class="c-article-references__text" id="ref-CR85">Pradhan, B. et al. Loop extrusion-mediated plasmid DNA cleavage by the bacterial SMC Wadjet complex. <i>Mol. Cell</i> <b>85</b>, 107–116 (2024).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="86."><p class="c-article-references__text" id="ref-CR86">Jalal A. S., Tran N. T. &amp; Le T. B. ParB spreading on DNA requires cytidine triphosphate in vitro. <i>Elife</i> <b>9</b>, e53515 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="87."><p class="c-article-references__text" id="ref-CR87">Soh, Y. M. et al. Self-organization of parS centromeres by the ParB CTP hydrolase. <i>Science</i> <b>366</b>, 1129–1133 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 87" data-doi="10.1126/science.aay3965" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.aay3965" data-track-label="10.1126/science.aay3965" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.aay3965" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 87" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019Sci...366.1129S" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 87" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlWisr%2FF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 87" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31649139" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 87" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6927813" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 87" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1522.00131" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 87" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Self-organization%20of%20parS%20centromeres%20by%20the%20ParB%20CTP%20hydrolase&amp;journal=Science&amp;doi=10.1126%2Fscience.aay3965&amp;volume=366&amp;pages=1129-1133&amp;publication_year=2019&amp;author=Soh%2CY.%20M" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="88."><p class="c-article-references__text" id="ref-CR88">Osorio-Valeriano, M. et al. ParB-type DNA segregation proteins Are CTP-dependent molecular switches. <i>Cell</i> <b>179</b>, 1512–1524.e1515 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 88" data-doi="10.1016/j.cell.2019.11.015" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2019.11.015" data-track-label="10.1016/j.cell.2019.11.015" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.11.015" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 88" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXlt1eitLk%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 88" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31835030" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 88" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=ParB-type%20DNA%20segregation%20proteins%20Are%20CTP-dependent%20molecular%20switches&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.11.015&amp;volume=179&amp;pages=1512-1524.e1515&amp;publication_year=2019&amp;author=Osorio-Valeriano%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="89."><p class="c-article-references__text" id="ref-CR89">Makino, Y. et al. An archaeal ADP-dependent serine kinase involved in cysteine biosynthesis and serine metabolism. <i>Nat. Commun.</i> <b>7</b>, 13446 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 89" data-doi="10.1038/ncomms13446" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/ncomms13446" data-track-label="10.1038/ncomms13446" data-track-value="article reference" href="https://doi.org/10.1038%2Fncomms13446" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 89" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2016NatCo...713446M" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 89" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXjtVCmsro%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 89" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27857065" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 89" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5120207" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 89" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1520.14080" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 89" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20archaeal%20ADP-dependent%20serine%20kinase%20involved%20in%20cysteine%20biosynthesis%20and%20serine%20metabolism&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms13446&amp;volume=7&amp;publication_year=2016&amp;author=Makino%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="90."><p class="c-article-references__text" id="ref-CR90">Mercier, R. et al. The MatP/<i>matS</i> site-specific system organizes the terminus region of the <i>E. coli</i> chromosome into a macrodomain. <i>Cell</i> <b>135</b>, 475–485 (2008).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 90" data-doi="10.1016/j.cell.2008.08.031" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2008.08.031" data-track-label="10.1016/j.cell.2008.08.031" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2008.08.031" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 90" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlyqtrzE" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 90" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18984159" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 90" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20MatP%2FmatS%20site-specific%20system%20organizes%20the%20terminus%20region%20of%20the%20E.%20coli%20chromosome%20into%20a%20macrodomain&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2008.08.031&amp;volume=135&amp;pages=475-485&amp;publication_year=2008&amp;author=Mercier%2CR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="91."><p class="c-article-references__text" id="ref-CR91">Nolivos, S. et al. MatP regulates the coordinated action of topoisomerase IV and MukBEF in chromosome segregation. <i>Nat. Commun.</i> <b>7</b>, 10466 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 91" data-doi="10.1038/ncomms10466" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/ncomms10466" data-track-label="10.1038/ncomms10466" data-track-value="article reference" href="https://doi.org/10.1038%2Fncomms10466" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 91" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2016NatCo...710466N" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 91" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhslOntbY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 91" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26818444" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 91" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4738335" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 91" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1393.74036" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 91" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=MatP%20regulates%20the%20coordinated%20action%20of%20topoisomerase%20IV%20and%20MukBEF%20in%20chromosome%20segregation&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms10466&amp;volume=7&amp;publication_year=2016&amp;author=Nolivos%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="92."><p class="c-article-references__text" id="ref-CR92">Herrmann, U. &amp; Soppa, J. Cell cycle-dependent expression of an essential SMC-like protein and dynamic chromosome localization in the archaeon <i>Halobacterium salinarum</i>. <i>Mol. Microbiol.</i> <b>46</b>, 395–409 (2002).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 92" data-doi="10.1046/j.1365-2958.2002.03181.x" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1046/j.1365-2958.2002.03181.x" data-track-label="10.1046/j.1365-2958.2002.03181.x" data-track-value="article reference" href="https://doi.org/10.1046%2Fj.1365-2958.2002.03181.x" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 92" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38XosVarsbY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 92" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12406217" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 92" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1399.06023" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 92" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cell%20cycle-dependent%20expression%20of%20an%20essential%20SMC-like%20protein%20and%20dynamic%20chromosome%20localization%20in%20the%20archaeon%20Halobacterium%20salinarum&amp;journal=Mol.%20Microbiol.&amp;doi=10.1046%2Fj.1365-2958.2002.03181.x&amp;volume=46&amp;pages=395-409&amp;publication_year=2002&amp;author=Herrmann%2CU&amp;author=Soppa%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="93."><p class="c-article-references__text" id="ref-CR93">Conin, B. et al. Extended sister-chromosome catenation leads to massive reorganization of the <i>E. coli</i> genome. <i>Nucleic Acids Res.</i> <b>50</b>, 2635–2650 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 93" data-doi="10.1093/nar/gkac105" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkac105" data-track-label="10.1093/nar/gkac105" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkac105" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 93" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38Xhs1KksL3J" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 93" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35212387" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 93" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8934667" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 93" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1506.74159" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 93" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Extended%20sister-chromosome%20catenation%20leads%20to%20massive%20reorganization%20of%20the%20E.%20coli%20genome&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkac105&amp;volume=50&amp;pages=2635-2650&amp;publication_year=2022&amp;author=Conin%2CB" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="94."><p class="c-article-references__text" id="ref-CR94">El Sayyed, H. et al. Mapping topoisomerase IV binding and activity sites on the <i>E. coli</i> genome. <i>PLoS Genet.</i> <b>12</b>, e1006025 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 94" data-doi="10.1371/journal.pgen.1006025" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1371/journal.pgen.1006025" data-track-label="10.1371/journal.pgen.1006025" data-track-value="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1006025" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 94" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27171414" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 94" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4865107" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 94" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Mapping%20topoisomerase%20IV%20binding%20and%20activity%20sites%20on%20the%20E.%20coli%20genome&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1006025&amp;volume=12&amp;publication_year=2016&amp;author=El%20Sayyed%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="95."><p class="c-article-references__text" id="ref-CR95">Sutormin, D., Galivondzhyan, A., Gafurov, A. &amp; Severinov, K. Single-nucleotide resolution detection of Topo IV cleavage activity in the <i>Escherichia coli</i> genome with Topo-Seq. <i>Front. Microbiol.</i> <b>14</b>, 1160736 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 95" data-doi="10.3389/fmicb.2023.1160736" data-track="click_references" data-track-action="article reference" data-track-item_id="10.3389/fmicb.2023.1160736" data-track-label="10.3389/fmicb.2023.1160736" data-track-value="article reference" href="https://doi.org/10.3389%2Ffmicb.2023.1160736" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 95" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37089538" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 95" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10117906" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 95" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-nucleotide%20resolution%20detection%20of%20Topo%20IV%20cleavage%20activity%20in%20the%20Escherichia%20coli%20genome%20with%20Topo-Seq&amp;journal=Front.%20Microbiol.&amp;doi=10.3389%2Ffmicb.2023.1160736&amp;volume=14&amp;publication_year=2023&amp;author=Sutormin%2CD&amp;author=Galivondzhyan%2CA&amp;author=Gafurov%2CA&amp;author=Severinov%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="96."><p class="c-article-references__text" id="ref-CR96">Canela, A. et al. Genome organization drives chromosome fragility. <i>Cell</i> <b>170</b>, 507–521.e518 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 96" data-doi="10.1016/j.cell.2017.06.034" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2017.06.034" data-track-label="10.1016/j.cell.2017.06.034" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.06.034" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 96" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1Sls73N" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 96" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28735753" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 96" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6133249" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 96" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome%20organization%20drives%20chromosome%20fragility&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.06.034&amp;volume=170&amp;pages=507-521.e518&amp;publication_year=2017&amp;author=Canela%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="97."><p class="c-article-references__text" id="ref-CR97">Zerulla, K. et al. DNA as a phosphate storage polymer and the alternative advantages of polyploidy for growth or survival. <i>PLoS ONE</i> <b>9</b>, e94819 (2014).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 97" data-doi="10.1371/journal.pone.0094819" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1371/journal.pone.0094819" data-track-label="10.1371/journal.pone.0094819" data-track-value="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0094819" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 97" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2014PLoSO...994819Z" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 97" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24733558" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 97" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3986227" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 97" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1501.46020" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 97" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20as%20a%20phosphate%20storage%20polymer%20and%20the%20alternative%20advantages%20of%20polyploidy%20for%20growth%20or%20survival&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0094819&amp;volume=9&amp;publication_year=2014&amp;author=Zerulla%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="98."><p class="c-article-references__text" id="ref-CR98">Le Bihan, Y. V. et al. Effect of Rap1 binding on DNA distortion and potassium permanganate hypersensitivity. <i>Acta Crystallogr. D Biol. Crystallogr.</i> <b>69</b>, 409–419 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 98" data-doi="10.1107/S0907444912049311" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1107/S0907444912049311" data-track-label="10.1107/S0907444912049311" data-track-value="article reference" href="https://doi.org/10.1107%2FS0907444912049311" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 98" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013AcCrD..69..409L" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 98" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23519416" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 98" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1119.12002" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 98" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Effect%20of%20Rap1%20binding%20on%20DNA%20distortion%20and%20potassium%20permanganate%20hypersensitivity&amp;journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr.&amp;doi=10.1107%2FS0907444912049311&amp;volume=69&amp;pages=409-419&amp;publication_year=2013&amp;author=Bihan%2CY.%20V" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="99."><p class="c-article-references__text" id="ref-CR99">Analikwu B. T. et al. Telomere protein arrays stall DNA loop extrusion by condensin. <i>bioRxiv</i>, 2023.2010.2029.564563 (2023).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="100."><p class="c-article-references__text" id="ref-CR100">Yano, K., Noguchi, H. &amp; Niki, H. Profiling a single-stranded DNA region within an rDNA segment that affects the loading of bacterial condensin. <i>iScience</i> <b>25</b>, 105504 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 100" data-doi="10.1016/j.isci.2022.105504" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.isci.2022.105504" data-track-label="10.1016/j.isci.2022.105504" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.isci.2022.105504" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 100" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022iSci...25j5504Y" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 100" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XjtValtL7K" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 100" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36437875" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 100" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9682359" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 100" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Profiling%20a%20single-stranded%20DNA%20region%20within%20an%20rDNA%20segment%20that%20affects%20the%20loading%20of%20bacterial%20condensin&amp;journal=iScience&amp;doi=10.1016%2Fj.isci.2022.105504&amp;volume=25&amp;publication_year=2022&amp;author=Yano%2CK&amp;author=Noguchi%2CH&amp;author=Niki%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="101."><p class="c-article-references__text" id="ref-CR101">Forterre, P. A hot story from comparative genomics: reverse gyrase is the only hyperthermophile-specific protein. <i>Trends Genet.</i> <b>18</b>, 236–237 (2002).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 101" data-doi="10.1016/S0168-9525(02)02650-1" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/S0168-9525(02)02650-1" data-track-label="10.1016/S0168-9525(02)02650-1" data-track-value="article reference" href="https://doi.org/10.1016%2FS0168-9525%2802%2902650-1" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 101" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38XjvFCgtr4%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 101" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12047940" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 101" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0987.76522" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 101" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20hot%20story%20from%20comparative%20genomics%3A%20reverse%20gyrase%20is%20the%20only%20hyperthermophile-specific%20protein&amp;journal=Trends%20Genet.&amp;doi=10.1016%2FS0168-9525%2802%2902650-1&amp;volume=18&amp;pages=236-237&amp;publication_year=2002&amp;author=Forterre%2CP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="102."><p class="c-article-references__text" id="ref-CR102">Atomi, H., Matsumi, R. &amp; Imanaka, T. Reverse gyrase is not a prerequisite for hyperthermophilic life. <i>J. Bacteriol.</i> <b>186</b>, 4829–4833 (2004).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 102" data-doi="10.1128/JB.186.14.4829-4833.2004" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1128/JB.186.14.4829-4833.2004" data-track-label="10.1128/JB.186.14.4829-4833.2004" data-track-value="article reference" href="https://doi.org/10.1128%2FJB.186.14.4829-4833.2004" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 102" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXmtVemtr8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 102" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15231817" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 102" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC438624" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 102" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1065.30030" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 102" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Reverse%20gyrase%20is%20not%20a%20prerequisite%20for%20hyperthermophilic%20life&amp;journal=J.%20Bacteriol.&amp;doi=10.1128%2FJB.186.14.4829-4833.2004&amp;volume=186&amp;pages=4829-4833&amp;publication_year=2004&amp;author=Atomi%2CH&amp;author=Matsumi%2CR&amp;author=Imanaka%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="103."><p class="c-article-references__text" id="ref-CR103">Lipscomb, G. L., Hahn, E. M., Crowley, A. T. &amp; Adams, M. W. W. Reverse gyrase is essential for microbial growth at 95 degrees C. <i>Extremophiles</i> <b>21</b>, 603–608 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 103" data-doi="10.1007/s00792-017-0929-z" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1007/s00792-017-0929-z" data-track-label="10.1007/s00792-017-0929-z" data-track-value="article reference" href="https://link.springer.com/doi/10.1007/s00792-017-0929-z" rel="noopener">Article</a> 
    <a aria-label="CAS reference 103" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXkslOksLg%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 103" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28331998" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 103" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Reverse%20gyrase%20is%20essential%20for%20microbial%20growth%20at%2095%20degrees%20C&amp;journal=Extremophiles&amp;doi=10.1007%2Fs00792-017-0929-z&amp;volume=21&amp;pages=603-608&amp;publication_year=2017&amp;author=Lipscomb%2CG.%20L&amp;author=Hahn%2CE.%20M&amp;author=Crowley%2CA.%20T&amp;author=Adams%2CM.%20W.%20W" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="104."><p class="c-article-references__text" id="ref-CR104">Hocher, A. et al. Growth temperature and chromatinization in archaea. <i>Nat. Microbiol.</i> <b>7</b>, 1932–1942 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 104" data-doi="10.1038/s41564-022-01245-2" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41564-022-01245-2" data-track-label="10.1038/s41564-022-01245-2" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41564-022-01245-2" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 104" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38Xis12ju7fF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 104" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36266339" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 104" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7613761" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 104" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?07814482" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 104" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Growth%20temperature%20and%20chromatinization%20in%20archaea&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-022-01245-2&amp;volume=7&amp;pages=1932-1942&amp;publication_year=2022&amp;author=Hocher%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="105."><p class="c-article-references__text" id="ref-CR105">Chiu, T. P., Rao, S., Mann, R. S., Honig, B. &amp; Rohs, R. Genome-wide prediction of minor-groove electrostatic potential enables biophysical modeling of protein-DNA binding. <i>Nucleic Acids Res.</i> <b>45</b>, 12565–12576 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 105" data-doi="10.1093/nar/gkx915" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkx915" data-track-label="10.1093/nar/gkx915" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkx915" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 105" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1Wju7vI" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 105" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29040720" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 105" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5716191" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 105" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20prediction%20of%20minor-groove%20electrostatic%20potential%20enables%20biophysical%20modeling%20of%20protein-DNA%20binding&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkx915&amp;volume=45&amp;pages=12565-12576&amp;publication_year=2017&amp;author=Chiu%2CT.%20P&amp;author=Rao%2CS&amp;author=Mann%2CR.%20S&amp;author=Honig%2CB&amp;author=Rohs%2CR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="106."><p class="c-article-references__text" id="ref-CR106">Michimori, Y. et al. Removal of phosphoglycolate in hyperthermophilic archaea. <i>Proc. Natl. Acad. Sci. USA</i> <b>121</b>, e2311390121 (2024).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 106" data-doi="10.1073/pnas.2311390121" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.2311390121" data-track-label="10.1073/pnas.2311390121" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.2311390121" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 106" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB2cXhtVOjtrvJ" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 106" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=38593075" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 106" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11032457" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 106" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Removal%20of%20phosphoglycolate%20in%20hyperthermophilic%20archaea&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2311390121&amp;volume=121&amp;publication_year=2024&amp;author=Michimori%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="107."><p class="c-article-references__text" id="ref-CR107">Sato, T., Fukui, T., Atomi, H. &amp; Imanaka, T. Targeted gene disruption by homologous recombination in the hyperthermophilic archaeon <i>Thermococcus kodakaraensis</i> KOD1. <i>J. Bacteriol.</i> <b>185</b>, 210–220 (2003).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 107" data-doi="10.1128/JB.185.1.210-220.2003" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1128/JB.185.1.210-220.2003" data-track-label="10.1128/JB.185.1.210-220.2003" data-track-value="article reference" href="https://doi.org/10.1128%2FJB.185.1.210-220.2003" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 107" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3sXht1Cquw%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 107" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12486058" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 107" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC141832" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 107" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20gene%20disruption%20by%20homologous%20recombination%20in%20the%20hyperthermophilic%20archaeon%20Thermococcus%20kodakaraensis%20KOD1&amp;journal=J.%20Bacteriol.&amp;doi=10.1128%2FJB.185.1.210-220.2003&amp;volume=185&amp;pages=210-220&amp;publication_year=2003&amp;author=Sato%2CT&amp;author=Fukui%2CT&amp;author=Atomi%2CH&amp;author=Imanaka%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="108."><p class="c-article-references__text" id="ref-CR108">Samson, R. Y. et al. Specificity and function of archaeal DNA replication initiator proteins. <i>Cell Rep.</i> <b>3</b>, 485–496 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 108" data-doi="10.1016/j.celrep.2013.01.002" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.celrep.2013.01.002" data-track-label="10.1016/j.celrep.2013.01.002" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.celrep.2013.01.002" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 108" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFahs74%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 108" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23375370" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 108" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3607249" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 108" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1270.81040" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 108" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Specificity%20and%20function%20of%20archaeal%20DNA%20replication%20initiator%20proteins&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2013.01.002&amp;volume=3&amp;pages=485-496&amp;publication_year=2013&amp;author=Samson%2CR.%20Y" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="109."><p class="c-article-references__text" id="ref-CR109">Su, Y., Michimori, Y. &amp; Atomi, H. Biochemical and genetic examination of two aminotransferases from the hyperthermophilic archaeon <i>Thermococcus kodakarensis</i>. <i>Front. Microbiol.</i> <b>14</b>, 1126218 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 109" data-doi="10.3389/fmicb.2023.1126218" data-track="click_references" data-track-action="article reference" data-track-item_id="10.3389/fmicb.2023.1126218" data-track-label="10.3389/fmicb.2023.1126218" data-track-value="article reference" href="https://doi.org/10.3389%2Ffmicb.2023.1126218" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 109" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36891395" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 109" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9986279" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 109" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Biochemical%20and%20genetic%20examination%20of%20two%20aminotransferases%20from%20the%20hyperthermophilic%20archaeon%20Thermococcus%20kodakarensis&amp;journal=Front.%20Microbiol.&amp;doi=10.3389%2Ffmicb.2023.1126218&amp;volume=14&amp;publication_year=2023&amp;author=Su%2CY&amp;author=Michimori%2CY&amp;author=Atomi%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="110."><p class="c-article-references__text" id="ref-CR110">Jager, D., Forstner, K. U., Sharma, C. M., Santangelo, T. J. &amp; Reeve, J. N. Primary transcriptome map of the hyperthermophilic archaeon <i>Thermococcus kodakarensis</i>. <i>BMC Genom.</i> <b>15</b>, 684 (2014).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 110" data-doi="10.1186/1471-2164-15-684" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1186/1471-2164-15-684" data-track-label="10.1186/1471-2164-15-684" data-track-value="article reference" href="https://link.springer.com/doi/10.1186/1471-2164-15-684" rel="noopener">Article</a> 
    <a aria-label="Google Scholar reference 110" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Primary%20transcriptome%20map%20of%20the%20hyperthermophilic%20archaeon%20Thermococcus%20kodakarensis&amp;journal=BMC%20Genom.&amp;doi=10.1186%2F1471-2164-15-684&amp;volume=15&amp;publication_year=2014&amp;author=Jager%2CD&amp;author=Forstner%2CK.%20U&amp;author=Sharma%2CC.%20M&amp;author=Santangelo%2CT.%20J&amp;author=Reeve%2CJ.%20N" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="111."><p class="c-article-references__text" id="ref-CR111">Villain, P. et al. The hyperthermophilic archaeon <i>Thermococcus kodakarensis</i> is resistant to pervasive negative supercoiling activity of DNA gyrase. <i>Nucleic Acids Res.</i> <b>49</b>, 12332–12347 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 111" data-doi="10.1093/nar/gkab869" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkab869" data-track-label="10.1093/nar/gkab869" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkab869" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 111" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XlslKjug%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 111" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34755863" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 111" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8643681" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 111" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1135.62345" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 111" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20hyperthermophilic%20archaeon%20Thermococcus%20kodakarensis%20is%20resistant%20to%20pervasive%20negative%20supercoiling%20activity%20of%20DNA%20gyrase&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkab869&amp;volume=49&amp;pages=12332-12347&amp;publication_year=2021&amp;author=Villain%2CP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="112."><p class="c-article-references__text" id="ref-CR112">Ducret, A., Quardokus, E. M. &amp; Brun, Y. V. MicrobeJ, a tool for high throughput bacterial cell detection and quantitative analysis. <i>Nat. Microbiol.</i> <b>1</b>, 16077 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 112" data-doi="10.1038/nmicrobiol.2016.77" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nmicrobiol.2016.77" data-track-label="10.1038/nmicrobiol.2016.77" data-track-value="article reference" href="https://doi.org/10.1038%2Fnmicrobiol.2016.77" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 112" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXkvFyrsrY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 112" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27572972" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 112" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5010025" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 112" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1156.51010" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 112" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=MicrobeJ%2C%20a%20tool%20for%20high%20throughput%20bacterial%20cell%20detection%20and%20quantitative%20analysis&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fnmicrobiol.2016.77&amp;volume=1&amp;publication_year=2016&amp;author=Ducret%2CA&amp;author=Quardokus%2CE.%20M&amp;author=Brun%2CY.%20V" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="113."><p class="c-article-references__text" id="ref-CR113">Sas-Chen, A. et al. Dynamic RNA acetylation revealed by quantitative cross-evolutionary mapping. <i>Nature</i> <b>583</b>, 638–643 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 113" data-doi="10.1038/s41586-020-2418-2" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41586-020-2418-2" data-track-label="10.1038/s41586-020-2418-2" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41586-020-2418-2" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 113" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Natur.583..638S" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 113" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtF2ju7jN" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 113" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32555463" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 113" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8130014" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 113" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?56.0164.21" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 113" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Dynamic%20RNA%20acetylation%20revealed%20by%20quantitative%20cross-evolutionary%20mapping&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2418-2&amp;volume=583&amp;pages=638-643&amp;publication_year=2020&amp;author=Sas-Chen%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="114."><p class="c-article-references__text" id="ref-CR114">Servant, N. et al. HiC-Pro: an optimized and flexible pipeline for Hi-C data processing. <i>Genome Biol.</i> <b>16</b>, 259 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 114" data-doi="10.1186/s13059-015-0831-x" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1186/s13059-015-0831-x" data-track-label="10.1186/s13059-015-0831-x" data-track-value="article reference" href="https://link.springer.com/doi/10.1186/s13059-015-0831-x" rel="noopener">Article</a> 
    <a aria-label="PubMed reference 114" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26619908" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 114" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4665391" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 114" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0992.83084" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 114" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=HiC-Pro%3A%20an%20optimized%20and%20flexible%20pipeline%20for%20Hi-C%20data%20processing&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-015-0831-x&amp;volume=16&amp;publication_year=2015&amp;author=Servant%2CN" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="115."><p class="c-article-references__text" id="ref-CR115">Patro, R., Duggal, G., Love, M. I., Irizarry, R. A. &amp; Kingsford, C. Salmon provides fast and bias-aware quantification of transcript expression. <i>Nat. Methods</i> <b>14</b>, 417–419 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 115" data-doi="10.1038/nmeth.4197" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nmeth.4197" data-track-label="10.1038/nmeth.4197" data-track-value="article reference" href="https://doi.org/10.1038%2Fnmeth.4197" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 115" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXltVWgtL8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 115" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28263959" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 115" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5600148" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 115" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1355.92034" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 115" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Salmon%20provides%20fast%20and%20bias-aware%20quantification%20of%20transcript%20expression&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.4197&amp;volume=14&amp;pages=417-419&amp;publication_year=2017&amp;author=Patro%2CR&amp;author=Duggal%2CG&amp;author=Love%2CM.%20I&amp;author=Irizarry%2CR.%20A&amp;author=Kingsford%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="116."><p class="c-article-references__text" id="ref-CR116">Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. <i>Bioinformatics</i> <b>26</b>, 139–140 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 116" data-doi="10.1093/bioinformatics/btp616" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/bioinformatics/btp616" data-track-label="10.1093/bioinformatics/btp616" data-track-value="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp616" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 116" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXhs1WlurvO" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 116" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19910308" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 116" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1397.94089" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 116" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp616&amp;volume=26&amp;pages=139-140&amp;publication_year=2010&amp;author=Robinson%2CM.%20D&amp;author=McCarthy%2CD.%20J&amp;author=Smyth%2CG.%20K" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="117."><p class="c-article-references__text" id="ref-CR117">Langmead, B. &amp; Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. <i>Nat. Methods</i> <b>9</b>, 357–359 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 117" data-doi="10.1038/nmeth.1923" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nmeth.1923" data-track-label="10.1038/nmeth.1923" data-track-value="article reference" href="https://doi.org/10.1038%2Fnmeth.1923" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 117" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xjt1Oqt7c%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 117" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22388286" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 117" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322381" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 117" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?07740911" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 117" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.1923&amp;volume=9&amp;pages=357-359&amp;publication_year=2012&amp;author=Langmead%2CB&amp;author=Salzberg%2CS.%20L" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="118."><p class="c-article-references__text" id="ref-CR118">Li, H. et al. The sequence alignment/map format and SAMtools. <i>Bioinformatics</i> <b>25</b>, 2078–2079 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 118" data-doi="10.1093/bioinformatics/btp352" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/bioinformatics/btp352" data-track-label="10.1093/bioinformatics/btp352" data-track-value="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp352" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 118" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19505943" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 118" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 118" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1192.14050" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 118" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp352&amp;volume=25&amp;pages=2078-2079&amp;publication_year=2009&amp;author=Li%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="119."><p class="c-article-references__text" id="ref-CR119">Ramirez, F. et al. deepTools2: a next generation web server for deep-sequencing data analysis. <i>Nucleic Acids Res.</i> <b>44</b>, W160–W165 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 119" data-doi="10.1093/nar/gkw257" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkw257" data-track-label="10.1093/nar/gkw257" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkw257" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 119" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtV2itrfI" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 119" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27079975" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 119" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987876" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 119" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1372.32040" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 119" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=deepTools2%3A%20a%20next%20generation%20web%20server%20for%20deep-sequencing%20data%20analysis&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw257&amp;volume=44&amp;pages=W160-W165&amp;publication_year=2016&amp;author=Ramirez%2CF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="120."><p class="c-article-references__text" id="ref-CR120">Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). <i>Genome Biol.</i> <b>9</b>, R137 (2008).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 120" data-doi="10.1186/gb-2008-9-9-r137" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1186/gb-2008-9-9-r137" data-track-label="10.1186/gb-2008-9-9-r137" data-track-value="article reference" href="https://link.springer.com/doi/10.1186/gb-2008-9-9-r137" rel="noopener">Article</a> 
    <a aria-label="PubMed reference 120" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18798982" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 120" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2592715" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 120" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1204.05023" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 120" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Model-based%20analysis%20of%20ChIP-Seq%20%28MACS%29&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fgb-2008-9-9-r137&amp;volume=9&amp;publication_year=2008&amp;author=Zhang%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="121."><p class="c-article-references__text" id="ref-CR121">Gel, B. et al. regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests. <i>Bioinformatics</i> <b>32</b>, 289–291 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 121" data-doi="10.1093/bioinformatics/btv562" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/bioinformatics/btv562" data-track-label="10.1093/bioinformatics/btv562" data-track-value="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtv562" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 121" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVWhurnL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 121" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26424858" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 121" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1347.62198" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 121" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=regioneR%3A%20an%20R%2FBioconductor%20package%20for%20the%20association%20analysis%20of%20genomic%20regions%20based%20on%20permutation%20tests&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtv562&amp;volume=32&amp;pages=289-291&amp;publication_year=2016&amp;author=Gel%2CB" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="122."><p class="c-article-references__text" id="ref-CR122">Machanick, P. &amp; Bailey, T. L. MEME-ChIP: motif analysis of large DNA datasets. <i>Bioinformatics</i> <b>27</b>, 1696–1697 (2011).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 122" data-doi="10.1093/bioinformatics/btr189" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/bioinformatics/btr189" data-track-label="10.1093/bioinformatics/btr189" data-track-value="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtr189" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 122" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXntFGkurw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 122" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21486936" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 122" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3106185" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 122" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1368.93595" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 122" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=MEME-ChIP%3A%20motif%20analysis%20of%20large%20DNA%20datasets&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtr189&amp;volume=27&amp;pages=1696-1697&amp;publication_year=2011&amp;author=Machanick%2CP&amp;author=Bailey%2CT.%20L" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="123."><p class="c-article-references__text" id="ref-CR123">Quinlan, A. R. &amp; Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. <i>Bioinformatics</i> <b>26</b>, 841–842 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 123" data-doi="10.1093/bioinformatics/btq033" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/bioinformatics/btq033" data-track-label="10.1093/bioinformatics/btq033" data-track-value="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtq033" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 123" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXivFGkurc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 123" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20110278" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 123" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832824" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 123" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0942.65101" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 123" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtq033&amp;volume=26&amp;pages=841-842&amp;publication_year=2010&amp;author=Quinlan%2CA.%20R&amp;author=Hall%2CI.%20M" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="124."><p class="c-article-references__text" id="ref-CR124">Shen, W., Le, S., Li, Y. &amp; Hu, F. SeqKit: a cross-platform and ultrafast toolkit for FASTA/Q file manipulation. <i>PLoS ONE</i> <b>11</b>, e0163962 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 124" data-doi="10.1371/journal.pone.0163962" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1371/journal.pone.0163962" data-track-label="10.1371/journal.pone.0163962" data-track-value="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0163962" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 124" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27706213" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 124" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5051824" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 124" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=SeqKit%3A%20a%20cross-platform%20and%20ultrafast%20toolkit%20for%20FASTA%2FQ%20file%20manipulation&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0163962&amp;volume=11&amp;publication_year=2016&amp;author=Shen%2CW&amp;author=Le%2CS&amp;author=Li%2CY&amp;author=Hu%2CF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="125."><p class="c-article-references__text" id="ref-CR125">Hensel, R. &amp; König, H. Thermoadaptation of methanogenic bacteria by intracellular ion concentration. <i>FEMS Microbiol. Lett.</i> <b>49</b>, 75–79 (1988).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 125" data-doi="10.1111/j.1574-6968.1988.tb02685.x" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1111/j.1574-6968.1988.tb02685.x" data-track-label="10.1111/j.1574-6968.1988.tb02685.x" data-track-value="article reference" href="https://doi.org/10.1111%2Fj.1574-6968.1988.tb02685.x" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 125" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaL1cXhsVKku7s%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 125" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Thermoadaptation%20of%20methanogenic%20bacteria%20by%20intracellular%20ion%20concentration&amp;journal=FEMS%20Microbiol.%20Lett.&amp;doi=10.1111%2Fj.1574-6968.1988.tb02685.x&amp;volume=49&amp;pages=75-79&amp;publication_year=1988&amp;author=Hensel%2CR&amp;author=K%C3%B6nig%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="126."><p class="c-article-references__text" id="ref-CR126">Kenzaki, H. et al. CafeMol: A coarse-grained biomolecular simulator for simulating proteins at work. <i>J. Chem. Theory Comput.</i> <b>7</b>, 1979–1989 (2011).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 126" data-doi="10.1021/ct2001045" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/ct2001045" data-track-label="10.1021/ct2001045" data-track-value="article reference" href="https://doi.org/10.1021%2Fct2001045" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 126" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXmt1Gnurw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 126" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26596457" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 126" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1268.90098" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 126" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CafeMol%3A%20A%20coarse-grained%20biomolecular%20simulator%20for%20simulating%20proteins%20at%20work&amp;journal=J.%20Chem.%20Theory%20Comput.&amp;doi=10.1021%2Fct2001045&amp;volume=7&amp;pages=1979-1989&amp;publication_year=2011&amp;author=Kenzaki%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="127."><p class="c-article-references__text" id="ref-CR127">Lu, X. J. &amp; Olson, W. K. 3DNA: a versatile, integrated software system for the analysis, rebuilding and visualization of three-dimensional nucleic-acid structures. <i>Nat. Protoc.</i> <b>3</b>, 1213–1227 (2008).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 127" data-doi="10.1038/nprot.2008.104" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nprot.2008.104" data-track-label="10.1038/nprot.2008.104" data-track-value="article reference" href="https://doi.org/10.1038%2Fnprot.2008.104" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 127" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXotFaktbY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 127" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18600227" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 127" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3065354" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 127" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1162.30022" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 127" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=3DNA%3A%20a%20versatile%2C%20integrated%20software%20system%20for%20the%20analysis%2C%20rebuilding%20and%20visualization%20of%20three-dimensional%20nucleic-acid%20structures&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fnprot.2008.104&amp;volume=3&amp;pages=1213-1227&amp;publication_year=2008&amp;author=Lu%2CX.%20J&amp;author=Olson%2CW.%20K" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="128."><p class="c-article-references__text" id="ref-CR128">Okuda, S. et al. jPOSTrepo: an international standard data repository for proteomes. <i>Nucleic Acids Res.</i> <b>45</b>, D1107–D1111 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 128" data-doi="10.1093/nar/gkw1080" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkw1080" data-track-label="10.1093/nar/gkw1080" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkw1080" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 128" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslWgsL0%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 128" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27899654" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 128" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0799.62007" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 128" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=jPOSTrepo%3A%20an%20international%20standard%20data%20repository%20for%20proteomes&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw1080&amp;volume=45&amp;pages=D1107-D1111&amp;publication_year=2017&amp;author=Okuda%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1038/s41467-025-56197-y?format=refman&amp;flavour=references" rel="nofollow">Download references<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-download-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We thank Single-Cell Genome Information Analysis Core (SignAC) at WPI-ASHBi at Kyoto University for Illumina DNA sequencing. We thank Takeshi Yamagami and Suzuka Kajikawa for their technical assistance to purify the Smc protein, Hugo Maruyama for his kind gifts of the <i>T. kodakarensis</i> KCP1 strain and the TrmBL2 expression plasmid, and Stephen Bell for helpful comments on the manuscript. The <i>T. acidophilum</i> strain DSM 1728 (JCM 9062) was provided by Japan Collection of Microorganisms at RIKEN BRC, which is participating in the National BioResource Project of the MEXT, Japan. The supercomputer of Academic Center for Computing and Media Studies (Kyoto University) was used for the MD simulations. K.Y. is supported by JST, the establishment of university fellowships towards the creation of science technology innovation (JPMJFS2123) and SPRING (JPMJSP2110). N.T. is funded by JST PRESTO (JPMJPR20K7), JST FOREST (JPMJFR224V), JSPS Grants-in-Aid for Young Scientists Start-up (JP21K20636), JSPS Grants-in-Aid for Transformative Research Areas (JP23H04281), Takeda Science Foundation, and the Uehara Memorial Foundation. S.T. is funded by JSPS Grants-in-Aid for Transformative Research Areas (JP20H05934). Y.I. is funded by JSPS Grant-in-Aid for Challenging Research (Exploratory) (JP19K22289).</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="na1"><p>These authors contributed equally: Kodai Yamaura, Naomichi Takemata.</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Department of Synthetic Chemistry and Biological Chemistry, Graduate School of Engineering, Kyoto University, Kyoto, Japan</p><p class="c-article-author-affiliation__authors-list">Kodai Yamaura, Naomichi Takemata, Masashi Kariya, Ayami Osaka, Tomonori Tamura, Itaru Hamachi &amp; Haruyuki Atomi</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Department of Bioscience and Biotechnology, Graduate School of Bioresource and Bioenvironmental Sciences, Kyushu University, Fukuoka, Japan</p><p class="c-article-author-affiliation__authors-list">Sonoko Ishino &amp; Yoshizumi Ishino</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Genome Editing Research Institute, Nagahama Institute of Bio-Science and Technology, Nagahama, Shiga, Japan</p><p class="c-article-author-affiliation__authors-list">Sonoko Ishino &amp; Yoshizumi Ishino</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Department of Biophysics, Graduate School of Science, Kyoto University, Kyoto, Japan</p><p class="c-article-author-affiliation__authors-list">Masataka Yamauchi &amp; Shoji Takada</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Cell Biology Center, Institute of Innovative Research, Institute of Science Tokyo, Yokohama, Kanagawa, Japan</p><p class="c-article-author-affiliation__authors-list">Yoshizumi Ishino</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Kodai-Yamaura-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Kodai Yamaura</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Kodai%20Yamaura" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kodai%20Yamaura" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kodai%20Yamaura%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Naomichi-Takemata-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Naomichi Takemata</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Naomichi%20Takemata" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Naomichi%20Takemata" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Naomichi%20Takemata%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Masashi-Kariya-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Masashi Kariya</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Masashi%20Kariya" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Masashi%20Kariya" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Masashi%20Kariya%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ayami-Osaka-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Ayami Osaka</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Ayami%20Osaka" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ayami%20Osaka" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ayami%20Osaka%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Sonoko-Ishino-Aff2-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Sonoko Ishino</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Sonoko%20Ishino" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sonoko%20Ishino" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sonoko%20Ishino%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Masataka-Yamauchi-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Masataka Yamauchi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Masataka%20Yamauchi" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Masataka%20Yamauchi" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Masataka%20Yamauchi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Tomonori-Tamura-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Tomonori Tamura</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Tomonori%20Tamura" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Tomonori%20Tamura" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tomonori%20Tamura%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Itaru-Hamachi-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Itaru Hamachi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Itaru%20Hamachi" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Itaru%20Hamachi" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Itaru%20Hamachi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Shoji-Takada-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Shoji Takada</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Shoji%20Takada" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Shoji%20Takada" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Shoji%20Takada%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Yoshizumi-Ishino-Aff2-Aff3-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Yoshizumi Ishino</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Yoshizumi%20Ishino" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Yoshizumi%20Ishino" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Yoshizumi%20Ishino%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Haruyuki-Atomi-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Haruyuki Atomi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Haruyuki%20Atomi" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Haruyuki%20Atomi" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Haruyuki%20Atomi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>K.Y. and M.K. performed 3C-seq and strain construction for <i>T. kodakarensis</i>. K.Y. also performed growth measurement, RNA-seq, and ChIP-seq. N.T. performed 3C-seq on <i>T. acidophilum</i> and DNA affinity purification. T.T. and I.H. performed mass spectrometry analysis. K.Y. and S.I. performed protein purification to raise antibodies for ChIP-seq. K.Y., A.O., and N.T. performed microscopic analysis. N.T. performed data analysis on DNA sequencing data. M.Y. and S.T. performed MD simulations. N.T. and H.A. supervised the project. N.T., S.T., and Y.I. acquired funding. K.Y., N.T., A.O., M.Y., and T.T. wrote the manuscript. All co-authors critically read and edited the manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a href="mailto:takemata.naomichi.3a@kyoto-u.ac.jp" id="corresp-c1">Naomichi Takemata</a> or <a href="mailto:atomi.haruyuki.8r@kyoto-u.ac.jp" id="corresp-c2">Haruyuki Atomi</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
<h3 class="c-article__sub-heading" id="FPar2">Competing interests</h3>
<p>The authors declare no competing interests.</p>
</div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
<h3 class="c-article__sub-heading" id="FPar1">Peer review information</h3>
<p><i>Nature Communications</i> thanks Hironori Niki and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p>
</div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec61-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec61">Supplementary information</h2><div class="c-article-section__content" id="Sec61-content"><div data-test="supplementary-info"><div class="c-article-figshare-container" data-test="figshare-container" id="figshareContainer"></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_MOESM1_ESM.pdf">Supplementary Information</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="description of additional supplementary files" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_MOESM2_ESM.pdf">Description of Additional Supplementary Files</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary data 1" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_MOESM3_ESM.xlsx">Supplementary Data 1</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary data 2" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_MOESM4_ESM.xlsx">Supplementary Data 2</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM5"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_MOESM5_ESM.pdf">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="author checklist" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_MOESM6_ESM.docx">Author checklist</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="transparent peer review file" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_MOESM7_ESM.pdf">Transparent Peer Review file</a></h3></div></div></div></div></section><section data-title="Source data"><div class="c-article-section" id="Sec62-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec62">Source data</h2><div class="c-article-section__content" id="Sec62-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="source data" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56197-y/MediaObjects/41467_2025_56197_MOESM8_ESM.xlsx">Source Data</a></h3></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
<p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/" rel="license">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Chromosomal%20domain%20formation%20by%20archaeal%20SMC%2C%20a%20roadblock%20protein%2C%20and%20DNA%20structure&amp;author=Kodai%20Yamaura%20et%20al&amp;contentID=10.1038%2Fs41467-025-56197-y&amp;copyright=The%20Author%28s%29&amp;publication=2041-1723&amp;publicationDate=2025-02-19&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY-NC-ND">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41467-025-56197-y" data-test="crossmark" data-track="click" data-track-action="Click Crossmark" data-track-label="link" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41467-025-56197-y" rel="noopener" target="_blank"><img alt="Check for updates. Verify currency and authenticity via CrossMark" height="81" loading="lazy" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>" width="57"/></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Yamaura, K., Takemata, N., Kariya, M. <i>et al.</i> Chromosomal domain formation by archaeal SMC, a roadblock protein, and DNA structure.
                    <i>Nat Commun</i> <b>16</b>, 1312 (2025). https://doi.org/10.1038/s41467-025-56197-y</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-external="" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1038/s41467-025-56197-y?format=refman&amp;flavour=citation" rel="nofollow">Download citation<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-download-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2024-06-06">06 June 2024</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2025-01-10">10 January 2025</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2025-02-19">19 February 2025</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41467-025-56197-y</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" data-track="click" data-track-action="get shareable link" data-track-external="" data-track-label="button" id="get-share-url" type="button">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" data-track="click" data-track-action="select share url" data-track-label="button" id="share-url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" data-track="click" data-track-action="copy share url" data-track-external="" data-track-label="button" id="copy-share-url" type="button">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
</div>
</div>
</article>
</main>
<aside aria-label="Article navigation" class="c-article-extras u-hide-print" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
<div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41467-025-56197-y.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
<div class="c-reading-companion">
<div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
<div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
<div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu="">
<div class="c-ad c-ad--300x250">
<div class="c-ad__inner">
<p class="c-ad__label">Advertisement</p>
<div class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-gpt="" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41467-025-56197-y;doi=10.1038/s41467-025-56197-y;techmeta=1,119,14,15,23,38,58,63,82,91;subjmeta=100,101,2526,26,326,337,631;kwrd=Archaeal+genomics,Chromatin+structure" data-gpt-unitpath="/285/nature_communications/article" data-pa11y-ignore="" data-test="right-ad" id="div-gpt-ad-right-2">
<noscript>
<a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature_communications/article&amp;sz=300x250&amp;c=1773293014&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41467-025-56197-y%26doi%3D10.1038/s41467-025-56197-y%26techmeta%3D1,119,14,15,23,38,58,63,82,91%26subjmeta%3D100,101,2526,26,326,337,631%26kwrd%3DArchaeal+genomics,Chromatin+structure">
<img alt="Advertisement" data-test="gpt-advert-fallback-img" height="250" src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature_communications/article&amp;sz=300x250&amp;c=1773293014&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41467-025-56197-y%26doi%3D10.1038/s41467-025-56197-y%26techmeta%3D1,119,14,15,23,38,58,63,82,91%26subjmeta%3D100,101,2526,26,326,337,631%26kwrd%3DArchaeal+genomics,Chromatin+structure" width="300"/></a>
</noscript>
</div>
</div>
</div>
</div>
</div>
<div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
<div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
</div>
</div>
</aside>
</div>
<nav aria-labelledby="Explore-content" class="c-header__dropdown" data-test="Explore-content" data-track-component="nature-150-split-header" id="explore">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="Explore-content">Explore content</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="research articles" data-track-label="link" href="/ncomms/research-articles">
                                    Research articles
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" href="/ncomms/reviews-and-analysis">
                                    Reviews &amp; Analysis
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="news &amp; comment" data-track-label="link" href="/ncomms/news-and-comment">
                                    News &amp; Comment
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="videos" data-track-label="link" href="/ncomms/video">
                                    Videos
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="collections" data-track-label="link" href="/ncomms/collections">
                                    Collections
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="subjects" data-track-label="link" href="/ncomms/browse-subjects">
                                    Subjects
                                </a>
</li>
</ul>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="facebook" data-track-label="link" href="https://www.facebook.com/NatureCommunications">Follow us on Facebook
                            </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="twitter" data-track-label="link" href="https://twitter.com/NatureComms">Follow us on Twitter
                            </a>
</li>
<li class="c-header__item c-header__item--hide-lg">
<a class="c-header__link" data-track="click" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=264" rel="nofollow">Sign up for alerts<svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--hide-lg">
<a class="c-header__link" data-track="click" data-track-action="rss feed" data-track-label="link" href="https://www.nature.com/ncomms.rss">
<span>RSS feed</span>
</a>
</li>
</ul>
</div>
</nav>
<nav aria-labelledby="About-the-journal" class="c-header__dropdown" data-test="about-the-journal" data-track-component="nature-150-split-header" id="about-the-journal">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="About-the-journal">About the journal</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="aims &amp; scope" data-track-label="link" href="/ncomms/aims">
                                    Aims &amp; Scope
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editors" data-track-label="link" href="/ncomms/editors">
                                    Editors
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="journal information" data-track-label="link" href="/ncomms/journal-information">
                                    Journal Information
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="open access fees and funding" data-track-label="link" href="/ncomms/open-access">
                                    Open Access Fees and Funding
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="calls for papers" data-track-label="link" href="/ncomms/calls-for-papers">
                                    Calls for Papers
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editorial values statement" data-track-label="link" href="/ncomms/editorial-values-statement">
                                    Editorial Values Statement
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="journal metrics" data-track-label="link" href="/ncomms/journal-impact">
                                    Journal Metrics
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editors' highlights" data-track-label="link" href="/ncomms/editorshighlights">
                                    Editors' Highlights
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="contact" data-track-label="link" href="/ncomms/contact">
                                    Contact
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editorial policies" data-track-label="link" href="/ncomms/editorial-policies">
                                    Editorial policies
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="top articles" data-track-label="link" href="/ncomms/top-articles">
                                    Top Articles
                                </a>
</li>
</ul>
</div>
</nav>
<nav aria-labelledby="Publish-with-us-label" class="c-header__dropdown" data-test="publish-with-us" data-track-component="nature-150-split-header" id="publish-with-us">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="Publish-with-us-label">Publish with us</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="for authors" data-track-label="link" href="/ncomms/submit">
                                    For authors
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="for reviewers" data-track-label="link" href="/ncomms/for-reviewers">
                                    For Reviewers
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-action="manuscript author services" data-track-context="header publish with us dropdown menu" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
</li>
<li class="c-header__item c-header__item--keyline">
<a class="c-header__link" data-gtm-criteo="submit-manuscript" data-track="click_submit_manuscript" data-track-action="submit manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-external="" data-track-label="link (publish with us dropdown menu)" href="https://mts-ncomms.nature.com/">Submit manuscript<svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
</a>
</li>
</ul>
</div>
</nav>
<div class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header" id="search-menu">
<div class="c-header__container">
<h2 class="c-header__visually-hidden">Search</h2>
<form action="/search" autocomplete="off" class="c-header__search-form" data-test="inline-search" method="get" role="search">
<label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
<div class="c-header__search-layout c-header__search-layout--max-width">
<div>
<input class="c-header__input" id="keywords" name="q" required="" type="text" value=""/>
</div>
<div class="c-header__search-layout">
<div>
<label class="c-header__visually-hidden" for="results-from">Show results from</label>
<select class="c-header__select" id="results-from" name="journal">
<option selected="" value="">All journals</option>
<option value="ncomms">This journal</option>
</select>
</div>
<div>
<button class="c-header__search-button" type="submit">Search</button>
</div>
</div>
</div>
</form>
<div class="c-header__flush">
<a class="c-header__link" data-track="click" data-track-action="advanced search" data-track-label="link" href="/search/advanced">
                Advanced search
            </a>
</div>
<h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
<ul class="c-header__list">
<li><a class="c-header__link" data-track="click" data-track-action="explore articles by subject" data-track-label="link" href="/subjects">Explore articles by subject</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="find a job" data-track-label="link" href="/naturecareers">Find a job</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="guide to authors" data-track-label="link" href="/authors/index.html">Guide to authors</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="editorial policies" data-track-label="link" href="/authors/editorial_policies/">Editorial policies</a></li>
</ul>
</div>
</div>
<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
<meta content="Springer Nature" itemprop="publisher"/>
<div class="u-mt-16 u-mb-16">
<div class="u-container">
<div class="u-display-flex u-flex-wrap u-justify-content-space-between">
<p class="c-meta u-ma-0 u-flex-shrink">
<span class="c-meta__item">
                    Nature Communications (<i>Nat Commun</i>)
                </span>
<span class="c-meta__item">
<abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">2041-1723</span> (online)
    </span>
</p>
</div>
</div>
</div>
<div class="c-footer">
<div class="u-hide-print" data-track-component="footer">
<h2 class="u-visually-hidden">nature.com sitemap</h2>
<div class="c-footer__container">
<div class="c-footer__grid c-footer__group--separator">
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="about us" data-track-label="link" href="https://www.nature.com/npg_/company_info/index.html">About us</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="press releases" data-track-label="link" href="https://www.nature.com/npg_/press_room/press_releases.html">Press releases</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="press office" data-track-label="link" href="https://press.nature.com/">Press office</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="contact us" data-track-label="link" href="https://support.nature.com/support/home">Contact us</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Discover content</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="journals a-z" data-track-label="link" href="https://www.nature.com/siteindex">Journals A-Z</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="article by subject" data-track-label="link" href="https://www.nature.com/subjects">Articles by subject</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="protocols.io" data-track-label="link" href="https://www.protocols.io/">protocols.io</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature index" data-track-label="link" href="https://www.natureindex.com/">Nature Index</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link" href="https://www.nature.com/authors/editorial_policies">Nature portfolio policies</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="open access" data-track-label="link" href="https://www.nature.com/nature-research/open-access">Open access</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="reprints and permissions" data-track-label="link" href="https://www.nature.com/reprints">Reprints &amp; permissions</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="data research service" data-track-label="link" href="https://www.springernature.com/gp/authors/research-data">Research data</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="language editing" data-track-label="link" href="https://authorservices.springernature.com/language-editing/">Language editing</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="scientific editing" data-track-label="link" href="https://authorservices.springernature.com/scientific-editing/">Scientific editing</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature masterclasses" data-track-label="link" href="https://masterclasses.nature.com/">Nature Masterclasses</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="research solutions" data-track-label="link" href="https://solutions.springernature.com/">Research Solutions</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="librarian service and tools" data-track-label="link" href="https://www.springernature.com/gp/librarians/tools-services">Librarian service &amp; tools</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="librarian portal" data-track-label="link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal">Librarian portal</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="open research" data-track-label="link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions">Open research</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="Recommend to library" data-track-label="link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library">Recommend to library</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="advertising" data-track-label="link" href="https://partnerships.nature.com/product/digital-advertising/">Advertising</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="partnerships and services" data-track-label="link" href="https://partnerships.nature.com/">Partnerships &amp; Services</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="media kits" data-track-label="link" href="https://partnerships.nature.com/media-kits/">Media kits</a>
</li>
<li class="c-footer__item"><a class="c-footer__link" data-track-action="branded content" data-track-label="link" href="https://partnerships.nature.com/product/branded-content-native-advertising/">Branded
                        content</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Professional development</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature careers" data-track-label="link" href="https://www.nature.com/naturecareers/">Nature Careers</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature conferences" data-track-label="link" href="https://conferences.nature.com">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Regional websites</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature africa" data-track-label="link" href="https://www.nature.com/natafrica">Nature Africa</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature china" data-track-label="link" href="http://www.naturechina.com">Nature China</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature india" data-track-label="link" href="https://www.nature.com/nindia">Nature India</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature Italy" data-track-label="link" href="https://www.nature.com/natitaly">Nature Italy</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature japan" data-track-label="link" href="https://www.natureasia.com/ja-jp">Nature Japan</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature middle east" data-track-label="link" href="https://www.nature.com/nmiddleeast">Nature Middle East</a></li>
</ul>
</div>
</div>
</div>
<div class="c-footer__container">
<ul class="c-footer__links">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="privacy policy" data-track-label="link" href="https://www.nature.com/info/privacy">Privacy
                Policy</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="use of cookies" data-track-label="link" href="https://www.nature.com/info/cookies">Use
                of cookies</a></li>
<li class="c-footer__item">
<button class="optanon-toggle-display c-footer__link" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link" onclick="javascript:;">Your privacy choices/Manage cookies
                </button>
</li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="legal notice" data-track-label="link" href="https://www.nature.com/info/legal-notice">Legal
                notice</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="accessibility statement" data-track-label="link" href="https://www.nature.com/info/accessibility-statement">Accessibility
                statement</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="terms and conditions" data-track-label="link" href="https://www.nature.com/info/terms-and-conditions">Terms &amp; Conditions</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="california privacy statement" data-track-label="link" href="https://www.springernature.com/ccpa">Your US state privacy rights</a></li>
</ul>
</div>
</div>
<div class="c-footer__container">
<a class="c-footer__link" href="https://www.springernature.com/">
<img alt="Springer Nature" height="20" loading="lazy" src="/static/images/logos/sn-logo-white-ea63208b81.svg" width="200"/>
</a>
<p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>
</div>
<div aria-hidden="true" class="u-visually-hidden">
<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path d="M0 .74h56.72v55.24H0z" id="a"></path></defs><symbol id="icon-access" viewbox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewbox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewbox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewbox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewbox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewbox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewbox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewbox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewbox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewbox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewbox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewbox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewbox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewbox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewbox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewbox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewbox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewbox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewbox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewbox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewbox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewbox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewbox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewbox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewbox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewbox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewbox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewbox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewbox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewbox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewbox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewbox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewbox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewbox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewbox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewbox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewbox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewbox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewbox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewbox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewbox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewbox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewbox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewbox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewbox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewbox="0 0 56.69 56.69"><path d="M0 0h56.69v56.69H0z" style="fill:none"></path><clippath id="b"><use style="overflow:visible" xlink:href="#a"></use></clippath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewbox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewbox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewbox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewbox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewbox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewbox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-download-medium" viewbox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-mail-medium" viewbox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewbox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewbox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewbox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewbox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewbox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewbox="0 0 100 100"><path clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z" fill-rule="evenodd"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewbox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>
<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif" data-component-expirydays="30" data-component-id="nature-briefing-banner" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Flagship">
<div class="c-site-messages__banner-large">
<div class="c-site-messages__close-container">
<button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
<svg aria-hidden="true" focusable="false" height="25px" version="1.1" viewbox="0 0 25 25" width="25px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<title>Close banner</title>
<defs></defs>
<g fill="none" fill-rule="evenodd" stroke="none" stroke-width="1">
<rect height="25" opacity="0" width="25" x="0" y="0"></rect>
<path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
</g>
</svg>
<span class="visually-hidden">Close</span>
</button>
</div>
<div class="c-site-messages__form-container">
<div class="grid grid-12 last">
<div class="grid grid-4">
<img alt="Nature Briefing" height="40" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250"/>
<p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
</div>
<div class="grid grid-8 last">
<form action="https://www.nature.com/briefing/briefing" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship" method="post">
<input id="briefing-banner-signup-form-input-track-originReferralPoint" name="track_originReferralPoint" type="hidden" value="MainBriefingBanner"/>
<input id="briefing-banner-signup-form-input-track-formType" name="track_formType" type="hidden" value="DirectEmailBanner"/>
<input id="gdpr_tick_banner" name="gdpr_tick" type="hidden" value="false"/>
<input id="marketing_input_banner" name="marketing" type="hidden" value="false"/>
<input id="marketing_tick_banner" name="marketing_tick" type="hidden" value="false"/>
<input id="brieferEntryPoint_banner" name="brieferEntryPoint" type="hidden" value="MainBriefingBanner"/>
<label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>
<div class="nature-briefing-banner__email-wrapper">
<input class="nature-briefing-banner__email-input box-sizing text14" data-test-element="briefing-emailbanner-email-input" id="emailAddress" name="emailAddress" placeholder="e.g. jo.smith@university.ac.uk" required="" type="email" value=""/>
<input id="defaultNewsletter_banner" name="N:nature_briefing_daily" type="hidden" value="true"/>
<button class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button" type="submit">Sign up</button>
</div>
<div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
<input class="nature-briefing-banner__checkbox-checkbox" data-test-element="briefing-emailbanner-gdpr-checkbox" id="gdpr-briefing-banner-checkbox" name="gdpr" required="" type="checkbox" value="true"/>
<label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
</div>
</form>
</div>
</div>
</div>
</div>
<div class="c-site-messages__banner-small">
<div class="c-site-messages__close-container">
<button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
<svg aria-hidden="true" focusable="false" height="25px" version="1.1" viewbox="0 0 25 25" width="25px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<title>Close banner</title>
<defs></defs>
<g fill="none" fill-rule="evenodd" stroke="none" stroke-width="1">
<rect height="25" opacity="0" width="25" x="0" y="0"></rect>
<path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
</g>
</svg>
<span class="visually-hidden">Close</span>
</button>
</div>
<div class="c-site-messages__content text14">
<span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
<a class="nature-briefing__link text14 sans-serif" data-test-element="briefing-banner-link" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner" rel="noreferrer noopener" target="_blank">Sign up for Nature Briefing
            </a>
</div>
</div>
</div>
<noscript>
<img alt="" height="0" hidden="" src="https://verify.nature.com/verify/nature.png" style="display: none" width="0"/>
</noscript>
<script async="" src="//content.readcube.com/ping?doi=10.1038/s41467-025-56197-y&amp;format=js&amp;last_modified=2025-02-19"></script>
</body>
</html>