<!DOCTYPE html>
<html class="grade-c" lang="en">
<head>
<title>Increasingly efficient chromatin binding of cohesin and CTCF supports chromatin architecture formation during zebrafish embryogenesis | Nature Communications</title>
<link href="https://www.nature.com/ncomms.rss" rel="alternate" type="application/rss+xml"/>
<script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
<link crossorigin="" href="https://cmp.nature.com" rel="preconnect"/>
<meta content="IE=edge" http-equiv="X-UA-Compatible"/>
<meta content="pc,mobile" name="applicable-device"/>
<meta content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes" name="viewport"/>
<meta content="5a2dc4ab3fcb9b0393241ffbbb490480" name="360-site-verification"/>
<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"chromatin-structure;embryogenesis;nuclear-organization;single-molecule-biophysics","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Communications","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41467-025-56889-5"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Jonas Coßmann","Pavel I. Kos","Vassiliki Varamogianni-Mamatsi","Devin S. Assenheimer","Tobias A. Bischof","Timo Kuhn","Thomas Vomhof","Argyris Papantonis","Luca Giorgetti","J. Christof M. Gebhardt"],"publishedAt":1740096000,"publishedAtString":"2025-02-21","title":"Increasingly efficient chromatin binding of cohesin and CTCF supports chromatin architecture formation during zebrafish embryogenesis","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Chromatin structure,Embryogenesis,Nuclear organization,Single-molecule biophysics"},"journal":{"pcode":"ncomms","title":"nature communications","volume":"16","issue":"1","id":41467,"publishingModel":"Open Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>
<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
<style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>
<link data-inline-css-source="critical-css" data-test="critical-css-handler" href="/static/css/enhanced-article-nature-branded-d54f9178e8.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null" rel="stylesheet"/>
<noscript>
<link href="/static/css/enhanced-article-nature-branded-d54f9178e8.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" rel="stylesheet" type="text/css"/>
</noscript>
<link href="/static/css/article-print-122346e276.css" media="print" rel="stylesheet" type="text/css"/>
<link href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png" rel="apple-touch-icon" sizes="180x180"/>
<link href="/static/images/favicons/nature/favicon-48x48-b52890008c.png" rel="icon" sizes="48x48" type="image/png"/>
<link href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png" rel="icon" sizes="32x32" type="image/png"/>
<link href="/static/images/favicons/nature/favicon-16x16-951651ab72.png" rel="icon" sizes="16x16" type="image/png"/>
<link crossorigin="use-credentials" href="/static/manifest.json" rel="manifest"/>
<link color="#000000" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" rel="mask-icon"/>
<link href="/static/images/favicons/nature/favicon.ico" rel="shortcut icon"/>
<meta content="#000000" name="msapplication-TileColor"/>
<meta content="/static/browserconfig.xml" name="msapplication-config"/>
<meta content="#000000" name="theme-color"/>
<meta content="Nature" name="application-name"/>
<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>
<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->
<script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-74.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-74.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>
<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-606cb67187.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-e919764a53.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>
<meta content="noarchive" name="robots"/>
<meta content="Yes" name="access"/>
<link href="https://www.nature.com/search" rel="search"/>
<link href="https://www.nature.com/opensearch/opensearch.xml" rel="search" title="nature.com" type="application/opensearchdescription+xml"/>
<link href="https://www.nature.com/opensearch/request" rel="search" title="nature.com" type="application/sru+xml"/>
<script type="application/ld+json">{"mainEntity":{"headline":"Increasingly efficient chromatin binding of cohesin and CTCF supports chromatin architecture formation during zebrafish embryogenesis","description":"The three-dimensional folding of chromosomes is essential for nuclear functions such as DNA replication and gene regulation. The emergence of chromatin architecture is thus an important process during embryogenesis. To shed light on the molecular and kinetic underpinnings of chromatin architecture formation, we characterized biophysical properties of cohesin and CTCF binding to chromatin and their changes upon cofactor depletion using single-molecule imaging in live developing zebrafish embryos. We found that chromatin-bound fractions of both cohesin and CTCF increased significantly between the 1000-cell and shield stages, which we could explain through changes in both their association and dissociation rates. Moreover, increasing binding of cohesin restricted chromatin motion, potentially via loop extrusion, and showed distinct stage-dependent nuclear distribution. Polymer simulations with experimentally derived parameters recapitulated the experimentally observed gradual emergence of chromatin architecture. Our findings reveal molecular kinetics underlying chromatin architecture formation during zebrafish embryogenesis. Using single-molecule tracking of cohesin and CTCF during zebrafish embryogenesis, the molecular kinetics underlying chromatin architecture formation was revealed and gradual emergence of chromatin structure recapitulated by polymer simulations.","datePublished":"2025-02-21T00:00:00Z","dateModified":"2025-02-21T00:00:00Z","pageStart":"1","pageEnd":"18","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41467-025-56889-5","keywords":["Chromatin structure","Embryogenesis","Nuclear organization","Single-molecule biophysics","Science","Humanities and Social Sciences","multidisciplinary"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig4_HTML.png"],"isPartOf":{"name":"Nature Communications","issn":["2041-1723"],"volumeNumber":"16","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group UK","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Jonas Coßmann","affiliation":[{"name":"Ulm University","address":{"name":"Institute of Biophysics, Ulm University, Ulm, Germany","@type":"PostalAddress"},"@type":"Organization"},{"name":"Ulm University","address":{"name":"Institute of Experimental Physics and IQST, Ulm University, Ulm, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Pavel I. Kos","url":"http://orcid.org/0000-0002-3625-0939","affiliation":[{"name":"Friedrich Miescher Institute for Biomedical Research","address":{"name":"Friedrich Miescher Institute for Biomedical Research, Basel, Switzerland","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Basel","address":{"name":"Department of Biomedicine, University of Basel, Basel, Switzerland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Vassiliki Varamogianni-Mamatsi","affiliation":[{"name":"University Medical Center Göttingen","address":{"name":"Institute of Pathology, University Medical Center Göttingen, Göttingen, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Devin S. Assenheimer","affiliation":[{"name":"Ulm University","address":{"name":"Institute of Biophysics, Ulm University, Ulm, Germany","@type":"PostalAddress"},"@type":"Organization"},{"name":"Ulm University","address":{"name":"Institute of Experimental Physics and IQST, Ulm University, Ulm, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Tobias A. Bischof","url":"http://orcid.org/0009-0000-0201-7373","affiliation":[{"name":"Ulm University","address":{"name":"Institute of Biophysics, Ulm University, Ulm, Germany","@type":"PostalAddress"},"@type":"Organization"},{"name":"Ulm University","address":{"name":"Institute of Experimental Physics and IQST, Ulm University, Ulm, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Timo Kuhn","affiliation":[{"name":"Ulm University","address":{"name":"Institute of Biophysics, Ulm University, Ulm, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Thomas Vomhof","url":"http://orcid.org/0000-0003-0194-7186","affiliation":[{"name":"Ulm University","address":{"name":"Institute of Biophysics, Ulm University, Ulm, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Argyris Papantonis","url":"http://orcid.org/0000-0001-7551-1073","affiliation":[{"name":"University Medical Center Göttingen","address":{"name":"Institute of Pathology, University Medical Center Göttingen, Göttingen, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Luca Giorgetti","url":"http://orcid.org/0000-0002-9664-9087","affiliation":[{"name":"Friedrich Miescher Institute for Biomedical Research","address":{"name":"Friedrich Miescher Institute for Biomedical Research, Basel, Switzerland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"J. Christof M. Gebhardt","url":"http://orcid.org/0000-0003-1900-600X","affiliation":[{"name":"Ulm University","address":{"name":"Institute of Biophysics, Ulm University, Ulm, Germany","@type":"PostalAddress"},"@type":"Organization"},{"name":"Ulm University","address":{"name":"Institute of Experimental Physics and IQST, Ulm University, Ulm, Germany","@type":"PostalAddress"},"@type":"Organization"}],"email":"christof.gebhardt@uni-ulm.de","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>
<link href="https://www.nature.com/articles/s41467-025-56889-5" rel="canonical"/>
<meta content="41467" name="journal_id"/>
<meta content="Increasingly efficient chromatin binding of cohesin and CTCF supports chromatin architecture formation during zebrafish embryogenesis" name="dc.title"/>
<meta content="Nature Communications 2025 16:1" name="dc.source"/>
<meta content="text/html" name="dc.format"/>
<meta content="Nature Publishing Group" name="dc.publisher"/>
<meta content="2025-02-21" name="dc.date"/>
<meta content="OriginalPaper" name="dc.type"/>
<meta content="En" name="dc.language"/>
<meta content="2025 The Author(s)" name="dc.copyright"/>
<meta content="2025 The Author(s)" name="dc.rights"/>
<meta content="journalpermissions@springernature.com" name="dc.rightsAgent"/>
<meta content="The three-dimensional folding of chromosomes is essential for nuclear functions such as DNA replication and gene regulation. The emergence of chromatin architecture is thus an important process during embryogenesis. To shed light on the molecular and kinetic underpinnings of chromatin architecture formation, we characterized biophysical properties of cohesin and CTCF binding to chromatin and their changes upon cofactor depletion using single-molecule imaging in live developing zebrafish embryos. We found that chromatin-bound fractions of both cohesin and CTCF increased significantly between the 1000-cell and shield stages, which we could explain through changes in both their association and dissociation rates. Moreover, increasing binding of cohesin restricted chromatin motion, potentially via loop extrusion, and showed distinct stage-dependent nuclear distribution. Polymer simulations with experimentally derived parameters recapitulated the experimentally observed gradual emergence of chromatin architecture. Our findings reveal molecular kinetics underlying chromatin architecture formation during zebrafish embryogenesis. Using single-molecule tracking of cohesin and CTCF during zebrafish embryogenesis, the molecular kinetics underlying chromatin architecture formation was revealed and gradual emergence of chromatin structure recapitulated by polymer simulations." name="dc.description"/>
<meta content="2041-1723" name="prism.issn"/>
<meta content="Nature Communications" name="prism.publicationName"/>
<meta content="2025-02-21" name="prism.publicationDate"/>
<meta content="16" name="prism.volume"/>
<meta content="1" name="prism.number"/>
<meta content="OriginalPaper" name="prism.section"/>
<meta content="1" name="prism.startingPage"/>
<meta content="18" name="prism.endingPage"/>
<meta content="2025 The Author(s)" name="prism.copyright"/>
<meta content="journalpermissions@springernature.com" name="prism.rightsAgent"/>
<meta content="https://www.nature.com/articles/s41467-025-56889-5" name="prism.url"/>
<meta content="doi:10.1038/s41467-025-56889-5" name="prism.doi"/>
<meta content="https://www.nature.com/articles/s41467-025-56889-5.pdf" name="citation_pdf_url"/>
<meta content="https://www.nature.com/articles/s41467-025-56889-5" name="citation_fulltext_html_url"/>
<meta content="Nature Communications" name="citation_journal_title"/>
<meta content="Nat Commun" name="citation_journal_abbrev"/>
<meta content="Nature Publishing Group" name="citation_publisher"/>
<meta content="2041-1723" name="citation_issn"/>
<meta content="Increasingly efficient chromatin binding of cohesin and CTCF supports chromatin architecture formation during zebrafish embryogenesis" name="citation_title"/>
<meta content="16" name="citation_volume"/>
<meta content="1" name="citation_issue"/>
<meta content="2025/02/21" name="citation_online_date"/>
<meta content="1" name="citation_firstpage"/>
<meta content="18" name="citation_lastpage"/>
<meta content="Article" name="citation_article_type"/>
<meta content="" name="citation_fulltext_world_readable"/>
<meta content="en" name="citation_language"/>
<meta content="doi:10.1038/s41467-025-56889-5" name="dc.identifier"/>
<meta content="10.1038/s41467-025-56889-5" name="DOI"/>
<meta content="460731" name="size"/>
<meta content="10.1038/s41467-025-56889-5" name="citation_doi"/>
<meta content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41467-025-56889-5&amp;api_key=" name="citation_springer_api_url"/>
<meta content="The three-dimensional folding of chromosomes is essential for nuclear functions such as DNA replication and gene regulation. The emergence of chromatin architecture is thus an important process during embryogenesis. To shed light on the molecular and kinetic underpinnings of chromatin architecture formation, we characterized biophysical properties of cohesin and CTCF binding to chromatin and their changes upon cofactor depletion using single-molecule imaging in live developing zebrafish embryos. We found that chromatin-bound fractions of both cohesin and CTCF increased significantly between the 1000-cell and shield stages, which we could explain through changes in both their association and dissociation rates. Moreover, increasing binding of cohesin restricted chromatin motion, potentially via loop extrusion, and showed distinct stage-dependent nuclear distribution. Polymer simulations with experimentally derived parameters recapitulated the experimentally observed gradual emergence of chromatin architecture. Our findings reveal molecular kinetics underlying chromatin architecture formation during zebrafish embryogenesis. Using single-molecule tracking of cohesin and CTCF during zebrafish embryogenesis, the molecular kinetics underlying chromatin architecture formation was revealed and gradual emergence of chromatin structure recapitulated by polymer simulations." name="description"/>
<meta content="Coßmann, Jonas" name="dc.creator"/>
<meta content="Kos, Pavel I." name="dc.creator"/>
<meta content="Varamogianni-Mamatsi, Vassiliki" name="dc.creator"/>
<meta content="Assenheimer, Devin S." name="dc.creator"/>
<meta content="Bischof, Tobias A." name="dc.creator"/>
<meta content="Kuhn, Timo" name="dc.creator"/>
<meta content="Vomhof, Thomas" name="dc.creator"/>
<meta content="Papantonis, Argyris" name="dc.creator"/>
<meta content="Giorgetti, Luca" name="dc.creator"/>
<meta content="Gebhardt, J. Christof M." name="dc.creator"/>
<meta content="Chromatin structure" name="dc.subject"/>
<meta content="Embryogenesis" name="dc.subject"/>
<meta content="Nuclear organization" name="dc.subject"/>
<meta content="Single-molecule biophysics" name="dc.subject"/>
<meta content="citation_journal_title=Cold Spring Harb. Perspect. Biol.; citation_title=Chromosome territories; citation_author=T Cremer, M Cremer; citation_volume=2; citation_publication_date=2010; citation_pages=a003889; citation_doi=10.1101/cshperspect.a003889; citation_id=CR1" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Comprehensive mapping of long-range interactions reveals folding principles of the human genome; citation_author=E Lieberman-Aiden; citation_volume=326; citation_publication_date=2009; citation_pages=289-293; citation_doi=10.1126/science.1181369; citation_id=CR2" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Spatial organization of the mouse genome and its role in recurrent chromosomal translocations; citation_author=Y Zhang; citation_volume=148; citation_publication_date=2012; citation_pages=908-921; citation_doi=10.1016/j.cell.2012.02.002; citation_id=CR3" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Topological domains in mammalian genomes identified by analysis of chromatin interactions; citation_author=JR Dixon; citation_volume=485; citation_publication_date=2012; citation_pages=376-380; citation_doi=10.1038/nature11082; citation_id=CR4" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Spatial partitioning of the regulatory landscape of the X-inactivation centre; citation_author=EP Nora; citation_volume=485; citation_publication_date=2012; citation_pages=381-385; citation_doi=10.1038/nature11049; citation_id=CR5" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping; citation_author=SSP Rao; citation_volume=159; citation_publication_date=2014; citation_pages=1665-1680; citation_doi=10.1016/j.cell.2014.11.021; citation_id=CR6" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=The long-range interaction landscape of gene promoters; citation_author=A Sanyal, BR Lajoie, G Jain, J Dekker; citation_volume=489; citation_publication_date=2012; citation_pages=109-113; citation_doi=10.1038/nature11279; citation_id=CR7" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Chromatin fibers are formed by heterogeneous groups of nucleosomes in vivo; citation_author=MA Ricci, C Manzo, MF García-Parajo, M Lakadamyali, MP Cosma; citation_volume=160; citation_publication_date=2015; citation_pages=1145-1158; citation_doi=10.1016/j.cell.2015.01.054; citation_id=CR8" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=ChromEMT: Visualizing 3D chromatin structure and compaction in interphase and mitotic cells; citation_author=HD Ou; citation_volume=357; citation_publication_date=2017; citation_pages=139-148; citation_doi=10.1126/science.aag0025; citation_id=CR9" name="citation_reference"/>
<meta content="citation_journal_title=Cell Rep.; citation_title=Formation of chromosomal domains by loop extrusion; citation_author=G Fudenberg; citation_volume=15; citation_publication_date=2016; citation_pages=2038-2049; citation_doi=10.1016/j.celrep.2016.04.085; citation_id=CR10" name="citation_reference"/>
<meta content="citation_journal_title=Genes Dev.; citation_title=CTCF mediates long-range chromatin looping and local histone modification in the β-globin locus; citation_author=E Splinter; citation_volume=20; citation_publication_date=2006; citation_pages=2349-2354; citation_doi=10.1101/gad.399506; citation_id=CR11" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Cohesin loss eliminates all loop domains; citation_author=SSP Rao; citation_volume=171; citation_publication_date=2017; citation_pages=305-320.e24; citation_doi=10.1016/j.cell.2017.09.026; citation_id=CR12" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes; citation_author=AL Sanborn; citation_volume=112; citation_publication_date=2015; citation_pages=E6456-E6465; citation_doi=10.1073/pnas.1518552112; citation_id=CR13" name="citation_reference"/>
<meta content="Peters, J. M. &amp; Nishiyama, T. Sister chromatid cohesion. Cold Spring Harb. Perspect. Biol. 4, a011130 (2012)." name="citation_reference"/>
<meta content="citation_journal_title=Annu. Rev. Genet.; citation_title=Cohesin: Its roles and mechanisms; citation_author=K Nasmyth, CH Haering; citation_volume=43; citation_publication_date=2009; citation_pages=525-558; citation_doi=10.1146/annurev-genet-102108-134233; citation_id=CR15" name="citation_reference"/>
<meta content="citation_journal_title=Biodesign; citation_title=Structure-function relationships of SMC protein complexes for DNA loop extrusion; citation_author=H Lee, H Noh, J-K Ryu; citation_volume=9; citation_publication_date=2021; citation_pages=1-13; citation_doi=10.34184/kssb.2021.9.1.1; citation_id=CR16" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=DNA loop extrusion by human cohesin; citation_author=IF Davidson; citation_volume=366; citation_publication_date=2019; citation_pages=1338-1345; citation_doi=10.1126/science.aaz3418; citation_id=CR17" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Human cohesin compacts DNA by loop extrusion; citation_author=Y Kim, Z Shi, H Zhang, IJ Finkelstein, H Yu; citation_volume=366; citation_publication_date=2019; citation_pages=1345-1349; citation_doi=10.1126/science.aaz4475; citation_id=CR18" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Scc2 is a potent activator of cohesin’s ATPase that promotes loading by binding Scc1 without Pds5; citation_author=NJ Petela; citation_volume=70; citation_publication_date=2018; citation_pages=1134-1148.e7; citation_doi=10.1016/j.molcel.2018.05.022; citation_id=CR19" name="citation_reference"/>
<meta content="citation_journal_title=Elife; citation_title=Scc2/Nipbl hops between chromosomal cohesin rings after loading; citation_author=J Rhodes, D Mazza, K Nasmyth, S Uphoff; citation_volume=6; citation_publication_date=2017; citation_pages=1-20; citation_doi=10.7554/eLife.30000; citation_id=CR20" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Wapl controls the dynamic association of cohesin with chromatin; citation_author=S Kueng; citation_volume=127; citation_publication_date=2006; citation_pages=955-967; citation_doi=10.1016/j.cell.2006.09.040; citation_id=CR21" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=The cohesin release factor WAPL restricts chromatin loop extension; citation_author=JHI Haarhuis; citation_volume=169; citation_publication_date=2017; citation_pages=693-707.e14; citation_doi=10.1016/j.cell.2017.04.013; citation_id=CR22" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Sororin, a substrate of the anaphase-promoting complex, is required for sister chromatid cohesion in vertebrates; citation_author=S Rankin, NG Ayad, MW Kirschner; citation_volume=18; citation_publication_date=2005; citation_pages=185-200; citation_doi=10.1016/j.molcel.2005.03.017; citation_id=CR23" name="citation_reference"/>
<meta content="citation_journal_title=EMBO J.; citation_title=Sororin actively maintains sister chromatid cohesion; citation_author=R Ladurner; citation_volume=35; citation_publication_date=2016; citation_pages=635-653; citation_doi=10.15252/embj.201592532; citation_id=CR24" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Sororin mediates sister chromatid cohesion by antagonizing Wapl; citation_author=T Nishiyama; citation_volume=143; citation_publication_date=2010; citation_pages=737-749; citation_doi=10.1016/j.cell.2010.10.031; citation_id=CR25" name="citation_reference"/>
<meta content="citation_journal_title=Curr. Biol.; citation_title=Sororin is required for stable binding of cohesin to chromatin and for sister chromatid cohesion in interphase; citation_author=J Schmitz, E Watrin, P Lénárt, K Mechtler, JM Peters; citation_volume=17; citation_publication_date=2007; citation_pages=630-636; citation_doi=10.1016/j.cub.2007.02.029; citation_id=CR26" name="citation_reference"/>
<meta content="Corsi, F., Rusch, E. &amp; Goloborodko, A. Loop extrusion rules: the next generation. Curr. Opin. Genet. Dev. 81, 102061 (2023)." name="citation_reference"/>
<meta content="citation_journal_title=Nat. Rev. Mol. Cell Biol.; citation_title=Genome folding through loop extrusion by SMC complexes; citation_author=IF Davidson, JM Peters; citation_volume=22; citation_publication_date=2021; citation_pages=445-464; citation_doi=10.1038/s41580-021-00349-7; citation_id=CR28" name="citation_reference"/>
<meta content="citation_journal_title=Annu. Rev. Biochem.; citation_title=Looping the genome with SMC complexes; citation_author=E Kim, R Barth, C Dekker; citation_volume=92; citation_publication_date=2023; citation_pages=15-41; citation_doi=10.1146/annurev-biochem-032620-110506; citation_id=CR29" name="citation_reference"/>
<meta content="citation_journal_title=Cell Rep.; citation_title=Comparative Hi-C reveals that CTCF underlies evolution of chromosomal domain architecture; citation_author=M Vietri Rudan; citation_volume=10; citation_publication_date=2015; citation_pages=1297-1309; citation_doi=10.1016/j.celrep.2015.02.004; citation_id=CR30" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=CTCF Binding polarity determines chromatin looping; citation_author=E Wit; citation_volume=60; citation_publication_date=2015; citation_pages=676-684; citation_doi=10.1016/j.molcel.2015.09.023; citation_id=CR31" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=CTCF mediates chromatin looping via N-terminal domain-dependent cohesin retention; citation_author=EM Pugacheva; citation_volume=117; citation_publication_date=2020; citation_pages=2020-2031; citation_doi=10.1073/pnas.1911708117; citation_id=CR32" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Molecular basis of CTCF binding polarity in genome folding; citation_author=EP Nora; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-19283-x; citation_id=CR33" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=The structural basis for cohesin–CTCF-anchored loops; citation_author=Y Li; citation_volume=578; citation_publication_date=2020; citation_pages=472-476; citation_doi=10.1038/s41586-019-1910-z; citation_id=CR34" name="citation_reference"/>
<meta content="citation_journal_title=Curr. Opin. Cell Biol.; citation_title=The emergence of genome architecture and zygotic genome activation; citation_author=A Vallot, K Tachibana; citation_volume=64; citation_publication_date=2020; citation_pages=50-57; citation_doi=10.1016/j.ceb.2020.02.002; citation_id=CR35" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Allelic reprogramming of 3D chromatin architecture during early mammalian development; citation_author=Z Du; citation_volume=547; citation_publication_date=2017; citation_pages=232-235; citation_doi=10.1038/nature23263; citation_id=CR36" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=3D Chromatin structures of mature gametes and structural reprogramming during mammalian Embryogenesis; citation_author=Y Ke; citation_volume=170; citation_publication_date=2017; citation_pages=367-381.e20; citation_doi=10.1016/j.cell.2017.06.029; citation_id=CR37" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Key role for CTCF in establishing chromatin structure in human embryos; citation_author=X Chen; citation_volume=576; citation_publication_date=2019; citation_pages=306-310; citation_doi=10.1038/s41586-019-1812-0; citation_id=CR38" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Chromatin architecture emerges during zygotic genome activation independent of transcription; citation_author=CB Hug, AG Grimaldi, K Kruse, JM Vaquerizas; citation_volume=169; citation_publication_date=2017; citation_pages=216-228.e19; citation_doi=10.1016/j.cell.2017.03.024; citation_id=CR39" name="citation_reference"/>
<meta content="citation_journal_title=Genome Res.; citation_title=Chromatin architecture transitions from zebrafish sperm through early embryogenesis; citation_author=CL Wike; citation_volume=31; citation_publication_date=2021; citation_pages=981-994; citation_doi=10.1101/gr.269860.120; citation_id=CR40" name="citation_reference"/>
<meta content="citation_journal_title=Cell Rep.; citation_title=Systemic loss and gain of Chromatin architecture throughout Zebrafish development; citation_author=LJT Kaaij, RH Weide, RF Ketting, E Wit; citation_volume=24; citation_publication_date=2018; citation_pages=1-10.e4; citation_doi=10.1016/j.celrep.2018.06.003; citation_id=CR41" name="citation_reference"/>
<meta content="citation_journal_title=Genome Res.; citation_title=CTCF looping is established during gastrulation in medaka embryos; citation_author=R Nakamura; citation_volume=31; citation_publication_date=2021; citation_pages=968-980; citation_doi=10.1101/gr.269951.120; citation_id=CR42" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Superresolution imaging reveals spatiotemporal propagation of human replication foci mediated by CTCF-organized chromatin structures; citation_author=QP Su; citation_volume=117; citation_publication_date=2020; citation_pages=15036-15046; citation_doi=10.1073/pnas.2001521117; citation_id=CR43" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=CTCF is a DNA-tension-dependent barrier to cohesin-mediated loop extrusion; citation_author=IF Davidson; citation_volume=616; citation_publication_date=2023; citation_pages=822-827; citation_doi=10.1038/s41586-023-05961-5; citation_id=CR44" name="citation_reference"/>
<meta content="citation_journal_title=Elife; citation_title=Absolute quantification of cohesin, CTCF and their regulators in human cells; citation_author=J Holzmann; citation_volume=8; citation_publication_date=2019; citation_pages=1-31; citation_doi=10.7554/eLife.46269; citation_id=CR45" name="citation_reference"/>
<meta content="Hansen, A. S., Pustova, I., Cattoglio, C., Tjian, R. &amp; Darzacq, X. CTCF and cohesin regulate chromatin loop stability with distinct dynamics. Elife 6, e25776 (2017)." name="citation_reference"/>
<meta content="citation_journal_title=EMBO J.; citation_title=Topologically associating domains and chromatin loops depend on cohesin and are regulated by CTCF, WAPL, and PDS5 proteins; citation_author=G Wutz; citation_volume=36; citation_publication_date=2017; citation_pages=3573-3599; citation_doi=10.15252/embj.201798004; citation_id=CR47" name="citation_reference"/>
<meta content="citation_journal_title=Biophys. J.; citation_title=Direct observation of cell-cycle-dependent interactions between CTCF and Chromatin; citation_author=H Agarwal, M Reisser, C Wortmann, JCM Gebhardt; citation_volume=112; citation_publication_date=2017; citation_pages=2051-2055; citation_doi=10.1016/j.bpj.2017.04.018; citation_id=CR48" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Chem. Biol.; citation_title=Guided nuclear exploration increases CTCF target search efficiency; citation_author=AS Hansen, A Amitai, C Cattoglio, R Tjian, X Darzacq; citation_volume=16; citation_publication_date=2020; citation_pages=257-266; citation_doi=10.1038/s41589-019-0422-3; citation_id=CR49" name="citation_reference"/>
<meta content="citation_journal_title=Curr. Biol.; citation_title=Live-cell imaging reveals a stable cohesin-chromatin interaction after but not before DNA replication; citation_author=D Gerlich, B Koch, F Dupeux, JM Peters, J Ellenberg; citation_volume=16; citation_publication_date=2006; citation_pages=1571-1578; citation_doi=10.1016/j.cub.2006.06.068; citation_id=CR50" name="citation_reference"/>
<meta content="citation_journal_title=Elife; citation_title=Cohesin and condensin extrude DNA loops in a cell-cycle dependent manner; citation_author=S Golfier, T Quail, H Kimura, J Brugués; citation_volume=9; citation_publication_date=2020; citation_pages=1-34; citation_doi=10.7554/eLife.53885; citation_id=CR51" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development; citation_author=M Reisser; citation_volume=9; citation_publication_date=2018; citation_doi=10.1038/s41467-018-07731-8; citation_id=CR52" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=CTCF knockout in zebrafish induces alterations in regulatory landscapes and developmental gene expression; citation_author=M Franke; citation_volume=12; citation_publication_date=2021; citation_doi=10.1038/s41467-021-25604-5; citation_id=CR53" name="citation_reference"/>
<meta content="citation_journal_title=Development; citation_title=Cohesin-dependent regulation of Runx genes; citation_author=JA Horsfield; citation_volume=134; citation_publication_date=2007; citation_pages=2639-2649; citation_doi=10.1242/dev.002485; citation_id=CR54" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Molecular architecture of SMC proteins and the Yeast Cohesin Complex C-terminal domains forming a head would be part of; citation_author=CH Haering, J Löwe, A Hochwagen, K Nasmyth; citation_volume=9; citation_publication_date=2002; citation_pages=773-788; citation_doi=10.1016/S1097-2765(02)00515-4; citation_id=CR55" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=A general method to improve fluorophores for live-cell and single-molecule microscopy; citation_author=JB Grimm; citation_volume=12; citation_publication_date=2015; citation_pages=244-250; citation_doi=10.1038/nmeth.3256; citation_id=CR56" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Single-molecule tracking of Nodal and Lefty in live zebrafish embryos supports hindered diffusion model; citation_author=T Kuhn; citation_volume=13; citation_publication_date=2022; citation_doi=10.1038/s41467-022-33704-z; citation_id=CR57" name="citation_reference"/>
<meta content="citation_journal_title=Sci. Rep.; citation_title=Single molecule tracking and analysis framework including theory-predicted parameter settings; citation_author=T Kuhn, J Hettich, R Davtyan, JCM Gebhardt; citation_volume=11; citation_publication_date=2021; citation_pages=1-12; citation_doi=10.1038/s41598-021-88802-7; citation_id=CR58" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=A benchmark for chromatin binding measurements in live cells; citation_author=D Mazza, A Abernathy, N Golob, T Morisaki, JG McNally; citation_volume=40; citation_publication_date=2012; citation_pages=1-13; citation_doi=10.1093/nar/gks701; citation_id=CR59" name="citation_reference"/>
<meta content="citation_journal_title=Elife; citation_title=Robust model-based analysis of single-particle tracking experiments with spot-on; citation_author=AS Hansen; citation_volume=7; citation_publication_date=2018; citation_pages=1-33; citation_doi=10.7554/eLife.33125; citation_id=CR60" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=Single-molecule imaging of transcription factor binding to DNA in live mammalian cells; citation_author=JCM Gebhardt; citation_volume=10; citation_publication_date=2013; citation_pages=421-426; citation_doi=10.1038/nmeth.2411; citation_id=CR61" name="citation_reference"/>
<meta content="citation_journal_title=Biophys. J.; citation_title=Activated STAT1 transcription factors conduct distinct saltatory movements in the cell nucleus; citation_author=J Speil; citation_volume=101; citation_publication_date=2011; citation_pages=2592-2600; citation_doi=10.1016/j.bpj.2011.10.006; citation_id=CR62" name="citation_reference"/>
<meta content="Izeddin, I. et al. Single-molecule tracking in live cells reveals distinct target-search strategies of transcription factors in the nucleus. Elife 2014, e02230 (2014)." name="citation_reference"/>
<meta content="citation_journal_title=Genome Biol.; citation_title=&amp; Bystricky, K. Hi-D: Nanoscale mapping of nuclear dynamics in single living cells; citation_author=HA Shaban, R Barth, L Recoules; citation_volume=21; citation_publication_date=2020; citation_pages=1-21; citation_doi=10.1186/s13059-020-02002-6; citation_id=CR64" name="citation_reference"/>
<meta content="citation_journal_title=Biophys. J.; citation_title=Balbiani ring mRNPs diffuse through and bind to clusters of large intranuclear molecular structures; citation_author=R Veith; citation_volume=99; citation_publication_date=2010; citation_pages=2676-2685; citation_doi=10.1016/j.bpj.2010.08.004; citation_id=CR65" name="citation_reference"/>
<meta content="citation_journal_title=Rep. Prog. Phys.; citation_title=Anomalous transport in the crowded world of biological cells; citation_author=F Höfling, T Franosch; citation_volume=76; citation_publication_date=2013; citation_pages=046602; citation_doi=10.1088/0034-4885/76/4/046602; citation_id=CR66" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Structure and stability of cohesin’s Smc1-kleisin interaction; citation_author=CH Haering; citation_volume=15; citation_publication_date=2004; citation_pages=951-964; citation_doi=10.1016/j.molcel.2004.08.030; citation_id=CR67" name="citation_reference"/>
<meta content="citation_journal_title=Genome Biol.; citation_title=RAD21 is the core subunit of the cohesin complex involved in directing genome organization; citation_author=Y Sun; citation_volume=24; citation_publication_date=2023; citation_doi=10.1186/s13059-023-02982-1; citation_id=CR68" name="citation_reference"/>
<meta content="citation_journal_title=Cell Rep.; citation_title=A genome-wide Map of CTCF multivalency redefines the CTCF Code; citation_author=H Nakahashi; citation_volume=3; citation_publication_date=2013; citation_pages=1678-1689; citation_doi=10.1016/j.celrep.2013.04.024; citation_id=CR69" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Structural basis for the versatile and methylation-dependent binding of CTCF to DNA; citation_author=H Hashimoto; citation_volume=66; citation_publication_date=2017; citation_pages=711-720.e3; citation_doi=10.1016/j.molcel.2017.05.004; citation_id=CR70" name="citation_reference"/>
<meta content="citation_journal_title=Cell Res.; citation_title=Molecular mechanism of directional CTCF recognition of a diverse range of genomic sites; citation_author=M Yin; citation_volume=27; citation_publication_date=2017; citation_pages=1365-1377; citation_doi=10.1038/cr.2017.131; citation_id=CR71" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Opposing effects of cohesin and transcription on CTCF organization revealed by super-resolution imaging; citation_author=B Gu; citation_volume=80; citation_publication_date=2020; citation_pages=699-711.e7; citation_doi=10.1016/j.molcel.2020.10.001; citation_id=CR72" name="citation_reference"/>
<meta content="Cattoglio, C. et al. Determining cellular CTCF and cohesin abundances to constrain 3D genome models. Elife 8 (2019)." name="citation_reference"/>
<meta content="citation_journal_title=Elife; citation_title=MCM2–7-dependent cohesin loading during s phase promotes sister-chromatid cohesion; citation_author=G Zheng, M Kanchwala, C Xing, H Yu; citation_volume=7; citation_publication_date=2018; citation_pages=1-25; citation_doi=10.7554/eLife.33920; citation_id=CR74" name="citation_reference"/>
<meta content="citation_journal_title=Elife; citation_title=Cohesion is established during dna replication utilising chromosome associated cohesin rings as well as those loaded de novo onto nascent DNAs; citation_author=M Srinivasan, M Fumasoni, NJ Petela, A Murray, KA Nasmyth; citation_volume=9; citation_publication_date=2020; citation_pages=1-27; citation_doi=10.7554/eLife.56611; citation_id=CR75" name="citation_reference"/>
<meta content="citation_journal_title=Trends Cell Biol.; citation_title=NIPBL and cohesin: new take on a classic tale; citation_author=D Alonso-Gil, A Losada; citation_volume=33; citation_publication_date=2023; citation_pages=860-871; citation_doi=10.1016/j.tcb.2023.03.006; citation_id=CR76" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Cohesin’s binding to chromosomes depends on a separate complex consisting of Scc2 and Scc4 proteins; citation_author=R Ciosk; citation_volume=5; citation_publication_date=2000; citation_pages=243-254; citation_doi=10.1016/S1097-2765(00)80420-7; citation_id=CR77" name="citation_reference"/>
<meta content="citation_journal_title=Curr. Biol.; citation_title=Human Wapl is a Cohesin-binding protein that promotes sister-chromatid resolution in mitotic prophase; citation_author=R Gandhi, PJ Gillespie, T Hirano; citation_volume=16; citation_publication_date=2006; citation_pages=2406-2417; citation_doi=10.1016/j.cub.2006.10.061; citation_id=CR78" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Wapl is an essential regulator of chromatin structure and chromosome segregation; citation_author=A Tedeschi; citation_volume=501; citation_publication_date=2013; citation_pages=564-568; citation_doi=10.1038/nature12471; citation_id=CR79" name="citation_reference"/>
<meta content="citation_journal_title=Elife; citation_title=ESCO1 and CTCF enable formation of long chromatin loops by protecting cohesinstag1 from WAPL; citation_author=G Wutz; citation_volume=9; citation_publication_date=2020; citation_pages=1-33; citation_doi=10.7554/eLife.52091; citation_id=CR80" name="citation_reference"/>
<meta content="Muto, A., Calof, A. L., Lander, A. D. &amp; Schilling, T. F. Multifactorial origins of heart and gut defects in Nipbl-deficient zebrafish, a model of cornelia de Lange Syndrome. PLoS Biol. 9, e1001181 (2011)." name="citation_reference"/>
<meta content="Muto, A. et al. Nipbl and mediator cooperatively regulate gene expression to control limb development. PLoS Genet. 10, e1004671 (2014)." name="citation_reference"/>
<meta content="citation_journal_title=J. Biol. Chem.; citation_title=CTCF promotes muscle differentiation by modulating the activity of myogenic regulatory factors; citation_author=P Delgado-Olguín; citation_volume=286; citation_publication_date=2011; citation_pages=12483-12494; citation_doi=10.1074/jbc.M110.164574; citation_id=CR83" name="citation_reference"/>
<meta content="citation_journal_title=Dev; citation_title=Cohesin facilitates zygotic genome activation in zebrafish; citation_author=M Meier; citation_volume=145; citation_publication_date=2018; citation_pages=1-13; citation_id=CR84" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Transcription-mediated supercoiling regulates genome folding and loop formation; citation_author=MV Neguembor; citation_volume=81; citation_publication_date=2021; citation_pages=3065-3081.e12; citation_doi=10.1016/j.molcel.2021.06.009; citation_id=CR85" name="citation_reference"/>
<meta content="Sept, C. E. et al. High-resolution CTCF footprinting reveals impact of chromatin state on cohesin extrusion dynamics. bioRxiv Prepr. Serv. Biol. 
                  https://doi.org/10.1101/2023.10.20.563340
                  
                 (2023)." name="citation_reference"/>
<meta content="citation_journal_title=Sci. Adv.; citation_title=RNA polymerase II is required for spatial chromatin reorganization following exit from mitosis; citation_author=S Zhang; citation_volume=7; citation_publication_date=2021; citation_pages=1-14; citation_doi=10.1126/sciadv.abg8205; citation_id=CR87" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Genet.; citation_title=Enhancer–promoter contact formation requires RNAPII and antagonizes loop extrusion; citation_author=S Zhang, N Übelmesser, M Barbieri, A Papantonis; citation_volume=55; citation_publication_date=2023; citation_pages=832-840; citation_doi=10.1038/s41588-023-01364-4; citation_id=CR88" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Struct. Mol. Biol.; citation_title=Structural basis of transcription inhibition by α-amanitin and implications for RNA polymerase II translocation; citation_author=F Brueckner, P Cramer; citation_volume=15; citation_publication_date=2008; citation_pages=811-818; citation_doi=10.1038/nsmb.1458; citation_id=CR89" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=The RNA Polymerase II Trigger Loop functions in substrate selection and is directly targeted by α-Amanitin; citation_author=CD Kaplan, KM Larsson, RD Kornberg; citation_volume=30; citation_publication_date=2008; citation_pages=547-556; citation_doi=10.1016/j.molcel.2008.04.023; citation_id=CR90" name="citation_reference"/>
<meta content="citation_journal_title=Transcription; citation_title=Inhibiting eukaryotic transcription: Which compound to choose? How to evaluate its activity?; citation_author=O Bensaude; citation_volume=2; citation_publication_date=2011; citation_pages=103-108; citation_doi=10.4161/trns.2.3.16172; citation_id=CR91" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Widespread rearrangement of 3D Chromatin organization underlies polycomb-mediated stress-induced silencing; citation_author=L Li; citation_volume=58; citation_publication_date=2015; citation_pages=216-231; citation_doi=10.1016/j.molcel.2015.02.023; citation_id=CR92" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Genet.; citation_title=Paused RNA polymerase II inhibits new transcriptional initiation; citation_author=W Shao, J Zeitlinger; citation_volume=49; citation_publication_date=2017; citation_pages=1045-1051; citation_doi=10.1038/ng.3867; citation_id=CR93" name="citation_reference"/>
<meta content="citation_journal_title=PLoS Genet.; citation_title=Chromatin accessibility established by Pou5f3, Sox19b and Nanog primes genes for activity during zebrafish genome activation; citation_author=M Pálfy, G Schulze, E Valen, NL Vastenhouw; citation_volume=16; citation_publication_date=2020; citation_pages=1-25; citation_doi=10.1371/journal.pgen.1008546; citation_id=CR94" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Nanog, Pou5f1 and SoxB1 activate zygotic gene expression during the maternal-to-zygotic transition; citation_author=MT Lee; citation_volume=503; citation_publication_date=2013; citation_pages=360-364; citation_doi=10.1038/nature12632; citation_id=CR95" name="citation_reference"/>
<meta content="citation_journal_title=Development; citation_title=The zebrafish epiboly mutants; citation_author=DA Kane; citation_volume=123; citation_publication_date=1996; citation_pages=47-55; citation_doi=10.1242/dev.123.1.47; citation_id=CR96" name="citation_reference"/>
<meta content="citation_journal_title=J. Appl. Toxicol.; citation_title=Triptolide-induced hepatotoxicity via apoptosis and autophagy in zebrafish; citation_author=J Huo; citation_volume=39; citation_publication_date=2019; citation_pages=1532-1540; citation_doi=10.1002/jat.3837; citation_id=CR97" name="citation_reference"/>
<meta content="citation_journal_title=Toxicol. Sci.; citation_title=Characterization of triptolide-induced hepatotoxicity by imaging and transcriptomics in a novel zebrafish model; citation_author=ADB Vliegenthart; citation_volume=159; citation_publication_date=2017; citation_pages=380-391; citation_doi=10.1093/toxsci/kfx144; citation_id=CR98" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Codon usage and 3’ UTR length determine maternal mRNA stability in Zebrafish; citation_author=Y Mishima, Y Tomari; citation_volume=61; citation_publication_date=2016; citation_pages=874-885; citation_doi=10.1016/j.molcel.2016.02.027; citation_id=CR99" name="citation_reference"/>
<meta content="Song, Y. S. et al. Validation, optimization, and application of the Zebrafish developmental toxicity assay for pharmaceuticals under the ICH S5(R3) Guideline. Front. Cell Dev. Biol. 9, 721130 (2021)." name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Dynamic organization of chromatin domains revealed by super-resolution live-cell imaging; citation_author=T Nozaki; citation_volume=67; citation_publication_date=2017; citation_pages=282-293.e7; citation_doi=10.1016/j.molcel.2017.06.018; citation_id=CR101" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Organization of fast and slow chromatin revealed by single-nucleosome dynamics; citation_author=SS Ashwin, T Nozaki, K Maeshima, M Sasai; citation_volume=116; citation_publication_date=2019; citation_pages=19939-19944; citation_doi=10.1073/pnas.1907342116; citation_id=CR102" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Genet.; citation_title=Cohesin and CTCF control the dynamics of chromosome folding; citation_author=P Mach; citation_volume=54; citation_publication_date=2022; citation_pages=1907-1918; citation_doi=10.1038/s41588-022-01232-7; citation_id=CR103" name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=Dynamics of CTCF- and cohesin-mediated chromatin looping revealed by live-cell imaging; citation_author=M Gabriele; citation_volume=376; citation_publication_date=2022; citation_pages=476-501; citation_doi=10.1126/science.abn6583; citation_id=CR104" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Two independent modes of chromatin organization revealed by cohesin removal; citation_author=W Schwarzer; citation_volume=551; citation_publication_date=2017; citation_pages=51-56; citation_doi=10.1038/nature24281; citation_id=CR105" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Different NIPBL requirements of cohesin-STAG1 and cohesin-STAG2; citation_author=D Alonso-Gil, A Cuadrado, D Giménez-Llorente, M Rodríguez-Corsino, A Losada; citation_volume=14; citation_publication_date=2023; citation_pages=1-11; citation_doi=10.1038/s41467-023-36900-7; citation_id=CR106" name="citation_reference"/>
<meta content="citation_journal_title=Clin. Epigenet.; citation_title=The effect of Nipped-B-like (Nipbl) haploinsufficiency on genome-wide cohesin binding and target gene expression: Modeling Cornelia de Lange syndrome; citation_author=DA Newkirk; citation_volume=9; citation_publication_date=2017; citation_pages=1-20; citation_doi=10.1186/s13148-017-0391-x; citation_id=CR107" name="citation_reference"/>
<meta content="citation_journal_title=Elife; citation_title=Cohesin-independent STAG proteins interact with RNA and R-loops and promote complex loading; citation_author=H Porter; citation_volume=12; citation_publication_date=2023; citation_pages=1-33; citation_doi=10.7554/eLife.79386; citation_id=CR108" name="citation_reference"/>
<meta content="citation_journal_title=Hum. Mol. Genet.; citation_title=Regional chromatin decompaction in cornelia de lange syndrome associated with NIPBL disruption can be uncoupled from cohesin and CTCF; citation_author=LD Nolen, S Boyle, M Ansari, E Pritchard, WA Bickmore; citation_volume=22; citation_publication_date=2013; citation_pages=4180-4193; citation_doi=10.1093/hmg/ddt265; citation_id=CR109" name="citation_reference"/>
<meta content="citation_journal_title=Sci. Rep.; citation_title=Inferring quantity and qualities of superimposed reaction rates from single molecule survival time distributions; citation_author=M Reisser; citation_volume=10; citation_publication_date=2020; citation_pages=1-13; citation_doi=10.1038/s41598-020-58634-y; citation_id=CR110" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=Altering transcription factor binding reveals comprehensive transcriptional kinetics of a basic gene; citation_author=AP Popp, J Hettich, JCM Gebhardt; citation_volume=49; citation_publication_date=2021; citation_pages=6249-6266; citation_doi=10.1093/nar/gkab443; citation_id=CR111" name="citation_reference"/>
<meta content="citation_journal_title=Biochemistry; citation_title=Diffusion-DRIVEN MECHANISMS OF PROTEIN TRANSLOCATION ON NUCLEIC ACids. 1. Models and Theory; citation_author=OG Berg, RB Winter, PH Hippel; citation_volume=20; citation_publication_date=1981; citation_pages=6929-6948; citation_doi=10.1021/bi00527a028; citation_id=CR112" name="citation_reference"/>
<meta content="citation_journal_title=Nucleic Acids Res.; citation_title=Equilibrium and kinetic aspects of protein-DNA recognition; citation_author=MA Livhitz, GV Gursky, AS Zasedatelev, MV Volkenstein; citation_volume=6; citation_publication_date=1979; citation_pages=2217-2236; citation_doi=10.1093/nar/6.6.2217; citation_id=CR113" name="citation_reference"/>
<meta content="Loffreda, A. et al. Live-cell p53 single-molecule binding is modulated by C-terminal acetylation and correlates with transcriptional activity. Nat. Commun. 8, 313 (2017)." name="citation_reference"/>
<meta content="citation_journal_title=Cold Spring Harb. Symp. Quant. Biol.; citation_title=Emerging Evidence of Chromosome Folding by Loop Extrusion; citation_author=G Fudenberg, N Abdennur, M Imakaev, A Goloborodko, LA Mirny; citation_volume=82; citation_publication_date=2017; citation_pages=45-55; citation_doi=10.1101/sqb.2017.82.034710; citation_id=CR115" name="citation_reference"/>
<meta content="Galitsyna, A. et al. Extrusion fountains are hallmarks of chromosome organization emerging upon zygotic genome activation. bioRxiv Prepr. Serv. Biol. 
                  https://doi.org/10.1101/2023.07.15.549120
                  
                 (2023)." name="citation_reference"/>
<meta content="citation_journal_title=Phys. Rev. X; citation_title=Crumpled polymer with loops recapitulates key features of chromosome organization; citation_author=KE Polovnikov; citation_volume=13; citation_publication_date=2023; citation_pages=41029; citation_id=CR117" name="citation_reference"/>
<meta content="citation_journal_title=iScience; citation_title=Demarcation of topologically associating domains is uncoupled from enriched CTCF binding in developing Zebrafish; citation_author=YA Pérez-Rico, E Barillot, A Shkumatava; citation_volume=23; citation_publication_date=2020; citation_pages=101046; citation_doi=10.1016/j.isci.2020.101046; citation_id=CR118" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Struct. Mol. Biol.; citation_title=Mechanical disengagement of the cohesin ring; citation_author=M Richeldi; citation_volume=31; citation_publication_date=2024; citation_pages=23-31; citation_doi=10.1038/s41594-023-01122-4; citation_id=CR119" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell. Biol.; citation_title=Transcription-dependent induction of G 1 phase during the Zebra Fish Midblastula transition; citation_author=E Zamir, Z Kam, A Yarden; citation_volume=17; citation_publication_date=1997; citation_pages=529-536; citation_doi=10.1128/MCB.17.2.529; citation_id=CR120" name="citation_reference"/>
<meta content="citation_journal_title=Dev. Dyn.; citation_title=Stages of embryonic development of the zebrafish; citation_author=CB Kimmel, WW Ballard, SR Kimmel, B Ullmann, TF Schilling; citation_volume=203; citation_publication_date=1995; citation_pages=253-310; citation_doi=10.1002/aja.1002030302; citation_id=CR121" name="citation_reference"/>
<meta content="citation_journal_title=Phys. Rev. Lett.; citation_title=Interphase Chromatin undergoes a local sol-gel transition upon cell differentiation; citation_author=I Eshghi, JA Eaton, A Zidovska; citation_volume=126; citation_publication_date=2021; citation_pages=228101; citation_doi=10.1103/PhysRevLett.126.228101; citation_id=CR122" name="citation_reference"/>
<meta content="Subramanian, V. et al. H2A.Z Acidic patch couples chromatin dynamics to regulation of gene expression programs during ESC Differentiation. PLoS Genet. 9, e1003725 (2013)." name="citation_reference"/>
<meta content="citation_journal_title=Nat. Biotechnol.; citation_title=GPSeq reveals the radial organization of chromatin in the cell nucleus; citation_author=G Girelli; citation_volume=38; citation_publication_date=2020; citation_pages=1184-1193; citation_doi=10.1038/s41587-020-0519-y; citation_id=CR124" name="citation_reference"/>
<meta content="citation_journal_title=Cell; citation_title=Genome-scale imaging of the 3D organization and transcriptional activity of chromatin; citation_author=JH Su, P Zheng, SS Kinrot, B Bintu, X Zhuang; citation_volume=182; citation_publication_date=2020; citation_pages=1641-1659.e26; citation_doi=10.1016/j.cell.2020.07.032; citation_id=CR125" name="citation_reference"/>
<meta content="citation_journal_title=Genome Biol.; citation_title=CTCF and cohesin promote focal detachment of DNA from the nuclear lamina; citation_author=T Schaik; citation_volume=23; citation_publication_date=2022; citation_pages=1-26; citation_id=CR126" name="citation_reference"/>
<meta content="citation_journal_title=F1000Research; citation_title=Chromatin dynamics at the maternal to zygotic transition: Recent advances from the zebrafish model; citation_author=MG Goll, B Akdogan-Ozdilek, KL Duval; citation_volume=9; citation_publication_date=2020; citation_pages=1-12; citation_id=CR127" name="citation_reference"/>
<meta content="Laue, K., Rajshekar, S., Courtney, A. J., Lewis, Z. A. &amp; Goll, M. G. The maternal to zygotic transition regulates genome-wide heterochromatin establishment in the zebrafish embryo. Nat. Commun. 10, 1551 (2019)." name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Cohesin is positioned in mammalian genomes by transcription, CTCF and Wapl; citation_author=GA Busslinger; citation_volume=544; citation_publication_date=2017; citation_pages=503-507; citation_doi=10.1038/nature22063; citation_id=CR129" name="citation_reference"/>
<meta content="citation_journal_title=EMBO J.; citation_title=Rapid movement and transcriptional re‐localization of human cohesin on DNA; citation_author=IF Davidson; citation_volume=35; citation_publication_date=2016; citation_pages=2671-2685; citation_doi=10.15252/embj.201695402; citation_id=CR130" name="citation_reference"/>
<meta content="citation_journal_title=Proc. Natl Acad. Sci.; citation_title=Transcription shapes 3D chromatin organization by interacting with loop extrusion; citation_author=EJ Banigan; citation_volume=120; citation_publication_date=2023; citation_pages=2017; citation_doi=10.1073/pnas.2210480120; citation_id=CR131" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Rev. Genet.; citation_title=Mechanisms regulating zygotic genome activation; citation_author=KN Schulz, MM Harrison; citation_volume=20; citation_publication_date=2019; citation_pages=221-234; citation_doi=10.1038/s41576-018-0087-x; citation_id=CR132" name="citation_reference"/>
<meta content="citation_journal_title=Genome Res.; citation_title=Redundant and specific roles of cohesin STAG subunits in chromatin looping and transcriptional control; citation_author=V Casa; citation_volume=30; citation_publication_date=2020; citation_pages=515-527; citation_doi=10.1101/gr.253211.119; citation_id=CR133" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Regulation of Cohesin-mediated chromosome folding by Eco1 and other partners; citation_author=L Dauban; citation_volume=77; citation_publication_date=2020; citation_pages=1279-1293.e4; citation_doi=10.1016/j.molcel.2020.01.019; citation_id=CR134" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Struct. Mol. Biol.; citation_title=The cohesin acetylation cycle controls chromatin loop length through a PDS5A brake mechanism; citation_author=MS Ruiten; citation_volume=29; citation_publication_date=2022; citation_pages=586-591; citation_doi=10.1038/s41594-022-00773-z; citation_id=CR135" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Struct. Mol. Biol.; citation_title=Smc3 acetylation, Pds5 and Scc2 control the translocase activity that establishes cohesin-dependent chromatin loops; citation_author=N Bastié; citation_volume=29; citation_publication_date=2022; citation_pages=575-585; citation_doi=10.1038/s41594-022-00780-0; citation_id=CR136" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Functional crosstalk between the cohesin loader and chromatin remodelers; citation_author=S Muñoz; citation_volume=13; citation_publication_date=2022; citation_pages=1-12; citation_doi=10.1038/s41467-022-35444-6; citation_id=CR137" name="citation_reference"/>
<meta content="citation_journal_title=Sci. Rep.; citation_title=The chromatin remodeler Chd1 regulates cohesin in budding yeast and humans; citation_author=A Boginya, R Detroja, A Matityahu, M Frenkel-Morgenstern, I Onn; citation_volume=9; citation_publication_date=2019; citation_pages=1-10; citation_doi=10.1038/s41598-019-45263-3; citation_id=CR138" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=A role for Chromatin remodeling in cohesin loading onto chromosomes; citation_author=S Muñoz, M Minamino, CS Casas-Delucchi, H Patel, F Uhlmann; citation_volume=74; citation_publication_date=2019; citation_pages=664-673.e5; citation_doi=10.1016/j.molcel.2019.02.027; citation_id=CR139" name="citation_reference"/>
<meta content="citation_journal_title=Cells; citation_title=CTCF and its multi-partner network for chromatin regulation; citation_author=A Moral-Morales; citation_volume=12; citation_publication_date=2023; citation_pages=1-19; citation_id=CR140" name="citation_reference"/>
<meta content="citation_journal_title=PLoS One; citation_title=Improved long-term imaging of embryos with genetically encoded α-bungarotoxin; citation_author=IA Swinburne, KR Mosaliganti, AA Green, SG Megason; citation_volume=10; citation_publication_date=2015; citation_pages=1-15; citation_doi=10.1371/journal.pone.0134005; citation_id=CR141" name="citation_reference"/>
<meta content="citation_journal_title=BMC Dev. Biol.; citation_title=Proteomics of early zebrafish embryos; citation_author=V Link, A Shevchenko, CP Heisenberg; citation_volume=6; citation_publication_date=2006; citation_pages=1-9; citation_doi=10.1186/1471-213X-6-1; citation_id=CR142" name="citation_reference"/>
<meta content="Sato, Y. et al. Histone H3K27 acetylation precedes active transcription during zebrafish zygotic genome activation as revealed by live-cell analysis. Dev. 146, dev179127 (2019)." name="citation_reference"/>
<meta content="citation_journal_title=RNA Biol.; citation_title=Activation of transcription enforces the formation of distinct nuclear bodies in zebrafish embryos; citation_author=P Heyn, H Salmonowicz, J Rodenfels, KM Neugebauer; citation_volume=14; citation_publication_date=2017; citation_pages=752-760; citation_doi=10.1080/15476286.2016.1255397; citation_id=CR144" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Syst. Biol.; citation_title=RNA polymerase II clusters form in line with surface condensation on regulatory chromatin; citation_author=A Pancholi; citation_volume=17; citation_publication_date=2021; citation_pages=1-26; citation_doi=10.15252/msb.202110272; citation_id=CR145" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Transcription organizes euchromatin via microphase separation; citation_author=L Hilbert; citation_volume=12; citation_publication_date=2021; citation_pages=1-12; citation_id=CR146" name="citation_reference"/>
<meta content="citation_journal_title=Dev. Cell; citation_title=The miR-430 locus with extreme promoter density forms a transcription body during the minor wave of zygotic genome activation; citation_author=Y Hadzhiev; citation_volume=58; citation_publication_date=2023; citation_pages=155-170.e8; citation_doi=10.1016/j.devcel.2022.12.007; citation_id=CR147" name="citation_reference"/>
<meta content="citation_journal_title=Cell Rep.; citation_title=Ythdf m6A readers function redundantly during Zebrafish development; citation_author=C Kontur, M Jeong, D Cifuentes, AJ Giraldez; citation_volume=33; citation_publication_date=2020; citation_pages=108598; citation_doi=10.1016/j.celrep.2020.108598; citation_id=CR148" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=Fiji: An open-source platform for biological-image analysis; citation_author=J Schindelin; citation_volume=9; citation_publication_date=2012; citation_pages=676-682; citation_doi=10.1038/nmeth.2019; citation_id=CR149" name="citation_reference"/>
<meta content="Chen, B. C. et al. Lattice light-sheet microscopy: Imaging molecules to embryos at high spatiotemporal resolution. Science. 346, 1257998 (2014)." name="citation_reference"/>
<meta content="Lambert, Talley Shao, L. &amp; VolkerH. tlambert03/LLSpy: v0.4.8 (0.4.8). 
                  https://doi.org/10.5281/zenodo.3554482
                  
                 (2019)." name="citation_reference"/>
<meta content="citation_journal_title=Bioinformatics; citation_title=TANGO: A generic tool for high-throughput 3D image analysis for studying nuclear organization; citation_author=J Ollion, J Cochennec, F Loll, C Escudé, T Boudier; citation_volume=29; citation_publication_date=2013; citation_pages=1840-1841; citation_doi=10.1093/bioinformatics/btt276; citation_id=CR152" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Methods; citation_title=Fast gapped-read alignment with Bowtie 2; citation_author=B Langmead, SL Salzberg; citation_volume=9; citation_publication_date=2012; citation_pages=357-359; citation_doi=10.1038/nmeth.1923; citation_id=CR153" name="citation_reference"/>
<meta content="Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). Genome Biol. 9, R137 (2008)." name="citation_reference"/>
<meta content="citation_journal_title=Bioinformatics; citation_title=BEDTools: A flexible suite of utilities for comparing genomic features; citation_author=AR Quinlan, IM Hall; citation_volume=26; citation_publication_date=2010; citation_pages=841-842; citation_doi=10.1093/bioinformatics/btq033; citation_id=CR155" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Cell; citation_title=Simple Combinations of lineage-determining transcription factors Prime cis-regulatory elements required for macrophage and B cell identities; citation_author=S Heinz; citation_volume=38; citation_publication_date=2010; citation_pages=576-589; citation_doi=10.1016/j.molcel.2010.05.004; citation_id=CR156" name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Phys.; citation_title=Fast Parallel Algorithms For Short-range Molecular Dynamics; citation_author=S Plimpton; citation_volume=117; citation_publication_date=1995; citation_pages=1-19; citation_doi=10.1006/jcph.1995.1039; citation_id=CR157" name="citation_reference"/>
<meta content="De Gennes, P. G. Scaling Concepts in Polymer Physics. Cornell University Press (1979)." name="citation_reference"/>
<meta content="Barshad, G. et al. RNA polymerase II dynamics shape enhancer-promoter interactions. Nat. Genet. 
                  https://doi.org/10.1038/s41588-023-01442-7
                  
                 (2023)." name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=MCM complexes are barriers that restrict cohesin-mediated loop extrusion; citation_author=BJH Dequeker; citation_volume=606; citation_publication_date=2022; citation_pages=197-203; citation_doi=10.1038/s41586-022-04730-0; citation_id=CR160" name="citation_reference"/>
<meta content="Coßmann, J. et al. Increasingly efficient chromatin binding of cohesin and CTCF supports chromatin architecture formation during zebrafish embryogenesis [Dataset]. 
                  https://doi.org/10.5061/dryad.3bk3j9ks8
                  
                 (2025)." name="citation_reference"/>
<meta content="Kuhn, T., Hettich, J., Davtyan, R. &amp; Gebhardt, J. C. M. TrackIt\_v1.5. 
                  https://doi.org/10.5281/zenodo.7092296
                  
                 (2022)." name="citation_reference"/>
<meta content="Coßmann, Jonas" name="citation_author"/>
<meta content="Institute of Biophysics, Ulm University, Ulm, Germany" name="citation_author_institution"/>
<meta content="Institute of Experimental Physics and IQST, Ulm University, Ulm, Germany" name="citation_author_institution"/>
<meta content="Kos, Pavel I." name="citation_author"/>
<meta content="Friedrich Miescher Institute for Biomedical Research, Basel, Switzerland" name="citation_author_institution"/>
<meta content="Department of Biomedicine, University of Basel, Basel, Switzerland" name="citation_author_institution"/>
<meta content="Varamogianni-Mamatsi, Vassiliki" name="citation_author"/>
<meta content="Institute of Pathology, University Medical Center Göttingen, Göttingen, Germany" name="citation_author_institution"/>
<meta content="Assenheimer, Devin S." name="citation_author"/>
<meta content="Institute of Biophysics, Ulm University, Ulm, Germany" name="citation_author_institution"/>
<meta content="Institute of Experimental Physics and IQST, Ulm University, Ulm, Germany" name="citation_author_institution"/>
<meta content="Bischof, Tobias A." name="citation_author"/>
<meta content="Institute of Biophysics, Ulm University, Ulm, Germany" name="citation_author_institution"/>
<meta content="Institute of Experimental Physics and IQST, Ulm University, Ulm, Germany" name="citation_author_institution"/>
<meta content="Kuhn, Timo" name="citation_author"/>
<meta content="Institute of Biophysics, Ulm University, Ulm, Germany" name="citation_author_institution"/>
<meta content="Vomhof, Thomas" name="citation_author"/>
<meta content="Institute of Biophysics, Ulm University, Ulm, Germany" name="citation_author_institution"/>
<meta content="Papantonis, Argyris" name="citation_author"/>
<meta content="Institute of Pathology, University Medical Center Göttingen, Göttingen, Germany" name="citation_author_institution"/>
<meta content="Giorgetti, Luca" name="citation_author"/>
<meta content="Friedrich Miescher Institute for Biomedical Research, Basel, Switzerland" name="citation_author_institution"/>
<meta content="Gebhardt, J. Christof M." name="citation_author"/>
<meta content="Institute of Biophysics, Ulm University, Ulm, Germany" name="citation_author_institution"/>
<meta content="Institute of Experimental Physics and IQST, Ulm University, Ulm, Germany" name="citation_author_institution"/>
<meta content="https://www.nature.com/platform/readcube-access" name="access_endpoint"/>
<meta content="@NatureComms" name="twitter:site"/>
<meta content="summary_large_image" name="twitter:card"/>
<meta content="Content cover image" name="twitter:image:alt"/>
<meta content="Increasingly efficient chromatin binding of cohesin and CTCF supports chromatin architecture formation during zebrafish embryogenesis" name="twitter:title"/>
<meta content="Nature Communications - Using single-molecule tracking of cohesin and CTCF during zebrafish embryogenesis, the molecular kinetics underlying chromatin architecture formation was revealed and..." name="twitter:description"/>
<meta content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig1_HTML.png" name="twitter:image"/>
<meta content="https://www.nature.com/articles/s41467-025-56889-5" property="og:url"/>
<meta content="article" property="og:type"/>
<meta content="Nature" property="og:site_name"/>
<meta content="Increasingly efficient chromatin binding of cohesin and CTCF supports chromatin architecture formation during zebrafish embryogenesis - Nature Communications" property="og:title"/>
<meta content="Using single-molecule tracking of cohesin and CTCF during zebrafish embryogenesis, the molecular kinetics underlying chromatin architecture formation was revealed and gradual emergence of chromatin structure recapitulated by polymer simulations." property="og:description"/>
<meta content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig1_HTML.png" property="og:image"/>
<script>
        window.eligibleForRa21 = 'false'; 
    </script>
</head>
<body class="article-page">
<noscript><iframe height="0" src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ" style="display:none;visibility:hidden" width="0"></iframe></noscript>
<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
<a class="c-skip-link" href="#content">Skip to main content</a>
<div class="c-grade-c-banner u-hide">
<div class="c-grade-c-banner__container">
<p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>
</div>
</div>
<div class="u-hide u-show-following-ad"></div>
<aside class="c-ad c-ad--728x90">
<div class="c-ad__inner" data-container-type="banner-advert">
<p class="c-ad__label">Advertisement</p>
<div class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-gpt="" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41467-025-56889-5;doi=10.1038/s41467-025-56889-5;techmeta=14,34,63;subjmeta=100,101,136,2086,2265,337,386,57,631;kwrd=Chromatin+structure,Embryogenesis,Nuclear+organization,Single-molecule+biophysics" data-gpt-unitpath="/285/nature_communications/article" data-pa11y-ignore="" data-test="top-ad" id="div-gpt-ad-top-1">
<noscript>
<a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature_communications/article&amp;sz=728x90&amp;c=1337038477&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41467-025-56889-5%26doi%3D10.1038/s41467-025-56889-5%26techmeta%3D14,34,63%26subjmeta%3D100,101,136,2086,2265,337,386,57,631%26kwrd%3DChromatin+structure,Embryogenesis,Nuclear+organization,Single-molecule+biophysics">
<img alt="Advertisement" data-test="gpt-advert-fallback-img" height="90" src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature_communications/article&amp;sz=728x90&amp;c=1337038477&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41467-025-56889-5%26doi%3D10.1038/s41467-025-56889-5%26techmeta%3D14,34,63%26subjmeta%3D100,101,136,2086,2265,337,386,57,631%26kwrd%3DChromatin+structure,Embryogenesis,Nuclear+organization,Single-molecule+biophysics" width="728"/></a>
</noscript>
</div>
</div>
</aside>
<header class="c-header" data-header="" data-track-component="nature-150-split-header" id="header" style="border-color:#e63323">
<div class="c-header__row">
<div class="c-header__container">
<div class="c-header__split">
<div class="c-header__logo-container">
<a data-track="click" data-track-action="home" data-track-label="image" href="/ncomms">
<picture class="c-header__logo">
<source media="(min-width: 875px)" srcset="https://media.springernature.com/full/nature-cms/uploads/product/ncomms/header-03d2e325c0a02f6df509e5730e9be304.svg">
<img alt="Nature Communications" height="32" src="https://media.springernature.com/full/nature-cms/uploads/product/ncomms/header-7001f06bc3fe2437048388e9f2f44215.svg"/>
</source></picture>
</a>
</div>
<ul class="c-header__menu c-header__menu--global">
<li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
<a class="c-header__link" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link" href="https://www.nature.com/siteindex">
<span>View all journals</span>
</a>
</li>
<li class="c-header__item c-header__item--padding c-header__item--pipe">
<a class="c-header__link c-header__link--search" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" href="#search-menu">
<svg aria-hidden="true" focusable="false" height="22" role="img" viewbox="0 0 18 18" width="22" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
</a>
</li>
<li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
<a class="c-header__link eds-c-header__link" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41467-025-56889-5" id="identity-account-widget"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
</li>
</ul>
</div>
</div>
</div>
<div class="c-header__row">
<div class="c-header__container" data-test="navigation-row">
<div class="c-header__split">
<ul class="c-header__menu c-header__menu--journal">
<li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
<a class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" href="#explore">
<span><span class="c-header__show-text">Explore</span> content</span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--dropdown-menu">
<a class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" href="#about-the-journal">
<span>About <span class="c-header__show-text">the journal</span></span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
<a class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" href="#publish-with-us">
<span>Publish <span class="c-header__show-text">with us</span></span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
</ul>
<ul class="c-header__menu c-header__menu--hide-lg-max">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (desktop site header)" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D264%26journal-link%3Dhttps%253A%252F%252Fwww.nature.com%252Fncomms%252F" rel="nofollow">
<span>Sign up for alerts</span><svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--pipe">
<a class="c-header__link" data-track="click" data-track-action="rss feed" data-track-label="link" href="https://www.nature.com/ncomms.rss">
<span>RSS feed</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</header>
<nav aria-label="breadcrumbs" class="u-mb-16">
<div class="u-container">
<ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
<li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature" href="/" itemprop="item"><span itemprop="name">nature</span></a><meta content="1" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature communications" href="/ncomms" itemprop="item"><span itemprop="name">nature communications</span></a><meta content="2" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles" href="/ncomms/articles?type=article" itemprop="item"><span itemprop="name">articles</span></a><meta content="3" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
<span itemprop="name">article</span><meta content="4" itemprop="position"/></li>
</ol>
</div>
</nav>
</div>
<div class="u-container u-mt-32 u-mb-32 u-clearfix" data-component="article-container" data-container-type="article" id="content">
<main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
<div aria-hidden="true" class="c-context-bar u-hide" data-context-bar="" data-test="context-bar">
<div class="c-context-bar__container u-container" data-track-context="sticky banner">
<div class="c-context-bar__title">
                            Increasingly efficient chromatin binding of cohesin and CTCF supports chromatin architecture formation during zebrafish embryogenesis
                        </div>
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41467-025-56889-5.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
</div>
<article lang="en">
<div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
<div class="c-pdf-container" data-track-context="article body">
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41467-025-56889-5.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
</div>
<div class="c-article-header">
<header>
<ul class="c-article-identifiers" data-test="article-identifier">
<li class="c-article-identifiers__item" data-test="article-category">Article</li>
<li class="c-article-identifiers__item">
<a class="u-color-open-access" data-test="open-access" data-track="click" data-track-action="open access" data-track-label="link" href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research">Open access</a>
</li>
<li class="c-article-identifiers__item">Published: <time datetime="2025-02-21">21 February 2025</time></li>
</ul>
<h1 class="c-article-title" data-article-title="" data-test="article-title">Increasingly efficient chromatin binding of cohesin and CTCF supports chromatin architecture formation during zebrafish embryogenesis</h1>
<ul class="c-article-author-list c-article-author-list--short" data-component-authors-activator="authors-list" data-test="authors-list"><li class="c-article-author-list__item"><a data-author-popup="auth-Jonas-Co_mann-Aff1-Aff4" data-author-search="Coßmann, Jonas" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jonas-Co_mann-Aff1-Aff4">Jonas Coßmann</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup><sup class="u-js-hide"> <a href="#nAff4">nAff4</a></sup>, </li><li class="c-article-author-list__item"><a data-author-popup="auth-Pavel_I_-Kos-Aff2-Aff5" data-author-search="Kos, Pavel I." data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Pavel_I_-Kos-Aff2-Aff5">Pavel I. Kos</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-3625-0939"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3625-0939</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup><sup class="u-js-hide"> <a href="#nAff5">nAff5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Vassiliki-Varamogianni_Mamatsi-Aff3" data-author-search="Varamogianni-Mamatsi, Vassiliki" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Vassiliki-Varamogianni_Mamatsi-Aff3">Vassiliki Varamogianni-Mamatsi</a><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Devin_S_-Assenheimer-Aff1-Aff4" data-author-search="Assenheimer, Devin S." data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Devin_S_-Assenheimer-Aff1-Aff4">Devin S. Assenheimer</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup><sup class="u-js-hide"> <a href="#nAff4">nAff4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Tobias_A_-Bischof-Aff1-Aff4" data-author-search="Bischof, Tobias A." data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Tobias_A_-Bischof-Aff1-Aff4">Tobias A. Bischof</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0009-0000-0201-7373"><span class="u-visually-hidden">ORCID: </span>orcid.org/0009-0000-0201-7373</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup><sup class="u-js-hide"> <a href="#nAff4">nAff4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Timo-Kuhn-Aff1" data-author-search="Kuhn, Timo" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Timo-Kuhn-Aff1">Timo Kuhn</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Thomas-Vomhof-Aff1" data-author-search="Vomhof, Thomas" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Thomas-Vomhof-Aff1">Thomas Vomhof</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-0194-7186"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-0194-7186</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Argyris-Papantonis-Aff3" data-author-search="Papantonis, Argyris" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Argyris-Papantonis-Aff3">Argyris Papantonis</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0001-7551-1073"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7551-1073</a></span><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Luca-Giorgetti-Aff2" data-author-search="Giorgetti, Luca" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Luca-Giorgetti-Aff2">Luca Giorgetti</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-9664-9087"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-9664-9087</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup> &amp; </li><li aria-label="Show all 10 authors for this article" class="c-article-author-list__show-more" title="Show all 10 authors for this article">…</li><li class="c-article-author-list__item"><a data-author-popup="auth-J__Christof_M_-Gebhardt-Aff1-Aff4" data-author-search="Gebhardt, J. Christof M." data-corresp-id="c1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-J__Christof_M_-Gebhardt-Aff1-Aff4">J. Christof M. Gebhardt<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-mail-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-1900-600X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1900-600X</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup><sup class="u-js-hide"> <a href="#nAff4">nAff4</a></sup> </li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-down-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg><span>Show authors</span></button>
<p class="c-article-info-details" data-container-section="info">
<a data-test="journal-link" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link" href="/ncomms"><i data-test="journal-title">Nature Communications</i></a>
<b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 16</b>, Article number: <span data-test="article-number">1833</span> (<span data-test="article-publication-year">2025</span>)
            <a class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link" href="#citeas">Cite this article</a>
</p>
<div class="c-article-metrics-bar__wrapper u-clear-both">
<ul class="c-article-metrics-bar u-list-reset">
<li class="c-article-metrics-bar__item" data-test="access-count">
<p class="c-article-metrics-bar__count">3001 <span class="c-article-metrics-bar__label">Accesses</span></p>
</li>
<li class="c-article-metrics-bar__item" data-test="altmetric-score">
<p class="c-article-metrics-bar__count">9 <span class="c-article-metrics-bar__label">Altmetric</span></p>
</li>
<li class="c-article-metrics-bar__item">
<p class="c-article-metrics-bar__details"><a data-track="click" data-track-action="view metrics" data-track-label="link" href="/articles/s41467-025-56889-5/metrics" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
</li>
</ul>
</div>
</header>
<div class="u-js-hide" data-component="article-subject-links">
<h3 class="c-article__sub-heading">Subjects</h3>
<ul class="c-article-subject-list">
<li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/chromatin-structure">Chromatin structure</a></li><li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/embryogenesis">Embryogenesis</a></li><li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/nuclear-organization">Nuclear organization</a></li><li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/single-molecule-biophysics">Single-molecule biophysics</a></li>
</ul>
</div>
</div>
<div class="c-article-body">
<section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>The three-dimensional folding of chromosomes is essential for nuclear functions such as DNA replication and gene regulation. The emergence of chromatin architecture is thus an important process during embryogenesis. To shed light on the molecular and kinetic underpinnings of chromatin architecture formation, we characterized biophysical properties of cohesin and CTCF binding to chromatin and their changes upon cofactor depletion using single-molecule imaging in live developing zebrafish embryos. We found that chromatin-bound fractions of both cohesin and CTCF increased significantly between the 1000-cell and shield stages, which we could explain through changes in both their association and dissociation rates. Moreover, increasing binding of cohesin restricted chromatin motion, potentially via loop extrusion, and showed distinct stage-dependent nuclear distribution. Polymer simulations with experimentally derived parameters recapitulated the experimentally observed gradual emergence of chromatin architecture. Our findings reveal molecular kinetics underlying chromatin architecture formation during zebrafish embryogenesis.</p></div></div></section>
<section aria-labelledby="inline-recommendations" class="c-article-recommendations" data-title="Inline Recommendations" data-track-component="inline-recommendations">
<h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
<div class="c-article-recommendations-list">
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-020-19283-x/MediaObjects/41467_2020_19283_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_1" data-track-action="click recommendations inline - 1" data-track-context="inline recommendations" data-track-label="10.1038/s41467-020-19283-x" href="https://www.nature.com/articles/s41467-020-19283-x?fromPaywallRec=false" itemprop="url">Molecular basis of CTCF binding polarity in genome folding
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__access-type">Open access</span>
<span class="c-article-meta-recommendations__date">05 November 2020</span>
</div>
</div>
</article>
</div>
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-022-35444-6/MediaObjects/41467_2022_35444_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_2" data-track-action="click recommendations inline - 2" data-track-context="inline recommendations" data-track-label="10.1038/s41467-022-35444-6" href="https://www.nature.com/articles/s41467-022-35444-6?fromPaywallRec=false" itemprop="url">Functional crosstalk between the cohesin loader and chromatin remodelers
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__access-type">Open access</span>
<span class="c-article-meta-recommendations__date">13 December 2022</span>
</div>
</div>
</article>
</div>
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-020-16394-3/MediaObjects/41467_2020_16394_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_3" data-track-action="click recommendations inline - 3" data-track-context="inline recommendations" data-track-label="10.1038/s41467-020-16394-3" href="https://www.nature.com/articles/s41467-020-16394-3?fromPaywallRec=false" itemprop="url">Nucleoporin 153 links nuclear pore complex to chromatin architecture by mediating CTCF and cohesin binding
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__access-type">Open access</span>
<span class="c-article-meta-recommendations__date">25 May 2020</span>
</div>
</div>
</article>
</div>
</div>
</section>
<script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1740764633,
                        embedded_user: 'null'
                    }
                });
            </script>
<div class="main-content">
<section data-title="Introduction"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Introduction</h2><div class="c-article-section__content" id="Sec1-content"><p>The three-dimensional organization of chromatin is important for nuclear functions such as DNA replication and gene regulation. In interphase, the organizational structures span several hierarchical levels, from chromosome territories<sup><a aria-label="Reference 1" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR1" id="ref-link-section-d21244865e599" title="Cremer, T. &amp; Cremer, M. Chromosome territories. Cold Spring Harb. Perspect. Biol. 2, a003889 (2010).">1</a></sup> to compartments<sup><a aria-label="Reference 2" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR2" id="ref-link-section-d21244865e603" title="Lieberman-Aiden, E. et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. Science 326, 289–293 (2009).">2</a>,<a aria-label="Reference 3" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR3" id="ref-link-section-d21244865e606" title="Zhang, Y. et al. Spatial organization of the mouse genome and its role in recurrent chromosomal translocations. Cell 148, 908–921 (2012).">3</a></sup>, topologically associating domains (TADs)<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR4" id="ref-link-section-d21244865e610" title="Dixon, J. R. et al. Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature 485, 376–380 (2012).">4</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR5" id="ref-link-section-d21244865e610_1" title="Nora, E. P. et al. Spatial partitioning of the regulatory landscape of the X-inactivation centre. Nature 485, 381–385 (2012).">5</a>,<a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR6" id="ref-link-section-d21244865e613" title="Rao, S. S. P. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell 159, 1665–1680 (2014).">6</a></sup>, and DNA loops<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR6" id="ref-link-section-d21244865e617" title="Rao, S. S. P. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell 159, 1665–1680 (2014).">6</a>,<a aria-label="Reference 7" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR7" id="ref-link-section-d21244865e620" title="Sanyal, A., Lajoie, B. R., Jain, G. &amp; Dekker, J. The long-range interaction landscape of gene promoters. Nature 489, 109–113 (2012).">7</a></sup> to clutched and separated nucleosomes<sup><a aria-label="Reference 8" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR8" id="ref-link-section-d21244865e624" title="Ricci, M. A., Manzo, C., García-Parajo, M. F., Lakadamyali, M. &amp; Cosma, M. P. Chromatin fibers are formed by heterogeneous groups of nucleosomes in vivo. Cell 160, 1145–1158 (2015).">8</a>,<a aria-label="Reference 9" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR9" id="ref-link-section-d21244865e627" title="Ou, H. D. et al. ChromEMT: Visualizing 3D chromatin structure and compaction in interphase and mitotic cells. Science 357, 139–148 (2017).">9</a></sup>. TADs and loops contribute to gene regulation by bringing together or insulating enhancers and promoters. They are formed by the architectural protein cohesin through loop extrusion and barrier-forming proteins such as the CCCTC-binding factor (CTCF)<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR10" id="ref-link-section-d21244865e632" title="Fudenberg, G. et al. Formation of chromosomal domains by loop extrusion. Cell Rep. 15, 2038–2049 (2016).">10</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR11" id="ref-link-section-d21244865e632_1" title="Splinter, E. et al. CTCF mediates long-range chromatin looping and local histone modification in the β-globin locus. Genes Dev. 20, 2349–2354 (2006).">11</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR12" id="ref-link-section-d21244865e632_2" title="Rao, S. S. P. et al. Cohesin loss eliminates all loop domains. Cell 171, 305–320.e24 (2017).">12</a>,<a aria-label="Reference 13" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR13" id="ref-link-section-d21244865e635" title="Sanborn, A. L. et al. Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes. Proc. Natl Acad. Sci. USA 112, E6456–E6465 (2015).">13</a></sup>. Moreover, concomitantly with replication, cohesin ensures the cohesion of sister chromatids<sup><a aria-label="Reference 14" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR14" id="ref-link-section-d21244865e639" title="Peters, J. M. &amp; Nishiyama, T. Sister chromatid cohesion. Cold Spring Harb. Perspect. Biol. 4, a011130 (2012).">14</a>,<a aria-label="Reference 15" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR15" id="ref-link-section-d21244865e642" title="Nasmyth, K. &amp; Haering, C. H. Cohesin: Its roles and mechanisms. Annu. Rev. Genet. 43, 525–558 (2009).">15</a></sup>.</p><p>Cohesin is a multi-subunit protein complex that forms a ring-like core complex through the ATPases Smc1/3 and Rad21<sup><a aria-label="Reference 16" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR16" id="ref-link-section-d21244865e649" title="Lee, H., Noh, H. &amp; Ryu, J.-K. Structure-function relationships of SMC protein complexes for DNA loop extrusion. Biodesign 9, 1–13 (2021).">16</a></sup>. Its ATPase activity and, thereby, the loop extrusion function of the core complex depend on Mau2-Nibpl<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR17" id="ref-link-section-d21244865e653" title="Davidson, I. F. et al. DNA loop extrusion by human cohesin. Science 366, 1338–1345 (2019).">17</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR18" id="ref-link-section-d21244865e653_1" title="Kim, Y., Shi, Z., Zhang, H., Finkelstein, I. J. &amp; Yu, H. Human cohesin compacts DNA by loop extrusion. Science 366, 1345–1349 (2019).">18</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR19" id="ref-link-section-d21244865e653_2" title="Petela, N. J. et al. Scc2 is a potent activator of cohesin’s ATPase that promotes loading by binding Scc1 without Pds5. Mol. Cell 70, 1134–1148.e7 (2018).">19</a>,<a aria-label="Reference 20" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR20" id="ref-link-section-d21244865e656" title="Rhodes, J., Mazza, D., Nasmyth, K. &amp; Uphoff, S. Scc2/Nipbl hops between chromosomal cohesin rings after loading. Elife 6, 1–20 (2017).">20</a></sup>, while Wapl determines the release of cohesin from DNA<sup><a aria-label="Reference 21" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR21" id="ref-link-section-d21244865e660" title="Kueng, S. et al. Wapl controls the dynamic association of cohesin with chromatin. Cell 127, 955–967 (2006).">21</a>,<a aria-label="Reference 22" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR22" id="ref-link-section-d21244865e663" title="Haarhuis, J. H. I. et al. The cohesin release factor WAPL restricts chromatin loop extension. Cell 169, 693–707.e14 (2017).">22</a></sup>. For the cohesion function, a fraction of cohesin is stabilized on chromatin through Sororin (Cdca5), which antagonizes the activity of Wapl<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR23" id="ref-link-section-d21244865e667" title="Rankin, S., Ayad, N. G. &amp; Kirschner, M. W. Sororin, a substrate of the anaphase-promoting complex, is required for sister chromatid cohesion in vertebrates. Mol. Cell 18, 185–200 (2005).">23</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR24" id="ref-link-section-d21244865e667_1" title="Ladurner, R. et al. Sororin actively maintains sister chromatid cohesion. EMBO J. 35, 635–653 (2016).">24</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR25" id="ref-link-section-d21244865e667_2" title="Nishiyama, T. et al. Sororin mediates sister chromatid cohesion by antagonizing Wapl. Cell 143, 737–749 (2010).">25</a>,<a aria-label="Reference 26" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR26" id="ref-link-section-d21244865e670" title="Schmitz, J., Watrin, E., Lénárt, P., Mechtler, K. &amp; Peters, J. M. Sororin is required for stable binding of cohesin to chromatin and for sister chromatid cohesion in interphase. Curr. Biol. 17, 630–636 (2007).">26</a></sup>. The exact extrusion mechanism of cohesin-mediated loop extrusion is still unclear. Pieces of evidence point towards loading of the cohesin complex on the DNA, followed by cohesin reeling in DNA symmetrically or by periodically switching between one-sided extrusion<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR27" id="ref-link-section-d21244865e674" title="Corsi, F., Rusch, E. &amp; Goloborodko, A. Loop extrusion rules: the next generation. Curr. Opin. Genet. Dev. 81, 102061 (2023).">27</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR28" id="ref-link-section-d21244865e674_1" title="Davidson, I. F. &amp; Peters, J. M. Genome folding through loop extrusion by SMC complexes. Nat. Rev. Mol. Cell Biol. 22, 445–464 (2021).">28</a>,<a aria-label="Reference 29" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR29" id="ref-link-section-d21244865e677" title="Kim, E., Barth, R. &amp; Dekker, C. Looping the genome with SMC complexes. Annu. Rev. Biochem. 92, 15–41 (2023).">29</a></sup>. CTCF binds in an orientation-dependent manner to specific DNA motifs, often forming a TAD boundary<sup><a aria-label="Reference 6" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR6" id="ref-link-section-d21244865e682" title="Rao, S. S. P. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. Cell 159, 1665–1680 (2014).">6</a>,<a aria-label="Reference 30" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR30" id="ref-link-section-d21244865e685" title="Vietri Rudan, M. et al. Comparative Hi-C reveals that CTCF underlies evolution of chromosomal domain architecture. Cell Rep. 10, 1297–1309 (2015).">30</a>,<a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR31" id="ref-link-section-d21244865e688" title="de Wit, E. et al. CTCF Binding polarity determines chromatin looping. Mol. Cell 60, 676–684 (2015).">31</a></sup>, and guides the formation of regulatory DNA loops by stalling the loop-extrusion process upon direct interaction with cohesin<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR32" id="ref-link-section-d21244865e692" title="Pugacheva, E. M. et al. CTCF mediates chromatin looping via N-terminal domain-dependent cohesin retention. Proc. Natl Acad. Sci. USA 117, 2020–2031 (2020).">32</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR33" id="ref-link-section-d21244865e692_1" title="Nora, E. P. et al. Molecular basis of CTCF binding polarity in genome folding. Nat. Commun. 11, 5612 (2020).">33</a>,<a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR34" id="ref-link-section-d21244865e695" title="Li, Y. et al. The structural basis for cohesin–CTCF-anchored loops. Nature 578, 472–476 (2020).">34</a></sup>.</p><p>Chromatin architecture in interphase is absent in zygotes of animals as diverse as flies, zebrafish, and humans<sup><a aria-label="Reference 35" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR35" id="ref-link-section-d21244865e702" title="Vallot, A. &amp; Tachibana, K. The emergence of genome architecture and zygotic genome activation. Curr. Opin. Cell Biol. 64, 50–57 (2020).">35</a></sup>, but is gradually reestablished during early embryo development<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR36" id="ref-link-section-d21244865e706" title="Du, Z. et al. Allelic reprogramming of 3D chromatin architecture during early mammalian development. Nature 547, 232–235 (2017).">36</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR37" id="ref-link-section-d21244865e706_1" title="Ke, Y. et al. 3D Chromatin structures of mature gametes and structural reprogramming during mammalian Embryogenesis. Cell 170, 367–381.e20 (2017).">37</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR38" id="ref-link-section-d21244865e706_2" title="Chen, X. et al. Key role for CTCF in establishing chromatin structure in human embryos. Nature 576, 306–310 (2019).">38</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR39" id="ref-link-section-d21244865e706_3" title="Hug, C. B., Grimaldi, A. G., Kruse, K. &amp; Vaquerizas, J. M. Chromatin architecture emerges during zygotic genome activation independent of transcription. Cell 169, 216–228.e19 (2017).">39</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR40" id="ref-link-section-d21244865e706_4" title="Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. Genome Res. 31, 981–994 (2021).">40</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR41" id="ref-link-section-d21244865e706_5" title="Kaaij, L. J. T., van der Weide, R. H., Ketting, R. F. &amp; de Wit, E. Systemic loss and gain of Chromatin architecture throughout Zebrafish development. Cell Rep. 24, 1–10.e4 (2018).">41</a>,<a aria-label="Reference 42" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR42" id="ref-link-section-d21244865e709" title="Nakamura, R. et al. CTCF looping is established during gastrulation in medaka embryos. Genome Res. 31, 968–980 (2021).">42</a></sup>. In Zebrafish embryos, the new formation of chromatin architecture starts around the 1000-cell stage, concurrent with zygotic genome activation (ZGA), when the embryo begins transcribing its own genome and becomes independent of maternal RNA and protein<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR40" id="ref-link-section-d21244865e713" title="Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. Genome Res. 31, 981–994 (2021).">40</a></sup>.</p><p>While important insight on the mechanistic underpinnings of cohesin-mediated loop extrusion and CTCF, including the interaction kinetics with chromatin and the cellular distribution have been obtained in vitro and in cells<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR17" id="ref-link-section-d21244865e720" title="Davidson, I. F. et al. DNA loop extrusion by human cohesin. Science 366, 1338–1345 (2019).">17</a>,<a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR18" id="ref-link-section-d21244865e723" title="Kim, Y., Shi, Z., Zhang, H., Finkelstein, I. J. &amp; Yu, H. Human cohesin compacts DNA by loop extrusion. Science 366, 1345–1349 (2019).">18</a>,<a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR34" id="ref-link-section-d21244865e726" title="Li, Y. et al. The structural basis for cohesin–CTCF-anchored loops. Nature 578, 472–476 (2020).">34</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR43" id="ref-link-section-d21244865e729" title="Su, Q. P. et al. Superresolution imaging reveals spatiotemporal propagation of human replication foci mediated by CTCF-organized chromatin structures. Proc. Natl Acad. Sci. USA 117, 15036–15046 (2020).">43</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR44" id="ref-link-section-d21244865e729_1" title="Davidson, I. F. et al. CTCF is a DNA-tension-dependent barrier to cohesin-mediated loop extrusion. Nature 616, 822–827 (2023).">44</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR45" id="ref-link-section-d21244865e729_2" title="Holzmann, J. et al. Absolute quantification of cohesin, CTCF and their regulators in human cells. Elife 8, 1–31 (2019).">45</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR46" id="ref-link-section-d21244865e729_3" title="Hansen, A. S., Pustova, I., Cattoglio, C., Tjian, R. &amp; Darzacq, X. CTCF and cohesin regulate chromatin loop stability with distinct dynamics. Elife 6, e25776 (2017).">46</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR47" id="ref-link-section-d21244865e729_4" title="Wutz, G. et al. Topologically associating domains and chromatin loops depend on cohesin and are regulated by CTCF, WAPL, and PDS5 proteins. EMBO J. 36, 3573–3599 (2017).">47</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR48" id="ref-link-section-d21244865e729_5" title="Agarwal, H., Reisser, M., Wortmann, C. &amp; Gebhardt, J. C. M. Direct observation of cell-cycle-dependent interactions between CTCF and Chromatin. Biophys. J. 112, 2051–2055 (2017).">48</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR49" id="ref-link-section-d21244865e729_6" title="Hansen, A. S., Amitai, A., Cattoglio, C., Tjian, R. &amp; Darzacq, X. Guided nuclear exploration increases CTCF target search efficiency. Nat. Chem. Biol. 16, 257–266 (2020).">49</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR50" id="ref-link-section-d21244865e729_7" title="Gerlich, D., Koch, B., Dupeux, F., Peters, J. M. &amp; Ellenberg, J. Live-cell imaging reveals a stable cohesin-chromatin interaction after but not before DNA replication. Curr. Biol. 16, 1571–1578 (2006).">50</a>,<a aria-label="Reference 51" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR51" id="ref-link-section-d21244865e732" title="Golfier, S., Quail, T., Kimura, H. &amp; Brugués, J. Cohesin and condensin extrude DNA loops in a cell-cycle dependent manner. Elife 9, 1–34 (2020).">51</a></sup>, detailed information on the temporal and spatial behavior in vivo is missing. In particular, it is unclear how frequently and for how long cohesin and CTCF interact with chromatin, how cohesin action affects chromatin motion, and how these parameters support the formation of chromatin architecture during embryo development.</p><p>Here, we characterized the binding of cohesin and CTCF to chromatin during early embryogenesis using single molecule tracking in live developing zebrafish embryos and polymer simulations. We measured the fraction of cohesin and CTCF molecules associated with chromatin, their association and dissociation kinetics, and the motion of cohesin-bound chromatin at multiple developmental stages. Our experiments and simulations revealed mechanistic insight into these architectural proteins that inform the process of chromatin architecture formation in zebrafish embryogenesis.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">The fraction of chromatin-bound cohesin and CTCF increases during development</h3><p>To be able to visualize cohesin and CTCF in zebrafish embryos, we fused the cohesin subunit Rad21 and CTCF to a HaloTag (HT) (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1a</a> and Methods). We injected mRNA encoding for either fusion protein together with mRNA encoding for the nuclear membrane marker GFP-lap2b into 1-cell stage embryos<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR52" id="ref-link-section-d21244865e755" title="Reisser, M. et al. Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development. Nat. Commun. 9, 5218 (2018).">52</a></sup> (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1b</a> and Methods), which resulted in ectopic protein expression of ca. (11.4 ± 2.6)% for HT-Rad21 and (24.0 ± 2.7)% for HT-CTCF of endogenous and maternally<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR40" id="ref-link-section-d21244865e762" title="Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. Genome Res. 31, 981–994 (2021).">40</a>,<a aria-label="Reference 53" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR53" id="ref-link-section-d21244865e765" title="Franke, M. et al. CTCF knockout in zebrafish induces alterations in regulatory landscapes and developmental gene expression. Nat. Commun. 12, 5415 (2021).">53</a>,<a aria-label="Reference 54" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR54" id="ref-link-section-d21244865e768" title="Horsfield, J. A. et al. Cohesin-dependent regulation of Runx genes. Development 134, 2639–2649 (2007).">54</a></sup> provided protein in shield-stage embryos (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">1</a>). We confirmed the incorporation of HT-Rad21 into the cohesin complex through co-immunoprecipitation of the N-terminally interacting Smc3 protein<sup><a aria-label="Reference 55" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR55" id="ref-link-section-d21244865e776" title="Haering, C. H., Löwe, J., Hochwagen, A. &amp; Nasmyth, K. Molecular architecture of SMC proteins and the Yeast Cohesin Complex C-terminal domains forming a head would be part of. Mol. Cell 9, 773–788 (2002).">55</a></sup> (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">2</a>). We further confirmed chromatin association of HT-Rad21 and HT-CTCF by subcellular protein fractionation (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">3</a>). The HT-rad21 and HT-ctcf injected embryos developed normally during our imaging period and up to 48 hpf (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">4</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">5</a>). Prior to imaging, we labeled HT-tagged proteins with HT-dye JF549<sup><a aria-label="Reference 56" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR56" id="ref-link-section-d21244865e792" title="Grimm, J. B. et al. A general method to improve fluorophores for live-cell and single-molecule microscopy. Nat. Methods 12, 244–250 (2015).">56</a></sup> and transferred them onto glass-bottom imaging dishes. Illumination of embryos with either 488 nm or 561 nm laser light in a reflected light-sheet microscope<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR52" id="ref-link-section-d21244865e797" title="Reisser, M. et al. Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development. Nat. Commun. 9, 5218 (2018).">52</a>,<a aria-label="Reference 57" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR57" id="ref-link-section-d21244865e800" title="Kuhn, T. et al. Single-molecule tracking of Nodal and Lefty in live zebrafish embryos supports hindered diffusion model. Nat. Commun. 13, 6101 (2022).">57</a></sup> enabled detection the nuclear membrane and the signal of individual HT-Rad21 or HT-CTCF molecules in the nucleus.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Cohesin and CTCF bind chromatin increasingly more efficient during zebrafish development." id="figure-1"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig1">Fig. 1: Cohesin and CTCF bind chromatin increasingly more efficient during zebrafish development.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-025-56889-5/figures/1" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig1_HTML.png?as=webp" type="image/webp"><img alt="figure 1" aria-describedby="Fig1" height="601" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig1_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b> Scheme of the cohesin complex. <b>b</b> Scheme outlining the workflow of zebrafish measurements. <b>c</b> Top: Scheme of continuous illumination. Bottom: Example single-molecule tracks of the 64-cell stage (bright pink) and shield stage (dark pink). Scale bar: 1 µm. Fractions of a three-component diffusion model fitted to jump distance distributions of (<b>d</b>) HT-Rad21 and (<b>e</b>) HT-CTCF. Colors indicate stages of development (64-, 128-, 256-, 512-, 1k-cell, high, oblong, sphere, shield). Bars represent mean values ± s.d. from 500 resamplings using 80% of randomly selected jump distances. For fits and diffusion coefficients see Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">6</a>-<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">8</a> and statistics in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">1</a>. <b>f</b> Scheme of interlaced time-lapse microscopy (ITM). Molecules are classified into different binding classes. Long (green): tracks surviving at least two long dark times (8.2 s). Short (red): Tracks surviving one short dark time (0.2 s). Spot (yellow): Molecules only detected in a single frame. Top: Example nuclei of (<b>g</b>) HT-Rad21 and (<b>h</b>) HT-CTCF ITM movies with tracks colored according to the binding classes in panel (<b>f</b>). Grey tracks survived only one long dark time. Scale bar: 5 µm. Bottom: Fractions of short and long binding events of (<b>g</b>) HT-Rad21 (WT) or HT-Rad21-3x mutant and (<b>h</b>) HT-CTCF (WT) or HT-CTCF-ΔZF4-7 mutant. Data represent mean ± s.d. of movie-wise determined fractions. Insets show zooms into the respective graphs. Lines serve as guides to the eye. Statistics and p-Values are provided in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">2</a>–<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">7</a>. Raw data plots are provided in Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">9</a>. <b>i</b> Fractions of short (left) and long (right) binding events of HT-Rad21 (WT) in the presence of morpholinos (MO) or RNA-polymerase inhibitors in the shield stage. Data represent mean ± s.d. of movie-wise determined fractions. All developmental stages are shown in Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">14</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">16</a>. P-values were calculated using two-sided Multiple Mann-Whitney test with a false discovery rate set to 1% (Supplementary Tables <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">8</a>, <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">9</a>). Source data are provided as a Source Data file for Fig. 1d, e, <b>g</b>–<b>i</b>.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 1" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure1 Full size image" data-track-label="button" href="/articles/s41467-025-56889-5/figures/1" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>We characterized the overall mobility of HT-Rad21 and HT-CTCF in the developmental stages between the 64-cell stage and the shield stage comprising the period of ZGA. In every stage, both species showed mobile and immobile molecules (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1c</a>, Supplementary Movie <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM3">1</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM4">2</a> and Methods). To measure the fraction of immobile molecules, we recorded continuous movies of single HT-tagged molecules with 11.7 ms frame rate (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1c</a>). We analyzed movies using TrackIt<sup><a aria-label="Reference 58" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR58" id="ref-link-section-d21244865e913" title="Kuhn, T., Hettich, J., Davtyan, R. &amp; Gebhardt, J. C. M. Single molecule tracking and analysis framework including theory-predicted parameter settings. Sci. Rep. 11, 1–12 (2021).">58</a></sup>, compiled cumulative histograms of jump distances of tracked molecules (Supplementary Figs. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">6</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">7</a>) and analyzed these using a model comprising three components of Brownian diffusion (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">8</a> and Methods)<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR59" id="ref-link-section-d21244865e927" title="Mazza, D., Abernathy, A., Golob, N., Morisaki, T. &amp; McNally, J. G. A benchmark for chromatin binding measurements in live cells. Nucleic Acids Res. 40, 1–13 (2012).">59</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR60" id="ref-link-section-d21244865e927_1" title="Hansen, A. S. et al. Robust model-based analysis of single-particle tracking experiments with spot-on. Elife 7, 1–33 (2018).">60</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR61" id="ref-link-section-d21244865e927_2" title="Gebhardt, J. C. M. et al. Single-molecule imaging of transcription factor binding to DNA in live mammalian cells. Nat. Methods 10, 421–426 (2013).">61</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR62" id="ref-link-section-d21244865e927_3" title="Speil, J. et al. Activated STAT1 transcription factors conduct distinct saltatory movements in the cell nucleus. Biophys. J. 101, 2592–2600 (2011).">62</a>,<a aria-label="Reference 63" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR63" id="ref-link-section-d21244865e930" title="Izeddin, I. et al. Single-molecule tracking in live cells reveals distinct target-search strategies of transcription factors in the nucleus. Elife 2014, e02230 (2014).">63</a></sup>. The first, slow diffusion component with fraction <i>A</i><sub><i>1</i></sub> represents immobile molecules presumably bound to chromatin that show apparent Brownian diffusion mainly due to the limited localization precision of diffraction-limited imaging and slow motion of the underlying chromatin. The two other diffusion components with fractions <i>A</i><sub><i>2</i></sub> and <i>A</i><sub><i>3</i></sub> are used to approximate the anomalous diffusion that has been observed for mobile molecules in the nucleus<sup><a aria-label="Reference 59" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR59" id="ref-link-section-d21244865e953" title="Mazza, D., Abernathy, A., Golob, N., Morisaki, T. &amp; McNally, J. G. A benchmark for chromatin binding measurements in live cells. Nucleic Acids Res. 40, 1–13 (2012).">59</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR64" id="ref-link-section-d21244865e956" title="Shaban, H. A., Barth, R. &amp; Recoules, L. &amp; Bystricky, K. Hi-D: Nanoscale mapping of nuclear dynamics in single living cells. Genome Biol. 21, 1–21 (2020).">64</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR65" id="ref-link-section-d21244865e956_1" title="Veith, R. et al. Balbiani ring mRNPs diffuse through and bind to clusters of large intranuclear molecular structures. Biophys. J. 99, 2676–2685 (2010).">65</a>,<a aria-label="Reference 66" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR66" id="ref-link-section-d21244865e959" title="Höfling, F. &amp; Franosch, T. Anomalous transport in the crowded world of biological cells. Rep. Prog. Phys. 76, 046602 (2013).">66</a></sup>. For HT-Rad21, the fraction of immobile, potentially chromatin-bound molecules increased ca. 8.7-fold, and for HT-CTCF 1.5-fold from 64-cell to shield stage embryos. In contrast, the intermediate and fast diffusion fractions, which presumably comprise freely diffusing molecules in the crowded nucleus, remained roughly constant (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1d, e</a>). Thus, HT-Rad21 and HT-CTCF are more and more driven to the chromatin-bound state during early embryo development.</p><h3 class="c-article__sub-heading" id="Sec4">Long, not short binding of cohesin and CTCF is associated with chromatin binding</h3><p>While analysis of continuous movies reveals the overall bound fraction of molecules, we used an interlaced time-lapse microscopy (ITM) approach to further distinguish chromatin-bound molecules within different binding time classes<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR52" id="ref-link-section-d21244865e974" title="Reisser, M. et al. Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development. Nat. Commun. 9, 5218 (2018).">52</a></sup>. We chose a repetitive ITM illumination pattern comprising 11.7 ms images interlaced with alternating dark times of 0.2 s and 4 s, which enabled us to separate short-bound molecules that survived a 0.2 s dark time from long-bound molecules that survived at least two 4 s dark times (&gt;8 s) while being confined to a small area (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1f</a>, Supplementary Movie <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM5">3</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM6">4</a>). Importantly, due to the Poissonian nature of molecular interactions, this temporal scheme allowed separating binding classes with distinct dissociation kinetics<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR52" id="ref-link-section-d21244865e987" title="Reisser, M. et al. Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development. Nat. Commun. 9, 5218 (2018).">52</a></sup>.</p><p>Our ITM approach revealed that the fraction of short-bound molecules of both HT-Rad21 and HT-CTCF increased ca. 1.5-fold from (7.4 ± 1.8)% (cohesin)/(6.2 ± 2.5)% (CTCF) at 64-cell stage to (10.3 ± 3.8)% (cohesin)/(11.4 ± 2.5)% (CTCF) in shield stage embryos (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1g, h</a>, right; Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">9</a>, right). The fraction of long-bound HT-Rad21 molecules increased ca. 21.5-fold from (0.6 ± 0.4)% to (12.9 ± 5.5)% during this time period, while the fraction of long-bound HT-CTCF molecules increased only ca. 3.2-fold from (2.0 ± 1.3)% to (6.2 ± 2.6)% (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1g, h</a>, left; Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">9</a>, left).</p><p>To shed further light onto the nature of the short and long binding classes of HT-Rad21 and HT-CTCF, we performed several control measurements. First, we repeated the ITM experiments with HaloTag alone. For this HT control, short-binding events were up to 4-fold reduced and long-binding events were reduced on average by 98 % of HT-Rad21 or HT-CTCF (Supplementary Figs. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">10</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">11</a>). Second, we imaged an HT-tagged triple mutant of Rad21, HT-Rad21-3x, which cannot form a DNA-binding cohesin complex anymore<sup><a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR46" id="ref-link-section-d21244865e1015" title="Hansen, A. S., Pustova, I., Cattoglio, C., Tjian, R. &amp; Darzacq, X. CTCF and cohesin regulate chromatin loop stability with distinct dynamics. Elife 6, e25776 (2017).">46</a>,<a aria-label="Reference 67" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR67" id="ref-link-section-d21244865e1018" title="Haering, C. H. et al. Structure and stability of cohesin’s Smc1-kleisin interaction. Mol. Cell 15, 951–964 (2004).">67</a>,<a aria-label="Reference 68" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR68" id="ref-link-section-d21244865e1021" title="Sun, Y. et al. RAD21 is the core subunit of the cohesin complex involved in directing genome organization. Genome Biol. 24, 155 (2023).">68</a></sup>. We also imaged an HT-tagged CTCF mutant lacking the intermediate zinc finger domains 4 to 7, HT-CTCF-ΔZF4-7, which impedes binding of CTCF to its core motif<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR69" id="ref-link-section-d21244865e1025" title="Nakahashi, H. et al. A genome-wide Map of CTCF multivalency redefines the CTCF Code. Cell Rep. 3, 1678–1689 (2013).">69</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR70" id="ref-link-section-d21244865e1025_1" title="Hashimoto, H. et al. Structural basis for the versatile and methylation-dependent binding of CTCF to DNA. Mol. Cell 66, 711–720.e3 (2017).">70</a>,<a aria-label="Reference 71" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR71" id="ref-link-section-d21244865e1028" title="Yin, M. et al. Molecular mechanism of directional CTCF recognition of a diverse range of genomic sites. Cell Res. 27, 1365–1377 (2017).">71</a></sup>. We confirmed impaired chromatin binding of HT-Rad21-3x and HT-CTCF-ΔZF4-7 by subcellular protein fractionation (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">12</a>). Both HT-Rad21-3x and HT-CTCF-ΔZF4-7 still exhibited a fraction of molecules in the short binding class (short-bound fraction) comparable to the HT-tagged wild-type proteins, while their fraction in the long binding class (long-bound fraction) was, on average, reduced by 93% (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1g, h</a>; Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">13</a>).</p><p>Since Rad21 interacts with both its N- and C-terminus with the SMC1/3 dimer, we further tested whether the position of the HaloTag affected the bound fraction of Rad21. We thus repeated our ITM measurements with a C-terminally Halo-tagged Rad21 construct (Rad21-HT) until the shield stage (Supplementary Figs. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">14</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">15</a>). Rad21-HT bound was comparable to the N-terminally tagged HT-Rad21 at most developmental stages and showed a similar trend for both the long- and the short-bound fractions. For CTCF, N-terminal tagging has been previously established and was reported to be preferable<sup><a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR46" id="ref-link-section-d21244865e1051" title="Hansen, A. S., Pustova, I., Cattoglio, C., Tjian, R. &amp; Darzacq, X. CTCF and cohesin regulate chromatin loop stability with distinct dynamics. Elife 6, e25776 (2017).">46</a>,<a aria-label="Reference 72" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR72" id="ref-link-section-d21244865e1054" title="Gu, B. et al. Opposing effects of cohesin and transcription on CTCF organization revealed by super-resolution imaging. Mol. Cell 80, 699–711.e7 (2020).">72</a>,<a aria-label="Reference 73" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR73" id="ref-link-section-d21244865e1057" title="Cattoglio, C. et al. Determining cellular CTCF and cohesin abundances to constrain 3D genome models. Elife 8 (2019).">73</a></sup>.</p><p>Overall, our ITM results indicate that the long-bound fractions of HT-Rad21 or HT-CTCF are predominantly associated with chromatin binding, while short-bound fractions are presumably associated with unspecific transient modes of binding to DNA, RNA, proteins, or other biomolecules, or transient spatial confinement. Notably, the percentage of long-bound molecules is significantly increased in the shield stage compared to the 64-cell stage, demonstrating that chromatin binding of architectural proteins is dynamically modulated during development.</p><h3 class="c-article__sub-heading" id="Sec5">Nipbl, CTCF, and RNA Polymerase II influence the long-bound fraction of cohesin</h3><p>Cohesin relies on various cofactors, including Nipbl, which is involved in the establishment of cohesion<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR74" id="ref-link-section-d21244865e1073" title="Zheng, G., Kanchwala, M., Xing, C. &amp; Yu, H. MCM2–7-dependent cohesin loading during s phase promotes sister-chromatid cohesion. Elife 7, 1–25 (2018).">74</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR75" id="ref-link-section-d21244865e1073_1" title="Srinivasan, M., Fumasoni, M., Petela, N. J., Murray, A. &amp; Nasmyth, K. A. Cohesion is established during dna replication utilising chromosome associated cohesin rings as well as those loaded de novo onto nascent DNAs. Elife 9, 1–27 (2020).">75</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR76" id="ref-link-section-d21244865e1073_2" title="Alonso-Gil, D. &amp; Losada, A. NIPBL and cohesin: new take on a classic tale. Trends Cell Biol. 33, 860–871 (2023).">76</a>,<a aria-label="Reference 77" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR77" id="ref-link-section-d21244865e1076" title="Ciosk, R. et al. Cohesin’s binding to chromosomes depends on a separate complex consisting of Scc2 and Scc4 proteins. Mol. Cell 5, 243–254 (2000).">77</a></sup> and supports its ATPase activity<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR17" id="ref-link-section-d21244865e1080" title="Davidson, I. F. et al. DNA loop extrusion by human cohesin. Science 366, 1338–1345 (2019).">17</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR18" id="ref-link-section-d21244865e1080_1" title="Kim, Y., Shi, Z., Zhang, H., Finkelstein, I. J. &amp; Yu, H. Human cohesin compacts DNA by loop extrusion. Science 366, 1345–1349 (2019).">18</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR19" id="ref-link-section-d21244865e1080_2" title="Petela, N. J. et al. Scc2 is a potent activator of cohesin’s ATPase that promotes loading by binding Scc1 without Pds5. Mol. Cell 70, 1134–1148.e7 (2018).">19</a>,<a aria-label="Reference 20" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR20" id="ref-link-section-d21244865e1083" title="Rhodes, J., Mazza, D., Nasmyth, K. &amp; Uphoff, S. Scc2/Nipbl hops between chromosomal cohesin rings after loading. Elife 6, 1–20 (2017).">20</a></sup>, the cohesin unloader Wapl<sup><a aria-label="Reference 21" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR21" id="ref-link-section-d21244865e1087" title="Kueng, S. et al. Wapl controls the dynamic association of cohesin with chromatin. Cell 127, 955–967 (2006).">21</a>,<a aria-label="Reference 78" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR78" id="ref-link-section-d21244865e1090" title="Gandhi, R., Gillespie, P. J. &amp; Hirano, T. Human Wapl is a Cohesin-binding protein that promotes sister-chromatid resolution in mitotic prophase. Curr. Biol. 16, 2406–2417 (2006).">78</a>,<a aria-label="Reference 79" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR79" id="ref-link-section-d21244865e1093" title="Tedeschi, A. et al. Wapl is an essential regulator of chromatin structure and chromosome segregation. Nature 501, 564–568 (2013).">79</a></sup>, and CTCF, which protects cohesin unloading by Wapl<sup><a aria-label="Reference 33" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR33" id="ref-link-section-d21244865e1097" title="Nora, E. P. et al. Molecular basis of CTCF binding polarity in genome folding. Nat. Commun. 11, 5612 (2020).">33</a>,<a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR34" id="ref-link-section-d21244865e1100" title="Li, Y. et al. The structural basis for cohesin–CTCF-anchored loops. Nature 578, 472–476 (2020).">34</a>,<a aria-label="Reference 80" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR80" id="ref-link-section-d21244865e1103" title="Wutz, G. et al. ESCO1 and CTCF enable formation of long chromatin loops by protecting cohesinstag1 from WAPL. Elife 9, 1–33 (2020).">80</a></sup>. We hypothesized that reducing the expression level of Nipbl or CTCF should decrease the long-bound fraction of Rad21 but not affect the short-bound, unspecific fraction. We, therefore, repeated our ITM measurements in the presence of established morpholinos (MOs) impeding the expression of either Nipbl or CTCF (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1i</a>)<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR81" id="ref-link-section-d21244865e1111" title="Muto, A., Calof, A. L., Lander, A. D. &amp; Schilling, T. F. Multifactorial origins of heart and gut defects in Nipbl-deficient zebrafish, a model of cornelia de Lange Syndrome. PLoS Biol. 9, e1001181 (2011).">81</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR82" id="ref-link-section-d21244865e1111_1" title="Muto, A. et al. Nipbl and mediator cooperatively regulate gene expression to control limb development. PLoS Genet. 10, e1004671 (2014).">82</a>,<a aria-label="Reference 83" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR83" id="ref-link-section-d21244865e1114" title="Delgado-Olguín, P. et al. CTCF promotes muscle differentiation by modulating the activity of myogenic regulatory factors. J. Biol. Chem. 286, 12483–12494 (2011).">83</a></sup>. We focused on the shield stage since treatment with MOs is expected to be less effective at earlier stages due to the maternal supply of proteins. Moreover, the comparison of large bound fractions is more robust than for the small bound fractions observed at earlier stages. We confirmed normal development of nipbl-MO injected embryos until shield-stage and severe developmental defects at 48 hpf (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">16</a>), as reported previously<sup><a aria-label="Reference 81" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR81" id="ref-link-section-d21244865e1121" title="Muto, A., Calof, A. L., Lander, A. D. &amp; Schilling, T. F. Multifactorial origins of heart and gut defects in Nipbl-deficient zebrafish, a model of cornelia de Lange Syndrome. PLoS Biol. 9, e1001181 (2011).">81</a></sup>. Upon injection of ctcf-MO, the expression level of endogenous CTCF protein was reduced to (15.6 ± 2.0)% at the time point of shield stage in untreated embryos (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">17</a>). Moreover, we confirmed a developmental delay in shield-stage and lethality at later stages in ctcf-MO injected embryos, as previously reported (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">18</a>)<sup><a aria-label="Reference 53" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR53" id="ref-link-section-d21244865e1131" title="Franke, M. et al. CTCF knockout in zebrafish induces alterations in regulatory landscapes and developmental gene expression. Nat. Commun. 12, 5415 (2021).">53</a>,<a aria-label="Reference 84" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR84" id="ref-link-section-d21244865e1134" title="Meier, M. et al. Cohesin facilitates zygotic genome activation in zebrafish. Dev 145, 1–13 (2018).">84</a></sup>.</p><p>Both injections of nipbl-MO or ctcf-MO significantly reduced the long-bound fraction of HT-Rad21 at the time point of shield stage in untreated embryos, compared to untreated embryos or control-MO injected embryos (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1i</a>, for earlier stages and control-MO measurements, see Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">19</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">20</a>). Short-bound fractions were not changed (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1i</a>). This is expected for nipbl-MO due to disturbed loading and extrusion at reduced Nipbl levels. The presence of ctcf-MO might reduce the bound fraction due to decreased protection of cohesin from Wapl, but we cannot exclude a contribution due to the developmental delay compared to untreated embryos. Together, our measured influence of the cofactors Nipbl and CTCF on HT-Rad21 binding was compatible with their reported effects on cohesin.</p><p>Previous studies suggested that the initial formation of chromatin architecture in <i>Drosophila</i> and mice occurs independent of transcription<sup><a aria-label="Reference 37" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR37" id="ref-link-section-d21244865e1159" title="Ke, Y. et al. 3D Chromatin structures of mature gametes and structural reprogramming during mammalian Embryogenesis. Cell 170, 367–381.e20 (2017).">37</a>,<a aria-label="Reference 39" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR39" id="ref-link-section-d21244865e1162" title="Hug, C. B., Grimaldi, A. G., Kruse, K. &amp; Vaquerizas, J. M. Chromatin architecture emerges during zygotic genome activation independent of transcription. Cell 169, 216–228.e19 (2017).">39</a></sup>. On the other hand, an influence of RNA Polymerase II on the function of cohesin has been reported<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR85" id="ref-link-section-d21244865e1166" title="Neguembor, M. V. et al. Transcription-mediated supercoiling regulates genome folding and loop formation. Mol. Cell 81, 3065–3081.e12 (2021).">85</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR86" id="ref-link-section-d21244865e1166_1" title="Sept, C. E. et al. High-resolution CTCF footprinting reveals impact of chromatin state on cohesin extrusion dynamics. bioRxiv Prepr. Serv. Biol. 
                  https://doi.org/10.1101/2023.10.20.563340
                  
                 (2023).">86</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR87" id="ref-link-section-d21244865e1166_2" title="Zhang, S. et al. RNA polymerase II is required for spatial chromatin reorganization following exit from mitosis. Sci. Adv. 7, 1–14 (2021).">87</a>,<a aria-label="Reference 88" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR88" id="ref-link-section-d21244865e1169" title="Zhang, S., Übelmesser, N., Barbieri, M. &amp; Papantonis, A. Enhancer–promoter contact formation requires RNAPII and antagonizes loop extrusion. Nat. Genet. 55, 832–840 (2023).">88</a></sup>. We therefore tested the impact of RNA Polymerase II, in particular RNA polymerase II elongation, on the binding of cohesin. Therefore, we repeated our ITM measurements in the presence of α-Amanitin or Triptolide (see Methods), which both prevent RNA Polymerase II elongation<sup><a aria-label="Reference 39" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR39" id="ref-link-section-d21244865e1173" title="Hug, C. B., Grimaldi, A. G., Kruse, K. &amp; Vaquerizas, J. M. Chromatin architecture emerges during zygotic genome activation independent of transcription. Cell 169, 216–228.e19 (2017).">39</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR89" id="ref-link-section-d21244865e1176" title="Brueckner, F. &amp; Cramer, P. Structural basis of transcription inhibition by α-amanitin and implications for RNA polymerase II translocation. Nat. Struct. Mol. Biol. 15, 811–818 (2008).">89</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR90" id="ref-link-section-d21244865e1176_1" title="Kaplan, C. D., Larsson, K. M. &amp; Kornberg, R. D. The RNA Polymerase II Trigger Loop functions in substrate selection and is directly targeted by α-Amanitin. Mol. Cell 30, 547–556 (2008).">90</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR91" id="ref-link-section-d21244865e1176_2" title="Bensaude, O. Inhibiting eukaryotic transcription: Which compound to choose? How to evaluate its activity? Transcription 2, 103–108 (2011).">91</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR92" id="ref-link-section-d21244865e1176_3" title="Li, L. et al. Widespread rearrangement of 3D Chromatin organization underlies polycomb-mediated stress-induced silencing. Mol. Cell 58, 216–231 (2015).">92</a>,<a aria-label="Reference 93" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR93" id="ref-link-section-d21244865e1179" title="Shao, W. &amp; Zeitlinger, J. Paused RNA polymerase II inhibits new transcriptional initiation. Nat. Genet. 49, 1045–1051 (2017).">93</a></sup>. As expected, the development of embryos was arrested after ZGA, similar to previous observations<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR94" id="ref-link-section-d21244865e1183" title="Pálfy, M., Schulze, G., Valen, E. &amp; Vastenhouw, N. L. Chromatin accessibility established by Pou5f3, Sox19b and Nanog primes genes for activity during zebrafish genome activation. PLoS Genet. 16, 1–25 (2020).">94</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR95" id="ref-link-section-d21244865e1183_1" title="Lee, M. T. et al. Nanog, Pou5f1 and SoxB1 activate zygotic gene expression during the maternal-to-zygotic transition. Nature 503, 360–364 (2013).">95</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR96" id="ref-link-section-d21244865e1183_2" title="Kane, D. A. et al. The zebrafish epiboly mutants. Development 123, 47–55 (1996).">96</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR97" id="ref-link-section-d21244865e1183_3" title="Huo, J. et al. Triptolide-induced hepatotoxicity via apoptosis and autophagy in zebrafish. J. Appl. Toxicol. 39, 1532–1540 (2019).">97</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR98" id="ref-link-section-d21244865e1183_4" title="Vliegenthart, A. D. B. et al. Characterization of triptolide-induced hepatotoxicity by imaging and transcriptomics in a novel zebrafish model. Toxicol. Sci. 159, 380–391 (2017).">98</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR99" id="ref-link-section-d21244865e1183_5" title="Mishima, Y. &amp; Tomari, Y. Codon usage and 3’ UTR length determine maternal mRNA stability in Zebrafish. Mol. Cell 61, 874–885 (2016).">99</a>,<a aria-label="Reference 100" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR100" id="ref-link-section-d21244865e1186" title="Song, Y. S. et al. Validation, optimization, and application of the Zebrafish developmental toxicity assay for pharmaceuticals under the ICH S5(R3) Guideline. Front. Cell Dev. Biol. 9, 721130 (2021).">100</a></sup> (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">21</a>). Both drugs increased the short- as well as the long-bound fractions of HT-Rad21 at the time point of shield stage in untreated embryos (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1i</a>, for earlier stages, see Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">22</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">23</a>). Increased long binding of HT-Rad21 indicates that elongation of RNA Polymerase II might disturb the binding of cohesin molecules to chromatin.</p><h3 class="c-article__sub-heading" id="Sec6">Cohesin bound long to chromatin shows reduced spatial confinement in the presence of Nipbl</h3><p>It has been shown by monitoring histone mobility and by simulations that the overall mobility of chromatin increases if the expression level of cohesin is reduced<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR101" id="ref-link-section-d21244865e1211" title="Nozaki, T. et al. Dynamic organization of chromatin domains revealed by super-resolution live-cell imaging. Mol. Cell 67, 282–293.e7 (2017).">101</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR102" id="ref-link-section-d21244865e1211_1" title="Ashwin, S. S., Nozaki, T., Maeshima, K. &amp; Sasai, M. Organization of fast and slow chromatin revealed by single-nucleosome dynamics. Proc. Natl Acad. Sci. USA 116, 19939–19944 (2019).">102</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR103" id="ref-link-section-d21244865e1211_2" title="Mach, P. et al. Cohesin and CTCF control the dynamics of chromosome folding. Nat. Genet. 54, 1907–1918 (2022).">103</a>,<a aria-label="Reference 104" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR104" id="ref-link-section-d21244865e1214" title="Gabriele, M. et al. Dynamics of CTCF- and cohesin-mediated chromatin looping revealed by live-cell imaging. Science 376, 476–501 (2022).">104</a></sup>. In our experiments, the DNA binding-deficient mutant HT-Rad21-3x showed predominantly short binding events (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1h</a>), suggesting that long-bound HT-Rad21 mirrors the motion of chromatin. Based on these previous observations, we hypothesized that the long-bound fraction of cohesin contributed to restricting chromatin movement.</p><p>To test our hypothesis, we measured the spatial confinement of short- and long-bound HT-Rad21 in the shield stage. We devised an illumination scheme, time-lapse alternated with continuous intervals (TACO), which combined our ITM scheme to filter for short- or long-bound molecules with periods of ten continuously illuminated images. We analyzed the confinement of HT-Rad21 molecules in short or long-binding classes by quantifying the mean jump distances of tracked molecules (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig2">2a</a>, Supplementary Movies <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM7">5</a>–<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM10">8</a> and Methods). Supporting our hypotheses, we observed that HT-Rad21 bound long to chromatin exhibited significantly shorter mean jump distances of (91 ± 47) nm compared to short-bound HT-Rad21 with (166 ± 73) nm (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig2">2b</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Long-bound cohesin confines chromatin motion and shows distinct nuclear localization." id="figure-2"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig2">Fig. 2: Long-bound cohesin confines chromatin motion and shows distinct nuclear localization.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-025-56889-5/figures/2" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig2_HTML.png?as=webp" type="image/webp"><img alt="figure 2" aria-describedby="Fig2" height="381" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig2_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b> Schemes of time-lapse illumination alternated with continuous intervals (TACO). Molecules are classified in short and long-bound binding classes comparable to ITM (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1f</a>) and analyzed during continuous intervals (dark green and dark red bars, see Methods). <b>b</b> Mean jump distances of HT-Rad21 molecules classified as short bound (circles) and long bound (squares) at different developmental phases. pre-ZGA (red): 64-, 128-, 256, 512-cell stages pooled; post-ZGA (green): high, oblong, sphere stages pooled; shield stage (blue) and shield stage with the addition of nibpl-morpholino (MO, cyan). Insets: example tracks. Data represent mean ± s.d. <i>P</i>-values were calculated using a two-sided Kruskal-Wallis test followed by Dunn’s multiple comparison test and Multiple Mann-Whitney test with a false discovery rate set to 1% (see <i>p</i>-values in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">10</a> and statistics in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">11</a>). Scale bar: 1 µm. <b>c</b> Sketch of cohesin-mediated decrease in chromatin motion. <b>d</b> Left: Example nucleus (bold white circle) subdivided into five bins (dotted lines) with the initial positions of long-bound HT-Rad21 tracks (pink). Right: Pooled initial positions of long-bound tracks of all ITM- and TACO measurements at pre-ZGA stages shown in a unit circle. For post-ZGA and shield stage, see Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">24</a>. <b>e</b> HT-Rad21 track counts per radial bin (compare panel (<b>d</b>), right) normalized to bin area and the highest bin value. Data represented as value ± statistical error (square root of track counts per bin). Statistics are provided in Supplementary Tables <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">6</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">11</a>. CBD Center-Border-Distance. Source data are provided as a Source Data file for Fig. 2b, e.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 2" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure2 Full size image" data-track-label="button" href="/articles/s41467-025-56889-5/figures/2" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>To further shed light on the mechanism with which long-bound cohesin restricted chromatin mobility, we reduced Nipbl levels by injecting nipbl-MO. Previous publications demonstrated a reduced association of cohesin to chromatin upon lower Nipbl levels<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR105" id="ref-link-section-d21244865e1302" title="Schwarzer, W. et al. Two independent modes of chromatin organization revealed by cohesin removal. Nature 551, 51–56 (2017).">105</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR106" id="ref-link-section-d21244865e1302_1" title="Alonso-Gil, D., Cuadrado, A., Giménez-Llorente, D., Rodríguez-Corsino, M. &amp; Losada, A. Different NIPBL requirements of cohesin-STAG1 and cohesin-STAG2. Nat. Commun. 14, 1–11 (2023).">106</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR107" id="ref-link-section-d21244865e1302_2" title="Newkirk, D. A. et al. The effect of Nipped-B-like (Nipbl) haploinsufficiency on genome-wide cohesin binding and target gene expression: Modeling Cornelia de Lange syndrome. Clin. Epigenet. 9, 1–20 (2017).">107</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR108" id="ref-link-section-d21244865e1302_3" title="Porter, H. et al. Cohesin-independent STAG proteins interact with RNA and R-loops and promote complex loading. Elife 12, 1–33 (2023).">108</a>,<a aria-label="Reference 109" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR109" id="ref-link-section-d21244865e1305" title="Nolen, L. D., Boyle, S., Ansari, M., Pritchard, E. &amp; Bickmore, W. A. Regional chromatin decompaction in cornelia de lange syndrome associated with NIPBL disruption can be uncoupled from cohesin and CTCF. Hum. Mol. Genet. 22, 4180–4193 (2013).">109</a></sup>. In addition, lower Nipbl levels reduced the ATPase activity of cohesin<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR17" id="ref-link-section-d21244865e1309" title="Davidson, I. F. et al. DNA loop extrusion by human cohesin. Science 366, 1338–1345 (2019).">17</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR18" id="ref-link-section-d21244865e1309_1" title="Kim, Y., Shi, Z., Zhang, H., Finkelstein, I. J. &amp; Yu, H. Human cohesin compacts DNA by loop extrusion. Science 366, 1345–1349 (2019).">18</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR19" id="ref-link-section-d21244865e1309_2" title="Petela, N. J. et al. Scc2 is a potent activator of cohesin’s ATPase that promotes loading by binding Scc1 without Pds5. Mol. Cell 70, 1134–1148.e7 (2018).">19</a>,<a aria-label="Reference 20" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR20" id="ref-link-section-d21244865e1312" title="Rhodes, J., Mazza, D., Nasmyth, K. &amp; Uphoff, S. Scc2/Nipbl hops between chromosomal cohesin rings after loading. Elife 6, 1–20 (2017).">20</a></sup>. As expected, mean jump distances of long-bound HT-Rad21 increased significantly to (112 ± 58) nm upon nipbl-MO injection (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig2">2b</a>), while the short-bound fraction showed no significant difference (174 ± 8) nm compared to undisturbed conditions. Increased chromatin mobility was not due to a developmental delay of nipbl-MO-injected embryos (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">16</a>). Our observation is compatible with the notion that reduced levels of chromatin-bound cohesin contribute to increased chromatin mobility. This might be due to less cohesin-mediated cohesion or less Nipbl-facilitated ATPase activity and, thereby, potentially loop extrusion of long-bound cohesin. (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig2">2c</a>).</p><p>We next repeated the TACO experiment for two additional phases during early development, where the first phase comprised the 64- to 512-cell stages (pre-ZGA) and the second phase the high to sphere stages (post-ZGA). Mean jump distances of both short-bound and long-bound cohesin molecules decreased from pre-ZGA to the shield stage (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig2">2b</a>). Notably, within every developmental phase, chromatin loci bound long by HT-Rad21 were significantly less mobile than short-bound HT-Rad21 molecules. The decrease in mobility of HT-Rad21-bound chromatin is compatible with loops and TADs becoming more prominent during embryo development<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR40" id="ref-link-section-d21244865e1331" title="Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. Genome Res. 31, 981–994 (2021).">40</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec7">Long-bound cohesin shows distinct nuclear distribution</h3><p>We further characterized the nuclear distribution of long-bound cohesin. Therefore, we devised a measure of radial nuclear position, the center-border distance (CBD), which ranged from the nuclear center (CBD = 0) to the periphery (CBD = 1) in five equidistant radial steps (see Methods). We then sorted every long-bound track from both ITM (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1f</a>) and TACO (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig2">2a</a>) measurements into CBD bins (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig2">2d</a> and Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">24</a>) and normalized to bin area (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig2">2e</a>). We found that the density of long-bound cohesin was highest at the nuclear center in the pre-ZGA phase but shifted towards a more isotropic distribution in the post-ZGA phase. At the shield stage, we observed that long-bound cohesin was sparse at the nuclear periphery.</p><h3 class="c-article__sub-heading" id="Sec8">Changes in the binding kinetics of cohesin and CTCF facilitate chromatin binding during development</h3><p>Our ITM experiments enabled comparing chromatin binding of HT-Rad21 and HT-CTCF at multiple experimental conditions. However, they yield relative, not absolute binding times and fractions of binding classes<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR52" id="ref-link-section-d21244865e1367" title="Reisser, M. et al. Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development. Nat. Commun. 9, 5218 (2018).">52</a></sup>. To obtain absolute bound fractions and dissociation rates (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3a</a>), we employed a classical time-lapse illumination approach in which we imaged HT-Rad21 or HT-CTCF molecules either continuously with 502 ms frame time or interspersed with a dark time of 4 s (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3b</a>). This approach allows correcting for tracking errors and photobleaching of fluorophores and enables covering a broad temporal bandwidth<sup><a aria-label="Reference 57" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR57" id="ref-link-section-d21244865e1377" title="Kuhn, T. et al. Single-molecule tracking of Nodal and Lefty in live zebrafish embryos supports hindered diffusion model. Nat. Commun. 13, 6101 (2022).">57</a>,<a aria-label="Reference 58" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR58" id="ref-link-section-d21244865e1380" title="Kuhn, T., Hettich, J., Davtyan, R. &amp; Gebhardt, J. C. M. Single molecule tracking and analysis framework including theory-predicted parameter settings. Sci. Rep. 11, 1–12 (2021).">58</a></sup>. Here, we counted fluorescent molecules as bound if they were confined for at least three frames (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3c</a> and Methods) and collected the fluorescence survival times of such binding events (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3d, e</a>). Again, we considered the three developmental phases, pre-ZGA, post-ZGA, and shield stage, which revealed a shift of fluorescence survival time distributions to longer times at later stages. We analyzed the fluorescence survival time distributions using the GRID method, which applies a Laplace transformation to infer full dissociation rate spectra of the respective survival time distribution (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3f, g</a>; event spectra in Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">25</a>)<sup><a aria-label="Reference 110" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR110" id="ref-link-section-d21244865e1397" title="Reisser, M. et al. Inferring quantity and qualities of superimposed reaction rates from single molecule survival time distributions. Sci. Rep. 10, 1–13 (2020).">110</a>,<a aria-label="Reference 111" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR111" id="ref-link-section-d21244865e1400" title="Popp, A. P., Hettich, J. &amp; Gebhardt, J. C. M. Altering transcription factor binding reveals comprehensive transcriptional kinetics of a basic gene. Nucleic Acids Res. 49, 6249–6266 (2021).">111</a></sup>. Dissociation rates can be converted to residence times by inversion (see Methods).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Binding kinetics of cohesin and CTCF change during zebrafish embryogenesis." id="figure-3"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig3">Fig. 3: Binding kinetics of cohesin and CTCF change during zebrafish embryogenesis.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-025-56889-5/figures/3" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig3_HTML.png?as=webp" type="image/webp"><img alt="figure 3" aria-describedby="Fig3" height="1007" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig3_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b> Scheme depicting the rate constants of chromatin association (<i>k</i><sub><i>on</i></sub>) and dissociation (<i>k</i><sub><i>off</i></sub>) of HT-Rad21 and HT-CTCF. <b>b</b> Schemes of continuous illumination. <b>c</b> Left: Kymographs propagating with the spot-center of continuous and 4.5 s time-lapse illumination. Right: Corresponding tracks, color-coded according to time. Scale bar: 1 µm. Survival time distributions of (<b>d</b>) HT-Rad21 and (<b>e</b>) HT-CTCF of different time-lapse conditions indicated on the top and the respective GRID fits (dashed lines). pre-ZGA (red): 64-, 128-, 256, 512-cell stages pooled; post-ZGA (green): high, oblong, sphere stages pooled; shield (blue). Statistics are provided in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">14</a>. GRID state spectra of dissociation rates of (<b>f</b>) HT-Rad21 and (<b>g</b>) HT-CTCF using all data (solid line, colored according to stages) and 500 resampling runs with randomly selected 80% of data (black spots) as an error estimation of the spectra. Grey insets: Average residence times and percentages of dissociation rates larger or smaller than 0.1 s<sup>−1</sup> (dashed line). Detailed residence times and fractions are provided in Supplementary Tables <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">12</a>−<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">13</a>. <b>h</b> Measured bound fractions (circle) and bound fractions calculated from kinetic rates (square) of HT-CTCF (left) and HT-Rad21 (right). Data represented as value ± s.d. (Gaussian error propagation (see Methods). Statistics are provided in Supplementary Tables <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">1</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">14</a>. <b>i</b> Search times (inverse on-rates) for a single HT-CTCF or HT-Rad21 molecule to find any binding site (Methods and Supplementary Tables <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">15</a>−<a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">16</a>). <b>j</b> Relative effective search times for any HT-CTCF or HT-Rad21 molecule to find any specific binding site. Data represented as value ± s.d. (Gaussian error propagation). Calculated values in (<b>h</b>)–(<b>j</b>) are based on data in (<b>d</b>), (<b>e</b>) and Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1d, e</a>. Statistics are provided in Supplementary Tables <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">1</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">14</a>. <b>k</b> Scheme depicting the concentration model: During early embryo development, the nuclear volume (outer black circle) decreases, CTCF- (brown) and cohesin (purple) molecule counts, and accessible CTCF binding sites increase. Changes in kinetic rates are depicted as variations in arrow width and color. unsp: unpecific, sp: specific binding. Source data are provided as a Source Data file for Fig. 3d–j.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 3" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure3 Full size image" data-track-label="button" href="/articles/s41467-025-56889-5/figures/3" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>For both cohesin and CTCF, dissociation rate spectra showed several clusters. The residence times corresponding to the smallest rate cluster increased in value and fraction during early development (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3f, g</a>), up to comparably long average residence times of ~100 s in shield stage (cohesin: 99 ± 6 s, CTCF: 89 ± 11 s, Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">12</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">13</a>), with individual binding events up to ~4 min (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3d, e</a>). In agreement with an increasing residence time, the survival times of long-bound fluorescent molecules in ITM measurements also increased during development (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">26</a>). Informed by our ITM measurements (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1f–h</a>), we separated the distribution of residence times into short-bound (&lt;10 s) and long-bound (&gt;10 s) regimes (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3f, g</a>, grey inserts). As expected from the ITM measurements, the overall long-bound fraction from time-lapse measurements increased moderately for HT-CTCF between pre-ZGA and shield stage, while it showed a strong increase for HT-Rad21 (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3h</a> and Methods).</p><p>CTCF has been proposed to transiently associate with RNA-containing clusters while searching for a specific target site<sup><a aria-label="Reference 49" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR49" id="ref-link-section-d21244865e1547" title="Hansen, A. S., Amitai, A., Cattoglio, C., Tjian, R. &amp; Darzacq, X. Guided nuclear exploration increases CTCF target search efficiency. Nat. Chem. Biol. 16, 257–266 (2020).">49</a></sup>. This process is formally similar to a search mechanism, including free diffusion and one-dimensional sliding on unspecific sites commonly applied to DNA-binding proteins<sup><a aria-label="Reference 44" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR44" id="ref-link-section-d21244865e1551" title="Davidson, I. F. et al. CTCF is a DNA-tension-dependent barrier to cohesin-mediated loop extrusion. Nature 616, 822–827 (2023).">44</a>,<a aria-label="Reference 112" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR112" id="ref-link-section-d21244865e1554" title="Berg, O. G., Winter, R. B. &amp; von Hippel, P. H. Diffusion-DRIVEN MECHANISMS OF PROTEIN TRANSLOCATION ON NUCLEIC ACids. 1. Models and Theory. Biochemistry 20, 6929–6948 (1981).">112</a></sup>. Therefore, we considered such a mechanism to quantify the time one of the HT-CTCF molecules needed to find any of the accessible specific target sequences on chromatin. This target site search time depends on the overall bound fraction, the fraction and residence time of specific binding events, and the residence time of unspecific binding events<sup><a aria-label="Reference 113" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR113" id="ref-link-section-d21244865e1558" title="Livhitz, M. A., Gursky, G. V., Zasedatelev, A. S. &amp; Volkenstein, M. V. Equilibrium and kinetic aspects of protein-DNA recognition. Nucleic Acids Res. 6, 2217–2236 (1979).">113</a>,<a aria-label="Reference 114" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR114" id="ref-link-section-d21244865e1561" title="Loffreda, A. et al. Live-cell p53 single-molecule binding is modulated by C-terminal acetylation and correlates with transcriptional activity. Nat. Commun. 8, 313 (2017).">114</a></sup> (see Methods). Since the DNA binding mutant HT-CTCF-ΔZF4-7 showed predominantly short binding events (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1g</a>), we identified the long-bound fraction of HT-CTCF with specific binding. The association times we calculated for one HT-CTCF molecule to any accessible specific site varied between ~150 s (pre-ZGA), ~330 s (post-ZGA), and ~210 s (shield stage) (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3i</a>). For cohesin, the search mechanism is not known. We, therefore, estimated the association rate from an equilibrium model including two types of binding classes of short and long cohesin binding<sup><a aria-label="Reference 114" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR114" id="ref-link-section-d21244865e1572" title="Loffreda, A. et al. Live-cell p53 single-molecule binding is modulated by C-terminal acetylation and correlates with transcriptional activity. Nat. Commun. 8, 313 (2017).">114</a></sup> (see Methods). Similar to CTCF, the time we obtained for cohesin to find a long-bound site increased from pre-ZGA (~420 s) to post-ZGA (~902 s) and dropped to ~180 s in the shield stage (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3i</a>). These association times can be converted to association rates by inversion.</p><p>We further estimated the search times of any HT-Rad21 or HT-CTCF molecule to chromatin. Therefore, we included an order-of-magnitude estimate of changes in relative protein concentrations during early zebrafish development determined from published data<sup><a aria-label="Reference 84" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR84" id="ref-link-section-d21244865e1582" title="Meier, M. et al. Cohesin facilitates zygotic genome activation in zebrafish. Dev 145, 1–13 (2018).">84</a></sup> and our nuclear volume measurements (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">27</a> and Methods). For both HT-CTCF and HT-Rad21, the relative search time dropped during development (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3j</a>).</p><p>The chromatin-bound fraction of proteins is coupled to the association and dissociation kinetics as well as the concentration of accessible binding sites and the nuclear size via the physicochemical law of mass action<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR52" id="ref-link-section-d21244865e1596" title="Reisser, M. et al. Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development. Nat. Commun. 9, 5218 (2018).">52</a></sup>. We compared the measured fraction of long-bound HT-CTCF and HT-Rad21 molecules with the bound fractions calculated from the association and dissociation rates within the physicochemical model and found good agreement (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3h</a> and Methods). This indicates that the altered binding kinetics are sufficient to explain the increase in the bound fractions of HT-Rad21 and HT-CTCF during zebrafish development (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3k</a>).</p><h3 class="c-article__sub-heading" id="Sec9">Polymer modeling recapitulates the emergence of loop extrusion patterns caused by changing cohesin kinetics</h3><p>Our measurements revealed a notable increase in the residence time and a decrease in the effective search time of both cohesin and CTCF during zebrafish embryogenesis (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3</a>). To obtain insight into whether the experimentally observed changes in binding kinetics of cohesin and CTCF could support a trend toward more structured chromosomes as observed in Hi-C experiments<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR40" id="ref-link-section-d21244865e1617" title="Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. Genome Res. 31, 981–994 (2021).">40</a></sup>, we turned to polymer simulations of cohesin-mediated loop extrusion. We used a physical model enabling the comparison of simulated chromosome interaction features against published Hi-C experimental data<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR40" id="ref-link-section-d21244865e1621" title="Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. Genome Res. 31, 981–994 (2021).">40</a></sup>. Our model was constructed as a polymer chain, including volume interactions and loop extrusion<sup><a aria-label="Reference 103" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR103" id="ref-link-section-d21244865e1625" title="Mach, P. et al. Cohesin and CTCF control the dynamics of chromosome folding. Nat. Genet. 54, 1907–1918 (2022).">103</a></sup>. Within this model, we established a direct correspondence between spatial and temporal units (see Methods). Spatial units are important for the spatial organization of chromosomes, while time units are crucial to evaluate the processivity rate of extrusion. We calibrated temporal units based on measurements in mouse embryonic stem cells<sup><a aria-label="Reference 103" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR103" id="ref-link-section-d21244865e1629" title="Mach, P. et al. Cohesin and CTCF control the dynamics of chromosome folding. Nat. Genet. 54, 1907–1918 (2022).">103</a></sup>.</p><p>To assess the effect of changing cohesin kinetics, we chose an extrusion speed of 0.5 kb/sec (an intermediate value between in vitro measurements<sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR17" id="ref-link-section-d21244865e1636" title="Davidson, I. F. et al. DNA loop extrusion by human cohesin. Science 366, 1338–1345 (2019).">17</a>,<a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR18" id="ref-link-section-d21244865e1639" title="Kim, Y., Shi, Z., Zhang, H., Finkelstein, I. J. &amp; Yu, H. Human cohesin compacts DNA by loop extrusion. Science 366, 1345–1349 (2019).">18</a></sup> and in vivo estimates<sup><a aria-label="Reference 73" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR73" id="ref-link-section-d21244865e1643" title="Cattoglio, C. et al. Determining cellular CTCF and cohesin abundances to constrain 3D genome models. Elife 8 (2019).">73</a></sup>) and performed an extensive sweeping of the parameter space of cohesin binding kinetics, covering one order of magnitude of association rates and residence times in the range of our measured values (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">28a, b</a>). From this, we selected a subset of relative association rates, 0.6, 3, and 15 [Mb*min]<sup>-1</sup>, for pre-ZGA, post-ZGA, and shield stages, whose ratios were comparable to the experimentally determined changes in the effective cohesin association rate (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3j</a>). We also calculated processivity<sup><a aria-label="Reference 115" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR115" id="ref-link-section-d21244865e1656" title="Fudenberg, G., Abdennur, N., Imakaev, M., Goloborodko, A. &amp; Mirny, L. A. Emerging Evidence of Chromosome Folding by Loop Extrusion. Cold Spring Harb. Symp. Quant. Biol. 82, 45–55 (2017).">115</a></sup>, yielding values from 25 kb in pre-ZGA and 50 kb in shield stage. For cohesin, we obtained density values on chromosomes of 0.7 Mb<sup>-1</sup> in pre-ZGA and 33.3 Mb<sup>−1</sup> in the shield stage, comparable to other experimental estimates of 2.5–5 Mb<sup>−1<a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR46" id="ref-link-section-d21244865e1665" title="Hansen, A. S., Pustova, I., Cattoglio, C., Tjian, R. &amp; Darzacq, X. CTCF and cohesin regulate chromatin loop stability with distinct dynamics. Elife 6, e25776 (2017).">46</a>,<a aria-label="Reference 73" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR73" id="ref-link-section-d21244865e1668" title="Cattoglio, C. et al. Determining cellular CTCF and cohesin abundances to constrain 3D genome models. Elife 8 (2019).">73</a></sup>. Our simulations focused on the pre-ZGA and shield stages with the two cohesin residence times of 50 s and 100 s, respectively (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">12</a>).</p><p>We represented our simulated trajectories as contact maps, which resemble the output of Hi-C experiments that capture contacts between pairs of DNA loci (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig4">4a, b</a>)<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR40" id="ref-link-section-d21244865e1681" title="Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. Genome Res. 31, 981–994 (2021).">40</a>,<a aria-label="Reference 116" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR116" id="ref-link-section-d21244865e1684" title="Galitsyna, A. et al. Extrusion fountains are hallmarks of chromosome organization emerging upon zygotic genome activation. bioRxiv Prepr. Serv. Biol. 
                  https://doi.org/10.1101/2023.07.15.549120
                  
                 (2023).">116</a></sup>. To reveal the general principles of the spatial chromosome organization, we determined the average contact probability <i>P(s)</i> as a function of the genomic distance <i>s</i> for each stage (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig4">4c, d</a>). As expected from theoretical analysis of looped polymers<sup><a aria-label="Reference 117" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR117" id="ref-link-section-d21244865e1698" title="Polovnikov, K. E. et al. Crumpled polymer with loops recapitulates key features of chromosome organization. Phys. Rev. X 13, 41029 (2023).">117</a></sup>, increasing the extruder association rate and residence time resulted in increased <i>P(s)</i> around the average size of the extruded loops (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig4">4c</a>) and depletion of contacts at longer length scales (orange arrowhead). This is reflected in the local slope of <i>P(s)</i> being smaller around the average loop size and bigger at long length scales (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig4">4e</a> and Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">28c, d</a>). These results are in qualitative agreement with experimentally observed changes in contact maps of the pre-ZGA and shield stages (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig4">4d, f</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Simulated contact maps including single-molecule data recapitulate experimental contact maps." id="figure-4"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig4">Fig. 4: Simulated contact maps including single-molecule data recapitulate experimental contact maps.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41467-025-56889-5/figures/4" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig4_HTML.png?as=webp" type="image/webp"><img alt="figure 4" aria-describedby="Fig4" height="550" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_Fig4_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b> Contact maps of a polymer model with 1 kb mapping for pre-ZGA (left) or shield stages (right). Data is averaged across 10 independent runs. <b>b</b> Contact maps derived from a published Hi-C dataset<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR40" id="ref-link-section-d21244865e1738" title="Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. Genome Res. 31, 981–994 (2021).">40</a></sup> for 2.25 hpf (hours post fertilization, similar to pre-ZGA) and 5.3 hpf (similar to shield). <b>c</b> Contact probability <i>P(s)</i> derived from the maps shown in (<b>a</b>) as a function of genomic distance. The table indicates simulation parameters for Speed of extrusion (S), Processivity of an extruder (P), and Density of extruders on chromatin (D) in pre-ZGA (red) and shield stage (blue). <b>d</b> Same as panel (<b>c</b>) but for Hi-C contacts maps from panel <b>b</b>. <b>e</b> Ratio of P(s) slopes between the shield and pre-ZGA stages using modeling data from panel (<b>c</b>). <b>f</b> Ratio of P(s) slopes between 5.3 hpf and 2.25 hpf for slopes of <i>P(s)</i> from panel <b>d</b>. <b>g</b> Pile-up plots representing average observed contact probabilities divided by expected (average contact probabilities at the same genomic distance) in +/−100kb windows around accessible CTCF sites for different developmental stages. The top row represents modeling, and the bottom row is experimental Hi-C data. <b>h</b> Scheme depicting the emergence of cohesin-mediated loops and CTCF binding due to altered binding kinetics of both species during early zebrafish development.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 4" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure4 Full size image" data-track-label="button" href="/articles/s41467-025-56889-5/figures/4" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>To estimate the influence of increasing CTCF binding, we conducted simulations with a lower spatial resolution of 10 kb. This enabled us to simulate a longer chromosome region of 50 Mb and thus compensate for the sparsity of accessible CTCF sites in the initial phases of development (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">28e, f</a>). We included CTCF sites identified in ChIP-seq data for the pre-ZGA and shield stages (see Methods)<sup><a aria-label="Reference 94" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR94" id="ref-link-section-d21244865e1797" title="Pálfy, M., Schulze, G., Valen, E. &amp; Vastenhouw, N. L. Chromatin accessibility established by Pou5f3, Sox19b and Nanog primes genes for activity during zebrafish genome activation. PLoS Genet. 16, 1–25 (2020).">94</a>,<a aria-label="Reference 118" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR118" id="ref-link-section-d21244865e1800" title="Pérez-Rico, Y. A., Barillot, E. &amp; Shkumatava, A. Demarcation of topologically associating domains is uncoupled from enriched CTCF binding in developing Zebrafish. iScience 23, 101046 (2020).">118</a></sup>. Again, in qualitative agreement with published experimental Hi-C data, we observed progressively increased insulation around CTCF sites, which is indicative of increased stalling of loop-extruding cohesin (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig4">4g</a>).</p><p>Comparison of polymer simulations with Hi-C data thus supports the notion that increased binding efficiency of cohesin and CTCF molecules during early development is accompanied by increased loop extrusion activity and physical insulation upon stalling of cohesin at CTCF sites (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig4">4h</a>).</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec10-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec10">Discussion</h2><div class="c-article-section__content" id="Sec10-content"><p>Here, we characterized the binding properties of the architectural proteins cohesin and CTCF during early development in live zebrafish embryos. We initially set out to understand the nature of single-molecule binding events of CTCF and cohesin (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1</a>). For CTCF, a comparison of the wild-type protein with the DNA binding-deficient mutant CTCF-ΔZF4-7 and an HaloTag control in ITM experiments enabled associating binding events longer than 10 s with binding to specific CTCF target sequences. The Rad21-3x mutant, which prevents DNA binding of the cohesin complex, significantly reduced binding events of cohesin longer than 10 s in ITM experiments, while shorter events were unaffected. This enabled associating long cohesin interactions with DNA binding. The origin of short binding events of CTCF and Rad21 were less clear but presumably represented unspecific transient interactions to DNA, RNA, proteins, or other biomolecules, or transient spatial confinement. To identify the origin of binding events, ITM is advantageous over continuous illumination<sup><a aria-label="Reference 33" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR33" id="ref-link-section-d21244865e1826" title="Nora, E. P. et al. Molecular basis of CTCF binding polarity in genome folding. Nat. Commun. 11, 5612 (2020).">33</a></sup> since it enables separating binding time classes and thereby distinguishing long binding events of molecules above a transient background. Overall, our experiments enabled discerning CTCF bound to specific target sequences and cohesin molecules bound to chromatin.</p><p>An important function of cohesin is sister chromatid cohesion by cohesin molecules stably connecting both daughter strands after replication until early mitosis<sup><a aria-label="Reference 14" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR14" id="ref-link-section-d21244865e1833" title="Peters, J. M. &amp; Nishiyama, T. Sister chromatid cohesion. Cold Spring Harb. Perspect. Biol. 4, a011130 (2012).">14</a>,<a aria-label="Reference 15" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR15" id="ref-link-section-d21244865e1836" title="Nasmyth, K. &amp; Haering, C. H. Cohesin: Its roles and mechanisms. Annu. Rev. Genet. 43, 525–558 (2009).">15</a></sup>. The question arises whether the smallest fraction of ~0.4% of long-bound cohesin molecules we observed at early developmental stages is compatible with this function. If all these long-bound cohesins participated in cohesion, and assuming at least 100 cohesive cohesins for each of the 50 diploid chromosomes for proper chromosome segregation at nuclear division<sup><a aria-label="Reference 45" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR45" id="ref-link-section-d21244865e1840" title="Holzmann, J. et al. Absolute quantification of cohesin, CTCF and their regulators in human cells. Elife 8, 1–31 (2019).">45</a>,<a aria-label="Reference 119" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR119" id="ref-link-section-d21244865e1843" title="Richeldi, M. et al. Mechanical disengagement of the cohesin ring. Nat. Struct. Mol. Biol. 31, 23–31 (2024).">119</a></sup>, approximately 1.1 million cohesins would need to reside in the nucleus. This value is larger than a previous estimate of 508,000 nuclear Scc1 cohesins in Prometa phase-synchronized HeLa cells<sup><a aria-label="Reference 45" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR45" id="ref-link-section-d21244865e1847" title="Holzmann, J. et al. Absolute quantification of cohesin, CTCF and their regulators in human cells. Elife 8, 1–31 (2019).">45</a></sup> and an estimate of approximately 109,000 Rad21 molecules in mESC cells<sup><a aria-label="Reference 73" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR73" id="ref-link-section-d21244865e1851" title="Cattoglio, C. et al. Determining cellular CTCF and cohesin abundances to constrain 3D genome models. Elife 8 (2019).">73</a></sup>. However, in zebrafish, Rad21 is maternally provided to compensate for transcriptional quiescence in the embryo before ZGA, which starts after ca. ten cell divisions<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR40" id="ref-link-section-d21244865e1855" title="Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. Genome Res. 31, 981–994 (2021).">40</a>,<a aria-label="Reference 54" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR54" id="ref-link-section-d21244865e1858" title="Horsfield, J. A. et al. Cohesin-dependent regulation of Runx genes. Development 134, 2639–2649 (2007).">54</a></sup>. Maternally provided proteins typically exceed the need of a single cell, as they are diluted by successive cell divisions. Such a mechanism may account for the very low bound fraction observed in early developmental stages.</p><p>Several of our observations and experiments suggested that long Rad21 binding events were at least partially associated with the loop extrusion function of cohesin: (i) The number of cohesive cohesin molecules required per cell for sister chromatid cohesion is not expected to change during development. Thus, even if all long-bound HT-Rad21 molecules we observed in early developmental stages were involved in cohesion, the increase in the bound fraction during development suggests at least partial involvement of Rad21 in functions of cohesin other than cohesion. In particular, as cell cycles lengthen and G1 phases appear around the 1000-cell stage after ZGA<sup><a aria-label="Reference 120" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR120" id="ref-link-section-d21244865e1865" title="Zamir, E., Kam, Z. &amp; Yarden, A. Transcription-dependent induction of G 1 phase during the Zebra Fish Midblastula transition. Mol. Cell. Biol. 17, 529–536 (1997).">120</a>,<a aria-label="Reference 121" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR121" id="ref-link-section-d21244865e1868" title="Kimmel, C. B., Ballard, W. W., Kimmel, S. R., Ullmann, B. &amp; Schilling, T. F. Stages of embryonic development of the zebrafish. Dev. Dyn. 203, 253–310 (1995).">121</a></sup>, the loop extrusion function of cohesin likely becomes more prominent, consistent with our observation of a strong increase in cohesin bound fraction. (ii) Reducing the expression level of CTCF significantly reduced long but not short Rad21 DNA bound fractions in ITM experiments (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1i</a>). This could be due to a stabilizing effect of CTCF bound to TAD borders on cohesin through their mutual binding interaction<sup><a aria-label="Reference 33" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR33" id="ref-link-section-d21244865e1875" title="Nora, E. P. et al. Molecular basis of CTCF binding polarity in genome folding. Nat. Commun. 11, 5612 (2020).">33</a></sup>. Additionally, CTCF protects cohesin from Wapl-induced unloading<sup><a aria-label="Reference 33" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR33" id="ref-link-section-d21244865e1879" title="Nora, E. P. et al. Molecular basis of CTCF binding polarity in genome folding. Nat. Commun. 11, 5612 (2020).">33</a>,<a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR34" id="ref-link-section-d21244865e1882" title="Li, Y. et al. The structural basis for cohesin–CTCF-anchored loops. Nature 578, 472–476 (2020).">34</a>,<a aria-label="Reference 80" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR80" id="ref-link-section-d21244865e1885" title="Wutz, G. et al. ESCO1 and CTCF enable formation of long chromatin loops by protecting cohesinstag1 from WAPL. Elife 9, 1–33 (2020).">80</a></sup>. However, we cannot exclude a contribution due to the developmental delay compared to untreated embryos. (iii) Mobility of Rad21 bound long to chromatin in TACO experiments was significantly increased at reduced expression levels of Nipbl (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig2">2b</a>). In the case of nipbl-MO treatment, we did not observe a developmental delay at shield stage. Since Nipbl reduces chromatin-bound cohesin<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR105" id="ref-link-section-d21244865e1893" title="Schwarzer, W. et al. Two independent modes of chromatin organization revealed by cohesin removal. Nature 551, 51–56 (2017).">105</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR106" id="ref-link-section-d21244865e1893_1" title="Alonso-Gil, D., Cuadrado, A., Giménez-Llorente, D., Rodríguez-Corsino, M. &amp; Losada, A. Different NIPBL requirements of cohesin-STAG1 and cohesin-STAG2. Nat. Commun. 14, 1–11 (2023).">106</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR107" id="ref-link-section-d21244865e1893_2" title="Newkirk, D. A. et al. The effect of Nipped-B-like (Nipbl) haploinsufficiency on genome-wide cohesin binding and target gene expression: Modeling Cornelia de Lange syndrome. Clin. Epigenet. 9, 1–20 (2017).">107</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR108" id="ref-link-section-d21244865e1893_3" title="Porter, H. et al. Cohesin-independent STAG proteins interact with RNA and R-loops and promote complex loading. Elife 12, 1–33 (2023).">108</a>,<a aria-label="Reference 109" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR109" id="ref-link-section-d21244865e1896" title="Nolen, L. D., Boyle, S., Ansari, M., Pritchard, E. &amp; Bickmore, W. A. Regional chromatin decompaction in cornelia de lange syndrome associated with NIPBL disruption can be uncoupled from cohesin and CTCF. Hum. Mol. Genet. 22, 4180–4193 (2013).">109</a></sup> and stimulates the ATPase activity of cohesin<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR17" id="ref-link-section-d21244865e1900" title="Davidson, I. F. et al. DNA loop extrusion by human cohesin. Science 366, 1338–1345 (2019).">17</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR18" id="ref-link-section-d21244865e1900_1" title="Kim, Y., Shi, Z., Zhang, H., Finkelstein, I. J. &amp; Yu, H. Human cohesin compacts DNA by loop extrusion. Science 366, 1345–1349 (2019).">18</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR19" id="ref-link-section-d21244865e1900_2" title="Petela, N. J. et al. Scc2 is a potent activator of cohesin’s ATPase that promotes loading by binding Scc1 without Pds5. Mol. Cell 70, 1134–1148.e7 (2018).">19</a>,<a aria-label="Reference 20" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR20" id="ref-link-section-d21244865e1903" title="Rhodes, J., Mazza, D., Nasmyth, K. &amp; Uphoff, S. Scc2/Nipbl hops between chromosomal cohesin rings after loading. Elife 6, 1–20 (2017).">20</a></sup>, a reduced cohesive function and reduced loop extrusion activity might contribute to increased chromatin mobility. A low mobility of Rad21-bound chromatin in the presence of loop extrusion is consistent with polymer simulations<sup><a aria-label="Reference 103" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR103" id="ref-link-section-d21244865e1907" title="Mach, P. et al. Cohesin and CTCF control the dynamics of chromosome folding. Nat. Genet. 54, 1907–1918 (2022).">103</a></sup>.</p><p>We found that the binding properties of cohesin and CTCF changed during embryo development and affected chromatin architecture in several ways.</p><p>First, we observed a decrease in mobility of cohesin-bound chromatin during early zebrafish embryogenesis (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig2">2b</a>). This is consistent with higher chromatin mobility in pluripotent embryonic stem cells than in differentiated cells<sup><a aria-label="Reference 122" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR122" id="ref-link-section-d21244865e1921" title="Eshghi, I., Eaton, J. A. &amp; Zidovska, A. Interphase Chromatin undergoes a local sol-gel transition upon cell differentiation. Phys. Rev. Lett. 126, 228101 (2021).">122</a>,<a aria-label="Reference 123" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR123" id="ref-link-section-d21244865e1924" title="Subramanian, V. et al. H2A.Z Acidic patch couples chromatin dynamics to regulation of gene expression programs during ESC Differentiation. PLoS Genet. 9, e1003725 (2013).">123</a></sup> and compatible with an increase in chromatin structures such as compartments, TADs, or loops during embryo development<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR40" id="ref-link-section-d21244865e1928" title="Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. Genome Res. 31, 981–994 (2021).">40</a>,<a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR41" id="ref-link-section-d21244865e1931" title="Kaaij, L. J. T., van der Weide, R. H., Ketting, R. F. &amp; de Wit, E. Systemic loss and gain of Chromatin architecture throughout Zebrafish development. Cell Rep. 24, 1–10.e4 (2018).">41</a></sup>. For the radial distribution of long-bound, potentially loop-extruding cohesin, we found a transition from a high central density in pre-ZGA to a more isotropic density with a pronounced decrease at the nuclear border in shield-stage embryos (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig2">2e</a>). In mammals, the chromatin of differentiated cells is enriched in active histone marks at the nuclear center and repressive marks at the nuclear periphery<sup><a aria-label="Reference 124" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR124" id="ref-link-section-d21244865e1938" title="Girelli, G. et al. GPSeq reveals the radial organization of chromatin in the cell nucleus. Nat. Biotechnol. 38, 1184–1193 (2020).">124</a>,<a aria-label="Reference 125" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR125" id="ref-link-section-d21244865e1941" title="Su, J. H., Zheng, P., Kinrot, S. S., Bintu, B. &amp; Zhuang, X. Genome-scale imaging of the 3D organization and transcriptional activity of chromatin. Cell 182, 1641–1659.e26 (2020).">125</a></sup>. Moreover, it was found that CTCF and Rad21 levels were decreased in heterochromatin-enriched lamina-associated domains of mouse embryonic stem cells<sup><a aria-label="Reference 126" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR126" id="ref-link-section-d21244865e1946" title="van Schaik, T. et al. CTCF and cohesin promote focal detachment of DNA from the nuclear lamina. Genome Biol. 23, 1–26 (2022).">126</a></sup>. Since histone modifications and chromatin accessibility are gradually established over the zygotic genome activation in zebrafish<sup><a aria-label="Reference 127" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR127" id="ref-link-section-d21244865e1950" title="Goll, M. G., Akdogan-Ozdilek, B. &amp; Duval, K. L. Chromatin dynamics at the maternal to zygotic transition: Recent advances from the zebrafish model. F1000Research 9, 1–12 (2020).">127</a>,<a aria-label="Reference 128" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR128" id="ref-link-section-d21244865e1953" title="Laue, K., Rajshekar, S., Courtney, A. J., Lewis, Z. A. &amp; Goll, M. G. The maternal to zygotic transition regulates genome-wide heterochromatin establishment in the zebrafish embryo. Nat. Commun. 10, 1551 (2019).">128</a></sup>, our observations might reflect the transition from pluripotent cells to more differentiated cells.</p><p>Second, we observed an increase in the chromatin-bound fractions of cohesin upon Triptolide or α-Amanitin treatment (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1i</a>). Since both Triptolide and α-Amanitin show distinct inhibitory effects on RNA PolII<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR89" id="ref-link-section-d21244865e1963" title="Brueckner, F. &amp; Cramer, P. Structural basis of transcription inhibition by α-amanitin and implications for RNA polymerase II translocation. Nat. Struct. Mol. Biol. 15, 811–818 (2008).">89</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR90" id="ref-link-section-d21244865e1963_1" title="Kaplan, C. D., Larsson, K. M. &amp; Kornberg, R. D. The RNA Polymerase II Trigger Loop functions in substrate selection and is directly targeted by α-Amanitin. Mol. Cell 30, 547–556 (2008).">90</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR91" id="ref-link-section-d21244865e1963_2" title="Bensaude, O. Inhibiting eukaryotic transcription: Which compound to choose? How to evaluate its activity? Transcription 2, 103–108 (2011).">91</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR92" id="ref-link-section-d21244865e1963_3" title="Li, L. et al. Widespread rearrangement of 3D Chromatin organization underlies polycomb-mediated stress-induced silencing. Mol. Cell 58, 216–231 (2015).">92</a>,<a aria-label="Reference 93" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR93" id="ref-link-section-d21244865e1966" title="Shao, W. &amp; Zeitlinger, J. Paused RNA polymerase II inhibits new transcriptional initiation. Nat. Genet. 49, 1045–1051 (2017).">93</a></sup>, this increase might be due to more efficient loading at sites of stalled PolII<sup><a aria-label="Reference 87" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR87" id="ref-link-section-d21244865e1970" title="Zhang, S. et al. RNA polymerase II is required for spatial chromatin reorganization following exit from mitosis. Sci. Adv. 7, 1–14 (2021).">87</a>,<a aria-label="Reference 88" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR88" id="ref-link-section-d21244865e1973" title="Zhang, S., Übelmesser, N., Barbieri, M. &amp; Papantonis, A. Enhancer–promoter contact formation requires RNAPII and antagonizes loop extrusion. Nat. Genet. 55, 832–840 (2023).">88</a></sup> or due to negative interference of elongating RNA PolII with DNA-bound cohesin. The latter case is still compatible with work showing that PolII is able to push cohesin<sup><a aria-label="Reference 129" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR129" id="ref-link-section-d21244865e1977" title="Busslinger, G. A. et al. Cohesin is positioned in mammalian genomes by transcription, CTCF and Wapl. Nature 544, 503–507 (2017).">129</a>,<a aria-label="Reference 130" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR130" id="ref-link-section-d21244865e1980" title="Davidson, I. F. et al. Rapid movement and transcriptional re‐localization of human cohesin on DNA. EMBO J. 35, 2671–2685 (2016).">130</a></sup>, if a fraction of cohesin remains bound to DNA, and compatible with counteraction of loop extrusion and formation of new TADs by RNA-PolII<sup><a aria-label="Reference 87" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR87" id="ref-link-section-d21244865e1984" title="Zhang, S. et al. RNA polymerase II is required for spatial chromatin reorganization following exit from mitosis. Sci. Adv. 7, 1–14 (2021).">87</a>,<a aria-label="Reference 131" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR131" id="ref-link-section-d21244865e1987" title="Banigan, E. J. et al. Transcription shapes 3D chromatin organization by interacting with loop extrusion. Proc. Natl Acad. Sci. 120, 2017 (2023).">131</a></sup>. Our experiments suggest that disturbing transcription should not interfere with cohesin-mediated chromatin organization. This agrees with recent observations where the initial formation of chromatin architecture in <i>Drosophila</i> and mice was independent of transcription<sup><a aria-label="Reference 37" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR37" id="ref-link-section-d21244865e1995" title="Ke, Y. et al. 3D Chromatin structures of mature gametes and structural reprogramming during mammalian Embryogenesis. Cell 170, 367–381.e20 (2017).">37</a>,<a aria-label="Reference 39" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR39" id="ref-link-section-d21244865e1998" title="Hug, C. B., Grimaldi, A. G., Kruse, K. &amp; Vaquerizas, J. M. Chromatin architecture emerges during zygotic genome activation independent of transcription. Cell 169, 216–228.e19 (2017).">39</a></sup>.</p><p>Third, we found that the binding kinetics of cohesin and CTCF changed during embryogenesis. For both molecular species, the effective association time decreased during development (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3k</a>). This can be understood by an increase in the absolute protein numbers<sup><a aria-label="Reference 84" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR84" id="ref-link-section-d21244865e2008" title="Meier, M. et al. Cohesin facilitates zygotic genome activation in zebrafish. Dev 145, 1–13 (2018).">84</a></sup>, a decrease in the nuclear size, and, for CTCF, an increase in the number of accessible target sites<sup><a aria-label="Reference 118" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR118" id="ref-link-section-d21244865e2012" title="Pérez-Rico, Y. A., Barillot, E. &amp; Shkumatava, A. Demarcation of topologically associating domains is uncoupled from enriched CTCF binding in developing Zebrafish. iScience 23, 101046 (2020).">118</a></sup>. The longest residence time clusters of both species increased during early embryo development, up to similar values of ~100 s in shield stage (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3f, g</a>), with individual binding events up to ~4 min. For CTCF, the longest average residence time is comparable to a previously reported value of ~120 s<sup><a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR46" id="ref-link-section-d21244865e2019" title="Hansen, A. S., Pustova, I., Cattoglio, C., Tjian, R. &amp; Darzacq, X. CTCF and cohesin regulate chromatin loop stability with distinct dynamics. Elife 6, e25776 (2017).">46</a></sup>. For cohesin, our value is considerably shorter than previous estimates of up to ~20 min from fluorescence recovery after photobleaching experiments in mammalian cells<sup><a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR46" id="ref-link-section-d21244865e2024" title="Hansen, A. S., Pustova, I., Cattoglio, C., Tjian, R. &amp; Darzacq, X. CTCF and cohesin regulate chromatin loop stability with distinct dynamics. Elife 6, e25776 (2017).">46</a>,<a aria-label="Reference 50" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR50" id="ref-link-section-d21244865e2027" title="Gerlich, D., Koch, B., Dupeux, F., Peters, J. M. &amp; Ellenberg, J. Live-cell imaging reveals a stable cohesin-chromatin interaction after but not before DNA replication. Curr. Biol. 16, 1571–1578 (2006).">50</a>,<a aria-label="Reference 80" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR80" id="ref-link-section-d21244865e2030" title="Wutz, G. et al. ESCO1 and CTCF enable formation of long chromatin loops by protecting cohesinstag1 from WAPL. Elife 9, 1–33 (2020).">80</a>,<a aria-label="Reference 103" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR103" id="ref-link-section-d21244865e2033" title="Mach, P. et al. Cohesin and CTCF control the dynamics of chromosome folding. Nat. Genet. 54, 1907–1918 (2022).">103</a></sup>. This might be due to species-specific differences. For example, longer cell cycles of ~24 h in mammalian cells compared to ~20 min in zebrafish embryos at pre-ZGA stages<sup><a aria-label="Reference 121" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR121" id="ref-link-section-d21244865e2037" title="Kimmel, C. B., Ballard, W. W., Kimmel, S. R., Ullmann, B. &amp; Schilling, T. F. Stages of embryonic development of the zebrafish. Dev. Dyn. 203, 253–310 (1995).">121</a>,<a aria-label="Reference 132" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR132" id="ref-link-section-d21244865e2040" title="Schulz, K. N. &amp; Harrison, M. M. Mechanisms regulating zygotic genome activation. Nat. Rev. Genet. 20, 221–234 (2019).">132</a></sup> might allow longer residence times. However, due to experimental limitations such as photobleaching and movement of nuclei in the living embryo, we cannot exclude that a fraction of molecules stay bound for longer times. The longest average residence times we obtained from GRID analysis should represent a lower limit. The increase in average residence times we observed during early development might be due to a mutual stabilization of CTCF and cohesin at TAD borders<sup><a aria-label="Reference 33" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR33" id="ref-link-section-d21244865e2044" title="Nora, E. P. et al. Molecular basis of CTCF binding polarity in genome folding. Nat. Commun. 11, 5612 (2020).">33</a></sup>, changes in the relative concentrations of Stag1 and Stag2 that were suggested to give rise to differently stable cohesin binding<sup><a aria-label="Reference 80" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR80" id="ref-link-section-d21244865e2048" title="Wutz, G. et al. ESCO1 and CTCF enable formation of long chromatin loops by protecting cohesinstag1 from WAPL. Elife 9, 1–33 (2020).">80</a>,<a aria-label="Reference 133" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR133" id="ref-link-section-d21244865e2051" title="Casa, V. et al. Redundant and specific roles of cohesin STAG subunits in chromatin looping and transcriptional control. Genome Res. 30, 515–527 (2020).">133</a></sup>, changes in the degree of acetylation of SMC3<sup><a aria-label="Reference 80" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR80" id="ref-link-section-d21244865e2055" title="Wutz, G. et al. ESCO1 and CTCF enable formation of long chromatin loops by protecting cohesinstag1 from WAPL. Elife 9, 1–33 (2020).">80</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR134" id="ref-link-section-d21244865e2058" title="Dauban, L. et al. Regulation of Cohesin-mediated chromosome folding by Eco1 and other partners. Mol. Cell 77, 1279–1293.e4 (2020).">134</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR135" id="ref-link-section-d21244865e2058_1" title="van Ruiten, M. S. et al. The cohesin acetylation cycle controls chromatin loop length through a PDS5A brake mechanism. Nat. Struct. Mol. Biol. 29, 586–591 (2022).">135</a>,<a aria-label="Reference 136" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR136" id="ref-link-section-d21244865e2061" title="Bastié, N. et al. Smc3 acetylation, Pds5 and Scc2 control the translocase activity that establishes cohesin-dependent chromatin loops. Nat. Struct. Mol. Biol. 29, 575–585 (2022).">136</a></sup>, the level of other interaction partners such as Wapl, or changes in the activity of enzymes such as chromatin remodelers<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR137" id="ref-link-section-d21244865e2065" title="Muñoz, S. et al. Functional crosstalk between the cohesin loader and chromatin remodelers. Nat. Commun. 13, 1–12 (2022).">137</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR138" id="ref-link-section-d21244865e2065_1" title="Boginya, A., Detroja, R., Matityahu, A., Frenkel-Morgenstern, M. &amp; Onn, I. The chromatin remodeler Chd1 regulates cohesin in budding yeast and humans. Sci. Rep. 9, 1–10 (2019).">138</a>,<a aria-label="Reference 139" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR139" id="ref-link-section-d21244865e2068" title="Muñoz, S., Minamino, M., Casas-Delucchi, C. S., Patel, H. &amp; Uhlmann, F. A role for Chromatin remodeling in cohesin loading onto chromosomes. Mol. Cell 74, 664–673.e5 (2019).">139</a></sup>. It will be interesting to screen for the origin of the increasing residence times in the future.</p><p>Fourth, for both cohesin and CTCF, the fraction of long-binding events significantly increased during early embryo development. We could recapitulate this increase by inserting our observed changes in the binding kinetics of both species into a physicochemical model of binding considering a decreasing nuclear size and an increasing target site accessibility during early development (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig3">3k</a>)<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR52" id="ref-link-section-d21244865e2078" title="Reisser, M. et al. Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development. Nat. Commun. 9, 5218 (2018).">52</a></sup>. This model previously could explain a comparable increase in the bound fractions of the transcription factors TBP and Sox19b<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR52" id="ref-link-section-d21244865e2082" title="Reisser, M. et al. Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development. Nat. Commun. 9, 5218 (2018).">52</a></sup>. The monotonous change in binding kinetics and, thus, the increase in the bound fraction of long-bound cohesin and CTCF molecules imply that the absolute number of bound architectural proteins involved in organizing chromatin increases during embryogenesis. This is compatible with a gradual formation of chromatin architecture<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR40" id="ref-link-section-d21244865e2086" title="Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. Genome Res. 31, 981–994 (2021).">40</a></sup>. We additionally characterized the effect of changing binding kinetics of cohesin and CTCF on the organization of chromatin by considering our measured kinetic parameters in polymer simulations (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig4">4</a>). These revealed changes in chromatin architecture between the pre-ZGA and shield stages roughly comparable to Hi-C experiments<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR40" id="ref-link-section-d21244865e2094" title="Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. Genome Res. 31, 981–994 (2021).">40</a></sup>. We attribute the more pronounced increase in contact probability of our simulations to omitting factors that can pause or compete with loop extrusion activity and to using a low-density coil in a theta solvent, which has a steeper scaling of contact probability (~-1.5) than what was observed experimentally for mammalian chromosomes after depletion of loop extrusion factors (~−1)<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR12" id="ref-link-section-d21244865e2098" title="Rao, S. S. P. et al. Cohesin loss eliminates all loop domains. Cell 171, 305–320.e24 (2017).">12</a>,<a aria-label="Reference 117" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR117" id="ref-link-section-d21244865e2101" title="Polovnikov, K. E. et al. Crumpled polymer with loops recapitulates key features of chromosome organization. Phys. Rev. X 13, 41029 (2023).">117</a></sup>. Our results indicate that the changes in the residence time and the search time of cohesin and, therefore, the increase of its chromatin-bound fraction are important to globally establish chromatin organization by DNA looping (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig4">4j</a>).</p><p>To conclude, our single-molecule measurements of cohesin and CTCF binding in live developing zebrafish embryos, in particular the different illumination schemes and altered cofactor levels, revealed the molecular basis of gradual chromatin architecture formation and provided a starting point to characterize the function of architectural proteins such as cohesin in the natural environment of a living embryo.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec11-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec11">Methods</h2><div class="c-article-section__content" id="Sec11-content"><h3 class="c-article__sub-heading" id="Sec12">Zebrafish husbandry</h3><p>Zebrafish of the Wild Indian Karyotype (WIK) were maintained by the Weidinger lab (Institute of Biochemistry and Molecular Biology, Ulm University, Ulm, Germany) following the guidelines of the EU directive 2010/63/EU, the German Animal Welfare Act and the State of Baden-Württemberg (Germany) in addition to being permitted by the Regierungspräsidium Tübingen. Zebrafish were solely kept for breeding, and all experiments exclusively used zebrafish embryos before 48 hours post-fertilization.</p><h3 class="c-article__sub-heading" id="Sec13">Generation of wild-type HaloTag constructs, RNA synthesis and Western blot validation</h3><p>We generated N-terminally tagged HaloTag (HT) constructs (HT-rad21, HT-ctcf) by overhang PCR with restriction enzyme sites for PacI, AscI, and complementary DNA sequences for <i>rad21a</i> (OTTDART00000061990, ZFIN) or <i>ctcf</i> (NM_001001844.1, NCBI). The DNA template was cDNA isolated from 1-cell stage zebrafish embryos<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR52" id="ref-link-section-d21244865e2137" title="Reisser, M. et al. Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development. Nat. Commun. 9, 5218 (2018).">52</a></sup>. Primers are listed in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">25</a>.</p><p>We cloned digested PCR fragments in a pCS2-2xHA-HALO-3xGS backbone comprising two HA-tags, the HaloTag, a flexible 3xGS-linker upstream, and an SV40 PolyA tail downstream of the insertion site. The cloning products were amplified in DH5-α cells and isolated using the QIAprep Spin Miniprep Kit (Qiagen) according to the manufacturer’s protocol. All cloning products were validated by Sanger sequencing before proceeding with RNA synthesis.</p><p>We obtained the control constructs (HT-rad21-3x, HT-ctcf-ΔZF4-7) by site-directed mutagenesis of the HT-rad21 and HT-ctcf plasmids using the Q5 Site-Directed Mutagenesis Kit (New England BioLabs) according to the manufacturer’s protocol with primers given in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">25</a>. The HT-rad21-3x construct contained three point mutations (F609R, L613R, Q625K) that were previously described to prevent proper cohesin complex formation<sup><a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR46" id="ref-link-section-d21244865e2153" title="Hansen, A. S., Pustova, I., Cattoglio, C., Tjian, R. &amp; Darzacq, X. CTCF and cohesin regulate chromatin loop stability with distinct dynamics. Elife 6, e25776 (2017).">46</a>,<a aria-label="Reference 67" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR67" id="ref-link-section-d21244865e2156" title="Haering, C. H. et al. Structure and stability of cohesin’s Smc1-kleisin interaction. Mol. Cell 15, 951–964 (2004).">67</a>,<a aria-label="Reference 68" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR68" id="ref-link-section-d21244865e2159" title="Sun, Y. et al. RAD21 is the core subunit of the cohesin complex involved in directing genome organization. Genome Biol. 24, 155 (2023).">68</a></sup>. The HT-ctcf-ΔZF4-7 construct was depleted of its zinc finger domains 4-7 (annotation from Q6JAA4, Uniprot), responsible for CTCF’s primary DNA binding to the core CTCF motif<sup><a aria-label="Reference 69" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR69" id="ref-link-section-d21244865e2163" title="Nakahashi, H. et al. A genome-wide Map of CTCF multivalency redefines the CTCF Code. Cell Rep. 3, 1678–1689 (2013).">69</a>,<a aria-label="Reference 70" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR70" id="ref-link-section-d21244865e2166" title="Hashimoto, H. et al. Structural basis for the versatile and methylation-dependent binding of CTCF to DNA. Mol. Cell 66, 711–720.e3 (2017).">70</a>,<a aria-label="Reference 140" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR140" id="ref-link-section-d21244865e2169" title="Del Moral-Morales, A. et al. CTCF and its multi-partner network for chromatin regulation. Cells 12, 1–19 (2023).">140</a></sup>. Successful mutagenesis was validated by Sanger sequencing before proceeding with RNA synthesis.</p><p>We synthesized RNA using the mMESSAGE mMACHINE SP6 Transcription Kit (Invitrogen) following the manufacturer’s protocol after plasmid linearization with NotI-HF (New England BioLabs). RNA cleanup was performed with the RNeasy Plus Mini Kit (Qiagen) according to the manufacturer’s protocol. α-Bungarotoxin RNA was synthesized as described above using the pmtb-t7-alpha-bungarotoxin plasmid, a gift from Sean Megason (#69542, Addgene)<sup><a aria-label="Reference 141" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR141" id="ref-link-section-d21244865e2176" title="Swinburne, I. A., Mosaliganti, K. R., Green, A. A. &amp; Megason, S. G. Improved long-term imaging of embryos with genetically encoded α-bungarotoxin. PLoS One 10, 1–15 (2015).">141</a></sup>. eGFP-lap2β mRNA was generated as described previously<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR52" id="ref-link-section-d21244865e2180" title="Reisser, M. et al. Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development. Nat. Commun. 9, 5218 (2018).">52</a></sup>. Control HaloTag mRNA was produced from the pCS2-2xHA-HALO-3xGS plasmid without any insert.</p><p>Ectopic protein expression levels were determined by Western blotting of samples obtained and injected on three independent days. We isolated protein from 20 shield-stage wild-type embryos and embryos injected with 6.7 pg of mRNA encoding for HT-rad21 or 10 pg of mRNA encoding for HT-ctcf based on a standard Western Blot protocol<sup><a aria-label="Reference 142" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR142" id="ref-link-section-d21244865e2188" title="Link, V., Shevchenko, A. &amp; Heisenberg, C. P. Proteomics of early zebrafish embryos. BMC Dev. Biol. 6, 1–9 (2006).">142</a></sup>. We chose 10-fold larger injection amounts compared to our single-molecule measurements to observe the HT-protein band. The primary anti-Rad21 antibody (ab992, Abcam) was diluted 1:500, and the anti-CTCF antibody (ab128873, Abcam) 1:3000. Quantification was performed with Image Lab 6.0 by dividing the background subtracted HT-protein band intensity with the endogenous protein band intensity. Since we injected 10-fold less mRNA for single-molecule measurements, we divided the value by a factor of 10 to obtain the expression levels in single-molecule experiments. WT lysate was used as a control and to identify the HT-protein band.</p><h3 class="c-article__sub-heading" id="Sec14">Sample preparation for single-molecule imaging</h3><p>After spawning, we dechorionated zebrafish embryos in a 1 mg/ml pronase (10165921001, Sigma-Aldrich) Danieau’s medium solution for three minutes, followed by washing in Danieau’s medium. Still in the 1-cell stage, embryos were injected with 5 pg of eGFP-lap2β mRNA together with either 1 pg HT-ctcf, 1 pg HT-ctcf-ΔZF4-7, 0.67 pg HT-rad21, 0.67 pg HT-rad21-3x mRNA (33.5 pg for 24 hpf measurements), or 1 pg HT mRNA (control). Measurements at 24 hpf required an additional 50 pg of α-Bungarotoxin mRNA to prevent embryos from twitching<sup><a aria-label="Reference 141" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR141" id="ref-link-section-d21244865e2200" title="Swinburne, I. A., Mosaliganti, K. R., Green, A. A. &amp; Megason, S. G. Improved long-term imaging of embryos with genetically encoded α-bungarotoxin. PLoS One 10, 1–15 (2015).">141</a></sup>. To set the injection volume of 1 nl on the pneumatic PicoPump (World Precision Instruments), we adjusted the injection duration to yield droplets of 124 µm, determined by a stereo microscope (Olympus SZX2-ZB10) and a camera (CAM-SC50).</p><p>HaloTag proteins were labeled by transferring zebrafish embryos to glass tubes followed by incubation in 10 nM HaloTag-JF549 dye solution (50 nM for 24 hpf measurements). After 15 and 30 min past labeling, embryos were washed twice and further bred in Danieau’s medium at room temperature (22 °C) or at 28 °C for shield-stage and 24 hpf measurements. We visually monitored the development of room-temperature embryos until they synchronously reached the 32-cell stage. Then, 4-6 embryos were transferred to a 170 µm thick glass bottom imaging dish (Delta T, Bioptechs) on the reflected light-sheet microscope. Embryos were measured at room temperature in Danieau’s buffer while progressing in development. During measurement, we identified cell divisions and developmental stages using the eGFP-labeled nuclear membrane marker Lap2β. Movies were recorded after the fusion of karyomeres and at the largest nuclear cross-section. Measurements were concluded after sphere-stage. The shield-stage and 24 hpf embryos were taken out from the incubator 30 min before measurement, placed on the imaging dish as aforementioned, and measured for a maximum duration of 1 h.</p><p>To inhibit RNA Polymerase II, we used α-Amanitin (A2263, Sigma-Aldrich), which was dissolved in water and included in the RNA injection mix at a final 0.4 mg/ml concentration. As described above, the injection mix was injected into 1-cell stage zebrafish embryos with HT-rad21 and eGFP-lap2β mRNA. The concentration was similar to previous reports<sup><a aria-label="Reference 96" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR96" id="ref-link-section-d21244865e2210" title="Kane, D. A. et al. The zebrafish epiboly mutants. Development 123, 47–55 (1996).">96</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR143" id="ref-link-section-d21244865e2213" title="Sato, Y. et al. Histone H3K27 acetylation precedes active transcription during zebrafish zygotic genome activation as revealed by live-cell analysis. Dev. 146, dev179127 (2019).">143</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR144" id="ref-link-section-d21244865e2213_1" title="Heyn, P., Salmonowicz, H., Rodenfels, J. &amp; Neugebauer, K. M. Activation of transcription enforces the formation of distinct nuclear bodies in zebrafish embryos. RNA Biol. 14, 752–760 (2017).">144</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR145" id="ref-link-section-d21244865e2213_2" title="Pancholi, A. et al. RNA polymerase II clusters form in line with surface condensation on regulatory chromatin. Mol. Syst. Biol. 17, 1–26 (2021).">145</a>,<a aria-label="Reference 146" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR146" id="ref-link-section-d21244865e2216" title="Hilbert, L. et al. Transcription organizes euchromatin via microphase separation. Nat. Commun. 12, 1–12 (2021).">146</a></sup>. Moreover, we used Triptolide (645900, Sigma-Aldrich) dissolved in DMSO and diluted to a 5 µM solution in Danieau’s medium on the day of usage. After dye incubation, around the 4- to 8-cell stage, embryos were washed and kept in a Triptolide solution for further development and imaging. Triptolide concentration and handling are comparable to previous reports<sup><a aria-label="Reference 147" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR147" id="ref-link-section-d21244865e2220" title="Hadzhiev, Y. et al. The miR-430 locus with extreme promoter density forms a transcription body during the minor wave of zygotic genome activation. Dev. Cell 58, 155–170.e8 (2023).">147</a>,<a aria-label="Reference 148" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR148" id="ref-link-section-d21244865e2223" title="Kontur, C., Jeong, M., Cifuentes, D. &amp; Giraldez, A. J. Ythdf m6A readers function redundantly during Zebrafish development. Cell Rep. 33, 108598 (2020).">148</a></sup>. The duration between treatment and shield stage measurement were approximately 9 hours for α-Amanitin and 8 hours for Triptolide.</p><h3 class="c-article__sub-heading" id="Sec15">Morpholino (MO) handling and Western blot validation</h3><p>All MOs targeting RNA translation were obtained from Gene Tools LLC (Philomath, USA) and coinjected with 0.67 pg HT-rad21 and 5 pg eGFP-lap2β mRNA as described above. We used 4 ng of the single ctcf-morpholino (ctcf-MO) or 1 ng of each nipbl-morpholino (nipbla and nipblb, in short nipbl-MO) per injection. Both the ctcf<sup><a aria-label="Reference 83" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR83" id="ref-link-section-d21244865e2235" title="Delgado-Olguín, P. et al. CTCF promotes muscle differentiation by modulating the activity of myogenic regulatory factors. J. Biol. Chem. 286, 12483–12494 (2011).">83</a></sup> and the nipbl-MOs<sup><a aria-label="Reference 81" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR81" id="ref-link-section-d21244865e2239" title="Muto, A., Calof, A. L., Lander, A. D. &amp; Schilling, T. F. Multifactorial origins of heart and gut defects in Nipbl-deficient zebrafish, a model of cornelia de Lange Syndrome. PLoS Biol. 9, e1001181 (2011).">81</a></sup> were previously validated with comparable injection amounts. Negative control measurements were performed by coinjection of 4 ng standard control morpholino mRNA obtained from Gene Tools LLC (Philomath, USA).</p><p>We validated the previously published ctcf-MO by Western blotting<sup><a aria-label="Reference 83" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR83" id="ref-link-section-d21244865e2246" title="Delgado-Olguín, P. et al. CTCF promotes muscle differentiation by modulating the activity of myogenic regulatory factors. J. Biol. Chem. 286, 12483–12494 (2011).">83</a></sup> of samples obtained and injected on three independent days. Therefore, we isolated protein from 20 shield-stage wild-type embryos or embryos injected with 4 ng ctcf-MO, based on a standard Western Blot protocol<sup><a aria-label="Reference 142" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR142" id="ref-link-section-d21244865e2250" title="Link, V., Shevchenko, A. &amp; Heisenberg, C. P. Proteomics of early zebrafish embryos. BMC Dev. Biol. 6, 1–9 (2006).">142</a></sup>. The primary anti-CTCF antibody (ab128873, Abcam) was diluted 1:3000, and the anti-γ-Tubulin antibody was diluted 1:100,000 (ab11316, Abcam). We quantified Western blots of three biological replicates with Image Lab 6.0 by first normalizing the background subtracted CTCF-band to the γ-Tubulin band, followed by dividing the morpholino-injected value with the wild-type value.</p><h3 class="c-article__sub-heading" id="Sec16">Subcellular Protein Fractionation</h3><p>We injected zebrafish embryos at the 1-cell stage with either 6.7 pg HT-rad21 mRNA, 6.7 pg HT-rad21 3x-mutant mRNA, 10 pg HT-ctcf mRNA or 10 pg HT-ctcf-ΔZF4-7 mutant mRNA. To ensure detection of HT-protein bands, injection amounts were increased 10-fold compared to our single-molecule measurements, consistent with our Western blot methodology. Similarly, we removed the comparably large proportion of interfering yolk proteins from 150-190 shield-stage embryos based on standard protocols<sup><a aria-label="Reference 142" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR142" id="ref-link-section-d21244865e2263" title="Link, V., Shevchenko, A. &amp; Heisenberg, C. P. Proteomics of early zebrafish embryos. BMC Dev. Biol. 6, 1–9 (2006).">142</a></sup>. The resulting cell pellet was washed once in PBS, followed by centrifugation at 500 <i>g</i> for 2 min. To enhance subsequent lysis, nearly all supernatant was removed followed by a short vortex on the lowest setting to gently disperse cells in the residual supernatant. Subcellular fractionation was performed using the Subcellular Protein Fractioning Kit for Cultured Cells (78840, Thermo Fisher Scientific), according to the manufacturers protocol for a 20 µL cell volume. For Western blot analysis, we used 20 µl of each extract. The primary anti-Rad21 antibody (ab992, Abcam) was diluted 1:500, the anti-CTCF antibody (ab128873, Abcam) 1:2000 and the anti-Histone H2B antibody (MA5-14835, Thermo Fisher Scientific) 1:1000. We calculated the ratio of HT-protein to endogenous protein from background subtracted bands using Image Lab 6.0.</p><p>Lane profiles for HT-CTCF-ΔZF4-7 were generated in FIJI<sup><a aria-label="Reference 149" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR149" id="ref-link-section-d21244865e2273" title="Schindelin, J. et al. Fiji: An open-source platform for biological-image analysis. Nat. Methods 9, 676–682 (2012).">149</a></sup>. Western blots images were first rotated to vertically align the 3-SNE and 4-SNE bands. Using a rectangular selection over a straight part of each band, we obtained intensity profiles by using the “Plot Profile” function. For each lane, data was normalized by first subtracting the minimum value of the dataset, followed by dividing by the range of the dataset (maximum minus the minimum value).</p><h3 class="c-article__sub-heading" id="Sec17">Co-Immunoprecipitation (Co-IP) of Smc3 through HA-tagged HT-Rad21</h3><p>To investigate the incorporation of our HA-tagged HT-Rad21 subunit in the cohesin complex, we performed Co-Immunoprecipitation (Co-IP) using anti-HA-Tag beads. We injected zebrafish embryos at the 1-cell stage with either 6.7 pg HT-rad21 mRNA (10-fold mRNA amounts compared to our single-molecule measurements) or left them uninjected as a wild-type (WT) negative control, comparable to our Western blot methodology. We deyolked approximately 165 shield stage zebrafish embryos of each sample based on standard protocols<sup><a aria-label="Reference 142" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR142" id="ref-link-section-d21244865e2285" title="Link, V., Shevchenko, A. &amp; Heisenberg, C. P. Proteomics of early zebrafish embryos. BMC Dev. Biol. 6, 1–9 (2006).">142</a></sup>. The resulting cell pellet was washed in PBS and centrifuged at 300 <i>g</i> for 1 min. To enhance subsequent lysis, nearly all supernatant was removed followed by a short vortex on the lowest setting to gently disperse cells. Cells were then lysed in ice-cold Pierce™ IP Lysis Buffer (87787, Thermo Fisher Scientific) for 5 min on ice after which 10% was taken as the Input reference. We centrifuged the remainder at 13,000 <i>g</i> for 10 min at 4 °C to pellet cell debris, and used the liquid phase for the following Co-IP procedure. We performed the Co-IP using the Pierce™ HA-Tag IP/Co-IP Kit (26180, Thermo Fisher Scientific) according to the manufactures protocol, incubating the anti-HA agarose beads and sample overnight, followed by Elution Protocol 2.</p><p>To detect proteins, we equally distributed the Input and Co-IP samples from injected and WT embryos across two gels and resolved the proteins simultaneously by SDS-PAGE, resulting in two equally loaded blots. To unambiguously identify the presence of each protein, we probed one blot with 1:1000 anti-Smc3 antibody (JM10-75, Thermo Fisher Scientific) and the other with 1:5000 anti-HA Tag antibody (2-2.2.14 #26183, Thermo Fisher Scientific). To confirm the presence of both Smc3 and HA-tagged HT-Rad21 protein on a single blot, the first blot probed with anti-Smc3 antibody was subsequently probed with 1:2500 anti-HA Tag antibody.</p><h3 class="c-article__sub-heading" id="Sec18">Reflected light-sheet microscopy</h3><p>Single-molecule movies of live developing zebrafish embryos were recorded on a custom-built reflected light-sheet microscope, allowing lateral excitation of the ~700 µm large zebrafish embryos<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR52" id="ref-link-section-d21244865e2306" title="Reisser, M. et al. Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development. Nat. Commun. 9, 5218 (2018).">52</a></sup>. A commercial Nikon TI microscope body laid the foundation together with a 1.2 NA water-immersion objective (60× 1.20 NA Plan Apo VC W, NIKON), dichroic mirror (F73–866/F58–533, AHF), emission filter (F72–866/ F57–532, AHF), notch filter (F40–072/F40–513, AHF) and an EM-CCD camera (iXon Ultra DU 897U, Andor). An additional 1.5x post-magnification in the detection path resulted in an effective pixel size of 166 nm. Peripheral devices were controlled with a NIDAQ data acquisition card (National Instruments) and the NIS Elements software (Version 4.40.00 64-bit, Nikon). Laser illuminations during camera exposures were driven by a custom controller, reacting to camera triggers and following predefined patterns (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">26</a>).</p><p>The excitation path used 488 nm (iBEAM- SMART-488-S-HP, 200 mW, Toptica) and 561 nm (Jive 300, 300 mW, Cobolt) lasers, combined and then controlled by an AOTF (AOTFnC-400.650-TN, AA Optoelectronics), followed by a single-mode fiber connected into a custom-built tower. The light sheet is created in the tower by a cylindrical lens that is focused in the back-focal plane of a vertically mounted water-dipping objective (40× 0.8 NA HCX Apo L W, Leica). The tower was placed above the imaging dish and reflected the excitation light sheet laterally with a custom-coated cantilever (40 nm Al coating on both sides of HYDRA2R-100N-TL-20, AppNano). The approximately 3 µm thick light sheet was set to 47 mW and 5 mW after the excitation objective for the 561 nm and 488 nm laser, respectively.</p><p>We recorded continuous, interlaced time-lapse microscopy (ITM) and time-lapse alternated with continuous intervals (TACO) movies with 10 ms of exposure time and 1.7 ms of camera readout time, resulting in a total frame cycle time of 11.7 ms. Movies for time-lapse illumination were recorded with 500 ms exposure and a total frame cycle time of 501.7 ms. All illumination schemes contained combinations of the nuclear membrane marker eGFP-Lap2β (488 nm) to track the nucleus and drift, 561 nm exposure for HaloTag-JF549 excitation, and dark times without laser excitation (see Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">26</a>).</p><h3 class="c-article__sub-heading" id="Sec19">Analysis of single-molecule microscopy data</h3><p>Single continuous movies were split into single files for the 488 nm and 561 nm channels using the Movie splitter tool of TrackIt<sup><a aria-label="Reference 58" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR58" id="ref-link-section-d21244865e2330" title="Kuhn, T., Hettich, J., Davtyan, R. &amp; Gebhardt, J. C. M. Single molecule tracking and analysis framework including theory-predicted parameter settings. Sci. Rep. 11, 1–12 (2021).">58</a></sup>. Movies with considerable drift were discarded to prevent tracking and subsequent analysis errors.</p><p>We analyzed all single-molecule microscopy data with the nearest neighbor algorithm of TrackIt and tracking parameters shown in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">27</a>. Nuclear tracking regions were drawn based on z-projections of the Lap2β signal if not indicated otherwise.</p><p>We used a custom MATLAB script to analyze the time intervals between consecutive early developmental stages (64-cell to the sphere stage) that were recorded with our ITM scheme (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1f</a>). The script determined the time difference between consecutive stages for each embryo separately, by comparing the creation dates of the original movie files. If a stage was missing for a particular embryo, time differences for that stage were not calculated.</p><h3 class="c-article__sub-heading" id="Sec20">Data analysis of continuous movies: bound fractions and diffusion coefficients</h3><p>First, we recorded continuous movies with a frame cycle time of 11.7 ms and tracked 100 frames with TrackIt (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">27</a>). Next, we determined diffusion rates and amplitudes by fitting cumulative distributions of the jump distances within detected tracks with a three-component Brownian diffusion model using the data analysis tool of TrackIt<sup><a aria-label="Reference 58" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR58" id="ref-link-section-d21244865e2357" title="Kuhn, T., Hettich, J., Davtyan, R. &amp; Gebhardt, J. C. M. Single molecule tracking and analysis framework including theory-predicted parameter settings. Sci. Rep. 11, 1–12 (2021).">58</a></sup>. The bin width was set to 1 nm, resulting in a total of 747 bins. Jumps over gap frames were removed, and the number of jumps to consider per track was set to 10 in order to prevent overrepresentation of bound molecules<sup><a aria-label="Reference 57" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR57" id="ref-link-section-d21244865e2361" title="Kuhn, T. et al. Single-molecule tracking of Nodal and Lefty in live zebrafish embryos supports hindered diffusion model. Nat. Commun. 13, 6101 (2022).">57</a></sup>. Effects from molecules moving out of the focus plane (overestimation of bound fraction) or tracking of 3D movement in a 2D plane (underestimation of diffusion coefficients) were not corrected. However, these effects are present to a similar extent in all stages, and relative comparisons are not affected. Errors of diffusion coefficients and amplitudes were estimated based on the standard deviation from analyzing 500 subsets of the jump distances, including 80% of randomly selected jump distances. The amplitude of the slowest diffusion component represented the overall bound fraction <i>f</i><sub><i>b</i></sub> of the tracked molecules. The unbound fraction of tracked molecules is given by <i>p</i><sub><i>f</i></sub> = 1-<i>f</i><sub><i>b</i></sub>.</p><p>To evaluate whether a three-component diffusion model was superior to a two-component diffusion model, the reduced chi-squared and Akaike Information Criterion (AIC) were calculated for each model. The AIC provides a measure for the balance between the goodness of a fit and the complexity of a model, where more complex models with a higher number of parameters are punished. The differences between the models were calculated according to Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ1">1</a>):</p><div class="c-article-equation" id="Equ1"><div class="c-article-equation__content"><span class="mathjax-tex">$$\Delta {{AIC}}_{i}=\left(2{k}_{2{rate}}+{nln}\left({{RSS}}_{2{rate}}\right)\right)-\left(2{k}_{3{rate}}+{nln}\left({{RSS}}_{3{rate}}\right)\right)$$</span></div><div class="c-article-equation__number">
                    (1)
                </div></div><p>here, the residual sum of squares (RSS) reflects the sum of differences between data and fit from the cumulative jump distance plot, <i>k</i> reflects the number of parameters from the two- or three-component models, and <i>n</i> is the total number of bins. A positive ΔAIC value indicates a preferential three-component model, whereas a negative value indicates a preferential two-component model.</p><h3 class="c-article__sub-heading" id="Sec21">Data analysis of interlaced time-lapse microscopy (ITM): relative fractions in binding time classes</h3><p>We recorded ITM movies to classify molecules into three binding time classes, namely spots (single frame), short bound (0.2 s), and long bound (&gt;8.2 s), and to calculate the relative fraction of molecules binding in each class. The illumination scheme includes short and long dark times between frames of molecule detection (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">26</a>).</p><p>The tracking and analysis parameters for TrackIt are provided in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">27</a>. Bound fractions of the binding classes were calculated movie-wise with TrackIts data analysis tool using the following Eqs. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ2">2</a>) and (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ3">3</a>):</p><div class="c-article-equation" id="Equ2"><div class="c-article-equation__content"><span class="mathjax-tex">$${long\; bound\; fraction}=\,\frac{{N}_{{long}}}{{N}_{{all\; events}}}=\frac{{N}_{{long}}}{{N}_{{long}}+\,{N}_{{short}}+{N}_{{spot}}}$$</span></div><div class="c-article-equation__number">
                    (2)
                </div></div><p>here <i>N</i><sub><i>long</i></sub> represented the number of all tracks surviving two or more long dark times (&gt;8.2 s), and <i>N</i><sub><i>short</i></sub> all tracks with one long or no long dark time. <i>N</i><sub><i>spot</i></sub> depicted the number of all single detections.</p><div class="c-article-equation" id="Equ3"><div class="c-article-equation__content"><span class="mathjax-tex">$${short\; bound\; fraction}=\,\frac{{N}_{{short}}}{{N}_{{all\; events}}}=\frac{{N}_{{short}}}{{N}_{{long}}+\,{N}_{{short}}+{N}_{{spot}}}$$</span></div><div class="c-article-equation__number">
                    (3)
                </div></div><p>here <i>N</i><sub><i>short</i></sub> was the sum of all tracks that lasted for one short dark time (0.2 s), and <i>N</i><sub><i>long</i></sub> was the sum of tracks that passed at least one long dark time.</p><h3 class="c-article__sub-heading" id="Sec22">Data analysis of time-lapse alternated with continuous intervals (TACO): mobility of molecules in certain binding time classes</h3><p>The TACO scheme can classify tracks in binding time classes similar to ITM, namely short and long; however, the scheme adds fast and continuous tracking periods. Thus, TACO can provide additional diffusion analysis of similarly classified tracks examined with ITM.</p><p>The illumination scheme consisted of single detection frames followed by dark times (detection periods) (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">26</a>) to track whether a molecule stayed bound while minimizing laser exposure and photobleaching. Tracking and analysis of molecules were facilitated by appending 10 continuous frames of 11.7 ms frame cycle time (analysis periods). We pooled multiple stages: pre-ZGA (64-cell, 128-cell, 256-cell, 512-cell), post-ZGA (high, oblong, sphere), and shield. After tracking of molecules (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">27</a>), each track was split into the first complete analysis period if, in the case of the long TACO scheme, the track survived at least 2 dark times (=8.4 s) or in the case of the short TACO scheme, the track only survived a single short dark time (=0.2 s) without being detected in the following long dark time. To prevent overrepresentation of single detections, only the first analysis region without gap frames was used, and all other analysis regions of this track were discarded.</p><h3 class="c-article__sub-heading" id="Sec23">Data analysis of time-lapse microscopy: residence times and bound fractions</h3><p>We used time-lapse microscopy and our previously published genuine rate identification method (GRID) to determine residence times of the HT-rad21 and HT-ctcf constructs<sup><a aria-label="Reference 110" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR110" id="ref-link-section-d21244865e3107" title="Reisser, M. et al. Inferring quantity and qualities of superimposed reaction rates from single molecule survival time distributions. Sci. Rep. 10, 1–13 (2020).">110</a></sup>. GRID is based on solving the inverse Laplace transformation of fluorescence survival time distributions gained from multiple time-lapse conditions.</p><p>The first time-lapse condition builds on 501.7 ms frame-cycle time movies with continuous excitation of the JF549 labeled HaloTag construct. The second condition consisted of a repeating pattern with a single 501.7 ms excitation frame followed by a dark time of 4013.6 ms without laser illumination (Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">26</a>). We chose 500 ms exposure time to blur out diffusing molecules and focus on tracking bound molecules. We pooled movies of both time-lapse conditions into pre-ZGA (64-cell, 128-cell, 256-cell, 512-cell), post-ZGA (high, oblong, sphere), and shield stage. Tracking radii for each time-lapse condition were determined to yield equal loss probabilities of 0.0025 in TrackIt. The loss probability, accounting for tracking errors and photobleaching, was set equally for time-lapse conditions and stages of each construct. The resulting tracking parameters were the same for all stages and are listed in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">27</a>.</p><p>We used GRID to simultaneously analyze the fluorescence survival time distributions of both imaging conditions and obtain the event spectrum of dissociation rates ranging from 10<sup>−3</sup> s<sup>−1</sup> to 10<sup>1</sup> s<sup>−1</sup> (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">25</a>). The amplitudes <i>A</i><sup><i>e</i></sup> of the event spectrum of dissociation rates correspond to the relative fraction of molecular binding events occurring with a certain dissociation rate in a period of time. We transferred the event spectrum to the state spectrum of dissociation rates by dividing each amplitude with the corresponding dissociation rate and renormalization<sup><a aria-label="Reference 110" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR110" id="ref-link-section-d21244865e3141" title="Reisser, M. et al. Inferring quantity and qualities of superimposed reaction rates from single molecule survival time distributions. Sci. Rep. 10, 1–13 (2020).">110</a></sup>. The amplitudes <i>A</i><sup><i>s</i></sup> of the state spectrum correspond to the relative fraction of molecular binding events that belong to a certain dissociation rate in a snapshot of time. We estimated the error of both event and state spectra by resampling 500 data subsets, each including 80% of randomly selected survival times.</p><p>To obtain the residence times <i>τ</i><sub><i>s</i></sub> of long-binding events, we averaged all dissociation rates below 0.1 s<sup>−1</sup> (<i>k</i><sub><i>off,s</i></sub>) based on the binding time classification determined in ITM experiments and inverted the resulting value. The relative fraction <i>A</i><sup><i>s</i></sup><sub><i>s</i></sub> of long-binding events is given by the corresponding averaged amplitudes of the state spectrum. To obtain the residence times <i>τ</i><sub><i>u</i></sub> of short-binding events, we performed these operations with dissociation rates above 0.1 s<sup>-1</sup> (<i>k</i><sub><i>off,u</i></sub>). Again, the relative fraction <i>A</i><sup><i>s</i></sup><sub><i>u</i></sub> of short-binding events is given by the corresponding averaged amplitudes of the state spectrum. We calculated the overall fractions <i>p</i><sub><i>b,s</i></sub> <i>= A</i><sup><i>s</i></sup><sub><i>s</i></sub> <i>* f</i><sub><i>b</i></sub> of long-binding events and <i>p</i><sub><i>b,u</i></sub> <i>= A</i><sup><i>s</i></sup><sub><i>u</i></sub> <i>* f</i><sub><i>b</i></sub> of short-binding events by multiplying the relative bound fractions with the overall bound fraction <i>f</i><sub><i>b</i></sub> obtained from the analysis of continuous movies (see above). We obtained the errors of the residence times and fractions from the standard deviation of the resampled state spectra and Gaussian error propagation, if necessary.</p><h3 class="c-article__sub-heading" id="Sec24">Calculation of search times</h3><p>DNA-binding proteins follow a search mechanism of facilitated diffusion, which includes three-dimensional diffusion in the nucleoplasm and one-dimensional sliding along unspecific DNA, to search for their specific target sites<sup><a aria-label="Reference 112" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR112" id="ref-link-section-d21244865e3260" title="Berg, O. G., Winter, R. B. &amp; von Hippel, P. H. Diffusion-DRIVEN MECHANISMS OF PROTEIN TRANSLOCATION ON NUCLEIC ACids. 1. Models and Theory. Biochemistry 20, 6929–6948 (1981).">112</a></sup>. To estimate the search time <i>τ</i><sub><i>search</i></sub>, that is the average time a DNA-binding protein needs to find any of its specific target sites, we considered an equilibrium binding model with two different types of binding sites, unspecific and specific<sup><a aria-label="Reference 114" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR114" id="ref-link-section-d21244865e3270" title="Loffreda, A. et al. Live-cell p53 single-molecule binding is modulated by C-terminal acetylation and correlates with transcriptional activity. Nat. Commun. 8, 313 (2017).">114</a></sup>. In this model, the DNA-binding protein performs the unspecific and specific binding reactions:</p><div class="c-article-equation" id="Equ4"><div class="c-article-equation__content"><span class="mathjax-tex">$$\left[T\right]+{\left[{D}_{u}\right]\,}\mathop{\leftrightarrows}\limits_{{k}_{{off},u}}^{{k}_{{on},u}}\,\left[T{D}_{u}\right]$$</span></div><div class="c-article-equation__number">
                    (4)
                </div></div><div class="c-article-equation" id="Equ5"><div class="c-article-equation__content"><span class="mathjax-tex">$$\left[T\right]+{\left[{D}_{s}\right]\,}\mathop{\leftrightarrows}\limits_{{k}_{{off},s}}^{{k}_{{on},s}}\,\left[T{D}_{s}\right]$$</span></div><div class="c-article-equation__number">
                    (5)
                </div></div><p>Were <i>[T]</i> is the concentration of the free protein, <i>[D</i><sub><i>u</i></sub><i>]</i> and <i>[D</i><sub><i>s</i></sub><i>]</i> are the concentrations of free unspecific and specific binding sites, <i>[TD</i><sub><i>u</i></sub><i>]</i> and <i>[TD</i><sub><i>s</i></sub><i>]</i> are the concentrations of protein-bound binding sites, <i>k</i><sub><i>on,u</i>,</sub> and <i>k</i><sub><i>on,s</i></sub> are the bimolecular association rates and <i>k</i><sub><i>off,u</i></sub> and <i>k</i><sub><i>off,s</i></sub> are the dissociation rates (see time-lapse microscopy analysis). If the concentration of binding sites is much larger than that of the DNA-binding protein and the bound fractions are small, both reactions are approximately decoupled.</p><p>The dissociation constant of either reaction is given by</p><div class="c-article-equation" id="Equ6"><div class="c-article-equation__content"><span class="mathjax-tex">$${K}_{d,i}=\,\frac{{k}_{{off},i}}{{k}_{{on},i}}=\,\frac{\left[T\right]\left[{D}_{i}\right]}{\left[T{D}_{i}\right]}$$</span></div><div class="c-article-equation__number">
                    (6)
                </div></div><p>with <i>i</i> = <i>u,s</i>. Thus, there is a relation between the ratio of kinetic rates and the ratio of the free fraction <i>p</i><sub><i>f</i></sub> (see continuous movie analysis above) to the bound fraction <i>p</i><sub><i>b,i</i></sub> (see time-lapse microscopy analysis above) of the DNA-binding protein:</p><div class="c-article-equation" id="Equ7"><div class="c-article-equation__content"><span class="mathjax-tex">$$\frac{{k}_{{off},i}}{{k}_{{on},i}\,[{D}_{i}]}=\,\frac{\left[T\right]}{\left[T{D}_{i}\right]}=\frac{{p}_{f}}{{p}_{b,i}}$$</span></div><div class="c-article-equation__number">
                    (7)
                </div></div><p>We set <i>k</i><sup><i>*</i></sup><sub><i>on,i</i></sub> <i>= k</i><sub><i>on,i</i></sub> <i>[D</i><sub><i>i</i></sub><i>]</i> in the following.</p><p>Within this search model, the target site search time <i>τ</i><sub><i>search</i></sub> of the DNA-binding protein depends on the time spent finding and binding to an unspecific binding site and the number of unspecific encounters before a specific target sequence is found. It can be estimated to<sup><a aria-label="Reference 113" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR113" id="ref-link-section-d21244865e3974" title="Livhitz, M. A., Gursky, G. V., Zasedatelev, A. S. &amp; Volkenstein, M. V. Equilibrium and kinetic aspects of protein-DNA recognition. Nucleic Acids Res. 6, 2217–2236 (1979).">113</a>,<a aria-label="Reference 114" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR114" id="ref-link-section-d21244865e3977" title="Loffreda, A. et al. Live-cell p53 single-molecule binding is modulated by C-terminal acetylation and correlates with transcriptional activity. Nat. Commun. 8, 313 (2017).">114</a></sup></p><div class="c-article-equation" id="Equ8"><div class="c-article-equation__content"><span class="mathjax-tex">$${\tau }_{{search}}=\,\frac{{N}_{{trials}}}{{k}_{{on},u}^{*}}+\frac{({N}_{{trials}}-1)}{{k}_{{off},u}}$$</span></div><div class="c-article-equation__number">
                    (8)
                </div></div><p>where <i>N</i><sub><i>trial</i></sub> is the number of unspecific encounters. It is given by</p><div class="c-article-equation" id="Equ9"><div class="c-article-equation__content"><span class="mathjax-tex">$${N}_{{trials}}=\frac{1}{{A}_{s}^{e}}$$</span></div><div class="c-article-equation__number">
                    (9)
                </div></div><p>with <i>A</i><sup><i>e</i></sup><sub><i>s</i></sub> the amplitude of the long-binding events in the GRID event spectrum of dissociation rates (see time-lapse microscopy analysis).</p><p>For CTCF, which follows a search mechanism that is formally equivalent to the mechanism of facilitated diffusion<sup><a aria-label="Reference 44" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR44" id="ref-link-section-d21244865e4248" title="Davidson, I. F. et al. CTCF is a DNA-tension-dependent barrier to cohesin-mediated loop extrusion. Nature 616, 822–827 (2023).">44</a>,<a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR46" id="ref-link-section-d21244865e4251" title="Hansen, A. S., Pustova, I., Cattoglio, C., Tjian, R. &amp; Darzacq, X. CTCF and cohesin regulate chromatin loop stability with distinct dynamics. Elife 6, e25776 (2017).">46</a></sup>, we also used this formalism. We obtained the search time <i>τ</i><sub><i>search</i></sub> from Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ8">8</a>) by calculating <i>k</i><sup><i>*</i></sup><sub><i>on,u</i>,</sub> and <i>N</i><sub><i>trial</i></sub> with Eqs. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ7">7</a>) and (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ9">9</a>) and inserting the measured parameters <i>A</i><sup><i>e</i></sup><sub><i>s</i></sub><i>, k</i><sub><i>off,u</i></sub>, <i>p</i><sub><i>f</i>,</sub> and <i>p</i><sub><i>b,u</i></sub>. All measured parameters are provided in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">15</a>.</p><p>For cohesin, a detailed model of the search mechanism is missing. Since we observed distinct short and long binding events for cohesin similar to CTCF, we considered the same equilibrium binding model with two types of binding events. From this model, we obtained the search time <i>τ</i><sub><i>search</i></sub> of cohesin by calculating <i>k</i><sup><i>*</i></sup><sub><i>on,s</i></sub> directly with Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ7">7</a>), the measured dissociation rate of long binding events <i>k</i><sub><i>off,s</i></sub> and the fractions <i>p</i><sub><i>f</i></sub> and <i>p</i><sub><i>b,s</i></sub>, and taking the inverse <i>τ</i><sub><i>search</i></sub> = 1/<i>k</i><sup><i>*</i></sup><sub><i>on,s</i></sub>. All parameters inserted in this equation can be found in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">16</a>.</p><h3 class="c-article__sub-heading" id="Sec25">Calculation of the relative effective search time</h3><p>For our polymer simulations, we additionally estimated the effective search time <i>τ</i><sub><i>effective</i></sub> of any cohesin or CTCF molecule to find any long-binding site by including estimates of the protein concentrations. With the number of molecules <i>N</i><sub><i>mol</i></sub> per nuclear volume <i>V</i><sub><i>nuc</i></sub>, the effective search time is given by:</p><div class="c-article-equation" id="Equ10"><div class="c-article-equation__content"><span class="mathjax-tex">$${\tau }_{{effective}}=\,{\tau }_{{search}}{V}_{{nuc}}\,{N}_{{mol}}^{-1}$$</span></div><div class="c-article-equation__number">
                    (10)
                </div></div><p>We obtained the nuclear volumes for pre- and post-ZGA stages from the area of the cross-section given by the eGFP-Lap2β signal in HT-Rad21 ITM movies (Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">27</a>) and assuming a spherical nuclear shape<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR52" id="ref-link-section-d21244865e4528" title="Reisser, M. et al. Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development. Nat. Commun. 9, 5218 (2018).">52</a></sup>.</p><p>For the more elliptical nuclei in the shield stage, we obtained the nuclear volume from the signal of GFP-H2B in z-scans with a Lattice Light-Sheet microscope (see section below).</p><p>The amount of molecules <i>N</i><sub><i>mol</i></sub> at different developmental stages of zebrafish embryos were approximated from published quantification data of Western blots<sup><a aria-label="Reference 84" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR84" id="ref-link-section-d21244865e4544" title="Meier, M. et al. Cohesin facilitates zygotic genome activation in zebrafish. Dev 145, 1–13 (2018).">84</a></sup>. We used the Rad21 and CTCF protein levels of stages closest to our pooled stages, namely 2.5 hpf for pre-ZGA, 3.3 hpf for post-ZGA, and 5.4 hpf for shield stage. We normalized to the effective search time in pre-ZGA to obtain relative effects of the effective search times. Gaussian error propagation was used for error estimations.</p><h3 class="c-article__sub-heading" id="Sec26">Calculation of long-bound fractions from kinetic rates</h3><p>The overall bound fraction of short- and long-bound molecules, <i>f</i><sub><i>b</i></sub> (see continuous movie analysis), is given by</p><div class="c-article-equation" id="Equ11"><div class="c-article-equation__content"><span class="mathjax-tex">$${f}_{b}=\,\frac{\left[T{D}_{s}\right]+\,\left[T{D}_{u}\right]}{\left[T{D}_{s}\right]+\left[T{D}_{u}\right]+[T]}$$</span></div><div class="c-article-equation__number">
                    (11)
                </div></div><p>and the relative bound fraction of long-binding events, A<sup>s</sup><sub>s</sub> (see time-lapse microscopy analysis), by</p><div class="c-article-equation" id="Equ12"><div class="c-article-equation__content"><span class="mathjax-tex">$${A}_{s}^{s}=\,\frac{[T{D}_{s}]}{\left[T{D}_{s}\right]+[T{D}_{u}]}$$</span></div><div class="c-article-equation__number">
                    (12)
                </div></div><p>Thus, the overall fraction of long-bound molecules, <i>p</i><sub><i>b,s</i></sub>, is</p><div class="c-article-equation" id="Equ13"><div class="c-article-equation__content"><span class="mathjax-tex">$${p}_{b,s}={f}_{b}*\,{A}_{s}^{s}=\,\frac{\left[T{D}_{s}\right]}{\left[T{D}_{s}\right]+\left[T{D}_{u}\right]+\left[T\right]}$$</span></div><div class="c-article-equation__number">
                    (13)
                </div></div><p>Using Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ7">7</a>), the expressions for <i>f</i><sub><i>b</i></sub> (Eq. <a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ11">11</a>) and <i>A</i><sup><i>s</i></sup><sub><i>s</i></sub> (Eq. <a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ12">12</a>) can be expressed in terms of kinetic rates, and <i>p</i><sub><i>b,s</i></sub> becomes</p><div class="c-article-equation" id="Equ14"><div class="c-article-equation__content"><span class="mathjax-tex">$${p}_{b,s}=\,\frac{{k}_{{on},s}^{*}/{k}_{{off},s}}{{k}_{{on},s}^{*}/{k}_{{off},s}+{k}_{{on},u}^{*}/{k}_{{off},u}+1}$$</span></div><div class="c-article-equation__number">
                    (14)
                </div></div><p>Notably, the on-rates <i>k</i><sup><i>*</i></sup><sub><i>on,i</i></sub><i>, i</i> = <i>u,s</i>, include the concentration of binding sites, which depend on the number of accessible binding sites and the nuclear volume.</p><p>We compared <i>p</i><sub><i>b,s</i></sub> obtained by Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ13">13</a>) by inserting the measured bound fractions and <i>p</i><sub><i>b,s</i></sub> obtained by Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ14">14</a>) by inserting the kinetic rates.</p><h3 class="c-article__sub-heading" id="Sec27">Lattice Light-Sheet microscopy (LLSM): nuclear volume determination</h3><p>We obtained nuclear volumes for pre- and post-ZGA measurements by assuming a spherical nucleus shape and calculating the sphere volume using the nuclear membrane eGFP-Lap2β signal as the cross-sectional area. Shield stage nuclei, however, are primarily elliptical and prevent volume estimations based on the nuclear cross-section alone. Therefore, we switched to lattice light-sheet microscopy to determine the nuclear volume from image stacks of GFP-labeled histone H2B. We injected zebrafish embryos in the 1-cell stage with 18.5 pg of mRNA encoding for GFP-H2B and incubated embryos that passed the 2-cell stage at 28 °C in an incubator until the shield stage.</p><p>To account for the larger size of zebrafish embryos compared to cells, we devised a method to mount zebrafish embryos on the lattice light sheet microscope. This was necessary due to its orthogonal geometry of excitation and detection objectives<sup><a aria-label="Reference 150" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR150" id="ref-link-section-d21244865e5284" title="Chen, B. C. et al. Lattice light-sheet microscopy: Imaging molecules to embryos at high spatiotemporal resolution. Science. 346, 1257998 (2014).">150</a></sup>. We placed zebrafish embryos upright on the sample holder by mounting them in agarose wells without fully covering the embryo, thereby preventing possible aberrations. We glued a 5 mm glass coverslip to the bottom of the sample holder to form a trough. This trough was slightly overfilled with 3% of low-gelling temperature agarose (A9045, Sigma-Aldrich) dissolved in Danieau’s medium. To insert wells into the agarose, we immediately placed the sample holder upside down on a custom 3D print (see Supplementary Fig. <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">27</a>). The 3D print contained notches to center the sample holder and 700 µm tall vertical pins, which stamped small wells into the gelling agarose. We filled the agarose wells with Danieau’s buffer and transferred a single embryo to a well with the animal cap pointing up. Ideally, the yolk reached the bottom of the well, and the animal cap protruded out of the well. This approach did not require adjustments to the sample holder setup, since the animal cap is mounted at a similar spatial location as cells.</p><p>We used a lattice-light sheet microscope custom-built according to a published design<sup><a aria-label="Reference 150" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR150" id="ref-link-section-d21244865e5294" title="Chen, B. C. et al. Lattice light-sheet microscopy: Imaging molecules to embryos at high spatiotemporal resolution. Science. 346, 1257998 (2014).">150</a></sup>. To image the nuclear histone marker GFP-H2B of shield stage embryos, GFP was excited with a dithered light sheet from a 488 nm laser (Genesis MX488 STM OPSL, Coherent Europe B.V.) that was formed by a square lattice and an annular mask with an outer NA of 0.55 and an inner NA of 0.52. We recorded image stacks scanning multiple nuclear volumes with a step size of 332 nm and 50 ms exposure time. Image stacks were deskewed and deconvolved using the lattice light-sheet post-processing utility LLSpy<sup><a aria-label="Reference 151" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR151" id="ref-link-section-d21244865e5298" title="Lambert, Talley Shao, L. &amp; VolkerH. tlambert03/LLSpy: v0.4.8 (0.4.8). 
                  https://doi.org/10.5281/zenodo.3554482
                  
                 (2019).">151</a></sup>. We then performed nuclear volume analysis in FIJI<sup><a aria-label="Reference 149" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR149" id="ref-link-section-d21244865e5302" title="Schindelin, J. et al. Fiji: An open-source platform for biological-image analysis. Nat. Methods 9, 676–682 (2012).">149</a></sup> by cropping image stacks to exclude frames at the start or end of a movie without a stable GFP signal, followed by equalizing irregular H2B signal with a median filter (20 px) and background subtraction (1 px). Leveraging the 3D ImageJ Suite<sup><a aria-label="Reference 152" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR152" id="ref-link-section-d21244865e5306" title="Ollion, J., Cochennec, J., Loll, F., Escudé, C. &amp; Boudier, T. TANGO: A generic tool for high-throughput 3D image analysis for studying nuclear organization. Bioinformatics 29, 1840–1841 (2013).">152</a></sup>, we segmented nuclei in every image stack by thresholding the H2B signal. Interphase nuclei were manually selected, and their volume analysis was done with FIJIs’ integrated 3D objects counter.</p><h3 class="c-article__sub-heading" id="Sec28">Center-Border-Distance (CBD) analysis</h3><p>We determined the relative position of long-bound tracks between the center of a nucleus and its border.</p><p>We started by pooling ITM and TACO movies in three stages: pre-ZGA (64-cell, 128-cell, 256-cell, 512-cell), post-ZGA (high, oblong, sphere), and shield stage. Next, we manually identified the nuclear outline at the first, middle, and last frame of a movie based on the nuclear lamina signal (eGFP-Lap2β), and interpolated the shape between these outlines for every frame of the movie to account for cell movement. We proceeded with tracks of the long-bound classification (see ITM: Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig1">1f</a>, TACO: Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Fig2">2a</a>). Since nuclei showed different sizes and forms, we normalized the CBD of tracks between the nucleus center (CBD = 0) and border (CBD = 1). The CBD distances were determined for every track based on a vector spanning the center of mass of the current nuclear outline, the initial xy-position of a track, and the border of the current nuclear outline. Finally, we sorted track-CBDs in a histogram with five ring-shaped area bins. To account for the increasing area of bins closer to the border, we divided the track counts of a bin with the bin area and further normalized the values by the largest histogram value. The errors represented the statistical counting error and were calculated as the square root of track counts per bin.</p><h3 class="c-article__sub-heading" id="Sec29">Identification of accessible CTCF sites from ATAC-seq and ChiP-seq data</h3><p>We identified CTCF sites in open chromatin regions to be used in our simulations.</p><p>For ATAC-seq analysis and peak calling, we used published ATAC-seq pairs end fastq reads from the Gene Expression Omnibus database (GEO) with the GEO accession number GSE130944<sup><a aria-label="Reference 94" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR94" id="ref-link-section-d21244865e5339" title="Pálfy, M., Schulze, G., Valen, E. &amp; Vastenhouw, N. L. Chromatin accessibility established by Pou5f3, Sox19b and Nanog primes genes for activity during zebrafish genome activation. PLoS Genet. 16, 1–25 (2020).">94</a></sup>. For the analysis, the developmental stages 256-cell (SRR9032650, SRR9032651), oblong (SRR9032665, SRR9032666) and shield stage (SRR9032668, SRR9032669, SRR9032670, SRR9032671, SRR9032672, SRR9032673) were used. All ATAC-seq fastq files were analyzed according to the pipeline described in the original publication. Next, we aligned Fastq files to zebrafish reference genome danRer10 (see NCBI RefSeq assembly GCF_000002035.5) using the bowtie2 tool<sup><a aria-label="Reference 153" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR153" id="ref-link-section-d21244865e5343" title="Langmead, B. &amp; Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. Nat. Methods 9, 357–359 (2012).">153</a></sup> with the parameters -X 2000 --no-mixed --no-discordant. Peak calling was performed after filtering out duplicates and low-quality alignments with <i>q</i> &lt; 30. Finally, ATAC peaks were called using MACS2<sup><a aria-label="Reference 154" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR154" id="ref-link-section-d21244865e5350" title="Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). Genome Biol. 9, R137 (2008).">154</a></sup> with parameters --no-model --no-lambda, and a cutoff False Discovery Rate (FDR) of 5% was applied using the parameter --q 0.05. As a control (-c), a digestion of genomic DNA was used (SRR9032640). The common peaks of each replicate from each developmental stage were used for further downstream analysis.</p><p>To extract the ATAC signal, Chromosome 18 was digested in 1 kb windows with the command bedtools makewindows<sup><a aria-label="Reference 155" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR155" id="ref-link-section-d21244865e5357" title="Quinlan, A. R. &amp; Hall, I. M. BEDTools: A flexible suite of utilities for comparing genomic features. Bioinformatics 26, 841–842 (2010).">155</a></sup>. For each window, the signal from the bigwig of each developmental stage was calculated (256, oblong, and shield from GEO: GSE130944) with bigWigSummary.</p><p>For CTCF orientation analysis, CTCF peaks and CTCF motifs were obtained from a published data set with the GEO accession number GSE133437<sup><a aria-label="Reference 118" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR118" id="ref-link-section-d21244865e5364" title="Pérez-Rico, Y. A., Barillot, E. &amp; Shkumatava, A. Demarcation of topologically associating domains is uncoupled from enriched CTCF binding in developing Zebrafish. iScience 23, 101046 (2020).">118</a></sup>. Each peak center was extended +/− 5 kb, and CTCF orientation per extended peak was extracted using homer<sup><a aria-label="Reference 156" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR156" id="ref-link-section-d21244865e5368" title="Heinz, S. et al. Simple Combinations of lineage-determining transcription factors Prime cis-regulatory elements required for macrophage and B cell identities. Mol. Cell 38, 576–589 (2010).">156</a></sup> and the parameters –find known_zebra_motif and –size given. The CTCF motif used is given in Supplementary Table <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM1">28</a>.</p><h3 class="c-article__sub-heading" id="Sec30">Polymer simulations</h3><p>For the polymer-chain model, we used Langevin molecular dynamics simulations of beads connected with springs implemented in LAMMPS<sup><a aria-label="Reference 157" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR157" id="ref-link-section-d21244865e5383" title="Plimpton, S. Fast Parallel Algorithms For Short-range Molecular Dynamics. J. Comput. Phys. 117, 1–19 (1995).">157</a></sup>. To simulate volume interactions between beads, we utilized the Lennard-Jones potential in Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ15">15</a>) with coefficients representing θ-solvent<sup><a aria-label="Reference 158" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR158" id="ref-link-section-d21244865e5390" title="De Gennes, P. G. Scaling Concepts in Polymer Physics. Cornell University Press (1979).">158</a></sup></p><div class="c-article-equation" id="Equ15"><div class="c-article-equation__content"><span class="mathjax-tex">$${U}_{{LJ}}\left(r\right)=4\epsilon \left[{\left(\sigma /r\right)}^{12}-{\left(\sigma /r\right)}^{6}\right]$$</span></div><div class="c-article-equation__number">
                    (15)
                </div></div><p>Where <span class="mathjax-tex">\(\epsilon\)</span> is the potential well depth, <span class="mathjax-tex">\(\sigma\)</span> the distance at which the potential vanishes (cutoff radius), and <span class="mathjax-tex">\(r\)</span> the distance between beads. The Lennard-Jones potential prevents the chain from self-intersections.</p><p>Bonds between beads were simulated using the harmonic potential given in Eq. (<a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Equ16">16</a>):</p><div class="c-article-equation" id="Equ16"><div class="c-article-equation__content"><span class="mathjax-tex">$${U}_{H}=\frac{k}{2}{\left(r-b\right)}^{2}$$</span></div><div class="c-article-equation__number">
                    (16)
                </div></div><p>Where <span class="mathjax-tex">\(k\)</span> is the spring constant, r the distance between beads and b the distance at which the potential is zero. We used a relatively high <span class="mathjax-tex">\(k=40\)</span> that prevents stretches allowing self-intersections.</p><p>Loop extrusion was simulated as a process comprised of 4 steps:</p><ol class="u-list-style-none">
<li>
<span class="u-custom-list-number">1.</span>
<p>Extruding molecules are stochastically loaded onto an adjacent pair of polymer beads every 400k simulation steps ( ~ 4 sec). We set the constant probability representing loading rate which we varied. Extruders are simulated as additional bonds between beads that can slide along the chain.</p>
</li>
<li>
<span class="u-custom-list-number">2.</span>
<p>Every 400k simulation steps, each extruder makes a step to increase the size of the loop. This step results in an extrusion processivity of 0.5 kb/sec. Starting from the initial loop [i, i + 1], the extruder will bind [i-1, i + 2] in the next step and so on.</p>
</li>
<li>
<span class="u-custom-list-number">3.</span>
<p>Every 400k simulation steps, each extruder can be unloaded with a constant probability. This probability determines the residence time of the extruder.</p>
</li>
<li>
<span class="u-custom-list-number">4.</span>
<p>When two loops encounter each other during loop extrusion on the chain, for instance loop 1 [i, i + 10] and loop 2 [i + 11, i + 20], they cannot pass one another. The common base of loop 1 [i + 10] and loop 2 [i + 11] becomes stationary, whereas the loop can continue extruding at the other end of loop 1 [i] and loop 2 [i + 20].</p>
</li>
</ol><p>We did not include CTCF and the loop-extrusion blocking activities of other complexes, such as PolII<sup><a aria-label="Reference 88" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR88" id="ref-link-section-d21244865e5715" title="Zhang, S., Übelmesser, N., Barbieri, M. &amp; Papantonis, A. Enhancer–promoter contact formation requires RNAPII and antagonizes loop extrusion. Nat. Genet. 55, 832–840 (2023).">88</a>,<a aria-label="Reference 159" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR159" id="ref-link-section-d21244865e5718" title="Barshad, G. et al. RNA polymerase II dynamics shape enhancer-promoter interactions. Nat. Genet. 
                  https://doi.org/10.1038/s41588-023-01442-7
                  
                 (2023).">159</a></sup> and the MCM complex<sup><a aria-label="Reference 160" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR160" id="ref-link-section-d21244865e5722" title="Dequeker, B. J. H. et al. MCM complexes are barriers that restrict cohesin-mediated loop extrusion. Nature 606, 197–203 (2022).">160</a></sup>, nor any other features that can pause or compete with loop extrusion activity.</p><p>We had to map arbitrary units of space and time to real units. We chose 1 bead to represent 1 kb of chromatin. Considering the size of a nucleosome with a linker of ~15 nm and assuming that the chain of nucleosomes is folded in a random walk configuration, we estimated that 1 bead corresponds to <span class="mathjax-tex">\(15*\sqrt{5}\)</span> nm <span class="mathjax-tex">\(\approx 33.5\)</span> nm. To map simulation onto experimental time, we used previously acquired data for mouse embryonic stem cells, where we determined the time it takes for a chromosome segment of 8 kb to move about its own size<sup><a aria-label="Reference 103" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR103" id="ref-link-section-d21244865e5780" title="Mach, P. et al. Cohesin and CTCF control the dynamics of chromosome folding. Nat. Genet. 54, 1907–1918 (2022).">103</a></sup>. We estimated that for the current system and resolution, 100k steps correspond to 1 sec. We fixed extrusion processivity at 0.5 kb/sec and we varied loading and unloading rates, as described above.</p><p>To test the effect of CTCF sites we remapped our system to 10 kb. Accessible and occupied CTCF sites were identified as described above for three developmental stages, namely 2.25, 4, and 5.3 hpf. Since the permeability of these sites to loop extrusion is a priori variable and unknown, we did not account for such variability and simulated a simplified model with impermeable sites. The time was mapped as for 1 kb simulations, yielding that 40k simulation steps corresponded to ~1 sec.</p><h3 class="c-article__sub-heading" id="Sec31">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41467-025-56889-5#MOESM11">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
</div>
<div class="u-mt-32">
<section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
<p>Source data are provided with this paper as a separate file named ‘Source Data.xlsx’. All single-particle tracking data and simulation data are freely available at Dryad [<a href="https://doi.org/10.5061/dryad.3bk3j9ks8">https://doi.org/10.5061/dryad.3bk3j9ks8</a>]<sup><a aria-label="Reference 161" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR161" id="ref-link-section-d21244865e5879" title="Coßmann, J. et al. Increasingly efficient chromatin binding of cohesin and CTCF supports chromatin architecture formation during zebrafish embryogenesis [Dataset]. 
                  https://doi.org/10.5061/dryad.3bk3j9ks8
                  
                 (2025).">161</a></sup>. Data supporting the findings of this manuscript are additionally available from the corresponding author upon reasonable request. <a data-track="click" data-track-action="section anchor" data-track-label="link" href="/articles/s41467-025-56889-5#Sec33">Source data</a> are provided with this paper.</p>
</div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
<p>The single-molecule tracking software TrackIt is freely available on GitLab [<a href="https://gitlab.com/GebhardtLab/TrackIt">https://gitlab.com/GebhardtLab/TrackIt</a>] or Zenodo [<a href="https://zenodo.org/records/7092296">https://zenodo.org/records/7092296</a>]<sup><a aria-label="Reference 162" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41467-025-56889-5#ref-CR162" id="ref-link-section-d21244865e5908" title="Kuhn, T., Hettich, J., Davtyan, R. &amp; Gebhardt, J. C. M. TrackIt\_v1.5. 
                  https://doi.org/10.5281/zenodo.7092296
                  
                 (2022).">162</a></sup>. For polymer modeling, LAMMPS with an implemented Loop Extrusion module is freely available on GitHub [<a href="https://github.com/polly-code/lammps_le">https://github.com/polly-code/lammps_le</a>].</p>
</div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Cremer, T. &amp; Cremer, M. Chromosome territories. <i>Cold Spring Harb. Perspect. Biol.</i> <b>2</b>, a003889 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 1" data-doi="10.1101/cshperspect.a003889" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1101/cshperspect.a003889" data-track-label="10.1101/cshperspect.a003889" data-track-value="article reference" href="https://doi.org/10.1101%2Fcshperspect.a003889" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 1" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20300217" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 1" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2829961" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 1" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromosome%20territories&amp;journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&amp;doi=10.1101%2Fcshperspect.a003889&amp;volume=2&amp;publication_year=2010&amp;author=Cremer%2CT&amp;author=Cremer%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Lieberman-Aiden, E. et al. Comprehensive mapping of long-range interactions reveals folding principles of the human genome. <i>Science</i> <b>326</b>, 289–293 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 2" data-doi="10.1126/science.1181369" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.1181369" data-track-label="10.1126/science.1181369" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.1181369" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 2" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2009Sci...326..289L" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 2" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXht1agsbjM" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 2" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19815776" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 2" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2858594" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 2" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0653.35032" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 2" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Comprehensive%20mapping%20of%20long-range%20interactions%20reveals%20folding%20principles%20of%20the%20human%20genome&amp;journal=Science&amp;doi=10.1126%2Fscience.1181369&amp;volume=326&amp;pages=289-293&amp;publication_year=2009&amp;author=Lieberman-Aiden%2CE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Zhang, Y. et al. Spatial organization of the mouse genome and its role in recurrent chromosomal translocations. <i>Cell</i> <b>148</b>, 908–921 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 3" data-doi="10.1016/j.cell.2012.02.002" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2012.02.002" data-track-label="10.1016/j.cell.2012.02.002" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2012.02.002" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 3" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XisFyit7Y%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 3" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22341456" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 3" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3320767" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 3" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1274.93278" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 3" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Spatial%20organization%20of%20the%20mouse%20genome%20and%20its%20role%20in%20recurrent%20chromosomal%20translocations&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2012.02.002&amp;volume=148&amp;pages=908-921&amp;publication_year=2012&amp;author=Zhang%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Dixon, J. R. et al. Topological domains in mammalian genomes identified by analysis of chromatin interactions. <i>Nature</i> <b>485</b>, 376–380 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 4" data-doi="10.1038/nature11082" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature11082" data-track-label="10.1038/nature11082" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature11082" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 4" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012Natur.485..376D" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 4" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XlslShurw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 4" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22495300" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 4" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3356448" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 4" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0211.37401" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 4" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Topological%20domains%20in%20mammalian%20genomes%20identified%20by%20analysis%20of%20chromatin%20interactions&amp;journal=Nature&amp;doi=10.1038%2Fnature11082&amp;volume=485&amp;pages=376-380&amp;publication_year=2012&amp;author=Dixon%2CJR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Nora, E. P. et al. Spatial partitioning of the regulatory landscape of the X-inactivation centre. <i>Nature</i> <b>485</b>, 381–385 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 5" data-doi="10.1038/nature11049" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature11049" data-track-label="10.1038/nature11049" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature11049" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 5" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012Natur.485..381N" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 5" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XlslSisrw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 5" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22495304" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 5" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3555144" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 5" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1281.62090" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 5" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Spatial%20partitioning%20of%20the%20regulatory%20landscape%20of%20the%20X-inactivation%20centre&amp;journal=Nature&amp;doi=10.1038%2Fnature11049&amp;volume=485&amp;pages=381-385&amp;publication_year=2012&amp;author=Nora%2CEP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Rao, S. S. P. et al. A 3D map of the human genome at kilobase resolution reveals principles of chromatin looping. <i>Cell</i> <b>159</b>, 1665–1680 (2014).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 6" data-doi="10.1016/j.cell.2014.11.021" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2014.11.021" data-track-label="10.1016/j.cell.2014.11.021" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2014.11.021" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 6" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFCrtrrK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 6" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25497547" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 6" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5635824" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 6" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0663.68043" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 6" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%203D%20map%20of%20the%20human%20genome%20at%20kilobase%20resolution%20reveals%20principles%20of%20chromatin%20looping&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2014.11.021&amp;volume=159&amp;pages=1665-1680&amp;publication_year=2014&amp;author=Rao%2CSSP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Sanyal, A., Lajoie, B. R., Jain, G. &amp; Dekker, J. The long-range interaction landscape of gene promoters. <i>Nature</i> <b>489</b>, 109–113 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 7" data-doi="10.1038/nature11279" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature11279" data-track-label="10.1038/nature11279" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature11279" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 7" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2012Natur.489..109S" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 7" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhtlGnsLnM" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 7" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22955621" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 7" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3555147" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 7" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20long-range%20interaction%20landscape%20of%20gene%20promoters&amp;journal=Nature&amp;doi=10.1038%2Fnature11279&amp;volume=489&amp;pages=109-113&amp;publication_year=2012&amp;author=Sanyal%2CA&amp;author=Lajoie%2CBR&amp;author=Jain%2CG&amp;author=Dekker%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Ricci, M. A., Manzo, C., García-Parajo, M. F., Lakadamyali, M. &amp; Cosma, M. P. Chromatin fibers are formed by heterogeneous groups of nucleosomes in vivo. <i>Cell</i> <b>160</b>, 1145–1158 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 8" data-doi="10.1016/j.cell.2015.01.054" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2015.01.054" data-track-label="10.1016/j.cell.2015.01.054" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2015.01.054" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 8" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXks1agur4%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 8" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25768910" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 8" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromatin%20fibers%20are%20formed%20by%20heterogeneous%20groups%20of%20nucleosomes%20in%20vivo&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2015.01.054&amp;volume=160&amp;pages=1145-1158&amp;publication_year=2015&amp;author=Ricci%2CMA&amp;author=Manzo%2CC&amp;author=Garc%C3%ADa-Parajo%2CMF&amp;author=Lakadamyali%2CM&amp;author=Cosma%2CMP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Ou, H. D. et al. ChromEMT: Visualizing 3D chromatin structure and compaction in interphase and mitotic cells. <i>Science</i> <b>357</b>, 139–148 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 9" data-doi="10.1126/science.aag0025" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.aag0025" data-track-label="10.1126/science.aag0025" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.aag0025" rel="nofollow noopener">Article</a> 
    <a aria-label="MATH reference 9" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1313.11095" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 9" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=ChromEMT%3A%20Visualizing%203D%20chromatin%20structure%20and%20compaction%20in%20interphase%20and%20mitotic%20cells&amp;journal=Science&amp;doi=10.1126%2Fscience.aag0025&amp;volume=357&amp;pages=139-148&amp;publication_year=2017&amp;author=Ou%2CHD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Fudenberg, G. et al. Formation of chromosomal domains by loop extrusion. <i>Cell Rep.</i> <b>15</b>, 2038–2049 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 10" data-doi="10.1016/j.celrep.2016.04.085" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.celrep.2016.04.085" data-track-label="10.1016/j.celrep.2016.04.085" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.celrep.2016.04.085" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 10" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XotlentLw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 10" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27210764" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 10" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4889513" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 10" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1369.17020" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 10" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Formation%20of%20chromosomal%20domains%20by%20loop%20extrusion&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2016.04.085&amp;volume=15&amp;pages=2038-2049&amp;publication_year=2016&amp;author=Fudenberg%2CG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Splinter, E. et al. CTCF mediates long-range chromatin looping and local histone modification in the β-globin locus. <i>Genes Dev.</i> <b>20</b>, 2349–2354 (2006).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 11" data-doi="10.1101/gad.399506" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1101/gad.399506" data-track-label="10.1101/gad.399506" data-track-value="article reference" href="https://doi.org/10.1101%2Fgad.399506" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 11" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XptlSntrY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 11" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16951251" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 11" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1560409" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 11" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1109.81025" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 11" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CTCF%20mediates%20long-range%20chromatin%20looping%20and%20local%20histone%20modification%20in%20the%20%CE%B2-globin%20locus&amp;journal=Genes%20Dev.&amp;doi=10.1101%2Fgad.399506&amp;volume=20&amp;pages=2349-2354&amp;publication_year=2006&amp;author=Splinter%2CE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Rao, S. S. P. et al. Cohesin loss eliminates all loop domains. <i>Cell</i> <b>171</b>, 305–320.e24 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 12" data-doi="10.1016/j.cell.2017.09.026" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2017.09.026" data-track-label="10.1016/j.cell.2017.09.026" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.09.026" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 12" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1WqsrfN" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 12" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28985562" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 12" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5846482" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 12" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohesin%20loss%20eliminates%20all%20loop%20domains&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.09.026&amp;volume=171&amp;pages=305-320.e24&amp;publication_year=2017&amp;author=Rao%2CSSP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Sanborn, A. L. et al. Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes. <i>Proc. Natl Acad. Sci. USA</i> <b>112</b>, E6456–E6465 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 13" data-doi="10.1073/pnas.1518552112" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.1518552112" data-track-label="10.1073/pnas.1518552112" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.1518552112" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 13" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhslWks7%2FN" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 13" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26499245" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 13" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4664323" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 13" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?37.0701.01" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 13" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromatin%20extrusion%20explains%20key%20features%20of%20loop%20and%20domain%20formation%20in%20wild-type%20and%20engineered%20genomes&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1518552112&amp;volume=112&amp;pages=E6456-E6465&amp;publication_year=2015&amp;author=Sanborn%2CAL" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Peters, J. M. &amp; Nishiyama, T. Sister chromatid cohesion. <i>Cold Spring Harb. Perspect. Biol</i>. <b>4</b>, a011130 (2012).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Nasmyth, K. &amp; Haering, C. H. Cohesin: Its roles and mechanisms. <i>Annu. Rev. Genet.</i> <b>43</b>, 525–558 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 15" data-doi="10.1146/annurev-genet-102108-134233" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1146/annurev-genet-102108-134233" data-track-label="10.1146/annurev-genet-102108-134233" data-track-value="article reference" href="https://doi.org/10.1146%2Fannurev-genet-102108-134233" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 15" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsF2qtrbI" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 15" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19886810" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 15" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1079.74585" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 15" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohesin%3A%20Its%20roles%20and%20mechanisms&amp;journal=Annu.%20Rev.%20Genet.&amp;doi=10.1146%2Fannurev-genet-102108-134233&amp;volume=43&amp;pages=525-558&amp;publication_year=2009&amp;author=Nasmyth%2CK&amp;author=Haering%2CCH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Lee, H., Noh, H. &amp; Ryu, J.-K. Structure-function relationships of SMC protein complexes for DNA loop extrusion. <i>Biodesign</i> <b>9</b>, 1–13 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 16" data-doi="10.34184/kssb.2021.9.1.1" data-track="click_references" data-track-action="article reference" data-track-item_id="10.34184/kssb.2021.9.1.1" data-track-label="10.34184/kssb.2021.9.1.1" data-track-value="article reference" href="https://doi.org/10.34184%2Fkssb.2021.9.1.1" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 16" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXitVCrtL3O" rel="nofollow noopener">CAS</a> 
    <a aria-label="MATH reference 16" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1092.62102" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 16" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structure-function%20relationships%20of%20SMC%20protein%20complexes%20for%20DNA%20loop%20extrusion&amp;journal=Biodesign&amp;doi=10.34184%2Fkssb.2021.9.1.1&amp;volume=9&amp;pages=1-13&amp;publication_year=2021&amp;author=Lee%2CH&amp;author=Noh%2CH&amp;author=Ryu%2CJ-K" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Davidson, I. F. et al. DNA loop extrusion by human cohesin. <i>Science</i> <b>366</b>, 1338–1345 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 17" data-doi="10.1126/science.aaz3418" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.aaz3418" data-track-label="10.1126/science.aaz3418" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.aaz3418" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 17" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019Sci...366.1338D" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 17" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSnsLrL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 17" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31753851" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 17" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1355.68230" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 17" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20loop%20extrusion%20by%20human%20cohesin&amp;journal=Science&amp;doi=10.1126%2Fscience.aaz3418&amp;volume=366&amp;pages=1338-1345&amp;publication_year=2019&amp;author=Davidson%2CIF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Kim, Y., Shi, Z., Zhang, H., Finkelstein, I. J. &amp; Yu, H. Human cohesin compacts DNA by loop extrusion. <i>Science</i> <b>366</b>, 1345–1349 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 18" data-doi="10.1126/science.aaz4475" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.aaz4475" data-track-label="10.1126/science.aaz4475" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.aaz4475" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 18" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019Sci...366.1345K" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 18" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSnsLzE" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 18" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31780627" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 18" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7387118" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 18" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20cohesin%20compacts%20DNA%20by%20loop%20extrusion&amp;journal=Science&amp;doi=10.1126%2Fscience.aaz4475&amp;volume=366&amp;pages=1345-1349&amp;publication_year=2019&amp;author=Kim%2CY&amp;author=Shi%2CZ&amp;author=Zhang%2CH&amp;author=Finkelstein%2CIJ&amp;author=Yu%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Petela, N. J. et al. Scc2 is a potent activator of cohesin’s ATPase that promotes loading by binding Scc1 without Pds5. <i>Mol. Cell</i> <b>70</b>, 1134–1148.e7 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 19" data-doi="10.1016/j.molcel.2018.05.022" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2018.05.022" data-track-label="10.1016/j.molcel.2018.05.022" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2018.05.022" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 19" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018cqmd.book.....P" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 19" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtF2jtbjE" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 19" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29932904" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 19" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6028919" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 19" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0392.76063" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 19" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Scc2%20is%20a%20potent%20activator%20of%20cohesin%E2%80%99s%20ATPase%20that%20promotes%20loading%20by%20binding%20Scc1%20without%20Pds5&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2018.05.022&amp;volume=70&amp;pages=1134-1148.e7&amp;publication_year=2018&amp;author=Petela%2CNJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Rhodes, J., Mazza, D., Nasmyth, K. &amp; Uphoff, S. Scc2/Nipbl hops between chromosomal cohesin rings after loading. <i>Elife</i> <b>6</b>, 1–20 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 20" data-doi="10.7554/eLife.30000" data-track="click_references" data-track-action="article reference" data-track-item_id="10.7554/eLife.30000" data-track-label="10.7554/eLife.30000" data-track-value="article reference" href="https://doi.org/10.7554%2FeLife.30000" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 20" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Scc2%2FNipbl%20hops%20between%20chromosomal%20cohesin%20rings%20after%20loading&amp;journal=Elife&amp;doi=10.7554%2FeLife.30000&amp;volume=6&amp;pages=1-20&amp;publication_year=2017&amp;author=Rhodes%2CJ&amp;author=Mazza%2CD&amp;author=Nasmyth%2CK&amp;author=Uphoff%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Kueng, S. et al. Wapl controls the dynamic association of cohesin with chromatin. <i>Cell</i> <b>127</b>, 955–967 (2006).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 21" data-doi="10.1016/j.cell.2006.09.040" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2006.09.040" data-track-label="10.1016/j.cell.2006.09.040" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2006.09.040" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 21" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XhtlWktr%2FL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 21" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17113138" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 21" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1390.94254" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 21" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Wapl%20controls%20the%20dynamic%20association%20of%20cohesin%20with%20chromatin&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2006.09.040&amp;volume=127&amp;pages=955-967&amp;publication_year=2006&amp;author=Kueng%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Haarhuis, J. H. I. et al. The cohesin release factor WAPL restricts chromatin loop extension. <i>Cell</i> <b>169</b>, 693–707.e14 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 22" data-doi="10.1016/j.cell.2017.04.013" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2017.04.013" data-track-label="10.1016/j.cell.2017.04.013" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.04.013" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 22" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXnt1CmsL4%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 22" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28475897" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 22" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5422210" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 22" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20cohesin%20release%20factor%20WAPL%20restricts%20chromatin%20loop%20extension&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.04.013&amp;volume=169&amp;pages=693-707.e14&amp;publication_year=2017&amp;author=Haarhuis%2CJHI" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Rankin, S., Ayad, N. G. &amp; Kirschner, M. W. Sororin, a substrate of the anaphase-promoting complex, is required for sister chromatid cohesion in vertebrates. <i>Mol. Cell</i> <b>18</b>, 185–200 (2005).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 23" data-doi="10.1016/j.molcel.2005.03.017" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2005.03.017" data-track-label="10.1016/j.molcel.2005.03.017" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2005.03.017" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 23" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXjvFSrs7c%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 23" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15837422" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 23" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Sororin%2C%20a%20substrate%20of%20the%20anaphase-promoting%20complex%2C%20is%20required%20for%20sister%20chromatid%20cohesion%20in%20vertebrates&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2005.03.017&amp;volume=18&amp;pages=185-200&amp;publication_year=2005&amp;author=Rankin%2CS&amp;author=Ayad%2CNG&amp;author=Kirschner%2CMW" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Ladurner, R. et al. Sororin actively maintains sister chromatid cohesion. <i>EMBO J.</i> <b>35</b>, 635–653 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 24" data-doi="10.15252/embj.201592532" data-track="click_references" data-track-action="article reference" data-track-item_id="10.15252/embj.201592532" data-track-label="10.15252/embj.201592532" data-track-value="article reference" href="https://doi.org/10.15252%2Fembj.201592532" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 24" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XivVWnu74%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 24" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26903600" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 24" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4801952" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 24" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Sororin%20actively%20maintains%20sister%20chromatid%20cohesion&amp;journal=EMBO%20J.&amp;doi=10.15252%2Fembj.201592532&amp;volume=35&amp;pages=635-653&amp;publication_year=2016&amp;author=Ladurner%2CR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Nishiyama, T. et al. Sororin mediates sister chromatid cohesion by antagonizing Wapl. <i>Cell</i> <b>143</b>, 737–749 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 25" data-doi="10.1016/j.cell.2010.10.031" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2010.10.031" data-track-label="10.1016/j.cell.2010.10.031" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2010.10.031" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 25" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsVygsLbF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 25" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21111234" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 25" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1432.62078" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 25" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Sororin%20mediates%20sister%20chromatid%20cohesion%20by%20antagonizing%20Wapl&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2010.10.031&amp;volume=143&amp;pages=737-749&amp;publication_year=2010&amp;author=Nishiyama%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Schmitz, J., Watrin, E., Lénárt, P., Mechtler, K. &amp; Peters, J. M. Sororin is required for stable binding of cohesin to chromatin and for sister chromatid cohesion in interphase. <i>Curr. Biol.</i> <b>17</b>, 630–636 (2007).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 26" data-doi="10.1016/j.cub.2007.02.029" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cub.2007.02.029" data-track-label="10.1016/j.cub.2007.02.029" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cub.2007.02.029" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 26" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXjslGnu7k%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 26" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17349791" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 26" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Sororin%20is%20required%20for%20stable%20binding%20of%20cohesin%20to%20chromatin%20and%20for%20sister%20chromatid%20cohesion%20in%20interphase&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2007.02.029&amp;volume=17&amp;pages=630-636&amp;publication_year=2007&amp;author=Schmitz%2CJ&amp;author=Watrin%2CE&amp;author=L%C3%A9n%C3%A1rt%2CP&amp;author=Mechtler%2CK&amp;author=Peters%2CJM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Corsi, F., Rusch, E. &amp; Goloborodko, A. Loop extrusion rules: the next generation. <i>Curr. Opin. Genet. Dev</i>. <b>81</b>, 102061 (2023).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Davidson, I. F. &amp; Peters, J. M. Genome folding through loop extrusion by SMC complexes. <i>Nat. Rev. Mol. Cell Biol.</i> <b>22</b>, 445–464 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 28" data-doi="10.1038/s41580-021-00349-7" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41580-021-00349-7" data-track-label="10.1038/s41580-021-00349-7" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41580-021-00349-7" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 28" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXnsFKjs78%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 28" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33767413" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 28" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome%20folding%20through%20loop%20extrusion%20by%20SMC%20complexes&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;doi=10.1038%2Fs41580-021-00349-7&amp;volume=22&amp;pages=445-464&amp;publication_year=2021&amp;author=Davidson%2CIF&amp;author=Peters%2CJM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Kim, E., Barth, R. &amp; Dekker, C. Looping the genome with SMC complexes. <i>Annu. Rev. Biochem.</i> <b>92</b>, 15–41 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 29" data-doi="10.1146/annurev-biochem-032620-110506" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1146/annurev-biochem-032620-110506" data-track-label="10.1146/annurev-biochem-032620-110506" data-track-value="article reference" href="https://doi.org/10.1146%2Fannurev-biochem-032620-110506" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 29" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXpsVynsLw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 29" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37137166" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 29" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1543.46031" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 29" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Looping%20the%20genome%20with%20SMC%20complexes&amp;journal=Annu.%20Rev.%20Biochem.&amp;doi=10.1146%2Fannurev-biochem-032620-110506&amp;volume=92&amp;pages=15-41&amp;publication_year=2023&amp;author=Kim%2CE&amp;author=Barth%2CR&amp;author=Dekker%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Vietri Rudan, M. et al. Comparative Hi-C reveals that CTCF underlies evolution of chromosomal domain architecture. <i>Cell Rep.</i> <b>10</b>, 1297–1309 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 30" data-doi="10.1016/j.celrep.2015.02.004" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.celrep.2015.02.004" data-track-label="10.1016/j.celrep.2015.02.004" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.celrep.2015.02.004" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 30" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXjs1Omt7s%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 30" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25732821" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 30" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4542312" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 30" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1198.35214" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 30" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparative%20Hi-C%20reveals%20that%20CTCF%20underlies%20evolution%20of%20chromosomal%20domain%20architecture&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2015.02.004&amp;volume=10&amp;pages=1297-1309&amp;publication_year=2015&amp;author=Vietri%20Rudan%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">de Wit, E. et al. CTCF Binding polarity determines chromatin looping. <i>Mol. Cell</i> <b>60</b>, 676–684 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 31" data-doi="10.1016/j.molcel.2015.09.023" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2015.09.023" data-track-label="10.1016/j.molcel.2015.09.023" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2015.09.023" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 31" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26527277" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 31" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CTCF%20Binding%20polarity%20determines%20chromatin%20looping&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2015.09.023&amp;volume=60&amp;pages=676-684&amp;publication_year=2015&amp;author=Wit%2CE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Pugacheva, E. M. et al. CTCF mediates chromatin looping via N-terminal domain-dependent cohesin retention. <i>Proc. Natl Acad. Sci. USA</i> <b>117</b>, 2020–2031 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 32" data-doi="10.1073/pnas.1911708117" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.1911708117" data-track-label="10.1073/pnas.1911708117" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.1911708117" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 32" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020PNAS..117.2020P" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 32" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVOltrw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 32" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31937660" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 32" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6995019" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 32" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CTCF%20mediates%20chromatin%20looping%20via%20N-terminal%20domain-dependent%20cohesin%20retention&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1911708117&amp;volume=117&amp;pages=2020-2031&amp;publication_year=2020&amp;author=Pugacheva%2CEM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Nora, E. P. et al. Molecular basis of CTCF binding polarity in genome folding. <i>Nat. Commun.</i> <b>11</b>, 5612 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 33" data-doi="10.1038/s41467-020-19283-x" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-020-19283-x" data-track-label="10.1038/s41467-020-19283-x" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-020-19283-x" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 33" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatCo..11.5612N" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 33" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXit12hsrjP" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 33" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33154377" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 33" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7645679" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 33" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0934.82043" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 33" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20basis%20of%20CTCF%20binding%20polarity%20in%20genome%20folding&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-19283-x&amp;volume=11&amp;publication_year=2020&amp;author=Nora%2CEP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Li, Y. et al. The structural basis for cohesin–CTCF-anchored loops. <i>Nature</i> <b>578</b>, 472–476 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 34" data-doi="10.1038/s41586-019-1910-z" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41586-019-1910-z" data-track-label="10.1038/s41586-019-1910-z" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41586-019-1910-z" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 34" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020Natur.578..472L" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 34" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXislCnsLY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 34" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31905366" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 34" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7035113" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 34" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1451.65200" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 34" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20structural%20basis%20for%20cohesin%E2%80%93CTCF-anchored%20loops&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1910-z&amp;volume=578&amp;pages=472-476&amp;publication_year=2020&amp;author=Li%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Vallot, A. &amp; Tachibana, K. The emergence of genome architecture and zygotic genome activation. <i>Curr. Opin. Cell Biol.</i> <b>64</b>, 50–57 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 35" data-doi="10.1016/j.ceb.2020.02.002" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.ceb.2020.02.002" data-track-label="10.1016/j.ceb.2020.02.002" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.ceb.2020.02.002" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 35" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXjvVKgt7g%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 35" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32220807" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 35" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7374442" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 35" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0053.42905" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 35" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20emergence%20of%20genome%20architecture%20and%20zygotic%20genome%20activation&amp;journal=Curr.%20Opin.%20Cell%20Biol.&amp;doi=10.1016%2Fj.ceb.2020.02.002&amp;volume=64&amp;pages=50-57&amp;publication_year=2020&amp;author=Vallot%2CA&amp;author=Tachibana%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Du, Z. et al. Allelic reprogramming of 3D chromatin architecture during early mammalian development. <i>Nature</i> <b>547</b>, 232–235 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 36" data-doi="10.1038/nature23263" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature23263" data-track-label="10.1038/nature23263" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature23263" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 36" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017Natur.547..232D" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 36" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFOjt7zI" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 36" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28703188" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 36" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1084.53029" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 36" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Allelic%20reprogramming%20of%203D%20chromatin%20architecture%20during%20early%20mammalian%20development&amp;journal=Nature&amp;doi=10.1038%2Fnature23263&amp;volume=547&amp;pages=232-235&amp;publication_year=2017&amp;author=Du%2CZ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Ke, Y. et al. 3D Chromatin structures of mature gametes and structural reprogramming during mammalian Embryogenesis. <i>Cell</i> <b>170</b>, 367–381.e20 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 37" data-doi="10.1016/j.cell.2017.06.029" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2017.06.029" data-track-label="10.1016/j.cell.2017.06.029" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.06.029" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 37" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFygtbzO" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 37" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28709003" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 37" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0967.78008" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 37" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=3D%20Chromatin%20structures%20of%20mature%20gametes%20and%20structural%20reprogramming%20during%20mammalian%20Embryogenesis&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.06.029&amp;volume=170&amp;pages=367-381.e20&amp;publication_year=2017&amp;author=Ke%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Chen, X. et al. Key role for CTCF in establishing chromatin structure in human embryos. <i>Nature</i> <b>576</b>, 306–310 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 38" data-doi="10.1038/s41586-019-1812-0" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41586-019-1812-0" data-track-label="10.1038/s41586-019-1812-0" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41586-019-1812-0" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 38" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitlWku7jN" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 38" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31801998" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 38" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1428.35547" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 38" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Key%20role%20for%20CTCF%20in%20establishing%20chromatin%20structure%20in%20human%20embryos&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1812-0&amp;volume=576&amp;pages=306-310&amp;publication_year=2019&amp;author=Chen%2CX" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Hug, C. B., Grimaldi, A. G., Kruse, K. &amp; Vaquerizas, J. M. Chromatin architecture emerges during zygotic genome activation independent of transcription. <i>Cell</i> <b>169</b>, 216–228.e19 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 39" data-doi="10.1016/j.cell.2017.03.024" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2017.03.024" data-track-label="10.1016/j.cell.2017.03.024" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.03.024" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 39" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXlvVKiu7w%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 39" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28388407" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 39" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromatin%20architecture%20emerges%20during%20zygotic%20genome%20activation%20independent%20of%20transcription&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.03.024&amp;volume=169&amp;pages=216-228.e19&amp;publication_year=2017&amp;author=Hug%2CCB&amp;author=Grimaldi%2CAG&amp;author=Kruse%2CK&amp;author=Vaquerizas%2CJM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Wike, C. L. et al. Chromatin architecture transitions from zebrafish sperm through early embryogenesis. <i>Genome Res.</i> <b>31</b>, 981–994 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 40" data-doi="10.1101/gr.269860.120" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1101/gr.269860.120" data-track-label="10.1101/gr.269860.120" data-track-value="article reference" href="https://doi.org/10.1101%2Fgr.269860.120" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 40" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB2cXit1ejsbfM" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 40" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34006569" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 40" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8168589" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 40" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1482.11132" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 40" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromatin%20architecture%20transitions%20from%20zebrafish%20sperm%20through%20early%20embryogenesis&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.269860.120&amp;volume=31&amp;pages=981-994&amp;publication_year=2021&amp;author=Wike%2CCL" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Kaaij, L. J. T., van der Weide, R. H., Ketting, R. F. &amp; de Wit, E. Systemic loss and gain of Chromatin architecture throughout Zebrafish development. <i>Cell Rep.</i> <b>24</b>, 1–10.e4 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 41" data-doi="10.1016/j.celrep.2018.06.003" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.celrep.2018.06.003" data-track-label="10.1016/j.celrep.2018.06.003" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.celrep.2018.06.003" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 41" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1yis7jL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 41" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29972771" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 41" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6047509" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 41" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Systemic%20loss%20and%20gain%20of%20Chromatin%20architecture%20throughout%20Zebrafish%20development&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2018.06.003&amp;volume=24&amp;pages=1-10.e4&amp;publication_year=2018&amp;author=Kaaij%2CLJT&amp;author=Weide%2CRH&amp;author=Ketting%2CRF&amp;author=Wit%2CE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Nakamura, R. et al. CTCF looping is established during gastrulation in medaka embryos. <i>Genome Res.</i> <b>31</b>, 968–980 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 42" data-doi="10.1101/gr.269951.120" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1101/gr.269951.120" data-track-label="10.1101/gr.269951.120" data-track-value="article reference" href="https://doi.org/10.1101%2Fgr.269951.120" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 42" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB2cXitF2ksrjJ" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 42" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34006570" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 42" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8168583" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 42" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1485.91168" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 42" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CTCF%20looping%20is%20established%20during%20gastrulation%20in%20medaka%20embryos&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.269951.120&amp;volume=31&amp;pages=968-980&amp;publication_year=2021&amp;author=Nakamura%2CR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Su, Q. P. et al. Superresolution imaging reveals spatiotemporal propagation of human replication foci mediated by CTCF-organized chromatin structures. <i>Proc. Natl Acad. Sci. USA</i> <b>117</b>, 15036–15046 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 43" data-doi="10.1073/pnas.2001521117" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.2001521117" data-track-label="10.1073/pnas.2001521117" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.2001521117" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 43" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020PNAS..11715036S" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 43" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsV2gtbnF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 43" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32541019" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 43" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334649" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 43" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Superresolution%20imaging%20reveals%20spatiotemporal%20propagation%20of%20human%20replication%20foci%20mediated%20by%20CTCF-organized%20chromatin%20structures&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.2001521117&amp;volume=117&amp;pages=15036-15046&amp;publication_year=2020&amp;author=Su%2CQP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Davidson, I. F. et al. CTCF is a DNA-tension-dependent barrier to cohesin-mediated loop extrusion. <i>Nature</i> <b>616</b>, 822–827 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 44" data-doi="10.1038/s41586-023-05961-5" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41586-023-05961-5" data-track-label="10.1038/s41586-023-05961-5" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41586-023-05961-5" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 44" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2023Natur.616..822D" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 44" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXotFKrsr8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 44" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37076620" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 44" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10132984" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 44" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?62.1150.18" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 44" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CTCF%20is%20a%20DNA-tension-dependent%20barrier%20to%20cohesin-mediated%20loop%20extrusion&amp;journal=Nature&amp;doi=10.1038%2Fs41586-023-05961-5&amp;volume=616&amp;pages=822-827&amp;publication_year=2023&amp;author=Davidson%2CIF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Holzmann, J. et al. Absolute quantification of cohesin, CTCF and their regulators in human cells. <i>Elife</i> <b>8</b>, 1–31 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 45" data-doi="10.7554/eLife.46269" data-track="click_references" data-track-action="article reference" data-track-item_id="10.7554/eLife.46269" data-track-label="10.7554/eLife.46269" data-track-value="article reference" href="https://doi.org/10.7554%2FeLife.46269" rel="nofollow noopener">Article</a> 
    <a aria-label="MATH reference 45" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1010.68789" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 45" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Absolute%20quantification%20of%20cohesin%2C%20CTCF%20and%20their%20regulators%20in%20human%20cells&amp;journal=Elife&amp;doi=10.7554%2FeLife.46269&amp;volume=8&amp;pages=1-31&amp;publication_year=2019&amp;author=Holzmann%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Hansen, A. S., Pustova, I., Cattoglio, C., Tjian, R. &amp; Darzacq, X. CTCF and cohesin regulate chromatin loop stability with distinct dynamics. <i>Elife</i> <b>6</b>, e25776 (2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Wutz, G. et al. Topologically associating domains and chromatin loops depend on cohesin and are regulated by CTCF, WAPL, and PDS5 proteins. <i>EMBO J.</i> <b>36</b>, 3573–3599 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 47" data-doi="10.15252/embj.201798004" data-track="click_references" data-track-action="article reference" data-track-item_id="10.15252/embj.201798004" data-track-label="10.15252/embj.201798004" data-track-value="article reference" href="https://doi.org/10.15252%2Fembj.201798004" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 47" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFejtL3I" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 47" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29217591" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 47" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5730888" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 47" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1369.46028" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 47" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Topologically%20associating%20domains%20and%20chromatin%20loops%20depend%20on%20cohesin%20and%20are%20regulated%20by%20CTCF%2C%20WAPL%2C%20and%20PDS5%20proteins&amp;journal=EMBO%20J.&amp;doi=10.15252%2Fembj.201798004&amp;volume=36&amp;pages=3573-3599&amp;publication_year=2017&amp;author=Wutz%2CG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Agarwal, H., Reisser, M., Wortmann, C. &amp; Gebhardt, J. C. M. Direct observation of cell-cycle-dependent interactions between CTCF and Chromatin. <i>Biophys. J.</i> <b>112</b>, 2051–2055 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 48" data-doi="10.1016/j.bpj.2017.04.018" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.bpj.2017.04.018" data-track-label="10.1016/j.bpj.2017.04.018" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.bpj.2017.04.018" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 48" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017BpJ...112.2051A" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 48" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXntVKltLc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 48" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28487148" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 48" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5444008" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 48" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Direct%20observation%20of%20cell-cycle-dependent%20interactions%20between%20CTCF%20and%20Chromatin&amp;journal=Biophys.%20J.&amp;doi=10.1016%2Fj.bpj.2017.04.018&amp;volume=112&amp;pages=2051-2055&amp;publication_year=2017&amp;author=Agarwal%2CH&amp;author=Reisser%2CM&amp;author=Wortmann%2CC&amp;author=Gebhardt%2CJCM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Hansen, A. S., Amitai, A., Cattoglio, C., Tjian, R. &amp; Darzacq, X. Guided nuclear exploration increases CTCF target search efficiency. <i>Nat. Chem. Biol.</i> <b>16</b>, 257–266 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 49" data-doi="10.1038/s41589-019-0422-3" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41589-019-0422-3" data-track-label="10.1038/s41589-019-0422-3" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41589-019-0422-3" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 49" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXit1yjsrzP" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 49" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31792445" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 49" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Guided%20nuclear%20exploration%20increases%20CTCF%20target%20search%20efficiency&amp;journal=Nat.%20Chem.%20Biol.&amp;doi=10.1038%2Fs41589-019-0422-3&amp;volume=16&amp;pages=257-266&amp;publication_year=2020&amp;author=Hansen%2CAS&amp;author=Amitai%2CA&amp;author=Cattoglio%2CC&amp;author=Tjian%2CR&amp;author=Darzacq%2CX" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Gerlich, D., Koch, B., Dupeux, F., Peters, J. M. &amp; Ellenberg, J. Live-cell imaging reveals a stable cohesin-chromatin interaction after but not before DNA replication. <i>Curr. Biol.</i> <b>16</b>, 1571–1578 (2006).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 50" data-doi="10.1016/j.cub.2006.06.068" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cub.2006.06.068" data-track-label="10.1016/j.cub.2006.06.068" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cub.2006.06.068" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 50" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XotFWlsbs%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 50" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16890534" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 50" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Live-cell%20imaging%20reveals%20a%20stable%20cohesin-chromatin%20interaction%20after%20but%20not%20before%20DNA%20replication&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2006.06.068&amp;volume=16&amp;pages=1571-1578&amp;publication_year=2006&amp;author=Gerlich%2CD&amp;author=Koch%2CB&amp;author=Dupeux%2CF&amp;author=Peters%2CJM&amp;author=Ellenberg%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Golfier, S., Quail, T., Kimura, H. &amp; Brugués, J. Cohesin and condensin extrude DNA loops in a cell-cycle dependent manner. <i>Elife</i> <b>9</b>, 1–34 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 51" data-doi="10.7554/eLife.53885" data-track="click_references" data-track-action="article reference" data-track-item_id="10.7554/eLife.53885" data-track-label="10.7554/eLife.53885" data-track-value="article reference" href="https://doi.org/10.7554%2FeLife.53885" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 51" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohesin%20and%20condensin%20extrude%20DNA%20loops%20in%20a%20cell-cycle%20dependent%20manner&amp;journal=Elife&amp;doi=10.7554%2FeLife.53885&amp;volume=9&amp;pages=1-34&amp;publication_year=2020&amp;author=Golfier%2CS&amp;author=Quail%2CT&amp;author=Kimura%2CH&amp;author=Brugu%C3%A9s%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Reisser, M. et al. Single-molecule imaging correlates decreasing nuclear volume with increasing TF-chromatin associations during zebrafish development. <i>Nat. Commun.</i> <b>9</b>, 5218 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 52" data-doi="10.1038/s41467-018-07731-8" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-018-07731-8" data-track-label="10.1038/s41467-018-07731-8" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-018-07731-8" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 52" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018NatCo...9.5218R" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 52" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVKisLfF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 52" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30523256" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 52" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6283880" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 52" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?07881563" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 52" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-molecule%20imaging%20correlates%20decreasing%20nuclear%20volume%20with%20increasing%20TF-chromatin%20associations%20during%20zebrafish%20development&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-07731-8&amp;volume=9&amp;publication_year=2018&amp;author=Reisser%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Franke, M. et al. CTCF knockout in zebrafish induces alterations in regulatory landscapes and developmental gene expression. <i>Nat. Commun.</i> <b>12</b>, 5415 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 53" data-doi="10.1038/s41467-021-25604-5" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-021-25604-5" data-track-label="10.1038/s41467-021-25604-5" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-021-25604-5" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 53" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021NatCo..12.5415F" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 53" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXitVCnu7%2FN" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 53" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34518536" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 53" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8438036" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 53" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?09.0106.03" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 53" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CTCF%20knockout%20in%20zebrafish%20induces%20alterations%20in%20regulatory%20landscapes%20and%20developmental%20gene%20expression&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-25604-5&amp;volume=12&amp;publication_year=2021&amp;author=Franke%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Horsfield, J. A. et al. Cohesin-dependent regulation of Runx genes. <i>Development</i> <b>134</b>, 2639–2649 (2007).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 54" data-doi="10.1242/dev.002485" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1242/dev.002485" data-track-label="10.1242/dev.002485" data-track-value="article reference" href="https://doi.org/10.1242%2Fdev.002485" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 54" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXpsFeisrg%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 54" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17567667" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 54" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0326.92006" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 54" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohesin-dependent%20regulation%20of%20Runx%20genes&amp;journal=Development&amp;doi=10.1242%2Fdev.002485&amp;volume=134&amp;pages=2639-2649&amp;publication_year=2007&amp;author=Horsfield%2CJA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Haering, C. H., Löwe, J., Hochwagen, A. &amp; Nasmyth, K. Molecular architecture of SMC proteins and the Yeast Cohesin Complex C-terminal domains forming a head would be part of. <i>Mol. Cell</i> <b>9</b>, 773–788 (2002).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 55" data-doi="10.1016/S1097-2765(02)00515-4" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/S1097-2765(02)00515-4" data-track-label="10.1016/S1097-2765(02)00515-4" data-track-value="article reference" href="https://doi.org/10.1016%2FS1097-2765%2802%2900515-4" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 55" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38XjsFWgsLk%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 55" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11983169" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 55" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20architecture%20of%20SMC%20proteins%20and%20the%20Yeast%20Cohesin%20Complex%20C-terminal%20domains%20forming%20a%20head%20would%20be%20part%20of&amp;journal=Mol.%20Cell&amp;doi=10.1016%2FS1097-2765%2802%2900515-4&amp;volume=9&amp;pages=773-788&amp;publication_year=2002&amp;author=Haering%2CCH&amp;author=L%C3%B6we%2CJ&amp;author=Hochwagen%2CA&amp;author=Nasmyth%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Grimm, J. B. et al. A general method to improve fluorophores for live-cell and single-molecule microscopy. <i>Nat. Methods</i> <b>12</b>, 244–250 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 56" data-doi="10.1038/nmeth.3256" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nmeth.3256" data-track-label="10.1038/nmeth.3256" data-track-value="article reference" href="https://doi.org/10.1038%2Fnmeth.3256" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 56" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFKjsb8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 56" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25599551" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 56" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4344395" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 56" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1334.12002" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 56" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20general%20method%20to%20improve%20fluorophores%20for%20live-cell%20and%20single-molecule%20microscopy&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.3256&amp;volume=12&amp;pages=244-250&amp;publication_year=2015&amp;author=Grimm%2CJB" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Kuhn, T. et al. Single-molecule tracking of Nodal and Lefty in live zebrafish embryos supports hindered diffusion model. <i>Nat. Commun.</i> <b>13</b>, 6101 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 57" data-doi="10.1038/s41467-022-33704-z" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-022-33704-z" data-track-label="10.1038/s41467-022-33704-z" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-022-33704-z" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 57" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022NatCo..13.6101K" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 57" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1Oru7jK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 57" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36243734" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 57" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9569377" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 57" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1508.92264" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 57" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-molecule%20tracking%20of%20Nodal%20and%20Lefty%20in%20live%20zebrafish%20embryos%20supports%20hindered%20diffusion%20model&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-022-33704-z&amp;volume=13&amp;publication_year=2022&amp;author=Kuhn%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Kuhn, T., Hettich, J., Davtyan, R. &amp; Gebhardt, J. C. M. Single molecule tracking and analysis framework including theory-predicted parameter settings. <i>Sci. Rep.</i> <b>11</b>, 1–12 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 58" data-doi="10.1038/s41598-021-88802-7" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41598-021-88802-7" data-track-label="10.1038/s41598-021-88802-7" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41598-021-88802-7" rel="nofollow noopener">Article</a> 
    <a aria-label="MATH reference 58" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1406.92183" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 58" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Single%20molecule%20tracking%20and%20analysis%20framework%20including%20theory-predicted%20parameter%20settings&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-021-88802-7&amp;volume=11&amp;pages=1-12&amp;publication_year=2021&amp;author=Kuhn%2CT&amp;author=Hettich%2CJ&amp;author=Davtyan%2CR&amp;author=Gebhardt%2CJCM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Mazza, D., Abernathy, A., Golob, N., Morisaki, T. &amp; McNally, J. G. A benchmark for chromatin binding measurements in live cells. <i>Nucleic Acids Res.</i> <b>40</b>, 1–13 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 59" data-doi="10.1093/nar/gks701" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gks701" data-track-label="10.1093/nar/gks701" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgks701" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 59" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20benchmark%20for%20chromatin%20binding%20measurements%20in%20live%20cells&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgks701&amp;volume=40&amp;pages=1-13&amp;publication_year=2012&amp;author=Mazza%2CD&amp;author=Abernathy%2CA&amp;author=Golob%2CN&amp;author=Morisaki%2CT&amp;author=McNally%2CJG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Hansen, A. S. et al. Robust model-based analysis of single-particle tracking experiments with spot-on. <i>Elife</i> <b>7</b>, 1–33 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 60" data-doi="10.7554/eLife.33125" data-track="click_references" data-track-action="article reference" data-track-item_id="10.7554/eLife.33125" data-track-label="10.7554/eLife.33125" data-track-value="article reference" href="https://doi.org/10.7554%2FeLife.33125" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 60" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFKru7rI" rel="nofollow noopener">CAS</a> 
    <a aria-label="MATH reference 60" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?02.0365.03" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 60" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Robust%20model-based%20analysis%20of%20single-particle%20tracking%20experiments%20with%20spot-on&amp;journal=Elife&amp;doi=10.7554%2FeLife.33125&amp;volume=7&amp;pages=1-33&amp;publication_year=2018&amp;author=Hansen%2CAS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Gebhardt, J. C. M. et al. Single-molecule imaging of transcription factor binding to DNA in live mammalian cells. <i>Nat. Methods</i> <b>10</b>, 421–426 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 61" data-doi="10.1038/nmeth.2411" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nmeth.2411" data-track-label="10.1038/nmeth.2411" data-track-value="article reference" href="https://doi.org/10.1038%2Fnmeth.2411" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 61" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXksVKqt7c%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 61" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23524394" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 61" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3664538" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 61" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1253.05145" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 61" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-molecule%20imaging%20of%20transcription%20factor%20binding%20to%20DNA%20in%20live%20mammalian%20cells&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.2411&amp;volume=10&amp;pages=421-426&amp;publication_year=2013&amp;author=Gebhardt%2CJCM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Speil, J. et al. Activated STAT1 transcription factors conduct distinct saltatory movements in the cell nucleus. <i>Biophys. J.</i> <b>101</b>, 2592–2600 (2011).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 62" data-doi="10.1016/j.bpj.2011.10.006" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.bpj.2011.10.006" data-track-label="10.1016/j.bpj.2011.10.006" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.bpj.2011.10.006" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 62" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2011BpJ...101.2592S" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 62" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1SqsrnJ" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 62" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22261046" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 62" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3297778" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 62" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1422.37003" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 62" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Activated%20STAT1%20transcription%20factors%20conduct%20distinct%20saltatory%20movements%20in%20the%20cell%20nucleus&amp;journal=Biophys.%20J.&amp;doi=10.1016%2Fj.bpj.2011.10.006&amp;volume=101&amp;pages=2592-2600&amp;publication_year=2011&amp;author=Speil%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Izeddin, I. et al. Single-molecule tracking in live cells reveals distinct target-search strategies of transcription factors in the nucleus. <i>Elife</i> <b>2014</b>, e02230 (2014).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Shaban, H. A., Barth, R. &amp; Recoules, L. &amp; Bystricky, K. Hi-D: Nanoscale mapping of nuclear dynamics in single living cells. <i>Genome Biol.</i> <b>21</b>, 1–21 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 64" data-doi="10.1186/s13059-020-02002-6" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1186/s13059-020-02002-6" data-track-label="10.1186/s13059-020-02002-6" data-track-value="article reference" href="https://link.springer.com/doi/10.1186/s13059-020-02002-6" rel="noopener">Article</a> 
    <a aria-label="MATH reference 64" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0597.60074" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 64" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=%26%20Bystricky%2C%20K.%20Hi-D%3A%20Nanoscale%20mapping%20of%20nuclear%20dynamics%20in%20single%20living%20cells&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-020-02002-6&amp;volume=21&amp;pages=1-21&amp;publication_year=2020&amp;author=Shaban%2CHA&amp;author=Barth%2CR&amp;author=Recoules%2CL" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Veith, R. et al. Balbiani ring mRNPs diffuse through and bind to clusters of large intranuclear molecular structures. <i>Biophys. J.</i> <b>99</b>, 2676–2685 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 65" data-doi="10.1016/j.bpj.2010.08.004" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.bpj.2010.08.004" data-track-label="10.1016/j.bpj.2010.08.004" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.bpj.2010.08.004" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 65" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2010BpJ....99.2676V" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 65" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlShsbfK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 65" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20959109" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 65" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2955395" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 65" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1368.81090" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 65" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Balbiani%20ring%20mRNPs%20diffuse%20through%20and%20bind%20to%20clusters%20of%20large%20intranuclear%20molecular%20structures&amp;journal=Biophys.%20J.&amp;doi=10.1016%2Fj.bpj.2010.08.004&amp;volume=99&amp;pages=2676-2685&amp;publication_year=2010&amp;author=Veith%2CR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Höfling, F. &amp; Franosch, T. Anomalous transport in the crowded world of biological cells. <i>Rep. Prog. Phys.</i> <b>76</b>, 046602 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 66" data-doi="10.1088/0034-4885/76/4/046602" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1088/0034-4885/76/4/046602" data-track-label="10.1088/0034-4885/76/4/046602" data-track-value="article reference" href="https://doi.org/10.1088%2F0034-4885%2F76%2F4%2F046602" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 66" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013RPPh...76d6602H" rel="nofollow noopener">ADS</a> 
    <a aria-label="MathSciNet reference 66" data-track="click_references" data-track-action="mathscinet reference" data-track-item_id="link" data-track-label="link" data-track-value="mathscinet reference" href="http://www.ams.org/mathscinet-getitem?mr=3042495" rel="nofollow noopener">MathSciNet</a> 
    <a aria-label="PubMed reference 66" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23481518" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 66" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1312.60035" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 66" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Anomalous%20transport%20in%20the%20crowded%20world%20of%20biological%20cells&amp;journal=Rep.%20Prog.%20Phys.&amp;doi=10.1088%2F0034-4885%2F76%2F4%2F046602&amp;volume=76&amp;publication_year=2013&amp;author=H%C3%B6fling%2CF&amp;author=Franosch%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Haering, C. H. et al. Structure and stability of cohesin’s Smc1-kleisin interaction. <i>Mol. Cell</i> <b>15</b>, 951–964 (2004).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 67" data-doi="10.1016/j.molcel.2004.08.030" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2004.08.030" data-track-label="10.1016/j.molcel.2004.08.030" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2004.08.030" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 67" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXosl2jsbc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 67" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15383284" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 67" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1092.74023" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 67" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structure%20and%20stability%20of%20cohesin%E2%80%99s%20Smc1-kleisin%20interaction&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2004.08.030&amp;volume=15&amp;pages=951-964&amp;publication_year=2004&amp;author=Haering%2CCH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Sun, Y. et al. RAD21 is the core subunit of the cohesin complex involved in directing genome organization. <i>Genome Biol.</i> <b>24</b>, 155 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 68" data-doi="10.1186/s13059-023-02982-1" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1186/s13059-023-02982-1" data-track-label="10.1186/s13059-023-02982-1" data-track-value="article reference" href="https://link.springer.com/doi/10.1186/s13059-023-02982-1" rel="noopener">Article</a> 
    <a aria-label="CAS reference 68" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXhsVWks7rP" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 68" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37381036" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 68" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10303866" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 68" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1513.34308" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 68" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=RAD21%20is%20the%20core%20subunit%20of%20the%20cohesin%20complex%20involved%20in%20directing%20genome%20organization&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-023-02982-1&amp;volume=24&amp;publication_year=2023&amp;author=Sun%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Nakahashi, H. et al. A genome-wide Map of CTCF multivalency redefines the CTCF Code. <i>Cell Rep.</i> <b>3</b>, 1678–1689 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 69" data-doi="10.1016/j.celrep.2013.04.024" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.celrep.2013.04.024" data-track-label="10.1016/j.celrep.2013.04.024" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.celrep.2013.04.024" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 69" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXotlGju7c%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 69" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23707059" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 69" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3770538" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 69" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1408.76002" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 69" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20genome-wide%20Map%20of%20CTCF%20multivalency%20redefines%20the%20CTCF%20Code&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2013.04.024&amp;volume=3&amp;pages=1678-1689&amp;publication_year=2013&amp;author=Nakahashi%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Hashimoto, H. et al. Structural basis for the versatile and methylation-dependent binding of CTCF to DNA. <i>Mol. Cell</i> <b>66</b>, 711–720.e3 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 70" data-doi="10.1016/j.molcel.2017.05.004" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2017.05.004" data-track-label="10.1016/j.molcel.2017.05.004" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2017.05.004" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 70" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXotVCnt7g%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 70" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28529057" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 70" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5542067" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 70" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1373.22023" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 70" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20basis%20for%20the%20versatile%20and%20methylation-dependent%20binding%20of%20CTCF%20to%20DNA&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2017.05.004&amp;volume=66&amp;pages=711-720.e3&amp;publication_year=2017&amp;author=Hashimoto%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Yin, M. et al. Molecular mechanism of directional CTCF recognition of a diverse range of genomic sites. <i>Cell Res.</i> <b>27</b>, 1365–1377 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 71" data-doi="10.1038/cr.2017.131" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/cr.2017.131" data-track-label="10.1038/cr.2017.131" data-track-value="article reference" href="https://doi.org/10.1038%2Fcr.2017.131" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 71" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslehs73M" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 71" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29076501" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 71" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5674162" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 71" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1195.92045" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 71" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20mechanism%20of%20directional%20CTCF%20recognition%20of%20a%20diverse%20range%20of%20genomic%20sites&amp;journal=Cell%20Res.&amp;doi=10.1038%2Fcr.2017.131&amp;volume=27&amp;pages=1365-1377&amp;publication_year=2017&amp;author=Yin%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Gu, B. et al. Opposing effects of cohesin and transcription on CTCF organization revealed by super-resolution imaging. <i>Mol. Cell</i> <b>80</b>, 699–711.e7 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 72" data-doi="10.1016/j.molcel.2020.10.001" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2020.10.001" data-track-label="10.1016/j.molcel.2020.10.001" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2020.10.001" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 72" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFGrsbbM" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 72" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33091336" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 72" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7725164" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 72" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1257.74131" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 72" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Opposing%20effects%20of%20cohesin%20and%20transcription%20on%20CTCF%20organization%20revealed%20by%20super-resolution%20imaging&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2020.10.001&amp;volume=80&amp;pages=699-711.e7&amp;publication_year=2020&amp;author=Gu%2CB" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Cattoglio, C. et al. Determining cellular CTCF and cohesin abundances to constrain 3D genome models. <i>Elife</i> <b>8</b> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Zheng, G., Kanchwala, M., Xing, C. &amp; Yu, H. MCM2–7-dependent cohesin loading during s phase promotes sister-chromatid cohesion. <i>Elife</i> <b>7</b>, 1–25 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 74" data-doi="10.7554/eLife.33920" data-track="click_references" data-track-action="article reference" data-track-item_id="10.7554/eLife.33920" data-track-label="10.7554/eLife.33920" data-track-value="article reference" href="https://doi.org/10.7554%2FeLife.33920" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 74" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=MCM2%E2%80%937-dependent%20cohesin%20loading%20during%20s%20phase%20promotes%20sister-chromatid%20cohesion&amp;journal=Elife&amp;doi=10.7554%2FeLife.33920&amp;volume=7&amp;pages=1-25&amp;publication_year=2018&amp;author=Zheng%2CG&amp;author=Kanchwala%2CM&amp;author=Xing%2CC&amp;author=Yu%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Srinivasan, M., Fumasoni, M., Petela, N. J., Murray, A. &amp; Nasmyth, K. A. Cohesion is established during dna replication utilising chromosome associated cohesin rings as well as those loaded de novo onto nascent DNAs. <i>Elife</i> <b>9</b>, 1–27 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 75" data-doi="10.7554/eLife.56611" data-track="click_references" data-track-action="article reference" data-track-item_id="10.7554/eLife.56611" data-track-label="10.7554/eLife.56611" data-track-value="article reference" href="https://doi.org/10.7554%2FeLife.56611" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 75" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohesion%20is%20established%20during%20dna%20replication%20utilising%20chromosome%20associated%20cohesin%20rings%20as%20well%20as%20those%20loaded%20de%20novo%20onto%20nascent%20DNAs&amp;journal=Elife&amp;doi=10.7554%2FeLife.56611&amp;volume=9&amp;pages=1-27&amp;publication_year=2020&amp;author=Srinivasan%2CM&amp;author=Fumasoni%2CM&amp;author=Petela%2CNJ&amp;author=Murray%2CA&amp;author=Nasmyth%2CKA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Alonso-Gil, D. &amp; Losada, A. NIPBL and cohesin: new take on a classic tale. <i>Trends Cell Biol.</i> <b>33</b>, 860–871 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 76" data-doi="10.1016/j.tcb.2023.03.006" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.tcb.2023.03.006" data-track-label="10.1016/j.tcb.2023.03.006" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.tcb.2023.03.006" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 76" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXmvVyqsbo%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 76" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37062615" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 76" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?07983764" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 76" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=NIPBL%20and%20cohesin%3A%20new%20take%20on%20a%20classic%20tale&amp;journal=Trends%20Cell%20Biol.&amp;doi=10.1016%2Fj.tcb.2023.03.006&amp;volume=33&amp;pages=860-871&amp;publication_year=2023&amp;author=Alonso-Gil%2CD&amp;author=Losada%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Ciosk, R. et al. Cohesin’s binding to chromosomes depends on a separate complex consisting of Scc2 and Scc4 proteins. <i>Mol. Cell</i> <b>5</b>, 243–254 (2000).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 77" data-doi="10.1016/S1097-2765(00)80420-7" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/S1097-2765(00)80420-7" data-track-label="10.1016/S1097-2765(00)80420-7" data-track-value="article reference" href="https://doi.org/10.1016%2FS1097-2765%2800%2980420-7" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 77" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3cXhslyht7Y%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 77" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10882066" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 77" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1049.78505" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 77" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohesin%E2%80%99s%20binding%20to%20chromosomes%20depends%20on%20a%20separate%20complex%20consisting%20of%20Scc2%20and%20Scc4%20proteins&amp;journal=Mol.%20Cell&amp;doi=10.1016%2FS1097-2765%2800%2980420-7&amp;volume=5&amp;pages=243-254&amp;publication_year=2000&amp;author=Ciosk%2CR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Gandhi, R., Gillespie, P. J. &amp; Hirano, T. Human Wapl is a Cohesin-binding protein that promotes sister-chromatid resolution in mitotic prophase. <i>Curr. Biol.</i> <b>16</b>, 2406–2417 (2006).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 78" data-doi="10.1016/j.cub.2006.10.061" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cub.2006.10.061" data-track-label="10.1016/j.cub.2006.10.061" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cub.2006.10.061" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 78" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XhtlalsL3E" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 78" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17112726" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 78" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1850625" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 78" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20Wapl%20is%20a%20Cohesin-binding%20protein%20that%20promotes%20sister-chromatid%20resolution%20in%20mitotic%20prophase&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2006.10.061&amp;volume=16&amp;pages=2406-2417&amp;publication_year=2006&amp;author=Gandhi%2CR&amp;author=Gillespie%2CPJ&amp;author=Hirano%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Tedeschi, A. et al. Wapl is an essential regulator of chromatin structure and chromosome segregation. <i>Nature</i> <b>501</b>, 564–568 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 79" data-doi="10.1038/nature12471" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature12471" data-track-label="10.1038/nature12471" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature12471" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 79" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013Natur.501..564T" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 79" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtlCnsrnO" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 79" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23975099" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 79" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6080692" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 79" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0818.90034" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 79" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Wapl%20is%20an%20essential%20regulator%20of%20chromatin%20structure%20and%20chromosome%20segregation&amp;journal=Nature&amp;doi=10.1038%2Fnature12471&amp;volume=501&amp;pages=564-568&amp;publication_year=2013&amp;author=Tedeschi%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80."><p class="c-article-references__text" id="ref-CR80">Wutz, G. et al. ESCO1 and CTCF enable formation of long chromatin loops by protecting cohesinstag1 from WAPL. <i>Elife</i> <b>9</b>, 1–33 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 80" data-doi="10.7554/eLife.52091" data-track="click_references" data-track-action="article reference" data-track-item_id="10.7554/eLife.52091" data-track-label="10.7554/eLife.52091" data-track-value="article reference" href="https://doi.org/10.7554%2FeLife.52091" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 80" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=ESCO1%20and%20CTCF%20enable%20formation%20of%20long%20chromatin%20loops%20by%20protecting%20cohesinstag1%20from%20WAPL&amp;journal=Elife&amp;doi=10.7554%2FeLife.52091&amp;volume=9&amp;pages=1-33&amp;publication_year=2020&amp;author=Wutz%2CG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81."><p class="c-article-references__text" id="ref-CR81">Muto, A., Calof, A. L., Lander, A. D. &amp; Schilling, T. F. Multifactorial origins of heart and gut defects in Nipbl-deficient zebrafish, a model of cornelia de Lange Syndrome. <i>PLoS Biol</i>. <b>9</b>, e1001181 (2011).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="82."><p class="c-article-references__text" id="ref-CR82">Muto, A. et al. Nipbl and mediator cooperatively regulate gene expression to control limb development. <i>PLoS Genet</i>. <b>10</b>, e1004671 (2014).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="83."><p class="c-article-references__text" id="ref-CR83">Delgado-Olguín, P. et al. CTCF promotes muscle differentiation by modulating the activity of myogenic regulatory factors. <i>J. Biol. Chem.</i> <b>286</b>, 12483–12494 (2011).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 83" data-doi="10.1074/jbc.M110.164574" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1074/jbc.M110.164574" data-track-label="10.1074/jbc.M110.164574" data-track-value="article reference" href="https://doi.org/10.1074%2Fjbc.M110.164574" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 83" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21288905" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 83" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3069451" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 83" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CTCF%20promotes%20muscle%20differentiation%20by%20modulating%20the%20activity%20of%20myogenic%20regulatory%20factors&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M110.164574&amp;volume=286&amp;pages=12483-12494&amp;publication_year=2011&amp;author=Delgado-Olgu%C3%ADn%2CP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="84."><p class="c-article-references__text" id="ref-CR84">Meier, M. et al. Cohesin facilitates zygotic genome activation in zebrafish. <i>Dev</i> <b>145</b>, 1–13 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="MATH reference 84" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0799.05004" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 84" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohesin%20facilitates%20zygotic%20genome%20activation%20in%20zebrafish&amp;journal=Dev&amp;volume=145&amp;pages=1-13&amp;publication_year=2018&amp;author=Meier%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="85."><p class="c-article-references__text" id="ref-CR85">Neguembor, M. V. et al. Transcription-mediated supercoiling regulates genome folding and loop formation. <i>Mol. Cell</i> <b>81</b>, 3065–3081.e12 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 85" data-doi="10.1016/j.molcel.2021.06.009" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2021.06.009" data-track-label="10.1016/j.molcel.2021.06.009" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2021.06.009" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 85" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhs1WkurvM" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 85" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34297911" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 85" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9482096" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 85" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1481.18013" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 85" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcription-mediated%20supercoiling%20regulates%20genome%20folding%20and%20loop%20formation&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2021.06.009&amp;volume=81&amp;pages=3065-3081.e12&amp;publication_year=2021&amp;author=Neguembor%2CMV" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="86."><p class="c-article-references__text" id="ref-CR86">Sept, C. E. et al. High-resolution CTCF footprinting reveals impact of chromatin state on cohesin extrusion dynamics. <i>bioRxiv Prepr. Serv. Biol</i>. <a data-track="click_references" data-track-action="external reference" data-track-label="10.1101/2023.10.20.563340" data-track-value="external reference" href="https://doi.org/10.1101/2023.10.20.563340">https://doi.org/10.1101/2023.10.20.563340</a> (2023).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="87."><p class="c-article-references__text" id="ref-CR87">Zhang, S. et al. RNA polymerase II is required for spatial chromatin reorganization following exit from mitosis. <i>Sci. Adv.</i> <b>7</b>, 1–14 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 87" data-doi="10.1126/sciadv.abg8205" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/sciadv.abg8205" data-track-label="10.1126/sciadv.abg8205" data-track-value="article reference" href="https://doi.org/10.1126%2Fsciadv.abg8205" rel="nofollow noopener">Article</a> 
    <a aria-label="MathSciNet reference 87" data-track="click_references" data-track-action="mathscinet reference" data-track-item_id="link" data-track-label="link" data-track-value="mathscinet reference" href="http://www.ams.org/mathscinet-getitem?mr=4216434" rel="nofollow noopener">MathSciNet</a> 
    <a aria-label="Google Scholar reference 87" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=RNA%20polymerase%20II%20is%20required%20for%20spatial%20chromatin%20reorganization%20following%20exit%20from%20mitosis&amp;journal=Sci.%20Adv.&amp;doi=10.1126%2Fsciadv.abg8205&amp;volume=7&amp;pages=1-14&amp;publication_year=2021&amp;author=Zhang%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="88."><p class="c-article-references__text" id="ref-CR88">Zhang, S., Übelmesser, N., Barbieri, M. &amp; Papantonis, A. Enhancer–promoter contact formation requires RNAPII and antagonizes loop extrusion. <i>Nat. Genet.</i> <b>55</b>, 832–840 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 88" data-doi="10.1038/s41588-023-01364-4" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41588-023-01364-4" data-track-label="10.1038/s41588-023-01364-4" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41588-023-01364-4" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 88" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXmvV2rtr0%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 88" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37012454" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 88" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Enhancer%E2%80%93promoter%20contact%20formation%20requires%20RNAPII%20and%20antagonizes%20loop%20extrusion&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-023-01364-4&amp;volume=55&amp;pages=832-840&amp;publication_year=2023&amp;author=Zhang%2CS&amp;author=%C3%9Cbelmesser%2CN&amp;author=Barbieri%2CM&amp;author=Papantonis%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="89."><p class="c-article-references__text" id="ref-CR89">Brueckner, F. &amp; Cramer, P. Structural basis of transcription inhibition by α-amanitin and implications for RNA polymerase II translocation. <i>Nat. Struct. Mol. Biol.</i> <b>15</b>, 811–818 (2008).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 89" data-doi="10.1038/nsmb.1458" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nsmb.1458" data-track-label="10.1038/nsmb.1458" data-track-value="article reference" href="https://doi.org/10.1038%2Fnsmb.1458" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 89" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXptlOisLw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 89" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18552824" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 89" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1533.62076" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 89" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20basis%20of%20transcription%20inhibition%20by%20%CE%B1-amanitin%20and%20implications%20for%20RNA%20polymerase%20II%20translocation&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;doi=10.1038%2Fnsmb.1458&amp;volume=15&amp;pages=811-818&amp;publication_year=2008&amp;author=Brueckner%2CF&amp;author=Cramer%2CP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="90."><p class="c-article-references__text" id="ref-CR90">Kaplan, C. D., Larsson, K. M. &amp; Kornberg, R. D. The RNA Polymerase II Trigger Loop functions in substrate selection and is directly targeted by α-Amanitin. <i>Mol. Cell</i> <b>30</b>, 547–556 (2008).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 90" data-doi="10.1016/j.molcel.2008.04.023" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2008.04.023" data-track-label="10.1016/j.molcel.2008.04.023" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2008.04.023" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 90" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXntlWms7o%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 90" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18538653" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 90" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2475549" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 90" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20RNA%20Polymerase%20II%20Trigger%20Loop%20functions%20in%20substrate%20selection%20and%20is%20directly%20targeted%20by%20%CE%B1-Amanitin&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2008.04.023&amp;volume=30&amp;pages=547-556&amp;publication_year=2008&amp;author=Kaplan%2CCD&amp;author=Larsson%2CKM&amp;author=Kornberg%2CRD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="91."><p class="c-article-references__text" id="ref-CR91">Bensaude, O. Inhibiting eukaryotic transcription: Which compound to choose? How to evaluate its activity? <i>Transcription</i> <b>2</b>, 103–108 (2011).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 91" data-doi="10.4161/trns.2.3.16172" data-track="click_references" data-track-action="article reference" data-track-item_id="10.4161/trns.2.3.16172" data-track-label="10.4161/trns.2.3.16172" data-track-value="article reference" href="https://doi.org/10.4161%2Ftrns.2.3.16172" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 91" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21922053" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 91" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3173647" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 91" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Inhibiting%20eukaryotic%20transcription%3A%20Which%20compound%20to%20choose%3F%20How%20to%20evaluate%20its%20activity%3F&amp;journal=Transcription&amp;doi=10.4161%2Ftrns.2.3.16172&amp;volume=2&amp;pages=103-108&amp;publication_year=2011&amp;author=Bensaude%2CO" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="92."><p class="c-article-references__text" id="ref-CR92">Li, L. et al. Widespread rearrangement of 3D Chromatin organization underlies polycomb-mediated stress-induced silencing. <i>Mol. Cell</i> <b>58</b>, 216–231 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 92" data-doi="10.1016/j.molcel.2015.02.023" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2015.02.023" data-track-label="10.1016/j.molcel.2015.02.023" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2015.02.023" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 92" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXlsVOisr4%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 92" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25818644" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 92" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4402144" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 92" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1333.05183" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 92" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Widespread%20rearrangement%20of%203D%20Chromatin%20organization%20underlies%20polycomb-mediated%20stress-induced%20silencing&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2015.02.023&amp;volume=58&amp;pages=216-231&amp;publication_year=2015&amp;author=Li%2CL" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="93."><p class="c-article-references__text" id="ref-CR93">Shao, W. &amp; Zeitlinger, J. Paused RNA polymerase II inhibits new transcriptional initiation. <i>Nat. Genet.</i> <b>49</b>, 1045–1051 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 93" data-doi="10.1038/ng.3867" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/ng.3867" data-track-label="10.1038/ng.3867" data-track-value="article reference" href="https://doi.org/10.1038%2Fng.3867" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 93" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXnslKlurk%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 93" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28504701" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 93" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Paused%20RNA%20polymerase%20II%20inhibits%20new%20transcriptional%20initiation&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3867&amp;volume=49&amp;pages=1045-1051&amp;publication_year=2017&amp;author=Shao%2CW&amp;author=Zeitlinger%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="94."><p class="c-article-references__text" id="ref-CR94">Pálfy, M., Schulze, G., Valen, E. &amp; Vastenhouw, N. L. Chromatin accessibility established by Pou5f3, Sox19b and Nanog primes genes for activity during zebrafish genome activation. <i>PLoS Genet.</i> <b>16</b>, 1–25 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 94" data-doi="10.1371/journal.pgen.1008546" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1371/journal.pgen.1008546" data-track-label="10.1371/journal.pgen.1008546" data-track-value="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1008546" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 94" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromatin%20accessibility%20established%20by%20Pou5f3%2C%20Sox19b%20and%20Nanog%20primes%20genes%20for%20activity%20during%20zebrafish%20genome%20activation&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1008546&amp;volume=16&amp;pages=1-25&amp;publication_year=2020&amp;author=P%C3%A1lfy%2CM&amp;author=Schulze%2CG&amp;author=Valen%2CE&amp;author=Vastenhouw%2CNL" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="95."><p class="c-article-references__text" id="ref-CR95">Lee, M. T. et al. Nanog, Pou5f1 and SoxB1 activate zygotic gene expression during the maternal-to-zygotic transition. <i>Nature</i> <b>503</b>, 360–364 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 95" data-doi="10.1038/nature12632" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature12632" data-track-label="10.1038/nature12632" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature12632" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 95" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013Natur.503..360L" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 95" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsFSltr7O" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 95" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24056933" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 95" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3925760" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 95" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Nanog%2C%20Pou5f1%20and%20SoxB1%20activate%20zygotic%20gene%20expression%20during%20the%20maternal-to-zygotic%20transition&amp;journal=Nature&amp;doi=10.1038%2Fnature12632&amp;volume=503&amp;pages=360-364&amp;publication_year=2013&amp;author=Lee%2CMT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="96."><p class="c-article-references__text" id="ref-CR96">Kane, D. A. et al. The zebrafish epiboly mutants. <i>Development</i> <b>123</b>, 47–55 (1996).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 96" data-doi="10.1242/dev.123.1.47" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1242/dev.123.1.47" data-track-label="10.1242/dev.123.1.47" data-track-value="article reference" href="https://doi.org/10.1242%2Fdev.123.1.47" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 96" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK2sXhtVOmtL4%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 96" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9007228" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 96" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0431.55007" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 96" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20zebrafish%20epiboly%20mutants&amp;journal=Development&amp;doi=10.1242%2Fdev.123.1.47&amp;volume=123&amp;pages=47-55&amp;publication_year=1996&amp;author=Kane%2CDA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="97."><p class="c-article-references__text" id="ref-CR97">Huo, J. et al. Triptolide-induced hepatotoxicity via apoptosis and autophagy in zebrafish. <i>J. Appl. Toxicol.</i> <b>39</b>, 1532–1540 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 97" data-doi="10.1002/jat.3837" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/jat.3837" data-track-label="10.1002/jat.3837" data-track-value="article reference" href="https://doi.org/10.1002%2Fjat.3837" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 97" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVSit7zF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 97" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31321794" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 97" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1422.65393" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 97" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Triptolide-induced%20hepatotoxicity%20via%20apoptosis%20and%20autophagy%20in%20zebrafish&amp;journal=J.%20Appl.%20Toxicol.&amp;doi=10.1002%2Fjat.3837&amp;volume=39&amp;pages=1532-1540&amp;publication_year=2019&amp;author=Huo%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="98."><p class="c-article-references__text" id="ref-CR98">Vliegenthart, A. D. B. et al. Characterization of triptolide-induced hepatotoxicity by imaging and transcriptomics in a novel zebrafish model. <i>Toxicol. Sci.</i> <b>159</b>, 380–391 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 98" data-doi="10.1093/toxsci/kfx144" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/toxsci/kfx144" data-track-label="10.1093/toxsci/kfx144" data-track-value="article reference" href="https://doi.org/10.1093%2Ftoxsci%2Fkfx144" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 98" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXps1Khsb4%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 98" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28962522" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 98" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5837554" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 98" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0211.29201" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 98" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Characterization%20of%20triptolide-induced%20hepatotoxicity%20by%20imaging%20and%20transcriptomics%20in%20a%20novel%20zebrafish%20model&amp;journal=Toxicol.%20Sci.&amp;doi=10.1093%2Ftoxsci%2Fkfx144&amp;volume=159&amp;pages=380-391&amp;publication_year=2017&amp;author=Vliegenthart%2CADB" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="99."><p class="c-article-references__text" id="ref-CR99">Mishima, Y. &amp; Tomari, Y. Codon usage and 3’ UTR length determine maternal mRNA stability in Zebrafish. <i>Mol. Cell</i> <b>61</b>, 874–885 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 99" data-doi="10.1016/j.molcel.2016.02.027" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2016.02.027" data-track-label="10.1016/j.molcel.2016.02.027" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2016.02.027" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 99" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xks1OqsLg%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 99" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26990990" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 99" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0991.13500" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 99" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Codon%20usage%20and%203%E2%80%99%20UTR%20length%20determine%20maternal%20mRNA%20stability%20in%20Zebrafish&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2016.02.027&amp;volume=61&amp;pages=874-885&amp;publication_year=2016&amp;author=Mishima%2CY&amp;author=Tomari%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="100."><p class="c-article-references__text" id="ref-CR100">Song, Y. S. et al. Validation, optimization, and application of the Zebrafish developmental toxicity assay for pharmaceuticals under the ICH S5(R3) Guideline. <i>Front. Cell Dev. Biol</i>. <b>9</b>, 721130 (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="101."><p class="c-article-references__text" id="ref-CR101">Nozaki, T. et al. Dynamic organization of chromatin domains revealed by super-resolution live-cell imaging. <i>Mol. Cell</i> <b>67</b>, 282–293.e7 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 101" data-doi="10.1016/j.molcel.2017.06.018" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2017.06.018" data-track-label="10.1016/j.molcel.2017.06.018" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2017.06.018" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 101" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFOlsbfE" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 101" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28712725" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 101" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1497.57017" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 101" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Dynamic%20organization%20of%20chromatin%20domains%20revealed%20by%20super-resolution%20live-cell%20imaging&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2017.06.018&amp;volume=67&amp;pages=282-293.e7&amp;publication_year=2017&amp;author=Nozaki%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="102."><p class="c-article-references__text" id="ref-CR102">Ashwin, S. S., Nozaki, T., Maeshima, K. &amp; Sasai, M. Organization of fast and slow chromatin revealed by single-nucleosome dynamics. <i>Proc. Natl Acad. Sci. USA</i> <b>116</b>, 19939–19944 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 102" data-doi="10.1073/pnas.1907342116" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.1907342116" data-track-label="10.1073/pnas.1907342116" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.1907342116" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 102" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019PNAS..11619939A" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 102" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFSit7zE" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 102" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31527274" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 102" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6778247" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 102" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Organization%20of%20fast%20and%20slow%20chromatin%20revealed%20by%20single-nucleosome%20dynamics&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1907342116&amp;volume=116&amp;pages=19939-19944&amp;publication_year=2019&amp;author=Ashwin%2CSS&amp;author=Nozaki%2CT&amp;author=Maeshima%2CK&amp;author=Sasai%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="103."><p class="c-article-references__text" id="ref-CR103">Mach, P. et al. Cohesin and CTCF control the dynamics of chromosome folding. <i>Nat. Genet.</i> <b>54</b>, 1907–1918 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 103" data-doi="10.1038/s41588-022-01232-7" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41588-022-01232-7" data-track-label="10.1038/s41588-022-01232-7" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41588-022-01232-7" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 103" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XjtVCjtb3O" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 103" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36471076" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 103" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9729113" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 103" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1503.60047" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 103" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohesin%20and%20CTCF%20control%20the%20dynamics%20of%20chromosome%20folding&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-022-01232-7&amp;volume=54&amp;pages=1907-1918&amp;publication_year=2022&amp;author=Mach%2CP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="104."><p class="c-article-references__text" id="ref-CR104">Gabriele, M. et al. Dynamics of CTCF- and cohesin-mediated chromatin looping revealed by live-cell imaging. <i>Science</i> <b>376</b>, 476–501 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 104" data-doi="10.1126/science.abn6583" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.abn6583" data-track-label="10.1126/science.abn6583" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.abn6583" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 104" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022Sci...376..496G" rel="nofollow noopener">ADS</a> 
    <a aria-label="Google Scholar reference 104" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Dynamics%20of%20CTCF-%20and%20cohesin-mediated%20chromatin%20looping%20revealed%20by%20live-cell%20imaging&amp;journal=Science&amp;doi=10.1126%2Fscience.abn6583&amp;volume=376&amp;pages=476-501&amp;publication_year=2022&amp;author=Gabriele%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="105."><p class="c-article-references__text" id="ref-CR105">Schwarzer, W. et al. Two independent modes of chromatin organization revealed by cohesin removal. <i>Nature</i> <b>551</b>, 51–56 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 105" data-doi="10.1038/nature24281" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature24281" data-track-label="10.1038/nature24281" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature24281" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 105" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017Natur.551...51S" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 105" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29094699" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 105" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5687303" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 105" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0722.90051" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 105" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Two%20independent%20modes%20of%20chromatin%20organization%20revealed%20by%20cohesin%20removal&amp;journal=Nature&amp;doi=10.1038%2Fnature24281&amp;volume=551&amp;pages=51-56&amp;publication_year=2017&amp;author=Schwarzer%2CW" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="106."><p class="c-article-references__text" id="ref-CR106">Alonso-Gil, D., Cuadrado, A., Giménez-Llorente, D., Rodríguez-Corsino, M. &amp; Losada, A. Different NIPBL requirements of cohesin-STAG1 and cohesin-STAG2. <i>Nat. Commun.</i> <b>14</b>, 1–11 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 106" data-doi="10.1038/s41467-023-36900-7" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-023-36900-7" data-track-label="10.1038/s41467-023-36900-7" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-023-36900-7" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 106" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Different%20NIPBL%20requirements%20of%20cohesin-STAG1%20and%20cohesin-STAG2&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-023-36900-7&amp;volume=14&amp;pages=1-11&amp;publication_year=2023&amp;author=Alonso-Gil%2CD&amp;author=Cuadrado%2CA&amp;author=Gim%C3%A9nez-Llorente%2CD&amp;author=Rodr%C3%ADguez-Corsino%2CM&amp;author=Losada%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="107."><p class="c-article-references__text" id="ref-CR107">Newkirk, D. A. et al. The effect of Nipped-B-like (Nipbl) haploinsufficiency on genome-wide cohesin binding and target gene expression: Modeling Cornelia de Lange syndrome. <i>Clin. Epigenet.</i> <b>9</b>, 1–20 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 107" data-doi="10.1186/s13148-017-0391-x" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1186/s13148-017-0391-x" data-track-label="10.1186/s13148-017-0391-x" data-track-value="article reference" href="https://link.springer.com/doi/10.1186/s13148-017-0391-x" rel="noopener">Article</a> 
    <a aria-label="MATH reference 107" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1398.53016" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 107" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20effect%20of%20Nipped-B-like%20%28Nipbl%29%20haploinsufficiency%20on%20genome-wide%20cohesin%20binding%20and%20target%20gene%20expression%3A%20Modeling%20Cornelia%20de%20Lange%20syndrome&amp;journal=Clin.%20Epigenet.&amp;doi=10.1186%2Fs13148-017-0391-x&amp;volume=9&amp;pages=1-20&amp;publication_year=2017&amp;author=Newkirk%2CDA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="108."><p class="c-article-references__text" id="ref-CR108">Porter, H. et al. Cohesin-independent STAG proteins interact with RNA and R-loops and promote complex loading. <i>Elife</i> <b>12</b>, 1–33 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 108" data-doi="10.7554/eLife.79386" data-track="click_references" data-track-action="article reference" data-track-item_id="10.7554/eLife.79386" data-track-label="10.7554/eLife.79386" data-track-value="article reference" href="https://doi.org/10.7554%2FeLife.79386" rel="nofollow noopener">Article</a> 
    <a aria-label="MATH reference 108" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0506.93040" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 108" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohesin-independent%20STAG%20proteins%20interact%20with%20RNA%20and%20R-loops%20and%20promote%20complex%20loading&amp;journal=Elife&amp;doi=10.7554%2FeLife.79386&amp;volume=12&amp;pages=1-33&amp;publication_year=2023&amp;author=Porter%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="109."><p class="c-article-references__text" id="ref-CR109">Nolen, L. D., Boyle, S., Ansari, M., Pritchard, E. &amp; Bickmore, W. A. Regional chromatin decompaction in cornelia de lange syndrome associated with NIPBL disruption can be uncoupled from cohesin and CTCF. <i>Hum. Mol. Genet.</i> <b>22</b>, 4180–4193 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 109" data-doi="10.1093/hmg/ddt265" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/hmg/ddt265" data-track-label="10.1093/hmg/ddt265" data-track-value="article reference" href="https://doi.org/10.1093%2Fhmg%2Fddt265" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 109" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsFegu73P" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 109" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23760082" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 109" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3781641" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 109" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Regional%20chromatin%20decompaction%20in%20cornelia%20de%20lange%20syndrome%20associated%20with%20NIPBL%20disruption%20can%20be%20uncoupled%20from%20cohesin%20and%20CTCF&amp;journal=Hum.%20Mol.%20Genet.&amp;doi=10.1093%2Fhmg%2Fddt265&amp;volume=22&amp;pages=4180-4193&amp;publication_year=2013&amp;author=Nolen%2CLD&amp;author=Boyle%2CS&amp;author=Ansari%2CM&amp;author=Pritchard%2CE&amp;author=Bickmore%2CWA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="110."><p class="c-article-references__text" id="ref-CR110">Reisser, M. et al. Inferring quantity and qualities of superimposed reaction rates from single molecule survival time distributions. <i>Sci. Rep.</i> <b>10</b>, 1–13 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 110" data-doi="10.1038/s41598-020-58634-y" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41598-020-58634-y" data-track-label="10.1038/s41598-020-58634-y" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41598-020-58634-y" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 110" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatSR..10....1R" rel="nofollow noopener">ADS</a> 
    <a aria-label="MATH reference 110" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?07817057" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 110" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Inferring%20quantity%20and%20qualities%20of%20superimposed%20reaction%20rates%20from%20single%20molecule%20survival%20time%20distributions&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-020-58634-y&amp;volume=10&amp;pages=1-13&amp;publication_year=2020&amp;author=Reisser%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="111."><p class="c-article-references__text" id="ref-CR111">Popp, A. P., Hettich, J. &amp; Gebhardt, J. C. M. Altering transcription factor binding reveals comprehensive transcriptional kinetics of a basic gene. <i>Nucleic Acids Res.</i> <b>49</b>, 6249–6266 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 111" data-doi="10.1093/nar/gkab443" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkab443" data-track-label="10.1093/nar/gkab443" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkab443" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 111" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1Crt7bE" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 111" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34060631" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 111" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8216454" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 111" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1406.92183" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 111" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Altering%20transcription%20factor%20binding%20reveals%20comprehensive%20transcriptional%20kinetics%20of%20a%20basic%20gene&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkab443&amp;volume=49&amp;pages=6249-6266&amp;publication_year=2021&amp;author=Popp%2CAP&amp;author=Hettich%2CJ&amp;author=Gebhardt%2CJCM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="112."><p class="c-article-references__text" id="ref-CR112">Berg, O. G., Winter, R. B. &amp; von Hippel, P. H. Diffusion-DRIVEN MECHANISMS OF PROTEIN TRANSLOCATION ON NUCLEIC ACids. 1. Models and Theory. <i>Biochemistry</i> <b>20</b>, 6929–6948 (1981).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 112" data-doi="10.1021/bi00527a028" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/bi00527a028" data-track-label="10.1021/bi00527a028" data-track-value="article reference" href="https://doi.org/10.1021%2Fbi00527a028" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 112" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaL3MXlvF2nt7c%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 112" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7317363" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 112" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0929.35143" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 112" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Diffusion-DRIVEN%20MECHANISMS%20OF%20PROTEIN%20TRANSLOCATION%20ON%20NUCLEIC%20ACids.%201.%20Models%20and%20Theory&amp;journal=Biochemistry&amp;doi=10.1021%2Fbi00527a028&amp;volume=20&amp;pages=6929-6948&amp;publication_year=1981&amp;author=Berg%2COG&amp;author=Winter%2CRB&amp;author=Hippel%2CPH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="113."><p class="c-article-references__text" id="ref-CR113">Livhitz, M. A., Gursky, G. V., Zasedatelev, A. S. &amp; Volkenstein, M. V. Equilibrium and kinetic aspects of protein-DNA recognition. <i>Nucleic Acids Res.</i> <b>6</b>, 2217–2236 (1979).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 113" data-doi="10.1093/nar/6.6.2217" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/6.6.2217" data-track-label="10.1093/nar/6.6.2217" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2F6.6.2217" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 113" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Equilibrium%20and%20kinetic%20aspects%20of%20protein-DNA%20recognition&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2F6.6.2217&amp;volume=6&amp;pages=2217-2236&amp;publication_year=1979&amp;author=Livhitz%2CMA&amp;author=Gursky%2CGV&amp;author=Zasedatelev%2CAS&amp;author=Volkenstein%2CMV" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="114."><p class="c-article-references__text" id="ref-CR114">Loffreda, A. et al. Live-cell p53 single-molecule binding is modulated by C-terminal acetylation and correlates with transcriptional activity. <i>Nat. Commun</i>. <b>8</b>, 313 (2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="115."><p class="c-article-references__text" id="ref-CR115">Fudenberg, G., Abdennur, N., Imakaev, M., Goloborodko, A. &amp; Mirny, L. A. Emerging Evidence of Chromosome Folding by Loop Extrusion. <i>Cold Spring Harb. Symp. Quant. Biol.</i> <b>82</b>, 45–55 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 115" data-doi="10.1101/sqb.2017.82.034710" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1101/sqb.2017.82.034710" data-track-label="10.1101/sqb.2017.82.034710" data-track-value="article reference" href="https://doi.org/10.1101%2Fsqb.2017.82.034710" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 115" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29728444" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 115" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Emerging%20Evidence%20of%20Chromosome%20Folding%20by%20Loop%20Extrusion&amp;journal=Cold%20Spring%20Harb.%20Symp.%20Quant.%20Biol.&amp;doi=10.1101%2Fsqb.2017.82.034710&amp;volume=82&amp;pages=45-55&amp;publication_year=2017&amp;author=Fudenberg%2CG&amp;author=Abdennur%2CN&amp;author=Imakaev%2CM&amp;author=Goloborodko%2CA&amp;author=Mirny%2CLA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="116."><p class="c-article-references__text" id="ref-CR116">Galitsyna, A. et al. Extrusion fountains are hallmarks of chromosome organization emerging upon zygotic genome activation. <i>bioRxiv Prepr. Serv. Biol</i>. <a data-track="click_references" data-track-action="external reference" data-track-label="10.1101/2023.07.15.549120" data-track-value="external reference" href="https://doi.org/10.1101/2023.07.15.549120">https://doi.org/10.1101/2023.07.15.549120</a> (2023).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="117."><p class="c-article-references__text" id="ref-CR117">Polovnikov, K. E. et al. Crumpled polymer with loops recapitulates key features of chromosome organization. <i>Phys. Rev. X</i> <b>13</b>, 41029 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="MathSciNet reference 117" data-track="click_references" data-track-action="mathscinet reference" data-track-item_id="link" data-track-label="link" data-track-value="mathscinet reference" href="http://www.ams.org/mathscinet-getitem?mr=4605978" rel="nofollow noopener">MathSciNet</a> 
    <a aria-label="CAS reference 117" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB2cXhsF2ht70%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="MATH reference 117" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1083.92001" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 117" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Crumpled%20polymer%20with%20loops%20recapitulates%20key%20features%20of%20chromosome%20organization&amp;journal=Phys.%20Rev.%20X&amp;volume=13&amp;publication_year=2023&amp;author=Polovnikov%2CKE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="118."><p class="c-article-references__text" id="ref-CR118">Pérez-Rico, Y. A., Barillot, E. &amp; Shkumatava, A. Demarcation of topologically associating domains is uncoupled from enriched CTCF binding in developing Zebrafish. <i>iScience</i> <b>23</b>, 101046 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 118" data-doi="10.1016/j.isci.2020.101046" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.isci.2020.101046" data-track-label="10.1016/j.isci.2020.101046" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.isci.2020.101046" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 118" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020iSci...23j1046P" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 118" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32334414" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 118" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7182764" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 118" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Demarcation%20of%20topologically%20associating%20domains%20is%20uncoupled%20from%20enriched%20CTCF%20binding%20in%20developing%20Zebrafish&amp;journal=iScience&amp;doi=10.1016%2Fj.isci.2020.101046&amp;volume=23&amp;publication_year=2020&amp;author=P%C3%A9rez-Rico%2CYA&amp;author=Barillot%2CE&amp;author=Shkumatava%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="119."><p class="c-article-references__text" id="ref-CR119">Richeldi, M. et al. Mechanical disengagement of the cohesin ring. <i>Nat. Struct. Mol. Biol.</i> <b>31</b>, 23–31 (2024).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 119" data-doi="10.1038/s41594-023-01122-4" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41594-023-01122-4" data-track-label="10.1038/s41594-023-01122-4" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41594-023-01122-4" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 119" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXitF2js7nK" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 119" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37872232" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 119" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Mechanical%20disengagement%20of%20the%20cohesin%20ring&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;doi=10.1038%2Fs41594-023-01122-4&amp;volume=31&amp;pages=23-31&amp;publication_year=2024&amp;author=Richeldi%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="120."><p class="c-article-references__text" id="ref-CR120">Zamir, E., Kam, Z. &amp; Yarden, A. Transcription-dependent induction of G 1 phase during the Zebra Fish Midblastula transition. <i>Mol. Cell. Biol.</i> <b>17</b>, 529–536 (1997).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 120" data-doi="10.1128/MCB.17.2.529" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1128/MCB.17.2.529" data-track-label="10.1128/MCB.17.2.529" data-track-value="article reference" href="https://doi.org/10.1128%2FMCB.17.2.529" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 120" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK2sXmvVSqsA%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 120" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9001205" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 120" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC231777" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 120" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcription-dependent%20induction%20of%20G%201%20phase%20during%20the%20Zebra%20Fish%20Midblastula%20transition&amp;journal=Mol.%20Cell.%20Biol.&amp;doi=10.1128%2FMCB.17.2.529&amp;volume=17&amp;pages=529-536&amp;publication_year=1997&amp;author=Zamir%2CE&amp;author=Kam%2CZ&amp;author=Yarden%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="121."><p class="c-article-references__text" id="ref-CR121">Kimmel, C. B., Ballard, W. W., Kimmel, S. R., Ullmann, B. &amp; Schilling, T. F. Stages of embryonic development of the zebrafish. <i>Dev. Dyn.</i> <b>203</b>, 253–310 (1995).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 121" data-doi="10.1002/aja.1002030302" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/aja.1002030302" data-track-label="10.1002/aja.1002030302" data-track-value="article reference" href="https://doi.org/10.1002%2Faja.1002030302" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 121" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK287nt12qsw%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 121" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8589427" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 121" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?07771583" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 121" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Stages%20of%20embryonic%20development%20of%20the%20zebrafish&amp;journal=Dev.%20Dyn.&amp;doi=10.1002%2Faja.1002030302&amp;volume=203&amp;pages=253-310&amp;publication_year=1995&amp;author=Kimmel%2CCB&amp;author=Ballard%2CWW&amp;author=Kimmel%2CSR&amp;author=Ullmann%2CB&amp;author=Schilling%2CTF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="122."><p class="c-article-references__text" id="ref-CR122">Eshghi, I., Eaton, J. A. &amp; Zidovska, A. Interphase Chromatin undergoes a local sol-gel transition upon cell differentiation. <i>Phys. Rev. Lett.</i> <b>126</b>, 228101 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 122" data-doi="10.1103/PhysRevLett.126.228101" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1103/PhysRevLett.126.228101" data-track-label="10.1103/PhysRevLett.126.228101" data-track-value="article reference" href="https://doi.org/10.1103%2FPhysRevLett.126.228101" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 122" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021PhRvL.126v8101E" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 122" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsVWku73P" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 122" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34152157" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 122" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8405102" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 122" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Interphase%20Chromatin%20undergoes%20a%20local%20sol-gel%20transition%20upon%20cell%20differentiation&amp;journal=Phys.%20Rev.%20Lett.&amp;doi=10.1103%2FPhysRevLett.126.228101&amp;volume=126&amp;publication_year=2021&amp;author=Eshghi%2CI&amp;author=Eaton%2CJA&amp;author=Zidovska%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="123."><p class="c-article-references__text" id="ref-CR123">Subramanian, V. et al. H2A.Z Acidic patch couples chromatin dynamics to regulation of gene expression programs during ESC Differentiation. <i>PLoS Genet</i>. <b>9</b>, e1003725 (2013).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="124."><p class="c-article-references__text" id="ref-CR124">Girelli, G. et al. GPSeq reveals the radial organization of chromatin in the cell nucleus. <i>Nat. Biotechnol.</i> <b>38</b>, 1184–1193 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 124" data-doi="10.1038/s41587-020-0519-y" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41587-020-0519-y" data-track-label="10.1038/s41587-020-0519-y" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41587-020-0519-y" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 124" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVSis73N" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 124" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32451505" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 124" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7610410" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 124" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0988.81124" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 124" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=GPSeq%20reveals%20the%20radial%20organization%20of%20chromatin%20in%20the%20cell%20nucleus&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-020-0519-y&amp;volume=38&amp;pages=1184-1193&amp;publication_year=2020&amp;author=Girelli%2CG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="125."><p class="c-article-references__text" id="ref-CR125">Su, J. H., Zheng, P., Kinrot, S. S., Bintu, B. &amp; Zhuang, X. Genome-scale imaging of the 3D organization and transcriptional activity of chromatin. <i>Cell</i> <b>182</b>, 1641–1659.e26 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 125" data-doi="10.1016/j.cell.2020.07.032" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cell.2020.07.032" data-track-label="10.1016/j.cell.2020.07.032" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cell.2020.07.032" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 125" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Gqsr%2FF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 125" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32822575" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 125" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7851072" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 125" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-scale%20imaging%20of%20the%203D%20organization%20and%20transcriptional%20activity%20of%20chromatin&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2020.07.032&amp;volume=182&amp;pages=1641-1659.e26&amp;publication_year=2020&amp;author=Su%2CJH&amp;author=Zheng%2CP&amp;author=Kinrot%2CSS&amp;author=Bintu%2CB&amp;author=Zhuang%2CX" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="126."><p class="c-article-references__text" id="ref-CR126">van Schaik, T. et al. CTCF and cohesin promote focal detachment of DNA from the nuclear lamina. <i>Genome Biol.</i> <b>23</b>, 1–26 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="MATH reference 126" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1472.60146" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 126" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CTCF%20and%20cohesin%20promote%20focal%20detachment%20of%20DNA%20from%20the%20nuclear%20lamina&amp;journal=Genome%20Biol.&amp;volume=23&amp;pages=1-26&amp;publication_year=2022&amp;author=Schaik%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="127."><p class="c-article-references__text" id="ref-CR127">Goll, M. G., Akdogan-Ozdilek, B. &amp; Duval, K. L. Chromatin dynamics at the maternal to zygotic transition: Recent advances from the zebrafish model. <i>F1000Research</i> <b>9</b>, 1–12 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Google Scholar reference 127" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromatin%20dynamics%20at%20the%20maternal%20to%20zygotic%20transition%3A%20Recent%20advances%20from%20the%20zebrafish%20model&amp;journal=F1000Research&amp;volume=9&amp;pages=1-12&amp;publication_year=2020&amp;author=Goll%2CMG&amp;author=Akdogan-Ozdilek%2CB&amp;author=Duval%2CKL" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="128."><p class="c-article-references__text" id="ref-CR128">Laue, K., Rajshekar, S., Courtney, A. J., Lewis, Z. A. &amp; Goll, M. G. The maternal to zygotic transition regulates genome-wide heterochromatin establishment in the zebrafish embryo. <i>Nat. Commun</i>. <b>10</b>, 1551 (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="129."><p class="c-article-references__text" id="ref-CR129">Busslinger, G. A. et al. Cohesin is positioned in mammalian genomes by transcription, CTCF and Wapl. <i>Nature</i> <b>544</b>, 503–507 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 129" data-doi="10.1038/nature22063" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nature22063" data-track-label="10.1038/nature22063" data-track-value="article reference" href="https://doi.org/10.1038%2Fnature22063" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 129" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017Natur.544..503B" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 129" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXmtlalt7s%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 129" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28424523" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 129" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6080695" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 129" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1378.00050" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 129" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohesin%20is%20positioned%20in%20mammalian%20genomes%20by%20transcription%2C%20CTCF%20and%20Wapl&amp;journal=Nature&amp;doi=10.1038%2Fnature22063&amp;volume=544&amp;pages=503-507&amp;publication_year=2017&amp;author=Busslinger%2CGA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="130."><p class="c-article-references__text" id="ref-CR130">Davidson, I. F. et al. Rapid movement and transcriptional re‐localization of human cohesin on DNA. <i>EMBO J.</i> <b>35</b>, 2671–2685 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 130" data-doi="10.15252/embj.201695402" data-track="click_references" data-track-action="article reference" data-track-item_id="10.15252/embj.201695402" data-track-label="10.15252/embj.201695402" data-track-value="article reference" href="https://doi.org/10.15252%2Fembj.201695402" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 130" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhslKnurvN" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 130" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27799150" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 130" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5167347" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 130" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1353.35056" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 130" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Rapid%20movement%20and%20transcriptional%20re%E2%80%90localization%20of%20human%20cohesin%20on%20DNA&amp;journal=EMBO%20J.&amp;doi=10.15252%2Fembj.201695402&amp;volume=35&amp;pages=2671-2685&amp;publication_year=2016&amp;author=Davidson%2CIF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="131."><p class="c-article-references__text" id="ref-CR131">Banigan, E. J. et al. Transcription shapes 3D chromatin organization by interacting with loop extrusion. <i>Proc. Natl Acad. Sci.</i> <b>120</b>, 2017 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 131" data-doi="10.1073/pnas.2210480120" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.2210480120" data-track-label="10.1073/pnas.2210480120" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.2210480120" rel="nofollow noopener">Article</a> 
    <a aria-label="MATH reference 131" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1520.94032" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 131" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcription%20shapes%203D%20chromatin%20organization%20by%20interacting%20with%20loop%20extrusion&amp;journal=Proc.%20Natl%20Acad.%20Sci.&amp;doi=10.1073%2Fpnas.2210480120&amp;volume=120&amp;publication_year=2023&amp;author=Banigan%2CEJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="132."><p class="c-article-references__text" id="ref-CR132">Schulz, K. N. &amp; Harrison, M. M. Mechanisms regulating zygotic genome activation. <i>Nat. Rev. Genet.</i> <b>20</b>, 221–234 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 132" data-doi="10.1038/s41576-018-0087-x" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41576-018-0087-x" data-track-label="10.1038/s41576-018-0087-x" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41576-018-0087-x" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 132" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXisFOmsrbL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 132" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30573849" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 132" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6558659" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 132" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Mechanisms%20regulating%20zygotic%20genome%20activation&amp;journal=Nat.%20Rev.%20Genet.&amp;doi=10.1038%2Fs41576-018-0087-x&amp;volume=20&amp;pages=221-234&amp;publication_year=2019&amp;author=Schulz%2CKN&amp;author=Harrison%2CMM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="133."><p class="c-article-references__text" id="ref-CR133">Casa, V. et al. Redundant and specific roles of cohesin STAG subunits in chromatin looping and transcriptional control. <i>Genome Res.</i> <b>30</b>, 515–527 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 133" data-doi="10.1101/gr.253211.119" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1101/gr.253211.119" data-track-label="10.1101/gr.253211.119" data-track-value="article reference" href="https://doi.org/10.1101%2Fgr.253211.119" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 133" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1KntrbP" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 133" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32253279" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 133" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7197483" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 133" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1495.62119" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 133" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Redundant%20and%20specific%20roles%20of%20cohesin%20STAG%20subunits%20in%20chromatin%20looping%20and%20transcriptional%20control&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.253211.119&amp;volume=30&amp;pages=515-527&amp;publication_year=2020&amp;author=Casa%2CV" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="134."><p class="c-article-references__text" id="ref-CR134">Dauban, L. et al. Regulation of Cohesin-mediated chromosome folding by Eco1 and other partners. <i>Mol. Cell</i> <b>77</b>, 1279–1293.e4 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 134" data-doi="10.1016/j.molcel.2020.01.019" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2020.01.019" data-track-label="10.1016/j.molcel.2020.01.019" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2020.01.019" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 134" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXis12murk%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 134" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32032532" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 134" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0861.05056" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 134" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20Cohesin-mediated%20chromosome%20folding%20by%20Eco1%20and%20other%20partners&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2020.01.019&amp;volume=77&amp;pages=1279-1293.e4&amp;publication_year=2020&amp;author=Dauban%2CL" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="135."><p class="c-article-references__text" id="ref-CR135">van Ruiten, M. S. et al. The cohesin acetylation cycle controls chromatin loop length through a PDS5A brake mechanism. <i>Nat. Struct. Mol. Biol.</i> <b>29</b>, 586–591 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 135" data-doi="10.1038/s41594-022-00773-z" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41594-022-00773-z" data-track-label="10.1038/s41594-022-00773-z" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41594-022-00773-z" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 135" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35710836" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 135" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9205776" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 135" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20cohesin%20acetylation%20cycle%20controls%20chromatin%20loop%20length%20through%20a%20PDS5A%20brake%20mechanism&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;doi=10.1038%2Fs41594-022-00773-z&amp;volume=29&amp;pages=586-591&amp;publication_year=2022&amp;author=Ruiten%2CMS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="136."><p class="c-article-references__text" id="ref-CR136">Bastié, N. et al. Smc3 acetylation, Pds5 and Scc2 control the translocase activity that establishes cohesin-dependent chromatin loops. <i>Nat. Struct. Mol. Biol.</i> <b>29</b>, 575–585 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 136" data-doi="10.1038/s41594-022-00780-0" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41594-022-00780-0" data-track-label="10.1038/s41594-022-00780-0" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41594-022-00780-0" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 136" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35710835" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 136" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Smc3%20acetylation%2C%20Pds5%20and%20Scc2%20control%20the%20translocase%20activity%20that%20establishes%20cohesin-dependent%20chromatin%20loops&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;doi=10.1038%2Fs41594-022-00780-0&amp;volume=29&amp;pages=575-585&amp;publication_year=2022&amp;author=Basti%C3%A9%2CN" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="137."><p class="c-article-references__text" id="ref-CR137">Muñoz, S. et al. Functional crosstalk between the cohesin loader and chromatin remodelers. <i>Nat. Commun.</i> <b>13</b>, 1–12 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 137" data-doi="10.1038/s41467-022-35444-6" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-022-35444-6" data-track-label="10.1038/s41467-022-35444-6" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-022-35444-6" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 137" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022NatCo..13....1M" rel="nofollow noopener">ADS</a> 
    <a aria-label="MATH reference 137" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1509.35320" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 137" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20crosstalk%20between%20the%20cohesin%20loader%20and%20chromatin%20remodelers&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-022-35444-6&amp;volume=13&amp;pages=1-12&amp;publication_year=2022&amp;author=Mu%C3%B1oz%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="138."><p class="c-article-references__text" id="ref-CR138">Boginya, A., Detroja, R., Matityahu, A., Frenkel-Morgenstern, M. &amp; Onn, I. The chromatin remodeler Chd1 regulates cohesin in budding yeast and humans. <i>Sci. Rep.</i> <b>9</b>, 1–10 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 138" data-doi="10.1038/s41598-019-45263-3" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41598-019-45263-3" data-track-label="10.1038/s41598-019-45263-3" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41598-019-45263-3" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 138" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1CnsLrM" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 138" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20chromatin%20remodeler%20Chd1%20regulates%20cohesin%20in%20budding%20yeast%20and%20humans&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-019-45263-3&amp;volume=9&amp;pages=1-10&amp;publication_year=2019&amp;author=Boginya%2CA&amp;author=Detroja%2CR&amp;author=Matityahu%2CA&amp;author=Frenkel-Morgenstern%2CM&amp;author=Onn%2CI" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="139."><p class="c-article-references__text" id="ref-CR139">Muñoz, S., Minamino, M., Casas-Delucchi, C. S., Patel, H. &amp; Uhlmann, F. A role for Chromatin remodeling in cohesin loading onto chromosomes. <i>Mol. Cell</i> <b>74</b>, 664–673.e5 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 139" data-doi="10.1016/j.molcel.2019.02.027" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2019.02.027" data-track-label="10.1016/j.molcel.2019.02.027" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2019.02.027" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 139" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30922844" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 139" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6527865" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 139" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20role%20for%20Chromatin%20remodeling%20in%20cohesin%20loading%20onto%20chromosomes&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2019.02.027&amp;volume=74&amp;pages=664-673.e5&amp;publication_year=2019&amp;author=Mu%C3%B1oz%2CS&amp;author=Minamino%2CM&amp;author=Casas-Delucchi%2CCS&amp;author=Patel%2CH&amp;author=Uhlmann%2CF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="140."><p class="c-article-references__text" id="ref-CR140">Del Moral-Morales, A. et al. CTCF and its multi-partner network for chromatin regulation. <i>Cells</i> <b>12</b>, 1–19 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="MATH reference 140" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1122.68665" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 140" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CTCF%20and%20its%20multi-partner%20network%20for%20chromatin%20regulation&amp;journal=Cells&amp;volume=12&amp;pages=1-19&amp;publication_year=2023&amp;author=Moral-Morales%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="141."><p class="c-article-references__text" id="ref-CR141">Swinburne, I. A., Mosaliganti, K. R., Green, A. A. &amp; Megason, S. G. Improved long-term imaging of embryos with genetically encoded α-bungarotoxin. <i>PLoS One</i> <b>10</b>, 1–15 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 141" data-doi="10.1371/journal.pone.0134005" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1371/journal.pone.0134005" data-track-label="10.1371/journal.pone.0134005" data-track-value="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0134005" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 141" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20long-term%20imaging%20of%20embryos%20with%20genetically%20encoded%20%CE%B1-bungarotoxin&amp;journal=PLoS%20One&amp;doi=10.1371%2Fjournal.pone.0134005&amp;volume=10&amp;pages=1-15&amp;publication_year=2015&amp;author=Swinburne%2CIA&amp;author=Mosaliganti%2CKR&amp;author=Green%2CAA&amp;author=Megason%2CSG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="142."><p class="c-article-references__text" id="ref-CR142">Link, V., Shevchenko, A. &amp; Heisenberg, C. P. Proteomics of early zebrafish embryos. <i>BMC Dev. Biol.</i> <b>6</b>, 1–9 (2006).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 142" data-doi="10.1186/1471-213X-6-1" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1186/1471-213X-6-1" data-track-label="10.1186/1471-213X-6-1" data-track-value="article reference" href="https://link.springer.com/doi/10.1186/1471-213X-6-1" rel="noopener">Article</a> 
    <a aria-label="PubMed reference 142" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16412219" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 142" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1363346" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 142" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1076.03001" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 142" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Proteomics%20of%20early%20zebrafish%20embryos&amp;journal=BMC%20Dev.%20Biol.&amp;doi=10.1186%2F1471-213X-6-1&amp;volume=6&amp;pages=1-9&amp;publication_year=2006&amp;author=Link%2CV&amp;author=Shevchenko%2CA&amp;author=Heisenberg%2CCP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="143."><p class="c-article-references__text" id="ref-CR143">Sato, Y. et al. Histone H3K27 acetylation precedes active transcription during zebrafish zygotic genome activation as revealed by live-cell analysis. <i>Dev</i>. <b>146</b>, dev179127 (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="144."><p class="c-article-references__text" id="ref-CR144">Heyn, P., Salmonowicz, H., Rodenfels, J. &amp; Neugebauer, K. M. Activation of transcription enforces the formation of distinct nuclear bodies in zebrafish embryos. <i>RNA Biol.</i> <b>14</b>, 752–760 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 144" data-doi="10.1080/15476286.2016.1255397" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1080/15476286.2016.1255397" data-track-label="10.1080/15476286.2016.1255397" data-track-value="article reference" href="https://doi.org/10.1080%2F15476286.2016.1255397" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 144" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27858508" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 144" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Activation%20of%20transcription%20enforces%20the%20formation%20of%20distinct%20nuclear%20bodies%20in%20zebrafish%20embryos&amp;journal=RNA%20Biol.&amp;doi=10.1080%2F15476286.2016.1255397&amp;volume=14&amp;pages=752-760&amp;publication_year=2017&amp;author=Heyn%2CP&amp;author=Salmonowicz%2CH&amp;author=Rodenfels%2CJ&amp;author=Neugebauer%2CKM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="145."><p class="c-article-references__text" id="ref-CR145">Pancholi, A. et al. RNA polymerase II clusters form in line with surface condensation on regulatory chromatin. <i>Mol. Syst. Biol.</i> <b>17</b>, 1–26 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 145" data-doi="10.15252/msb.202110272" data-track="click_references" data-track-action="article reference" data-track-item_id="10.15252/msb.202110272" data-track-label="10.15252/msb.202110272" data-track-value="article reference" href="https://doi.org/10.15252%2Fmsb.202110272" rel="nofollow noopener">Article</a> 
    <a aria-label="MATH reference 145" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1533.57060" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 145" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=RNA%20polymerase%20II%20clusters%20form%20in%20line%20with%20surface%20condensation%20on%20regulatory%20chromatin&amp;journal=Mol.%20Syst.%20Biol.&amp;doi=10.15252%2Fmsb.202110272&amp;volume=17&amp;pages=1-26&amp;publication_year=2021&amp;author=Pancholi%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="146."><p class="c-article-references__text" id="ref-CR146">Hilbert, L. et al. Transcription organizes euchromatin via microphase separation. <i>Nat. Commun.</i> <b>12</b>, 1–12 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Google Scholar reference 146" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcription%20organizes%20euchromatin%20via%20microphase%20separation&amp;journal=Nat.%20Commun.&amp;volume=12&amp;pages=1-12&amp;publication_year=2021&amp;author=Hilbert%2CL" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="147."><p class="c-article-references__text" id="ref-CR147">Hadzhiev, Y. et al. The miR-430 locus with extreme promoter density forms a transcription body during the minor wave of zygotic genome activation. <i>Dev. Cell</i> <b>58</b>, 155–170.e8 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 147" data-doi="10.1016/j.devcel.2022.12.007" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.devcel.2022.12.007" data-track-label="10.1016/j.devcel.2022.12.007" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.devcel.2022.12.007" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 147" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXhs1eiu7s%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 147" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36693321" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 147" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9904021" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 147" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0223.57033" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 147" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20miR-430%20locus%20with%20extreme%20promoter%20density%20forms%20a%20transcription%20body%20during%20the%20minor%20wave%20of%20zygotic%20genome%20activation&amp;journal=Dev.%20Cell&amp;doi=10.1016%2Fj.devcel.2022.12.007&amp;volume=58&amp;pages=155-170.e8&amp;publication_year=2023&amp;author=Hadzhiev%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="148."><p class="c-article-references__text" id="ref-CR148">Kontur, C., Jeong, M., Cifuentes, D. &amp; Giraldez, A. J. Ythdf m6A readers function redundantly during Zebrafish development. <i>Cell Rep.</i> <b>33</b>, 108598 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 148" data-doi="10.1016/j.celrep.2020.108598" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.celrep.2020.108598" data-track-label="10.1016/j.celrep.2020.108598" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.celrep.2020.108598" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 148" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXjtVOhuw%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 148" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33378672" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 148" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11407899" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 148" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Ythdf%20m6A%20readers%20function%20redundantly%20during%20Zebrafish%20development&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2020.108598&amp;volume=33&amp;publication_year=2020&amp;author=Kontur%2CC&amp;author=Jeong%2CM&amp;author=Cifuentes%2CD&amp;author=Giraldez%2CAJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="149."><p class="c-article-references__text" id="ref-CR149">Schindelin, J. et al. Fiji: An open-source platform for biological-image analysis. <i>Nat. Methods</i> <b>9</b>, 676–682 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 149" data-doi="10.1038/nmeth.2019" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nmeth.2019" data-track-label="10.1038/nmeth.2019" data-track-value="article reference" href="https://doi.org/10.1038%2Fnmeth.2019" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 149" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhtVKnurbJ" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 149" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22743772" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 149" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1294.62003" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 149" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Fiji%3A%20An%20open-source%20platform%20for%20biological-image%20analysis&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.2019&amp;volume=9&amp;pages=676-682&amp;publication_year=2012&amp;author=Schindelin%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="150."><p class="c-article-references__text" id="ref-CR150">Chen, B. C. et al. Lattice light-sheet microscopy: Imaging molecules to embryos at high spatiotemporal resolution. <i>Science</i>. <b>346</b>, 1257998 (2014).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="151."><p class="c-article-references__text" id="ref-CR151">Lambert, Talley Shao, L. &amp; VolkerH. tlambert03/LLSpy: v0.4.8 (0.4.8). <a data-track="click_references" data-track-action="external reference" data-track-label="10.5281/zenodo.3554482" data-track-value="external reference" href="https://doi.org/10.5281/zenodo.3554482">https://doi.org/10.5281/zenodo.3554482</a> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="152."><p class="c-article-references__text" id="ref-CR152">Ollion, J., Cochennec, J., Loll, F., Escudé, C. &amp; Boudier, T. TANGO: A generic tool for high-throughput 3D image analysis for studying nuclear organization. <i>Bioinformatics</i> <b>29</b>, 1840–1841 (2013).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 152" data-doi="10.1093/bioinformatics/btt276" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/bioinformatics/btt276" data-track-label="10.1093/bioinformatics/btt276" data-track-value="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtt276" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 152" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtVOjsrvJ" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 152" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23681123" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 152" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3702251" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 152" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=TANGO%3A%20A%20generic%20tool%20for%20high-throughput%203D%20image%20analysis%20for%20studying%20nuclear%20organization&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtt276&amp;volume=29&amp;pages=1840-1841&amp;publication_year=2013&amp;author=Ollion%2CJ&amp;author=Cochennec%2CJ&amp;author=Loll%2CF&amp;author=Escud%C3%A9%2CC&amp;author=Boudier%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="153."><p class="c-article-references__text" id="ref-CR153">Langmead, B. &amp; Salzberg, S. L. Fast gapped-read alignment with Bowtie 2. <i>Nat. Methods</i> <b>9</b>, 357–359 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 153" data-doi="10.1038/nmeth.1923" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/nmeth.1923" data-track-label="10.1038/nmeth.1923" data-track-value="article reference" href="https://doi.org/10.1038%2Fnmeth.1923" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 153" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xjt1Oqt7c%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 153" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22388286" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 153" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322381" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 153" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?07740911" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 153" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.1923&amp;volume=9&amp;pages=357-359&amp;publication_year=2012&amp;author=Langmead%2CB&amp;author=Salzberg%2CSL" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="154."><p class="c-article-references__text" id="ref-CR154">Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). <i>Genome Biol</i>. <b>9</b>, R137 (2008).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="155."><p class="c-article-references__text" id="ref-CR155">Quinlan, A. R. &amp; Hall, I. M. BEDTools: A flexible suite of utilities for comparing genomic features. <i>Bioinformatics</i> <b>26</b>, 841–842 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 155" data-doi="10.1093/bioinformatics/btq033" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/bioinformatics/btq033" data-track-label="10.1093/bioinformatics/btq033" data-track-value="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtq033" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 155" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXivFGkurc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 155" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20110278" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 155" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832824" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 155" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0942.65101" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 155" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=BEDTools%3A%20A%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtq033&amp;volume=26&amp;pages=841-842&amp;publication_year=2010&amp;author=Quinlan%2CAR&amp;author=Hall%2CIM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="156."><p class="c-article-references__text" id="ref-CR156">Heinz, S. et al. Simple Combinations of lineage-determining transcription factors Prime cis-regulatory elements required for macrophage and B cell identities. <i>Mol. Cell</i> <b>38</b>, 576–589 (2010).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 156" data-doi="10.1016/j.molcel.2010.05.004" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.molcel.2010.05.004" data-track-label="10.1016/j.molcel.2010.05.004" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.molcel.2010.05.004" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 156" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXns1SlsLc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 156" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20513432" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 156" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898526" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 156" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1191.68211" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 156" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Simple%20Combinations%20of%20lineage-determining%20transcription%20factors%20Prime%20cis-regulatory%20elements%20required%20for%20macrophage%20and%20B%20cell%20identities&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2010.05.004&amp;volume=38&amp;pages=576-589&amp;publication_year=2010&amp;author=Heinz%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="157."><p class="c-article-references__text" id="ref-CR157">Plimpton, S. Fast Parallel Algorithms For Short-range Molecular Dynamics. <i>J. Comput. Phys.</i> <b>117</b>, 1–19 (1995).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 157" data-doi="10.1006/jcph.1995.1039" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1006/jcph.1995.1039" data-track-label="10.1006/jcph.1995.1039" data-track-value="article reference" href="https://doi.org/10.1006%2Fjcph.1995.1039" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 157" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1995JCoPh.117....1P" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 157" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK2MXlt1ejs7Y%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="MATH reference 157" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0830.65120" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 157" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20Parallel%20Algorithms%20For%20Short-range%20Molecular%20Dynamics&amp;journal=J.%20Comput.%20Phys.&amp;doi=10.1006%2Fjcph.1995.1039&amp;volume=117&amp;pages=1-19&amp;publication_year=1995&amp;author=Plimpton%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="158."><p class="c-article-references__text" id="ref-CR158">De Gennes, P. G. <i>Scaling Concepts in Polymer Physics</i>. <i>Cornell University Press</i> (1979).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="159."><p class="c-article-references__text" id="ref-CR159">Barshad, G. et al. RNA polymerase II dynamics shape enhancer-promoter interactions. <i>Nat. Genet</i>. <a data-track="click_references" data-track-action="external reference" data-track-label="10.1038/s41588-023-01442-7" data-track-value="external reference" href="https://doi.org/10.1038/s41588-023-01442-7">https://doi.org/10.1038/s41588-023-01442-7</a> (2023).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="160."><p class="c-article-references__text" id="ref-CR160">Dequeker, B. J. H. et al. MCM complexes are barriers that restrict cohesin-mediated loop extrusion. <i>Nature</i> <b>606</b>, 197–203 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 160" data-doi="10.1038/s41586-022-04730-0" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41586-022-04730-0" data-track-label="10.1038/s41586-022-04730-0" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41586-022-04730-0" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 160" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022Natur.606..197D" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 160" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38Xhtlajtr3L" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 160" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35585235" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 160" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9159944" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 160" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=MCM%20complexes%20are%20barriers%20that%20restrict%20cohesin-mediated%20loop%20extrusion&amp;journal=Nature&amp;doi=10.1038%2Fs41586-022-04730-0&amp;volume=606&amp;pages=197-203&amp;publication_year=2022&amp;author=Dequeker%2CBJH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="161."><p class="c-article-references__text" id="ref-CR161">Coßmann, J. et al. Increasingly efficient chromatin binding of cohesin and CTCF supports chromatin architecture formation during zebrafish embryogenesis [Dataset]. <a data-track="click_references" data-track-action="external reference" data-track-label="10.5061/dryad.3bk3j9ks8" data-track-value="external reference" href="https://doi.org/10.5061/dryad.3bk3j9ks8">https://doi.org/10.5061/dryad.3bk3j9ks8</a> (2025).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="162."><p class="c-article-references__text" id="ref-CR162">Kuhn, T., Hettich, J., Davtyan, R. &amp; Gebhardt, J. C. M. TrackIt\_v1.5. <a data-track="click_references" data-track-action="external reference" data-track-label="10.5281/zenodo.7092296" data-track-value="external reference" href="https://doi.org/10.5281/zenodo.7092296">https://doi.org/10.5281/zenodo.7092296</a> (2022).</p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1038/s41467-025-56889-5?format=refman&amp;flavour=references" rel="nofollow">Download references<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-download-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We thank Gilbert Weidinger (Ulm University) for generously granting us access to his zebrafish facility and for offering insightful discussion, as well as Doris Weber (Ulm University) for her expert guidance on zebrafish handling. Moreover, we are thankful to Astrid Bellan-Koch (Ulm University) and Jutta Hegler (Ulm University) for adding in cloning the fusion constructs and performing Western blots, Jens Michaelis (Ulm University) for his excellent contributions in developing the lattice light-sheet microscope, and members of the Gebhardt and Michaelis labs for supportive discussions. We further thank Aleksandra Galitsyna (Massachusetts Institute of Technology) for important discussions on Hi-C data. pmtb-t7-alpha-bungarotoxin was a gift from Sean Megason (Harward Medical School). The work was funded by the European Research Council (ERC) under the European Union’s Horizon 2020 Research and Innovation Program (no. 637987 ChromArch to J.C.M.G.) and the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation no. 422780363 SPP 2202 and no. 427512076 to J.C.M.G.). Support by the Collaborative Research Centers no. 316249678 (CRC 1279) and no. 450627322 (CRC 1506) to J.C.M.G. and the DFG Center for Translational Imaging MoMAN (no. 447235146) of Ulm University is acknowledged. Research of P.I.K. and L.G. was funded by the Novartis Foundation, the European Research Council (ERC grant no. 759366, ‘BioMeTre’), and the Swiss National Science Foundation (grant no. 310030_192642). Work in the Papantonis lab was supported by the German Research Foundation (DFG) via the Collaborative Research Center 1565 (Project no. 469281184) and the Priority Program 2202 (Project no. 422389065).</p></div></div></section><section data-title="Funding"><div class="c-article-section" id="Fun-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Fun">Funding</h2><div class="c-article-section__content" id="Fun-content"><p>Open Access funding enabled and organized by Projekt DEAL.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="nAff4"><p class="c-article-author-information__authors-list">Jonas Coßmann, Devin S. Assenheimer, Tobias A. Bischof &amp; J. Christof M. Gebhardt</p><p class="js-present-address">Present address: Institute of Experimental Physics and IQST, Ulm University, Ulm, Germany</p></li><li class="c-article-author-information__item" id="nAff5"><p class="c-article-author-information__authors-list">Pavel I. Kos</p><p class="js-present-address">Present address: Department of Biomedicine, University of Basel, Basel, Switzerland</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Institute of Biophysics, Ulm University, Ulm, Germany</p><p class="c-article-author-affiliation__authors-list">Jonas Coßmann, Devin S. Assenheimer, Tobias A. Bischof, Timo Kuhn, Thomas Vomhof &amp; J. Christof M. Gebhardt</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Friedrich Miescher Institute for Biomedical Research, Basel, Switzerland</p><p class="c-article-author-affiliation__authors-list">Pavel I. Kos &amp; Luca Giorgetti</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Institute of Pathology, University Medical Center Göttingen, Göttingen, Germany</p><p class="c-article-author-affiliation__authors-list">Vassiliki Varamogianni-Mamatsi &amp; Argyris Papantonis</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Jonas-Co_mann-Aff1-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Jonas Coßmann</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Jonas%20Co%C3%9Fmann" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jonas%20Co%C3%9Fmann" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jonas%20Co%C3%9Fmann%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Pavel_I_-Kos-Aff2-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Pavel I. Kos</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Pavel%20I.%20Kos" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Pavel%20I.%20Kos" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Pavel%20I.%20Kos%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Vassiliki-Varamogianni_Mamatsi-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Vassiliki Varamogianni-Mamatsi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Vassiliki%20Varamogianni-Mamatsi" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Vassiliki%20Varamogianni-Mamatsi" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Vassiliki%20Varamogianni-Mamatsi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Devin_S_-Assenheimer-Aff1-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Devin S. Assenheimer</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Devin%20S.%20Assenheimer" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Devin%20S.%20Assenheimer" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Devin%20S.%20Assenheimer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Tobias_A_-Bischof-Aff1-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Tobias A. Bischof</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Tobias%20A.%20Bischof" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Tobias%20A.%20Bischof" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tobias%20A.%20Bischof%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Timo-Kuhn-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Timo Kuhn</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Timo%20Kuhn" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Timo%20Kuhn" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Timo%20Kuhn%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Thomas-Vomhof-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Thomas Vomhof</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Thomas%20Vomhof" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Thomas%20Vomhof" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Thomas%20Vomhof%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Argyris-Papantonis-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Argyris Papantonis</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Argyris%20Papantonis" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Argyris%20Papantonis" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Argyris%20Papantonis%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Luca-Giorgetti-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Luca Giorgetti</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Luca%20Giorgetti" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Luca%20Giorgetti" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Luca%20Giorgetti%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-J__Christof_M_-Gebhardt-Aff1-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">J. Christof M. Gebhardt</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=J.%20Christof%20M.%20Gebhardt" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=J.%20Christof%20M.%20Gebhardt" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22J.%20Christof%20M.%20Gebhardt%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>J.C.M.G. conceived the project; J.C. and J.C.M.G. designed the project; J.C. performed the single-molecule measurements; T.A.B. contributed to the single-molecule measurements; J.C. analyzed data with contributions from D.S.A. and J.C.M.G.; J.C. performed biochemical techniques; T.A.B. performed phenotype analysis with contributions from J.C.; T.K. and T.V. set up the lattice light-sheet microscope; V.V.M. and A. P. performed bioinformatics analysis; P.I.K. and L.G. performed computational modeling and analysis; J.C., P.I.K., and J.C.M.G. wrote the manuscript with comments from all authors.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a href="mailto:christof.gebhardt@uni-ulm.de" id="corresp-c1">J. Christof M. Gebhardt</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
<h3 class="c-article__sub-heading" id="FPar2">Competing interests</h3>
<p>The authors declare no competing interests.</p>
</div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
<h3 class="c-article__sub-heading" id="FPar1">Peer review information</h3>
<p><i>Nature Communications</i> thanks Julia Horsfield, José Santos-Pereira and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p>
</div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec32-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec32">Supplementary information</h2><div class="c-article-section__content" id="Sec32-content"><div data-test="supplementary-info"><div class="c-article-figshare-container" data-test="figshare-container" id="figshareContainer"></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_MOESM1_ESM.pdf">Supplementary Information</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="description of additional supplementary files" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_MOESM2_ESM.pdf">Description of Additional Supplementary Files</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary movie 1" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_MOESM3_ESM.mp4">Supplementary Movie 1</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary movie 2" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_MOESM4_ESM.mp4">Supplementary Movie 2</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM5"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary movie 3" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_MOESM5_ESM.mp4">Supplementary Movie 3</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary movie 4" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_MOESM6_ESM.mp4">Supplementary Movie 4</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary movie 5" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_MOESM7_ESM.mp4">Supplementary Movie 5</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary movie 6" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_MOESM8_ESM.mp4">Supplementary Movie 6</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary movie 7" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_MOESM9_ESM.mp4">Supplementary Movie 7</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary movie 8" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_MOESM10_ESM.mp4">Supplementary Movie 8</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_MOESM11_ESM.pdf">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM12"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="transparent peer review file" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_MOESM12_ESM.pdf">Transparent Peer Review file</a></h3></div></div></div></div></section><section data-title="Source data"><div class="c-article-section" id="Sec33-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec33">Source data</h2><div class="c-article-section__content" id="Sec33-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM13"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="source data" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-025-56889-5/MediaObjects/41467_2025_56889_MOESM13_ESM.xlsx">Source Data</a></h3></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
<p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Increasingly%20efficient%20chromatin%20binding%20of%20cohesin%20and%20CTCF%20supports%20chromatin%20architecture%20formation%20during%20zebrafish%20embryogenesis&amp;author=Jonas%20Co%C3%9Fmann%20et%20al&amp;contentID=10.1038%2Fs41467-025-56889-5&amp;copyright=The%20Author%28s%29&amp;publication=2041-1723&amp;publicationDate=2025-02-21&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41467-025-56889-5" data-test="crossmark" data-track="click" data-track-action="Click Crossmark" data-track-label="link" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41467-025-56889-5" rel="noopener" target="_blank"><img alt="Check for updates. Verify currency and authenticity via CrossMark" height="81" loading="lazy" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>" width="57"/></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Coßmann, J., Kos, P.I., Varamogianni-Mamatsi, V. <i>et al.</i> Increasingly efficient chromatin binding of cohesin and CTCF supports chromatin architecture formation during zebrafish embryogenesis.
                    <i>Nat Commun</i> <b>16</b>, 1833 (2025). https://doi.org/10.1038/s41467-025-56889-5</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-external="" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1038/s41467-025-56889-5?format=refman&amp;flavour=citation" rel="nofollow">Download citation<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-download-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2023-12-15">15 December 2023</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2025-02-05">05 February 2025</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2025-02-21">21 February 2025</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41467-025-56889-5</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" data-track="click" data-track-action="get shareable link" data-track-external="" data-track-label="button" id="get-share-url" type="button">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" data-track="click" data-track-action="select share url" data-track-label="button" id="share-url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" data-track="click" data-track-action="copy share url" data-track-external="" data-track-label="button" id="copy-share-url" type="button">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
</div>
</div>
</article>
</main>
<aside aria-label="Article navigation" class="c-article-extras u-hide-print" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
<div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41467-025-56889-5.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
<div class="c-reading-companion">
<div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
<div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
<div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu="">
<div class="c-ad c-ad--300x250">
<div class="c-ad__inner">
<p class="c-ad__label">Advertisement</p>
<div class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-gpt="" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41467-025-56889-5;doi=10.1038/s41467-025-56889-5;techmeta=14,34,63;subjmeta=100,101,136,2086,2265,337,386,57,631;kwrd=Chromatin+structure,Embryogenesis,Nuclear+organization,Single-molecule+biophysics" data-gpt-unitpath="/285/nature_communications/article" data-pa11y-ignore="" data-test="right-ad" id="div-gpt-ad-right-2">
<noscript>
<a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/nature_communications/article&amp;sz=300x250&amp;c=2066501333&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41467-025-56889-5%26doi%3D10.1038/s41467-025-56889-5%26techmeta%3D14,34,63%26subjmeta%3D100,101,136,2086,2265,337,386,57,631%26kwrd%3DChromatin+structure,Embryogenesis,Nuclear+organization,Single-molecule+biophysics">
<img alt="Advertisement" data-test="gpt-advert-fallback-img" height="250" src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/nature_communications/article&amp;sz=300x250&amp;c=2066501333&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41467-025-56889-5%26doi%3D10.1038/s41467-025-56889-5%26techmeta%3D14,34,63%26subjmeta%3D100,101,136,2086,2265,337,386,57,631%26kwrd%3DChromatin+structure,Embryogenesis,Nuclear+organization,Single-molecule+biophysics" width="300"/></a>
</noscript>
</div>
</div>
</div>
</div>
</div>
<div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
<div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
</div>
</div>
</aside>
</div>
<nav aria-labelledby="Explore-content" class="c-header__dropdown" data-test="Explore-content" data-track-component="nature-150-split-header" id="explore">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="Explore-content">Explore content</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="research articles" data-track-label="link" href="/ncomms/research-articles">
                                    Research articles
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" href="/ncomms/reviews-and-analysis">
                                    Reviews &amp; Analysis
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="news &amp; comment" data-track-label="link" href="/ncomms/news-and-comment">
                                    News &amp; Comment
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="videos" data-track-label="link" href="/ncomms/video">
                                    Videos
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="collections" data-track-label="link" href="/ncomms/collections">
                                    Collections
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="subjects" data-track-label="link" href="/ncomms/browse-subjects">
                                    Subjects
                                </a>
</li>
</ul>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="facebook" data-track-label="link" href="https://www.facebook.com/NatureCommunications">Follow us on Facebook
                            </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="twitter" data-track-label="link" href="https://twitter.com/NatureComms">Follow us on Twitter
                            </a>
</li>
<li class="c-header__item c-header__item--hide-lg">
<a class="c-header__link" data-track="click" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=264" rel="nofollow">Sign up for alerts<svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--hide-lg">
<a class="c-header__link" data-track="click" data-track-action="rss feed" data-track-label="link" href="https://www.nature.com/ncomms.rss">
<span>RSS feed</span>
</a>
</li>
</ul>
</div>
</nav>
<nav aria-labelledby="About-the-journal" class="c-header__dropdown" data-test="about-the-journal" data-track-component="nature-150-split-header" id="about-the-journal">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="About-the-journal">About the journal</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="aims &amp; scope" data-track-label="link" href="/ncomms/aims">
                                    Aims &amp; Scope
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editors" data-track-label="link" href="/ncomms/editors">
                                    Editors
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="journal information" data-track-label="link" href="/ncomms/journal-information">
                                    Journal Information
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="open access fees and funding" data-track-label="link" href="/ncomms/open-access">
                                    Open Access Fees and Funding
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="calls for papers" data-track-label="link" href="/ncomms/calls-for-papers">
                                    Calls for Papers
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editorial values statement" data-track-label="link" href="/ncomms/editorial-values-statement">
                                    Editorial Values Statement
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="journal metrics" data-track-label="link" href="/ncomms/journal-impact">
                                    Journal Metrics
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editors' highlights" data-track-label="link" href="/ncomms/editorshighlights">
                                    Editors' Highlights
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="contact" data-track-label="link" href="/ncomms/contact">
                                    Contact
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editorial policies" data-track-label="link" href="/ncomms/editorial-policies">
                                    Editorial policies
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="top articles" data-track-label="link" href="/ncomms/top-articles">
                                    Top Articles
                                </a>
</li>
</ul>
</div>
</nav>
<nav aria-labelledby="Publish-with-us-label" class="c-header__dropdown" data-test="publish-with-us" data-track-component="nature-150-split-header" id="publish-with-us">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="Publish-with-us-label">Publish with us</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="for authors" data-track-label="link" href="/ncomms/submit">
                                    For authors
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="for reviewers" data-track-label="link" href="/ncomms/for-reviewers">
                                    For Reviewers
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-action="manuscript author services" data-track-context="header publish with us dropdown menu" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
</li>
<li class="c-header__item c-header__item--keyline">
<a class="c-header__link" data-gtm-criteo="submit-manuscript" data-track="click_submit_manuscript" data-track-action="submit manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-external="" data-track-label="link (publish with us dropdown menu)" href="https://mts-ncomms.nature.com/">Submit manuscript<svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
</a>
</li>
</ul>
</div>
</nav>
<div class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header" id="search-menu">
<div class="c-header__container">
<h2 class="c-header__visually-hidden">Search</h2>
<form action="/search" autocomplete="off" class="c-header__search-form" data-test="inline-search" method="get" role="search">
<label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
<div class="c-header__search-layout c-header__search-layout--max-width">
<div>
<input class="c-header__input" id="keywords" name="q" required="" type="text" value=""/>
</div>
<div class="c-header__search-layout">
<div>
<label class="c-header__visually-hidden" for="results-from">Show results from</label>
<select class="c-header__select" id="results-from" name="journal">
<option selected="" value="">All journals</option>
<option value="ncomms">This journal</option>
</select>
</div>
<div>
<button class="c-header__search-button" type="submit">Search</button>
</div>
</div>
</div>
</form>
<div class="c-header__flush">
<a class="c-header__link" data-track="click" data-track-action="advanced search" data-track-label="link" href="/search/advanced">
                Advanced search
            </a>
</div>
<h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
<ul class="c-header__list">
<li><a class="c-header__link" data-track="click" data-track-action="explore articles by subject" data-track-label="link" href="/subjects">Explore articles by subject</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="find a job" data-track-label="link" href="/naturecareers">Find a job</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="guide to authors" data-track-label="link" href="/authors/index.html">Guide to authors</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="editorial policies" data-track-label="link" href="/authors/editorial_policies/">Editorial policies</a></li>
</ul>
</div>
</div>
<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
<meta content="Springer Nature" itemprop="publisher"/>
<div class="u-mt-16 u-mb-16">
<div class="u-container">
<div class="u-display-flex u-flex-wrap u-justify-content-space-between">
<p class="c-meta u-ma-0 u-flex-shrink">
<span class="c-meta__item">
                    Nature Communications (<i>Nat Commun</i>)
                </span>
<span class="c-meta__item">
<abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">2041-1723</span> (online)
    </span>
</p>
</div>
</div>
</div>
<div class="c-footer">
<div class="u-hide-print" data-track-component="footer">
<h2 class="u-visually-hidden">nature.com sitemap</h2>
<div class="c-footer__container">
<div class="c-footer__grid c-footer__group--separator">
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="about us" data-track-label="link" href="https://www.nature.com/npg_/company_info/index.html">About us</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="press releases" data-track-label="link" href="https://www.nature.com/npg_/press_room/press_releases.html">Press releases</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="press office" data-track-label="link" href="https://press.nature.com/">Press office</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="contact us" data-track-label="link" href="https://support.nature.com/support/home">Contact us</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Discover content</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="journals a-z" data-track-label="link" href="https://www.nature.com/siteindex">Journals A-Z</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="article by subject" data-track-label="link" href="https://www.nature.com/subjects">Articles by subject</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="protocols.io" data-track-label="link" href="https://www.protocols.io/">protocols.io</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature index" data-track-label="link" href="https://www.natureindex.com/">Nature Index</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link" href="https://www.nature.com/authors/editorial_policies">Nature portfolio policies</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="open access" data-track-label="link" href="https://www.nature.com/nature-research/open-access">Open access</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="reprints and permissions" data-track-label="link" href="https://www.nature.com/reprints">Reprints &amp; permissions</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="data research service" data-track-label="link" href="https://www.springernature.com/gp/authors/research-data">Research data</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="language editing" data-track-label="link" href="https://authorservices.springernature.com/language-editing/">Language editing</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="scientific editing" data-track-label="link" href="https://authorservices.springernature.com/scientific-editing/">Scientific editing</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature masterclasses" data-track-label="link" href="https://masterclasses.nature.com/">Nature Masterclasses</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="research solutions" data-track-label="link" href="https://solutions.springernature.com/">Research Solutions</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="librarian service and tools" data-track-label="link" href="https://www.springernature.com/gp/librarians/tools-services">Librarian service &amp; tools</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="librarian portal" data-track-label="link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal">Librarian portal</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="open research" data-track-label="link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions">Open research</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="Recommend to library" data-track-label="link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library">Recommend to library</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="advertising" data-track-label="link" href="https://partnerships.nature.com/product/digital-advertising/">Advertising</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="partnerships and services" data-track-label="link" href="https://partnerships.nature.com/">Partnerships &amp; Services</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="media kits" data-track-label="link" href="https://partnerships.nature.com/media-kits/">Media kits</a>
</li>
<li class="c-footer__item"><a class="c-footer__link" data-track-action="branded content" data-track-label="link" href="https://partnerships.nature.com/product/branded-content-native-advertising/">Branded
                        content</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Professional development</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature careers" data-track-label="link" href="https://www.nature.com/naturecareers/">Nature Careers</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature conferences" data-track-label="link" href="https://conferences.nature.com">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Regional websites</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature africa" data-track-label="link" href="https://www.nature.com/natafrica">Nature Africa</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature china" data-track-label="link" href="http://www.naturechina.com">Nature China</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature india" data-track-label="link" href="https://www.nature.com/nindia">Nature India</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature Italy" data-track-label="link" href="https://www.nature.com/natitaly">Nature Italy</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature japan" data-track-label="link" href="https://www.natureasia.com/ja-jp">Nature Japan</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature middle east" data-track-label="link" href="https://www.nature.com/nmiddleeast">Nature Middle East</a></li>
</ul>
</div>
</div>
</div>
<div class="c-footer__container">
<ul class="c-footer__links">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="privacy policy" data-track-label="link" href="https://www.nature.com/info/privacy">Privacy
                Policy</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="use of cookies" data-track-label="link" href="https://www.nature.com/info/cookies">Use
                of cookies</a></li>
<li class="c-footer__item">
<button class="optanon-toggle-display c-footer__link" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link" onclick="javascript:;">Your privacy choices/Manage cookies
                </button>
</li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="legal notice" data-track-label="link" href="https://www.nature.com/info/legal-notice">Legal
                notice</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="accessibility statement" data-track-label="link" href="https://www.nature.com/info/accessibility-statement">Accessibility
                statement</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="terms and conditions" data-track-label="link" href="https://www.nature.com/info/terms-and-conditions">Terms &amp; Conditions</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="california privacy statement" data-track-label="link" href="https://www.springernature.com/ccpa">Your US state privacy rights</a></li>
</ul>
</div>
</div>
<div class="c-footer__container">
<a class="c-footer__link" href="https://www.springernature.com/">
<img alt="Springer Nature" height="20" loading="lazy" src="/static/images/logos/sn-logo-white-ea63208b81.svg" width="200"/>
</a>
<p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>
</div>
<div aria-hidden="true" class="u-visually-hidden">
<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path d="M0 .74h56.72v55.24H0z" id="a"></path></defs><symbol id="icon-access" viewbox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewbox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewbox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewbox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewbox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewbox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewbox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewbox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewbox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewbox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewbox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewbox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewbox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewbox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewbox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewbox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewbox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewbox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewbox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewbox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewbox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewbox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewbox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewbox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewbox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewbox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewbox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewbox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewbox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewbox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewbox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewbox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewbox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewbox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewbox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewbox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewbox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewbox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewbox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewbox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewbox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewbox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewbox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewbox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewbox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewbox="0 0 56.69 56.69"><path d="M0 0h56.69v56.69H0z" style="fill:none"></path><clippath id="b"><use style="overflow:visible" xlink:href="#a"></use></clippath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewbox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewbox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewbox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewbox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewbox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewbox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-download-medium" viewbox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-mail-medium" viewbox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewbox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewbox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewbox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewbox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewbox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewbox="0 0 100 100"><path clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z" fill-rule="evenodd"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewbox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>
<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif" data-component-expirydays="30" data-component-id="nature-briefing-banner" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Flagship">
<div class="c-site-messages__banner-large">
<div class="c-site-messages__close-container">
<button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
<svg aria-hidden="true" focusable="false" height="25px" version="1.1" viewbox="0 0 25 25" width="25px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<title>Close banner</title>
<defs></defs>
<g fill="none" fill-rule="evenodd" stroke="none" stroke-width="1">
<rect height="25" opacity="0" width="25" x="0" y="0"></rect>
<path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
</g>
</svg>
<span class="visually-hidden">Close</span>
</button>
</div>
<div class="c-site-messages__form-container">
<div class="grid grid-12 last">
<div class="grid grid-4">
<img alt="Nature Briefing" height="40" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250"/>
<p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
</div>
<div class="grid grid-8 last">
<form action="https://www.nature.com/briefing/briefing" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship" method="post">
<input id="briefing-banner-signup-form-input-track-originReferralPoint" name="track_originReferralPoint" type="hidden" value="MainBriefingBanner"/>
<input id="briefing-banner-signup-form-input-track-formType" name="track_formType" type="hidden" value="DirectEmailBanner"/>
<input id="gdpr_tick_banner" name="gdpr_tick" type="hidden" value="false"/>
<input id="marketing_input_banner" name="marketing" type="hidden" value="false"/>
<input id="marketing_tick_banner" name="marketing_tick" type="hidden" value="false"/>
<input id="brieferEntryPoint_banner" name="brieferEntryPoint" type="hidden" value="MainBriefingBanner"/>
<label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>
<div class="nature-briefing-banner__email-wrapper">
<input class="nature-briefing-banner__email-input box-sizing text14" data-test-element="briefing-emailbanner-email-input" id="emailAddress" name="emailAddress" placeholder="e.g. jo.smith@university.ac.uk" required="" type="email" value=""/>
<input id="defaultNewsletter_banner" name="N:nature_briefing_daily" type="hidden" value="true"/>
<button class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button" type="submit">Sign up</button>
</div>
<div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
<input class="nature-briefing-banner__checkbox-checkbox" data-test-element="briefing-emailbanner-gdpr-checkbox" id="gdpr-briefing-banner-checkbox" name="gdpr" required="" type="checkbox" value="true"/>
<label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
</div>
</form>
</div>
</div>
</div>
</div>
<div class="c-site-messages__banner-small">
<div class="c-site-messages__close-container">
<button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
<svg aria-hidden="true" focusable="false" height="25px" version="1.1" viewbox="0 0 25 25" width="25px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<title>Close banner</title>
<defs></defs>
<g fill="none" fill-rule="evenodd" stroke="none" stroke-width="1">
<rect height="25" opacity="0" width="25" x="0" y="0"></rect>
<path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
</g>
</svg>
<span class="visually-hidden">Close</span>
</button>
</div>
<div class="c-site-messages__content text14">
<span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
<a class="nature-briefing__link text14 sans-serif" data-test-element="briefing-banner-link" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner" rel="noreferrer noopener" target="_blank">Sign up for Nature Briefing
            </a>
</div>
</div>
</div>
<noscript>
<img alt="" height="0" hidden="" src="https://verify.nature.com/verify/nature.png" style="display: none" width="0"/>
</noscript>
<script async="" src="//content.readcube.com/ping?doi=10.1038/s41467-025-56889-5&amp;format=js&amp;last_modified=2025-02-21"></script>
</body>
</html>