<!DOCTYPE html>
<html class="grade-c" lang="en">
<head>
<title>Evaluating cutinase from Fusarium oxysporum as a biocatalyst for the degradation of nine synthetic polymer | Scientific Reports</title>
<link href="https://www.nature.com/srep.rss" rel="alternate" type="application/rss+xml"/>
<link crossorigin="" href="https://cmp.nature.com" rel="preconnect"/>
<meta content="IE=edge" http-equiv="X-UA-Compatible"/>
<meta content="pc,mobile" name="applicable-device"/>
<meta content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes" name="viewport"/>
<meta content="5a2dc4ab3fcb9b0393241ffbbb490480" name="360-site-verification"/>
<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"biochemistry;biophysics;biotechnology;chemistry;computational-biology-and-bioinformatics","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Scientific Reports","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41598-024-84718-0"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by-nc-nd/4.0/"}}},"contentInfo":{"authors":["Maycon Vinicius Damasceno de Oliveira","Gabriel Calandrini","Clauber Henrique Souza da Costa","Carlos Gabriel da Silva de Souza","Cláudio Nahum Alves","José Rogério A. Silva","Anderson H. Lima","Jerônimo Lameira"],"publishedAt":1737504000,"publishedAtString":"2025-01-22","title":"Evaluating cutinase from Fusarium oxysporum as a biocatalyst for the degradation of nine synthetic polymer","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Biochemistry,Biophysics,Biotechnology,Chemistry,Computational biology and bioinformatics"},"journal":{"pcode":"srep","title":"scientific reports","volume":"15","issue":"1","id":41598,"publishingModel":"Open Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>
<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
<style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-card--major .c-card__title,.u-h1,.u-h2,h1,h2{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h3,h4,h5,h6{letter-spacing:-.0117156rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}.c-card--major .c-card__title,.u-h1,.u-h2,button,h1,h2{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}button{border-radius:0;cursor:pointer}.c-card--major .c-card__title,.u-h1,.u-h2,h1,h2{font-weight:700}h1{font-size:2rem;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,h2{font-size:1.5rem;letter-spacing:-.0117156rem;line-height:1.6rem}.u-h3{letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h3,h4,h5,h6{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-reading-companion__figure-title,.u-h4,h3,h4,h5,h6{letter-spacing:-.0117156rem}.c-reading-companion__figure-title,.u-h4,h4{font-size:1.125rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-link-inherit{color:inherit}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-text-bold{font-weight:700}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>
<link data-inline-css-source="critical-css" data-test="critical-css-handler" href="/static/css/enhanced-article-c2d4d414fd.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null" rel="stylesheet"/>
<noscript>
<link href="/static/css/enhanced-article-c2d4d414fd.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" rel="stylesheet" type="text/css"/>
</noscript>
<link href="/static/css/article-print-122346e276.css" media="print" rel="stylesheet" type="text/css"/>
<link href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png" rel="apple-touch-icon" sizes="180x180"/>
<link href="/static/images/favicons/nature/favicon-48x48-b52890008c.png" rel="icon" sizes="48x48" type="image/png"/>
<link href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png" rel="icon" sizes="32x32" type="image/png"/>
<link href="/static/images/favicons/nature/favicon-16x16-951651ab72.png" rel="icon" sizes="16x16" type="image/png"/>
<link crossorigin="use-credentials" href="/static/manifest.json" rel="manifest"/>
<link color="#000000" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" rel="mask-icon"/>
<link href="/static/images/favicons/nature/favicon.ico" rel="shortcut icon"/>
<meta content="#000000" name="msapplication-TileColor"/>
<meta content="/static/browserconfig.xml" name="msapplication-config"/>
<meta content="#000000" name="theme-color"/>
<meta content="Nature" name="application-name"/>
<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>
<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->
<script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-74.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-74.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>
<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-606cb67187.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-e919764a53.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>
<meta content="noarchive" name="robots"/>
<meta content="Yes" name="access"/>
<link href="https://www.nature.com/search" rel="search"/>
<link href="https://www.nature.com/opensearch/opensearch.xml" rel="search" title="nature.com" type="application/opensearchdescription+xml"/>
<link href="https://www.nature.com/opensearch/request" rel="search" title="nature.com" type="application/sru+xml"/>
<script type="application/ld+json">{"mainEntity":{"headline":"Evaluating cutinase from Fusarium oxysporum as a biocatalyst for the degradation of nine synthetic polymer","description":"Plastic poses a significant environmental impact due to its chemical resilience, leading to prolonged and degradation times and resulting in widespread adverse effects on global flora and fauna. Cutinases are essential enzymes in the biodegradation process of synthetic polymers like polyethylene terephthalate (PET), which recognized organisms can break down. Here, we used molecular dynamics and binding free energy calculations to explore the interaction of nine synthetic polymers, including PET, with Cutinase from Fusarium oxysporum (FoCut). According to our findings, the polymers poly(ethylene terephthalate) (PET), poly(3-hydroxybutyrate-co-3-hydroxyhexanoate) (PHBH), poly(butylene succinate) (PBS), poly(butylene adipate-co-terephthalate) (PBAT) and poly(ε-caprolactone) (PCL) can bind to the Cutinase enzyme from F. oxysporum, indicating potential biodegradation activity for these polymers. PET exhibited the highest binding affinity (− 34.26 kcal/mol). Besides PET, the polymers PHBH, PBS, PBAT, and PCL also demonstrated significant affinities for the FoCut enzyme, with binding values of − 18.44, − 29.71, − 22.78, and − 22.26 kcal/mol, respectively. Additionally, analysis of the phylogenetic tree of cutinases produced by different organisms demonstrated that even though the organisms belong to different kingdoms, the cutinase from F. oxysporum (FoCut) showed biological similarity in its activity in degrading polymers with the cutinase enzyme from the bacterium Kineococcus radiotolerans and the fungus Moniliophthora roreri. Furthermore, the phylogenetic analysis demonstrated that the PETase enzyme has a very high similarity with the bacterial cutinase enzyme than with the fungal cutinase, therefore demonstrating that the PETase enzyme from Ideonella sakaiensis can easily be a modified bacterial cutinase enzyme that created a unique feature in biodegrading only the pet polymer through an evolutionary process due to its environment and its biochemical need for carbon. Our data demonstrate that bacterial cutinase enzymes have the same common ancestor as the PETase enzyme. Therefore, cutinases and PETase are interconnected through their biological similarity in biodegrading polymers. We demonstrated that important conserved regions, such as the Ser-Asp-His catalytic triad, exist in the enzyme’s catalytic site and that all Cut enzymes from different organisms have the same region to couple with the polymer structures.","datePublished":"2025-01-22T00:00:00Z","dateModified":"2025-01-22T00:00:00Z","pageStart":"1","pageEnd":"18","license":"http://creativecommons.org/licenses/by-nc-nd/4.0/","sameAs":"https://doi.org/10.1038/s41598-024-84718-0","keywords":["Biochemistry","Biophysics","Biotechnology","Chemistry","Computational biology and bioinformatics","Cutinase","Polymer","\n                        Fusarium oxysporum\n                     ","Molecular dynamic","Phylogentic","Science","Humanities and Social Sciences","multidisciplinary"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig6_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig7_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig8_HTML.png"],"isPartOf":{"name":"Scientific Reports","issn":["2045-2322"],"volumeNumber":"15","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group UK","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Maycon Vinicius Damasceno de Oliveira","url":"http://orcid.org/0000-0003-2263-2124","affiliation":[{"name":"Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará","address":{"name":"Laboratório de Planejamento e Desenvolvimento de Fármacos, Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará, Belém, Brazil","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Gabriel Calandrini","url":"http://orcid.org/0000-0001-6321-2228","affiliation":[{"name":"Universidade Federal do Pará","address":{"name":"Núcleo de Ecologia Aquática e Pesca (NEAP), Universidade Federal do Pará, Belém, Brazil","@type":"PostalAddress"},"@type":"Organization"}],"email":"gabriel.calandrini@icb.ufpa.br","@type":"Person"},{"name":"Clauber Henrique Souza da Costa","url":"http://orcid.org/0000-0002-6915-1056","affiliation":[{"name":"IQ/UNICAMP","address":{"name":"Center for Computing in Engineering & Sciences, IQ/UNICAMP, Campinas, Brazil","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Carlos Gabriel da Silva de Souza","url":"http://orcid.org/0000-0001-6572-7196","affiliation":[{"name":"Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará","address":{"name":"Laboratório de Planejamento e Desenvolvimento de Fármacos, Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará, Belém, Brazil","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Cláudio Nahum Alves","affiliation":[{"name":"Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará","address":{"name":"Laboratório de Planejamento e Desenvolvimento de Fármacos, Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará, Belém, Brazil","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"José Rogério A. Silva","url":"http://orcid.org/0000-0003-2310-5107","affiliation":[{"name":"Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará","address":{"name":"Laboratório de Planejamento e Desenvolvimento de Fármacos, Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará, Belém, Brazil","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of KwaZulu-Natal","address":{"name":"Catalysis and Peptide Research Unit, University of KwaZulu-Natal, Durban, South Africa","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Anderson H. Lima","url":"http://orcid.org/0000-0002-8451-9912","affiliation":[{"name":"Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará","address":{"name":"Laboratório de Planejamento e Desenvolvimento de Fármacos, Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará, Belém, Brazil","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jerônimo Lameira","url":"http://orcid.org/0000-0001-7270-1517","affiliation":[{"name":"Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará","address":{"name":"Laboratório de Planejamento e Desenvolvimento de Fármacos, Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará, Belém, Brazil","@type":"PostalAddress"},"@type":"Organization"}],"email":"lameira@ufpa.br","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>
<link href="https://www.nature.com/articles/s41598-024-84718-0" rel="canonical"/>
<meta content="41598" name="journal_id"/>
<meta content="Evaluating cutinase from Fusarium oxysporum as a biocatalyst for the degradation of nine synthetic polymer" name="dc.title"/>
<meta content="Scientific Reports 2024 15:1" name="dc.source"/>
<meta content="text/html" name="dc.format"/>
<meta content="Nature Publishing Group" name="dc.publisher"/>
<meta content="2025-01-22" name="dc.date"/>
<meta content="OriginalPaper" name="dc.type"/>
<meta content="En" name="dc.language"/>
<meta content="2024 The Author(s)" name="dc.copyright"/>
<meta content="2024 The Author(s)" name="dc.rights"/>
<meta content="journalpermissions@springernature.com" name="dc.rightsAgent"/>
<meta content="Plastic poses a significant environmental impact due to its chemical resilience, leading to prolonged and degradation times and resulting in widespread adverse effects on global flora and fauna. Cutinases are essential enzymes in the biodegradation process of synthetic polymers like polyethylene terephthalate (PET), which recognized organisms can break down. Here, we used molecular dynamics and binding free energy calculations to explore the interaction of nine synthetic polymers, including PET, with Cutinase from Fusarium oxysporum (FoCut). According to our findings, the polymers poly(ethylene terephthalate) (PET),&amp;nbsp;poly(3-hydroxybutyrate-co-3-hydroxyhexanoate) (PHBH), poly(butylene succinate) (PBS), poly(butylene adipate-co-terephthalate) (PBAT) and poly(ε-caprolactone) (PCL)&amp;nbsp;can bind to the Cutinase enzyme from&amp;nbsp;F. oxysporum, indicating potential biodegradation activity for these polymers. PET exhibited the highest binding affinity (−&amp;nbsp;34.26&amp;nbsp;kcal/mol). Besides PET, the polymers PHBH, PBS, PBAT, and PCL also demonstrated significant affinities for the FoCut enzyme, with binding values of −&amp;nbsp;18.44, −&amp;nbsp;29.71, −&amp;nbsp;22.78, and −&amp;nbsp;22.26&amp;nbsp;kcal/mol, respectively. Additionally, analysis of the phylogenetic tree of cutinases produced by different organisms demonstrated that even though the organisms belong to different kingdoms, the cutinase from F. oxysporum&amp;nbsp;(FoCut) showed biological similarity in its activity in degrading polymers with the cutinase enzyme from the bacterium Kineococcus radiotolerans&amp;nbsp;and the fungus&amp;nbsp;Moniliophthora roreri. Furthermore, the phylogenetic analysis demonstrated that the PETase enzyme has a very high similarity with the bacterial cutinase enzyme than with the fungal cutinase, therefore demonstrating that the PETase enzyme from Ideonella sakaiensis can easily be a modified bacterial cutinase enzyme that created a unique feature in biodegrading only the pet polymer through an evolutionary process due to its environment and its biochemical need for carbon. Our data demonstrate that bacterial cutinase enzymes have the same common ancestor as the PETase enzyme. Therefore, cutinases and PETase are interconnected through their biological similarity in biodegrading polymers. We demonstrated that important conserved regions, such as the Ser-Asp-His catalytic triad, exist in the enzyme’s catalytic site and that all Cut enzymes from different organisms have the same region to couple with the polymer structures." name="dc.description"/>
<meta content="2045-2322" name="prism.issn"/>
<meta content="Scientific Reports" name="prism.publicationName"/>
<meta content="2025-01-22" name="prism.publicationDate"/>
<meta content="15" name="prism.volume"/>
<meta content="1" name="prism.number"/>
<meta content="OriginalPaper" name="prism.section"/>
<meta content="1" name="prism.startingPage"/>
<meta content="18" name="prism.endingPage"/>
<meta content="2024 The Author(s)" name="prism.copyright"/>
<meta content="journalpermissions@springernature.com" name="prism.rightsAgent"/>
<meta content="https://www.nature.com/articles/s41598-024-84718-0" name="prism.url"/>
<meta content="doi:10.1038/s41598-024-84718-0" name="prism.doi"/>
<meta content="https://www.nature.com/articles/s41598-024-84718-0.pdf" name="citation_pdf_url"/>
<meta content="https://www.nature.com/articles/s41598-024-84718-0" name="citation_fulltext_html_url"/>
<meta content="Scientific Reports" name="citation_journal_title"/>
<meta content="Sci Rep" name="citation_journal_abbrev"/>
<meta content="Nature Publishing Group" name="citation_publisher"/>
<meta content="2045-2322" name="citation_issn"/>
<meta content="Evaluating cutinase from Fusarium oxysporum as a biocatalyst for the degradation of nine synthetic polymer" name="citation_title"/>
<meta content="15" name="citation_volume"/>
<meta content="1" name="citation_issue"/>
<meta content="2025/01/22" name="citation_online_date"/>
<meta content="1" name="citation_firstpage"/>
<meta content="18" name="citation_lastpage"/>
<meta content="Article" name="citation_article_type"/>
<meta content="" name="citation_fulltext_world_readable"/>
<meta content="en" name="citation_language"/>
<meta content="doi:10.1038/s41598-024-84718-0" name="dc.identifier"/>
<meta content="10.1038/s41598-024-84718-0" name="DOI"/>
<meta content="280391" name="size"/>
<meta content="10.1038/s41598-024-84718-0" name="citation_doi"/>
<meta content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41598-024-84718-0&amp;api_key=" name="citation_springer_api_url"/>
<meta content="Plastic poses a significant environmental impact due to its chemical resilience, leading to prolonged and degradation times and resulting in widespread adverse effects on global flora and fauna. Cutinases are essential enzymes in the biodegradation process of synthetic polymers like polyethylene terephthalate (PET), which recognized organisms can break down. Here, we used molecular dynamics and binding free energy calculations to explore the interaction of nine synthetic polymers, including PET, with Cutinase from Fusarium oxysporum (FoCut). According to our findings, the polymers poly(ethylene terephthalate) (PET),&amp;nbsp;poly(3-hydroxybutyrate-co-3-hydroxyhexanoate) (PHBH), poly(butylene succinate) (PBS), poly(butylene adipate-co-terephthalate) (PBAT) and poly(ε-caprolactone) (PCL)&amp;nbsp;can bind to the Cutinase enzyme from&amp;nbsp;F. oxysporum, indicating potential biodegradation activity for these polymers. PET exhibited the highest binding affinity (−&amp;nbsp;34.26&amp;nbsp;kcal/mol). Besides PET, the polymers PHBH, PBS, PBAT, and PCL also demonstrated significant affinities for the FoCut enzyme, with binding values of −&amp;nbsp;18.44, −&amp;nbsp;29.71, −&amp;nbsp;22.78, and −&amp;nbsp;22.26&amp;nbsp;kcal/mol, respectively. Additionally, analysis of the phylogenetic tree of cutinases produced by different organisms demonstrated that even though the organisms belong to different kingdoms, the cutinase from F. oxysporum&amp;nbsp;(FoCut) showed biological similarity in its activity in degrading polymers with the cutinase enzyme from the bacterium Kineococcus radiotolerans&amp;nbsp;and the fungus&amp;nbsp;Moniliophthora roreri. Furthermore, the phylogenetic analysis demonstrated that the PETase enzyme has a very high similarity with the bacterial cutinase enzyme than with the fungal cutinase, therefore demonstrating that the PETase enzyme from Ideonella sakaiensis can easily be a modified bacterial cutinase enzyme that created a unique feature in biodegrading only the pet polymer through an evolutionary process due to its environment and its biochemical need for carbon. Our data demonstrate that bacterial cutinase enzymes have the same common ancestor as the PETase enzyme. Therefore, cutinases and PETase are interconnected through their biological similarity in biodegrading polymers. We demonstrated that important conserved regions, such as the Ser-Asp-His catalytic triad, exist in the enzyme’s catalytic site and that all Cut enzymes from different organisms have the same region to couple with the polymer structures." name="description"/>
<meta content="de Oliveira, Maycon Vinicius Damasceno" name="dc.creator"/>
<meta content="Calandrini, Gabriel" name="dc.creator"/>
<meta content="da Costa, Clauber Henrique Souza" name="dc.creator"/>
<meta content="da Silva de Souza, Carlos Gabriel" name="dc.creator"/>
<meta content="Alves, Cláudio Nahum" name="dc.creator"/>
<meta content="Silva, José Rogério A." name="dc.creator"/>
<meta content="Lima, Anderson H." name="dc.creator"/>
<meta content="Lameira, Jerônimo" name="dc.creator"/>
<meta content="Biochemistry" name="dc.subject"/>
<meta content="Biophysics" name="dc.subject"/>
<meta content="Biotechnology" name="dc.subject"/>
<meta content="Chemistry" name="dc.subject"/>
<meta content="Computational biology and bioinformatics" name="dc.subject"/>
<meta content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Characterization and engineering of a plastic-degrading aromatic polyesterase; citation_author=HP Austin; citation_volume=115; citation_publication_date=2018; citation_pages=E4350-E4357; citation_doi=10.1073/pnas.1718804115; citation_id=CR1" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Structural insight into catalytic mechanism of PET hydrolase; citation_author=X Han; citation_volume=8; citation_publication_date=2017; citation_pages=2106; citation_doi=10.1038/s41467-017-02255-z; citation_id=CR2" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Structural insight into molecular mechanism of poly(ethylene terephthalate) degradation; citation_author=S Joo; citation_volume=9; citation_publication_date=2018; citation_pages=382; citation_doi=10.1038/s41467-018-02881-1; citation_id=CR3" name="citation_reference"/>
<meta content="citation_journal_title=Sci. Prog.; citation_title=Plastic pollution and potential solutions; citation_author=CJ Rhodes; citation_volume=101; citation_publication_date=2018; citation_pages=207-260; citation_doi=10.3184/003685018X15294876706211; citation_id=CR4" name="citation_reference"/>
<meta content="citation_journal_title=Sci. Total Environ.; citation_title=Smaller-sized micro-plastics (MPs) contamination in single-use PET-bottled water in Thailand; citation_author=D Kankanige, S Babel; citation_volume=717; citation_publication_date=2020; citation_doi=10.1016/j.scitotenv.2020.137232; citation_id=CR5" name="citation_reference"/>
<meta content="citation_journal_title=Npj Mater. Degrad.; citation_title=Recent advances in biodegradable polymers for sustainable applications; citation_author=A Samir, FH Ashour, AAA Hakim, M Bassyouni; citation_volume=6; citation_publication_date=2022; citation_pages=68; citation_doi=10.1038/s41529-022-00277-7; citation_id=CR6" name="citation_reference"/>
<meta content="citation_journal_title=Conserv. Lett.; citation_title=Plastic pollution is killing marine megafauna, but how do we prioritize policies to reduce mortality?; citation_author=L Roman, Q Schuyler, C Wilcox, BD Hardesty; citation_volume=14; citation_publication_date=2021; citation_pages=12871; citation_doi=10.1111/conl.12781; citation_id=CR7" name="citation_reference"/>
<meta content="citation_journal_title=SN Appl. Sci.; citation_title=Microbial degradation of polyethylene terephthalate: A systematic review; citation_author=CD Benavides Fernández, MP Guzmán Castillo, SA Quijano Pérez, LV Carvajal Rodríguez; citation_volume=4; citation_publication_date=2022; citation_pages=263; citation_doi=10.1007/s42452-022-05143-4; citation_id=CR8" name="citation_reference"/>
<meta content="citation_journal_title=Polym. Bull.; citation_title=Polymer pollution and its solutions with special emphasis on Poly (butylene adipate terephthalate (PBAT)); citation_author=A Kanwal, M Zhang, F Sharaf, C Li; citation_volume=79; citation_publication_date=2022; citation_pages=9303-9330; citation_doi=10.1007/s00289-021-04065-2; citation_id=CR9" name="citation_reference"/>
<meta content="citation_journal_title=Biochemistry; citation_title=Biochemical Characterization and NMR Study of a PET-hydrolyzing cutinase from Fusarium solani pisi; citation_author=KN Hellesnes, S Vijayaraj, P Fojan, E Petersen, G Courtade; citation_volume=62; citation_publication_date=2023; citation_pages=1369-1375; citation_doi=10.1021/acs.biochem.2c00619; citation_id=CR10" name="citation_reference"/>
<meta content="citation_journal_title=Biochim. Biophys. Acta BBA General Subjects; citation_title=Structural and functional studies of a Fusarium oxysporum cutinase with polyethylene terephthalate modification potential; citation_author=M Dimarogona; citation_volume=1850; citation_publication_date=2015; citation_pages=2308-2317; citation_doi=10.1016/j.bbagen.2015.08.009; citation_id=CR11" name="citation_reference"/>
<meta content="citation_journal_title=Microbiol. Spectr.; citation_title=Expression of a cutinase of Moniliophthora roreri with polyester and PET-plastic residues degradation activity; citation_author=L Vázquez-Alcántara, RM Oliart-Ros, A García-Bórquez, C Peña-Montes; citation_volume=9; citation_issue=3; citation_publication_date=2021; citation_pages=e00976-e1021; citation_doi=10.1128/Spectrum.00976-21; citation_id=CR12" name="citation_reference"/>
<meta content="citation_journal_title=Biochemistry; citation_title=Structural dynamics of the PET-degrading cutinase-like enzyme from Saccharomonospora viridis AHK190 in substrate-bound states elucidates the Ca2+—driven catalytic cycle; citation_author=N Numoto; citation_volume=57; citation_publication_date=2018; citation_pages=5289-5300; citation_doi=10.1021/acs.biochem.8b00624; citation_id=CR13" name="citation_reference"/>
<meta content="citation_journal_title=Biocatal Biotransform.; citation_title=Characterization of a new cutinase from Thermobifida alba for PET-surface hydrolysis; citation_author=D Ribitsch; citation_volume=30; citation_publication_date=2012; citation_pages=2-9; citation_doi=10.3109/10242422.2012.644435; citation_id=CR14" name="citation_reference"/>
<meta content="citation_journal_title=Green Chem.; citation_title=Depolymerization of post-consumer PET bottles with engineered cutinase 1 from Thermobifida cellulosilytica; citation_author=Z Zhang; citation_volume=24; citation_publication_date=2022; citation_pages=5998-6007; citation_doi=10.1039/D2GC01834A; citation_id=CR15" name="citation_reference"/>
<meta content="citation_journal_title=Sci. Rep.; citation_title=Efficient degradation of poly(ethylene terephthalate) with thermobifida fusca cutinase exhibiting improved catalytic activity generated using mutagenesis and additive-based approaches; citation_author=M Furukawa, N Kawakami, A Tomizawa, K Miyamoto; citation_volume=9; citation_publication_date=2019; citation_pages=16038; citation_doi=10.1038/s41598-019-52379-z; citation_id=CR16" name="citation_reference"/>
<meta content="Aristizábal-Lanza, L., Mankar, S. V., Tullberg, C., Zhang, B. &amp; Linares-Pastén, J. A. Comparison of the enzymatic depolymerization of polyethylene terephthalate and AkestraTM using Humicola insolens cutinase. Front. Chem. Eng. 4, 1048744  (2022)." name="citation_reference"/>
<meta content="Abokitse, K. et al. A novel actinobacterial cutinase containing a noncatalytic polymer-binding domain. Appl. Environ. Microbiol. 88,  e0152221. 
                  https://doi.org/10.1128/AEM.01522-21
                  
                 (2022)." name="citation_reference"/>
<meta content="citation_journal_title=FEBS J.; citation_title=Structural insights into the unique polylactate-degrading mechanism of Thermobifida alba cutinase; citation_author=K Kitadokoro; citation_volume=286; citation_publication_date=2019; citation_pages=2087-2098; citation_doi=10.1111/febs.14781; citation_id=CR19" name="citation_reference"/>
<meta content="Gamerith, C. et al. Enzymatic degradation of aromatic and aliphatic polyesters by P. pastoris expressed cutinase 1 from Thermobifida cellulosilytica. Front. Microbiol. 8, (2017)." name="citation_reference"/>
<meta content="citation_journal_title=Polym. Degrad. Stab.; citation_title=Enzymatic degradation of poly(butylene succinate) by cutinase cloned from Fusarium solani; citation_author=X Hu; citation_volume=134; citation_publication_date=2016; citation_pages=211-219; citation_doi=10.1016/j.polymdegradstab.2016.10.012; citation_id=CR21" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Complete bio-degradation of poly(butylene adipate-co-terephthalate) via engineered cutinases; citation_author=Y Yang; citation_volume=14; citation_publication_date=2023; citation_pages=1645; citation_doi=10.1038/s41467-023-37374-3; citation_id=CR22" name="citation_reference"/>
<meta content="citation_journal_title=Polym. Degrad. Stab.; citation_title=Enzymatic hydrolysis of poly(butylene adipate-co-terephthalate) by Fusarium solani cutinase; citation_author=W Lin, Y Zhao, T Su, Z Wang; citation_volume=211; citation_publication_date=2023; citation_doi=10.1016/j.polymdegradstab.2023.110335; citation_id=CR23" name="citation_reference"/>
<meta content="Abdel-Azeem, A. M., Abdel-Azeem, M. A., Darwish, A. G., Nafady, N. A. &amp; Ibrahim, N. A. Fusarium: Biodiversity, ecological significances, and industrial applications. 201–261 (Springer Nature, 2019)." name="citation_reference"/>
<meta content="citation_journal_title=Water Res.; citation_title=Adaptation of Fusarium oxysporum and Fusarium dimerum to the specific aquatic environment provided by the water systems of hospitals; citation_author=C Steinberg; citation_volume=76; citation_publication_date=2015; citation_pages=53-65; citation_doi=10.1016/j.watres.2015.02.036; citation_id=CR25" name="citation_reference"/>
<meta content="citation_journal_title=Brazil. Int. J. Mol. Sci.; citation_title=Secondary metabolic profile as a tool for distinction and characterization of cultivars of black pepper (Piper nigrum L.) cultivated in Pará State; citation_author=LM Barata; citation_volume=22; citation_publication_date=2021; citation_pages=890; citation_doi=10.3390/ijms22020890; citation_id=CR26" name="citation_reference"/>
<meta content="citation_journal_title=Rev. De Agric. Neotrop.; citation_title=Quality Of black pepper produced in northeastern Pará; citation_author=NE Paracampo, LF Abreu, OF Lemos, JP Both; citation_volume=9; citation_issue=3; citation_publication_date=2022; citation_pages=e7020; citation_doi=10.32404/rean.v9i3.7020; citation_id=CR27" name="citation_reference"/>
<meta content="Bubici, G., Kaushal, M., Prigigallo, M. I., Gómez-Lama Cabanás, C. &amp; Mercado-Blanco, J. Biological control agents against fusarium wilt of Banana. Front. Microbiol. 10, (2019)." name="citation_reference"/>
<meta content="citation_journal_title=Crop Protection; citation_title=Management of tomato diseases caused by Fusarium oxysporum; citation_author=RJ McGovern; citation_volume=73; citation_publication_date=2015; citation_pages=78-92; citation_doi=10.1016/j.cropro.2015.02.021; citation_id=CR29" name="citation_reference"/>
<meta content="citation_journal_title=RSC Advances.; citation_title=Virtual screening of natural products against 5-enolpyruvylshikimate-3-phosphate synthase using the Anagreen herbicide-like natural compound library; citation_author=MV Oliveira, GM Fernandes, KS Costa, S Vakal, AH Lima; citation_volume=12; citation_issue=29; citation_publication_date=2022; citation_pages=18834-18847; citation_doi=10.1039/D2RA02645G; citation_id=CR30" name="citation_reference"/>
<meta content="De Oliveira, M. D., Araújo, J. de O., Galúcio, J. M. P., Santana, K. &amp; Lima, A. H. Targeting shikimate pathway: In silico analysis of phosphoenolpyruvate derivatives as inhibitors of EPSP synthase and DAHP synthase. J. Mol. Graph Model 101, 107735 (2020)." name="citation_reference"/>
<meta content="De Souza, E. M. de C. et al. Molecular characterization and in silico evaluation of surfactins produced by endophytic bacteria from Phanera splendens. Front. Chem. 11, 1240704 (2023)." name="citation_reference"/>
<meta content="citation_journal_title=J. Braz. Chem. Soc.; citation_title=A combined approach for predicting binding affinity of Zika Virus NS2B-NS3 protease inhibitors using semi-empirical methods and molecular docking algorithms; citation_author=MV Oliveira, L Almeida, J Rocha, AL Wanzeller, A Lima; citation_publication_date=2024; citation_doi=10.21577/0103-5053.20230130; citation_id=CR33" name="citation_reference"/>
<meta content="citation_journal_title=Proteins Struct. Funct. Bioinform.; citation_title=Assessment of the PETase conformational changes induced by poly(ethylene terephthalate) binding; citation_author=CHS Costa; citation_volume=89; citation_publication_date=2021; citation_pages=1340-1352; citation_doi=10.1002/prot.26155; citation_id=CR34" name="citation_reference"/>
<meta content="citation_journal_title=Sci. Rep.; citation_title=Structural, energetic and lipophilic analysis of SARS-CoV-2 non-structural protein 9 (NSP9); citation_author=OJ Araújo; citation_volume=11; citation_publication_date=2021; citation_pages=23003; citation_doi=10.1038/s41598-021-02366-0; citation_id=CR35" name="citation_reference"/>
<meta content="citation_journal_title=Mol. Biol. Evol.; citation_title=MEGA11: Molecular evolutionary genetics analysis version 11; citation_author=K Tamura, G Stecher, S Kumar; citation_volume=38; citation_publication_date=2021; citation_pages=3022-3027; citation_doi=10.1093/molbev/msab120; citation_id=CR36" name="citation_reference"/>
<meta content="citation_journal_title=ChemSusChem.; citation_title=Concentration-dependent inhibition of mesophilic PETases on poly (ethylene terephthalate) can be eliminated by enzyme engineering; citation_author=L Avilan, BR Lichtenstein, G König, M Zahn, MD Allen, L Oliveira, M Clark, V Bemmer, R Graham, HP Austin, G Dominick; citation_volume=16; citation_issue=8; citation_publication_date=2023; citation_pages=e202202277; citation_doi=10.1002/cssc.202202277; citation_id=CR37" name="citation_reference"/>
<meta content="citation_journal_title=J. Mol. Biol.; citation_title=Atomic resolution (1.0 Å) crystal structure of Fusarium solani cutinase: stereochemical analysis; citation_author=S Longhi, M Czjzek, V Lamzin, A Nicolas, C Cambillau; citation_volume=268; citation_publication_date=1997; citation_pages=779-799; citation_doi=10.1006/jmbi.1997.1000; citation_id=CR38" name="citation_reference"/>
<meta content="citation_journal_title=Nature; citation_title=Fusarium solani cutinase is a lipolytic enzyme with a catalytic serine accessible to solvent; citation_author=C Martinez, P Geus, M Lauwereys, G Matthyssens, C Cambillau; citation_volume=356; citation_publication_date=1992; citation_pages=615-618; citation_doi=10.1038/356615a0; citation_id=CR39" name="citation_reference"/>
<meta content="citation_journal_title=Protein Science; citation_title=Thermodynamic and structural investigation of the specific SDS binding of humicola insolens cutinase; citation_author=D Kold; citation_volume=23; citation_publication_date=2014; citation_pages=1023-1035; citation_doi=10.1002/pro.2489; citation_id=CR40" name="citation_reference"/>
<meta content="citation_journal_title=J. Mol. Biol.; citation_title=Catalysis by glomerella cingulata cutinase requires conformational cycling between the active and inactive states of its catalytic triad; citation_author=MP Nyon; citation_volume=385; citation_publication_date=2009; citation_pages=226-235; citation_doi=10.1016/j.jmb.2008.10.050; citation_id=CR41" name="citation_reference"/>
<meta content="citation_journal_title=J. Am. Chem. Soc.; citation_title=Structural and functional studies of Aspergillus oryzae cutinase: Enhanced thermostability and hydrolytic activity of synthetic ester and polyester degradation; citation_author=Z Liu; citation_volume=131; citation_publication_date=2009; citation_pages=15711-15716; citation_doi=10.1021/ja9046697; citation_id=CR42" name="citation_reference"/>
<meta content="citation_journal_title=Biotechnol. Biofuels; citation_title=High-level expression and characterization of a novel cutinase from Malbranchea cinnamomea suitable for butyl butyrate production; citation_author=X Duan; citation_volume=10; citation_publication_date=2017; citation_pages=223; citation_doi=10.1186/s13068-017-0912-z; citation_id=CR43" name="citation_reference"/>
<meta content="citation_journal_title=J Mol. Biol.; citation_title=A cutinase from trichoderma reesei with a lid-covered active site and kinetic properties of true lipases; citation_author=A Roussel; citation_volume=426; citation_publication_date=2014; citation_pages=3757-3772; citation_doi=10.1016/j.jmb.2014.09.003; citation_id=CR44" name="citation_reference"/>
<meta content="citation_journal_title=Appl. Microbiol. Biotechnol.; citation_title=Structural basis for the Ca2+-enhanced thermostability and activity of PET-degrading cutinase-like enzyme from Saccharomonospora viridis AHK190; citation_author=T Miyakawa; citation_volume=99; citation_publication_date=2015; citation_pages=4297-4307; citation_doi=10.1007/s00253-014-6272-8; citation_id=CR45" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Sourcing thermotolerant poly(ethylene terephthalate) hydrolase scaffolds from natural diversity; citation_author=E Erickson; citation_volume=13; citation_publication_date=2022; citation_pages=7850; citation_doi=10.1038/s41467-022-35237-x; citation_id=CR46" name="citation_reference"/>
<meta content="citation_journal_title=Biotechnol. Bioeng.; citation_title=Small cause, large effect: Structural characterization of cutinases from Thermobifida cellulosilytica; citation_author=D Ribitsch; citation_volume=114; citation_publication_date=2017; citation_pages=2481-2488; citation_doi=10.1002/bit.26372; citation_id=CR47" name="citation_reference"/>
<meta content="citation_journal_title=Polym. Degrad. Stab.; citation_title=Crystal structure of cutinase Est119 from Thermobifida alba AHK119 that can degrade modified polyethylene terephthalate at 1.76Å resolution; citation_author=K Kitadokoro; citation_volume=97; citation_publication_date=2012; citation_pages=771-775; citation_doi=10.1016/j.polymdegradstab.2012.02.003; citation_id=CR48" name="citation_reference"/>
<meta content="citation_journal_title=Nucl. Acids Res.; citation_title=The protein data bank; citation_author=HM Berman; citation_publication_date=2000; citation_doi=10.1093/nar/28.1.235; citation_id=CR49" name="citation_reference"/>
<meta content="ChemAxon. ChemAxon—Software Solutions and Services for Chemistry and Biology. MarvinSketch, Version 16.10.31 Preprint at (2016)." name="citation_reference"/>
<meta content="citation_journal_title=J. Cheminform.; citation_title=Avogadro: An advanced semantic chemical editor, visualization, and analysis platform; citation_author=MD Hanwell; citation_publication_date=2012; citation_doi=10.1186/1758-2946-4-17; citation_id=CR51" name="citation_reference"/>
<meta content="Frisch, M. J. ; et. al, Nathan, A. J. &amp; Scobell, A. Gaussian 09. Gaussian, Inc. Wallingford CT 2–3 Preprint at 111 (2009)." name="citation_reference"/>
<meta content="citation_journal_title=Rev. Mod. Phys.; citation_title=New developments in molecular orbital theory; citation_author=CCJ Roothaan; citation_volume=23; citation_publication_date=1951; citation_pages=69-89; citation_doi=10.1103/RevModPhys.23.69; citation_id=CR53" name="citation_reference"/>
<meta content="citation_journal_title=Sci. Rep.; citation_title=Exploring the infiltrative and degradative ability of Fusarium oxysporum on polyethylene terephthalate (PET) using correlative microscopy and deep learning; citation_author=F Cognigni; citation_volume=13; citation_publication_date=2023; citation_pages=22987; citation_doi=10.1038/s41598-023-50199-w; citation_id=CR54" name="citation_reference"/>
<meta content="citation_journal_title=Microorganisms; citation_title=Screening enzymes that can depolymerize commercial biodegradable polymers: Heterologous expression of fusarium solani cutinase in Escherichia coli; citation_author=F Santos-Beneit; citation_volume=11; citation_publication_date=2023; citation_pages=328; citation_doi=10.3390/microorganisms11020328; citation_id=CR55" name="citation_reference"/>
<meta content="citation_journal_title=SynBio; citation_title=Biotechnological application of cutinase: A powerful tool in synthetic biology; citation_author=X Liang, H Zou; citation_volume=1; citation_publication_date=2022; citation_pages=54-64; citation_doi=10.3390/synbio1010004; citation_id=CR56" name="citation_reference"/>
<meta content="citation_journal_title=Biomacromolecules; citation_title=Thermal embedding of Humicola insolens cutinase: A strategy for improving polyester biodegradation in seawater; citation_author=Q Huang, S Kimura, T Iwata; citation_volume=24; citation_publication_date=2023; citation_pages=5836-5846; citation_doi=10.1021/acs.biomac.3c00835; citation_id=CR57" name="citation_reference"/>
<meta content="citation_journal_title=Polym. J.; citation_title=Biodegradability of poly(3-hydroxyalkanoate) and poly(ε-caprolactone) via biological carbon cycles in marine environments; citation_author=M Suzuki, Y Tachibana, K Kasuya; citation_volume=53; citation_publication_date=2021; citation_pages=47-66; citation_doi=10.1038/s41428-020-00396-5; citation_id=CR58" name="citation_reference"/>
<meta content="citation_journal_title=Appl. Environ. Microbiol.; citation_title=Fusarium polycaprolactone depolymerase is cutinase; citation_author=CA Murphy, JA Cameron, SJ Huang, RT Vinopal; citation_volume=62; citation_publication_date=1996; citation_pages=456-460; citation_doi=10.1128/aem.62.2.456-460.1996; citation_id=CR59" name="citation_reference"/>
<meta content="citation_journal_title=J. Med. Chem.; citation_title=MolDock: A new technique for high-accuracy molecular docking; citation_author=R Thomsen, MH Christensen; citation_volume=49; citation_publication_date=2006; citation_pages=3315-3321; citation_doi=10.1021/jm051197e; citation_id=CR60" name="citation_reference"/>
<meta content="citation_journal_title=J. Mol. Biol.; citation_title=Development and validation of a genetic algorithm for flexible docking 1 1Edited by F E. Cohen; citation_author=G Jones, P Willett, RC Glen, AR Leach, R Taylor; citation_volume=267; citation_publication_date=1997; citation_pages=727-748; citation_doi=10.1006/jmbi.1996.0897; citation_id=CR61" name="citation_reference"/>
<meta content="citation_journal_title=Protein Sci.; citation_title=Improvements to the &lt;scp&gt;APBS&lt;/scp&gt; biomolecular solvation software suite; citation_author=E Jurrus; citation_volume=27; citation_publication_date=2018; citation_pages=112-128; citation_doi=10.1002/pro.3280; citation_id=CR62" name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Chem.; citation_title=CHARMM-GUI: A web-based graphical user interface for CHARMM; citation_author=S Jo, T Kim, VG Iyer, W Im; citation_volume=29; citation_publication_date=2008; citation_pages=1859-1865; citation_doi=10.1002/jcc.20945; citation_id=CR63" name="citation_reference"/>
<meta content="citation_journal_title=J. Comput. Chem.; citation_title=CHARMM: The biomolecular simulation program; citation_author=BR Brooks; citation_volume=30; citation_publication_date=2009; citation_pages=1545-1614; citation_doi=10.1002/jcc.21287; citation_id=CR64" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Theory Comput.; citation_title=CHARMM-GUI input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field; citation_author=J Lee; citation_volume=12; citation_publication_date=2016; citation_pages=405-413; citation_doi=10.1021/acs.jctc.5b00935; citation_id=CR65" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Phys.; citation_title=Comparison of simple potential functions for simulating liquid water; citation_author=WL Jorgensen, J Chandrasekhar, JD Madura, RW Impey, ML Klein; citation_publication_date=1983; citation_doi=10.1063/1.445869; citation_id=CR66" name="citation_reference"/>
<meta content="Case, D. A. et al. Amber 20. (2020)." name="citation_reference"/>
<meta content="citation_journal_title=J. Am. Chem. Soc.; citation_title=Continuum solvent studies of the stability of DNA, RNA, and phosphoramidate−DNA helices; citation_author=J Srinivasan, TE Cheatham, P Cieplak, PA Kollman, DA Case; citation_volume=120; citation_publication_date=1998; citation_pages=9401-9409; citation_doi=10.1021/ja981844+; citation_id=CR68" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Theory Comput.; citation_title=MMPBSA. py: An efficient program for end-state free energy calculations; citation_author=BR Miller, TD McGee, JM Swails, N Homeyer, H Gohlke, AE Roitberg; citation_volume=8; citation_issue=9; citation_publication_date=2012; citation_pages=3314-3321; citation_doi=10.1021/ct300418h; citation_id=CR69" name="citation_reference"/>
<meta content="citation_journal_title=Expert Opin. Drug Discov.; citation_title=The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities; citation_author=S Genheden, U Ryde; citation_volume=10; citation_publication_date=2015; citation_pages=449-461; citation_doi=10.1517/17460441.2015.1032936; citation_id=CR70" name="citation_reference"/>
<meta content="citation_journal_title=Bioinformatics; citation_title=RADER: A RApid DEcoy retriever to facilitate decoy based assessment of virtual screening; citation_author=L Wang, X Pang, Y Li, Z Zhang, W Tan; citation_volume=33; citation_publication_date=2017; citation_pages=1235-1237; citation_doi=10.1093/bioinformatics/btw783; citation_id=CR71" name="citation_reference"/>
<meta content="citation_journal_title=Chem. Rev.; citation_title=End-point binding free energy calculation with MM/PBSA and MM/GBSA: Strategies and applications in drug design; citation_author=E Wang; citation_volume=119; citation_publication_date=2019; citation_pages=9478-9508; citation_doi=10.1021/acs.chemrev.9b00055; citation_id=CR72" name="citation_reference"/>
<meta content="citation_journal_title=J Oleo Sci; citation_title=Biodegradable plastic-degrading activity of various species of &lt;i&gt;P&lt;/i&gt;&lt;i&gt;araphoma&lt;/i&gt;; citation_author=M Koitabashi; citation_volume=65; citation_publication_date=2016; citation_pages=621-627; citation_doi=10.5650/jos.ess16067; citation_id=CR73" name="citation_reference"/>
<meta content="citation_journal_title=Heliyon; citation_title=Fungal biodegradation of low-density polyethylene using consortium of Aspergillus species under controlled conditions; citation_author=GC Dsouza; citation_volume=7; citation_publication_date=2021; citation_pages=e7008; citation_doi=10.1016/j.heliyon.2021.e07008; citation_id=CR74" name="citation_reference"/>
<meta content="citation_journal_title=RSC Adv.; citation_title=Recent advances in microbial and enzymatic engineering for the biodegradation of micro- and nanoplastics; citation_author=J Choi; citation_volume=14; citation_publication_date=2024; citation_pages=9943-9966; citation_doi=10.1039/D4RA00844H; citation_id=CR75" name="citation_reference"/>
<meta content="citation_journal_title=Sci. Total Environ.; citation_title=Accelerated biodegradation of polyethylene terephthalate by Thermobifida fusca cutinase mediated by Stenotrophomonas pavanii; citation_author=Q-S Huang; citation_volume=808; citation_publication_date=2022; citation_doi=10.1016/j.scitotenv.2021.152107; citation_id=CR76" name="citation_reference"/>
<meta content="Scattolini Rimada, A. C., Hernández Rodríguez, L. I. &amp; González Idiarte, H. Identification of Fusarium oxysporum causing sweet potato rot (Ipomoea batatas L &amp; Lamb) in Uruguay. Agrociencia 24,  e124 (2020)." name="citation_reference"/>
<meta content="Rodríguez-Grimaldo, J. E., M. González, G. &amp; M. Montoya, A. Fusarium: Un fitopatógeno Que Amenaza La Salud Humana. Revista Ciencia UANL 25, 37–43 (2023)." name="citation_reference"/>
<meta content="Cighir, A. et al. Fusarium spp. in Human Disease: Exploring the Boundaries between Commensalism and Pathogenesis. Life 13, 1440 (2023)." name="citation_reference"/>
<meta content="Anderson, K. S. Detection of novel enzyme intermediates in PEP-utilizing enzymes. Arch. Biochem. Biophys. 433 47–58 (2005)." name="citation_reference"/>
<meta content="citation_journal_title=Science; citation_title=A bacterium that degrades and assimilates poly(ethylene terephthalate); citation_author=S Yoshida; citation_volume=1979; citation_issue=351; citation_publication_date=2016; citation_pages=1196-1199; citation_doi=10.1126/science.aad6359; citation_id=CR81" name="citation_reference"/>
<meta content="citation_journal_title=Appl. Biochem. Biotechnol.; citation_title=Emerging roles of PETase and MHETase in the biodegradation of plastic wastes; citation_author=W Maity, S Maity, S Bera, A Roy; citation_volume=193; citation_publication_date=2021; citation_pages=2699-2716; citation_doi=10.1007/s12010-021-03562-4; citation_id=CR82" name="citation_reference"/>
<meta content="citation_journal_title=J. Phys. Chem. B; citation_title=Exploring the reaction mechanism of polyethylene terephthalate biodegradation through QM/MM approach; citation_author=AM Santos, CHS Costa, PHA Silva, MS Skaf, J Lameira; citation_volume=128; citation_publication_date=2024; citation_pages=7486-7499; citation_doi=10.1021/acs.jpcb.4c02207; citation_id=CR83" name="citation_reference"/>
<meta content="citation_journal_title=Polymers; citation_title=Degradation of PET bottles by an engineered Ideonella sakaiensis PETase; citation_author=ME Sevilla; citation_volume=15; citation_publication_date=2023; citation_pages=1779; citation_doi=10.3390/polym15071779; citation_id=CR84" name="citation_reference"/>
<meta content="citation_journal_title=ChemBioChem; citation_title=Protein crystallography and site-direct mutagenesis analysis of the poly(Ethylene terephthalate) hydrolase petase from Ideonella sakaiensis; citation_author=B Liu; citation_volume=19; citation_publication_date=2018; citation_pages=1471-1475; citation_doi=10.1002/cbic.201800097; citation_id=CR85" name="citation_reference"/>
<meta content="citation_journal_title=Food Biosci.; citation_title=Contribution of disulfide bond to the stability of Thermobifida fusca cutinase; citation_author=S Chen, Y Wu, L Su, J Wu; citation_volume=27; citation_publication_date=2019; citation_pages=6-10; citation_doi=10.1016/j.fbio.2018.11.001; citation_id=CR86" name="citation_reference"/>
<meta content="citation_journal_title=Int. J. Syst. Evol. Microbiol.; citation_title=Ideonella sakaiensis sp. nov, isolated from a microbial consortium that degrades poly(ethylene terephthalate); citation_author=S Tanasupawat, T Takehana, S Yoshida, K Hiraga, K Oda; citation_volume=66; citation_publication_date=2016; citation_pages=2813-2818; citation_doi=10.1099/ijsem.0.001058; citation_id=CR87" name="citation_reference"/>
<meta content="citation_journal_title=Nat. Commun.; citation_title=Structure of the plastic-degrading Ideonella sakaiensis MHETase bound to a substrate; citation_author=GJ Palm; citation_volume=10; citation_publication_date=2019; citation_pages=1717; citation_doi=10.1038/s41467-019-09326-3; citation_id=CR88" name="citation_reference"/>
<meta content="Longhi, S. &amp; Cambillau, C. Structure-activity of cutinase, a small lipolytic enzyme. Biochim. Biophys. Acta (BBA)Mol. Cell Biol. Lipids 1441, 185–196 (1999)." name="citation_reference"/>
<meta content="citation_journal_title=Biochem. Biophys. Res. Commun.; citation_title=The role of short-range Cys171–Cys178 disulfide bond in maintaining cutinase active site integrity: A molecular dynamics simulation; citation_author=MY Matak, ME Moghaddam; citation_volume=390; citation_publication_date=2009; citation_pages=201-204; citation_doi=10.1016/j.bbrc.2009.09.073; citation_id=CR90" name="citation_reference"/>
<meta content="citation_journal_title=Nucl. Acids Res.; citation_title=ProteinsPlus: A web portal for structure analysis of macromolecules; citation_author=R Fährrolfes; citation_volume=45; citation_publication_date=2017; citation_pages=W337-W343; citation_doi=10.1093/nar/gkx333; citation_id=CR91" name="citation_reference"/>
<meta content="citation_journal_title=Nucl. Acids Res.; citation_title=ProteinsPlus: Interactive analysis of protein–ligand binding interfaces; citation_author=K Schöning-Stierand; citation_volume=48; citation_publication_date=2020; citation_pages=W48-W53; citation_doi=10.1093/nar/gkaa235; citation_id=CR92" name="citation_reference"/>
<meta content="citation_journal_title=J. Chem. Inf. Model.; citation_title=Binding site detection remastered: Enabling fast, robust, and reliable binding site detection and descriptor calculation with DoGSite3; citation_author=J Graef, C Ehrt, M Rarey; citation_volume=63; citation_publication_date=2023; citation_pages=3128-3137; citation_doi=10.1021/acs.jcim.3c00336; citation_id=CR93" name="citation_reference"/>
<meta content="citation_journal_title=Bioinformatics; citation_title=GeoMine: Interactive pattern mining of protein–ligand interfaces in the Protein Data Bank; citation_author=K Diedrich, J Graef, K Schöning-Stierand, M Rarey; citation_volume=37; citation_publication_date=2021; citation_pages=424-425; citation_doi=10.1093/bioinformatics/btaa693; citation_id=CR94" name="citation_reference"/>
<meta content="citation_journal_title=Int. J. Mol. Sci.; citation_title=Perspectives on the role of enzymatic biocatalysis for the degradation of plastic PET; citation_author=RP Magalhães, JM Cunha, SF Sousa; citation_volume=22; citation_publication_date=2021; citation_pages=11257; citation_doi=10.3390/ijms222011257; citation_id=CR95" name="citation_reference"/>
<meta content="de Oliveira, Maycon Vinicius Damasceno" name="citation_author"/>
<meta content="Laboratório de Planejamento e Desenvolvimento de Fármacos, Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará, Belém, Brazil" name="citation_author_institution"/>
<meta content="Calandrini, Gabriel" name="citation_author"/>
<meta content="Núcleo de Ecologia Aquática e Pesca (NEAP), Universidade Federal do Pará, Belém, Brazil" name="citation_author_institution"/>
<meta content="da Costa, Clauber Henrique Souza" name="citation_author"/>
<meta content="Center for Computing in Engineering &amp; Sciences, IQ/UNICAMP, Campinas, Brazil" name="citation_author_institution"/>
<meta content="da Silva de Souza, Carlos Gabriel" name="citation_author"/>
<meta content="Laboratório de Planejamento e Desenvolvimento de Fármacos, Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará, Belém, Brazil" name="citation_author_institution"/>
<meta content="Alves, Cláudio Nahum" name="citation_author"/>
<meta content="Laboratório de Planejamento e Desenvolvimento de Fármacos, Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará, Belém, Brazil" name="citation_author_institution"/>
<meta content="Silva, José Rogério A." name="citation_author"/>
<meta content="Laboratório de Planejamento e Desenvolvimento de Fármacos, Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará, Belém, Brazil" name="citation_author_institution"/>
<meta content="Catalysis and Peptide Research Unit, University of KwaZulu-Natal, Durban, South Africa" name="citation_author_institution"/>
<meta content="Lima, Anderson H." name="citation_author"/>
<meta content="Laboratório de Planejamento e Desenvolvimento de Fármacos, Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará, Belém, Brazil" name="citation_author_institution"/>
<meta content="Lameira, Jerônimo" name="citation_author"/>
<meta content="Laboratório de Planejamento e Desenvolvimento de Fármacos, Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará, Belém, Brazil" name="citation_author_institution"/>
<meta content="https://www.nature.com/platform/readcube-access" name="access_endpoint"/>
<meta content="@SciReports" name="twitter:site"/>
<meta content="summary_large_image" name="twitter:card"/>
<meta content="Content cover image" name="twitter:image:alt"/>
<meta content="Evaluating cutinase from Fusarium oxysporum as a biocatalyst for the degradation of nine synthetic polymer" name="twitter:title"/>
<meta content="Scientific Reports - Evaluating cutinase from Fusarium oxysporum as a biocatalyst for the degradation of nine synthetic polymer" name="twitter:description"/>
<meta content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig1_HTML.png" name="twitter:image"/>
<meta content="https://www.nature.com/articles/s41598-024-84718-0" property="og:url"/>
<meta content="article" property="og:type"/>
<meta content="Nature" property="og:site_name"/>
<meta content="Evaluating cutinase from Fusarium oxysporum as a biocatalyst for the degradation of nine synthetic polymer - Scientific Reports" property="og:title"/>
<meta content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig1_HTML.png" property="og:image"/>
<script>
        window.eligibleForRa21 = 'false'; 
    </script>
</head>
<body class="article-page">
<noscript><iframe height="0" src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ" style="display:none;visibility:hidden" width="0"></iframe></noscript>
<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
<a class="c-skip-link" href="#content">Skip to main content</a>
<div class="c-grade-c-banner u-hide">
<div class="c-grade-c-banner__container">
<p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>
</div>
</div>
<div class="u-hide u-show-following-ad"></div>
<aside class="c-ad c-ad--728x90">
<div class="c-ad__inner" data-container-type="banner-advert">
<p class="c-ad__label">Advertisement</p>
<div class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-gpt="" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41598-024-84718-0;doi=10.1038/s41598-024-84718-0;subjmeta=114,45,57,61,631,638,639;kwrd=Biochemistry,Biophysics,Biotechnology,Chemistry,Computational+biology+and+bioinformatics" data-gpt-unitpath="/285/scientific_reports/article" data-pa11y-ignore="" data-test="top-ad" id="div-gpt-ad-top-1">
<noscript>
<a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/scientific_reports/article&amp;sz=728x90&amp;c=-2048442541&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41598-024-84718-0%26doi%3D10.1038/s41598-024-84718-0%26subjmeta%3D114,45,57,61,631,638,639%26kwrd%3DBiochemistry,Biophysics,Biotechnology,Chemistry,Computational+biology+and+bioinformatics">
<img alt="Advertisement" data-test="gpt-advert-fallback-img" height="90" src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/scientific_reports/article&amp;sz=728x90&amp;c=-2048442541&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41598-024-84718-0%26doi%3D10.1038/s41598-024-84718-0%26subjmeta%3D114,45,57,61,631,638,639%26kwrd%3DBiochemistry,Biophysics,Biotechnology,Chemistry,Computational+biology+and+bioinformatics" width="728"/></a>
</noscript>
</div>
</div>
</aside>
<header class="c-header" data-header="" data-track-component="nature-150-split-header" id="header" style="border-color:#cedde4">
<div class="c-header__row">
<div class="c-header__container">
<div class="c-header__split">
<div class="c-header__logo-container">
<a data-track="click" data-track-action="home" data-track-label="image" href="/srep">
<picture class="c-header__logo">
<source media="(min-width: 875px)" srcset="https://media.springernature.com/full/nature-cms/uploads/product/srep/header-d3c533c187c710c1bedbd8e293815d5f.svg">
<img alt="Scientific Reports" height="32" src="https://media.springernature.com/full/nature-cms/uploads/product/srep/header-d3c533c187c710c1bedbd8e293815d5f.svg"/>
</source></picture>
</a>
</div>
<ul class="c-header__menu c-header__menu--global">
<li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
<a class="c-header__link" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link" href="https://www.nature.com/siteindex">
<span>View all journals</span>
</a>
</li>
<li class="c-header__item c-header__item--padding c-header__item--pipe">
<a class="c-header__link c-header__link--search" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" href="#search-menu">
<svg aria-hidden="true" focusable="false" height="22" role="img" viewbox="0 0 18 18" width="22" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
</a>
</li>
<li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
<a class="c-header__link eds-c-header__link" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41598-024-84718-0" id="identity-account-widget"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
</li>
</ul>
</div>
</div>
</div>
<div class="c-header__row">
<div class="c-header__container" data-test="navigation-row">
<div class="c-header__split">
<ul class="c-header__menu c-header__menu--journal">
<li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
<a class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" href="#explore">
<span><span class="c-header__show-text">Explore</span> content</span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--dropdown-menu">
<a class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" href="#about-the-journal">
<span>About <span class="c-header__show-text">the journal</span></span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
<a class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" href="#publish-with-us">
<span>Publish <span class="c-header__show-text">with us</span></span><svg aria-hidden="true" focusable="false" height="16" role="img" viewbox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
</a>
</li>
</ul>
<ul class="c-header__menu c-header__menu--hide-lg-max">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (desktop site header)" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D288%26journal-link%3Dhttps%253A%252F%252Fwww.nature.com%252Fsrep%252F" rel="nofollow">
<span>Sign up for alerts</span><svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--pipe">
<a class="c-header__link" data-track="click" data-track-action="rss feed" data-track-label="link" href="https://www.nature.com/srep.rss">
<span>RSS feed</span>
</a>
</li>
</ul>
</div>
</div>
</div>
</header>
<nav aria-label="breadcrumbs" class="u-mb-16">
<div class="u-container">
<ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
<li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature" href="/" itemprop="item"><span itemprop="name">nature</span></a><meta content="1" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:scientific reports" href="/srep" itemprop="item"><span itemprop="name">scientific reports</span></a><meta content="2" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles" href="/srep/articles?type=article" itemprop="item"><span itemprop="name">articles</span></a><meta content="3" itemprop="position"/>
<svg aria-hidden="true" class="c-breadcrumbs__chevron" focusable="false" height="10" role="img" viewbox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
<path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
</svg>
</li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
<span itemprop="name">article</span><meta content="4" itemprop="position"/></li>
</ol>
</div>
</nav>
</div>
<div class="u-container u-mt-32 u-mb-32 u-clearfix" data-component="article-container" data-container-type="article" id="content">
<main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
<div aria-hidden="true" class="c-context-bar u-hide" data-context-bar="" data-test="context-bar">
<div class="c-context-bar__container u-container" data-track-context="sticky banner">
<div class="c-context-bar__title">
                            Evaluating cutinase from<i> Fusarium oxysporum</i> as a biocatalyst for the degradation of nine synthetic polymer
                        </div>
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41598-024-84718-0.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
</div>
<article lang="en">
<div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
<div class="c-pdf-container" data-track-context="article body">
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41598-024-84718-0.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
</div>
<div class="c-article-header">
<header>
<ul class="c-article-identifiers" data-test="article-identifier">
<li class="c-article-identifiers__item" data-test="article-category">Article</li>
<li class="c-article-identifiers__item">
<a class="u-color-open-access" data-test="open-access" data-track="click" data-track-action="open access" data-track-label="link" href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research">Open access</a>
</li>
<li class="c-article-identifiers__item">Published: <time datetime="2025-01-22">22 January 2025</time></li>
</ul>
<h1 class="c-article-title" data-article-title="" data-test="article-title">Evaluating cutinase from<i> Fusarium oxysporum</i> as a biocatalyst for the degradation of nine synthetic polymer</h1>
<ul class="c-article-author-list c-article-author-list--short" data-component-authors-activator="authors-list" data-test="authors-list"><li class="c-article-author-list__item"><a data-author-popup="auth-Maycon_Vinicius_Damasceno-Oliveira-Aff1" data-author-search="de Oliveira, Maycon Vinicius Damasceno" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Maycon_Vinicius_Damasceno-Oliveira-Aff1">Maycon Vinicius Damasceno de Oliveira</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-2263-2124"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-2263-2124</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-author-popup="auth-Gabriel-Calandrini-Aff2" data-author-search="Calandrini, Gabriel" data-corresp-id="c1" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Gabriel-Calandrini-Aff2">Gabriel Calandrini<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-mail-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0001-6321-2228"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6321-2228</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Clauber_Henrique_Souza-Costa-Aff3" data-author-search="da Costa, Clauber Henrique Souza" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Clauber_Henrique_Souza-Costa-Aff3">Clauber Henrique Souza da Costa</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-6915-1056"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-6915-1056</a></span><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Carlos_Gabriel-de_Souza-Aff1" data-author-search="da Silva de Souza, Carlos Gabriel" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Carlos_Gabriel-de_Souza-Aff1">Carlos Gabriel da Silva de Souza</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0001-6572-7196"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6572-7196</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Cl_udio_Nahum-Alves-Aff1" data-author-search="Alves, Cláudio Nahum" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Cl_udio_Nahum-Alves-Aff1">Cláudio Nahum Alves</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Jos__Rog_rio_A_-Silva-Aff1-Aff4" data-author-search="Silva, José Rogério A." data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jos__Rog_rio_A_-Silva-Aff1-Aff4">José Rogério A. Silva</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-2310-5107"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-2310-5107</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-author-popup="auth-Anderson_H_-Lima-Aff1" data-author-search="Lima, Anderson H." data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Anderson_H_-Lima-Aff1">Anderson H. Lima</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-8451-9912"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-8451-9912</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup> &amp; </li><li aria-label="Show all 8 authors for this article" class="c-article-author-list__show-more" title="Show all 8 authors for this article">…</li><li class="c-article-author-list__item"><a data-author-popup="auth-Jer_nimo-Lameira-Aff1" data-author-search="Lameira, Jerônimo" data-corresp-id="c2" data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jer_nimo-Lameira-Aff1">Jerônimo Lameira<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-mail-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0001-7270-1517"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7270-1517</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup> </li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-down-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg><span>Show authors</span></button>
<p class="c-article-info-details" data-container-section="info">
<a data-test="journal-link" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link" href="/srep"><i data-test="journal-title">Scientific Reports</i></a>
<b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 15</b>, Article number: <span data-test="article-number">2887</span> (<span data-test="article-publication-year">2025</span>)
            <a class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link" href="#citeas">Cite this article</a>
</p>
<div class="c-article-metrics-bar__wrapper u-clear-both">
<ul class="c-article-metrics-bar u-list-reset">
<li class="c-article-metrics-bar__item" data-test="altmetric-score">
<p class="c-article-metrics-bar__count">2 <span class="c-article-metrics-bar__label">Altmetric</span></p>
</li>
<li class="c-article-metrics-bar__item">
<p class="c-article-metrics-bar__details"><a data-track="click" data-track-action="view metrics" data-track-label="link" href="/articles/s41598-024-84718-0/metrics" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
</li>
</ul>
</div>
</header>
<div class="u-js-hide" data-component="article-subject-links">
<h3 class="c-article__sub-heading">Subjects</h3>
<ul class="c-article-subject-list">
<li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/biochemistry">Biochemistry</a></li><li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/biophysics">Biophysics</a></li><li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/biotechnology">Biotechnology</a></li><li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/chemistry">Chemistry</a></li><li class="c-article-subject-list__subject"><a data-track="click" data-track-action="view subject" data-track-label="link" href="/subjects/computational-biology-and-bioinformatics">Computational biology and bioinformatics</a></li>
</ul>
</div>
</div>
<div class="c-article-body">
<section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Plastic poses a significant environmental impact due to its chemical resilience, leading to prolonged and degradation times and resulting in widespread adverse effects on global flora and fauna. Cutinases are essential enzymes in the biodegradation process of synthetic polymers like polyethylene terephthalate (PET), which recognized organisms can break down. Here, we used molecular dynamics and binding free energy calculations to explore the interaction of nine synthetic polymers, including PET, with Cutinase from <i>Fusarium oxysporum</i> (<i>Fo</i>Cut). According to our findings, the polymers poly(ethylene terephthalate) (PET), poly(3-hydroxybutyrate-co-3-hydroxyhexanoate) (PHBH), poly(butylene succinate) (PBS), poly(butylene adipate-co-terephthalate) (PBAT) and poly(ε-caprolactone) (PCL) can bind to the Cutinase enzyme from <i>F. oxysporum</i>, indicating potential biodegradation activity for these polymers. PET exhibited the highest binding affinity (− 34.26 kcal/mol). Besides PET, the polymers PHBH, PBS, PBAT, and PCL also demonstrated significant affinities for the <i>Fo</i>Cut enzyme, with binding values of − 18.44, − 29.71, − 22.78, and − 22.26 kcal/mol, respectively. Additionally, analysis of the phylogenetic tree of cutinases produced by different organisms demonstrated that even though the organisms belong to different kingdoms, the cutinase from <i>F. oxysporum</i> (<i>Fo</i>Cut) showed biological similarity in its activity in degrading polymers with the cutinase enzyme from the bacterium <i>Kineococcus radiotolerans</i> and the fungus <i>Moniliophthora roreri</i>. Furthermore, the phylogenetic analysis demonstrated that the PETase enzyme has a very high similarity with the bacterial cutinase enzyme than with the fungal cutinase, therefore demonstrating that the PETase enzyme from <i>Ideonella sakaiensis</i> can easily be a modified bacterial cutinase enzyme that created a unique feature in biodegrading only the pet polymer through an evolutionary process due to its environment and its biochemical need for carbon. Our data demonstrate that bacterial cutinase enzymes have the same common ancestor as the PETase enzyme. Therefore, cutinases and PETase are interconnected through their biological similarity in biodegrading polymers. We demonstrated that important conserved regions, such as the Ser-Asp-His catalytic triad, exist in the enzyme’s catalytic site and that all Cut enzymes from different organisms have the same region to couple with the polymer structures.</p></div></div></section>
<section aria-labelledby="inline-recommendations" class="c-article-recommendations" data-title="Inline Recommendations" data-track-component="inline-recommendations">
<h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
<div class="c-article-recommendations-list">
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-023-37374-3/MediaObjects/41467_2023_37374_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_1" data-track-action="click recommendations inline - 1" data-track-context="inline recommendations" data-track-label="10.1038/s41467-023-37374-3" href="https://www.nature.com/articles/s41467-023-37374-3?fromPaywallRec=false" itemprop="url">Complete bio-degradation of poly(butylene adipate-co-terephthalate) via engineered cutinases
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__access-type">Open access</span>
<span class="c-article-meta-recommendations__date">24 March 2023</span>
</div>
</div>
</article>
</div>
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-023-39201-1/MediaObjects/41467_2023_39201_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_2" data-track-action="click recommendations inline - 2" data-track-context="inline recommendations" data-track-label="10.1038/s41467-023-39201-1" href="https://www.nature.com/articles/s41467-023-39201-1?fromPaywallRec=false" itemprop="url">Mechanistic studies of a lipase unveil effect of pH on hydrolysis products of small PET modules
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__access-type">Open access</span>
<span class="c-article-meta-recommendations__date">15 June 2023</span>
</div>
</div>
</article>
</div>
<div class="c-article-recommendations-list__item">
<article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
<div class="c-article-recommendations-card__img"><img alt="" loading="lazy" src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-021-03408-3/MediaObjects/41586_2021_3408_Fig1_HTML.png"/></div>
<div class="c-article-recommendations-card__main">
<h3 class="c-article-recommendations-card__heading" itemprop="name headline">
<a class="c-article-recommendations-card__link" data-track="select_recommendations_3" data-track-action="click recommendations inline - 3" data-track-context="inline recommendations" data-track-label="10.1038/s41586-021-03408-3" href="https://www.nature.com/articles/s41586-021-03408-3?fromPaywallRec=false" itemprop="url">Near-complete depolymerization of polyesters with nano-dispersed enzymes
                                        </a>
</h3>
<div class="c-article-meta-recommendations" data-test="recommendation-info">
<span class="c-article-meta-recommendations__item-type">Article</span>
<span class="c-article-meta-recommendations__date">21 April 2021</span>
</div>
</div>
</article>
</div>
</div>
</section>
<script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1738882480,
                        embedded_user: 'null'
                    }
                });
            </script>
<div class="main-content">
<section data-title="Introduction"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Introduction</h2><div class="c-article-section__content" id="Sec1-content"><p>Plastics are essential materials for modern society due to their distinct properties, including lightness, durability, versatility, and low production costs<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR1" id="ref-link-section-d32394649e566" title="Austin, H. P. et al. Characterization and engineering of a plastic-degrading aromatic polyesterase. Proc. Natl. Acad. Sci. USA 115, E4350–E4357 (2018).">1</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR2" id="ref-link-section-d32394649e566_1" title="Han, X. et al. Structural insight into catalytic mechanism of PET hydrolase. Nat. Commun. 8, 2106 (2017).">2</a>,<a aria-label="Reference 3" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR3" id="ref-link-section-d32394649e569" title="Joo, S. et al. Structural insight into molecular mechanism of poly(ethylene terephthalate) degradation. Nat. Commun. 9, 382 (2018).">3</a></sup>. However, their non-degradability poses a significant threat to the environment, primarily because synthetic polymers are chemically resistant and take a long time to degrade, remaining for years, especially in oceans<sup><a aria-label="Reference 4" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR4" id="ref-link-section-d32394649e573" title="Rhodes, C. J. Plastic pollution and potential solutions. Sci. Prog. 101, 207–260 (2018).">4</a></sup>. The use of biodegradable polymers, both synthetic and natural, has emerged as a more sustainable alternative for plastic production, since pollution by non-biodegradable plastics affects everything from marine animals, interfering with their feeding behaviour, to humans with the presence of microplastics<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR5" id="ref-link-section-d32394649e577" title="Kankanige, D. &amp; Babel, S. Smaller-sized micro-plastics (MPs) contamination in single-use PET-bottled water in Thailand. Sci. Total Environ. 717, 137232 (2020).">5</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR6" id="ref-link-section-d32394649e577_1" title="Samir, A., Ashour, F. H., Hakim, A. A. A. &amp; Bassyouni, M. Recent advances in biodegradable polymers for sustainable applications. Npj Mater. Degrad. 6, 68 (2022).">6</a>,<a aria-label="Reference 7" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR7" id="ref-link-section-d32394649e580" title="Roman, L., Schuyler, Q., Wilcox, C. &amp; Hardesty, B. D. Plastic pollution is killing marine megafauna, but how do we prioritize policies to reduce mortality?. Conserv. Lett. 14, 12871 (2021).">7</a></sup>. The biodegradation of polymers occurs through the activity of enzymes produced by microorganisms such as fungi and bacteria, capable of secreting enzymes such as PETases and Cutinases<sup><a aria-label="Reference 8" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR8" id="ref-link-section-d32394649e584" title="Benavides Fernández, C. D., Guzmán Castillo, M. P., Quijano Pérez, S. A. &amp; Carvajal Rodríguez, L. V. Microbial degradation of polyethylene terephthalate: A systematic review. SN Appl. Sci. 4, 263 (2022).">8</a>,<a aria-label="Reference 9" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR9" id="ref-link-section-d32394649e587" title="Kanwal, A., Zhang, M., Sharaf, F. &amp; Li, C. Polymer pollution and its solutions with special emphasis on Poly (butylene adipate terephthalate (PBAT)). Polym. Bull. 79, 9303–9330 (2022).">9</a></sup>.</p><p>Cutinases (Cut), a type of serine hydrolase found in various organisms (e.g. <i>Fusarium solani pisi</i><sup><a aria-label="Reference 10" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR10" id="ref-link-section-d32394649e596" title="Hellesnes, K. N., Vijayaraj, S., Fojan, P., Petersen, E. &amp; Courtade, G. Biochemical Characterization and NMR Study of a PET-hydrolyzing cutinase from Fusarium solani pisi. Biochemistry 62, 1369–1375 (2023).">10</a></sup>, <i>Fusarium oxysporum</i><sup><a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR11" id="ref-link-section-d32394649e602" title="Dimarogona, M. et al. Structural and functional studies of a Fusarium oxysporum cutinase with polyethylene terephthalate modification potential. Biochim. Biophys. Acta BBA General Subjects 1850, 2308–2317 (2015).">11</a></sup>, <i>Moniliophthora roreri</i><sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR12" id="ref-link-section-d32394649e608" title="Vázquez-Alcántara, L., Oliart-Ros, R. M., García-Bórquez, A. &amp; Peña-Montes, C. Expression of a cutinase of Moniliophthora roreri with polyester and PET-plastic residues degradation activity. Microbiol. Spectr. 9(3), e00976-e1021 (2021).">12</a></sup>, <i>Saccharomonospora viridis</i><sup><a aria-label="Reference 13" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR13" id="ref-link-section-d32394649e615" title="Numoto, N. et al. Structural dynamics of the PET-degrading cutinase-like enzyme from Saccharomonospora viridis AHK190 in substrate-bound states elucidates the Ca2+—driven catalytic cycle. Biochemistry 57, 5289–5300 (2018).">13</a></sup>, <i>Thermobifida alba</i><sup><a aria-label="Reference 14" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR14" id="ref-link-section-d32394649e621" title="Ribitsch, D. et al. Characterization of a new cutinase from Thermobifida alba for PET-surface hydrolysis. Biocatal Biotransform. 30, 2–9 (2012).">14</a></sup>, <i>Thermobifida cellulosilytica</i><sup><a aria-label="Reference 15" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR15" id="ref-link-section-d32394649e627" title="Zhang, Z. et al. Depolymerization of post-consumer PET bottles with engineered cutinase 1 from Thermobifida cellulosilytica. Green Chem. 24, 5998–6007 (2022).">15</a></sup>, <i>Thermobifida fusca</i><sup><a aria-label="Reference 16" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR16" id="ref-link-section-d32394649e633" title="Furukawa, M., Kawakami, N., Tomizawa, A. &amp; Miyamoto, K. Efficient degradation of poly(ethylene terephthalate) with thermobifida fusca cutinase exhibiting improved catalytic activity generated using mutagenesis and additive-based approaches. Sci. Rep. 9, 16038 (2019).">16</a></sup>, <i>Humicola insolens</i><sup><a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR17" id="ref-link-section-d32394649e640" title="Aristizábal-Lanza, L., Mankar, S. V., Tullberg, C., Zhang, B. &amp; Linares-Pastén, J. A. Comparison of the enzymatic depolymerization of polyethylene terephthalate and AkestraTM using Humicola insolens cutinase. Front. Chem. Eng. 4, 1048744  (2022).">17</a></sup>, <i>Kineococcus radiotolerans</i><sup><a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR18" id="ref-link-section-d32394649e646" title="Abokitse, K. et al. A novel actinobacterial cutinase containing a noncatalytic polymer-binding domain. Appl. Environ. Microbiol. 88,  e0152221. 
                  https://doi.org/10.1128/AEM.01522-21
                  
                 (2022).">18</a></sup>), are capable of degrading cutin, a fatty acid polyester that forms the main component of plant cuticles. These enzymes have emerged as vital biocatalysts in the breakdown of synthetic polymers such as poly(ethylene terephthalate) (PET)<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR12" id="ref-link-section-d32394649e650" title="Vázquez-Alcántara, L., Oliart-Ros, R. M., García-Bórquez, A. &amp; Peña-Montes, C. Expression of a cutinase of Moniliophthora roreri with polyester and PET-plastic residues degradation activity. Microbiol. Spectr. 9(3), e00976-e1021 (2021).">12</a></sup>, poly(lactic Acid) (PLA)<sup><a aria-label="Reference 19" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR19" id="ref-link-section-d32394649e654" title="Kitadokoro, K. et al. Structural insights into the unique polylactate-degrading mechanism of Thermobifida alba cutinase. FEBS J. 286, 2087–2098 (2019).">19</a></sup>, poly(3-hydroxybutyrate-co-3-hydroxyvalerate) (PHBV)<sup><a aria-label="Reference 20" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR20" id="ref-link-section-d32394649e658" title="Gamerith, C. et al. Enzymatic degradation of aromatic and aliphatic polyesters by P. pastoris expressed cutinase 1 from Thermobifida cellulosilytica. Front. Microbiol. 8, (2017).">20</a></sup>, poly(butylene succinate) (PBS)<sup><a aria-label="Reference 21" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR21" id="ref-link-section-d32394649e663" title="Hu, X. et al. Enzymatic degradation of poly(butylene succinate) by cutinase cloned from Fusarium solani. Polym. Degrad. Stab. 134, 211–219 (2016).">21</a></sup>, poly(butylene adipate-co-terephthalate) (PBAT)<sup><a aria-label="Reference 22" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR22" id="ref-link-section-d32394649e667" title="Yang, Y. et al. Complete bio-degradation of poly(butylene adipate-co-terephthalate) via engineered cutinases. Nat. Commun. 14, 1645 (2023).">22</a>,<a aria-label="Reference 23" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR23" id="ref-link-section-d32394649e670" title="Lin, W., Zhao, Y., Su, T. &amp; Wang, Z. Enzymatic hydrolysis of poly(butylene adipate-co-terephthalate) by Fusarium solani cutinase. Polym. Degrad. Stab. 211, 110335 (2023).">23</a></sup>, poly(ε-caprolactone) (PCL)<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR12" id="ref-link-section-d32394649e674" title="Vázquez-Alcántara, L., Oliart-Ros, R. M., García-Bórquez, A. &amp; Peña-Montes, C. Expression of a cutinase of Moniliophthora roreri with polyester and PET-plastic residues degradation activity. Microbiol. Spectr. 9(3), e00976-e1021 (2021).">12</a></sup>, poly(ethylene succinate) (PES)<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR12" id="ref-link-section-d32394649e678" title="Vázquez-Alcántara, L., Oliart-Ros, R. M., García-Bórquez, A. &amp; Peña-Montes, C. Expression of a cutinase of Moniliophthora roreri with polyester and PET-plastic residues degradation activity. Microbiol. Spectr. 9(3), e00976-e1021 (2021).">12</a></sup>. In 2022, Abokitse et al. demonstrated that the bacterium <i>Kineococcus radiotolerans</i> also produces a Cut capable of biodegrading polymers. Notably, <i>F. oxysporu</i> has so far only shown PET biodegradation activity among polymers.</p><p>The fungus <i>Fusarium oxysporum</i> is known to thrive in moist, warm environments. However, <i>Fusarium</i> genus can be quite versatile and adapt to a various environmental conditions, including dry conditions or aquatic habitats in certain cases<sup><a aria-label="Reference 24" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR24" id="ref-link-section-d32394649e697" title="Abdel-Azeem, A. M., Abdel-Azeem, M. A., Darwish, A. G., Nafady, N. A. &amp; Ibrahim, N. A. Fusarium: Biodiversity, ecological significances, and industrial applications. 201–261 (Springer Nature, 2019).">24</a>,<a aria-label="Reference 25" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR25" id="ref-link-section-d32394649e700" title="Steinberg, C. et al. Adaptation of Fusarium oxysporum and Fusarium dimerum to the specific aquatic environment provided by the water systems of hospitals. Water Res. 76, 53–65 (2015).">25</a></sup>. In the Amazon, for example, the fungus is the leading cause of the death of black pepper plantations in Pará<sup><a aria-label="Reference 26" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR26" id="ref-link-section-d32394649e704" title="Barata, L. M. et al. Secondary metabolic profile as a tool for distinction and characterization of cultivars of black pepper (Piper nigrum L.) cultivated in Pará State. Brazil. Int. J. Mol. Sci. 22, 890 (2021).">26</a></sup>. Pará is one of the primary producers and exporters of black pepper in Brazil<sup><a aria-label="Reference 27" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR27" id="ref-link-section-d32394649e708" title="Paracampo, N. E., Abreu, L. F., de Lemos, O. F. &amp; Both, J. P. Quality Of black pepper produced in northeastern Pará. Rev. De Agric. Neotrop. 9(3), e7020 (2022).">27</a></sup>. Therefore, <i>F. oxysporum</i>, in addition to being a problem in the health sector as it causes the disease fusariosis, is also a problem for the economic sector of a region<sup><a aria-label="Reference 24" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR24" id="ref-link-section-d32394649e716" title="Abdel-Azeem, A. M., Abdel-Azeem, M. A., Darwish, A. G., Nafady, N. A. &amp; Ibrahim, N. A. Fusarium: Biodiversity, ecological significances, and industrial applications. 201–261 (Springer Nature, 2019).">24</a></sup>. Furthermore, this organism attacks other crops such as bananas<sup><a aria-label="Reference 28" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR28" id="ref-link-section-d32394649e720" title="Bubici, G., Kaushal, M., Prigigallo, M. I., Gómez-Lama Cabanás, C. &amp; Mercado-Blanco, J. Biological control agents against fusarium wilt of Banana. Front. Microbiol. 10, (2019).">28</a></sup> and tomatoes<sup><a aria-label="Reference 29" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR29" id="ref-link-section-d32394649e724" title="McGovern, R. J. Management of tomato diseases caused by Fusarium oxysporum. Crop Protection 73, 78–92 (2015).">29</a></sup>.</p><p>Therefore, this work evaluates synthetic polymers’ binding affinity the <i>Fusarium oxysporum</i> Cutinase (Cut) enzyme using computational methods such as Molecular Dynamics, Molecular Docking, and Binding Free Energy calculations, as employed in previous studies<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR30" id="ref-link-section-d32394649e734" title="de Oliveira, M. V., Fernandes, G. M., da Costa, K. S., Vakal, S. &amp; Lima, A. H. Virtual screening of natural products against 5-enolpyruvylshikimate-3-phosphate synthase using the Anagreen herbicide-like natural compound library. RSC Advances. 12(29), 18834–18847 (2022).">30</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR31" id="ref-link-section-d32394649e734_1" title="De Oliveira, M. D., Araújo, J. de O., Galúcio, J. M. P., Santana, K. &amp; Lima, A. H. Targeting shikimate pathway: In silico analysis of phosphoenolpyruvate derivatives as inhibitors of EPSP synthase and DAHP synthase. J. Mol. Graph Model 101, 107735 (2020).">31</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR32" id="ref-link-section-d32394649e734_2" title="De Souza, E. M. de C. et al. Molecular characterization and in silico evaluation of surfactins produced by endophytic bacteria from Phanera splendens. Front. Chem. 11, 1240704 (2023).">32</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR33" id="ref-link-section-d32394649e734_3" title="de Oliveira, M. V., Almeida, L., da Rocha, J., Wanzeller, A. L. &amp; Lima, A. A combined approach for predicting binding affinity of Zika Virus NS2B-NS3 protease inhibitors using semi-empirical methods and molecular docking algorithms. J. Braz. Chem. Soc. 
                  https://doi.org/10.21577/0103-5053.20230130
                  
                 (2024).">33</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR34" id="ref-link-section-d32394649e734_4" title="Costa, C. H. S. et al. Assessment of the PETase conformational changes induced by poly(ethylene terephthalate) binding. Proteins Struct. Funct. Bioinform. 89, 1340–1352 (2021).">34</a>,<a aria-label="Reference 35" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR35" id="ref-link-section-d32394649e737" title="Araújo, O. J. et al. Structural, energetic and lipophilic analysis of SARS-CoV-2 non-structural protein 9 (NSP9). Sci. Rep. 11, 23003 (2021).">35</a></sup>. Overall, we provide insights into the potential use of this enzyme as a biocatalyst in the biodegradation of polymers other than PET.</p></div></div></section><section data-title="Material and methods"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Material and methods</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Phylogenetic analysis protocol</h3><p>We used the Molecular Evolutionary Genetics Analysis version 11 (MEGA11)<sup><a aria-label="Reference 36" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR36" id="ref-link-section-d32394649e753" title="Tamura, K., Stecher, G. &amp; Kumar, S. MEGA11: Molecular evolutionary genetics analysis version 11. Mol. Biol. Evol. 38, 3022–3027 (2021).">36</a></sup> software. We selected some organisms that produce the Cut enzyme, fungi, and bacteria deposited in the PDB. We selected the FASTA sequence from the PDB of these Cut proteins as <i>Halopseudomonas bauzanensis</i> (8AIT)<sup><a aria-label="Reference 37" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR37" id="ref-link-section-d32394649e760" title="Avilan, L. et al. Concentration-dependent inhibition of mesophilic PETases on poly (ethylene terephthalate) can be eliminated by enzyme engineering. ChemSusChem. 16(8), e202202277 (2023).">37</a></sup>, <i>Fusarium oxysporum</i> (5AJH)<sup><a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR11" id="ref-link-section-d32394649e767" title="Dimarogona, M. et al. Structural and functional studies of a Fusarium oxysporum cutinase with polyethylene terephthalate modification potential. Biochim. Biophys. Acta BBA General Subjects 1850, 2308–2317 (2015).">11</a></sup>, <i>Fusarium vanettenii</i> (1CEX)<sup><a aria-label="Reference 38" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR38" id="ref-link-section-d32394649e775" title="Longhi, S., Czjzek, M., Lamzin, V., Nicolas, A. &amp; Cambillau, C. Atomic resolution (1.0 Å) crystal structure of Fusarium solani cutinase: stereochemical analysis. J. Mol. Biol. 268, 779–799 (1997).">38</a></sup>, <i>Fusarium solani</i> (1CUS)<sup><a aria-label="Reference 39" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR39" id="ref-link-section-d32394649e782" title="Martinez, C., De Geus, P., Lauwereys, M., Matthyssens, G. &amp; Cambillau, C. Fusarium solani cutinase is a lipolytic enzyme with a catalytic serine accessible to solvent. Nature 356, 615–618 (1992).">39</a></sup>, <i>Paraphoma</i> sp. (7CY3), <i>Humicola insolens</i> (4OYY)<sup><a aria-label="Reference 40" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR40" id="ref-link-section-d32394649e793" title="Kold, D. et al. Thermodynamic and structural investigation of the specific SDS binding of humicola insolens cutinase. Protein Science 23, 1023–1035 (2014).">40</a></sup>, <i>Colletotrichum gloeosporioides</i> (3DCN)<sup><a aria-label="Reference 41" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR41" id="ref-link-section-d32394649e800" title="Nyon, M. P. et al. Catalysis by glomerella cingulata cutinase requires conformational cycling between the active and inactive states of its catalytic triad. J. Mol. Biol. 385, 226–235 (2009).">41</a></sup>, <i>Aspergillus oryzae</i> (3GBS)<sup><a aria-label="Reference 42" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR42" id="ref-link-section-d32394649e807" title="Liu, Z. et al. Structural and functional studies of Aspergillus oryzae cutinase: Enhanced thermostability and hydrolytic activity of synthetic ester and polyester degradation. J. Am. Chem. Soc. 131, 15711–15716 (2009).">42</a></sup>, <i>Malbranchea cinnamomea</i> (5X88)<sup><a aria-label="Reference 43" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR43" id="ref-link-section-d32394649e815" title="Duan, X. et al. High-level expression and characterization of a novel cutinase from Malbranchea cinnamomea suitable for butyl butyrate production. Biotechnol. Biofuels 10, 223 (2017).">43</a></sup>, <i>Trichoderma reesei</i> (4PSC)<sup><a aria-label="Reference 44" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR44" id="ref-link-section-d32394649e822" title="Roussel, A. et al. A cutinase from trichoderma reesei with a lid-covered active site and kinetic properties of true lipases. J Mol. Biol. 426, 3757–3772 (2014).">44</a></sup>, <i>Saccharomonosphora viridis</i> (4WFI)<sup><a aria-label="Reference 45" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR45" id="ref-link-section-d32394649e829" title="Miyakawa, T. et al. Structural basis for the Ca2+-enhanced thermostability and activity of PET-degrading cutinase-like enzyme from Saccharomonospora viridis AHK190. Appl. Microbiol. Biotechnol. 99, 4297–4307 (2015).">45</a></sup>, <i>Saccharopolyspora flava</i> (7QJP)<sup><a aria-label="Reference 46" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR46" id="ref-link-section-d32394649e837" title="Erickson, E. et al. Sourcing thermotolerant poly(ethylene terephthalate) hydrolase scaffolds from natural diversity. Nat. Commun. 13, 7850 (2022).">46</a></sup>, <i>Thermobifida cellulosilytica</i> (5LUI)<sup><a aria-label="Reference 47" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR47" id="ref-link-section-d32394649e844" title="Ribitsch, D. et al. Small cause, large effect: Structural characterization of cutinases from Thermobifida cellulosilytica. Biotechnol. Bioeng. 114, 2481–2488 (2017).">47</a></sup>, <i>Thermobifida fusca</i> (5ZOA), <i>Thermobifida alba</i> (3VIS)<sup><a aria-label="Reference 48" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR48" id="ref-link-section-d32394649e854" title="Kitadokoro, K. et al. Crystal structure of cutinase Est119 from Thermobifida alba AHK119 that can degrade modified polyethylene terephthalate at 1.76Å resolution. Polym. Degrad. Stab. 97, 771–775 (2012).">48</a></sup>. Two organisms were selected from recent studies and do not have the crystallized structure of the Cut enzyme deposited in the PDB, namely <i>Moniliophthora roreri</i> (GenBank: ESK97883.1)<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR12" id="ref-link-section-d32394649e862" title="Vázquez-Alcántara, L., Oliart-Ros, R. M., García-Bórquez, A. &amp; Peña-Montes, C. Expression of a cutinase of Moniliophthora roreri with polyester and PET-plastic residues degradation activity. Microbiol. Spectr. 9(3), e00976-e1021 (2021).">12</a></sup> and <i>Kineococcus radiotolerans</i> (GenBank: ABS05574.1)<sup><a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR18" id="ref-link-section-d32394649e869" title="Abokitse, K. et al. A novel actinobacterial cutinase containing a noncatalytic polymer-binding domain. Appl. Environ. Microbiol. 88,  e0152221. 
                  https://doi.org/10.1128/AEM.01522-21
                  
                 (2022).">18</a></sup>. Additionally, we added the PETase enzyme from the bacteria <i>Ideonella sakaiensis</i> (6EQE)<sup><a aria-label="Reference 1" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR1" id="ref-link-section-d32394649e876" title="Austin, H. P. et al. Characterization and engineering of a plastic-degrading aromatic polyesterase. Proc. Natl. Acad. Sci. USA 115, E4350–E4357 (2018).">1</a></sup>. We performed a global alignment of the sequences using ClustalW. The phylogenetic tree was plotted using the Neighbor–Joining statistical method, using Bootstrap method with 10,000 replications.</p><h3 class="c-article__sub-heading" id="Sec4">Structural preparation of polymers and <i>Fo</i>Cut</h3><p>The three-dimensional structure of the Cutinase hydrolase from <i>Fusarium oxysporum</i> (<i>Fo</i>Cut) was acquired from the RCSB Protein Data Bank<sup><a aria-label="Reference 49" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR49" id="ref-link-section-d32394649e898" title="Berman, H. M. et al. The protein data bank. Nucl. Acids Res. 
                  https://doi.org/10.1093/nar/28.1.235
                  
                 (2000).">49</a></sup> using the PDB code 5AJH<sup><a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR11" id="ref-link-section-d32394649e902" title="Dimarogona, M. et al. Structural and functional studies of a Fusarium oxysporum cutinase with polyethylene terephthalate modification potential. Biochim. Biophys. Acta BBA General Subjects 1850, 2308–2317 (2015).">11</a></sup>. This structure was determined by X-ray diffraction with a resolution of 1.90 Å. The nine polymer structures were designed in 2D (Table <a data-track="click" data-track-action="table anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Tab1">1</a>) using the MarvinSketch<sup><a aria-label="Reference 50" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR50" id="ref-link-section-d32394649e910" title="ChemAxon. ChemAxon—Software Solutions and Services for Chemistry and Biology. MarvinSketch, Version 16.10.31 Preprint at (2016).">50</a></sup> program (version 18.9) and subsequently converted and analyzed in the 3D model using the Avogadro<sup><a aria-label="Reference 51" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR51" id="ref-link-section-d32394649e914" title="Hanwell, M. D. et al. Avogadro: An advanced semantic chemical editor, visualization, and analysis platform. J. Cheminform. 
                  https://doi.org/10.1186/1758-2946-4-17
                  
                 (2012).">51</a></sup> program (version 1.2.0). The PET, PLA, PHB, PBS, PCL and PES structures were used a tetramer. PHBH and PHBV structure were used as a dimer. PBAT structure was used as a monomer. This difference of each ligand (tetramer, dimer or monomer) is related to the volume of each structure. Finally, 3D structures were optimized in the Gaussian09<sup><a aria-label="Reference 52" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR52" id="ref-link-section-d32394649e918" title="Frisch, M. J. ; et. al, Nathan, A. J. &amp; Scobell, A. Gaussian 09. Gaussian, Inc. Wallingford CT 2–3 Preprint at 111 (2009).">52</a></sup> program using the Hartree-Fock<sup><a aria-label="Reference 53" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR53" id="ref-link-section-d32394649e922" title="Roothaan, C. C. J. New developments in molecular orbital theory. Rev. Mod. Phys. 23, 69–89 (1951).">53</a></sup> method with the 6-31G* basis set.</p><div class="c-article-table" data-container-section="table" data-test="inline-table" id="table-1"><figure><figcaption class="c-article-table__figcaption"><b data-test="table-caption" id="Tab1">Table 1 Two-dimensional structure and name of each of the nine selected polymers and works that demonstrated the Cut enzyme from different organisms degrading the polymer.</b></figcaption><div class="u-text-right u-hide-print"><a aria-label="Full size table 1" class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" href="/articles/s41598-024-84718-0/tables/1" rel="nofollow"><span>Full size table</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec5">Docking molecular protocol</h3><p>We used the Molegro Virtual Docker (MVD) software (version 5.5) <sup><a aria-label="Reference 60" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR60" id="ref-link-section-d32394649e1374" title="Thomsen, R. &amp; Christensen, M. H. MolDock: A new technique for high-accuracy molecular docking. J. Med. Chem. 49, 3315–3321 (2006).">60</a></sup> for the molecular docking stage. MVD employs the MolDock heuristic search algorithm, which uses a piecewise linear potential (PLP) scoring function that includes hydrogen and electrostatic interaction terms<sup><a aria-label="Reference 60" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR60" id="ref-link-section-d32394649e1378" title="Thomsen, R. &amp; Christensen, M. H. MolDock: A new technique for high-accuracy molecular docking. J. Med. Chem. 49, 3315–3321 (2006).">60</a></sup>. Molecular docking was performed using a flexible docking approach<sup><a aria-label="Reference 61" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR61" id="ref-link-section-d32394649e1382" title="Jones, G., Willett, P., Glen, R. C., Leach, A. R. &amp; Taylor, R. Development and validation of a genetic algorithm for flexible docking 1 1Edited by F E. Cohen. J. Mol. Biol. 267, 727–748 (1997).">61</a></sup>, where the active site residues of <i>Fo</i>Cut were considered flexible to ensure optimal coupling with the nine polymer structures. This strategy allowed for more interactions at the enzyme site, eliminating the need to apply a restriction force to maintain these interactions during the Molecular Dynamics step. Then, for the molecular docking protocol, we used the MolDock SE algorithm with the following parameters: number of runs = 10, space coordinates (X = − 15.35, Y = 13.52, Z = − 41.65), maximum iterations = 1500, maximum population size = 50, and grid resolution = 25.0 Å.</p><h3 class="c-article__sub-heading" id="Sec6">Molecular Dynamics (MD) protocol</h3><p>Initially, we used the PROPKA<sup><a aria-label="Reference 62" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR62" id="ref-link-section-d32394649e1397" title="Jurrus, E. et al. Improvements to the &lt;scp&gt;APBS&lt;/scp&gt; biomolecular solvation software suite. Protein Sci. 27, 112–128 (2018).">62</a></sup> to determine the protonation states of the ionizable residues of <i>Fo</i>Cut. The nine polymers and the protein were submitted to the CHARMM-GUI web server using the CHARMM36 force field<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR63" id="ref-link-section-d32394649e1404" title="Jo, S., Kim, T., Iyer, V. G. &amp; Im, W. CHARMM-GUI: A web-based graphical user interface for CHARMM. J. Comput. Chem. 29, 1859–1865 (2008).">63</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR64" id="ref-link-section-d32394649e1404_1" title="Brooks, B. R. et al. CHARMM: The biomolecular simulation program. J. Comput. Chem. 30, 1545–1614 (2009).">64</a>,<a aria-label="Reference 65" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR65" id="ref-link-section-d32394649e1407" title="Lee, J. et al. CHARMM-GUI input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field. J. Chem. Theory Comput. 12, 405–413 (2016).">65</a></sup>, as applied in our previous work<sup><a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR34" id="ref-link-section-d32394649e1411" title="Costa, C. H. S. et al. Assessment of the PETase conformational changes induced by poly(ethylene terephthalate) binding. Proteins Struct. Funct. Bioinform. 89, 1340–1352 (2021).">34</a></sup>. The systems were solvated with water molecules using the TIP3P explicit solvation model<sup><a aria-label="Reference 66" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR66" id="ref-link-section-d32394649e1415" title="Jorgensen, W. L., Chandrasekhar, J., Madura, J. D., Impey, R. W. &amp; Klein, M. L. Comparison of simple potential functions for simulating liquid water. J. Chem. Phys. 
                  https://doi.org/10.1063/1.445869
                  
                 (1983).">66</a></sup>, within an octahedral box, with a radius of 12.0 Å (distance from the protein to the box), and Cl⁻ counter-ions to maintain the electroneutrality of the system. We prepared nine complexes: PEP–, PLA–, PHB–, PHBH–, PHBV–, PBS–, PBAT–, PCL–, PES–<i>Fo</i>Cut. Then, the structural-energetic minimization step was performed in four stages. In the first step, we included only the counter-ions and water molecules. In the second step, we minimized only the hydrogen atoms of the protein structure. The third step included the hydrogen atoms of the entire complex (ligand–protein structure and water molecules). The final step involved minimizing the entire system. Then, all the systems were subjected to a heating step from 0 to 300 K using 200 ps of MD in a constant volume with a restraint weight of 5.0 kcal·mol⁻<sup>1</sup>·Å<sup>2</sup> at the positions of the atoms (complex). Subsequently, all systems were equilibrated with 500 ps of MD with no restraints at a constant pressure of 1 bar. The temperature of the systems was maintained at 300 K using a Langevin thermostat with a collision frequency of 2 ps<sup>⁻1</sup>, and the isotropic constant pressure was maintained at 1 bar using a Berendsen barostat. We used a cutoff of 10 Å for non-bonded interactions and the Particle Mesh Ewald (PME) method to calculate long-range electrostatic interactions. MD simulations were performed assuming a simulation time of 300 ns, conducted in triplicate (100 ns for each replicate). All MM simulations were carried out in AMBER20<sup><a aria-label="Reference 67" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR67" id="ref-link-section-d32394649e1429" title="Case, D. A. et al. Amber 20. (2020).">67</a></sup> package. Additionally, we performed an MD simulation of <i>Fo</i>Cut without any polymer, assuming a simulation time of 1 μs.</p><h3 class="c-article__sub-heading" id="Sec7">Binding free energy calculations and per-residue free energy decomposition protocol</h3><p>To evaluate the affinity of the nine polymers to <i>Fo</i>Cut, we employed the Molecular Mechanics with Generalized Born and Surface Area Solvation (MM/GBSA) approache<sup><a aria-label="Reference 68" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR68" id="ref-link-section-d32394649e1448" title="Srinivasan, J., Cheatham, T. E., Cieplak, P., Kollman, P. A. &amp; Case, D. A. Continuum solvent studies of the stability of DNA, RNA, and phosphoramidate−DNA helices. J. Am. Chem. Soc. 120, 9401–9409 (1998).">68</a>,<a aria-label="Reference 69" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR69" id="ref-link-section-d32394649e1451" title="Miller, B. R. III. et al. MMPBSA. py: An efficient program for end-state free energy calculations. J. Chem. Theory Comput. 8(9), 3314–3321 (2012).">69</a></sup> available in Amber20<sup><a aria-label="Reference 67" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR67" id="ref-link-section-d32394649e1455" title="Case, D. A. et al. Amber 20. (2020).">67</a></sup>. We selected 10,000 frames from the MD trajectory, corresponding to the entire 100 ns simulation of each replicate, to evaluate the contributions and the binding free energy <span class="mathjax-tex">\((\Delta {G}_{binding}\)</span>) values throughout the simulation time. Subsequently, we performed a decomposition analysis to identify the central residues that contribute to the binding affinity of the polymer–<i>Fo</i>Cut complex.</p><p>The <span class="mathjax-tex">\(\Delta {G}_{binding}\)</span> is calculated through Eq. <a data-track="click" data-track-action="equation anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Equ1">1</a>, where the terms <span class="mathjax-tex">\(\Delta {E}_{MM}\)</span>, <span class="mathjax-tex">\(\Delta {G}_{sol}\)</span> and <span class="mathjax-tex">\(T\Delta S\)</span> represent the changes in the molecular mechanical energy of the gas phase (MM), free solvation energy, and conformational entropy after the binding state of the binder, respectively<sup><a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR31" id="ref-link-section-d32394649e1524" title="De Oliveira, M. D., Araújo, J. de O., Galúcio, J. M. P., Santana, K. &amp; Lima, A. H. Targeting shikimate pathway: In silico analysis of phosphoenolpyruvate derivatives as inhibitors of EPSP synthase and DAHP synthase. J. Mol. Graph Model 101, 107735 (2020).">31</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR69" id="ref-link-section-d32394649e1527" title="Miller, B. R. III. et al. MMPBSA. py: An efficient program for end-state free energy calculations. J. Chem. Theory Comput. 8(9), 3314–3321 (2012).">69</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR70" id="ref-link-section-d32394649e1527_1" title="Genheden, S. &amp; Ryde, U. The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities. Expert Opin. Drug Discov. 10, 449–461 (2015).">70</a>,<a aria-label="Reference 71" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR71" id="ref-link-section-d32394649e1530" title="Wang, L., Pang, X., Li, Y., Zhang, Z. &amp; Tan, W. RADER: A RApid DEcoy retriever to facilitate decoy based assessment of virtual screening. Bioinformatics 33, 1235–1237 (2017).">71</a></sup>.</p><div class="c-article-equation" id="Equ1"><div class="c-article-equation__content"><span class="mathjax-tex">$$\Delta {G}_{binding}=\Delta {E}_{MM}+\Delta {G}_{sol}-T\Delta S$$</span></div><div class="c-article-equation__number">
                    (1)
                </div></div><p>Therefore, <span class="mathjax-tex">\(\Delta {E}_{MM}\)</span> refers to the energy of molecular mechanics which can be described as the sum of intramolecular energy (internal, electrostatic and van der Waals energy)<sup><a aria-label="Reference 31" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR31" id="ref-link-section-d32394649e1560" title="De Oliveira, M. D., Araújo, J. de O., Galúcio, J. M. P., Santana, K. &amp; Lima, A. H. Targeting shikimate pathway: In silico analysis of phosphoenolpyruvate derivatives as inhibitors of EPSP synthase and DAHP synthase. J. Mol. Graph Model 101, 107735 (2020).">31</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR69" id="ref-link-section-d32394649e1563" title="Miller, B. R. III. et al. MMPBSA. py: An efficient program for end-state free energy calculations. J. Chem. Theory Comput. 8(9), 3314–3321 (2012).">69</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR70" id="ref-link-section-d32394649e1563_1" title="Genheden, S. &amp; Ryde, U. The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities. Expert Opin. Drug Discov. 10, 449–461 (2015).">70</a>,<a aria-label="Reference 71" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR71" id="ref-link-section-d32394649e1566" title="Wang, L., Pang, X., Li, Y., Zhang, Z. &amp; Tan, W. RADER: A RApid DEcoy retriever to facilitate decoy based assessment of virtual screening. Bioinformatics 33, 1235–1237 (2017).">71</a></sup>. <span class="mathjax-tex">\(\Delta {G}_{sol}\)</span> represents the sum of all the electrostatic solvation energy (polar contribution) and all the nonpolar contribution between the solute and the solvent and right after the third term of the <span class="mathjax-tex">\(\Delta {G}_{binding}\)</span> equation, <span class="mathjax-tex">\(T\Delta S\)</span>, represents the temperature, <span class="mathjax-tex">\(T,\)</span> and entropy, <span class="mathjax-tex">\(\Delta S\)</span>, of the system<sup><a aria-label="Reference 72" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR72" id="ref-link-section-d32394649e1626" title="Wang, E. et al. End-point binding free energy calculation with MM/PBSA and MM/GBSA: Strategies and applications in drug design. Chem. Rev. 119, 9478–9508 (2019).">72</a></sup>. In addition, the GB method has limitations in calculating entropy because, due to the high computational cost, changes in conformational entropy are ignored mainly since only the relative binding free energies of similar ligands are needed<sup><a aria-label="Reference 72" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR72" id="ref-link-section-d32394649e1630" title="Wang, E. et al. End-point binding free energy calculation with MM/PBSA and MM/GBSA: Strategies and applications in drug design. Chem. Rev. 119, 9478–9508 (2019).">72</a></sup>.</p></div></div></section><section data-title="Results and discussion"><div class="c-article-section" id="Sec8-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec8">Results and discussion</h2><div class="c-article-section__content" id="Sec8-content"><h3 class="c-article__sub-heading" id="Sec9">Phylogenetic analysis of organisms that produce Cutinases</h3><p>The phylogenetic tree (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig1">1</a>) was developed from the alignment of cutinases present in 17 different organisms to evaluate the similarity of cutinases produced between species compared to cutinases already described in the literature as biodegrading polymers, such as <i>Aspergillus oryzae, Moniliophthora roreri</i>, <i>Paraphoma</i> sp., <i>Humicola insolens</i>, <i>Saccharomonospora viridis, Thermobifida fusca, Fusarium solani,</i> among others<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR12" id="ref-link-section-d32394649e1663" title="Vázquez-Alcántara, L., Oliart-Ros, R. M., García-Bórquez, A. &amp; Peña-Montes, C. Expression of a cutinase of Moniliophthora roreri with polyester and PET-plastic residues degradation activity. Microbiol. Spectr. 9(3), e00976-e1021 (2021).">12</a>,<a aria-label="Reference 17" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR17" id="ref-link-section-d32394649e1666" title="Aristizábal-Lanza, L., Mankar, S. V., Tullberg, C., Zhang, B. &amp; Linares-Pastén, J. A. Comparison of the enzymatic depolymerization of polyethylene terephthalate and AkestraTM using Humicola insolens cutinase. Front. Chem. Eng. 4, 1048744  (2022).">17</a>,<a aria-label="Reference 23" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR23" id="ref-link-section-d32394649e1669" title="Lin, W., Zhao, Y., Su, T. &amp; Wang, Z. Enzymatic hydrolysis of poly(butylene adipate-co-terephthalate) by Fusarium solani cutinase. Polym. Degrad. Stab. 211, 110335 (2023).">23</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR73" id="ref-link-section-d32394649e1672" title="Koitabashi, M. et al. Biodegradable plastic-degrading activity of various species of &lt;i&gt;P&lt;/i&gt;&lt;i&gt;araphoma&lt;/i&gt;. J Oleo Sci 65, 621–627 (2016).">73</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR74" id="ref-link-section-d32394649e1672_1" title="Dsouza, G. C. et al. Fungal biodegradation of low-density polyethylene using consortium of Aspergillus species under controlled conditions. Heliyon 7, e7008 (2021).">74</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR75" id="ref-link-section-d32394649e1672_2" title="Choi, J. et al. Recent advances in microbial and enzymatic engineering for the biodegradation of micro- and nanoplastics. RSC Adv. 14, 9943–9966 (2024).">75</a>,<a aria-label="Reference 76" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR76" id="ref-link-section-d32394649e1675" title="Huang, Q.-S. et al. Accelerated biodegradation of polyethylene terephthalate by Thermobifida fusca cutinase mediated by Stenotrophomonas pavanii. Sci. Total Environ. 808, 152107 (2022).">76</a></sup><i>.</i></p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Fig. 1" id="figure-1"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig1">Fig. 1</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41598-024-84718-0/figures/1" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig1_HTML.png?as=webp" type="image/webp"><img alt="figure 1" aria-describedby="Fig1" height="779" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig1_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p>Phylogenetic tree of Cutinases from different organism fungus and bacteria, highlighting the organism used in this research.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 1" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure1 Full size image" data-track-label="button" href="/articles/s41598-024-84718-0/figures/1" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>The analysis of the phylogenetic tree showed the lineages of several organisms, both fungi and bacteria, that can produce the cutinase enzyme. Among the fungi, the species <i>Fusarium oxysporum</i> stands out, being the species studied in this work, belonging to the same genus as the species <i>F. solani</i> and <i>F. vanettenii</i>, which can be pathogenic to both plants and humans, causing diseases on plants’ roots and several diseases in humans such as pneumonia and other consequences due to contamination by toxins produced by the fungi<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR77" id="ref-link-section-d32394649e1709" title="Scattolini Rimada, A. C., Hernández Rodríguez, L. I. &amp; González Idiarte, H. Identification of Fusarium oxysporum causing sweet potato rot (Ipomoea batatas L &amp; Lamb) in Uruguay. Agrociencia 24,  e124 (2020).">77</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR78" id="ref-link-section-d32394649e1709_1" title="Rodríguez-Grimaldo, J. E., M. González, G. &amp; M. Montoya, A. Fusarium: Un fitopatógeno Que Amenaza La Salud Humana. Revista Ciencia UANL 25, 37–43 (2023).">78</a>,<a aria-label="Reference 79" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR79" id="ref-link-section-d32394649e1712" title="Cighir, A. et al. Fusarium spp. in Human Disease: Exploring the Boundaries between Commensalism and Pathogenesis. Life 13, 1440 (2023).">79</a></sup>.</p><p>Among the bacteria, the genus <i>Thermobifida</i>, which are thermophilic bacteria capable of adapting to environments with high temperatures and have great potential for biodegradation, the species <i>Thermobifida fusca</i> is widely used in studies of cutinase production for biodegradation of polyethylene terephthalate<sup><a aria-label="Reference 16" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR16" id="ref-link-section-d32394649e1725" title="Furukawa, M., Kawakami, N., Tomizawa, A. &amp; Miyamoto, K. Efficient degradation of poly(ethylene terephthalate) with thermobifida fusca cutinase exhibiting improved catalytic activity generated using mutagenesis and additive-based approaches. Sci. Rep. 9, 16038 (2019).">16</a></sup> tag. The species forms a clade with the species <i>T. cellulosilytica</i> and <i>T. alba</i> and are evolutionarily distant from the fungal species of the <i>Fusarium</i>.</p><p>The bacterial species <i>Ideonella sakaiensis</i> was also analyzed, and the organism is phylogenetically distinct from other cutinase-producing bacteria, but closer to other species as <i>Halopseudomonas bauzanensis</i> and <i>Kineococcus radiotolerans</i>. However, its grouping with those other bacteria may be due to a shared ability to degrade synthetic polymers, as <i>I. sakaiensis</i> produces the PETase enzyme.</p><p>Cutinases and PETases are distinct enzymes, but share similar catalytic mechanisms, which may indicate that the enzymes belong to the same family of cutinases<sup><a aria-label="Reference 80" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR80" id="ref-link-section-d32394649e1758" title="Anderson, K. S. Detection of novel enzyme intermediates in PEP-utilizing enzymes. Arch. Biochem. Biophys. 433 47–58 (2005).">80</a></sup>. Furthermore, the phylogenetic analysis revealed that the PETase enzyme shares more similarity with bacterial cutinases than fungal cutinases. Moreover, Yoshida et al.<sup><a aria-label="Reference 81" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR81" id="ref-link-section-d32394649e1762" title="Yoshida, S. et al. A bacterium that degrades and assimilates poly(ethylene terephthalate). Science 1979(351), 1196–1199 (2016).">81</a></sup> and Maity et al.<sup><a aria-label="Reference 82" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR82" id="ref-link-section-d32394649e1766" title="Maity, W., Maity, S., Bera, S. &amp; Roy, A. Emerging roles of PETase and MHETase in the biodegradation of plastic wastes. Appl. Biochem. Biotechnol. 193, 2699–2716 (2021).">82</a></sup> indicated a significant similarity (&gt; 52%) of the PETase enzyme with the cutinases produced by the thermophilic bacterium <i>Thermobifida fusca</i>. This suggests that the PETase enzyme from <i>Ideonella sakaiensis</i> likely evolved from a bacterial cutinase, acquiring a unique ability to specifically degrade PET polymers through an evolutionary process driven by environmental pressures and the organism’s biochemical need for carbon as a source of energy<sup><a aria-label="Reference 1" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR1" id="ref-link-section-d32394649e1777" title="Austin, H. P. et al. Characterization and engineering of a plastic-degrading aromatic polyesterase. Proc. Natl. Acad. Sci. USA 115, E4350–E4357 (2018).">1</a>,<a aria-label="Reference 80" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR80" id="ref-link-section-d32394649e1780" title="Anderson, K. S. Detection of novel enzyme intermediates in PEP-utilizing enzymes. Arch. Biochem. Biophys. 433 47–58 (2005).">80</a></sup>. Another point in PETase, this enzyme retains the ancestral fold of the α/β-hydrolase. However, unlike homologous cutinases, its active site is more favourable for the entry of the PET polymer since the active site cleft is in a more open form than in cutinases, which This is traumatic since the PETase enzyme has a high ability to degrade PET, demonstrated in computational and experimental studies<sup><a aria-label="Reference 1" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR1" id="ref-link-section-d32394649e1784" title="Austin, H. P. et al. Characterization and engineering of a plastic-degrading aromatic polyesterase. Proc. Natl. Acad. Sci. USA 115, E4350–E4357 (2018).">1</a>,<a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR34" id="ref-link-section-d32394649e1787" title="Costa, C. H. S. et al. Assessment of the PETase conformational changes induced by poly(ethylene terephthalate) binding. Proteins Struct. Funct. Bioinform. 89, 1340–1352 (2021).">34</a>,<a aria-label="Reference 81" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR81" id="ref-link-section-d32394649e1790" title="Yoshida, S. et al. A bacterium that degrades and assimilates poly(ethylene terephthalate). Science 1979(351), 1196–1199 (2016).">81</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR83" id="ref-link-section-d32394649e1793" title="dos Santos, A. M., da Costa, C. H. S., Silva, P. H. A., Skaf, M. S. &amp; Lameira, J. Exploring the reaction mechanism of polyethylene terephthalate biodegradation through QM/MM approach. J. Phys. Chem. B 128, 7486–7499 (2024).">83</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR84" id="ref-link-section-d32394649e1793_1" title="Sevilla, M. E. et al. Degradation of PET bottles by an engineered Ideonella sakaiensis PETase. Polymers 15, 1779 (2023).">84</a>,<a aria-label="Reference 85" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR85" id="ref-link-section-d32394649e1796" title="Liu, B. et al. Protein crystallography and site-direct mutagenesis analysis of the poly(Ethylene terephthalate) hydrolase petase from Ideonella sakaiensis. ChemBioChem 19, 1471–1475 (2018).">85</a></sup>.</p><p>Our phylogenetic tree data indicate that the similarity between the genetic sequences of PETase and Cutinase is supported by enzymatic engineering<sup><a aria-label="Reference 1" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR1" id="ref-link-section-d32394649e1803" title="Austin, H. P. et al. Characterization and engineering of a plastic-degrading aromatic polyesterase. Proc. Natl. Acad. Sci. USA 115, E4350–E4357 (2018).">1</a></sup>. We demonstrated that a mutant version of the PETase enzyme, after introducing a double mutation inspired by the cutinase from <i>T. fusca</i><sup><a aria-label="Reference 1" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR1" id="ref-link-section-d32394649e1809" title="Austin, H. P. et al. Characterization and engineering of a plastic-degrading aromatic polyesterase. Proc. Natl. Acad. Sci. USA 115, E4350–E4357 (2018).">1</a></sup>, narrowed the binding site region for the polymer. and exhibited greater affinity for PET, resulting in more efficient polymer degradation. Thus, our study broadens the possibilities for future research in protein engineering, focusing on guided mutations based on conserved regions of bacterial and fungal cutinases, aiming to enhance the catalytic activity of PETase for the degradation of PET and potentially other polymers. We established the phylogenetic similarity between PETase and bacterial cutinases and conducted a phylogenetic analysis to map cutinases experimentally confirmed to degrade synthetic polymers. Table <a data-track="click" data-track-action="table anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Tab1">1</a> enumerates the polymers and the respective organisms either tested for degradation or demonstrated to exhibit this capability. The phylogenetic tree was examined to evaluate the proximity of these organisms and to determine the extent of similarity between the cutinase from <i>Fusarium oxysporum</i> and enzymes reported to possess polymer-degradation activity. Our findings reveal that, while the <i>F. oxysporum</i> cutinase does not exhibit a high degree of similarity to bacterial cutinases, it demonstrates considerable potential for degrading polymers beyond PET, as supported by experimental evidence (e.g., <i>Thermobifida fusca</i>, known to degrade PBAT). PET was utilized as the benchmark polymer for comparative analysis. We emphasized the structural resemblance between PETase and cutinase, particularly at their catalytic sites, illustrating that PET binds to the same region of the active site in both enzymes. This observation is significant, as both enzymes employ an identical catalytic mechanism involving the serine residue within their catalytic triad. The phylogenetic tree thus serves as a crucial framework for advancing research, facilitating the comparison of polymer-degrading organisms and investigating whether bacterial cutinases exhibit a broader capacity for synthetic polymer degradation compared to <i>F. oxysporum</i>.</p><p>The phylogenetic analysis shows that even though they are from different organisms, bacteria and fungi, these enzymes have structural similarity and, in addition, the cutinase from the bacteria <i>K. radiotolerans</i> has the same capacity to biodegrade PCL as does the cutinase from the organisms <i>M. roreri</i> , <i>F. solani</i>, <i>K. radiotolerans</i>, <i>A. oryzae</i>, <i>M. cinnamomea</i>, <i>H. insolens</i>, <i>F. vanettenii</i> <sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR12" id="ref-link-section-d32394649e1857" title="Vázquez-Alcántara, L., Oliart-Ros, R. M., García-Bórquez, A. &amp; Peña-Montes, C. Expression of a cutinase of Moniliophthora roreri with polyester and PET-plastic residues degradation activity. Microbiol. Spectr. 9(3), e00976-e1021 (2021).">12</a>,<a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR18" id="ref-link-section-d32394649e1860" title="Abokitse, K. et al. A novel actinobacterial cutinase containing a noncatalytic polymer-binding domain. Appl. Environ. Microbiol. 88,  e0152221. 
                  https://doi.org/10.1128/AEM.01522-21
                  
                 (2022).">18</a>,<a aria-label="Reference 43" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR43" id="ref-link-section-d32394649e1863" title="Duan, X. et al. High-level expression and characterization of a novel cutinase from Malbranchea cinnamomea suitable for butyl butyrate production. Biotechnol. Biofuels 10, 223 (2017).">43</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR55" id="ref-link-section-d32394649e1866" title="Santos-Beneit, F. et al. Screening enzymes that can depolymerize commercial biodegradable polymers: Heterologous expression of fusarium solani cutinase in Escherichia coli. Microorganisms 11, 328 (2023).">55</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR56" id="ref-link-section-d32394649e1866_1" title="Liang, X. &amp; Zou, H. Biotechnological application of cutinase: A powerful tool in synthetic biology. SynBio 1, 54–64 (2022).">56</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR57" id="ref-link-section-d32394649e1866_2" title="Huang, Q., Kimura, S. &amp; Iwata, T. Thermal embedding of Humicola insolens cutinase: A strategy for improving polyester biodegradation in seawater. Biomacromolecules 24, 5836–5846 (2023).">57</a>,<a aria-label="Reference 58" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR58" id="ref-link-section-d32394649e1869" title="Suzuki, M., Tachibana, Y. &amp; Kasuya, K. Biodegradability of poly(3-hydroxyalkanoate) and poly(ε-caprolactone) via biological carbon cycles in marine environments. Polym. J. 53, 47–66 (2021).">58</a></sup>. From a structural point of view, all cutinases from the 17 selected organisms have the Ser–His–Asp catalytic triad in common, in addition, they have at least one disulfide bond. Chemically and biologically, disulfide bonds can be correlated with the resistance capacity of cutinase to adverse conditions, from pH to temperature variations, showing that cutinase from different organisms that have adverse environmental conditions have different amounts of disulfide bonds between fungi and bacteria<sup><a aria-label="Reference 42" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR42" id="ref-link-section-d32394649e1873" title="Liu, Z. et al. Structural and functional studies of Aspergillus oryzae cutinase: Enhanced thermostability and hydrolytic activity of synthetic ester and polyester degradation. J. Am. Chem. Soc. 131, 15711–15716 (2009).">42</a>,<a aria-label="Reference 86" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR86" id="ref-link-section-d32394649e1876" title="Chen, S., Wu, Y., Su, L. &amp; Wu, J. Contribution of disulfide bond to the stability of Thermobifida fusca cutinase. Food Biosci. 27, 6–10 (2019).">86</a></sup>. The cutinase of the filamentous fungus <i>A. oryzae</i> has an additional disulfide bond, like the fungi <i>T. reesei</i> and <i>M. cinnamomea</i>, which is crucial to improving the thermal stability of <i>Ao</i>Cut in the PCL polymer biodegradation process<sup><a aria-label="Reference 42" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR42" id="ref-link-section-d32394649e1893" title="Liu, Z. et al. Structural and functional studies of Aspergillus oryzae cutinase: Enhanced thermostability and hydrolytic activity of synthetic ester and polyester degradation. J. Am. Chem. Soc. 131, 15711–15716 (2009).">42</a></sup>. Among the bacteria studied, the bacterium <i>K. radiotolerans</i> is the closest phylogenetically to fungi. Besides, it is also observed that bacterial cutinase has the affinity to biodegrade many more polymers other than PET—among the nine selected in this research—than fungal cutinase (Table <a data-track="click" data-track-action="table anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Tab1">1</a>).</p><p>Among the nine polymers studied, some fungal organisms are shown to be capable of biodegrading only one type of polymer. Fungi of the genus <i>Fusarium</i> are those most capable of biodegrading several synthetic polymers (such as PET, PBAT, PCL, PBS and PES) as well as <i>M. roreri</i>, which is capable of biodegrading three polymers (PET, PCL and PES) (Table <a data-track="click" data-track-action="table anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Tab1">1</a>). Phylogenetic analysis and MD simulations can be helpful for designing more active and thermostable proteins capable of degrading these polymers.</p><h3 class="c-article__sub-heading" id="Sec10">Biological similarity between PETase and <i>Fo</i>Cut</h3><p>The Yoshida group showed that the bacterium <i>Ideonella sakaiensis</i> expresses two enzymes that are involved in the biodegradation of PET, the enzyme PETase and MHETase<sup><a aria-label="Reference 81" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR81" id="ref-link-section-d32394649e1930" title="Yoshida, S. et al. A bacterium that degrades and assimilates poly(ethylene terephthalate). Science 1979(351), 1196–1199 (2016).">81</a>,<a aria-label="Reference 87" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR87" id="ref-link-section-d32394649e1933" title="Tanasupawat, S., Takehana, T., Yoshida, S., Hiraga, K. &amp; Oda, K. Ideonella sakaiensis sp. nov, isolated from a microbial consortium that degrades poly(ethylene terephthalate). Int. J. Syst. Evol. Microbiol. 66, 2813–2818 (2016).">87</a></sup>. This PETase enzyme catalyzes the biodegradation reaction of PET by converting it into mono(2-hydroxyethyl) terephthalic acid (MHET) and releasing bis-(hydroxyethyl) terephthalate (BHET) and terephthalic acid (TPA) as products.</p><p>In the proposed catalytic mechanism of polymer biodegradation by PETase, the carbonyl group attached to the first benzene ring is oriented toward the centre of the polymer-binding cleft, where the catalytic triad Ser160, Asp206, and His237 is located<sup><a aria-label="Reference 2" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR2" id="ref-link-section-d32394649e1940" title="Han, X. et al. Structural insight into catalytic mechanism of PET hydrolase. Nat. Commun. 8, 2106 (2017).">2</a>,<a aria-label="Reference 3" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR3" id="ref-link-section-d32394649e1943" title="Joo, S. et al. Structural insight into molecular mechanism of poly(ethylene terephthalate) degradation. Nat. Commun. 9, 382 (2018).">3</a>,<a aria-label="Reference 88" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR88" id="ref-link-section-d32394649e1946" title="Palm, G. J. et al. Structure of the plastic-degrading Ideonella sakaiensis MHETase bound to a substrate. Nat. Commun. 10, 1717 (2019).">88</a></sup>. The essential residue Ser160 is responsible for the nucleophilic attack of the enzyme on the substrate and then the ester bond is cleaved with the participation of a water molecule, similar to the mechanism of the Cutinase enzyme<sup><a aria-label="Reference 2" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR2" id="ref-link-section-d32394649e1950" title="Han, X. et al. Structural insight into catalytic mechanism of PET hydrolase. Nat. Commun. 8, 2106 (2017).">2</a>,<a aria-label="Reference 89" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR89" id="ref-link-section-d32394649e1953" title="Longhi, S. &amp; Cambillau, C. Structure-activity of cutinase, a small lipolytic enzyme. Biochim. Biophys. Acta (BBA)Mol. Cell Biol. Lipids 1441, 185–196 (1999).">89</a></sup>.</p><p>Here, we performed global align by ClustalW in MEGA 11<sup><a aria-label="Reference 36" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR36" id="ref-link-section-d32394649e1960" title="Tamura, K., Stecher, G. &amp; Kumar, S. MEGA11: Molecular evolutionary genetics analysis version 11. Mol. Biol. Evol. 38, 3022–3027 (2021).">36</a></sup> software of the enzyme PETase (PDB ID: 6EQE<sup><a aria-label="Reference 1" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR1" id="ref-link-section-d32394649e1964" title="Austin, H. P. et al. Characterization and engineering of a plastic-degrading aromatic polyesterase. Proc. Natl. Acad. Sci. USA 115, E4350–E4357 (2018).">1</a></sup>) and Cutinase (PDB ID: 5AJH<sup><a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR11" id="ref-link-section-d32394649e1968" title="Dimarogona, M. et al. Structural and functional studies of a Fusarium oxysporum cutinase with polyethylene terephthalate modification potential. Biochim. Biophys. Acta BBA General Subjects 1850, 2308–2317 (2015).">11</a></sup>) in order to identify the main conserved and semi-conserved residues (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig2">2</a>, panel A). Despite being from extremely different organisms, we identified that these enzymes have similarities in their structure, which provides insights into their ability to biodegrade the PET polymer. The Cutinase from <i>F. oxysporum</i> also has a catalytic triad of Ser121, Asp176 and His189 residues. Among the conserved residues are the catalytic residues of the <i>Is</i>PETase triad Ser160, Asp176 and His237 being Ser121, Asp176 and His189 for <i>Fo</i>Cut (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig2">2</a>, panel B and C).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Fig. 2" id="figure-2"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig2">Fig. 2</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41598-024-84718-0/figures/2" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig2_HTML.png?as=webp" type="image/webp"><img alt="figure 2" aria-describedby="Fig2" height="685" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig2_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p>Structural comparison between the enzyme <i>Is</i>PETase (PDB ID: 6EQE) and <i>Fo</i>Cut (PDB ID: 5AJH). (<b>A</b>) Non-conserved, conserved and semi-conserved residues observed in the global alignment by ClustalW, (<b>B</b>) overlap of the enzymes with the structure of the PET tetramer present in the catalytic site of the enzyme, (<b>C</b>) region of the catalytic triad in comparison for both enzymes, binding pocket for enzyme (<b>D</b>) <i>Fo</i>Cut and (<b>E</b>) <i>Is</i>PETase.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 2" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure2 Full size image" data-track-label="button" href="/articles/s41598-024-84718-0/figures/2" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>Carrying out an analysis on the overlap of the <i>Is</i>PETase and <i>Fo</i>Cut proteins, we noticed that the active site region is the same for both (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig2">2</a>, panel D and E), which agrees with a previous contribution<sup><a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR34" id="ref-link-section-d32394649e2046" title="Costa, C. H. S. et al. Assessment of the PETase conformational changes induced by poly(ethylene terephthalate) binding. Proteins Struct. Funct. Bioinform. 89, 1340–1352 (2021).">34</a></sup>. Thus, we observed that the PET using the spatial coordinates of another work on the catalytic site of the <i>Is</i>PETase enzyme<sup><a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR34" id="ref-link-section-d32394649e2054" title="Costa, C. H. S. et al. Assessment of the PETase conformational changes induced by poly(ethylene terephthalate) binding. Proteins Struct. Funct. Bioinform. 89, 1340–1352 (2021).">34</a></sup>, when superimposed, is in the same region in <i>Fo</i>Cut.</p><h3 class="c-article__sub-heading" id="Sec11">Structural analysis of the Cutinase protein from <i>Fusarium oxysporum</i>
</h3><p>MD simulations with a duration of 1 μs (1000 ns) enabled us to understand the enzyme’s behaviour without the presence of ligands in the active site. We observed that the enzyme hightly stable throughout the simulation, with an RMSD value of 1.09 Å (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig3">3</a>, panel A). A high number of hydrogen bonds between the protein residues was observed. Besides, the enzyme has two disulfide bridges between residues Cys172–Cys179 and Cys32–Cys110 as also observed in the homologous structure of Cut from <i>Fusarium solani</i><sup><a aria-label="Reference 11" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR11" id="ref-link-section-d32394649e2078" title="Dimarogona, M. et al. Structural and functional studies of a Fusarium oxysporum cutinase with polyethylene terephthalate modification potential. Biochim. Biophys. Acta BBA General Subjects 1850, 2308–2317 (2015).">11</a>,<a aria-label="Reference 89" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR89" id="ref-link-section-d32394649e2081" title="Longhi, S. &amp; Cambillau, C. Structure-activity of cutinase, a small lipolytic enzyme. Biochim. Biophys. Acta (BBA)Mol. Cell Biol. Lipids 1441, 185–196 (1999).">89</a>,<a aria-label="Reference 90" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR90" id="ref-link-section-d32394649e2084" title="Matak, M. Y. &amp; Moghaddam, M. E. The role of short-range Cys171–Cys178 disulfide bond in maintaining cutinase active site integrity: A molecular dynamics simulation. Biochem. Biophys. Res. Commun. 390, 201–204 (2009).">90</a></sup>. Furthermore, the PETase enzyme presents the same disulfide bonds in the <i>F. solani</i> cutinase enzyme responsible for maintaining the stability of the active site<sup><a aria-label="Reference 1" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR1" id="ref-link-section-d32394649e2091" title="Austin, H. P. et al. Characterization and engineering of a plastic-degrading aromatic polyesterase. Proc. Natl. Acad. Sci. USA 115, E4350–E4357 (2018).">1</a>,<a aria-label="Reference 90" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR90" id="ref-link-section-d32394649e2094" title="Matak, M. Y. &amp; Moghaddam, M. E. The role of short-range Cys171–Cys178 disulfide bond in maintaining cutinase active site integrity: A molecular dynamics simulation. Biochem. Biophys. Res. Commun. 390, 201–204 (2009).">90</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Fig. 3" id="figure-3"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig3">Fig. 3</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41598-024-84718-0/figures/3" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig3_HTML.png?as=webp" type="image/webp"><img alt="figure 3" aria-describedby="Fig3" height="359" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig3_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p>Structural analysis of the <i>Fo</i>Cut protein in 1 μs (1000 ns) simulation time. (<b>A</b>) RMSD, (<b>B</b>) RMSF and (<b>C</b>) overlapping of the centroids obtained through clustering highlighting the Catalytic Loop region based on Cluster ID and D) sequence of aminoacid described by regions.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 3" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure3 Full size image" data-track-label="button" href="/articles/s41598-024-84718-0/figures/3" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>The RMSF analysis show values below 3.0 Å (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig2">2</a>, panel B), indicating that the amino acid residues exhibited minimal fluctuations during the simulation, particularly in the catalytic site region. Only fluctuations in the N- and C-terminal regions were identified, such as Asn28, Ser29, and Gly30, with values greater than 1.5 Å. We also noted that Leu82, located in the alpha helix region at the top of the catalytic site, presented a fluctuation of 1.5 Å. This fluctuation was influenced by the absence of ligands at the site.</p><p>We noted that during the simulation, the Cut enzyme remained predominantly at 52.6% of the simulation time, with the active site region most exposed in the clustering analysis demonstrated by volume and surface region (Table <a data-track="click" data-track-action="table anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Tab2">2</a>). It is noted that the region of the loop referring to the active site of the enzyme between residues 172–192 has movement (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig3">3</a>, panel C). This movement is seen with the overlap of the clusters from the highest to the lowest occupancy value (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig3">3</a> and Table S1). Thus, the absence of a substrate in the active site of the enzyme causes it to remain in an open state.</p><div class="c-article-table" data-container-section="table" data-test="inline-table" id="table-2"><figure><figcaption class="c-article-table__figcaption"><b data-test="table-caption" id="Tab2">Table 2 Binding pocket grid properties by Cluster ID calculated by ProteinPlus<sup><a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR91" id="ref-link-section-d32394649e2156" title="Fährrolfes, R. et al. ProteinsPlus: A web portal for structure analysis of macromolecules. Nucl. Acids Res. 45, W337–W343 (2017).">91</a>,<a data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="#ref-CR92" id="ref-link-section-d32394649e2156_1" title="Schöning-Stierand, K. et al. ProteinsPlus: Interactive analysis of protein–ligand binding interfaces. Nucl. Acids Res. 48, W48–W53 (2020).">92</a>,<a aria-label="Reference 93" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR93" id="ref-link-section-d32394649e2159" title="Graef, J., Ehrt, C. &amp; Rarey, M. Binding site detection remastered: Enabling fast, robust, and reliable binding site detection and descriptor calculation with DoGSite3. J. Chem. Inf. Model. 63, 3128–3137 (2023).">93</a></sup> using GeoMine<sup><a aria-label="Reference 94" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR94" id="ref-link-section-d32394649e2163" title="Diedrich, K., Graef, J., Schöning-Stierand, K. &amp; Rarey, M. GeoMine: Interactive pattern mining of protein–ligand interfaces in the Protein Data Bank. Bioinformatics 37, 424–425 (2021).">94</a></sup>.</b></figcaption><div class="u-text-right u-hide-print"><a aria-label="Full size table 2" class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" href="/articles/s41598-024-84718-0/tables/2" rel="nofollow"><span>Full size table</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec12">Molecular Dynamics </h3><p>MD simulations were performed in triplicate to evaluate the obtained data statistically. The initial analysis focused on the behavior of the ligands in the enzyme’s active site. We observed that PET, PHBH, PBS, PBAT, and PCL remained in all replicates, as shown in Table <a data-track="click" data-track-action="table anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Tab3">3</a>. Therefore, only the complexes with these ligands, which remained stable across all replicates, were selected for the subsequent structural analyses.</p><div class="c-article-table" data-container-section="table" data-test="inline-table" id="table-3"><figure><figcaption class="c-article-table__figcaption"><b data-test="table-caption" id="Tab3">Table 3 Permanency of polymers in the catalytic site of the <i>Fo</i>Cut enzyme in each of the complexes and the percentage of polymers remaining in the three replicates.</b></figcaption><div class="u-text-right u-hide-print"><a aria-label="Full size table 3" class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" href="/articles/s41598-024-84718-0/tables/3" rel="nofollow"><span>Full size table</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>The polymers PLA, PHB, PHBV and PES showed few strong interactions in the enzyme’s active site, which resulted in the ligands leaving the active site during the simulation. The enzyme cutinase from <i>T. alba</i><sup><a aria-label="Reference 19" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR19" id="ref-link-section-d32394649e2612" title="Kitadokoro, K. et al. Structural insights into the unique polylactate-degrading mechanism of Thermobifida alba cutinase. FEBS J. 286, 2087–2098 (2019).">19</a></sup> biodegrades the PLA polymer. The cutinase of this organism, although they have the catalytic triad in common, the structure of the protein is not similar to that of the fungus <i>F. oxysporum</i>, presenting only one disulfide bond in contrast to two disulfide bonds in <i>Fo</i>Cut, which play a fundamental role in the stability of the protein and maintaining your active website. The polymer PHB and PHBH do not have data on biodegradation activity among the organisms used. In contrast to PHBH, the PHB polymer did not demonstrate strong interactions during the simulations and did not remain in the enzyme’s active site. The PHBV polymer showed only a single interaction with the Thr51 residue.</p><p>In RMSD analyses (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig4">4</a>), considering the last minimization step as reference to all replicates, PBS and PCL exhibited considerable fluctuations in some replicates due to the instability in the structure of the ligands. The RMSD value in replicate 3 of 3.38 Å (± 0.67) for the complex with PCL and 3.69 Å (± 0.97) for PBS is due to the structural change of the polymers, as their structures are mainly involved through single bonds between the C–C atoms and C–O with very few double bonds, the sigma bonds rotate, causing the ligand to change its conformation easily quickly. During the simulation, the structures of these polymers rotate entirely, losing their stretched position initially obtained by the docking stage. This is observed by the RMSD values referring solely to these two polymers, considering only the heavy atoms (C and O), in this replicates being 4.09 Å (± 1.38) and 6.53 Å (± 1.38) for PCL and PBS respectively. This is justifiable due to the chemical structure of these polymers.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Fig. 4" id="figure-4"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig4">Fig. 4</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41598-024-84718-0/figures/4" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig4_HTML.png?as=webp" type="image/webp"><img alt="figure 4" aria-describedby="Fig4" height="539" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig4_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>Root mean square deviation (RMSD) for the backbone atoms (C, O, Ca, and N) plots obtained from <i>Fo</i>Cut complexes for all replicates. (<b>A</b>) PET, (<b>B</b>) PHBH, (<b>C</b>) PBS, (<b>D</b>) PBAT and (<b>E</b>) PCL complexes.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 4" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure4 Full size image" data-track-label="button" href="/articles/s41598-024-84718-0/figures/4" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>The PHBH structure remained stable across all replicates, which can be attributed to its smaller size than to the other selected polymers. We observed that the structural integrity of the PET complexes in replicates 1 and 3 was maintained throughout the 100 ns of the MD simulations. In replicate 2, the increased RMSD value was due to a conformational change in the PET structure, a behaviour that has been observed in previous studies.<sup><a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR34" id="ref-link-section-d32394649e2668" title="Costa, C. H. S. et al. Assessment of the PETase conformational changes induced by poly(ethylene terephthalate) binding. Proteins Struct. Funct. Bioinform. 89, 1340–1352 (2021).">34</a></sup> Similarly, in the PBS complex, the structural change in replicate 3 was responsible for the increased RMSD value.The PBAT complex showed stability across all three replicates. This polymer contains an aromatic ring in its structure, similar to PET. This structural similarity is a crucial factor in interactions with residues at the active site, which may explain the stability of this polymer in the enzyme. The system with PBS exhibited high movement in the RMSD graph due to its structure being predominantly composed of carbon–carbon sigma bonds, which undergo rotations. These conformational changes account for the fluctuations observed in the RMSD graph. In additional RMSD analyses, we observed the behaviour of only the ligand and protein in each of the studied systems (Figures <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-024-84718-0#MOESM1">S2</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-024-84718-0#MOESM1">S3</a>). This analysis revealed that the protein remained stable throughout the simulation time, regardless of the ligand, across all replicates. This demonstrates that the desviations in the RMSD of the complexes are directly attributable to the ligands. Furthermore, an additional MD simulation analysis of only the protein further confirms the stability of this enzyme during the simulation.</p><p>All systems demonstrated that the enzyme remained stable, with fluctuations not exceeding 3.0 Å (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig5">5</a>). This is consistent with the RMSD analysis data, where we showed that the instability of a system is directly related to the ligand. The RMSF data indicate a similar fluctuation profile across all six best systems and across all replicates. A slight fluctuation was observed in the system with PCL in repetition 2, resulting in more significant fluctuations than the others.This fluctuation is attributed to a structural change in the ligand in this replicate, leading to weaker interactions and consequently greater fluctuation. None of the systems showed significant fluctuation for the catalytic triad residues Ser121, His189, and Asp176.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Fig. 5" id="figure-5"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig5">Fig. 5</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41598-024-84718-0/figures/5" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig5_HTML.png?as=webp" type="image/webp"><img alt="figure 5" aria-describedby="Fig5" height="573" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig5_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p>Root mean square fluctuation (RMSF) plots obtained from <i>Fo</i>Cut complexes for each Replicate and (<b>A</b>) tridimensional structure of <i>Fo</i>Cut highliting the loop regions, N–Terminal and C–Terminal. (<b>B</b>) PET, (<b>C</b>) PHBH, (<b>D</b>) PBS, (<b>E</b>) PBAT and (<b>F</b>) PCL complexes.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 5" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure5 Full size image" data-track-label="button" href="/articles/s41598-024-84718-0/figures/5" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>We applied the clustering methodology to each replicate individually, as well as to the entire set, to evaluate the ligand conformation in each replicate. We determined how the structure was maintained by identifying the most populous cluster (Table S2). The PET analysis demonstrated that in replicates 1 and 3 (with simulation time fractions of 43.5% and 50.5%, respectively). The PET presented a similar structure and in replicate 2 (with 46%) the single bonds between carbons rotated as demonstrated in another Molecular Dynamics study with PET complexed to PETase<sup><a aria-label="Reference 34" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR34" id="ref-link-section-d32394649e2731" title="Costa, C. H. S. et al. Assessment of the PETase conformational changes induced by poly(ethylene terephthalate) binding. Proteins Struct. Funct. Bioinform. 89, 1340–1352 (2021).">34</a></sup>. Considering the 3 replicates together in total, the structure obtained is similar to that obtained in replicates 1 and 3 with a time fraction of 68.9% (Figure <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-024-84718-0#MOESM1">S5</a>). For complexes with PBAT, the replicates did not show much similarity, even with the similar RMSD between the replicates. Each structure did not remain stable in the simulation with values below 40% of the simulation time fraction, with the total pose being 36.9%. The aromatic ring present in both PBAT and PET carries out pi interactions with protein residues in the catalytic site.</p><p>The interactions between the ligands with the amino acids of the catalytic site of the <i>Fo</i>Cut enzyme were computed through fingerprint analysis with the ProLif program, demonstrating that the main interactions that are carried out for the complex with PET in the three replicates are Leu82, Ala86, Leu183, Val185, Leu190. The first replicate presented more interactions with an occurrence greater than 0.5 with the residues of the site, which directly explains the stable RMSD during the simulation time.</p><p>Moreover, residues Tyr120 and His189 perform pi interactionts with the aromatic rings of both ligands (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig6">6</a>, panel B and D). For complexes with PBAT, the main residues that interact in the three replicates are Tyr120, Val185, Leu190. The same situation is present in PET, the most stable system in RMSD, which is consequently the one that performs the most fingerprint interactions in complexes with PBAT. We also highlight that both complexes are formed mainly through hydrophobic, van der Waals and pi interactions. Both interact with the critical residue His189, corresponding to a catalytic residue. Furthermore, the catalytic Serine in Cutinase from <i>T. fusca</i> is involved in the PBAT degradation mechanism<sup><a aria-label="Reference 22" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR22" id="ref-link-section-d32394649e2753" title="Yang, Y. et al. Complete bio-degradation of poly(butylene adipate-co-terephthalate) via engineered cutinases. Nat. Commun. 14, 1645 (2023).">22</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Fig. 6" id="figure-6"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig6">Fig. 6</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41598-024-84718-0/figures/6" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig6_HTML.png?as=webp" type="image/webp"><img alt="figure 6" aria-describedby="Fig6" height="320" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig6_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p>Interaction fingerprints (IFP) of <i>Fo</i>Cut complexes, including interaction information—and the occurrence of each interaction—as hydrophobic (HY), van der Waals (vdW) and PiStacking character interactions for A–B) PET and PBAT C–D) complexes, respectively.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 6" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure6 Full size image" data-track-label="button" href="/articles/s41598-024-84718-0/figures/6" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>Complexes with PHBH carry out interactions with residues Ser43, Leu82, Gln122, Leu183, Val185, Leu190 in all replicate (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig7">7</a>). However, the catalytic residue His189 in two replicates has an occurrence value greater than 0.5, in replicate 2 this value is the opposite (less than 0.5), but in the total calculation, this contribution appears as a strong interaction as well as Glu45 and Tyr120. Furthermore, in replicate 1, another catalytic residue interacts with the ligand, Ser121. The main interactions forming these complexes with PHBH are van der Waals and hydrophobic, with only a single hydrogen bond with the Ser43 residue.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Fig. 7" id="figure-7"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig7">Fig. 7</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41598-024-84718-0/figures/7" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig7_HTML.png?as=webp" type="image/webp"><img alt="figure 7" aria-describedby="Fig7" height="690" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig7_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-7-desc"><p>Interaction fingerprints (IFP) of <i>Fo</i>Cut complexes, including interaction information—and the occurrence of each interaction—as hydrophobic (HY), van der Waals (vdW) and Hydrogen bond donor (HBD), Hydrogen bond Acceptor (HBA) and Anionic (AN) character interactions for A) PHBH, B) PBS and C) PCL complexes, respectively.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 7" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure7 Full size image" data-track-label="button" href="/articles/s41598-024-84718-0/figures/7" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>Complexes with PBS are mainly formed by hydrophobic van der Waals interactions (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig7">7</a>). However, the polymer also performs an anionic interaction (Arg55) and two hydrogen bonds (Ser43, Arg55), with the Arg55 residue only in replicate 1. The residues Ser43, Leu82, Tyr120, Ser121, Gln122, Leu183, Val185, Leu190 perform interactions of van der Waals and hydrophobic in the 3 replicates. Furthermore, residue His189 appears in two replicates and is also a essential interaction residue. In this way, we observed that PBS is one of the best ligands due to its high number of interactions and the high occurrence value of these interactions that the ligand performs with residues of the catalytic site.</p><p>PCL interacts with residues Ser43, Leu82, Tyr120, Val185, Leu190 in the three replicates to form PCL–<i>Fo</i>Cut complexes (Figure <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-024-84718-0#MOESM1">S5</a>, panel C). The interactions involved in forming of this complex are predominantly van der Waals and hydrophobic, but the ligand performs hydrogen interactions with residues Ser43 and Leu82 in replicate 2. This demonstrates that during this replicate, the ligand was closer to these residues than in the other replicates, which justifies its having the lowest RMSD value. The absence of strong interactions in replicate 3 causes the ligand to carry out more movements at the site than a system with more entries, which justifies the high RMSD value of this replicate. PCL complex in <i>Mr</i>Cut can be biodegraded by the enzyme<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR12" id="ref-link-section-d32394649e2824" title="Vázquez-Alcántara, L., Oliart-Ros, R. M., García-Bórquez, A. &amp; Peña-Montes, C. Expression of a cutinase of Moniliophthora roreri with polyester and PET-plastic residues degradation activity. Microbiol. Spectr. 9(3), e00976-e1021 (2021).">12</a></sup>. Our study identified the affinity of the ligand to interact with residues from the <i>Fo</i>Cut enzyme site of another organism. This demonstrates that Cutinase enzymes can function as biocatalysts in polymer degradation reactions regardless of the organism. This information provides insights into the critical amino acid residues of the catalytic site that are involved in forming the enzyme–substrate complex.</p><p>Through the interactions that polymers carry out in the active site of the enzyme, the key residues Ser43, Val185 and Leu190 are in the active site and contribute for the formation of all polymer–<i>Fo</i>Cut complexes studied. Where the Ser43 residue, as well as Gln122, is present in the region of the oxyanion hole<sup><a aria-label="Reference 95" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR95" id="ref-link-section-d32394649e2837" title="Magalhães, R. P., Cunha, J. M. &amp; Sousa, S. F. Perspectives on the role of enzymatic biocatalysis for the degradation of plastic PET. Int. J. Mol. Sci. 22, 11257 (2021).">95</a></sup>.</p><p>In the clustering analysis (Figure <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-024-84718-0#MOESM1">S6</a>), PHBH presented similar structures in replicates 1 and 3 and in the total structure (considering all 3 replicates together). This demonstrates that the ligand remained predominantly in a similar conformation in the enzyme’s active site during all simulations (73.5%). The same is observed in the PBS structures in replicates 2 and total. This structure of the second replicate appears stable structure during 70.1% of the total simulation time of the three replicates together. For the system with PCL, the majority structure of replicate 3 is identical to the total structure of the replicates, however, replicates 1 and 2 are completely different, showing that the ligand can present these other two conformations due to its high number of sigma bonds that rotate during simulation. This conformational change demonstrates that the conformation of replicate 2 (with 54.3% presence during the simulation time) performs strong interactions at the site, which demonstrates that it is a critical conformation that performs interactions with the catalytic residue Ser121 in addition to interaction with Leu190, which is a residue key in protein stabilization through the alpha helix<sup><a aria-label="Reference 95" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR95" id="ref-link-section-d32394649e2847" title="Magalhães, R. P., Cunha, J. M. &amp; Sousa, S. F. Perspectives on the role of enzymatic biocatalysis for the degradation of plastic PET. Int. J. Mol. Sci. 22, 11257 (2021).">95</a></sup>.</p><p>The analysis of the distance between the polymers, considering the representative pose of the clustering carried out with the three simulations together (Total Cluster, Figures <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-024-84718-0#MOESM1">S5</a> and <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-024-84718-0#MOESM1">S6</a>) and the catalytic residues demonstrated that in some systems the catalytic Serine is considerably close to the carbon of the ester group in the polymer structure, enabling nucleophilic attack to cleave the polymer structure. The PBS polymers presented the smallest distance from the carbon of the ester group and Ser121. The PET polymer during the simulation approached Ser121, moving from an initial distance of 6.4 to 4.1 Å (Figure <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-024-84718-0#MOESM1">S1</a> and Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig8">8</a>). PBAT presented a distance of 4.2 Å and PCL with 4.8 Å. The rotation of the C–C single bonds caused the PHBH polymer to lose interaction with residue Ser121, however, in replicate 1 this interaction is maintained firmly during the simulation. This shows that no matter how much the conformational change occurred in the total replicate, Ser121 aims to interact with this carbon of the Ester group of the polymer so that the nucleophilic attack occurs. This information serves as a starting point for elucidating the reaction mechanism of polymers biodegraded by <i>Fo</i>Cut and also of other Cutinase from organisms.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-container-section="figure" data-test="figure" data-title="Fig. 8" id="figure-8"><figure><figcaption><b class="c-article-section__figure-caption" data-test="figure-caption-text" id="Fig8">Fig. 8</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-action="view figure" data-track-label="image" href="/articles/s41598-024-84718-0/figures/8" rel="nofollow"><picture><source srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig8_HTML.png?as=webp" type="image/webp"><img alt="figure 8" aria-describedby="Fig8" height="345" loading="lazy" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_Fig8_HTML.png" width="685"/></source></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-8-desc"><p>Total cluster centroid of (<b>A</b>) PET, (<b>B</b>) PHBH, (<b>C</b>) PBS, (<b>D</b>) PBAT and (<b>E</b>) PCL polymers in the catalytic site in relation to the distance from the <i>Fo</i>Cut catalytic triad.</p></div></div><div class="u-text-right u-hide-print"><a aria-label="Full size image figure 8" class="c-article__pill-button" data-test="article-link" data-track="click" data-track-action="view figure" data-track-dest="link:Figure8 Full size image" data-track-label="button" href="/articles/s41598-024-84718-0/figures/8" rel="nofollow"><span>Full size image</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec13">Polymers complexes binding affinities</h3><p>The binding free energy data demonstrate that PET is the polymer with the highest affinity for the Cutinase enzyme (Table <a data-track="click" data-track-action="table anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Tab4">4</a>). Experimental methods, such as microscopy, have shown and confirmed the ability of the fungus <i>Fusarium oxysporum</i> to biodegrade PET-type polymers through the Cutinase enzyme<sup><a aria-label="Reference 54" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR54" id="ref-link-section-d32394649e2922" title="Cognigni, F. et al. Exploring the infiltrative and degradative ability of Fusarium oxysporum on polyethylene terephthalate (PET) using correlative microscopy and deep learning. Sci. Rep. 13, 22987 (2023).">54</a></sup>. Among the nine polymers studied, five exhibit affinity for the Cutinase enzyme, with PET having the most escellent affinity at -34.26 (± 5.51) kcal/mol. In addition to PET, the polymers PHBH, PBS, PBAT, and PCL also showed high binding affinbity with the <i>Fo</i>Cut enzyme, with values of − 18.44 (± 4.41), − 29.71 (± 5.90), − 22.78 (± 5.36), and − 22.26 (± 6.56) kcal/mol, respectively. In 2021, it was discovered that Cutinase from <i>M. roreri</i> can biodegrade PES, PCL and PET polymers with degradation percentages of 59%, 43% and 31%, respectively, after 21 days<sup><a aria-label="Reference 12" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR12" id="ref-link-section-d32394649e2933" title="Vázquez-Alcántara, L., Oliart-Ros, R. M., García-Bórquez, A. &amp; Peña-Montes, C. Expression of a cutinase of Moniliophthora roreri with polyester and PET-plastic residues degradation activity. Microbiol. Spectr. 9(3), e00976-e1021 (2021).">12</a></sup>. This demonstrates that <i>Mr</i>Cut1 can biodegrade polymers other than PET, with a greater affinity for the PES polymer. In the case of <i>Fo</i>Cut, the PES polymer showed little affinity and did not remain bound for the entire simulation, dissociating at approximately 80 ns. Therefore, the binding free energy value of -13.57 (± 8.63) kcal/mol suggests that further studies are needed to explore the potential of <i>Fo</i>Cut to degrade this polymer. However, PCL and PET polymers showed a binding affinity with the <i>Fo</i>Cut enzyme. <i>Kr</i>Cut has demonstrated the ability to biodegrade PCL polymers and the synthetic polyester poly(1,3-propylene adipate)<sup><a aria-label="Reference 18" data-test="citation-ref" data-track="click" data-track-action="reference anchor" data-track-label="link" href="/articles/s41598-024-84718-0#ref-CR18" id="ref-link-section-d32394649e2953" title="Abokitse, K. et al. A novel actinobacterial cutinase containing a noncatalytic polymer-binding domain. Appl. Environ. Microbiol. 88,  e0152221. 
                  https://doi.org/10.1128/AEM.01522-21
                  
                 (2022).">18</a></sup>. As with <i>Kr</i>Cut, <i>Fo</i>Cut also showed binding affinity with the PCL polymer, demonstrating that even though they are from different organisms, the <i>Mr</i>Cut1 and <i>Kr</i>Cut enzymes have the same affinity for PCL.</p><div class="c-article-table" data-container-section="table" data-test="inline-table" id="table-4"><figure><figcaption class="c-article-table__figcaption"><b data-test="table-caption" id="Tab4">Table 4 Binding free energy values (from MM/GBSA method) for the <i>Fo</i>Cut complexes.</b></figcaption><div class="u-text-right u-hide-print"><a aria-label="Full size table 4" class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" href="/articles/s41598-024-84718-0/tables/4" rel="nofollow"><span>Full size table</span><svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-chevron-right-small" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></div></figure></div><p>The system with PET exhibited the most favorable residues, which is consistent with its highest affinity value. The smaller the ΔG value and the greater the number of interactions, the more favourable residues are involved. The main residues contributing to the ∆G<sub>total</sub> value in the three replicates for PET are Leu82, Ala86, Leu183, Val185, and Leu190. In the complexes with PHBH, the residues that consistently contributed across the three replicates are Ser43, Leu82, Gln122, Leu183, and Leu185 (Figure <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-024-84718-0#MOESM1">S7</a>). In the complex with PBS, most of the central residues contribute to the three replicates being Gly42, Ser43, Leu82, Tyr120, *Ser121, Gln122, Leu183, Val185 and Leu190. In the system with PBAT, only three main residues contribute to all replicates, Leu82, Val185, and Leu190. In the last system, the complexes with PCL, only two residues appear, contributing to all replicates Val185 and Leu190.</p><p>
                           The amino acid residues Val185 and Leu190 were favorable in all replicates across all five systems (Figure <a data-track="click" data-track-action="supplementary material anchor" data-track-label="link" href="/articles/s41598-024-84718-0#MOESM1">S7</a>), demonstrating their key role in complex formation through van der Waals and hydrogen interactions. These residues are part of the catalytic loop region (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig3">3</a>, panel C), confirming its importance in maintaining the polymer in the active site. These common residues in all systems across the three replicates indicate that all polymers remained in the same region of the enzyme’s active site. Additionally, the Leu82 residue, located in another loop region, is highlighted as crucial for maintaining the stability of the polymer complexes, as this region exhibits high fluctuation in the absence of polymers (Fig. <a data-track="click" data-track-action="figure anchor" data-track-label="link" href="/articles/s41598-024-84718-0#Fig3">3</a>). Furthermore, the Ser121 catalytic residues were favorable for at least one replicate of the complexes performing van der Waals interaction. As this movement only happens in these replicates, through fingerprint analysis we were able to visualize that some interactions carried out in these replicates are not carried out in the others due to the unique rotational movement of the single bonds in the polymer structure during these simulations. This character is not shown in the other replicates, which suggests that it is unique and particular to these replicates in isolation, as in replicate 2 of the complex with PBS this residue contributes favorably.</p></div></div></section><section data-title="Conclusion"><div class="c-article-section" id="Sec14-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec14">Conclusion</h2><div class="c-article-section__content" id="Sec14-content"><p>Our phylogenetic analysis provided important insights into the biological similarity of enzymes that biodegrade polymers in different types of organisms. It demonstrated that although F. oxysporum cutinase is not closely related to <i>Kr</i>Cut and <i>Mr</i>Cut, they all can biodegrade polymers. Thus, these discoveries show that even though bacteria or fungi produce them, these enzymes have this biological similarity of biodegrading polymers. Therefore, our findings provide the critical information that <i>Fo</i>Cut is like Cutinase from several organisms such as <i>Aspergillus oryzae</i>, <i>Moniliophthora roreri</i>, <i>Paraphoma</i> sp., <i>Humicola insolens</i>, <i>Saccharomonospora viridis</i>, <i>Thermobifida fusca</i>, <i>Fusarium solani</i>. Furthermore, an overlap analysis demonstrate that both <i>Fo</i>Cut and <i>Is</i>PETase enzymes have the catalytic triad in the same region, which also justifies the similarity of the region in which PET was inserted through Molecular Docking, being similar to the <i>Is</i>PETase enzyme site region as already demonstrated in the literature. Furthermore, our data demonstrate that the polymers PET, PHBH, PBS, PBAT, and PCL can bind into Cutinase enzyme from <i>Fusarium oxysporum</i>, demonstrating the possibility of biodegradation activity of these new polymers in addition to PET (− 34.26 ± 5.51), which has already been reported in the literature as a polymer degraded by this enzyme, with binding free energy values equal to − 18.44 (± 4.41), − 29.71 (± 5.90), − 22.78 (± 5.36), and − 22.26 (± 6.56) kcal/mol, respectively. We therefore demonstrated, through computational simulations, the stability through the intermolecular interactions of these polymers in the active site, highlighting the importance of key residues such as Val185 and Leu190 that are involved in an opening loop—also called catalytic loop—responsible for causing the polymers remain in the active site. We highlight the Ser43 and Gln122 exclusion as one of the key residues important to coordinate the ester group region and keep the ligands close to the catalytic triad through the oxyanion hole. Furthermore, we highlight the polymer binding mode through predominantly vdW and hydrophobic interactions to form the polymer–<i>Fo</i>Cut complex. The cleavage mechanism of these polymers through the ester group, by the nucleophilic attack of the catalytic Serine, in their structures demonstrates the similarity between cutinases and PETases enzymes. Thus, our work provides a new perspective for using Cut enzymes, especially <i>Fo</i>Cut, as biocatalysts for polymer degradation reactions, providing insights into the possibility of enzymatic improvement to increase their enzymatic activity.</p></div></div></section>
</div>
<div class="u-mt-32">
<section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
<p>The datasets used and/or analysed during the current study available from the corresponding author on reasonable request.</p>
</div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Austin, H. P. et al. Characterization and engineering of a plastic-degrading aromatic polyesterase. <i>Proc. Natl. Acad. Sci. USA</i> <b>115</b>, E4350–E4357 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 1" data-doi="10.1073/pnas.1718804115" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1073/pnas.1718804115" data-track-label="10.1073/pnas.1718804115" data-track-value="article reference" href="https://doi.org/10.1073%2Fpnas.1718804115" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 1" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvVGhtLnF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 1" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29666242" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 1" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5948967" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 1" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1462.60029" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 1" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Characterization%20and%20engineering%20of%20a%20plastic-degrading%20aromatic%20polyesterase&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1718804115&amp;volume=115&amp;pages=E4350-E4357&amp;publication_year=2018&amp;author=Austin%2CHP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Han, X. et al. Structural insight into catalytic mechanism of PET hydrolase. <i>Nat. Commun.</i> <b>8</b>, 2106 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 2" data-doi="10.1038/s41467-017-02255-z" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-017-02255-z" data-track-label="10.1038/s41467-017-02255-z" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-017-02255-z" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 2" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2017NatCo...8.2106H" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 2" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29235460" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 2" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5727383" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 2" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1419.58011" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 2" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20insight%20into%20catalytic%20mechanism%20of%20PET%20hydrolase&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-017-02255-z&amp;volume=8&amp;publication_year=2017&amp;author=Han%2CX" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Joo, S. et al. Structural insight into molecular mechanism of poly(ethylene terephthalate) degradation. <i>Nat. Commun.</i> <b>9</b>, 382 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 3" data-doi="10.1038/s41467-018-02881-1" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-018-02881-1" data-track-label="10.1038/s41467-018-02881-1" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-018-02881-1" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 3" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018NatCo...9..382J" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 3" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29374183" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 3" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5785972" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 3" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1409.00049" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 3" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20insight%20into%20molecular%20mechanism%20of%20poly%28ethylene%20terephthalate%29%20degradation&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-02881-1&amp;volume=9&amp;publication_year=2018&amp;author=Joo%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Rhodes, C. J. Plastic pollution and potential solutions. <i>Sci. Prog.</i> <b>101</b>, 207–260 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 4" data-doi="10.3184/003685018X15294876706211" data-track="click_references" data-track-action="article reference" data-track-item_id="10.3184/003685018X15294876706211" data-track-label="10.3184/003685018X15294876706211" data-track-value="article reference" href="https://doi.org/10.3184%2F003685018X15294876706211" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 4" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30025551" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 4" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10365157" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 4" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1027.20038" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 4" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Plastic%20pollution%20and%20potential%20solutions&amp;journal=Sci.%20Prog.&amp;doi=10.3184%2F003685018X15294876706211&amp;volume=101&amp;pages=207-260&amp;publication_year=2018&amp;author=Rhodes%2CCJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Kankanige, D. &amp; Babel, S. Smaller-sized micro-plastics (MPs) contamination in single-use PET-bottled water in Thailand. <i>Sci. Total Environ.</i> <b>717</b>, 137232 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 5" data-doi="10.1016/j.scitotenv.2020.137232" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.scitotenv.2020.137232" data-track-label="10.1016/j.scitotenv.2020.137232" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.scitotenv.2020.137232" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 5" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXktVGksLo%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 5" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32062244" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 5" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Smaller-sized%20micro-plastics%20%28MPs%29%20contamination%20in%20single-use%20PET-bottled%20water%20in%20Thailand&amp;journal=Sci.%20Total%20Environ.&amp;doi=10.1016%2Fj.scitotenv.2020.137232&amp;volume=717&amp;publication_year=2020&amp;author=Kankanige%2CD&amp;author=Babel%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Samir, A., Ashour, F. H., Hakim, A. A. A. &amp; Bassyouni, M. Recent advances in biodegradable polymers for sustainable applications. <i>Npj Mater. Degrad.</i> <b>6</b>, 68 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 6" data-doi="10.1038/s41529-022-00277-7" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41529-022-00277-7" data-track-label="10.1038/s41529-022-00277-7" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41529-022-00277-7" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 6" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XitF2itbzN" rel="nofollow noopener">CAS</a> 
    <a aria-label="MATH reference 6" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1524.62028" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 6" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Recent%20advances%20in%20biodegradable%20polymers%20for%20sustainable%20applications&amp;journal=Npj%20Mater.%20Degrad.&amp;doi=10.1038%2Fs41529-022-00277-7&amp;volume=6&amp;publication_year=2022&amp;author=Samir%2CA&amp;author=Ashour%2CFH&amp;author=Hakim%2CAAA&amp;author=Bassyouni%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Roman, L., Schuyler, Q., Wilcox, C. &amp; Hardesty, B. D. Plastic pollution is killing marine megafauna, but how do we prioritize policies to reduce mortality?. <i>Conserv. Lett.</i> <b>14</b>, 12871 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 7" data-doi="10.1111/conl.12781" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1111/conl.12781" data-track-label="10.1111/conl.12781" data-track-value="article reference" href="https://doi.org/10.1111%2Fconl.12781" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 7" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Plastic%20pollution%20is%20killing%20marine%20megafauna%2C%20but%20how%20do%20we%20prioritize%20policies%20to%20reduce%20mortality%3F&amp;journal=Conserv.%20Lett.&amp;doi=10.1111%2Fconl.12781&amp;volume=14&amp;publication_year=2021&amp;author=Roman%2CL&amp;author=Schuyler%2CQ&amp;author=Wilcox%2CC&amp;author=Hardesty%2CBD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Benavides Fernández, C. D., Guzmán Castillo, M. P., Quijano Pérez, S. A. &amp; Carvajal Rodríguez, L. V. Microbial degradation of polyethylene terephthalate: A systematic review. <i>SN Appl. Sci.</i> <b>4</b>, 263 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 8" data-doi="10.1007/s42452-022-05143-4" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1007/s42452-022-05143-4" data-track-label="10.1007/s42452-022-05143-4" data-track-value="article reference" href="https://link.springer.com/doi/10.1007/s42452-022-05143-4" rel="noopener">Article</a> 
    <a aria-label="Google Scholar reference 8" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Microbial%20degradation%20of%20polyethylene%20terephthalate%3A%20A%20systematic%20review&amp;journal=SN%20Appl.%20Sci.&amp;doi=10.1007%2Fs42452-022-05143-4&amp;volume=4&amp;publication_year=2022&amp;author=Benavides%20Fern%C3%A1ndez%2CCD&amp;author=Guzm%C3%A1n%20Castillo%2CMP&amp;author=Quijano%20P%C3%A9rez%2CSA&amp;author=Carvajal%20Rodr%C3%ADguez%2CLV" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Kanwal, A., Zhang, M., Sharaf, F. &amp; Li, C. Polymer pollution and its solutions with special emphasis on Poly (butylene adipate terephthalate (PBAT)). <i>Polym. Bull.</i> <b>79</b>, 9303–9330 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 9" data-doi="10.1007/s00289-021-04065-2" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1007/s00289-021-04065-2" data-track-label="10.1007/s00289-021-04065-2" data-track-value="article reference" href="https://link.springer.com/doi/10.1007/s00289-021-04065-2" rel="noopener">Article</a> 
    <a aria-label="CAS reference 9" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38Xmtl2ktw%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 9" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Polymer%20pollution%20and%20its%20solutions%20with%20special%20emphasis%20on%20Poly%20%28butylene%20adipate%20terephthalate%20%28PBAT%29%29&amp;journal=Polym.%20Bull.&amp;doi=10.1007%2Fs00289-021-04065-2&amp;volume=79&amp;pages=9303-9330&amp;publication_year=2022&amp;author=Kanwal%2CA&amp;author=Zhang%2CM&amp;author=Sharaf%2CF&amp;author=Li%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Hellesnes, K. N., Vijayaraj, S., Fojan, P., Petersen, E. &amp; Courtade, G. Biochemical Characterization and NMR Study of a PET-hydrolyzing cutinase from <i>Fusarium solani pisi</i>. <i>Biochemistry</i> <b>62</b>, 1369–1375 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 10" data-doi="10.1021/acs.biochem.2c00619" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.biochem.2c00619" data-track-label="10.1021/acs.biochem.2c00619" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.biochem.2c00619" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 10" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXmtVSgtrc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 10" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36967526" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 10" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Biochemical%20Characterization%20and%20NMR%20Study%20of%20a%20PET-hydrolyzing%20cutinase%20from%20Fusarium%20solani%20pisi&amp;journal=Biochemistry&amp;doi=10.1021%2Facs.biochem.2c00619&amp;volume=62&amp;pages=1369-1375&amp;publication_year=2023&amp;author=Hellesnes%2CKN&amp;author=Vijayaraj%2CS&amp;author=Fojan%2CP&amp;author=Petersen%2CE&amp;author=Courtade%2CG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Dimarogona, M. et al. Structural and functional studies of a Fusarium oxysporum cutinase with polyethylene terephthalate modification potential. <i>Biochim. Biophys. Acta BBA General Subjects</i> <b>1850</b>, 2308–2317 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 11" data-doi="10.1016/j.bbagen.2015.08.009" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.bbagen.2015.08.009" data-track-label="10.1016/j.bbagen.2015.08.009" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.bbagen.2015.08.009" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 11" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtlKktb7N" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 11" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26291558" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 11" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1360.68171" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 11" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20and%20functional%20studies%20of%20a%20Fusarium%20oxysporum%20cutinase%20with%20polyethylene%20terephthalate%20modification%20potential&amp;journal=Biochim.%20Biophys.%20Acta%20BBA%20General%20Subjects&amp;doi=10.1016%2Fj.bbagen.2015.08.009&amp;volume=1850&amp;pages=2308-2317&amp;publication_year=2015&amp;author=Dimarogona%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Vázquez-Alcántara, L., Oliart-Ros, R. M., García-Bórquez, A. &amp; Peña-Montes, C. Expression of a cutinase of <i>Moniliophthora roreri</i> with polyester and PET-plastic residues degradation activity. <i>Microbiol. Spectr.</i> <b>9</b>(3), e00976-e1021 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 12" data-doi="10.1128/Spectrum.00976-21" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1128/Spectrum.00976-21" data-track-label="10.1128/Spectrum.00976-21" data-track-value="article reference" href="https://doi.org/10.1128%2FSpectrum.00976-21" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 12" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34730414" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 12" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8567236" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 12" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Expression%20of%20a%20cutinase%20of%20Moniliophthora%20roreri%20with%20polyester%20and%20PET-plastic%20residues%20degradation%20activity&amp;journal=Microbiol.%20Spectr.&amp;doi=10.1128%2FSpectrum.00976-21&amp;volume=9&amp;issue=3&amp;pages=e00976-e1021&amp;publication_year=2021&amp;author=V%C3%A1zquez-Alc%C3%A1ntara%2CL&amp;author=Oliart-Ros%2CRM&amp;author=Garc%C3%ADa-B%C3%B3rquez%2CA&amp;author=Pe%C3%B1a-Montes%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Numoto, N. et al. Structural dynamics of the PET-degrading cutinase-like enzyme from <i>Saccharomonospora viridis</i> AHK190 in substrate-bound states elucidates the Ca<sup>2+</sup>—driven catalytic cycle. <i>Biochemistry</i> <b>57</b>, 5289–5300 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 13" data-doi="10.1021/acs.biochem.8b00624" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.biochem.8b00624" data-track-label="10.1021/acs.biochem.8b00624" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.biochem.8b00624" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 13" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsFSqsrbE" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 13" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30110540" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 13" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1406.46049" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 13" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20dynamics%20of%20the%20PET-degrading%20cutinase-like%20enzyme%20from%20Saccharomonospora%20viridis%20AHK190%20in%20substrate-bound%20states%20elucidates%20the%20Ca2%2B%E2%80%94driven%20catalytic%20cycle&amp;journal=Biochemistry&amp;doi=10.1021%2Facs.biochem.8b00624&amp;volume=57&amp;pages=5289-5300&amp;publication_year=2018&amp;author=Numoto%2CN" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Ribitsch, D. et al. Characterization of a new cutinase from <i>Thermobifida alba</i> for PET-surface hydrolysis. <i>Biocatal Biotransform.</i> <b>30</b>, 2–9 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 14" data-doi="10.3109/10242422.2012.644435" data-track="click_references" data-track-action="article reference" data-track-item_id="10.3109/10242422.2012.644435" data-track-label="10.3109/10242422.2012.644435" data-track-value="article reference" href="https://doi.org/10.3109%2F10242422.2012.644435" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 14" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhsVOmtb0%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 14" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Characterization%20of%20a%20new%20cutinase%20from%20Thermobifida%20alba%20for%20PET-surface%20hydrolysis&amp;journal=Biocatal%20Biotransform.&amp;doi=10.3109%2F10242422.2012.644435&amp;volume=30&amp;pages=2-9&amp;publication_year=2012&amp;author=Ribitsch%2CD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Zhang, Z. et al. Depolymerization of post-consumer PET bottles with engineered cutinase 1 from <i>Thermobifida cellulosilytica</i>. <i>Green Chem.</i> <b>24</b>, 5998–6007 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 15" data-doi="10.1039/D2GC01834A" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1039/D2GC01834A" data-track-label="10.1039/D2GC01834A" data-track-value="article reference" href="https://doi.org/10.1039%2FD2GC01834A" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 15" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XhvVOhsLzI" rel="nofollow noopener">CAS</a> 
    <a aria-label="MATH reference 15" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1508.92299" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 15" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Depolymerization%20of%20post-consumer%20PET%20bottles%20with%20engineered%20cutinase%201%20from%20Thermobifida%20cellulosilytica&amp;journal=Green%20Chem.&amp;doi=10.1039%2FD2GC01834A&amp;volume=24&amp;pages=5998-6007&amp;publication_year=2022&amp;author=Zhang%2CZ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Furukawa, M., Kawakami, N., Tomizawa, A. &amp; Miyamoto, K. Efficient degradation of poly(ethylene terephthalate) with thermobifida fusca cutinase exhibiting improved catalytic activity generated using mutagenesis and additive-based approaches. <i>Sci. Rep.</i> <b>9</b>, 16038 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 16" data-doi="10.1038/s41598-019-52379-z" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41598-019-52379-z" data-track-label="10.1038/s41598-019-52379-z" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41598-019-52379-z" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 16" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019NatSR...916038F" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 16" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31690819" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 16" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6831586" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 16" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20degradation%20of%20poly%28ethylene%20terephthalate%29%20with%20thermobifida%20fusca%20cutinase%20exhibiting%20improved%20catalytic%20activity%20generated%20using%20mutagenesis%20and%20additive-based%20approaches&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-019-52379-z&amp;volume=9&amp;publication_year=2019&amp;author=Furukawa%2CM&amp;author=Kawakami%2CN&amp;author=Tomizawa%2CA&amp;author=Miyamoto%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Aristizábal-Lanza, L., Mankar, S. V., Tullberg, C., Zhang, B. &amp; Linares-Pastén, J. A. Comparison of the enzymatic depolymerization of polyethylene terephthalate and Akestra<sup>TM</sup> using <i>Humicola insolens</i> cutinase. <i>Front. Chem. Eng.</i> <b>4</b>, 1048744  (2022).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Abokitse, K. et al. A novel actinobacterial cutinase containing a noncatalytic polymer-binding domain. <i>Appl. Environ. Microbiol.</i> <b>88</b>,  e0152221. <a data-track="click_references" data-track-action="external reference" data-track-label="10.1128/AEM.01522-21" data-track-value="external reference" href="https://doi.org/10.1128/AEM.01522-21">https://doi.org/10.1128/AEM.01522-21</a> (2022).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Kitadokoro, K. et al. Structural insights into the unique polylactate-degrading mechanism of <i>Thermobifida alba</i> cutinase. <i>FEBS J.</i> <b>286</b>, 2087–2098 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 19" data-doi="10.1111/febs.14781" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1111/febs.14781" data-track-label="10.1111/febs.14781" data-track-value="article reference" href="https://doi.org/10.1111%2Ffebs.14781" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 19" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXjslOku7c%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 19" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30761732" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 19" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1432.62082" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 19" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20insights%20into%20the%20unique%20polylactate-degrading%20mechanism%20of%20Thermobifida%20alba%20cutinase&amp;journal=FEBS%20J.&amp;doi=10.1111%2Ffebs.14781&amp;volume=286&amp;pages=2087-2098&amp;publication_year=2019&amp;author=Kitadokoro%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Gamerith, C. et al. Enzymatic degradation of aromatic and aliphatic polyesters by <i>P. pastoris</i> expressed cutinase 1 from <i>Thermobifida cellulosilytica</i>. <i>Front. Microbiol.</i> <b>8</b>, (2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Hu, X. et al. Enzymatic degradation of poly(butylene succinate) by cutinase cloned from Fusarium solani. <i>Polym. Degrad. Stab.</i> <b>134</b>, 211–219 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 21" data-doi="10.1016/j.polymdegradstab.2016.10.012" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.polymdegradstab.2016.10.012" data-track-label="10.1016/j.polymdegradstab.2016.10.012" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.polymdegradstab.2016.10.012" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 21" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhslansbjN" rel="nofollow noopener">CAS</a> 
    <a aria-label="MATH reference 21" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1374.35208" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 21" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Enzymatic%20degradation%20of%20poly%28butylene%20succinate%29%20by%20cutinase%20cloned%20from%20Fusarium%20solani&amp;journal=Polym.%20Degrad.%20Stab.&amp;doi=10.1016%2Fj.polymdegradstab.2016.10.012&amp;volume=134&amp;pages=211-219&amp;publication_year=2016&amp;author=Hu%2CX" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Yang, Y. et al. Complete bio-degradation of poly(butylene adipate-co-terephthalate) via engineered cutinases. <i>Nat. Commun.</i> <b>14</b>, 1645 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 22" data-doi="10.1038/s41467-023-37374-3" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-023-37374-3" data-track-label="10.1038/s41467-023-37374-3" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-023-37374-3" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 22" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2023NatCo..14.1645Y" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 22" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXmsV2kurs%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 22" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36964144" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 22" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10039075" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 22" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1306.16008" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 22" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Complete%20bio-degradation%20of%20poly%28butylene%20adipate-co-terephthalate%29%20via%20engineered%20cutinases&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-023-37374-3&amp;volume=14&amp;publication_year=2023&amp;author=Yang%2CY" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Lin, W., Zhao, Y., Su, T. &amp; Wang, Z. Enzymatic hydrolysis of poly(butylene adipate-co-terephthalate) by Fusarium solani cutinase. <i>Polym. Degrad. Stab.</i> <b>211</b>, 110335 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 23" data-doi="10.1016/j.polymdegradstab.2023.110335" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.polymdegradstab.2023.110335" data-track-label="10.1016/j.polymdegradstab.2023.110335" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.polymdegradstab.2023.110335" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 23" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXmt1yqt70%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 23" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Enzymatic%20hydrolysis%20of%20poly%28butylene%20adipate-co-terephthalate%29%20by%20Fusarium%20solani%20cutinase&amp;journal=Polym.%20Degrad.%20Stab.&amp;doi=10.1016%2Fj.polymdegradstab.2023.110335&amp;volume=211&amp;publication_year=2023&amp;author=Lin%2CW&amp;author=Zhao%2CY&amp;author=Su%2CT&amp;author=Wang%2CZ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Abdel-Azeem, A. M., Abdel-Azeem, M. A., Darwish, A. G., Nafady, N. A. &amp; Ibrahim, N. A. Fusarium: Biodiversity, ecological significances, and industrial applications. 201–261 (Springer Nature, 2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Steinberg, C. et al. Adaptation of Fusarium oxysporum and Fusarium dimerum to the specific aquatic environment provided by the water systems of hospitals. <i>Water Res.</i> <b>76</b>, 53–65 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 25" data-doi="10.1016/j.watres.2015.02.036" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.watres.2015.02.036" data-track-label="10.1016/j.watres.2015.02.036" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.watres.2015.02.036" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 25" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXjvFeqsLk%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 25" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25792434" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 25" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1308.76238" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 25" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Adaptation%20of%20Fusarium%20oxysporum%20and%20Fusarium%20dimerum%20to%20the%20specific%20aquatic%20environment%20provided%20by%20the%20water%20systems%20of%20hospitals&amp;journal=Water%20Res.&amp;doi=10.1016%2Fj.watres.2015.02.036&amp;volume=76&amp;pages=53-65&amp;publication_year=2015&amp;author=Steinberg%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Barata, L. M. et al. Secondary metabolic profile as a tool for distinction and characterization of cultivars of black pepper (<i>Piper nigrum </i>L.) cultivated in Pará State. <i>Brazil. Int. J. Mol. Sci.</i> <b>22</b>, 890 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 26" data-doi="10.3390/ijms22020890" data-track="click_references" data-track-action="article reference" data-track-item_id="10.3390/ijms22020890" data-track-label="10.3390/ijms22020890" data-track-value="article reference" href="https://doi.org/10.3390%2Fijms22020890" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 26" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXltlGjtLw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 26" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33477389" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 26" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Secondary%20metabolic%20profile%20as%20a%20tool%20for%20distinction%20and%20characterization%20of%20cultivars%20of%20black%20pepper%20%28Piper%20nigrum%20L.%29%20cultivated%20in%20Par%C3%A1%20State&amp;journal=Brazil.%20Int.%20J.%20Mol.%20Sci.&amp;doi=10.3390%2Fijms22020890&amp;volume=22&amp;publication_year=2021&amp;author=Barata%2CLM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Paracampo, N. E., Abreu, L. F., de Lemos, O. F. &amp; Both, J. P. Quality Of black pepper produced in northeastern Pará. <i>Rev. De Agric. Neotrop.</i> <b>9</b>(3), e7020 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 27" data-doi="10.32404/rean.v9i3.7020" data-track="click_references" data-track-action="article reference" data-track-item_id="10.32404/rean.v9i3.7020" data-track-label="10.32404/rean.v9i3.7020" data-track-value="article reference" href="https://doi.org/10.32404%2Frean.v9i3.7020" rel="nofollow noopener">Article</a> 
    <a aria-label="Google Scholar reference 27" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Quality%20Of%20black%20pepper%20produced%20in%20northeastern%20Par%C3%A1&amp;journal=Rev.%20De%20Agric.%20Neotrop.&amp;doi=10.32404%2Frean.v9i3.7020&amp;volume=9&amp;issue=3&amp;publication_year=2022&amp;author=Paracampo%2CNE&amp;author=Abreu%2CLF&amp;author=Lemos%2COF&amp;author=Both%2CJP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Bubici, G., Kaushal, M., Prigigallo, M. I., Gómez-Lama Cabanás, C. &amp; Mercado-Blanco, J. Biological control agents against fusarium wilt of Banana. <i>Front. Microbiol.</i> <b>10</b>, (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">McGovern, R. J. Management of tomato diseases caused by Fusarium oxysporum. <i>Crop Protection</i> <b>73</b>, 78–92 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 29" data-doi="10.1016/j.cropro.2015.02.021" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.cropro.2015.02.021" data-track-label="10.1016/j.cropro.2015.02.021" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.cropro.2015.02.021" rel="nofollow noopener">Article</a> 
    <a aria-label="MATH reference 29" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1521.83176" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 29" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Management%20of%20tomato%20diseases%20caused%20by%20Fusarium%20oxysporum&amp;journal=Crop%20Protection&amp;doi=10.1016%2Fj.cropro.2015.02.021&amp;volume=73&amp;pages=78-92&amp;publication_year=2015&amp;author=McGovern%2CRJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">de Oliveira, M. V., Fernandes, G. M., da Costa, K. S., Vakal, S. &amp; Lima, A. H. Virtual screening of natural products against 5-enolpyruvylshikimate-3-phosphate synthase using the <i>Anagreen herbicide</i>-like natural compound library. <i>RSC Advances.</i> <b>12</b>(29), 18834–18847 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 30" data-doi="10.1039/D2RA02645G" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1039/D2RA02645G" data-track-label="10.1039/D2RA02645G" data-track-value="article reference" href="https://doi.org/10.1039%2FD2RA02645G" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 30" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022RSCAd..1218834D" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 30" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35873314" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 30" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9240924" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 30" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Virtual%20screening%20of%20natural%20products%20against%205-enolpyruvylshikimate-3-phosphate%20synthase%20using%20the%20Anagreen%20herbicide-like%20natural%20compound%20library&amp;journal=RSC%20Advances.&amp;doi=10.1039%2FD2RA02645G&amp;volume=12&amp;issue=29&amp;pages=18834-18847&amp;publication_year=2022&amp;author=Oliveira%2CMV&amp;author=Fernandes%2CGM&amp;author=Costa%2CKS&amp;author=Vakal%2CS&amp;author=Lima%2CAH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">De Oliveira, M. D., Araújo, J. de O., Galúcio, J. M. P., Santana, K. &amp; Lima, A. H. Targeting shikimate pathway: In silico analysis of phosphoenolpyruvate derivatives as inhibitors of EPSP synthase and DAHP synthase. <i>J. Mol. Graph Model</i> <b>101</b>, 107735 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">De Souza, E. M. de C. et al. Molecular characterization and in silico evaluation of surfactins produced by endophytic bacteria from <i>Phanera splendens</i>. <i>Front. Chem.</i> <b>11</b>, 1240704 (2023).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">de Oliveira, M. V., Almeida, L., da Rocha, J., Wanzeller, A. L. &amp; Lima, A. A combined approach for predicting binding affinity of Zika Virus NS2B-NS3 protease inhibitors using semi-empirical methods and molecular docking algorithms. <i>J. Braz. Chem. Soc.</i> <a data-track="click_references" data-track-action="external reference" data-track-label="10.21577/0103-5053.20230130" data-track-value="external reference" href="https://doi.org/10.21577/0103-5053.20230130">https://doi.org/10.21577/0103-5053.20230130</a> (2024).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 33" data-doi="10.21577/0103-5053.20230130" data-track="click_references" data-track-action="article reference" data-track-item_id="10.21577/0103-5053.20230130" data-track-label="10.21577/0103-5053.20230130" data-track-value="article reference" href="https://doi.org/10.21577%2F0103-5053.20230130" rel="nofollow noopener">Article</a> 
    <a aria-label="MATH reference 33" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1545.91183" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 33" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20combined%20approach%20for%20predicting%20binding%20affinity%20of%20Zika%20Virus%20NS2B-NS3%20protease%20inhibitors%20using%20semi-empirical%20methods%20and%20molecular%20docking%20algorithms&amp;journal=J.%20Braz.%20Chem.%20Soc.&amp;doi=10.21577%2F0103-5053.20230130&amp;publication_year=2024&amp;author=Oliveira%2CMV&amp;author=Almeida%2CL&amp;author=Rocha%2CJ&amp;author=Wanzeller%2CAL&amp;author=Lima%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Costa, C. H. S. et al. Assessment of the PETase conformational changes induced by poly(ethylene terephthalate) binding. <i>Proteins Struct. Funct. Bioinform.</i> <b>89</b>, 1340–1352 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 34" data-doi="10.1002/prot.26155" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/prot.26155" data-track-label="10.1002/prot.26155" data-track-value="article reference" href="https://doi.org/10.1002%2Fprot.26155" rel="nofollow noopener">Article</a> 
    <a aria-label="MATH reference 34" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1158.35348" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 34" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Assessment%20of%20the%20PETase%20conformational%20changes%20induced%20by%20poly%28ethylene%20terephthalate%29%20binding&amp;journal=Proteins%20Struct.%20Funct.%20Bioinform.&amp;doi=10.1002%2Fprot.26155&amp;volume=89&amp;pages=1340-1352&amp;publication_year=2021&amp;author=Costa%2CCHS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Araújo, O. J. et al. Structural, energetic and lipophilic analysis of SARS-CoV-2 non-structural protein 9 (NSP9). <i>Sci. Rep.</i> <b>11</b>, 23003 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 35" data-doi="10.1038/s41598-021-02366-0" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41598-021-02366-0" data-track-label="10.1038/s41598-021-02366-0" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41598-021-02366-0" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 35" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2021NatSR..1123003D" rel="nofollow noopener">ADS</a> 
    <a aria-label="MATH reference 35" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1472.37043" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 35" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%2C%20energetic%20and%20lipophilic%20analysis%20of%20SARS-CoV-2%20non-structural%20protein%209%20%28NSP9%29&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-021-02366-0&amp;volume=11&amp;publication_year=2021&amp;author=Ara%C3%BAjo%2COJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Tamura, K., Stecher, G. &amp; Kumar, S. MEGA11: Molecular evolutionary genetics analysis version 11. <i>Mol. Biol. Evol.</i> <b>38</b>, 3022–3027 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 36" data-doi="10.1093/molbev/msab120" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/molbev/msab120" data-track-label="10.1093/molbev/msab120" data-track-value="article reference" href="https://doi.org/10.1093%2Fmolbev%2Fmsab120" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 36" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlCktrfN" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 36" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33892491" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 36" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8233496" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 36" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0539.60087" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 36" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=MEGA11%3A%20Molecular%20evolutionary%20genetics%20analysis%20version%2011&amp;journal=Mol.%20Biol.%20Evol.&amp;doi=10.1093%2Fmolbev%2Fmsab120&amp;volume=38&amp;pages=3022-3027&amp;publication_year=2021&amp;author=Tamura%2CK&amp;author=Stecher%2CG&amp;author=Kumar%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Avilan, L. et al. Concentration-dependent inhibition of mesophilic PETases on poly (ethylene terephthalate) can be eliminated by enzyme engineering. <i>ChemSusChem.</i> <b>16</b>(8), e202202277 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 37" data-doi="10.1002/cssc.202202277" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/cssc.202202277" data-track-label="10.1002/cssc.202202277" data-track-value="article reference" href="https://doi.org/10.1002%2Fcssc.202202277" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 37" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXmsFCitr8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 37" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36811288" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 37" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Concentration-dependent%20inhibition%20of%20mesophilic%20PETases%20on%20poly%20%28ethylene%20terephthalate%29%20can%20be%20eliminated%20by%20enzyme%20engineering&amp;journal=ChemSusChem.&amp;doi=10.1002%2Fcssc.202202277&amp;volume=16&amp;issue=8&amp;publication_year=2023&amp;author=Avilan%2CL&amp;author=Lichtenstein%2CBR&amp;author=K%C3%B6nig%2CG&amp;author=Zahn%2CM&amp;author=Allen%2CMD&amp;author=Oliveira%2CL&amp;author=Clark%2CM&amp;author=Bemmer%2CV&amp;author=Graham%2CR&amp;author=Austin%2CHP&amp;author=Dominick%2CG" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Longhi, S., Czjzek, M., Lamzin, V., Nicolas, A. &amp; Cambillau, C. Atomic resolution (1.0 Å) crystal structure of <i>Fusarium solani</i> cutinase: stereochemical analysis. <i>J. Mol. Biol.</i> <b>268</b>, 779–799 (1997).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 38" data-doi="10.1006/jmbi.1997.1000" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1006/jmbi.1997.1000" data-track-label="10.1006/jmbi.1997.1000" data-track-value="article reference" href="https://doi.org/10.1006%2Fjmbi.1997.1000" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 38" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK2sXjs1Cmu7k%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 38" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9175860" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 38" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Atomic%20resolution%20%281.0%20%C3%85%29%20crystal%20structure%20of%20Fusarium%20solani%20cutinase%3A%20stereochemical%20analysis&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1006%2Fjmbi.1997.1000&amp;volume=268&amp;pages=779-799&amp;publication_year=1997&amp;author=Longhi%2CS&amp;author=Czjzek%2CM&amp;author=Lamzin%2CV&amp;author=Nicolas%2CA&amp;author=Cambillau%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Martinez, C., De Geus, P., Lauwereys, M., Matthyssens, G. &amp; Cambillau, C. Fusarium solani cutinase is a lipolytic enzyme with a catalytic serine accessible to solvent. <i>Nature</i> <b>356</b>, 615–618 (1992).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 39" data-doi="10.1038/356615a0" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/356615a0" data-track-label="10.1038/356615a0" data-track-value="article reference" href="https://doi.org/10.1038%2F356615a0" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 39" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1992Natur.356..615M" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 39" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK38XitlSksL0%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 39" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1560844" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 39" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Fusarium%20solani%20cutinase%20is%20a%20lipolytic%20enzyme%20with%20a%20catalytic%20serine%20accessible%20to%20solvent&amp;journal=Nature&amp;doi=10.1038%2F356615a0&amp;volume=356&amp;pages=615-618&amp;publication_year=1992&amp;author=Martinez%2CC&amp;author=Geus%2CP&amp;author=Lauwereys%2CM&amp;author=Matthyssens%2CG&amp;author=Cambillau%2CC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Kold, D. et al. Thermodynamic and structural investigation of the specific SDS binding of <i>humicola insolens</i> cutinase. <i>Protein Science</i> <b>23</b>, 1023–1035 (2014).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 40" data-doi="10.1002/pro.2489" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/pro.2489" data-track-label="10.1002/pro.2489" data-track-value="article reference" href="https://doi.org/10.1002%2Fpro.2489" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 40" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFeltbjJ" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 40" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24832484" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 40" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4116652" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 40" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0977.57007" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 40" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Thermodynamic%20and%20structural%20investigation%20of%20the%20specific%20SDS%20binding%20of%20humicola%20insolens%20cutinase&amp;journal=Protein%20Science&amp;doi=10.1002%2Fpro.2489&amp;volume=23&amp;pages=1023-1035&amp;publication_year=2014&amp;author=Kold%2CD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Nyon, M. P. et al. Catalysis by glomerella cingulata cutinase requires conformational cycling between the active and inactive states of its catalytic triad. <i>J. Mol. Biol.</i> <b>385</b>, 226–235 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 41" data-doi="10.1016/j.jmb.2008.10.050" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.jmb.2008.10.050" data-track-label="10.1016/j.jmb.2008.10.050" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.jmb.2008.10.050" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 41" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsFCmsbjM" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 41" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18983850" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 41" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Catalysis%20by%20glomerella%20cingulata%20cutinase%20requires%20conformational%20cycling%20between%20the%20active%20and%20inactive%20states%20of%20its%20catalytic%20triad&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2Fj.jmb.2008.10.050&amp;volume=385&amp;pages=226-235&amp;publication_year=2009&amp;author=Nyon%2CMP" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Liu, Z. et al. Structural and functional studies of <i>Aspergillus oryzae</i> cutinase: Enhanced thermostability and hydrolytic activity of synthetic ester and polyester degradation. <i>J. Am. Chem. Soc.</i> <b>131</b>, 15711–15716 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 42" data-doi="10.1021/ja9046697" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/ja9046697" data-track-label="10.1021/ja9046697" data-track-value="article reference" href="https://doi.org/10.1021%2Fja9046697" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 42" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXht1artLrF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 42" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19810726" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 42" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796240" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 42" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20and%20functional%20studies%20of%20Aspergillus%20oryzae%20cutinase%3A%20Enhanced%20thermostability%20and%20hydrolytic%20activity%20of%20synthetic%20ester%20and%20polyester%20degradation&amp;journal=J.%20Am.%20Chem.%20Soc.&amp;doi=10.1021%2Fja9046697&amp;volume=131&amp;pages=15711-15716&amp;publication_year=2009&amp;author=Liu%2CZ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Duan, X. et al. High-level expression and characterization of a novel cutinase from Malbranchea cinnamomea suitable for butyl butyrate production. <i>Biotechnol. Biofuels</i> <b>10</b>, 223 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 43" data-doi="10.1186/s13068-017-0912-z" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1186/s13068-017-0912-z" data-track-label="10.1186/s13068-017-0912-z" data-track-value="article reference" href="https://link.springer.com/doi/10.1186/s13068-017-0912-z" rel="noopener">Article</a> 
    <a aria-label="PubMed reference 43" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28932264" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 43" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5606096" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 43" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=High-level%20expression%20and%20characterization%20of%20a%20novel%20cutinase%20from%20Malbranchea%20cinnamomea%20suitable%20for%20butyl%20butyrate%20production&amp;journal=Biotechnol.%20Biofuels&amp;doi=10.1186%2Fs13068-017-0912-z&amp;volume=10&amp;publication_year=2017&amp;author=Duan%2CX" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Roussel, A. et al. A cutinase from trichoderma reesei with a lid-covered active site and kinetic properties of true lipases. <i>J Mol. Biol.</i> <b>426</b>, 3757–3772 (2014).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 44" data-doi="10.1016/j.jmb.2014.09.003" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.jmb.2014.09.003" data-track-label="10.1016/j.jmb.2014.09.003" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.jmb.2014.09.003" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 44" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFyitb7K" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 44" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25219509" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 44" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1304.14033" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 44" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20cutinase%20from%20trichoderma%20reesei%20with%20a%20lid-covered%20active%20site%20and%20kinetic%20properties%20of%20true%20lipases&amp;journal=J%20Mol.%20Biol.&amp;doi=10.1016%2Fj.jmb.2014.09.003&amp;volume=426&amp;pages=3757-3772&amp;publication_year=2014&amp;author=Roussel%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Miyakawa, T. et al. Structural basis for the Ca2+-enhanced thermostability and activity of PET-degrading cutinase-like enzyme from Saccharomonospora viridis AHK190. <i>Appl. Microbiol. Biotechnol.</i> <b>99</b>, 4297–4307 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 45" data-doi="10.1007/s00253-014-6272-8" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1007/s00253-014-6272-8" data-track-label="10.1007/s00253-014-6272-8" data-track-value="article reference" href="https://link.springer.com/doi/10.1007/s00253-014-6272-8" rel="noopener">Article</a> 
    <a aria-label="CAS reference 45" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXitVKmu73F" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 45" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25492421" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 45" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1373.78250" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 45" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20basis%20for%20the%20Ca2%2B-enhanced%20thermostability%20and%20activity%20of%20PET-degrading%20cutinase-like%20enzyme%20from%20Saccharomonospora%20viridis%20AHK190&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;doi=10.1007%2Fs00253-014-6272-8&amp;volume=99&amp;pages=4297-4307&amp;publication_year=2015&amp;author=Miyakawa%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Erickson, E. et al. Sourcing thermotolerant poly(ethylene terephthalate) hydrolase scaffolds from natural diversity. <i>Nat. Commun.</i> <b>13</b>, 7850 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 46" data-doi="10.1038/s41467-022-35237-x" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-022-35237-x" data-track-label="10.1038/s41467-022-35237-x" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-022-35237-x" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 46" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2022NatCo..13.7850E" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 46" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XjtFOltb%2FF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 46" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36543766" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 46" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9772341" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 46" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Sourcing%20thermotolerant%20poly%28ethylene%20terephthalate%29%20hydrolase%20scaffolds%20from%20natural%20diversity&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-022-35237-x&amp;volume=13&amp;publication_year=2022&amp;author=Erickson%2CE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Ribitsch, D. et al. Small cause, large effect: Structural characterization of cutinases from <i>Thermobifida cellulosilytica</i>. <i>Biotechnol. Bioeng.</i> <b>114</b>, 2481–2488 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 47" data-doi="10.1002/bit.26372" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/bit.26372" data-track-label="10.1002/bit.26372" data-track-value="article reference" href="https://doi.org/10.1002%2Fbit.26372" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 47" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlart7vL" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 47" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28671263" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 47" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1369.46012" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 47" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Small%20cause%2C%20large%20effect%3A%20Structural%20characterization%20of%20cutinases%20from%20Thermobifida%20cellulosilytica&amp;journal=Biotechnol.%20Bioeng.&amp;doi=10.1002%2Fbit.26372&amp;volume=114&amp;pages=2481-2488&amp;publication_year=2017&amp;author=Ribitsch%2CD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Kitadokoro, K. et al. Crystal structure of cutinase Est119 from Thermobifida alba AHK119 that can degrade modified polyethylene terephthalate at 1.76Å resolution. <i>Polym. Degrad. Stab.</i> <b>97</b>, 771–775 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 48" data-doi="10.1016/j.polymdegradstab.2012.02.003" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.polymdegradstab.2012.02.003" data-track-label="10.1016/j.polymdegradstab.2012.02.003" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.polymdegradstab.2012.02.003" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 48" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xjt1elsrY%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 48" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Crystal%20structure%20of%20cutinase%20Est119%20from%20Thermobifida%20alba%20AHK119%20that%20can%20degrade%20modified%20polyethylene%20terephthalate%20at%201.76%C3%85%20resolution&amp;journal=Polym.%20Degrad.%20Stab.&amp;doi=10.1016%2Fj.polymdegradstab.2012.02.003&amp;volume=97&amp;pages=771-775&amp;publication_year=2012&amp;author=Kitadokoro%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Berman, H. M. et al. The protein data bank. <i>Nucl. Acids Res.</i> <a data-track="click_references" data-track-action="external reference" data-track-label="10.1093/nar/28.1.235" data-track-value="external reference" href="https://doi.org/10.1093/nar/28.1.235">https://doi.org/10.1093/nar/28.1.235</a> (2000).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 49" data-doi="10.1093/nar/28.1.235" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/28.1.235" data-track-label="10.1093/nar/28.1.235" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2F28.1.235" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 49" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2000spam.book.....B" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 49" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10592235" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 49" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC102472" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 49" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1006.08001" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 49" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20protein%20data%20bank&amp;journal=Nucl.%20Acids%20Res.&amp;doi=10.1093%2Fnar%2F28.1.235&amp;publication_year=2000&amp;author=Berman%2CHM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">ChemAxon. ChemAxon—Software Solutions and Services for Chemistry and Biology. <i>MarvinSketch, Version 16.10.31</i> Preprint at (2016).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Hanwell, M. D. et al. Avogadro: An advanced semantic chemical editor, visualization, and analysis platform. <i>J. Cheminform.</i> <a data-track="click_references" data-track-action="external reference" data-track-label="10.1186/1758-2946-4-17" data-track-value="external reference" href="https://doi.org/10.1186/1758-2946-4-17">https://doi.org/10.1186/1758-2946-4-17</a> (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 51" data-doi="10.1186/1758-2946-4-17" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1186/1758-2946-4-17" data-track-label="10.1186/1758-2946-4-17" data-track-value="article reference" href="https://link.springer.com/doi/10.1186/1758-2946-4-17" rel="noopener">Article</a> 
    <a aria-label="PubMed reference 51" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22889332" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 51" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3542060" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 51" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Avogadro%3A%20An%20advanced%20semantic%20chemical%20editor%2C%20visualization%2C%20and%20analysis%20platform&amp;journal=J.%20Cheminform.&amp;doi=10.1186%2F1758-2946-4-17&amp;publication_year=2012&amp;author=Hanwell%2CMD" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Frisch, M. J. ; et. al, Nathan, A. J. &amp; Scobell, A. Gaussian 09. <i>Gaussian, Inc. Wallingford CT</i> 2–3 Preprint at 111 (2009).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Roothaan, C. C. J. New developments in molecular orbital theory. <i>Rev. Mod. Phys.</i> <b>23</b>, 69–89 (1951).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 53" data-doi="10.1103/RevModPhys.23.69" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1103/RevModPhys.23.69" data-track-label="10.1103/RevModPhys.23.69" data-track-value="article reference" href="https://doi.org/10.1103%2FRevModPhys.23.69" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 53" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1951RvMP...23...69R" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 53" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaG38XksVakuw%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="MATH reference 53" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0045.28502" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 53" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=New%20developments%20in%20molecular%20orbital%20theory&amp;journal=Rev.%20Mod.%20Phys.&amp;doi=10.1103%2FRevModPhys.23.69&amp;volume=23&amp;pages=69-89&amp;publication_year=1951&amp;author=Roothaan%2CCCJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Cognigni, F. et al. Exploring the infiltrative and degradative ability of <i>Fusarium oxysporum</i> on polyethylene terephthalate (PET) using correlative microscopy and deep learning. <i>Sci. Rep.</i> <b>13</b>, 22987 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 54" data-doi="10.1038/s41598-023-50199-w" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41598-023-50199-w" data-track-label="10.1038/s41598-023-50199-w" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41598-023-50199-w" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 54" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2023NatSR..1322987C" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 54" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB2cXptFQ%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 54" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=38151495" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 54" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10752890" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 54" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Exploring%20the%20infiltrative%20and%20degradative%20ability%20of%20Fusarium%20oxysporum%20on%20polyethylene%20terephthalate%20%28PET%29%20using%20correlative%20microscopy%20and%20deep%20learning&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-023-50199-w&amp;volume=13&amp;publication_year=2023&amp;author=Cognigni%2CF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Santos-Beneit, F. et al. Screening enzymes that can depolymerize commercial biodegradable polymers: Heterologous expression of fusarium solani cutinase in <i>Escherichia coli</i>. <i>Microorganisms</i> <b>11</b>, 328 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 55" data-doi="10.3390/microorganisms11020328" data-track="click_references" data-track-action="article reference" data-track-item_id="10.3390/microorganisms11020328" data-track-label="10.3390/microorganisms11020328" data-track-value="article reference" href="https://doi.org/10.3390%2Fmicroorganisms11020328" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 55" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXltFOhsLg%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 55" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=36838293" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 55" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9963400" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 55" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Screening%20enzymes%20that%20can%20depolymerize%20commercial%20biodegradable%20polymers%3A%20Heterologous%20expression%20of%20fusarium%20solani%20cutinase%20in%20Escherichia%20coli&amp;journal=Microorganisms&amp;doi=10.3390%2Fmicroorganisms11020328&amp;volume=11&amp;publication_year=2023&amp;author=Santos-Beneit%2CF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Liang, X. &amp; Zou, H. Biotechnological application of cutinase: A powerful tool in synthetic biology. <i>SynBio</i> <b>1</b>, 54–64 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 56" data-doi="10.3390/synbio1010004" data-track="click_references" data-track-action="article reference" data-track-item_id="10.3390/synbio1010004" data-track-label="10.3390/synbio1010004" data-track-value="article reference" href="https://doi.org/10.3390%2Fsynbio1010004" rel="nofollow noopener">Article</a> 
    <a aria-label="MATH reference 56" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1195.81065" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 56" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Biotechnological%20application%20of%20cutinase%3A%20A%20powerful%20tool%20in%20synthetic%20biology&amp;journal=SynBio&amp;doi=10.3390%2Fsynbio1010004&amp;volume=1&amp;pages=54-64&amp;publication_year=2022&amp;author=Liang%2CX&amp;author=Zou%2CH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Huang, Q., Kimura, S. &amp; Iwata, T. Thermal embedding of <i>Humicola insolens</i> cutinase: A strategy for improving polyester biodegradation in seawater. <i>Biomacromolecules</i> <b>24</b>, 5836–5846 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 57" data-doi="10.1021/acs.biomac.3c00835" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.biomac.3c00835" data-track-label="10.1021/acs.biomac.3c00835" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.biomac.3c00835" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 57" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXitlSjsbnJ" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 57" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37940601" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 57" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1538.35015" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 57" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Thermal%20embedding%20of%20Humicola%20insolens%20cutinase%3A%20A%20strategy%20for%20improving%20polyester%20biodegradation%20in%20seawater&amp;journal=Biomacromolecules&amp;doi=10.1021%2Facs.biomac.3c00835&amp;volume=24&amp;pages=5836-5846&amp;publication_year=2023&amp;author=Huang%2CQ&amp;author=Kimura%2CS&amp;author=Iwata%2CT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Suzuki, M., Tachibana, Y. &amp; Kasuya, K. Biodegradability of poly(3-hydroxyalkanoate) and poly(ε-caprolactone) via biological carbon cycles in marine environments. <i>Polym. J.</i> <b>53</b>, 47–66 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 58" data-doi="10.1038/s41428-020-00396-5" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41428-020-00396-5" data-track-label="10.1038/s41428-020-00396-5" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41428-020-00396-5" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 58" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslyjsL3L" rel="nofollow noopener">CAS</a> 
    <a aria-label="Google Scholar reference 58" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Biodegradability%20of%20poly%283-hydroxyalkanoate%29%20and%20poly%28%CE%B5-caprolactone%29%20via%20biological%20carbon%20cycles%20in%20marine%20environments&amp;journal=Polym.%20J.&amp;doi=10.1038%2Fs41428-020-00396-5&amp;volume=53&amp;pages=47-66&amp;publication_year=2021&amp;author=Suzuki%2CM&amp;author=Tachibana%2CY&amp;author=Kasuya%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Murphy, C. A., Cameron, J. A., Huang, S. J. &amp; Vinopal, R. T. Fusarium polycaprolactone depolymerase is cutinase. <i>Appl. Environ. Microbiol.</i> <b>62</b>, 456–460 (1996).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 59" data-doi="10.1128/aem.62.2.456-460.1996" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1128/aem.62.2.456-460.1996" data-track-label="10.1128/aem.62.2.456-460.1996" data-track-value="article reference" href="https://doi.org/10.1128%2Faem.62.2.456-460.1996" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 59" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1996ApEnM..62..456M" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 59" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK28XovVClsQ%3D%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 59" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8593048" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 59" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC167813" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 59" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Fusarium%20polycaprolactone%20depolymerase%20is%20cutinase&amp;journal=Appl.%20Environ.%20Microbiol.&amp;doi=10.1128%2Faem.62.2.456-460.1996&amp;volume=62&amp;pages=456-460&amp;publication_year=1996&amp;author=Murphy%2CCA&amp;author=Cameron%2CJA&amp;author=Huang%2CSJ&amp;author=Vinopal%2CRT" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Thomsen, R. &amp; Christensen, M. H. MolDock: A new technique for high-accuracy molecular docking. <i>J. Med. Chem.</i> <b>49</b>, 3315–3321 (2006).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 60" data-doi="10.1021/jm051197e" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/jm051197e" data-track-label="10.1021/jm051197e" data-track-value="article reference" href="https://doi.org/10.1021%2Fjm051197e" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 60" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XjvFyiu7w%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 60" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16722650" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 60" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1494.46057" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 60" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=MolDock%3A%20A%20new%20technique%20for%20high-accuracy%20molecular%20docking&amp;journal=J.%20Med.%20Chem.&amp;doi=10.1021%2Fjm051197e&amp;volume=49&amp;pages=3315-3321&amp;publication_year=2006&amp;author=Thomsen%2CR&amp;author=Christensen%2CMH" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Jones, G., Willett, P., Glen, R. C., Leach, A. R. &amp; Taylor, R. Development and validation of a genetic algorithm for flexible docking 1 1Edited by F E. Cohen. <i>J. Mol. Biol.</i> <b>267</b>, 727–748 (1997).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 61" data-doi="10.1006/jmbi.1996.0897" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1006/jmbi.1996.0897" data-track-label="10.1006/jmbi.1996.0897" data-track-value="article reference" href="https://doi.org/10.1006%2Fjmbi.1996.0897" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 61" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK2sXis1KntLo%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 61" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9126849" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 61" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Development%20and%20validation%20of%20a%20genetic%20algorithm%20for%20flexible%20docking%201%201Edited%20by%20F%20E.%20Cohen&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1006%2Fjmbi.1996.0897&amp;volume=267&amp;pages=727-748&amp;publication_year=1997&amp;author=Jones%2CG&amp;author=Willett%2CP&amp;author=Glen%2CRC&amp;author=Leach%2CAR&amp;author=Taylor%2CR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Jurrus, E. et al. Improvements to the &lt;scp&gt;APBS&lt;/scp&gt; biomolecular solvation software suite. <i>Protein Sci.</i> <b>27</b>, 112–128 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 62" data-doi="10.1002/pro.3280" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/pro.3280" data-track-label="10.1002/pro.3280" data-track-value="article reference" href="https://doi.org/10.1002%2Fpro.3280" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 62" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslSkt7vI" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 62" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28836357" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 62" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Improvements%20to%20the%20%3Cscp%3EAPBS%3C%2Fscp%3E%20biomolecular%20solvation%20software%20suite&amp;journal=Protein%20Sci.&amp;doi=10.1002%2Fpro.3280&amp;volume=27&amp;pages=112-128&amp;publication_year=2018&amp;author=Jurrus%2CE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Jo, S., Kim, T., Iyer, V. G. &amp; Im, W. CHARMM-GUI: A web-based graphical user interface for CHARMM. <i>J. Comput. Chem.</i> <b>29</b>, 1859–1865 (2008).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 63" data-doi="10.1002/jcc.20945" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/jcc.20945" data-track-label="10.1002/jcc.20945" data-track-value="article reference" href="https://doi.org/10.1002%2Fjcc.20945" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 63" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXosVKksbc%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 63" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18351591" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 63" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM-GUI%3A%20A%20web-based%20graphical%20user%20interface%20for%20CHARMM&amp;journal=J.%20Comput.%20Chem.&amp;doi=10.1002%2Fjcc.20945&amp;volume=29&amp;pages=1859-1865&amp;publication_year=2008&amp;author=Jo%2CS&amp;author=Kim%2CT&amp;author=Iyer%2CVG&amp;author=Im%2CW" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Brooks, B. R. et al. CHARMM: The biomolecular simulation program. <i>J. Comput. Chem.</i> <b>30</b>, 1545–1614 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 64" data-doi="10.1002/jcc.21287" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/jcc.21287" data-track-label="10.1002/jcc.21287" data-track-value="article reference" href="https://doi.org/10.1002%2Fjcc.21287" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 64" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXms1Ciu70%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 64" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19444816" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 64" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2810661" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 64" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1170.83322" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 64" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM%3A%20The%20biomolecular%20simulation%20program&amp;journal=J.%20Comput.%20Chem.&amp;doi=10.1002%2Fjcc.21287&amp;volume=30&amp;pages=1545-1614&amp;publication_year=2009&amp;author=Brooks%2CBR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Lee, J. et al. CHARMM-GUI input generator for NAMD, GROMACS, AMBER, OpenMM, and CHARMM/OpenMM simulations using the CHARMM36 additive force field. <i>J. Chem. Theory Comput.</i> <b>12</b>, 405–413 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 65" data-doi="10.1021/acs.jctc.5b00935" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jctc.5b00935" data-track-label="10.1021/acs.jctc.5b00935" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jctc.5b00935" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 65" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVWru7nI" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 65" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26631602" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 65" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=CHARMM-GUI%20input%20generator%20for%20NAMD%2C%20GROMACS%2C%20AMBER%2C%20OpenMM%2C%20and%20CHARMM%2FOpenMM%20simulations%20using%20the%20CHARMM36%20additive%20force%20field&amp;journal=J.%20Chem.%20Theory%20Comput.&amp;doi=10.1021%2Facs.jctc.5b00935&amp;volume=12&amp;pages=405-413&amp;publication_year=2016&amp;author=Lee%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Jorgensen, W. L., Chandrasekhar, J., Madura, J. D., Impey, R. W. &amp; Klein, M. L. Comparison of simple potential functions for simulating liquid water. <i>J. Chem. Phys.</i> <a data-track="click_references" data-track-action="external reference" data-track-label="10.1063/1.445869" data-track-value="external reference" href="https://doi.org/10.1063/1.445869">https://doi.org/10.1063/1.445869</a> (1983).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 66" data-doi="10.1063/1.445869" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1063/1.445869" data-track-label="10.1063/1.445869" data-track-value="article reference" href="https://doi.org/10.1063%2F1.445869" rel="nofollow noopener">Article</a> 
    <a aria-label="MATH reference 66" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0499.30033" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 66" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparison%20of%20simple%20potential%20functions%20for%20simulating%20liquid%20water&amp;journal=J.%20Chem.%20Phys.&amp;doi=10.1063%2F1.445869&amp;publication_year=1983&amp;author=Jorgensen%2CWL&amp;author=Chandrasekhar%2CJ&amp;author=Madura%2CJD&amp;author=Impey%2CRW&amp;author=Klein%2CML" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Case, D. A. <i>et al.</i> Amber 20. (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Srinivasan, J., Cheatham, T. E., Cieplak, P., Kollman, P. A. &amp; Case, D. A. Continuum solvent studies of the stability of DNA, RNA, and phosphoramidate−DNA helices. <i>J. Am. Chem. Soc.</i> <b>120</b>, 9401–9409 (1998).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 68" data-doi="10.1021/ja981844+" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/ja981844+" data-track-label="10.1021/ja981844+" data-track-value="article reference" href="https://doi.org/10.1021%2Fja981844%2B" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 68" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK1cXlsVyqtLo%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="MATH reference 68" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0886.92035" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 68" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Continuum%20solvent%20studies%20of%20the%20stability%20of%20DNA%2C%20RNA%2C%20and%20phosphoramidate%E2%88%92DNA%20helices&amp;journal=J.%20Am.%20Chem.%20Soc.&amp;doi=10.1021%2Fja981844%2B&amp;volume=120&amp;pages=9401-9409&amp;publication_year=1998&amp;author=Srinivasan%2CJ&amp;author=Cheatham%2CTE&amp;author=Cieplak%2CP&amp;author=Kollman%2CPA&amp;author=Case%2CDA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Miller, B. R. III. et al. MMPBSA. py: An efficient program for end-state free energy calculations. <i>J. Chem. Theory Comput.</i> <b>8</b>(9), 3314–3321 (2012).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 69" data-doi="10.1021/ct300418h" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/ct300418h" data-track-label="10.1021/ct300418h" data-track-value="article reference" href="https://doi.org/10.1021%2Fct300418h" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 69" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhtV2gtrzP" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 69" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26605738" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 69" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=MMPBSA.%20py%3A%20An%20efficient%20program%20for%20end-state%20free%20energy%20calculations&amp;journal=J.%20Chem.%20Theory%20Comput.&amp;doi=10.1021%2Fct300418h&amp;volume=8&amp;issue=9&amp;pages=3314-3321&amp;publication_year=2012&amp;author=Miller%2CBR&amp;author=McGee%2CTD&amp;author=Swails%2CJM&amp;author=Homeyer%2CN&amp;author=Gohlke%2CH&amp;author=Roitberg%2CAE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Genheden, S. &amp; Ryde, U. The MM/PBSA and MM/GBSA methods to estimate ligand-binding affinities. <i>Expert Opin. Drug Discov.</i> <b>10</b>, 449–461 (2015).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 70" data-doi="10.1517/17460441.2015.1032936" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1517/17460441.2015.1032936" data-track-label="10.1517/17460441.2015.1032936" data-track-value="article reference" href="https://doi.org/10.1517%2F17460441.2015.1032936" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 70" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXntFGktr8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 70" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25835573" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 70" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4487606" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 70" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?48.0529.01" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 70" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20MM%2FPBSA%20and%20MM%2FGBSA%20methods%20to%20estimate%20ligand-binding%20affinities&amp;journal=Expert%20Opin.%20Drug%20Discov.&amp;doi=10.1517%2F17460441.2015.1032936&amp;volume=10&amp;pages=449-461&amp;publication_year=2015&amp;author=Genheden%2CS&amp;author=Ryde%2CU" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Wang, L., Pang, X., Li, Y., Zhang, Z. &amp; Tan, W. RADER: A RApid DEcoy retriever to facilitate decoy based assessment of virtual screening. <i>Bioinformatics</i> <b>33</b>, 1235–1237 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 71" data-doi="10.1093/bioinformatics/btw783" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/bioinformatics/btw783" data-track-label="10.1093/bioinformatics/btw783" data-track-value="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtw783" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 71" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28011765" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 71" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1240.65058" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 71" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=RADER%3A%20A%20RApid%20DEcoy%20retriever%20to%20facilitate%20decoy%20based%20assessment%20of%20virtual%20screening&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtw783&amp;volume=33&amp;pages=1235-1237&amp;publication_year=2017&amp;author=Wang%2CL&amp;author=Pang%2CX&amp;author=Li%2CY&amp;author=Zhang%2CZ&amp;author=Tan%2CW" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Wang, E. et al. End-point binding free energy calculation with MM/PBSA and MM/GBSA: Strategies and applications in drug design. <i>Chem. Rev.</i> <b>119</b>, 9478–9508 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 72" data-doi="10.1021/acs.chemrev.9b00055" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.chemrev.9b00055" data-track-label="10.1021/acs.chemrev.9b00055" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.chemrev.9b00055" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 72" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1ajs7%2FF" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 72" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31244000" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 72" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1016.74011" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 72" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=End-point%20binding%20free%20energy%20calculation%20with%20MM%2FPBSA%20and%20MM%2FGBSA%3A%20Strategies%20and%20applications%20in%20drug%20design&amp;journal=Chem.%20Rev.&amp;doi=10.1021%2Facs.chemrev.9b00055&amp;volume=119&amp;pages=9478-9508&amp;publication_year=2019&amp;author=Wang%2CE" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Koitabashi, M. et al. Biodegradable plastic-degrading activity of various species of &lt;i&gt;P&lt;/i&gt;&lt;i&gt;araphoma&lt;/i&gt;. <i>J Oleo Sci</i> <b>65</b>, 621–627 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 73" data-doi="10.5650/jos.ess16067" data-track="click_references" data-track-action="article reference" data-track-item_id="10.5650/jos.ess16067" data-track-label="10.5650/jos.ess16067" data-track-value="article reference" href="https://doi.org/10.5650%2Fjos.ess16067" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 73" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1arsbfJ" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 73" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27321123" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 73" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0663.14033" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 73" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Biodegradable%20plastic-degrading%20activity%20of%20various%20species%20of%20%3Ci%3EP%3C%2Fi%3E%3Ci%3Earaphoma%3C%2Fi%3E&amp;journal=J%20Oleo%20Sci&amp;doi=10.5650%2Fjos.ess16067&amp;volume=65&amp;pages=621-627&amp;publication_year=2016&amp;author=Koitabashi%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Dsouza, G. C. et al. Fungal biodegradation of low-density polyethylene using consortium of <i>Aspergillus</i> species under controlled conditions. <i>Heliyon</i> <b>7</b>, e7008 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 74" data-doi="10.1016/j.heliyon.2021.e07008" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.heliyon.2021.e07008" data-track-label="10.1016/j.heliyon.2021.e07008" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.heliyon.2021.e07008" rel="nofollow noopener">Article</a> 
    <a aria-label="MATH reference 74" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1423.57045" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 74" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Fungal%20biodegradation%20of%20low-density%20polyethylene%20using%20consortium%20of%20Aspergillus%20species%20under%20controlled%20conditions&amp;journal=Heliyon&amp;doi=10.1016%2Fj.heliyon.2021.e07008&amp;volume=7&amp;publication_year=2021&amp;author=Dsouza%2CGC" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Choi, J. et al. Recent advances in microbial and enzymatic engineering for the biodegradation of micro- and nanoplastics. <i>RSC Adv.</i> <b>14</b>, 9943–9966 (2024).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 75" data-doi="10.1039/D4RA00844H" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1039/D4RA00844H" data-track-label="10.1039/D4RA00844H" data-track-value="article reference" href="https://doi.org/10.1039%2FD4RA00844H" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 75" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2024RSCAd..14.9943C" rel="nofollow noopener">ADS</a> 
    <a aria-label="CAS reference 75" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB2cXms1WltL8%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 75" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=38528920" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 75" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10961967" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 75" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1155.37044" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 75" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Recent%20advances%20in%20microbial%20and%20enzymatic%20engineering%20for%20the%20biodegradation%20of%20micro-%20and%20nanoplastics&amp;journal=RSC%20Adv.&amp;doi=10.1039%2FD4RA00844H&amp;volume=14&amp;pages=9943-9966&amp;publication_year=2024&amp;author=Choi%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Huang, Q.-S. et al. Accelerated biodegradation of polyethylene terephthalate by <i>Thermobifida fusca</i> cutinase mediated by Stenotrophomonas pavanii. <i>Sci. Total Environ.</i> <b>808</b>, 152107 (2022).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 76" data-doi="10.1016/j.scitotenv.2021.152107" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.scitotenv.2021.152107" data-track-label="10.1016/j.scitotenv.2021.152107" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.scitotenv.2021.152107" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 76" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXislGju73P" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 76" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34864034" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 76" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Accelerated%20biodegradation%20of%20polyethylene%20terephthalate%20by%20Thermobifida%20fusca%20cutinase%20mediated%20by%20Stenotrophomonas%20pavanii&amp;journal=Sci.%20Total%20Environ.&amp;doi=10.1016%2Fj.scitotenv.2021.152107&amp;volume=808&amp;publication_year=2022&amp;author=Huang%2CQ-S" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Scattolini Rimada, A. C., Hernández Rodríguez, L. I. &amp; González Idiarte, H. Identification of <i>Fusarium oxysporum</i> causing sweet potato rot (<i>Ipomoea batatas</i> L &amp; Lamb) in Uruguay. <i>Agrociencia</i> <b>24</b>,  e124 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Rodríguez-Grimaldo, J. E., M. González, G. &amp; M. Montoya, A. <i>Fusarium</i>: Un fitopatógeno Que Amenaza La Salud Humana. <i>Revista Ciencia UANL</i> <b>25</b>, 37–43 (2023).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Cighir, A. et al. <i>Fusarium</i> spp. in Human Disease: Exploring the Boundaries between Commensalism and Pathogenesis. <i>Life</i> <b>13</b>, 1440 (2023).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80."><p class="c-article-references__text" id="ref-CR80">Anderson, K. S. Detection of novel enzyme intermediates in PEP-utilizing enzymes. <i>Arch. Biochem. Biophys.</i> 433 47–58 (2005).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81."><p class="c-article-references__text" id="ref-CR81">Yoshida, S. et al. A bacterium that degrades and assimilates poly(ethylene terephthalate). <i>Science</i> <b>1979</b>(351), 1196–1199 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 81" data-doi="10.1126/science.aad6359" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1126/science.aad6359" data-track-label="10.1126/science.aad6359" data-track-value="article reference" href="https://doi.org/10.1126%2Fscience.aad6359" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 81" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2016Sci...351.1196Y" rel="nofollow noopener">ADS</a> 
    <a aria-label="MATH reference 81" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1492.62082" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 81" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20bacterium%20that%20degrades%20and%20assimilates%20poly%28ethylene%20terephthalate%29&amp;journal=Science&amp;doi=10.1126%2Fscience.aad6359&amp;volume=1979&amp;issue=351&amp;pages=1196-1199&amp;publication_year=2016&amp;author=Yoshida%2CS" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="82."><p class="c-article-references__text" id="ref-CR82">Maity, W., Maity, S., Bera, S. &amp; Roy, A. Emerging roles of PETase and MHETase in the biodegradation of plastic wastes. <i>Appl. Biochem. Biotechnol.</i> <b>193</b>, 2699–2716 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 82" data-doi="10.1007/s12010-021-03562-4" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1007/s12010-021-03562-4" data-track-label="10.1007/s12010-021-03562-4" data-track-value="article reference" href="https://link.springer.com/doi/10.1007/s12010-021-03562-4" rel="noopener">Article</a> 
    <a aria-label="CAS reference 82" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXotVWks7k%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 82" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33797026" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 82" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1464.35188" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 82" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Emerging%20roles%20of%20PETase%20and%20MHETase%20in%20the%20biodegradation%20of%20plastic%20wastes&amp;journal=Appl.%20Biochem.%20Biotechnol.&amp;doi=10.1007%2Fs12010-021-03562-4&amp;volume=193&amp;pages=2699-2716&amp;publication_year=2021&amp;author=Maity%2CW&amp;author=Maity%2CS&amp;author=Bera%2CS&amp;author=Roy%2CA" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="83."><p class="c-article-references__text" id="ref-CR83">dos Santos, A. M., da Costa, C. H. S., Silva, P. H. A., Skaf, M. S. &amp; Lameira, J. Exploring the reaction mechanism of polyethylene terephthalate biodegradation through QM/MM approach. <i>J. Phys. Chem. B</i> <b>128</b>, 7486–7499 (2024).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 83" data-doi="10.1021/acs.jpcb.4c02207" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jpcb.4c02207" data-track-label="10.1021/acs.jpcb.4c02207" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jpcb.4c02207" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 83" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=39072475" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 83" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC11317977" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 83" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Exploring%20the%20reaction%20mechanism%20of%20polyethylene%20terephthalate%20biodegradation%20through%20QM%2FMM%20approach&amp;journal=J.%20Phys.%20Chem.%20B&amp;doi=10.1021%2Facs.jpcb.4c02207&amp;volume=128&amp;pages=7486-7499&amp;publication_year=2024&amp;author=Santos%2CAM&amp;author=Costa%2CCHS&amp;author=Silva%2CPHA&amp;author=Skaf%2CMS&amp;author=Lameira%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="84."><p class="c-article-references__text" id="ref-CR84">Sevilla, M. E. et al. Degradation of PET bottles by an engineered <i>Ideonella sakaiensis</i> PETase. <i>Polymers</i> <b>15</b>, 1779 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 84" data-doi="10.3390/polym15071779" data-track="click_references" data-track-action="article reference" data-track-item_id="10.3390/polym15071779" data-track-label="10.3390/polym15071779" data-track-value="article reference" href="https://doi.org/10.3390%2Fpolym15071779" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 84" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXotVKmtLw%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 84" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37050393" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 84" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10098701" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 84" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1459.37093" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 84" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Degradation%20of%20PET%20bottles%20by%20an%20engineered%20Ideonella%20sakaiensis%20PETase&amp;journal=Polymers&amp;doi=10.3390%2Fpolym15071779&amp;volume=15&amp;publication_year=2023&amp;author=Sevilla%2CME" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="85."><p class="c-article-references__text" id="ref-CR85">Liu, B. et al. Protein crystallography and site-direct mutagenesis analysis of the poly(Ethylene terephthalate) hydrolase petase from Ideonella sakaiensis. <i>ChemBioChem</i> <b>19</b>, 1471–1475 (2018).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 85" data-doi="10.1002/cbic.201800097" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1002/cbic.201800097" data-track-label="10.1002/cbic.201800097" data-track-value="article reference" href="https://doi.org/10.1002%2Fcbic.201800097" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 85" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVaju77P" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 85" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29603535" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 85" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1391.81103" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 85" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein%20crystallography%20and%20site-direct%20mutagenesis%20analysis%20of%20the%20poly%28Ethylene%20terephthalate%29%20hydrolase%20petase%20from%20Ideonella%20sakaiensis&amp;journal=ChemBioChem&amp;doi=10.1002%2Fcbic.201800097&amp;volume=19&amp;pages=1471-1475&amp;publication_year=2018&amp;author=Liu%2CB" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="86."><p class="c-article-references__text" id="ref-CR86">Chen, S., Wu, Y., Su, L. &amp; Wu, J. Contribution of disulfide bond to the stability of <i>Thermobifida fusca</i> cutinase. <i>Food Biosci.</i> <b>27</b>, 6–10 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 86" data-doi="10.1016/j.fbio.2018.11.001" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.fbio.2018.11.001" data-track-label="10.1016/j.fbio.2018.11.001" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.fbio.2018.11.001" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 86" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlyis7fO" rel="nofollow noopener">CAS</a> 
    <a aria-label="MATH reference 86" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0745.41027" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 86" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Contribution%20of%20disulfide%20bond%20to%20the%20stability%20of%20Thermobifida%20fusca%20cutinase&amp;journal=Food%20Biosci.&amp;doi=10.1016%2Fj.fbio.2018.11.001&amp;volume=27&amp;pages=6-10&amp;publication_year=2019&amp;author=Chen%2CS&amp;author=Wu%2CY&amp;author=Su%2CL&amp;author=Wu%2CJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="87."><p class="c-article-references__text" id="ref-CR87">Tanasupawat, S., Takehana, T., Yoshida, S., Hiraga, K. &amp; Oda, K. <i>Ideonella sakaiensis</i> sp. nov, isolated from a microbial consortium that degrades poly(ethylene terephthalate). <i>Int. J. Syst. Evol. Microbiol.</i> <b>66</b>, 2813–2818 (2016).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 87" data-doi="10.1099/ijsem.0.001058" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1099/ijsem.0.001058" data-track-label="10.1099/ijsem.0.001058" data-track-value="article reference" href="https://doi.org/10.1099%2Fijsem.0.001058" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 87" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1aktrk%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 87" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27045688" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 87" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Ideonella%20sakaiensis%20sp.%20nov%2C%20isolated%20from%20a%20microbial%20consortium%20that%20degrades%20poly%28ethylene%20terephthalate%29&amp;journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;doi=10.1099%2Fijsem.0.001058&amp;volume=66&amp;pages=2813-2818&amp;publication_year=2016&amp;author=Tanasupawat%2CS&amp;author=Takehana%2CT&amp;author=Yoshida%2CS&amp;author=Hiraga%2CK&amp;author=Oda%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="88."><p class="c-article-references__text" id="ref-CR88">Palm, G. J. et al. Structure of the plastic-degrading <i>Ideonella sakaiensis</i> MHETase bound to a substrate. <i>Nat. Commun.</i> <b>10</b>, 1717 (2019).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 88" data-doi="10.1038/s41467-019-09326-3" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1038/s41467-019-09326-3" data-track-label="10.1038/s41467-019-09326-3" data-track-value="article reference" href="https://doi.org/10.1038%2Fs41467-019-09326-3" rel="nofollow noopener">Article</a> 
    <a aria-label="ADS reference 88" data-track="click_references" data-track-action="ads reference" data-track-item_id="link" data-track-label="link" data-track-value="ads reference" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2019NatCo..10.1717P" rel="nofollow noopener">ADS</a> 
    <a aria-label="PubMed reference 88" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30979881" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 88" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6461665" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 88" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1530.74039" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 88" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Structure%20of%20the%20plastic-degrading%20Ideonella%20sakaiensis%20MHETase%20bound%20to%20a%20substrate&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-09326-3&amp;volume=10&amp;publication_year=2019&amp;author=Palm%2CGJ" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="89."><p class="c-article-references__text" id="ref-CR89">Longhi, S. &amp; Cambillau, C. Structure-activity of cutinase, a small lipolytic enzyme. <i>Biochim. Biophys. Acta (BBA)Mol. Cell Biol. Lipids</i> <b>1441</b>, 185–196 (1999).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="90."><p class="c-article-references__text" id="ref-CR90">Matak, M. Y. &amp; Moghaddam, M. E. The role of short-range Cys171–Cys178 disulfide bond in maintaining cutinase active site integrity: A molecular dynamics simulation. <i>Biochem. Biophys. Res. Commun.</i> <b>390</b>, 201–204 (2009).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 90" data-doi="10.1016/j.bbrc.2009.09.073" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1016/j.bbrc.2009.09.073" data-track-label="10.1016/j.bbrc.2009.09.073" data-track-value="article reference" href="https://doi.org/10.1016%2Fj.bbrc.2009.09.073" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 90" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtlWgu7%2FJ" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 90" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19781526" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 90" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20role%20of%20short-range%20Cys171%E2%80%93Cys178%20disulfide%20bond%20in%20maintaining%20cutinase%20active%20site%20integrity%3A%20A%20molecular%20dynamics%20simulation&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;doi=10.1016%2Fj.bbrc.2009.09.073&amp;volume=390&amp;pages=201-204&amp;publication_year=2009&amp;author=Matak%2CMY&amp;author=Moghaddam%2CME" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="91."><p class="c-article-references__text" id="ref-CR91">Fährrolfes, R. et al. ProteinsPlus: A web portal for structure analysis of macromolecules. <i>Nucl. Acids Res.</i> <b>45</b>, W337–W343 (2017).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 91" data-doi="10.1093/nar/gkx333" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkx333" data-track-label="10.1093/nar/gkx333" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkx333" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 91" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28472372" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 91" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5570178" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 91" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=ProteinsPlus%3A%20A%20web%20portal%20for%20structure%20analysis%20of%20macromolecules&amp;journal=Nucl.%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkx333&amp;volume=45&amp;pages=W337-W343&amp;publication_year=2017&amp;author=F%C3%A4hrrolfes%2CR" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="92."><p class="c-article-references__text" id="ref-CR92">Schöning-Stierand, K. et al. ProteinsPlus: Interactive analysis of protein–ligand binding interfaces. <i>Nucl. Acids Res.</i> <b>48</b>, W48–W53 (2020).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 92" data-doi="10.1093/nar/gkaa235" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/nar/gkaa235" data-track-label="10.1093/nar/gkaa235" data-track-value="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkaa235" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 92" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32297936" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 92" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7319454" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="Google Scholar reference 92" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=ProteinsPlus%3A%20Interactive%20analysis%20of%20protein%E2%80%93ligand%20binding%20interfaces&amp;journal=Nucl.%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkaa235&amp;volume=48&amp;pages=W48-W53&amp;publication_year=2020&amp;author=Sch%C3%B6ning-Stierand%2CK" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="93."><p class="c-article-references__text" id="ref-CR93">Graef, J., Ehrt, C. &amp; Rarey, M. Binding site detection remastered: Enabling fast, robust, and reliable binding site detection and descriptor calculation with DoGSite3. <i>J. Chem. Inf. Model.</i> <b>63</b>, 3128–3137 (2023).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 93" data-doi="10.1021/acs.jcim.3c00336" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1021/acs.jcim.3c00336" data-track-label="10.1021/acs.jcim.3c00336" data-track-value="article reference" href="https://doi.org/10.1021%2Facs.jcim.3c00336" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 93" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3sXovFersbg%3D" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 93" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=37130052" rel="nofollow noopener">PubMed</a> 
    <a aria-label="Google Scholar reference 93" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Binding%20site%20detection%20remastered%3A%20Enabling%20fast%2C%20robust%2C%20and%20reliable%20binding%20site%20detection%20and%20descriptor%20calculation%20with%20DoGSite3&amp;journal=J.%20Chem.%20Inf.%20Model.&amp;doi=10.1021%2Facs.jcim.3c00336&amp;volume=63&amp;pages=3128-3137&amp;publication_year=2023&amp;author=Graef%2CJ&amp;author=Ehrt%2CC&amp;author=Rarey%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="94."><p class="c-article-references__text" id="ref-CR94">Diedrich, K., Graef, J., Schöning-Stierand, K. &amp; Rarey, M. GeoMine: Interactive pattern mining of protein–ligand interfaces in the Protein Data Bank. <i>Bioinformatics</i> <b>37</b>, 424–425 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 94" data-doi="10.1093/bioinformatics/btaa693" data-track="click_references" data-track-action="article reference" data-track-item_id="10.1093/bioinformatics/btaa693" data-track-label="10.1093/bioinformatics/btaa693" data-track-value="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtaa693" rel="nofollow noopener">Article</a> 
    <a aria-label="CAS reference 94" data-track="click_references" data-track-action="cas reference" data-track-item_id="link" data-track-label="link" data-track-value="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFGisbbI" rel="nofollow noopener">CAS</a> 
    <a aria-label="PubMed reference 94" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32735322" rel="nofollow noopener">PubMed</a> 
    <a aria-label="MATH reference 94" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?0875.92034" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 94" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=GeoMine%3A%20Interactive%20pattern%20mining%20of%20protein%E2%80%93ligand%20interfaces%20in%20the%20Protein%20Data%20Bank&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtaa693&amp;volume=37&amp;pages=424-425&amp;publication_year=2021&amp;author=Diedrich%2CK&amp;author=Graef%2CJ&amp;author=Sch%C3%B6ning-Stierand%2CK&amp;author=Rarey%2CM" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="95."><p class="c-article-references__text" id="ref-CR95">Magalhães, R. P., Cunha, J. M. &amp; Sousa, S. F. Perspectives on the role of enzymatic biocatalysis for the degradation of plastic PET. <i>Int. J. Mol. Sci.</i> <b>22</b>, 11257 (2021).</p><p class="c-article-references__links u-hide-print"><a aria-label="Article reference 95" data-doi="10.3390/ijms222011257" data-track="click_references" data-track-action="article reference" data-track-item_id="10.3390/ijms222011257" data-track-label="10.3390/ijms222011257" data-track-value="article reference" href="https://doi.org/10.3390%2Fijms222011257" rel="nofollow noopener">Article</a> 
    <a aria-label="PubMed reference 95" data-track="click_references" data-track-action="pubmed reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34681915" rel="nofollow noopener">PubMed</a> 
    <a aria-label="PubMed Central reference 95" data-track="click_references" data-track-action="pubmed central reference" data-track-item_id="link" data-track-label="link" data-track-value="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8540959" rel="nofollow noopener">PubMed Central</a> 
    <a aria-label="MATH reference 95" data-track="click_references" data-track-action="math reference" data-track-item_id="link" data-track-label="link" data-track-value="math reference" href="http://www.emis.de/MATH-item?1040.37039" rel="nofollow noopener">MATH</a> 
    <a aria-label="Google Scholar reference 95" data-track="click_references" data-track-action="google scholar reference" data-track-item_id="link" data-track-label="link" data-track-value="google scholar reference" href="http://scholar.google.com/scholar_lookup?&amp;title=Perspectives%20on%20the%20role%20of%20enzymatic%20biocatalysis%20for%20the%20degradation%20of%20plastic%20PET&amp;journal=Int.%20J.%20Mol.%20Sci.&amp;doi=10.3390%2Fijms222011257&amp;volume=22&amp;publication_year=2021&amp;author=Magalh%C3%A3es%2CRP&amp;author=Cunha%2CJM&amp;author=Sousa%2CSF" rel="nofollow noopener">
                    Google Scholar</a> 
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1038/s41598-024-84718-0?format=refman&amp;flavour=references" rel="nofollow">Download references<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-download-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>The authors would like to thank Fundação Amazônica de Amparo a Estudos e Pesquisas (FAPESPA) and Coordenação de Aperfeiçoamento de Pessoal de Nível Superior (CAPES) for their financial support (073/2023) to improve the quality of this research; Pro-Reitoria de Pesquisa e Pós-Graduação (PROPESP/UFPA) for the financial support (PAPQ/2023); the Laboratorio Nacional de Computação Científica (LNCC) and Centro de Computação de Alto Desempenho (CCAD), for providing the supercomputing facilities. J.R.A.S. and J.L. thanks the CNPq funding agency for providing the grants 304610/2023-2, 440053/2022-6, 402141/2023-7, 305182/2021-8 and 402311/2024-8.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Laboratório de Planejamento e Desenvolvimento de Fármacos, Instituto de Ciências Exatas e Naturais, Universidade Federal do Pará, Belém, Pará, 66075-110, Brazil</p><p class="c-article-author-affiliation__authors-list">Maycon Vinicius Damasceno de Oliveira, Carlos Gabriel da Silva de Souza, Cláudio Nahum Alves, José Rogério A. Silva, Anderson H. Lima &amp; Jerônimo Lameira</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Núcleo de Ecologia Aquática e Pesca (NEAP), Universidade Federal do Pará, Belém, Pará, 66075-110, Brazil</p><p class="c-article-author-affiliation__authors-list">Gabriel Calandrini</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Center for Computing in Engineering &amp; Sciences, IQ/UNICAMP, Campinas, Brazil</p><p class="c-article-author-affiliation__authors-list">Clauber Henrique Souza da Costa</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Catalysis and Peptide Research Unit, University of KwaZulu-Natal, Durban, 4000, South Africa</p><p class="c-article-author-affiliation__authors-list">José Rogério A. Silva</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Maycon_Vinicius_Damasceno-Oliveira-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Maycon Vinicius Damasceno de Oliveira</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Maycon%20Vinicius%20Damasceno%20de%20Oliveira" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Maycon%20Vinicius%20Damasceno%20de%20Oliveira" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Maycon%20Vinicius%20Damasceno%20de%20Oliveira%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Gabriel-Calandrini-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Gabriel Calandrini</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Gabriel%20Calandrini" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Gabriel%20Calandrini" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Gabriel%20Calandrini%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Clauber_Henrique_Souza-Costa-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Clauber Henrique Souza da Costa</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Clauber%20Henrique%20Souza%20da%20Costa" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Clauber%20Henrique%20Souza%20da%20Costa" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Clauber%20Henrique%20Souza%20da%20Costa%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Carlos_Gabriel-de_Souza-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Carlos Gabriel da Silva de Souza</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Carlos%20Gabriel%20da%20Silva%20de%20Souza" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Carlos%20Gabriel%20da%20Silva%20de%20Souza" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Carlos%20Gabriel%20da%20Silva%20de%20Souza%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Cl_udio_Nahum-Alves-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Cláudio Nahum Alves</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Cl%C3%A1udio%20Nahum%20Alves" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Cl%C3%A1udio%20Nahum%20Alves" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Cl%C3%A1udio%20Nahum%20Alves%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jos__Rog_rio_A_-Silva-Aff1-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">José Rogério A. Silva</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Jos%C3%A9%20Rog%C3%A9rio%20A.%20Silva" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jos%C3%A9%20Rog%C3%A9rio%20A.%20Silva" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jos%C3%A9%20Rog%C3%A9rio%20A.%20Silva%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Anderson_H_-Lima-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Anderson H. Lima</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Anderson%20H.%20Lima" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Anderson%20H.%20Lima" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Anderson%20H.%20Lima%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jer_nimo-Lameira-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Jerônimo Lameira</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" href="/search?author=Jer%C3%B4nimo%20Lameira" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - pubmed" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jer%C3%B4nimo%20Lameira" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" data-track="click" data-track-action="author link - scholar" data-track-label="link" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jer%C3%B4nimo%20Lameira%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>Conceptualization, J.L. and M.V.D.O.; methodology, M.V.D.O., G.C., C.H.S.C, C.G.S.S.  and J.L.; formal analysis, J.R.A.S.. M.V.D.O., G.C., C.H.S.C., C.G.S.S. and J.L.,; investigation, M.V.D.O., G.C., C.H.S.C., C.G.S.S.  ; software, M.V.D.O., G.C., C.H.S.C., C.G.S.S. ; data curation, writing—original draft preparation, M.V.D.O., C.N.A., J.R.A.S., A.H.L. and J.L.; writing, review and editing, M.V.D.O., C.N.A., J.R.A.S., A.H.L. and J.L.; visualization, M.V.D.O., C.N.A., J.R.A.S., A.H.L. and J.L.; project administration, J.L.; funding acquisition, J.L. All authors have read an.d agreed to the published version of the manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a href="mailto:gabriel.calandrini@icb.ufpa.br" id="corresp-c1">Gabriel Calandrini</a> or <a href="mailto:lameira@ufpa.br" id="corresp-c2">Jerônimo Lameira</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
<h3 class="c-article__sub-heading" id="FPar1">Competing interests</h3>
<p>The authors declare no competing interests.</p>
</div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><h3 class="c-article__sub-heading">Publisher’s note</h3><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Supplementary Information"><div class="c-article-section" id="Sec15-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec15">Supplementary Information</h2><div class="c-article-section__content" id="Sec15-content"><div data-test="supplementary-info"><div class="c-article-figshare-container" data-test="figshare-container" id="figshareContainer"></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-supp-info-image="" data-test="supp-info-link" data-track="click" data-track-action="view supplementary info" data-track-label="supplementary information." href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41598-024-84718-0/MediaObjects/41598_2024_84718_MOESM1_ESM.docx">Supplementary Information.</a></h3></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
<p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="http://creativecommons.org/licenses/by-nc-nd/4.0/" rel="license">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Evaluating%20cutinase%20from%20Fusarium%20oxysporum%20as%20a%20biocatalyst%20for%20the%20degradation%20of%20nine%20synthetic%20polymer&amp;author=Maycon%20Vinicius%20Damasceno%20de%20Oliveira%20et%20al&amp;contentID=10.1038%2Fs41598-024-84718-0&amp;copyright=The%20Author%28s%29&amp;publication=2045-2322&amp;publicationDate=2025-01-22&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY-NC-ND">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41598-024-84718-0" data-test="crossmark" data-track="click" data-track-action="Click Crossmark" data-track-label="link" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41598-024-84718-0" rel="noopener" target="_blank"><img alt="Check for updates. Verify currency and authenticity via CrossMark" height="81" loading="lazy" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>" width="57"/></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">de Oliveira, M.V.D., Calandrini, G., da Costa, C.H.S. <i>et al.</i> Evaluating cutinase from<i> Fusarium oxysporum</i> as a biocatalyst for the degradation of nine synthetic polymer.
                    <i>Sci Rep</i> <b>15</b>, 2887 (2025). https://doi.org/10.1038/s41598-024-84718-0</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-external="" data-track-label="link" href="https://citation-needed.springer.com/v2/references/10.1038/s41598-024-84718-0?format=refman&amp;flavour=citation" rel="nofollow">Download citation<svg aria-hidden="true" class="u-icon" focusable="false" height="16" role="img" width="16"><use xlink:href="#icon-eds-i-download-medium" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2024-06-30">30 June 2024</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2024-12-26">26 December 2024</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2025-01-22">22 January 2025</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41598-024-84718-0</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" data-track="click" data-track-action="get shareable link" data-track-external="" data-track-label="button" id="get-share-url" type="button">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" data-track="click" data-track-action="select share url" data-track-label="button" id="share-url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" data-track="click" data-track-action="copy share url" data-track-external="" data-track-label="button" id="copy-share-url" type="button">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><h3 class="c-article__sub-heading">Keywords</h3><ul class="c-article-subject-list"><li class="c-article-subject-list__subject"><span><a data-track="click" data-track-action="view keyword" data-track-label="link" href='/search?query=Cutinase&amp;facet-discipline="Science%2C%20Humanities%20and%20Social%20Sciences%2C%20multidisciplinary"'>Cutinase</a></span></li><li class="c-article-subject-list__subject"><span><a data-track="click" data-track-action="view keyword" data-track-label="link" href='/search?query=Polymer&amp;facet-discipline="Science%2C%20Humanities%20and%20Social%20Sciences%2C%20multidisciplinary"'>Polymer</a></span></li><li class="c-article-subject-list__subject"><span><a data-track="click" data-track-action="view keyword" data-track-label="link" href='/search?query=%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20&amp;facet-discipline="Science%2C%20Humanities%20and%20Social%20Sciences%2C%20multidisciplinary"'>
<i>Fusarium oxysporum</i>
</a></span></li><li class="c-article-subject-list__subject"><span><a data-track="click" data-track-action="view keyword" data-track-label="link" href='/search?query=Molecular%20dynamic&amp;facet-discipline="Science%2C%20Humanities%20and%20Social%20Sciences%2C%20multidisciplinary"'>Molecular dynamic</a></span></li><li class="c-article-subject-list__subject"><span><a data-track="click" data-track-action="view keyword" data-track-label="link" href='/search?query=Phylogentic&amp;facet-discipline="Science%2C%20Humanities%20and%20Social%20Sciences%2C%20multidisciplinary"'>Phylogentic</a></span></li></ul><div data-component="article-info-list"></div></div></div></div></div></section>
</div>
</div>
</article>
</main>
<aside aria-label="Article navigation" class="c-article-extras u-hide-print" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
<div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
<div class="c-pdf-download u-clear-both js-pdf-download">
<a class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-draft-ignore="true" data-readcube-pdf-url="true" data-test="download-pdf" data-track="content_download" data-track-action="download pdf" data-track-external="" data-track-label="link" data-track-type="article pdf download" download="" href="/articles/s41598-024-84718-0.pdf">
<span class="c-pdf-download__text">Download PDF</span>
<svg aria-hidden="true" class="u-icon" focusable="false" height="16" width="16"><use xlink:href="#icon-download"></use></svg>
</a>
</div>
</div>
<div class="c-reading-companion">
<div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
<div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
<div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu="">
<div class="c-ad c-ad--300x250">
<div class="c-ad__inner">
<p class="c-ad__label">Advertisement</p>
<div class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-gpt="" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41598-024-84718-0;doi=10.1038/s41598-024-84718-0;subjmeta=114,45,57,61,631,638,639;kwrd=Biochemistry,Biophysics,Biotechnology,Chemistry,Computational+biology+and+bioinformatics" data-gpt-unitpath="/285/scientific_reports/article" data-pa11y-ignore="" data-test="right-ad" id="div-gpt-ad-right-2">
<noscript>
<a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/scientific_reports/article&amp;sz=300x250&amp;c=837549006&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41598-024-84718-0%26doi%3D10.1038/s41598-024-84718-0%26subjmeta%3D114,45,57,61,631,638,639%26kwrd%3DBiochemistry,Biophysics,Biotechnology,Chemistry,Computational+biology+and+bioinformatics">
<img alt="Advertisement" data-test="gpt-advert-fallback-img" height="250" src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/scientific_reports/article&amp;sz=300x250&amp;c=837549006&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41598-024-84718-0%26doi%3D10.1038/s41598-024-84718-0%26subjmeta%3D114,45,57,61,631,638,639%26kwrd%3DBiochemistry,Biophysics,Biotechnology,Chemistry,Computational+biology+and+bioinformatics" width="300"/></a>
</noscript>
</div>
</div>
</div>
</div>
</div>
<div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
<div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
</div>
</div>
</aside>
</div>
<nav aria-labelledby="Explore-content" class="c-header__dropdown" data-test="Explore-content" data-track-component="nature-150-split-header" id="explore">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="Explore-content">Explore content</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="research articles" data-track-label="link" href="/srep/research-articles">
                                    Research articles
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="news &amp; comment" data-track-label="link" href="/srep/news-and-comment">
                                    News &amp; Comment
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="collections" data-track-label="link" href="/srep/collections">
                                    Collections
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="explore-nav-item" data-track="click" data-track-action="subjects" data-track-label="link" href="/srep/browse-subjects">
                                    Subjects
                                </a>
</li>
</ul>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="facebook" data-track-label="link" href="https://www.facebook.com/scientificreports">Follow us on Facebook
                            </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="twitter" data-track-label="link" href="https://twitter.com/SciReports">Follow us on Twitter
                            </a>
</li>
<li class="c-header__item c-header__item--hide-lg">
<a class="c-header__link" data-track="click" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=288" rel="nofollow">Sign up for alerts<svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
</a>
</li>
<li class="c-header__item c-header__item--hide-lg">
<a class="c-header__link" data-track="click" data-track-action="rss feed" data-track-label="link" href="https://www.nature.com/srep.rss">
<span>RSS feed</span>
</a>
</li>
</ul>
</div>
</nav>
<nav aria-labelledby="About-the-journal" class="c-header__dropdown" data-test="about-the-journal" data-track-component="nature-150-split-header" id="about-the-journal">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="About-the-journal">About the journal</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="about scientific reports" data-track-label="link" href="/srep/about">
                                    About Scientific Reports
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="contact" data-track-label="link" href="/srep/contact">
                                    Contact
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="journal policies" data-track-label="link" href="/srep/journal-policies">
                                    Journal policies
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="guide to referees" data-track-label="link" href="/srep/guide-to-referees">
                                    Guide to referees
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="calls for papers" data-track-label="link" href="/srep/calls-for-papers">
                                    Calls for Papers
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="editor's choice" data-track-label="link" href="/srep/editorschoice">
                                    Editor's Choice
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="journal highlights" data-track-label="link" href="/srep/highlights">
                                    Journal highlights
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="open access fees and funding" data-track-label="link" href="/srep/open-access">
                                    Open Access Fees and Funding
                                </a>
</li>
</ul>
</div>
</nav>
<nav aria-labelledby="Publish-with-us-label" class="c-header__dropdown" data-test="publish-with-us" data-track-component="nature-150-split-header" id="publish-with-us">
<div class="c-header__container">
<h2 class="c-header__heading c-header__heading--js-hide" id="Publish-with-us-label">Publish with us</h2>
<ul class="c-header__list c-header__list--js-stack">
<li class="c-header__item">
<a class="c-header__link" data-track="click" data-track-action="for authors" data-track-label="link" href="/srep/author-instructions">
                                    For authors
                                </a>
</li>
<li class="c-header__item">
<a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-action="manuscript author services" data-track-context="header publish with us dropdown menu" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
</li>
<li class="c-header__item c-header__item--keyline">
<a class="c-header__link" data-gtm-criteo="submit-manuscript" data-track="click_submit_manuscript" data-track-action="submit manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-external="" data-track-label="link (publish with us dropdown menu)" href="https://author-welcome.nature.com/41598">Submit manuscript<svg aria-hidden="true" focusable="false" height="18" role="img" viewbox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
</a>
</li>
</ul>
</div>
</nav>
<div class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header" id="search-menu">
<div class="c-header__container">
<h2 class="c-header__visually-hidden">Search</h2>
<form action="/search" autocomplete="off" class="c-header__search-form" data-test="inline-search" method="get" role="search">
<label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
<div class="c-header__search-layout c-header__search-layout--max-width">
<div>
<input class="c-header__input" id="keywords" name="q" required="" type="text" value=""/>
</div>
<div class="c-header__search-layout">
<div>
<label class="c-header__visually-hidden" for="results-from">Show results from</label>
<select class="c-header__select" id="results-from" name="journal">
<option selected="" value="">All journals</option>
<option value="srep">This journal</option>
</select>
</div>
<div>
<button class="c-header__search-button" type="submit">Search</button>
</div>
</div>
</div>
</form>
<div class="c-header__flush">
<a class="c-header__link" data-track="click" data-track-action="advanced search" data-track-label="link" href="/search/advanced">
                Advanced search
            </a>
</div>
<h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
<ul class="c-header__list">
<li><a class="c-header__link" data-track="click" data-track-action="explore articles by subject" data-track-label="link" href="/subjects">Explore articles by subject</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="find a job" data-track-label="link" href="/naturecareers">Find a job</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="guide to authors" data-track-label="link" href="/authors/index.html">Guide to authors</a></li>
<li><a class="c-header__link" data-track="click" data-track-action="editorial policies" data-track-label="link" href="/authors/editorial_policies/">Editorial policies</a></li>
</ul>
</div>
</div>
<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
<meta content="Springer Nature" itemprop="publisher"/>
<div class="u-mt-16 u-mb-16">
<div class="u-container">
<div class="u-display-flex u-flex-wrap u-justify-content-space-between">
<p class="c-meta u-ma-0 u-flex-shrink">
<span class="c-meta__item">
                    Scientific Reports (<i>Sci Rep</i>)
                </span>
<span class="c-meta__item">
<abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">2045-2322</span> (online)
    </span>
</p>
</div>
</div>
</div>
<div class="c-footer">
<div class="u-hide-print" data-track-component="footer">
<h2 class="u-visually-hidden">nature.com sitemap</h2>
<div class="c-footer__container">
<div class="c-footer__grid c-footer__group--separator">
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="about us" data-track-label="link" href="https://www.nature.com/npg_/company_info/index.html">About us</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="press releases" data-track-label="link" href="https://www.nature.com/npg_/press_room/press_releases.html">Press releases</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="press office" data-track-label="link" href="https://press.nature.com/">Press office</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="contact us" data-track-label="link" href="https://support.nature.com/support/home">Contact us</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Discover content</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="journals a-z" data-track-label="link" href="https://www.nature.com/siteindex">Journals A-Z</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="article by subject" data-track-label="link" href="https://www.nature.com/subjects">Articles by subject</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="protocols.io" data-track-label="link" href="https://www.protocols.io/">protocols.io</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature index" data-track-label="link" href="https://www.natureindex.com/">Nature Index</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link" href="https://www.nature.com/authors/editorial_policies">Nature portfolio policies</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="open access" data-track-label="link" href="https://www.nature.com/nature-research/open-access">Open access</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="reprints and permissions" data-track-label="link" href="https://www.nature.com/reprints">Reprints &amp; permissions</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="data research service" data-track-label="link" href="https://www.springernature.com/gp/authors/research-data">Research data</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="language editing" data-track-label="link" href="https://authorservices.springernature.com/language-editing/">Language editing</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="scientific editing" data-track-label="link" href="https://authorservices.springernature.com/scientific-editing/">Scientific editing</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature masterclasses" data-track-label="link" href="https://masterclasses.nature.com/">Nature Masterclasses</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="research solutions" data-track-label="link" href="https://solutions.springernature.com/">Research Solutions</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="librarian service and tools" data-track-label="link" href="https://www.springernature.com/gp/librarians/tools-services">Librarian service &amp; tools</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="librarian portal" data-track-label="link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal">Librarian portal</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="open research" data-track-label="link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions">Open research</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="Recommend to library" data-track-label="link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library">Recommend to library</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="advertising" data-track-label="link" href="https://partnerships.nature.com/product/digital-advertising/">Advertising</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="partnerships and services" data-track-label="link" href="https://partnerships.nature.com/">Partnerships &amp; Services</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="media kits" data-track-label="link" href="https://partnerships.nature.com/media-kits/">Media kits</a>
</li>
<li class="c-footer__item"><a class="c-footer__link" data-track-action="branded content" data-track-label="link" href="https://partnerships.nature.com/product/branded-content-native-advertising/">Branded
                        content</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Professional development</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature careers" data-track-label="link" href="https://www.nature.com/naturecareers/">Nature Careers</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature conferences" data-track-label="link" href="https://conferences.nature.com">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
</ul>
</div>
<div class="c-footer__group">
<h3 class="c-footer__heading u-mt-0">Regional websites</h3>
<ul class="c-footer__list">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature africa" data-track-label="link" href="https://www.nature.com/natafrica">Nature Africa</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature china" data-track-label="link" href="http://www.naturechina.com">Nature China</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature india" data-track-label="link" href="https://www.nature.com/nindia">Nature India</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature Italy" data-track-label="link" href="https://www.nature.com/natitaly">Nature Italy</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature japan" data-track-label="link" href="https://www.natureasia.com/ja-jp">Nature Japan</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="nature middle east" data-track-label="link" href="https://www.nature.com/nmiddleeast">Nature Middle East</a></li>
</ul>
</div>
</div>
</div>
<div class="c-footer__container">
<ul class="c-footer__links">
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="privacy policy" data-track-label="link" href="https://www.nature.com/info/privacy">Privacy
                Policy</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="use of cookies" data-track-label="link" href="https://www.nature.com/info/cookies">Use
                of cookies</a></li>
<li class="c-footer__item">
<button class="optanon-toggle-display c-footer__link" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link" onclick="javascript:;">Your privacy choices/Manage cookies
                </button>
</li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="legal notice" data-track-label="link" href="https://www.nature.com/info/legal-notice">Legal
                notice</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="accessibility statement" data-track-label="link" href="https://www.nature.com/info/accessibility-statement">Accessibility
                statement</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="terms and conditions" data-track-label="link" href="https://www.nature.com/info/terms-and-conditions">Terms &amp; Conditions</a></li>
<li class="c-footer__item"><a class="c-footer__link" data-track="click" data-track-action="california privacy statement" data-track-label="link" href="https://www.springernature.com/ccpa">Your US state privacy rights</a></li>
</ul>
</div>
</div>
<div class="c-footer__container">
<a class="c-footer__link" href="https://www.springernature.com/">
<img alt="Springer Nature" height="20" loading="lazy" src="/static/images/logos/sn-logo-white-ea63208b81.svg" width="200"/>
</a>
<p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>
</div>
<div aria-hidden="true" class="u-visually-hidden">
<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path d="M0 .74h56.72v55.24H0z" id="a"></path></defs><symbol id="icon-access" viewbox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewbox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewbox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewbox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewbox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewbox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewbox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewbox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewbox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewbox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewbox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewbox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewbox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewbox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewbox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewbox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewbox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewbox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewbox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewbox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewbox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewbox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewbox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewbox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewbox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewbox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewbox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewbox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewbox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewbox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewbox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewbox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewbox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewbox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewbox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewbox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewbox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewbox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewbox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewbox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewbox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewbox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewbox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewbox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewbox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewbox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewbox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewbox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewbox="0 0 56.69 56.69"><path d="M0 0h56.69v56.69H0z" style="fill:none"></path><clippath id="b"><use style="overflow:visible" xlink:href="#a"></use></clippath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewbox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewbox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewbox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewbox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewbox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewbox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewbox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-download-medium" viewbox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewbox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-mail-medium" viewbox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewbox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewbox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewbox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewbox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewbox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewbox="0 0 100 100"><path clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z" fill-rule="evenodd"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewbox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>
<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif c-site-messages--nature-briefing-transres" data-component-expirydays="30" data-component-id="nature-briefing-banner" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: TranslationalResearch">
<div class="c-site-messages__banner-large">
<div class="c-site-messages__close-container">
<button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: TranslationalResearch">
<svg aria-hidden="true" focusable="false" height="25px" version="1.1" viewbox="0 0 25 25" width="25px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<title>Close banner</title>
<defs></defs>
<g fill="none" fill-rule="evenodd" stroke="none" stroke-width="1">
<rect height="25" opacity="0" width="25" x="0" y="0"></rect>
<path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
</g>
</svg>
<span class="visually-hidden">Close</span>
</button>
</div>
<div class="c-site-messages__form-container">
<div class="grid grid-12 last">
<div class="grid grid-4">
<img alt="Nature Briefing: Translational Research" height="40" src="/static/images/logos/nature-briefing-logo-transres-white-1245a3c374.svg" width="213"/>
<p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing: Translational Research</em> newsletter — top stories in biotechnology, drug discovery and pharma.</p>
</div>
<div class="grid grid-8 last">
<form action="https://www.nature.com/briefing/translational_research" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: TranslationalResearch" method="post">
<input id="briefing-banner-signup-form-input-track-originReferralPoint" name="track_originReferralPoint" type="hidden" value="TransResBriefingBanner"/>
<input id="briefing-banner-signup-form-input-track-formType" name="track_formType" type="hidden" value="DirectEmailBanner"/>
<input id="gdpr_tick_banner" name="gdpr_tick" type="hidden" value="false"/>
<input id="marketing_input_banner" name="marketing" type="hidden" value="false"/>
<input id="marketing_tick_banner" name="marketing_tick" type="hidden" value="false"/>
<input id="brieferEntryPoint_banner" name="brieferEntryPoint" type="hidden" value="TransResBriefingBanner"/>
<label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>
<div class="nature-briefing-banner__email-wrapper">
<input class="nature-briefing-banner__email-input box-sizing text14" data-test-element="briefing-emailbanner-email-input" id="emailAddress" name="emailAddress" placeholder="e.g. jo.smith@university.ac.uk" required="" type="email" value=""/>
<input id="defaultNewsletter_banner" name="N:translational_research" type="hidden" value="true"/>
<button class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button" type="submit">Sign up</button>
</div>
<div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
<input class="nature-briefing-banner__checkbox-checkbox" data-test-element="briefing-emailbanner-gdpr-checkbox" id="gdpr-briefing-banner-checkbox" name="gdpr" required="" type="checkbox" value="true"/>
<label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
</div>
</form>
</div>
</div>
</div>
</div>
<div class="c-site-messages__banner-small">
<div class="c-site-messages__close-container">
<button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: TranslationalResearch">
<svg aria-hidden="true" focusable="false" height="25px" version="1.1" viewbox="0 0 25 25" width="25px" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<title>Close banner</title>
<defs></defs>
<g fill="none" fill-rule="evenodd" stroke="none" stroke-width="1">
<rect height="25" opacity="0" width="25" x="0" y="0"></rect>
<path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
</g>
</svg>
<span class="visually-hidden">Close</span>
</button>
</div>
<div class="c-site-messages__content text14">
<span class="c-site-messages--nature-briefing__strapline strong">Get what matters in translational research, free to your inbox weekly.</span>
<a class="nature-briefing__link text14 sans-serif" data-test-element="briefing-banner-link" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" href="/briefing/translational-research/?brieferEntryPoint=TransResBriefingBanner" rel="noreferrer noopener" target="_blank">Sign up for Nature Briefing: Translational Research
            </a>
</div>
</div>
</div>
<noscript>
<img alt="" height="0" hidden="" src="https://verify.nature.com/verify/nature.png" style="display: none" width="0"/>
</noscript>
<script async="" src="//content.readcube.com/ping?doi=10.1038/s41598-024-84718-0&amp;format=js&amp;last_modified=2025-01-22"></script>
</body>
</html>